<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of Clinical Microbiology"><meta name="citation_title" content="Sequencing of the rpoB Gene and Flanking Spacers for Molecular Identification of Acinetobacter Species "><meta name="citation_authors" content="Bernard La Scola, Vijay A. K. B. Gundi, Atieh Khamis, and  Didier Raoult"><meta name="citation_date" content="2006 March"><meta name="citation_issue" content="3"><meta name="citation_volume" content="44"><meta name="citation_firstpage" content="827"><meta name="citation_doi" content="10.1128/JCM.44.3.827-832.2006"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1393131&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1393131&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1393131"><meta name="citation_pmid" content="16517861"><meta name="DC.Title" content="Sequencing of the rpoB Gene and Flanking Spacers for Molecular Identification of Acinetobacter Species "><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Society for Microbiology (ASM)"><meta name="DC.Contributor" content="Bernard La Scola"><meta name="DC.Contributor" content="Vijay A. K. B. Gundi"><meta name="DC.Contributor" content="Atieh Khamis"><meta name="DC.Contributor" content="Didier Raoult"><meta name="DC.Date" content="2006 March"><meta name="DC.Identifier" content="10.1128/JCM.44.3.827-832.2006"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody9.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs4.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Sequencing of the rpoB Gene and Flanking Spacers for Molecular Identification of Acinetobacter Species </title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0J9jSoiV90Xf_VmttPYk0QuGIp5mppzyGTOuqVAzU&amp;reftype=other&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-jcm.gif" alt="Logo of jcm" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="J Clin Microbiol Subscriptions" coords="0,64,190,75" href="redirect3.cgi?&amp;&amp;auth=0zwjXaCyhPmCndiFxM5-RiqSgZiYNXcEz4MkNzkKC&amp;reftype=publisher&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://jcm.asm.org/subscriptions/" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="DEFAULT" alt="J Clin Microbiol Web Site" coords="0,0,500,75" href="redirect3.cgi?&amp;&amp;auth=00mO8EOKecl1_h495m_vS9S6gIdUwj-lQc_dFhk2t&amp;reftype=publisher&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://jcm.asm.org" target="pmc_ext" onclick="focuswin('pmc_ext')"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=81&amp;action=archive">J Clin Microbiol</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=128355">v.44(3); Mar 2006</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=1393131&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="#supplementary-material-sec" class="sidefm-supplink">Supplementary Material</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=1393131&amp;blobtype=pdf" class="sidefm-pmclink">PDF (217K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=128355" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=81&amp;action=archive" class="sidefm-pmclink">Archive</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=81&amp;action=homepage" class="sidefm-pmclink">Journal Homepage</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0HmcpRpWKMoDaDdRvAycyuvj5sP0v7GyuT9cLVAtM&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16517861">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=0FkM9ngtDTa6Xd-Ny0flmew07VfzCQjKvkL0xb5mw&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=16517861">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0N7fqERGKp98vq2f3E7sqqCCkP9eR4YOPfyuICk2I&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=16517861">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0lm1J4aHg-Rx67kpI7_hL03gJt3Hg51Mn4Ca_yAV0&amp;reftype=Gene&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Gene&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1393131&amp;dopt=pmc_gene">Gene</option><option value="redirect3.cgi?&amp;&amp;auth=0ZG4pOX8eqBfYKGUSRHxOSDDQjZQJaz91cdhlg5w0&amp;reftype=Nucleotide&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Nucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1393131&amp;dopt=pmc_nuccore">Nucleotide</option><option value="redirect3.cgi?&amp;&amp;auth=06g8fDXKzK3_XKqGIIkLHlzK4pH7JQK8YDK9dhQO_&amp;reftype=PopSet&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Popset&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1393131&amp;dopt=pmc_popset">PopSet</option><option value="redirect3.cgi?&amp;&amp;auth=0Z3Vm0Q0vjw1qk3kKrAIczlF1aGGSNt_d_bOHd554&amp;reftype=Protein&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Protein&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1393131&amp;dopt=pmc_protein">Protein</option><option value="redirect3.cgi?&amp;&amp;auth=0lOc1ScNFkQ25id1G4YG-bkQrBOKv46kSVfUGO5w8&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1393131&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0TilqoPUyQ28BENlrAdAjWBJ89ROFlJPx97dPw5NT&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=1393131">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0hs-Pel2qh1CZlsBfvk6aq5QYP-0y_aHwXbm7BkGk&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20La%20Scola+B%5Bauth%5D" class="sidefm-pmclink"> La Scola, B.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0Vwii4fkgSDKgJeD4g4OqmLRvPz7uSriti7nVik9C&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Gundi+VA%5Bauth%5D" class="sidefm-pmclink"> Gundi, V.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0dGpLaYWyq0uMjcu_D2-Yd1ziyeYn7KyMiet2o5nk&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Khamis+A%5Bauth%5D" class="sidefm-pmclink"> Khamis, A.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0vuo4AN_tf_MKvCCDl614Y6elYBu0fxmjzf8sxQO7&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Raoult+D%5Bauth%5D" class="sidefm-pmclink"> Raoult, D.</a></div></div></div></td><td rowspan="104" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-jcm.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">J Clin Microbiol. </span><span class="citation-publication-date">2006 March; </span><span class="citation-volume">44</span><span class="citation-issue">(3)</span><span class="citation-flpages">: 827–832. </span></div><div><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1128/JCM.44.3.827-832.2006.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC1393131</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 2006, American Society for Microbiology</div><div class="fm-title">Sequencing of the <em>rpoB</em> Gene and Flanking Spacers for Molecular Identification of <em>Acinetobacter</em> Species<sup><a href="#fn1" class="cite-reflink fn">&#x02020;</a></sup> </div><div class="fm-author contrib-group">Bernard La Scola,<sup></sup><sup>&#x0002a;</sup> Vijay A. K. B. Gundi,<sup></sup> Atieh Khamis,<sup></sup> and  Didier Raoult<sup></sup><div class="fm-affl">Unit&#x000e9; des Rickettsies, CNRS UMR 6020, Facult&#x000e9; de M&#x000e9;decine, Universit&#x000e9; de la M&#x000e9;diterrann&#x000e9;e, 27 Boulevard Jean Moulin, 13385 Marseille Cedex 05, France</div></div><div class="fm-footnote"></div><div class="fm-footnote" id="cor1"><sup>&#x0002a;</sup>Corresponding author. Mailing address: Unit&#x000e9; des Rickettsies, CNRS UPRESA 6020, Facult&#x000e9; de M&#x000e9;decine, Universit&#x000e9; de la M&#x000e9;diterrann&#x000e9;e, 27 Boulevard Jean Moulin, 13385 Marseille Cedex 5, France. Phone: 33 04 91 38 55 17. Fax: 33 04 91 83 03 90. E-mail: <span class="e_id440164">bernard.lascola/at/medecine.univ-mrs.fr</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id440164", '<a class="ext-reflink" href="' + reverseAndReplaceString('rf.srm-vinu.enicedem/ta/alocsal.dranreb:otliam', '/at/', '@') + '">' + reverseAndReplaceString('rf.srm-vinu.enicedem/ta/alocsal.dranreb', '/at/','@') + '</a>')}catch(e){}
                                //--></script>.</div><div class="fm-pubdate">Received October 17, 2005; Revised December 15, 2005; Accepted January 3, 2006.</div><div class="links-box"><div class="fm-footnote"><img src="corehtml/pmc/pmcgifs/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="tocrender.fcgi?artid=1393131&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419318" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419597" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420072" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420580" class="sidefm-pmclink" style="text-transform: uppercase;">Supplementary Material</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420636" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id440231">Abstract</div><div class="section-content"><!--article-meta--><em>Acinetobacter</em> species are defined on the basis of several phenotypic characters, results of DNA-DNA homology, and more recently, similarities or dissimilarities in 16S rRNA gene sequences. However, the 16S rRNA gene is not polymorphic enough to clearly distinguish all <em>Acinetobacter</em> species. We used an RNA polymerase &#x003b2;-subunit gene (<em>rpoB</em>)-based identification scheme for the delineation of species within the genus <em>Acinetobacter</em>, and towards that end, we determined the complete <em>rpoB</em> gene and flanking spacer (<em>rplL</em>-<em>rpoB</em> and <em>rpoB</em>-<em>rpoC</em>) sequences of the 17 reference strains of <em>Acinetobacter</em> species and 7 unnamed genomospecies. By using complete gene sequences (4,089 bp), we clearly separated all species and grouped them into different clusters. A phylogenetic tree constructed using these sequences was supported by bootstrap values higher than those obtained with 16S rRNA or the <em>gyrB</em> or <em>recA</em> gene. Four pairs of primers enabled us to amplify and sequence two highly polymorphic partial sequences (350 and 450 bp) of the <em>rpoB</em> gene. These and flanking spacers were designed and tested for rapid identification of the 17 reference strains of <em>Acinetobacter</em> species and 7 unnamed genomospecies. Each of these four variable sequences enabled us to delineate most species. Sequences of at least two polymorphic sequences should be used to distinguish <em>Acinetobacter grimontii</em>, <em>Acinetobacter junii</em>, <em>Acinetobacter baylyi</em>, and genomic species 9 from one another. Finally, 21 clinical isolates of <em>Acinetobacter baumannii</em> were tested for intraspecies relationships and assigned correctly to the same species by comparing the partial sequences of the <em>rpoB</em> gene and its flanking spacers.</div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id440231" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419318" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419597" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420072" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420580" class="sidefm-pmclink" style="text-transform: uppercase;">Supplementary Material</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420636" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id418728"> </div><div class="section-content"><em>Acinetobacter</em> species are ubiquitous in the environment and have emerged as important nosocomial pathogens (<a href="#r3" class="cite-reflink bibr">3</a>). A propensity to tolerate drying (<a href="#r29" class="cite-reflink bibr">29</a>, <a href="#r30" class="cite-reflink bibr">30</a>) and resistance to many commonly used antibiotics (<a href="#r28" class="cite-reflink bibr">28</a>) are key factors in enabling the organism to survive and spread in the nosocomial environment. Furthermore, <em>Acinetobacter</em> species may serve as reservoirs of antibiotic-resistant genes, particularly in hospital environments (<a href="#r22" class="cite-reflink bibr">22</a>).<p>The genus <em>Acinetobacter</em>, originally proposed by Brisou and Prevot (<a href="#r6" class="cite-reflink bibr">6</a>), comprises a collection of bacteria which show a great deal of phenotypic similarities. Currently, the genus <em>Acinetobacter</em> comprises many taxons based on hybridization groups and classified as genomovars and genomospecies. From an official taxonomic point of view, there are currently 17 <em>Acinetobacter</em> nomenspecies with standing in nomenclature. However, several other genomospecies have been proposed, mainly on the basis of DNA hybridization studies (<a href="#r7" class="cite-reflink bibr">7</a>, <a href="#r20" class="cite-reflink bibr">20</a>, <a href="#r21" class="cite-reflink bibr">21</a>, <a href="#r28" class="cite-reflink bibr">28</a>).</p><p>Delineation of species within the genus <em>Acinetobacter</em> is often problematic (<a href="#r22" class="cite-reflink bibr">22</a>). A number of recent studies have attempted to set up reliable identification schemes based on DNA homology, phenotypic characters, and comparison of cell envelope and outer membrane protein patterns, but such methods yield inconsistent results (<a href="#r1" class="cite-reflink bibr">1</a>, <a href="#r10" class="cite-reflink bibr">10</a>, <a href="#r11" class="cite-reflink bibr">11</a>, <a href="#r12" class="cite-reflink bibr">12</a>, <a href="#r23" class="cite-reflink bibr">23</a>). The phenotypic characters of <em>Acinetobacter</em> species are susceptible to the conditions under which they are cultured (<a href="#r10" class="cite-reflink bibr">10</a>). Members of this genus have a large amount of genes devoted to catabolic pathways (named by authors as an &#x0201c;archipelago of catabolic diversity&#x0201d;) that lead to adaptation to most substrates (<a href="#r2" class="cite-reflink bibr">2</a>). Thus, auxanograms have only limited usefulness. It is therefore no wonder that the recent edition of a widely consulted clinical microbiology manual has chosen to separate <em>Acinetobacter</em> species into only two groups, depending on their abilities to oxidize glucose and show hemolytic activities (<a href="#r24" class="cite-reflink bibr">24</a>).</p><p>The currently used 16S rRNA gene sequencing determinations have failed to distinguish closely related genomic species of <em>Acinetobacter</em> due to its extremely low polymorphic nature and cannot replace the standard DNA-DNA hybridization method (<a href="#r11" class="cite-reflink bibr">11</a>, <a href="#r32" class="cite-reflink bibr">32</a>). We experienced misidentification of <em>Acinetobacter baumannii</em> isolates as <em>Acinetobacter calcoaceticus</em> using 16S rRNA gene sequence comparison (<a href="#r16" class="cite-reflink bibr">16</a>) that was later correctly identified using internal transcribed spacer and recombinase A (<em>recA</em>) gene sequence analyses (<a href="#r18" class="cite-reflink bibr">18</a>). Gene sequences, including those of internal transcribed spacer (<a href="#r9" class="cite-reflink bibr">9</a>), gyrase B (<em>gyrB</em>), and <em>recA</em> genes, have been evaluated to delineate species within the genus <em>Acinetobacter</em> (<a href="#r15" class="cite-reflink bibr">15</a>, <a href="#r31" class="cite-reflink bibr">31</a>, <a href="#r33" class="cite-reflink bibr">33</a>). However, unavailability of the sequences of <em>gyrB</em> and <em>recA</em> genes for 10 recently described <em>Acinetobacter</em> species precludes the use of these gene sequences for comparing different species of <em>Acinetobacter</em> (<a href="#r7" class="cite-reflink bibr">7</a>, <a href="#r19" class="cite-reflink bibr">19</a>, <a href="#r20" class="cite-reflink bibr">20</a>).</p><p>Several studies have demonstrated the usefulness of RNA polymerase &#x003b2;-subunit (<em>rpoB</em>) gene sequences for the identification and taxonomic classification of various bacterial species, including <em>Bartonella</em>, spirochete, <em>Staphylococcus</em>, <em>Mycobacterium</em>, <em>Legionella</em>, <em>Streptococcus</em>, <em>Enterobacteriaceae</em>, <em>Afipia</em> and <em>Bosea</em>, <em>Corynebacterium</em>, <em>Pasteurellaceae</em>, <em>Streptomyces</em> and <em>Kitasatospora</em>, <em>Mycoplasma</em>, <em>Salmonella</em>, <em>Bacillus</em>, <em>Paenibacillus</em>, <em>Ehrlichia</em>, <em>Anaplasma</em>, <em>Neorickettsia</em>, and <em>Wolbachia</em> species (<a href="#r19" class="cite-reflink bibr">19</a>). In one study, of the seven genes tested, the <em>rpoB</em> gene had the highest discriminatory power in delineating <em>Bartonella</em> species (<a href="#r17" class="cite-reflink bibr">17</a>). To our knowledge, <em>rpoB</em> has not been evaluated for the delineation of <em>Acinetobacter</em> species. Moreover, the presence of only one copy of the <em>rpoB</em> gene in the genome of <em>Acinetobacter</em> sp. ADP1 was reported, which shows that the gene may be phylogenetically reliable (<a href="#r2" class="cite-reflink bibr">2</a>).</p><p>In this study, we investigated the usefulness of <em>rpoB</em> gene sequencing for the differentiation and identification of 24 <em>Acinetobacter</em> species. Moreover, we reported herein four highly variable regions, including two polymorphic areas in the <em>rpoB</em> gene and two flanking spacers bordered by conserved sequences, with the objective of designing universal primers for amplification of a small but discriminative sequence for routine molecular identification of <em>Acinetobacter</em> species.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id440231" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>MATERIALS AND METHODS</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419597" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420072" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420580" class="sidefm-pmclink" style="text-transform: uppercase;">Supplementary Material</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420636" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id419318">MATERIALS AND METHODS</div><div class="section-content"><p><span class="head2">Bacterial strains. </span> A total of 24 <em>Acinetobacter</em> strains representing type strains of currently accepted species were obtained from the Collection de l'Institut Pasteur (CIP) and examined in the present study (Table <a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a>). Additionally, 21 isolates of <em>A. baumannii</em>, including the louse strain of La Scola and Raoult (<a href="#r16" class="cite-reflink bibr">16</a>, <a href="#r18" class="cite-reflink bibr">18</a>), were also studied (see Table S1 in the supplemental material). All strains were grown on Columbia agar plates with 5% sheep blood (Trypticase soy agar; bioM&#x000e9;rieux, Marcy-l'Etoile, France). These culture plates were incubated for 48 h at 37&#x000b0;C under aerobic conditions.<div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="t1" name="t1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="TABLE 1." title="TABLE 1."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>TABLE 1.</strong></a><div class="figure-table-caption-in-article"><span><em>Acinetobacter</em> strains used in the present study</span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span class="head2">Amplification and sequencing of the <em>rpoB</em> gene. </span> The sequences of <em>rplL</em>, <em>rpoB</em>, and <em>rpoC</em> of <em>Acinetobacter</em> sp. ADP1 (GenBank accession number NC005966) and <em>Pseudomonas syringae</em> pv. tomato strain DC3000 (GenBank accession number NC 006347) had been aligned to design primers that were subsequently used to amplify the complete <em>rpoB</em> gene and flanking spacers. Additional primers were designed as required. This procedure as well as the procedures for DNA extraction, PCR amplification, and sequencing has been detailed elsewhere (<a href="#r14" class="cite-reflink bibr">14</a>).</p><p><span class="head2">Analysis of complete <em>rpoB</em> sequences. </span> The nucleotide sequences of the target <em>rpoB</em> gene fragments obtained were processed into sequence data using standard software (Sequence Analysis Software; Applied Biosystems). The partially overlapping sequences encompassing the complete <em>rpoB</em> gene were combined into a single consensus sequence using sequence assembler software (Applied Biosystems). Multiple <em>rpoB</em> gene sequence alignments among all 24 species were made using the Clustal W program (<a href="#r26" class="cite-reflink bibr">26</a>) on the EMBL-EBI web server (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=06MrRof9G28U-ao7RDS0VxFnZelulCC9H14f8X509&amp;reftype=extlink&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.ebi.ac.uk/clustalw/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.ebi.ac.uk/clustalw/</a></span>).<p>The phylogenetic relationship between complete sequences of the <em>rpoB</em> gene and two spacers was estimated by the neighbor-joining method (<a href="#r8" class="cite-reflink bibr">8</a>). Phylogenetic trees were constructed, and their node reliabilities were estimated by performing bootstrap replicates. Bootstrap values were obtained from 1,000 trees generated randomly using SEQBOOT (PHYLIP software package). Proportional similarities of partial and complete <em>rpoB</em> sequences and spacers (<em>rplL</em>-<em>rpoB</em> and <em>rpoB</em>-<em>rpoC</em>) among species were computed using the MEGA 2.1 program.</p></p><p><span class="head2">Determination of discriminative partial sequences of the <em>rpoB</em> gene and flanking spacers. </span> SVARAP (for <em>S</em>equence <em>VAR</em>iability <em>A</em>nalysis <em>P</em>rogram; hypertext link &#x0201c;T&#x000e9;l&#x000e9;chargements&#x0201d;) software was used (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0xgGv0y5cD9IOWmcGTPJ4PURLj48WIaksK0qf1krx&amp;reftype=extlink&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://ifr48.free.fr/recherche/jeu_cadre/jeu_rickettsie.html" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://ifr48.free.fr/recherche/jeu_cadre/jeu_rickettsie.html</a></span>) to find the portions of sequences with high variability bordering highly conserved regions. These short fragments showing high variability were amplified and sequenced in all strains, including all isolates of <em>A. baumannii</em>. The primers used and their targets were as listed in Table <a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a>. PCRs were performed as described previously (<a href="#r14" class="cite-reflink bibr">14</a>), with the exception of primer annealing (48&#x000b0;C for 30 s) and extension (72&#x000b0;C for 1 min).<div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="t2" name="t2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t2" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="TABLE 2." title="TABLE 2."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>TABLE 2.</strong></a><div class="figure-table-caption-in-article"><span>Primers used for amplification and sequencing of zones 1 and 2 of the <em>rpoB</em> gene and flanking spacers of <em>Acinetobacter</em></span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span class="head2">Nucleotide sequence accession numbers. </span> GenBank accession numbers for sequences of the <em>rpoB</em> gene and flanking spacers obtained in the present study are listed in Table <a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a>.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id440231" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419318" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>RESULTS</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420072" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420580" class="sidefm-pmclink" style="text-transform: uppercase;">Supplementary Material</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420636" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id419597">RESULTS</div><div class="section-content"><p><span class="head2">Complete <em>rpoB</em> sequences of <em>Acinetobacter</em> species. </span> The primers, designed as described above, amplified specific regions of <em>rpoB</em> in all the strains tested. In all species tested, the total size of the <em>rpoB</em> gene was 4,089 bp. Pairwise comparison of the percent similarities of DNA of <em>rpoB</em> genes of various strains tested varied between 83 and 99% (see Table S2 in the supplemental material). Only two pairs of species, <em>Acinetobacter junii</em>/<em>Acinetobacter grimontii</em> and <em>Acinetobacter baylyi</em>/genomic species 11, showed 99% DNA homology. The others showed less than 95% DNA homology when pairwise comparisons were made.</p><p><span class="head2">Identification of <em>Acinetobacter</em> species based on partial sequences. </span> Using SVARAP software, we have identified two variable areas bordered by highly conserved regions: zone 1, between positions 2900 and 3250, and zone 2, between positions 3250 and 3700 (Fig. <a href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1</a>). The primers targeting these variable areas were designed. Using the primers, as indicated above, we amplified zones 1 and 2 of all reference strains and all isolates of <em>A. baumannii</em>. The sizes of the variable zone 1 and zone 2 were 350 and 450 bp, respectively. The percent similarities of zone 1 varied between 78.6 and 99.1%. In a pairwise comparison of DNA homology of the complete <em>rpoB</em> gene, <em>A. baylyi</em>/genomic species 11 and <em>A. junii</em>/<em>A. grimontii</em> showed proportional similarities of 98 and 99.1%, respectively. In the remaining strains tested, such homology was &#x0003c;96%. The percent similarity of zone 2 ranged between 75.8 and 99.6%. Again, in a pairwise comparison, the species pairs <em>A. junii</em>/<em>A. grimontii</em> and <em>A. baylyi</em>/genomic species 11 showed percent similarities of 98.8 and 99.6%, respectively. In contrast, in other species tested, such percent similarity in a pairwise comparison, as indicated above, was 96%. Overall, the percentage of sequence variability in the partial <em>rpoB</em> gene ranged between 0.4 and 24.2%. In contrast, in the complete <em>rpoB</em> gene sequence, the percentage of such variability ranged between 0.8 and 16.9%. The two partial sequences unambiguously established the identities of all 24 strains of <em>Acinetobacter</em>.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="f1" name="f1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1393131&amp;blobname=zjm0030662930001.gif" border="1" class="icon-reflink" alt="FIG. 1." title="FIG. 1."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>FIG. 1.</strong></a><div class="figure-table-caption-in-article"><span>Graphical representation of range site variability in <em>rpoB</em> gene and spacer sequences of <em>Acinetobacter</em> species in the present study using SVARAP software. The <em>x</em> axis indicates the positions of nucleotides, and the <em>y</em> axis indicates the percent variabilities</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>Intraspecies similarities in the partial <em>rpoB</em> gene sequence were observed among 21 clinical isolates of <em>A. baumannii</em>. In a pairwise comparison, the strain CIP 103655 showed between 94.9 and 95.7% similarity of zone 1 sequences with other strains tested. The other strains tested showed between 98.3 and 100% similarity of zone 1 sequences among one another. For zone 2 sequences, there was between 98.7 and 100% similarity when two strains other than strain CIP 103655 were compared. Compared with other strains, zone 2 sequences of strain CIP 103655 showed between 93.6 and 94.4% similarity with those of other strains. Two species were found to be the closest to <em>A. baumannii</em>. These included genospecies 3 (percent similarity, 95.1%) and <em>A. calcoaceticus</em> (percent similarity, 93.6%) for zone 1 and zone 2, respectively. Except for that of the CIP 103655 strain, the percent similarities in zone 1 and 2 sequences between <em>A. baumannii</em> and its closest species were lower than the intraspecies similarities. For CIP 103655, the most distant isolate of <em>A. baumannii</em> was 94.9% and 93.6% similar in zone 1 and zone 2, respectively.</p></p><p><span class="head2">Analysis of flanking spacers of the <em>rpoB</em> gene. </span> The lengths of two spacers (<em>rplL</em>-<em>rpo</em>B and <em>rpoB</em>-<em>rpoC</em> spacers) varied from species to species. The sizes of these spacers were as listed in Table <a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> and Table S1. The <em>rplL</em>-<em>rpoB</em> spacer (size, 301 to 310 bp) showed between 80.8 and 100% sequence similarity among various species studied herein. As was the case with partial and complete <em>rpoB</em> sequences, <em>A. junii</em> and <em>A. grimontii</em> showed 100% similarity in <em>rplL</em>-<em>rpoB</em> spacer sequences. Between <em>A. baylyi</em>/genomic species 11 and <em>Acinetobacter lwoffii</em>/genomic species 9, there was 98.4 to 99.7% similarity in <em>rplL</em>-<em>rpoB</em> spacer sequences.<p>The <em>rpoB</em>-<em>rpoC</em> spacer (size, 86 to 177 bp) sequences showed 70.2 to 99.5% similarity among various species studied herein. <em>A. junii</em> and <em>A. grimontii</em> showed 99.5% similarity in <em>rpoB</em>-<em>rpoC</em> spacer sequences. Among one another, the members (<em>A. calcoaceticus</em>, <em>A. baumannii</em>, and genomic species 3) of the so-called <em>A. calcoaceticus</em>/<em>A. baumannii</em> complex showed 98.5 to 99.0% similarity in <em>rpoB</em>-<em>rpoC</em> spacer sequences. In contrast, there were only 83.8 and 87.9% similarities in <em>rpoB</em>-<em>rpoC</em> spacer sequences between <em>A. baylyi</em>/genomic species 11 and <em>A. lwoffii</em>/genomic species 9, respectively.</p><p>The size of the <em>rplL</em>-<em>rpoB</em> spacer in strain CIP 103655 (size, 304 bp) was 1 base pair shorter than that found in other strains of <em>A. baumannii</em>. However, there was no difference among various <em>A. baumannii</em> strains tested in terms of the size (86 bp) of the <em>rpoB</em>-<em>rpoC</em> spacer. The strain CIP 103655 showed 96.1 to 96.4% similarity in the <em>rplL</em>-<em>rpoB</em> spacer sequence with other strains. The remaining strains of <em>A. baumannii</em> tested showed 99 to 100% similarity in the <em>rplL</em>-<em>rpoB</em> spacer sequence among one another. Compared in a pair, there was 97.7 to 98.8% similarity in <em>rpoB</em>-<em>rpoC</em> spacer sequences between strain CIP103655 and other <em>A. baumannii</em> strains. For other strains of <em>A. baumannii</em>, intrastrain similarities in <em>rpoB</em>-<em>rpoC</em> were 100%. The species found to be the closest to <em>A. baumannii</em> were genospecies 3 for <em>rplL</em>-<em>rpoB</em> (similarity, 95.9%) and <em>A. calcoaceticus</em> for <em>rpoB</em>-<em>rpoC</em> (similarity, 98.5%). Except for that of CIP 103655, these similarities were clearly lower than the intraspecies similarities observed among other strains of <em>A. baumannii</em> tested.</p></p><p><span class="head2">Phylogenetic analysis of <em>Acinetobacter</em> species. </span> A phylogenetic tree was constructed from complete <em>rpoB</em> gene sequences using the neighbor-joining method. The phylogenetic tree so constructed was supported by high bootstrap values (Fig. <a href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f2" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">2</a>). Compared to that obtained by analyzing 16S rRNA genes (7 of 22), a bootstrap value of &#x02265;75% was obtained significantly more frequently when complete <em>rpoB</em> genes of the strains (17 of 22) had been analyzed (<em>P</em> &#x0003c; 0.01). All the species were clearly separated into different groups. The tree based on partial <em>rpoB</em> (concatenated zone 1 and zone 2) showed homogeneous grouping of <em>A. baumannii</em> (Fig. <a href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f3" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">3</a>). Although the strain 103655 remained distinctly apart from other isolates of <em>A. baumannii</em>, it appeared in the same cluster. This clustering was supported by a bootstrap value of 85%.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="f2" name="f2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1393131&amp;blobname=zjm0030662930002.gif" border="1" class="icon-reflink" alt="FIG. 2." title="FIG. 2."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>FIG. 2.</strong></a><div class="figure-table-caption-in-article"><span>Dendrogram representing phylogenetic relationships of <em>Acinetobacter</em> species using the neighbor-joining method. The tree was derived from the alignment of complete <em>rpoB</em> gene sequences. The support of each branch, as determined from 1,000 bootstrap samples,</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="f3" name="f3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1393131&amp;blobname=zjm0030662930003.gif" border="1" class="icon-reflink" alt="FIG. 3." title="FIG. 3."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>FIG. 3.</strong></a><div class="figure-table-caption-in-article"><span>Dendrogram representing phylogenetic relationships of <em>Acinetobacter</em> species and intraspecies relationships of <em>A. baumannii</em> strains using the neighbor-joining method. The tree was derived from the alignment of concatenated zone 1 and zone 2 of the partial</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1393131&amp;rendertype=figure&amp;id=f3" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id440231" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419318" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419597" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>DISCUSSION</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420580" class="sidefm-pmclink" style="text-transform: uppercase;">Supplementary Material</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420636" class="sidefm-pmclink" style="text-transform: uppercase;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id420072">DISCUSSION</div><div class="section-content"><p>The interspecies relationships in the genus <em>Acinetobacter</em> need further definition. This may be understandable, considering the fact that many genospecies are very similar phenotypically and, therefore, are not readily distinguishable (<a href="#r10" class="cite-reflink bibr">10</a>). The increasing pathogenic importance of <em>Acinetobacter</em> has stimulated the development of reliable identification methods for these strains. Although considered the gold standard, the DNA-DNA hybridization method is time consuming and labor intensive (<a href="#r25" class="cite-reflink bibr">25</a>). Sequencing of housekeeping genes in general and protein-encoding genes in particular and sequencing of 16S DNA have been used in the identification of species or strains and delineation of phylogenetic relationships among different strains and species within the genus <em>Acinetobacter</em>. Compared to 16S rRNA genes, housekeeping genes provide a higher degree of resolution as the latter genes evolve faster than the former (<a href="#r32" class="cite-reflink bibr">32</a>). It may not therefore be surprising that various protein-encoding genes, including <em>recA</em>, <em>groEL</em>, <em>hsp</em>75, <em>rpoB</em>, <em>rpoD</em>, and <em>gyrB</em>, have been used for the classification of several unrelated bacteria at the intrageneric level (<a href="#r13" class="cite-reflink bibr">13</a>, <a href="#r14" class="cite-reflink bibr">14</a>, <a href="#r15" class="cite-reflink bibr">15</a>, <a href="#r31" class="cite-reflink bibr">31</a>, <a href="#r32" class="cite-reflink bibr">32</a>, <a href="#r34" class="cite-reflink bibr">34</a>). Unfortunately, the <em>gyrB</em> and <em>recA</em> gene sequences are not available for 10 recently described species of <em>Acinetobacter</em>. This makes it impossible to compare their performances with those of <em>rpoB</em> gene sequences in delineating various species of <em>Acinetobacter</em>.</p><p>Several investigators have sequenced <em>gyrB</em> and <em>recA</em> genes of 14 genomic species of <em>Acinetobacter</em> and studied them to delineate phylogenetic relationships among different members of the genus <em>Acinetobacter</em> (<a href="#r5" class="cite-reflink bibr">5</a>, <a href="#r15" class="cite-reflink bibr">15</a>, <a href="#r31" class="cite-reflink bibr">31</a>, <a href="#r33" class="cite-reflink bibr">33</a>). Furthermore, these investigators compared the performance of the gene sequence-based identification scheme, as indicated above, with that based on DNA-DNA hybridization (<a href="#r4" class="cite-reflink bibr">4</a>, <a href="#r27" class="cite-reflink bibr">27</a>). In our study, we constructed phylogenetic trees incorporating 14 genospecies of <em>Acinetobacter</em> and found no congruency among four individual phylogenetic trees constructed using the sequences of 16S rRNA, <em>rpoB</em>, <em>gyrB</em>, and <em>recA</em> genes (data not shown). However, a bootstrap value of &#x02265;75% was a statistically significant (<em>P</em> &#x0003c; 0.05) occurrence when a phylogenetic tree was based on <em>rpoB</em> (11 of 12) gene sequences compared to that based on either 16S rRNA (4 of 12), <em>gyr</em>B (5 of 12), or <em>recA</em> (6 of 12) gene sequences. This shows that the robustness of the phylogenetic tree was based on the <em>rpoB</em> gene sequence. <em>A. lwoffii</em> and <em>Acinetobacter</em> genomic species 9 were shown to be 100% identical when their partial <em>gyrB</em> gene sequences had been analyzed and compared. However, sequencing of <em>gyrD</em> and <em>recA</em> genes (<a href="#r15" class="cite-reflink bibr">15</a>, <a href="#r33" class="cite-reflink bibr">33</a>) separated them. In the present study, these two species were clearly differentiated as they showed variability between 4.9 and 6.7% in their complete <em>rpoB</em> gene sequence and two partial sequences of the <em>rpoB</em> gene. The sequence of the <em>rpoB</em> gene failed to clearly distinguish <em>A. baylyi</em> from genospecies 9 on one hand and <em>A. grimontii</em> from <em>A. baylyi</em> on the other. Unfortunately, the sequences of <em>gyrB</em> and <em>recA</em> genes are not available to use for such comparisons.</p><p>For the routine identification of <em>Acinetobacter</em>, either of two partial sequences and spacers can be used as they have high discriminative power and short lengths. However, such an identification scheme may not be sufficient to distinguish <em>A. grimontii</em> from <em>A. junii</em> and <em>A. baylyi</em> from <em>Acinetobacter</em> genomic species 9 (Table <a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>). One way of addressing this problem is to incorporate another short variable sequence into the identification scheme. In order to identify an <em>Acinetobacter</em> isolate, zone 1 should be determined first as it is the shortest and allows clear identification of 20 of 24 species. If the sequences of zone 1 in <em>A. grimontii</em> and <em>A. junii</em> are similar, then the sequences of zone 2 should be studied to make a clear distinction between these two species (Table <a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>). On the other hand, if the sequences of zone 1 in <em>A. baylyi</em> and genomic species 9 are similar, then it will be useful to determine the sequence of the <em>rpoB</em>-<em>rpoC</em> spacer as it may make a clear distinction between these two species (Table <a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>).</p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="t3" name="t3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t3" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="TABLE 3." title="TABLE 3."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1393131&amp;rendertype=table&amp;id=t3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>TABLE 3.</strong></a><div class="figure-table-caption-in-article"><span>Range of percent similarity for partial sequences between most-closely related <em>Acinetobacter</em> strains</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>The intraspecies variability of the short discriminative fragments observed within the <em>A. baumannii</em> species showed that, with exception of CIP 103655, all isolates have clearly higher degrees of similarities than that observed between <em>A. baumannii</em> and its closest species. However, the similarities observed between <em>A. baumannii</em> CIP 103655 and other isolates of the genus suggest that the identification of some isolates in this species could be ambiguous. All in all, an identification scheme based on analyzing partial <em>rpoB</em> gene sequences and flanking spacers may prove useful in the molecular identification of <em>Acinetobacter</em>. However, due to possible intraspecies dissimilarities and interspecies high similarities, such a diagnostic scheme should be undertaken for a large collection of <em>Acinetobacter</em> isolates. Moreover, the status of some isolates (e.g., <em>A. baumannii</em> CIP 103655) or some species (e.g., <em>A. grimontii</em> and <em>A. baylyi</em>) should be assessed by DNA-DNA hybridization studies and analyses of the sequences of such housekeeping genes as <em>recA</em> and <em>gyrB</em>. The molecular identification scheme proposed herein will provide better genetic definition of large collections of strains representing named species and will then serve as the basis for a large-scale phenotypic study to determine better biochemical tests to separate groups, especially the group of <em>Acinetobacter</em> species isolated in humans.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id420580"><a id="supplementary-material-sec"></a>Supplementary Material</div><div class="section-content"><!--body--><div id="PMC_1">
<div><strong>[Supplemental material]</strong></div>
<div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1393131&amp;blobname=jcm_44_3_827__index.html">Click here to view.</a></blockquote></div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="footnotes">Footnotes</div><div class="section-content"><div class="fm-footnote" id="fn1"><sup>&#x02020;</sup>Supplemental material for this article may be found at <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=00mSOtv1m4JruatTyDEfkez8wewl0TehNz4G7h5Wp&amp;reftype=extlink&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://jcm.asm.org/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://jcm.asm.org/</a></span>.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id440231" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419318" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id419597" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420072" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id420580" class="sidefm-pmclink" style="text-transform: uppercase;">Supplementary Material</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>REFERENCES</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id420636">REFERENCES</div><div class="section-content"><div class="back-matter-section"><div class="ref-cit-blk" id="r1"><div class="ref-label">1.</div><div class="ref-cit"><strong>Alexander, M., F. Ismail, P. J. H. Jackman, and W. C. Nobel.</strong> 1984. Fingerprinting <em>Acinetobacter</em> strains from clinical sources by numerical analysis of electrophoretic patterns. <span class="ref-journal">J. Med. Microbiol.</span> <span class="ref-vol">18</span><strong>:</strong>55-68.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pMeg8vM5MON5ir4jKmrtqUCRqDRQVXEluGv4LQZE&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/6748041">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r2"><div class="ref-label">2.</div><div class="ref-cit"><strong>Barbe, V., D. Vallenet, N. Fonknechten, A. Kreimeyer, S. Oztas, L. Labarre, S. Cruveiller, C. Robert, S. Duprat, P. Wincker, L. N. Ornston, J. Weissenbach, P. Marliere, G. N. Cohen, and C. Medigue.</strong> 2004. Unique features revealed by the genome sequence of Acinetobacter sp. ADP1, a versatile and naturally transformation competent bacterium. <span class="ref-journal">Nucleic Acids Res.</span> <span class="ref-vol">32</span><strong>:</strong>5766-5779.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0v5baLz5NUS_H_TtXmyKhXcnZiazFWdI3XLRkxQH2&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15514110">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r3"><div class="ref-label">3.</div><div class="ref-cit"><strong>Bergogne-Berezin, E., and K. J. Towner.</strong> 1996. <em>Acinetobacter</em> spp. as nosocomial pathogens: microbiological, clinical, and epidemiological features. <span class="ref-journal">Clin. Microbiol. Rev.</span> <span class="ref-vol">9</span><strong>:</strong>148-165.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0xGNv580Es7-HKO2TAacmtDrWwBpoOi67YXqlak7c&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8964033">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r4"><div class="ref-label">4.</div><div class="ref-cit"><strong>Bouvet, P. J. M., and P. A. D. Grimont.</strong> 1986. Taxonomy of the genus <em>Acinetobacter</em> with the recognition of <em>Acinetobacter baumannii</em> sp. nov., <em>Acinetobacter haemolyticus</em> sp. nov., <em>Acinetobacter johnsonii</em> sp. nov., and <em>Acinetobacter junii</em> sp. nov. and emended descriptions of <em>Acinetobacter calcoaceticus</em> and <em>Acinetobacter lwoffii</em>. <span class="ref-journal">Int. J. Syst. Bacteriol.</span> <span class="ref-vol">36</span><strong>:</strong>228-240.</div></div><div class="ref-cit-blk" id="r5"><div class="ref-label">5.</div><div class="ref-cit"><strong>Bouvet, P. J. M., and P. A. D. Grimont.</strong> 1987. Identification and biotyping of clinical isolates of <em>Acinetobacter</em>. <span class="ref-journal">Ann. Inst. Pasteur Microbiol.</span> <span class="ref-vol">138</span><strong>:</strong>569-578.</div></div><div class="ref-cit-blk" id="r6"><div class="ref-label">6.</div><div class="ref-cit"><strong>Brisou, J., and A. R. Prevot.</strong> 1954. [Studies on bacterial taxonomy. X. The revision of species under Achromobacter group.] <span class="ref-journal">Ann. Inst. Pasteur (Paris)</span> <span class="ref-vol">86</span><strong>:</strong>722-728.</div></div><div class="ref-cit-blk" id="r7"><div class="ref-label">7.</div><div class="ref-cit"><strong>Carr, E. L., P. Kampfer, B. K. Patel, V. Gurtler, and R. J. Seviour.</strong> 2003. Seven novel species of Acinetobacter isolated from activated sludge. <span class="ref-journal">Int. J. Syst. Evol. Microbiol.</span> <span class="ref-vol">53</span><strong>:</strong>953-963.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Jss7lcE1d86yM_X4p7dGGrZI5VcAj5ZFsEMBZQf2&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12892111">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r8"><div class="ref-label">8.</div><div class="ref-cit"><strong>Felsenstein, J.</strong> 1989. PHYLIP&#x02014;phylogeny inference package (version 3.2). <span class="ref-journal">Cladistics</span> <span class="ref-vol">5</span><strong>:</strong>164-166.</div></div><div class="ref-cit-blk" id="r9"><div class="ref-label">9.</div><div class="ref-cit"><strong>Garcia, A., R. Montoya, H. Bello, G. Gonzalez, M. Dominguez, and R. Zemelman.</strong> 1996. Differentiation of <em>Acinetobacter baumannii</em> biotypes by amplification of 16S-23S rRNA intergenic spacer sequences. <span class="ref-journal">Microbios</span> <span class="ref-vol">88</span><strong>:</strong>159-167.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hcWStBn7oLBOEea9DK-kr3EmmHNLsC_zhwz--5dS&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9141712">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r10"><div class="ref-label">10.</div><div class="ref-cit"><strong>Gerner-Smidt, P., I. Tjernberg, and J. Ursing.</strong> 1991. Reliability of phenotypic tests for identification of <em>Acinetobacter</em> species. <span class="ref-journal">J. Clin. Microbiol.</span> <span class="ref-vol">29</span><strong>:</strong>277-282.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Xm92Cv4vNqmsL8-T91824aUwWeu8FeuJnDBgBkba&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/2007635">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r11"><div class="ref-label">11.</div><div class="ref-cit"><strong>Ibrahim, A., P. Gerner-Smidt, and W. Liesack.</strong> 1997. Phylogenetic relationship of the twenty-one DNA groups of the genus <em>Acinetobacter</em> as revealed by 16S ribosomal DNA sequence analysis. <span class="ref-journal">Int. J. Syst. Bacteriol.</span> <span class="ref-vol">47</span><strong>:</strong>837-841.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hqvKRo7wdPcF1Tw8t1pj9O13WTOMogifu2LPFkB5&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9226915">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r12"><div class="ref-label">12.</div><div class="ref-cit"><strong>Ino, T., and Y. Nishimura.</strong> 1989. Taxonomic studies of <em>Acinetobacter</em> species based on outer membrane protein patterns. <span class="ref-journal">J. Gen. Appl. Microbiol.</span> <span class="ref-vol">16</span><strong>:</strong>213-224.</div></div><div class="ref-cit-blk" id="r13"><div class="ref-label">13.</div><div class="ref-cit"><strong>Karlin, S., G. M. Weinstock, and V. Brendel.</strong> 1995. Bacterial classifications derived from RecA protein sequence comparisons. <span class="ref-journal">J. Bacteriol.</span> <span class="ref-vol">177</span><strong>:</strong>6881-6893.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FguXpx7kKtnzGPc4m9hgaE1I9NWRKT4Goeu_Fk8G&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7592482">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r14"><div class="ref-label">14.</div><div class="ref-cit"><strong>Khamis, A., D. Raoult, and B. La Scola.</strong> 2004. <em>rpoB</em> gene sequencing for identification of <em>Corynebacterium</em> species. <span class="ref-journal">J. Clin. Microbiol.</span> <span class="ref-vol">42</span><strong>:</strong>3925-3931.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=00W4PP2_6CRUFREFxKdyZo25N99hAi-_ZdrouFQa4&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15364970">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r15"><div class="ref-label">15.</div><div class="ref-cit"><strong>Krawczyk, B., K. Lewandowski, and J. Kur.</strong> 2002. Comparative studies of the <em>Acinetobacter</em> genus and the species identification method based on the <em>recA</em> sequences. <span class="ref-journal">Mol. Cell. Probes</span> <span class="ref-vol">16</span><strong>:</strong>1-11.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0lENQnYaqXQZdegTRK9pI69ipaKyNYOYcFvCYzANm&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12005442">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r16"><div class="ref-label">16.</div><div class="ref-cit"><strong>La Scola, B., P.-E. Fournier, P. Brouqui, and D. Raoult.</strong> 2001. Detection and culture of <em>Bartonella quintana</em>, <em>Serratia marcescens</em>, and <em>Acinetobacter</em> spp. from decontaminated human body lice. <span class="ref-journal">J. Clin. Microbiol.</span> <span class="ref-vol">39</span><strong>:</strong>1707-1709.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0B9bUUO2Z_siT4v5-RVcUma9MnndMvl-vcGndoQaR&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11325978">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r17"><div class="ref-label">17.</div><div class="ref-cit"><strong>La Scola, B., Z. Zeaiter, A. Khamis, and D. Raoult.</strong> 2003. Gene-sequence-based criteria for species definition in bacteriology: the Bartonella paradigm. <span class="ref-journal">Trends Microbiol.</span> <span class="ref-vol">11</span><strong>:</strong>318-321.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0b3q4ks_QACZlOXfzPHveS81Sy37h2g_GrsP4Z563&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12875815">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r18"><div class="ref-label">18.</div><div class="ref-cit"><strong>La Scola, B., and D. Raoult.</strong> 2004. <em>Acinetobacter baumannii</em> in human body louse. <span class="ref-journal">Emerg. Infect. Dis.</span> <span class="ref-vol">10</span><strong>:</strong>1671-1673.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0DwiVjtVds-eCuRH7WAEGtFnfsjHskaa1ibaIJkhn&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15498175">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r19"><div class="ref-label">19.</div><div class="ref-cit"><strong>Nemec, A., L. Dijkshoorn, I. Cleenwerck, T. De Baere, D. Janssens, T. J. Van Der Reijden, P. Jezek, and M. Vaneechoutte.</strong> 2003. <em>Acinetobacter parvus</em> sp. nov., a small-colony-forming species isolated from human clinical specimens. <span class="ref-journal">Int. J. Syst. Evol. Microbiol.</span> <span class="ref-vol">53</span><strong>:</strong>1563-1567.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0tWspJDu3n3b9TLOw2p1hHpodhdrnizi9DN0ucAWX&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/13130049">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r20"><div class="ref-label">20.</div><div class="ref-cit"><strong>Nemec, A., T. De Baere, I. Tjernberg, M. Vaneechoutte, T. J. van der Reijden, and L. Dijkshoorn.</strong> 2001. <em>Acinetobacter ursingii</em> sp. nov. and <em>Acinetobacter schindleri</em> sp. nov., isolated from human clinical specimens. <span class="ref-journal">Int. J. Syst. Evol. Microbiol.</span> <span class="ref-vol">51</span><strong>:</strong>1891-1899.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0v1gWmikpwZuVzo9YnArEdPCB_wxnybvTqd-dEA2B&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11594623">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r21"><div class="ref-label">21.</div><div class="ref-cit"><strong>Nishimura, Y., T. Ino, and H. Lizuka.</strong> 1988. <em>Acinetobacter radioresistens</em> sp. nov. isolated from cotton and soil. <span class="ref-journal">Int. J. Syst. Bacteriol.</span> <span class="ref-vol">38</span><strong>:</strong>209-211.</div></div><div class="ref-cit-blk" id="r22"><div class="ref-label">22.</div><div class="ref-cit"><strong>Nowak, A., and J. Kur.</strong> 1996. Differentiation of seventeen genospecies of <em>Acinetobacter</em> by multiplex polymerase chain reaction and restriction fragment length polymorphism analysis. <span class="ref-journal">Mol. Cell. Probes</span> <span class="ref-vol">10</span><strong>:</strong>405-411.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0DwIWHdyUkL11SnrkrueAxiX1oJcXOhgAGEvR5QGT&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9025077">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r23"><div class="ref-label">23.</div><div class="ref-cit"><strong>Rainey, F. A., E. Lang, and E. Stackebrandt.</strong> 1994. The phylogenetic structure of the genus <em>Acinetobacter</em>. <span class="ref-journal">FEMS Microbiol. Lett.</span> <span class="ref-vol">124</span><strong>:</strong>349-353.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0F_bBXlhhKlfDXK8HQS6dRjBjiMoScxPqiPEkEk3q&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7851741">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r24"><div class="ref-label">24.</div><div class="ref-cit"><strong>Schreckenberger, P. C., M. I. Daneshvar, R. S. Weyant, and D. G. Hollis.</strong> 2003. <em>Acinetobacter</em>, <em>Achromobacter</em>, <em>Chryseobacterium</em>, <em>Moraxella</em>, and other nonfermentative gram-negative rods, p. 749-779. <em>In</em> P. R. Murray, E. J. Baron, J. H. Jorgensen, M. A. Pfaller, and R. H. Yolken (ed.), <span class="ref-journal">Manual of clinical microbiology</span>, 8th ed. American Society for Microbiology, Washington, D.C.</div></div><div class="ref-cit-blk" id="r25"><div class="ref-label">25.</div><div class="ref-cit"><strong>Stackebrandt, E., and B. M. Goebel.</strong> 1994. Taxonomic note: a place for DNA-DNA reassociation and 16S rRNA sequence analysis in the present species definition in bacteriology. <span class="ref-journal">Int. J. Syst. Bacteriol.</span> <span class="ref-vol">44</span><strong>:</strong>846-849.</div></div><div class="ref-cit-blk" id="r26"><div class="ref-label">26.</div><div class="ref-cit"><strong>Thompson, J. D., D. G. Higgins, and T. J. Gibson.</strong> 1994. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. <span class="ref-journal">Nucleic Acids Res.</span> <span class="ref-vol">22</span><strong>:</strong>4673-4680.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0BM3RQV7d4-BUSYyF8MSQzhXCIqmfSMP2gOXECQgc&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7984417">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r27"><div class="ref-label">27.</div><div class="ref-cit"><strong>Tjernberg, I., and J. Ursing.</strong> 1989. Clinical strains of <em>Acinetobacter</em> classified by DNA-DNA hybridization. <span class="ref-journal">APMIS</span> <span class="ref-vol">97</span><strong>:</strong>595-605.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0x7Hl_Jx-Nuhq3aUWMd6wCEF3XwoIFErn7jZuZ5ru&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/2751895">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r28"><div class="ref-label">28.</div><div class="ref-cit"><strong>Van Dessel, H., L. Dijkshoorn, T. Van Der Reijden, N. Bakker, A. Paauw, P. Van Den Broek, J. Verhoef, and S. Brisse.</strong> 2004. Identification of a new geographically widespread multiresistant <em>Acinetobacter baumannii</em> clone from European hospitals. <span class="ref-journal">Res. Microbiol.</span> <span class="ref-vol">155</span><strong>:</strong>105-112.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0bijoPJ78ufEgUHj7gmmHfHwQCP3amdRQz1qjHA3L&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14990262">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r29"><div class="ref-label">29.</div><div class="ref-cit"><strong>Webster, C. A., M. Crow, H. Humphreys, and K. S. Towner.</strong> 1998. Surveillance of an adult intensive care unit for a long-term persistence of a multi-resistant strain of <em>Acinetobacter baumannii</em>. <span class="ref-journal">Eur. J. Clin. Microbiol. Infect. Dis.</span> <span class="ref-vol">17</span><strong>:</strong>171-176.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0diEptiNE5xTcbuTT7qAeFLTeTczA-lcrG0IGhAw8&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9665298">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r30"><div class="ref-label">30.</div><div class="ref-cit"><strong>Wendt, C., B. Dietz, E. Dietz, and H. Ruden.</strong> 1997. Survival of <em>Acinetobacter baumannii</em> on dry surfaces. <span class="ref-journal">J. Clin. Microbiol.</span> <span class="ref-vol">35</span><strong>:</strong>1394-1397.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=02O1BLetlKt2CqEOTlidyT3j5fcz59H4U_IJijQOS&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9163451">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r31"><div class="ref-label">31.</div><div class="ref-cit"><strong>Yamamoto, S., and S. Harayama.</strong> 1996. Phylogenetic analysis of Acinetobacter strains based on the nucleotide sequences of gyrB genes and on the amino acid sequences of their products. <span class="ref-journal">Int. J. Syst. Bacteriol.</span> <span class="ref-vol">46</span><strong>:</strong>506-511.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=04cHJqlfB_D2uTnpBtSb74I_6spQgp0o2Re8ZXQHt&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8934907">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r32"><div class="ref-label">32.</div><div class="ref-cit"><strong>Yamamoto, S., and S. Harayama.</strong> 1998. Phylogenetic relationships of <em>Pseudomonas putida</em> strains deduced from the nucleotide sequences of <em>gyrB</em>, <em>rpoD</em> and 16S rRNA genes. <span class="ref-journal">Int. J. Syst. Bacteriol.</span> <span class="ref-vol">48</span><strong>:</strong>813-819.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FI-a7b6Axp5v7uGT90YUNr5IcoFD8seIEqKB0kL_&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9734035">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r33"><div class="ref-label">33.</div><div class="ref-cit"><strong>Yamamoto, S., P. J. Bouvet, and S. Harayama.</strong> 1999. Phylogenetic structures of the genus Acinetobacter based on gyrB sequences: comparison with the grouping by DNA-DNA hybridization. <span class="ref-journal">Int. J. Syst. Bacteriol.</span> <span class="ref-vol">49</span><strong>:</strong>87-95.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Do1gIYjpU-bDkcpJELXgOiT97mZ8ExRDZzEXhQLN&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10028249">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="r34"><div class="ref-label">34.</div><div class="ref-cit"><strong>Zeaiter, Z., P. E. Fournier, H. Ogata, and D. Raoult.</strong> 2002. Phylogenetic classification of <em>Bartonella</em> species by comparing <em>groEL</em> sequences. <span class="ref-journal">Int. J. Syst. Evol. Microbiol.</span> <span class="ref-vol">52</span><strong>:</strong>165-171.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0h_jqQh-fnS6aZt-zLsoCB6RFL1CAcsch1ICOx5xH&amp;reftype=pubmed&amp;artid=1393131&amp;article-id=1393131&amp;iid=128355&amp;issue-id=128355&amp;jid=81&amp;journal-id=81&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11837299">PubMed</a>]</span>.</div></div></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">Journal of Clinical Microbiology</span> are provided here courtesy of <br><b>American Society for Microbiology (ASM)</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc1-->

