<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="BMC Evolutionary Biology"><meta name="citation_title" content='Discovery of the photosynthetic relatives of the "Maltese mushroom" Cynomorium'><meta name="citation_authors" content="Daniel L Nickrent, Joshua P Der, and  Frank E Anderson"><meta name="citation_date" content="2005"><meta name="citation_volume" content="5"><meta name="citation_firstpage" content="38"><meta name="citation_doi" content="10.1186/1471-2148-5-38"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1182362&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1182362&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1182362"><meta name="citation_pmid" content="15969755"><meta name="DC.Title" content='Discovery of the photosynthetic relatives of the "Maltese mushroom" Cynomorium'><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="BioMed Central"><meta name="DC.Contributor" content="Daniel L Nickrent"><meta name="DC.Contributor" content="Joshua P Der"><meta name="DC.Contributor" content="Frank E Anderson"><meta name="DC.Date" content="2005"><meta name="DC.Identifier" content="10.1186/1471-2148-5-38"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody4.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs3.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Discovery of the photosynthetic relatives of the "Maltese mushroom" Cynomorium</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0RE1mQ_Qy1Swnc90eKSXFnBiL0-H3e4cJzX9Mtk2N&amp;reftype=other&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-bmceb.gif" alt="Logo of bmceb" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="BioMed Central web site" coords="0,0,99,74" onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0tIFaJ8HJMkDxQKsqjdaVnFah121oGkJyFR_bsQ0T&amp;reftype=publisher&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com" target="pmc_ext"><area shape="rect" coords="100,54,172,74" alt="Reference to the article." onclick="focuswin('pmc_ext') " href="redirect3.cgi?&amp;&amp;auth=047O1q4o8djKAaXOe1RWaAIiNJOifNQwnQg0Qek_h&amp;reftype=publisher&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7CArticle%7CFullText&amp;rendering-type=normal&amp;&amp;http://dx.doi.org/10.1186/1471-2148-5-38" target="pmc_ext"><area shape="rect" coords="173,54,223,74" alt="Search." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0VY7xia2pXnUDZeW7D6Gu_-V2UzFXCurS6sHMUQ9k&amp;reftype=publisher&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/search/bmcevolbiol" target="pmc_ext"><area shape="rect" coords="224,55,342,74" alt="Manuscript submission." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0vaHr7VICGzJ1QZ5R8ozGTD5SQxNYO-0EC_vY4Q86&amp;reftype=publisher&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/manuscript/bmcevolbiol" target="pmc_ext"><area shape="rect" coords="343,53,480,74" alt="Registration." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0V178kaNufHuF8i4n8B7mJAQn1uRCTOMqSmEdakk4&amp;reftype=publisher&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/registration" target="pmc_ext"><area shape="rect" coords="0,0,480,74" alt="Journal front page." onclick="focuswin('pmc_ext') " href="redirect3.cgi?&amp;&amp;auth=0p7wbme68YS9IzBBVM0FFLz1YrWVQi3r2rrv7Ek36&amp;reftype=publisher&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/bmcevolbiol" target="pmc_ext"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=28&amp;action=archive">BMC Evol Biol</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=18008">v.5; 2005</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=1182362&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="#supplementary-material-sec" class="sidefm-supplink">Supplementary Material</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=1182362&amp;blobtype=pdf" class="sidefm-pmclink">PDF (511K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=18008" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=28&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0PS-puaMyNj3QGNktAmnjtp5qTru-4CJGRZ5S3Q45&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15969755">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=02ACONwOezxPMpjrDeItju1EHnOAeOMQnfIFb9AgH&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=15969755">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=00qjCGBV67JqoA7XXAOFxIQR_-X4G1oBqteXab5cQ&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=15969755">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0Fq2GRa9mgTFtTESqgMuMyseCKUDOXnFgpC0WvkDy&amp;reftype=Nucleotide&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Nucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1182362&amp;dopt=pmc_nuccore">Nucleotide</option><option value="redirect3.cgi?&amp;&amp;auth=0lsWa1iAeDrEgVAwMFR44HnGuN-JVxUhDwtet6AWX&amp;reftype=Protein&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Protein&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1182362&amp;dopt=pmc_protein">Protein</option><option value="redirect3.cgi?&amp;&amp;auth=0x7Yk2hi2DhmCe3ZovhHrrSWnYGx8XnwHahdA8QEA&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1182362&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=02UHfTUOXyGoZpPlsSlIc-_PKZYQWXzFrSZgIRAwO&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=1182362">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0lskODp4RC4WzKcoy9fb_cFupAaNTlSENr-FOlAPL&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Nickrent+DL%5Bauth%5D" class="sidefm-pmclink"> Nickrent, D.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=04K7s3ZK3afVLPiJEmd3AroECOCz6cZMakbvJGkQh&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Der+JP%5Bauth%5D" class="sidefm-pmclink"> Der, J.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0X_9bFQfK-mziYLjVoz8TeaKglUnw_iPxNyQut5M1&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Anderson+FE%5Bauth%5D" class="sidefm-pmclink"> Anderson, F.</a></div></div></div></td><td rowspan="108" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-bmceb.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">BMC Evol Biol. </span><span class="citation-publication-date">2005; </span><span class="citation-volume">5</span><span class="citation-issue"></span><span class="citation-flpages">: 38. </span></div><div><span class="fm-vol-iss-date">Published online 2005 June 21. </span><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1186/1471-2148-5-38.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC1182362</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 2005 Nickrent et al; licensee BioMed Central Ltd.</div><div class="fm-title">Discovery of the photosynthetic relatives of the "Maltese mushroom" <em>Cynomorium</em></div><div class="fm-author contrib-group">Daniel L Nickrent,<sup><img src="corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>#</sup><sup>1</sup> Joshua P Der,<sup>1</sup> and  Frank E Anderson<sup>#</sup><sup>2</sup><div class="fm-affl"><sup>1</sup>Department of Plant Biology, Southern Illinois University, Carbondale, IL 62901-6509, USA</div><div class="fm-affl"><sup>2</sup>Department of Zoology, Southern Illinois University, Carbondale, IL 62901-6501, USA</div></div><div class="fm-footnote"><sup><img src="corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div class="fm-footnote"><sup>#</sup>Contributed equally.</div><div class="fm-footnote"><span style="white-space: nowrap;" id="id368038">Daniel L Nickrent: <span class="e_id368038">nickrent/at/plant.siu.edu</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id368038", '<a class="ext-reflink" href="' + reverseAndReplaceString('ude.uis.tnalp/ta/tnerkcin:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ude.uis.tnalp/ta/tnerkcin', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id465674">Joshua P Der: <span class="e_id465674">jpder/at/siu.edu</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id465674", '<a class="ext-reflink" href="' + reverseAndReplaceString('ude.uis/ta/redpj:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ude.uis/ta/redpj', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id422043">Frank E Anderson: <span class="e_id422043">feander/at/siu.edu</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id422043", '<a class="ext-reflink" href="' + reverseAndReplaceString('ude.uis/ta/rednaef:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ude.uis/ta/rednaef', '/at/','@') + '</a>')}catch(e){}
                                //--></script></span> </div><div class="fm-pubdate">Received April 26, 2005; Accepted June 21, 2005.</div><div class="fm-copyright"><p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=087Bp6-9RctXkSl55onDhL6uGDYFt3co9kGYDcAet&amp;reftype=extlink&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://creativecommons.org/licenses/by/2.0" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://creativecommons.org/licenses/by/2.0</a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="links-box"><div class="fm-footnote"><img src="corehtml/pmc/pmcgifs/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="tocrender.fcgi?artid=1182362&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470431" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470690" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470917" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471710" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471741" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472713" class="sidefm-pmclink" style="text-transform: none;">List of abbreviations</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472779" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472788" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473198" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id399411">Abstract</div><div class="section-content"><!--article-meta--><p><div class="abs-head2 head-separate">Background</div>Although recent molecular phylogenetic studies have identified the photosynthetic relatives of several enigmatic holoparasitic angiosperms, uncertainty remains for the last parasitic plant order, Balanophorales, often considered to include two families, Balanophoraceae and Cynomoriaceae. The nonphotosynthetic (holoparasitic) flowering plant <em>Cynomorium coccineum </em>has long been known to the Muslim world as "tarthuth" and to Europeans as the "Maltese mushroom"; <em>C. songaricum </em>is known in Chinese medicine as "suo yang." Interest in these plants is increasing and they are being extensively collected from wild populations for use in herbal medicines.</p><p><div class="abs-head2 head-separate">Results</div>Here we report molecular phylogenetic analyses of nuclear ribosomal DNA and mitochondrial <em>matR </em>sequence data that strongly support the independent origin of Balanophoraceae and Cynomoriaceae. Analyses of single gene and combined gene data sets place <em>Cynomorium </em>in Saxifragales, possibly near Crassulaceae (stonecrop family). Balanophoraceae appear related to Santalales (sandalwood order), a position previously suggested from morphological characters that are often assumed to be convergent.</p><p><div class="abs-head2 head-separate">Conclusion</div>Our work shows that <em>Cynomorium </em>and Balanophoraceae are not closely related as indicated in all past and present classifications. Thus, morphological features, such as inflorescences bearing numerous highly reduced flowers, are convergent and were attained independently by these two holoparasite lineages. Given the widespread harvest of wild <em>Cynomorium </em>species for herbal medicines, we here raise conservation concerns and suggest that further molecular phylogenetic work is needed to identify its photosynthetic relatives. These relatives, which will be easier to cultivate, should then be examined for phytochemical activity purported to be present in the more sensitive <em>Cynomorium</em>.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399411" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Background</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470690" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470917" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471710" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471741" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472713" class="sidefm-pmclink" style="text-transform: none;">List of abbreviations</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472779" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472788" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473198" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id470431">Background</div><div class="section-content"><p>Molecular phylogenetics has expanded understanding of relationships among all major angiosperm groups and has thereby strongly impacted their classification [<a href="#B1" class="cite-reflink bibr">1</a>]. More recently, such advances have also included some nonphotosynthetic holoparasites whose phylogenetic positions had previously been uncertain, such as Hydnoraceae [<a href="#B2" class="cite-reflink bibr">2</a>] and Rafflesiales [<a href="#B3" class="cite-reflink bibr">3</a>]. The latter study documented that Rafflesiales are actually a polyphyletic assemblage of three or four independent evolutionary lineages. The losses and reductions in features that are pervasive in parasitic plants have resulted in remarkable morphological convergences, thereby explaining their erroneous placement by traditional methods. Previous molecular phylogenetic work with such holoparasites highlighted the need to employ gene sequences from different subcellular compartments and analytical methods that accommodate rate heterogeneity, thus avoiding long-branch attraction artifacts [<a href="#B3" class="cite-reflink bibr">3</a>]. These steps are justified because congruence among different gene trees provides evidence that the organismal tree is being recovered, while incongruence suggests the presence of nonstandard processes such as introgression, lineage sorting, and horizontal gene transfer (HGT) [<a href="#B3" class="cite-reflink bibr">3</a>-<a href="#B6" class="cite-reflink bibr">6</a>].</p><p>In addition to the abovementioned holoparasites, another group with uncertain placement is Balanophorales. For over 150 years, there has been taxonomic debate as to whether the genus <em>Cynomorium </em>is part of Balanophoraceae [<a href="#B7" class="cite-reflink bibr">7</a>,<a href="#B8" class="cite-reflink bibr">8</a>] or whether it should be classified as a separate family, Cynomoriaceae [<a href="#B9" class="cite-reflink bibr">9</a>]. These plants are fleshy, monoecious or dioecious holoparasitic herbs that often produce swollen tuberous haustorial root connections to their host plants. Their stems bear scalelike leaves and their unisexual flowers represent the ultimate in reduction among angiosperms [<a href="#B10" class="cite-reflink bibr">10</a>] (Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1</a>). Balanophoraceae (in the strict sense) contains 17 genera and 44 species in the neo- and paleotropics whereas <em>Cynomorium </em>has two species in eastern Asia (<em>C. songaricum</em>) and the Mediterranean (<em>C. coccineum</em>). The presence of bisexual flowers, an unusual bimodal karyotype [<a href="#B11" class="cite-reflink bibr">11</a>] and features of the stamens, ovules, and embryo sac [<a href="#B12" class="cite-reflink bibr">12</a>] in <em>Cynomorium </em>all support its segregation from Balanophoraceae. Whether considered one or two families, all past and present classifications accept a relationships between <em>Cynomorium </em>and Balanophoraceae. Previous molecular phylogenetic work using only nuclear small-subunit ribosomal DNA [<a href="#B13" class="cite-reflink bibr">13</a>] suggested placement of <em>Cynomorium </em>with Saxifragales and Balanophoraceae with the sandalwood order (Santalales). The validity of these results could not be assessed because high substitution rates in these plants could potentially confound phylogeny estimation [<a href="#B14" class="cite-reflink bibr">14</a>] and because confirmatory data from other genes was lacking. Here we utilize both nuclear and mitochondrial gene sequence data, analyzed with maximum parsimony (MP) and Bayesian inference (BI) methods, to determine whether Balanophoraceae and Cynomoriaceae are related to one another and where these clades fit in the global angiosperm phylogeny.</p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F1" name="F1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-1.gif" border="1" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span><strong>1 Morphology of <em>Cynomorium coccineum </em>(a-e) and <em>Balanophora fungosa </em>(f)</strong>. a, Habit showing inflorescences emerging from the ground. b, Scanning electron micrograph (SEM) of staminate flower. c, SEM of carpellate flower. d, SEM of bisexual flower. e, SEM</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399411" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470431" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Results</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470917" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471710" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471741" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472713" class="sidefm-pmclink" style="text-transform: none;">List of abbreviations</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472779" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472788" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473198" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id470690">Results</div><div class="section-content"><p>Results of analyses of of the global data set using both MP (Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F2" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">2</a>) and BI (<a href="#S3" class="side-supplink supplementary-material">Additional File 3</a>) strongly indicate that <em>Cynomorium </em>is not closely related to Balanophoraceae but is a component of Saxifragales. Small-subunit (SSU) nuclear ribosomal DNA alone analyzed with BI (<a href="#S4" class="side-supplink supplementary-material">Additional File 4</a>) and ML (<a href="#S5" class="side-supplink supplementary-material">Additional File 5</a>) gave the same result as obtained with the global data set. Similarly, mitochondrial <em>matR </em>analyzed alone with BI (<a href="#S6" class="side-supplink supplementary-material">Additional File 6</a>) showed <em>Cynomorium </em>well separated from Balanophoraceae. Moreover, strong support is obtained from multigene and separate gene analyses for a relationship between Balanophoraceae and Santalales. Given these results, attempts were made to precisely determine the sister taxon of <em>Cynomorium </em>within Saxifragales, a clade that has been subjected to extensive molecular phylogenetic work [<a href="#B15" class="cite-reflink bibr">15</a>,<a href="#B16" class="cite-reflink bibr">16</a>]. The shortest MP trees contained a clade composed of <em>Cynomorium </em>and Crassulaceae (Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F3" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">3</a>), however, this relationship did not receive high bootstrap (BS) support. A similar lack of resolution for this clade was also seen in the BI tree (<a href="#S7" class="side-supplink supplementary-material">Additional File 7</a>). On both the MP and BI trees, <em>Cynomorium </em>is part of a clade with moderate support that contains Crassulaceae, Paeoniaceae, Aphanopetalaceae, Tetracarpaeaceae, Haloragaceae, Pterostemonaceae, Iteaceae, Grossulariaceae, and Saxifragaceae.</p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F2" name="F2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-2.gif" border="1" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span><strong>Maximum parsimony strict consensus tree from the <em>global </em>data set</strong>. Data set of combined nuclear SSU rDNA, chloroplast <em>rbcL </em>and <em>atpB </em>and mitochondrial <em>matR</em>. Strict consensus of 4 trees. Bootstrap support values are shown above the lines, Bayesian posterior</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F3" name="F3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-3.gif" border="1" class="icon-reflink" alt="Figure 3" title="Figure 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 3</strong></a><div class="figure-table-caption-in-article"><span><strong>Maximum parsimony strict consensus tree from the <em>Saxifragales </em>data set</strong>. Data set of combined nuclear SSU and LSU rDNA, chloroplast <em>rbcL</em>, <em>atpB</em>, and <em>matK</em>. Strict consensus of 9 trees. Bootstrap support values are shown above the lines. The minimum length</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399411" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470431" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470690" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Discussion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471710" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471741" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472713" class="sidefm-pmclink" style="text-transform: none;">List of abbreviations</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472779" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472788" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473198" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id470917">Discussion</div><div class="section-content"><p>This work documents that <em>Cynomorium </em>and Balanophoraceae are not closely related, a result in conflict with past and present classifications. Assuming our molecular gene trees reflect the organismal tree, these data indicate that morphological features, such as inflorescences bearing numerous highly reduced flowers, are convergent and were thus attained independently by these two holoparasite lineages.</p><p><div class="head2 head-separate"><em>Cynomorium </em>in Saxifragales</div>From an historical perspective, Hooker [<a href="#B7" class="cite-reflink bibr">7</a>] allied <em>Cynomorium </em>and Balanophoraceae with Haloragaceae (the latter is a family in Saxifragales, see Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F3" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">3</a>) because both groups have epigynous flowers with stamens opposite the valvate perianth lobes. He also noted the striking similarities between the female flowers of <em>Gunnera </em>and <em>Lophophytum </em>(Balanophoraceae). Our analyses that included <em>Gunnera </em>(data not shown) indicate it is not closely related to either Cynomoriaceae or Balanophoraceae, a result in agreement with other work that placed <em>Gunnera </em>as sister to all core eudicots [<a href="#B17" class="cite-reflink bibr">17</a>,<a href="#B18" class="cite-reflink bibr">18</a>]. Other 19<sup>th </sup>century workers such as Hoffmeister proposed a relationships between <em>Cynomorium </em>and <em>Hippuris </em>(Plantaginaceae, an asterid) because both have a single, epigynous stamen attached to the top of a unicarpellate ovary. This relationship was not confirmed following a detailed floral morphological study [<a href="#B19" class="cite-reflink bibr">19</a>] nor was <em>Cynomorium </em>shown to be related to asterids following molecular analyses reported here.<p>The exact position of Saxifragales within the core eudicot clade remains uncertain because various molecular phylogenetic analyses are in conflict (c. f. [<a href="#B17" class="cite-reflink bibr">17</a>,<a href="#B18" class="cite-reflink bibr">18</a>,<a href="#B20" class="cite-reflink bibr">20</a>,<a href="#B21" class="cite-reflink bibr">21</a>]). These studies have proposed relationships with rosids, asterids, caryophyllids, and Santalales, i.e. nearly all members of the core tricolpate clade (see review by Judd and Olmstead [<a href="#B22" class="cite-reflink bibr">22</a>]). The results reported here also demonstrate uncertainty with regard to the topology of the above clades (Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F2" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">2</a>). Given the variety of ways these clades could resolve, the possibility exists that Balanophoraceae and <em>Cynomorium </em>are more closely related than graphically depicted in Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F2" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">2</a>. Although future molecular phylogenetic studies may support this, it is unlikely that these two holoparasite groups will become sister given that the Balanophoraceae + Santalales clade has BS support of 94% and <em>Cynomorium </em>is sister to <em>Peridiscus </em>with 98% BS support. Moreover, the latter two genera are sister to <em>Hamamelis </em>with a 98% BS value.</p><p>Long-branch attraction has long been suggested as a confounding factor when conducting phylogenetic analyses of organisms with marked among-lineage rate heterogeneity [<a href="#B23" class="cite-reflink bibr">23</a>]. Indeed, our group has shown that Rafflesiales, another holoparasitic angiosperm group, is susceptible to long-branch attraction, particularly when analyzed with MP [<a href="#B3" class="cite-reflink bibr">3</a>]. The results reported here differ from that study in several respects. First, MP analysis of the global data set does not indicate that Balanophoraceae and <em>Cynomorium </em>exhibit particularly long branches with respect to other angiosperms, nor are these holoparasite clades "attracted" to each other (<a href="#S8" class="side-supplink supplementary-material">Additional File 8</a>). This result differs from that obtained when the Rafflesiales molecular data are analyzed with MP where the four distinct lineages (Apodanthaceae, Cytinaceae, Mitrastemonaceae and Rafflesiaceae) appear monophyletic [<a href="#B3" class="cite-reflink bibr">3</a>]. When the global data set is analyzed with BI, some degree of branch length heterogeneity is seen (<a href="#S9" class="side-supplink supplementary-material">Additional File 9</a>), however, Balanophoraceae and <em>Cynomorium </em>still remain on distinct clades. It should be pointed out that for the global data set, branch length comparisons between holoparasites and photosynthetic plants are somewhat biased given that the chloroplast genes were coded as missing for the holoparasites. Taken together, we feel our results are not being influence by long-branch attraction and that they provide strong evidence for the independent evolution of these two holoparasite lineages.</p><p>Attempts to find morphological features that link <em>Cynomorium </em>with Saxifragales are complicated, first because the latter order is morphologically heterogeneous and second because of morphological reductions in the holoparasite (see below). The heterogeneity of Saxifragales is evidenced by the fact that pre-molecular classifications placed many of its component families in distinct subclasses [<a href="#B8" class="cite-reflink bibr">8</a>]. Although a close relationship between <em>Cynomorium </em>and Crassulaceae was not seen on the BI tree, the shortest MP tree showed these genera as sister taxa, thus this possible affinity will here be considered. Tricolporate pollen (Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1e</a>), stem succulence, and proanthocyanidins (red pigments, see [<a href="#B24" class="cite-reflink bibr">24</a>]) are possible characters that may reflect a relationship between <em>Cynomorium </em>and Crassulaceae, but these are all general features found in many groups. Other than such facies, there are few floral characters that could be considered synapomorphies between the extremely reduced flowers of <em>Cynomorium </em>and those of Crassulaceae. The staminate flower of <em>Cynomorium </em>is composed of four to six distinct perianth parts forming a whorl below the single stamen and a wedge-shaped structure interpreted as a pistillode [<a href="#B25" class="cite-reflink bibr">25</a>]. The apical portion of the pistillode is concave and accepts the base of an anther theca prior to filament elongation (Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1b</a>) and its internal lateral surface has a groove that accepts the filament. Stamen morphology is not unlike many tricolpate angiosperms: the anthers are dorsifixed, tetrasporangiate, dithecal and open by longitudinal slits. The carpellate flower (Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1c</a>) is composed of a single carpel with an elongated, grooved style. The perianth (or perigonial scales) are much smaller than in the staminate flower, reduced to small papillae at the summit of the ovary or along its sides. The bisexual flowers are similar to carpellate flowers except for the addition of a stamen at the base of the ovary (Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1d</a>). As with many holoparasitic flowering plants, the reduction or loss of morphological features interferes with the comparative morphological approach when searching for relatives among less modified photosynthetic plants.</p></p><p><div class="head2 head-separate">Balanophoraceae in Santalales</div>Although precedents exists for placing Balanophoraceae with Santalales [<a href="#B8" class="cite-reflink bibr">8</a>,<a href="#B26" class="cite-reflink bibr">26</a>-<a href="#B29" class="cite-reflink bibr">29</a>], recovering this relationship with both nuclear and mitochondrial genes is surprising because any morphological similarities (e.g., reduction in the gynoecium) have usually been assumed to be cases of convergence [<a href="#B30" class="cite-reflink bibr">30</a>]. Moreover, BI of the Saxifragales data set indicates a derivation of Balanophoraceae from <em>within </em>Santalales, not as its sister (Figure <a href="articlerender.fcgi?artid=1182362&amp;rendertype=figure&amp;id=F3" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">3</a> and <a href="#S8" class="side-supplink supplementary-material">Additional File 8</a>). Although a multigene analysis of Santalales with robust taxon sampling exists [<a href="#B31" class="cite-reflink bibr">31</a>], it was generated with nuclear ribosomal DNA and chloroplast genes. To conduct analyses with Balanophoraceae and Santalales, mitochondrial genes from the latter will be needed to test relationships.<p>Cronquist [<a href="#B8" class="cite-reflink bibr">8</a>] viewed the sandalwoods as the best candidate ancestral group for Balanophoraceae, however, he admitted any similarities might reflect convergent adaptations to parasitism. Of the 17 genera in the family, only <em>Dactylanthus, Hachettea </em>and <em>Mystropetalon </em>were used in this study because they are sister (and basal) to the remaining genera and have lower substitution rates in SSU rDNA, thus avoiding potential long-branch artifacts. The floral morphology of <em>Mystropetalon </em>is less reduced than other genera in the family. Given that its staminate flowers have pistillodes (vestigial carpels) and carpellate flowers have staminodes (vestigial stamens), it is assumed this plesiomorphic condition reflects evolution from an ancestor with bisexual flowers. Floral features in common with Santalales include a valvate perianth and stamens with "typical" morphology (i.e. anther sacs and filaments as opposed to synangia as seen in <em>Balanophora</em>). When examining the haustorial structure of <em>Mystropetalon</em>, Weber [<a href="#B32" class="cite-reflink bibr">32</a>] noted the presence of a collapsed zone, graniferous tracheary elements, and "runners," which he associated with Santalales. Similarly, reduction in ovules is seen in several santalalean families, particularly the mistletoes in Viscaceae where no true ovule exists. Either an embryo sac is embedded within central placenta called a "mamelon" or the archesporium develops hypodermally in the ovary, functioning directly as megaspore mother cells [<a href="#B33" class="cite-reflink bibr">33</a>]. Similarly, ovules have been lost in some Santalaceae (<em>Exocarpos</em>) and Loranthaceae. During the course of floral reduction, the loss of ovules most influenced Fagerlind [<a href="#B27" class="cite-reflink bibr">27</a>] who drew a direct ancestor-descendant relationship between Santalaceae and Balanophoraceae.</p><p>As mentioned above, such apparent similarities between Balanophoraceae and Santalales have also been interpreted as convergent. Kuijt [<a href="#B10" class="cite-reflink bibr">10</a>] suggests that the reduction of santalalean ovules may permit retention of meristematic activity, a requirement for the formation of complex fruits in epiparasites. For root parasites such as Balanophoraceae, he suggests ovular reduction is related to selection for tiny seeds that require host stimulants for germination, as is seen in Orobanchaceae. In addition to ovules, the androecia of both Viscaceae and Balanophoraceae are diverse with both showing trends toward the formation of synandria and porose dehiscence. Whether these trends are convergent or follow from homologous suites of genes may yield to direct testing using floral homeotic mutants.</p><p>Although herbaceous root parasites have evolved in Santalaceae (e.g. <em>Thesium, Quinchamalium</em>), most members of Santalales are woody. Completely holoparasitic species do not occur in this order, with the possible exception of <em>Daenikera</em>, an endemic monotypic genus of New Caledonia whose photosynthetic capabilities remain to be determined. As adults, dwarf mistletoes (<em>Arceuthobium</em>, Viscaceae) fix only about 30% of the carbon needed for growth [<a href="#B34" class="cite-reflink bibr">34</a>,<a href="#B35" class="cite-reflink bibr">35</a>] but prior to host attachment seedlings actively photosynthesize. For this reason these mistletoes must be considered advanced obligate hemiparasites, not holoparasites. Thus, to include with Santalales the entirely holoparasitic group Balanophoraceae requires further investigation to identify the exact sister group.</p></p><p><div class="head2 head-separate">Ethnobotany and conservation biology of <em>Cynomorium</em></div><em>Cynomorium coccineum</em>, known to the Muslim world as "tarthuth," has been harvested from the deserts of north Africa and the Middle East for thousands of years. Arabs and Bedouins eat the interior portions of fresh young stems, prepare infusions of older stems to treat colic or stomach ulcers, or dry and pulverize the plant for use as a spice or condiment with meat dishes [<a href="#B36" class="cite-reflink bibr">36</a>]. Medicinal uses of tarthuth can be traced to Al-Kindi, Al-Razi (Rhazes), Ibn Masawayh, Ibn Wahshiya, and Maimonides but the plant became known to Europeans only in the 16<sup>th </sup>century. A group called the Knights Hospitaller of St. John operated a hospital in Jerusalem and learned of the medicinal qualities of tarthuth from local Muslim physicians. When the Crusaders lost Jerusalem to the Muslims, they moved to the island of Malta where <em>Cynomorium </em>was also native. The site where the "Maltese Mushroom" grew (Fungus Rock) was thereafter vigorously guarded and thieves were imprisoned or made galley slaves. The "treasure of drugs," as the Arabs called it, was used for a variety of purposes, including treating apoplexy, venereal disease, high blood pressure, vomiting, irregular menstrual periods and as a contraceptive and toothpaste.<p>Modern biomedical and phytochemical research on <em>Cynomorium coccineum </em>has demonstrated a variety of activities from plant extracts. Effects of <em>Cynomorium </em>extracts on mammalian reproductive cells modulation of pituitary gonadotrophins [<a href="#B37" class="cite-reflink bibr">37</a>] as well as changes in testicular development [<a href="#B38" class="cite-reflink bibr">38</a>] and epididymal sperm patterns [<a href="#B39" class="cite-reflink bibr">39</a>] in rats. <em>Cynomorium songaricum</em>, known in Chinese medicine as "suo yang," has been shown to contain triterpenes with HIV protease inhibitory activity [<a href="#B40" class="cite-reflink bibr">40</a>,<a href="#B41" class="cite-reflink bibr">41</a>]. Interest in herbal medicines is growing at a rapid pace, and attention has been focused upon <em>Cynomorium </em>as evidenced by its availability via hundreds of distributors advertising (via the internet) herbal remedies for kidney and intestinal ailments as well as for impotence. Although it is not at present clear how many original distributors of the plant exist, and whether plant material being marketed as <em>Cynomorium </em>is actually this plant, what is clear is that it is not being cultivated, thus authentic herbal preparations must be obtained from wild populations. Very little information exists on the cultivation of <em>Cynomorium </em>[<a href="#B25" class="cite-reflink bibr">25</a>] and certainly commercial suppliers are not practicing sustainable harvest by cultivating this obligate holoparasite. There is evidence that overexploitation of this plant has resulted in localized extinction [<a href="#B42" class="cite-reflink bibr">42</a>]. For these reasons, we here raise concerns for the conservation of both species of <em>Cynomorium </em>and strongly voice the need to develop cultivation methodologies. Given the potential and actual biomedical applications of extracts from this plant, and conservation concerns given extensive harvesting from wild populations, information on its phylogenetic position within angiosperms is timely. Further molecular work will likely illuminate its closest relatives within Saxifragales. These taxa should then become the subject of phytochemical analyses to determine whether they also contain compounds of biomedical interest. Cultivation of photosynthetic plants would be more straightforward than the holoparasite, thus possibly relieving some of the pressure to harvest this more sensitive species from the wild.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399411" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470431" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470690" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470917" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Conclusion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471741" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472713" class="sidefm-pmclink" style="text-transform: none;">List of abbreviations</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472779" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472788" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473198" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id471710">Conclusion</div><div class="section-content"><p>All previous classifications have allied <em>Cynomorium </em>and Balanophoraceae, likely owing to the holoparasitic habit and the presence of inflorescences with numerous tiny flowers. Molecular phylogenetic analyses using nuclear and mitochondrial gene sequences both indicate that these taxa are not closely related and that perceived similarities are a result of convergent evolution. <em>Cynomorium </em>is strongly supported as being a member of Saxifragales, however, its exact position within this order remains unresolved. Surprisingly, both nuclear and mitochondrial genes place Balanophoraceae with the sandalwood order, a relationship previously proposed but explained by some as a case of convergent evolution. Both species of wild <em>Cynomorium </em>are being harvested from wild populations for use in herbal medicines, and evidence exists for overexploitation. Given that methods to cultivate these holoparasites are not being employed, we here raise conservation concerns. If further molecular phylogenetic work identifies the nearest photosynthetic relatives of <em>Cynomorium</em>, these plants should be examined for phytochemical activity.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399411" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470431" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470690" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470917" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471710" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Methods</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472713" class="sidefm-pmclink" style="text-transform: none;">List of abbreviations</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472779" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472788" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473198" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id471741">Methods</div><div class="section-content"><p><div class="head2 head-separate">Taxon sampling and data collection</div>Voucher information for all newly sequenced taxa is given in <a href="#S1" class="side-supplink supplementary-material">Additional Files 1</a> and <a href="#S2" class="side-supplink supplementary-material">2</a>. DNA was extracted, PCR amplified, cloned, and sequenced following reported methods [<a href="#B3" class="cite-reflink bibr">3</a>,<a href="#B43" class="cite-reflink bibr">43</a>]. The nuclear and mitochondrial sequences were amplified using primers reported elsewhere [<a href="#B14" class="cite-reflink bibr">14</a>,<a href="#B44" class="cite-reflink bibr">44</a>,<a href="#B45" class="cite-reflink bibr">45</a>]. Sequencing was conducted using an ABI Prism<sup>&#x000ae; </sup>377 automated DNA sequencer (Applied Biosystems) according to the manufacturer's protocols. Two multiple sequence alignments were constructed. The purpose of the <strong><em>global </em></strong>data set was to test the position of <em>Cynomorium </em>and the three Balanophoraceae genera among angiosperm orders. This matrix included nuclear small-subunit (SSU) rDNA, chloroplast <em>rbcL </em>and <em>atpB</em>, and mitochondrial <em>matR </em>for 67 taxa with <em>Laurus </em>and <em>Cinnamomum </em>as outgroups. Balanophoraceae and Cynomorium lack the chloroplast genes <em>rbcL </em>and <em>atpB</em>, but they were included in the global data set to stabilize the overall tree topology as previously demonstrated [<a href="#B2" class="cite-reflink bibr">2</a>,<a href="#B3" class="cite-reflink bibr">3</a>]. Given the strong support for a relationship between <em>Cynomorium </em>and Saxifragales obtained from this global analysis, a <strong><em>Saxifragales </em></strong>data set was constructed to further resolve the position of the parasites within this clade. Here the genes used were nuclear SSU and LSU rDNA, chloroplast <em>rbcL</em>, <em>atpB</em>, and <em>matK </em>for 62 taxa with <em>Lea </em>and <em>Vitis </em>as outgroups. In both alignments, chloroplast genes were coded as missing for the holoparasites. Accession numbers for newly sequenced genes are: <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0JuKxnt942NrH6hVh-nt2vLf2PEn-x5WNbpjGnAaX&amp;reftype=extlink-entrez-nucleotide&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=63253787" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">AY957440</a></span> &#x02013; <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0HczyCvyyfN5Jlsgt5x2a7mXamALGpfHBpOw5S5V6&amp;reftype=extlink-entrez-nucleotide&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=63253808" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">AY957454</a></span>.</p><p><div class="head2 head-separate">Phylogenetic analyses</div>The <strong><em>global </em></strong>and <strong><em>Saxifragales </em></strong>data sets were analyzed using maximum parsimony (MP) in PAUP* 4.0b10 [<a href="#B46" class="cite-reflink bibr">46</a>] and Bayesian inference (BI) methods in MrBayes 3.0b4 [<a href="#B47" class="cite-reflink bibr">47</a>]. MP searches were performed using 100 random addition sequence replicates with tree-bisection-reconnection (TBR) branch-swapping, holding ten trees at each addition step, with all sites equally weighted. Fully partitioned Bayesian analyses were performed by partitioning each data set by gene and, for the protein-coding genes, by codon position. This resulted in a total of 10 partitions for the full data set and 11 partitions for the Saxifragales data set (Table <a href="articlerender.fcgi?artid=1182362&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a>). MP trees were constructed for each gene (following the protocol described above), and these trees were used in PAUP* to evaluate 24 nucleotide substitution models for each data partition. For example, models were evaluated for the "<em>rbcL </em>Pos1" partition (the first codon position of the <em>rbcL </em>data set) on the MP tree for the <em>rbcL </em>data set. MrModelTest 2.0 [<a href="#B48" class="cite-reflink bibr">48</a>] was used to select an appropriate model from the PAUP* output via a second-order version of the Akaike Information Criterion (AIC<em>c</em>) that takes sample size into account, as recommended by Posada and Buckley [<a href="#B49" class="cite-reflink bibr">49</a>]. AIC<em>c </em>values were computed using both the total number of characters and the number of variable characters per partition. For nearly all partitions, the Akaike weight for the chosen model was much higher than the Akaike weight of the next best model, so model-averaged analyses were not performed. The best-fitting models and their Akaike weights for each data partition are listed in Supplementary Data. Partitioned Bayesian analyses were performed with all model parameters unlinked (i.e., model parameters for each data partition were estimated separately from each partition), and with topology and branch lengths linked. Two separate analyses (with different MCMC seeds, random starting trees and default uniform priors for all parameters) were run for each data set for 15 million generations, with trees sampled every 500 generations. Trees recovered during the first 2.5 million generations (the first 5000 trees) in both runs were discarded as burn-in, leaving a total of 25,000 trees which were used to construct majority-rule consensus trees.<div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T1" name="T1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1182362&amp;rendertype=table&amp;id=T1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 1" title="Table 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1182362&amp;rendertype=table&amp;id=T1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 1</strong></a><div class="figure-table-caption-in-article"><span>Summary of output from MrModelTest for each data partition in the <em>global </em>and <em>Saxifragales </em>data sets &#x000a7;</span></div></td></tr></table></div></div><div style="clear:both;"></div></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399411" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470431" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470690" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470917" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471710" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471741" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>List of abbreviations</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472779" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472788" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473198" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id472713">List of abbreviations</div><div class="section-content"><p>AIC<em>c </em>&#x02013; Akaike Information criterion</p><p><em>atpB </em>&#x02013; ATP synthase beta subunit</p><p>BI &#x02013; Bayesian inference</p><p>BS &#x02013; bootstrap</p><p>GTR &#x02013; general time reversible model (a model of DNA sequence evolution)</p><p>HIV &#x02013; human immunodeficiency virus</p><p>I + &#x00393; &#x02013; invariant sites plus gamma distribution</p><p><em>matK </em>&#x02013; chloroplast maturase K</p><p><em>matR </em>&#x02013; mitochondrial maturase R</p><p>MCMC &#x02013; Markov Chain Monte Carlo (a simulation method used to approximate the posterior probabilities of trees)</p><p>ML &#x02013; maximum likelihood</p><p>MP &#x02013; maximum parsimony</p><p><em>rbcL </em>&#x02013; ribulose bisphosphate carboxylase/oxygenase, large subunit</p><p>nu SSU &#x02013; nuclear small-subunit ribosomal DNA</p><p>nu LSU &#x02013; nuclear large-subunit ribosomal DNA</p><p>TBR &#x02013; tree bisection-reconnection branch swapping</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id472779">Authors' contributions</div><div class="section-content"><p>DLN coordinated all aspects of the study, obtained directly or through colleagues all genomic DNAs used herein, conducted sequence aligments, and drafted the manuscript. JPD obtained all of the newly generated sequences. FEA performed all phylogenetic analyses. All of the authors read and approved the final manuscript.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399411" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470431" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470690" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470917" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471710" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471741" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472713" class="sidefm-pmclink" style="text-transform: none;">List of abbreviations</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472779" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Supplementary Material</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id473198" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id472788"><a id="supplementary-material-sec"></a>Supplementary Material</div><div class="section-content"><!--body--><div id="S1"><div><strong>Additional File 1</strong></div><p><strong>Taxon sampling &#x02013; global data set. </strong>MS Excel file giving taxon sampling by gene and GenBank accession numbers for the <strong><em>global </em></strong>data set.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-S1.xls">Click here for file</a><sup> (30K)</sup></blockquote></div></div><div id="S2"><div><strong>Additional File 2</strong></div><p><strong>Taxon sampling &#x02013; Saxifragales data set. </strong>MS Excel file giving taxon sampling by gene and GenBank accession numbers for the <strong><em>Saxifragales </em></strong>data set.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-S2.xls">Click here for file</a><sup> (31K)</sup></blockquote></div></div><div id="S3"><div><strong>Additional File 3</strong></div><p><strong>BI tree from global data set. </strong>Bayesian inference majority rule consensus tree of 50,000 trees derived from the <em>global </em>data set composed of nuclear SSU rDNA, chloroplast <em>rbcL</em>, <em>atpB</em>, and mitochondrial <em>matR</em>. Trees were generated in two separate BI analyses, each run for 15 million generations with trees from the first 2.5 million generations removed as burn-in.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-S3.PDF">Click here for file</a><sup> (864K)</sup></blockquote></div></div><div id="S4"><div><strong>Additional File 4</strong></div><p><strong>BI tree from the nuclear SSU rDNA partition. </strong>Bayesian inference majority rule consensus tree of 50,000 trees derived from the nuclear SSU rDNA partition. Trees were generated in two separate BI analyses, each run for 15 million generations with trees from the first 2.5 million generations removed as burn-in.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-S4.PDF">Click here for file</a><sup> (756K)</sup></blockquote></div></div><div id="S5"><div><strong>Additional File 5</strong></div><p><strong>ML tree from the nuclear SSU rDNA partition. </strong>Phylogram from maximum likelihood (ML) analysis of the nuclear SSU rDNA partition (GTR+I+&#x00393; model. The tree was generated using a successive approximations approach in which a MP tree was generated and used as a starting tree for additional branch swapping under ML.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-S5.PDF">Click here for file</a><sup> (544K)</sup></blockquote></div></div><div id="S6"><div><strong>Additional File 6</strong></div><p><strong>BI tree from the mitochondrial matR partition. </strong>Bayesian inference majority rule consensus tree of 50,000 trees derived from the mitochondrial <em>matR </em>partition. Trees were generated in two separate BI analyses, each run for 15 million generations with trees from the first 2.5 million generations removed as burn-in.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-S6.PDF">Click here for file</a><sup> (741K)</sup></blockquote></div></div><div id="S7"><div><strong>Additional File 7</strong></div><p><strong>BI tree from the Saxifragales data set. </strong>Bayesian inference majority rule consensus tree of 50,000 trees derived from the <em>Saxifragales </em>data set. Trees were generated in two separate BI analyses, each run for 15 million generations with trees from the first 2.5 million generations removed as burn-in.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-S7.PDF">Click here for file</a><sup> (748K)</sup></blockquote></div></div><div id="S8"><div><strong>Additional File 8</strong></div><p><strong>MP phylogram from the global data set. </strong>Data set of combined nuclear SSU rDNA, chloroplast <em>rbcL </em>and <em>atpB </em>and mitochondrial <em>matR</em>. Tree one of four is shown, with branch lengths drawn proportional to the number of changes.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-S8.PDF">Click here for file</a><sup> (524K)</sup></blockquote></div></div><div id="S9"><div><strong>Additional File 9</strong></div><p><strong>BI majority rule phylogram from the global data set. </strong>Bayesian inference majority rule consensus phylogram from the <em>global </em>data set (nuclear SSU rDNA, chloroplast <em>rbcL </em>and <em>atpB </em>and mitochondrial <em>matR</em>). Branch lengths are means of the branch length posterior probability distribution across all post burn-in trees from one of the two MrBayes runs (branch lengths resulting from the other run are nearly identical).</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1182362&amp;blobname=1471-2148-5-38-S9.PDF">Click here for file</a><sup> (529K)</sup></blockquote></div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id473161">Acknowledgements</div><div class="section-content"><p><p><p>The authors wish to thank all who assisted this project by collecting plant samples or assisting in field work: A. Aparicio and I. Sanchez Garc&#x000ed;a (<em>Cynomorium coccineum</em>), C. Ecroyd (<em>Dactylanthus taylori</em>), W. Forstreuter (<em>Benthamina alyxifolia</em>), J.-M. Groult (<em>Hachettea austrocaledonica</em>), R. Narayana (<em>Santalum album</em>), K. Steiner and A. Wolfe (<em>Mystropetalon thomii</em>). Data analysis was greatly facilitated by David Swofford and Peter Foster who generously allowed access to computer clusters at Florida State University and The Natural History Museum (London), respectively. J. Bozzola and S. Schmitt at the SIUC IMAGE facility assisted in SEM photography. Financial support was provided by the National Science Foundation to DLN. The helpful comments of S. Stefanovic improved the final manuscript.</p></p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id399411" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470431" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470690" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id470917" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471710" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id471741" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472713" class="sidefm-pmclink" style="text-transform: none;">List of abbreviations</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472779" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id472788" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id473198">References</div><div class="section-content"><div class="back-matter-section"><ul class="back-ref-list" style="list-style-type: decimal;"><li id="B1">APG.  An update of the Angiosperm Phylogeny Group classification for the orders and families of flowering plants: APG II. <span><span class="ref-journal">Botanical Journal of the Linnean Society London. </span>2003;<span class="ref-vol">141</span>:399–436. doi: 10.1046/j.1095-8339.2003.t01-1-00158.x.</span></li><li id="B2">Nickrent, DL; Blarer, A; Qiu, YL; Soltis, DE; Soltis, PS; Zanis, M. Molecular data place Hydnoraceae with Aristolochiaceae. <span><span class="ref-journal">Amer J Bot. </span>2002;<span class="ref-vol">89</span>:1809–1817.</span></li><li id="B3">Nickrent, DL; Blarer, A; Qiu, YL; Vidal-Russell, R; Anderson, FE. Phylogenetic inference in Rafflesiales: the influence of rate heterogeneity and horizontal gene transfer. <span><span class="ref-journal">BMC Evolutionary Biology. </span>2004;<span class="ref-vol">4:40</span></span></li><li id="B4">Wendel, JF; Doyle, JJ.  Phylogenetic incongruence: window into genome history and molecular evolution. In: Soltis DE, Soltis PS and Doyle JJ. ,  editor. <span class="ref-journal">Molecular Systematics of Plants II DNA Sequencing.</span> Boston, MA, Kluwer Academic Publishers; 1998. pp. 265–296.</li><li id="B5">Davis, CC; Wurdack, KJ. Host-to-parasite gene transfer in flowering plants: phylogenetic evidence from Malpighiales. <span><span class="ref-journal">Science. </span>2004;<span class="ref-vol">575</span>:676–678. doi: 10.1126/science.1100671.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0d92OMtcSlJUghyaUsjtAx9InHxbSk_ydkLlVKkQp&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15256617">PubMed</a>]</span></li><li id="B6">Mower, JP; Stefanovic, S; Young, GJ; Palmer, JD. Gene transfer from parasitic to host plants. <span><span class="ref-journal">Nature (London). </span>2004;<span class="ref-vol">432</span>:165–166. doi: 10.1038/432165b.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0dAaERIVMF2B_iY2wXgaOKcOkUJZ0dT2D-IsTuQyd&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15538356">PubMed</a>]</span></li><li id="B7">Hooker, JD. On the structure and affinities of Balanophoraceae. <span><span class="ref-journal">Trans Linn Soc London. </span>1856;<span class="ref-vol">22</span>:1–68 plus 16 plates.</span></li><li id="B8">Cronquist, A. <span class="ref-journal">An integrated system of classification of flowering plants.</span> Bronx, New York, Columbia Univ. Press; 1981. p. 1262.</li><li id="B9">Takhtajan, A. <span class="ref-journal">Diversity and classification of flowering plants.</span> New York, NY, Columbia University Press; 1997. p. 643.</li><li id="B10">Kuijt, J. <span class="ref-journal">The Biology of Parasitic Flowering Plants.</span> Berkeley, CA, University of California Press; 1969. p. 246.</li><li id="B11">Pazy, B; Plitmann, U; Cohen, O. Bimodal karyotype in Cynomorium coccineum L. and its systematic implications. <span><span class="ref-journal">Botanical Journal of the Linnean Society London. </span>1996;<span class="ref-vol">120</span>:279–281. doi: 10.1006/bojl.1996.0015.</span></li><li id="B12">Teryokhin, ES; Yakovlev, MS; Nikitcheva, ZI. Development of microsporangium, pollen grains, ovule and embryo sac in Cynomorium songaricum Rupr. (Cynomoriaceae). <span><span class="ref-journal">Botnicheskii Zhurnal. </span>1975;<span class="ref-vol">60</span>:153–162.</span></li><li id="B13">Nickrent, DL.  Or&#x000ed;genes filogen&#x000e9;ticos de las plantas par&#x000e1;sitas. In: L&#x000f3;pez-S&#x000e1;ez JA, Catal&#x000e1;n P and S&#x000e1;ez L. ,  editor. <span class="ref-journal">Plantas Par&#x000e1;sitas de la Pen&#x000ed;nsula Ib&#x000e9;rica e Islas Baleares.</span> Madrid, Mundi-Prensa Libros, S. A.; 2002. pp. 29–56.</li><li id="B14">Nickrent, DL; Starr, EM. High rates of nucleotide substitution in nuclear small-subunit (18S) rDNA from holoparasitic flowering plants. <span><span class="ref-journal">Journal of Molecular Evolution. </span>1994;<span class="ref-vol">39</span>:62–70. doi: 10.1007/BF00178250.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=02Su0vG_NrtedJDebGmcG7NJMx7eUCGbAmGIC4AGN&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8064875">PubMed</a>]</span></li><li id="B15">Fishbein, M; Soltis, DE. Further resolution of the rapid radiation of Saxifragales (angiosperms, eudicots) supported by mixed-model Bayesian analysis. <span><span class="ref-journal">Systematic Botany. </span>2004;<span class="ref-vol">29</span>:883–891. doi: 10.1600/0363644042450982.</span></li><li id="B16">Fishbein, M; Hibsch-Jetter, C; Soltis, DE; Hufford, L. Phylogeny of Saxifragales (angiosperms, eudicots): Analysis of a rapid, ancient radiation. <span><span class="ref-journal">Systematic Biology. </span>2001;<span class="ref-vol">50</span>:817–847. doi: 10.1080/106351501753462821.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0XSMeZcW-f4cn8Pqb3VLyKZThUDAuVunXLjlRukFY&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12116635">PubMed</a>]</span></li><li id="B17">Soltis, DE; Senters, AE; Zanis, MJ; Kim, S; Thompson, JD; Soltis, PS; Ronse De Craene, LP; Endress, PK; Farris, JS. Gunnerales are sister to other core eudicots: implications for the evolution of pentamery. <span><span class="ref-journal">Amer J Bot. </span>2003;<span class="ref-vol">90</span>:461–470.</span></li><li id="B18">Hilu, KW; Borsch, T; Muller, K; Soltis, DE; Soltis, PS; Savolainen, V; Chase, MW; Powell, MP; Alice, LA; Evans, R; Sauquet, H; Neinhuis, C; Slotta, TAB; Rohwer, JG; Campbell, CS; Chatrou, LW. Angiosperm phylogeny based on matK sequence information. <span><span class="ref-journal">American Journal of Botany. </span>2003;<span class="ref-vol">90</span>:1758–1776.</span></li><li id="B19">Juel, O. Cynomorium und Hippuris. <span><span class="ref-journal">Svensk Bot Tidskr. </span>1910;<span class="ref-vol">4</span>:151–159.</span></li><li id="B20">Soltis, DE; Soltis, PS; Chase, MW; Mort, ME; Albach, DC; Zanis, M; Savolainen, V; Hahn, WH; Hoot, SB; Fay, MF; Axtell, M; Swensen, SM; Prance, LM; Kress, WJ; Nixon, KC; Farris, JS. Angiosperm phylogeny inferred from 18S rDNA, rbcL, and atpB sequences. <span><span class="ref-journal">Botanical Journal of the Linnean Society London. </span>2000;<span class="ref-vol">133</span>:381–461. doi: 10.1006/bojl.2000.0380.</span></li><li id="B21">Kim, S; Soltis, DE; Soltis, PS; Zanis, MJ; Suh, Y. Phylogenetic relationships among early-diverging eudicots based on four genes: were the eudicots ancestrally woody? <span><span class="ref-journal">Molecular Phylogenetics and Evolution. </span>2004;<span class="ref-vol">31</span>:16–30. doi: 10.1016/j.ympev.2003.07.017.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0657nLCLTEWuPjD0kpV1Vh67oiaG-BMmY81iKPAiD&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15019605">PubMed</a>]</span></li><li id="B22">Judd, WS; Olmstead, RG. A survey of tricolpate (eudicot) phylogenetic relationships. <span><span class="ref-journal">Amer J Bot. </span>2004;<span class="ref-vol">91</span>:1627–1644.</span></li><li id="B23">Anderson, FE; Swofford, DL. Should we be worried about long-branch attraction in real data sets? Investigations using metazoan 18S rDNA. <span><span class="ref-journal">Mol Phylo Evol. </span>2004;<span class="ref-vol">33</span>:440–451. doi: 10.1016/j.ympev.2004.06.015.</span></li><li id="B24">Harborne, JB; Saito, N; Detoni, CH. Anthocyanins of Cephaelis, Cynomorium, Euterpe, Lavatera and Pinanga. <span><span class="ref-journal">Biochemical Systematics and Ecology. </span>1994;<span class="ref-vol">22</span>:835–836. doi: 10.1016/0305-1978(94)90088-4.</span></li><li id="B25">Weddell, HA. Memoire sur le Cynomorium coccineum, parasite de l'ordre des Balanophor&#x000e9;es. <span><span class="ref-journal">Archives du Mus&#x000e9;um National d'Histoire Naturelle. </span>1860;<span class="ref-vol">10</span>:269–308.</span></li><li id="B26">Engler, A.  Balanophoraceae. In: Engler A and Prantl K. ,  editor. <span class="ref-journal">Die Naturlichen Pflanzenfamilien, Nachtr.</span> Vol. 3. Leipzig, ; 1894. pp. 243–252.</li><li id="B27">Fagerlind, F. Beitr&#x000e4;ge z&#x000fc;r Kenntnis der Gyn&#x000e4;ceummorphologie und Phylogenie der Santalales-Familien. <span><span class="ref-journal">Svensk Botanisk Tidskrift. </span>1948;<span class="ref-vol">42</span>:195–229.</span></li><li id="B28">Takhtajan, AL. Outline of the classification of flowering plants (Magnoliophyta). <span><span class="ref-journal">Botanical Review. </span>1980;<span class="ref-vol">46</span>:225–359.</span></li><li id="B29">Thorne, RF. An updated phylogenetic classification of the flowering plants. <span><span class="ref-journal">Aliso. </span>1992;<span class="ref-vol">13</span>:365–389.</span></li><li id="B30">Kuijt, J. Mutual affinities of Santalalean families. <span><span class="ref-journal">Brittonia. </span>1968;<span class="ref-vol">20</span>: 136–147.</span></li><li id="B31">Nickrent, DL; Mal&#x000e9;cot, V.  A molecular phylogeny of Santalales. In: Fer A, Thalouarn P, Joel DM, Musselman LJ, Parker C and Verkleij JAC. ,  editor. <span class="ref-journal">Proceedings of the 7th International Parasitic Weed Symposium.</span> Nantes, France, Facult&#x000e9; des Sciences, Universit&#x000e9; de Nantes; 2001. pp. 69–74.</li><li id="B32">Weber, HC. Granulahaltige Xylem-Leitbahnen und andere den Santalales &#x000e4;hnliche anatomische Strukturen in den haustorialen Knollen von Mystropetalon thomii Harv. (Balanophoraceae). <span><span class="ref-journal">Flora. </span>1986;<span class="ref-vol">178</span>:315–328.</span></li><li id="B33">Bhandari, NN; Vohra, SCA.  Embryology and affinities of Viscaceae. In: Calder M and Bernhardt P. ,  editor. <span class="ref-journal">The Biology of Mistletoes.</span> New York, Academic Press; 1983. pp. 69–86.</li><li id="B34">Hull, RJ; Leonard, OA. Physiological aspects of parasitism in mistletoes (Arceuthobium and Phoradendron). 1.   The carbohydrate nutrition of mistletoe. <span><span class="ref-journal">Pl Phys. </span>1964;<span class="ref-vol">39</span>:996–1007.</span></li><li id="B35">Hull, RJ; Leonard, OA. Physiological aspects of parasitism in mistletoes (Arceuthobium and Phoradendron). 2. The photosynthetic capacity of mistletoes. <span><span class="ref-journal">Pl Phys. </span>1964;<span class="ref-vol">39</span>:1008–1017.</span></li><li id="B36">Lebling, RW. The treasure of tarthuth. <span><span class="ref-journal">Saudi Aramco World. </span>2003;<span class="ref-vol">54</span>:12–17.</span></li><li id="B37">Al-Qarawi, AA; Abdel-Rahman, HA; El-Badry, AA; Harraz, F; Razig, NA; Abdel-Magied, EM. The effect of extracts of Cynomorium coccineum and Withania somnifera on gonadotrophins and ovarian follicles of immature Wistar rats. <span><span class="ref-journal">Phytother Res. </span>2000;<span class="ref-vol">14</span>:288–290. doi: 10.1002/1099-1573(200006)14:4&#x0003c;288::AID-PTR603&#x0003e;3.0.CO;2-9.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0v7MyjbOuSzXrNr60f5GozyFXsE6rr_EjXBjIBkLl&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10861976">PubMed</a>]</span></li><li id="B38">Abd el-Magied, EM; Abd el-Rahman, HA; Harraz, FM. The effect of aqueous extracts of Cynomorium coccineum and Withania somnifera on testicular development in immature Wistar rats. <span><span class="ref-journal">J Ethnopharmacol. </span>2001;<span class="ref-vol">75</span>:1–4. doi: 10.1016/S0378-8741(00)00348-2.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0nSwyJ6kNX6sIvs863a0mD0_k5Iq4EwT1QJZ6tAX4&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11282435">PubMed</a>]</span></li><li id="B39">Abd el-Rahman, HA; El-Badry, AA; Mahmoud, OM; Harraz, FA. The effect of the aqueous extract of Cynomorium coccineum on the epididymal sperm pattern of the rat. <span><span class="ref-journal">Phytother Res. </span>1999;<span class="ref-vol">13</span>:248–250. doi: 10.1002/(SICI)1099-1573(199905)13:3&#x0003c;248::AID-PTR425&#x0003e;3.0.CO;2-D.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0jOc4OTR_rzfr9ZqUG0i4josgC924vl4PDX86TkCY&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10353171">PubMed</a>]</span></li><li id="B40">Ma, CM; Nakamura, N; Miyashiro, H; Hattori, M; Shimotohno, K. Inhibitory effects of constituents from Cynomorium songaricum and related triterpene derivatives on HIV-1 protease. <span><span class="ref-journal">Chem Pharm Bull. </span>1999;<span class="ref-vol">47</span>:141–145.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0v5paaE58a8goQD5kH1hKuhd4GkckcnK3oAW77knk&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10071849">PubMed</a>]</span></li><li id="B41">Nakamura, N. Inhibitory effects of some traditional medicines on proliferation of HIV-1 and its protease. <span><span class="ref-journal">Yakugaku Zasshi. </span>2004;<span class="ref-vol">124</span>:519–529. doi: 10.1248/yakushi.124.519.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0RY7y-js0q1oe2ZJi6-wfW2b7CiUOwU9ComMvAkdR&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15297721">PubMed</a>]</span></li><li id="B42">Xie, Y; Wang, S. <span class="ref-journal">Chapter 11: People and biodiversity.</span> , Institute of Zoology, Chinese Academy of Sciences, Beijing, 100080. (http://www.chinabiodiversity.com/shengwudyx2/training/chapter11.htm); 2001. </li><li id="B43">Nickrent, DL. From field to film: rapid sequencing methods for field collected plant species. <span><span class="ref-journal">BioTechniques. </span>1994;<span class="ref-vol">16</span>:470–475.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Zqh2_-IX1Qf0UJhXbPbv9DLuQs_lRcMI1QJ8Y5F3&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8185922">PubMed</a>]</span></li><li id="B44">Qiu, Y; Lee, Y; Bernasconi-Quadroni, F; Soltis, DE; Soltis, PS; Zanis, M; Zimmer, EA; Chen, Z; Savolainen, V; Chase, MW. The earliest angiosperms: evidence from mitochondrial, plastid and nuclear genomes. <span><span class="ref-journal">Nature. </span>1999;<span class="ref-vol">402</span>:404–409. doi: 10.1038/46536.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Fm5QJUuDrar2qXYTGTiTWQLzNbBf5dYsfB7dqk77&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10586879">PubMed</a>]</span></li><li id="B45">Kuzoff, RK; Sweere, JA; Soltis, DE; Soltis, PS; Zimmer, EA. The phylogenetic potential of entire 26S rDNA sequences in plants. <span><span class="ref-journal">Mol Biol Evol. </span>1998;<span class="ref-vol">15</span>:251–263.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0B3ftjpESUDKlsINhlZDo7WmiE-c9-eeucQyQNkTM&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9501492">PubMed</a>]</span></li><li id="B46">Swofford, DL. <span class="ref-journal">PAUP*: phylogenetic analysis using parsimony (* and other methods).</span> 4.0.b10. Sunderland, MA, Sinauer Associates; 2002. </li><li id="B47">Ronquist, F; Huelsenbeck, JP. MrBayes 3: Bayesian phylogenetic inference under mixed models. <span><span class="ref-journal">Bioinformatics. </span>2003;<span class="ref-vol">19</span>:1572–1574. doi: 10.1093/bioinformatics/btg180.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0jijGuHXXnZtbKIlrcXAcc8_M8_S9RoHOf5wySkUc&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12912839">PubMed</a>]</span></li><li id="B48">Nylander, JA. <span class="ref-journal">MrModelTest.</span> 2.0. Upsalla, Distributed by the author. Evolutionary Biology Centre, Uppsala University; 2004. </li><li id="B49">Posada, D; Buckley, TR. Model selection and model averaging in phylogenetics: advantages of Akaike information criterion and Bayesian approaches over likelihood ratio tests. <span><span class="ref-journal">Systematic Biology. </span>2004;<span class="ref-vol">53</span>:793–808. doi: 10.1080/10635150490522304.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=06GvhvjocadZWKQdF2ZcNvy6t5aJUfP6MM3pmq5GA&amp;reftype=pubmed&amp;artid=1182362&amp;article-id=1182362&amp;iid=18008&amp;issue-id=18008&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15545256">PubMed</a>]</span></li></ul></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">BMC Evolutionary Biology</span> are provided here courtesy of <br><b>BioMed Central</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc2-->

