<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Insect Molecular Biology"><meta name="citation_title" content="Genomic and transcriptional analysis of protein heterogeneity of the honeybee venom allergen Api m 6"><meta name="citation_authors" content="N Peiren, D C de Graaf, J D Evans, and  F J Jacobs"><meta name="citation_date" content="2006 October"><meta name="citation_issue" content="5"><meta name="citation_volume" content="15"><meta name="citation_firstpage" content="577"><meta name="citation_doi" content="10.1111/j.1365-2583.2006.00669.x"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1761135&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1761135&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1761135"><meta name="citation_pmid" content="17069633"><meta name="DC.Title" content="Genomic and transcriptional analysis of protein heterogeneity of the honeybee venom allergen Api m 6"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Blackwell Publishing"><meta name="DC.Contributor" content="N Peiren"><meta name="DC.Contributor" content="D C de Graaf"><meta name="DC.Contributor" content="J D Evans"><meta name="DC.Contributor" content="F J Jacobs"><meta name="DC.Date" content="2006 October"><meta name="DC.Identifier" content="10.1111/j.1365-2583.2006.00669.x"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody5.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs1.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Genomic and transcriptional analysis of protein heterogeneity of the honeybee venom allergen Api m 6</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0V3rBYLy1vO9EWa-_rfump-beuQuvFJJoJeTaXQll&amp;reftype=other&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div class="banner-generic-logo-background"><div class="banner-journal-publisher-over-image"><div class="banner-journal-name over-logo-journal-title-blackwellopen" style="overflow: hidden;">Insect Molecular Biology</div><div class="banner-publisher-name"><a class="ext-reflink" alt="Publisher URL" href="redirect3.cgi?&amp;&amp;auth=0-amnvr7gSa-_E04u5bhOpZeY5doQFt5eBuOVa5Ve&amp;reftype=publisher&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.blackwellpublishing.com/static/onlineopen.asp?site=1" onclick="focuswin('pmc_ext')" target="pmc_ext">Blackwell Publishing Ltd</a></div></div></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a href="redirect3.cgi?&amp;&amp;auth=0lYVrIOghxD77okJ5LIEMvi7lL5dyCc5pkY2jE5X7&amp;reftype=other&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CN%2FA&amp;TO=Entrez%7CSearch%7CWiley-Blackwell%20Online%20Open&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=search&amp;term=blackwell%20online%20open%5Bfilter%5D" class="navlink">Wiley-Blackwell Online Open</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=1761135&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=1761135&amp;blobtype=pdf" class="sidefm-pmclink">PDF (396K)</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=02kE03DhiqxIBYGhlhM7tQmsqvJTiS4fd04q_pAI_&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17069633">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=06ollEnjmBHgwoVRoa8WocWb_IdGDR_ZIa_B9GkpN&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=17069633">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0NQWcY11q-4AGidm5PNY5Gr8sxWvCww4SQjYwXA_I&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=17069633">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0FIN-MW1SK1xO0m4QclnB1p5eC_He5C0cKPmXh5Iq&amp;reftype=Gene&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Gene&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1761135&amp;dopt=pmc_gene">Gene</option><option value="redirect3.cgi?&amp;&amp;auth=0RS2VltTzrQSfzEZ9sBMcXCarPeGq8Hxz-3zsZkk4&amp;reftype=Nucleotide&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Nucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1761135&amp;dopt=pmc_nuccore">Nucleotide</option><option value="redirect3.cgi?&amp;&amp;auth=0zCD1VTmDk7e2iEeay-dcH03dP4ypfnztPz9y55N2&amp;reftype=Substance&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|PubChem%20Substance&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1761135&amp;dopt=pmc_pcsubstance">Substance</option><option value="redirect3.cgi?&amp;&amp;auth=0lgQv9IjHliGuAV3-SsA4nQsuC0TCT94XpemzMk13&amp;reftype=Protein&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Protein&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1761135&amp;dopt=pmc_protein">Protein</option><option value="redirect3.cgi?&amp;&amp;auth=0FSsglLdVLrEQGMHftUNPEA0mi0OXRTnt9jxJDQHM&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1761135&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0Z1Vtcl_m7WyyaL2-1_tw9g55t63FWvrFHPlFVQds&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=1761135">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0N1Y4AYUA-TqiVwzK4wKtTDQt-C5LfBhB4Hultkx9&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Peiren+N%5Bauth%5D" class="sidefm-pmclink"> Peiren, N.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0-qLz9tEPH4AztpFer9T8TsRdxX6k8lPumnU_uA44&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20de%20Graaf+DC%5Bauth%5D" class="sidefm-pmclink"> de Graaf, D.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0NAe6vU-MV2Ck_vVUN3wfrt3B5FAplB5j8AVGjQKl&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Evans+JD%5Bauth%5D" class="sidefm-pmclink"> Evans, J.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0NiRtRm6BH5nidnllsQHdNWIX6kil0fC8cltcEADQ&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Jacobs+FJ%5Bauth%5D" class="sidefm-pmclink"> Jacobs, F.</a></div></div></div></td><td rowspan="103" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-blackwellopen.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">Insect Mol Biol. </span><span class="citation-publication-date">2006 October; </span><span class="citation-volume">15</span><span class="citation-issue">(5)</span><span class="citation-flpages">: 577–581. </span></div><div><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1111/j.1365-2583.2006.00669.x.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC1761135</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a> &#x000a9; 2006 The Authors Journal compilation &#x000a9; 2006 The Royal Entomological Society</div><div class="fm-title">Genomic and transcriptional analysis of protein heterogeneity of the honeybee venom allergen Api m 6</div><div class="fm-author contrib-group">N Peiren, D C de Graaf, J D Evans,<sup>*</sup> and  F J Jacobs<div class="fm-affl">Laboratory of Zoophysiology, Ghent University, Ghent, Belgium</div><div class="fm-affl"><sup>*</sup>USDA-ARS Bee Research Laboratory, Beltsville, MD, USA</div></div><div class="fm-footnote"></div><div class="fm-footnote" id="cor1">Correspondence: Dirk C. de Graaf, Laboratory of Zoophysiology, K.L. Ledeganckstraat 35, B-9000 Ghent, Belgium. Tel.: +32 92648732; fax: +32 92645242; e-mail: <span class="e_id440814">Dirk.deGraaf/at/Ugent.be</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id440814", '<a class="ext-reflink" href="' + reverseAndReplaceString('eb.tnegU/ta/faarGed.kriD:otliam', '/at/', '@') + '">' + reverseAndReplaceString('eb.tnegU/ta/faarGed.kriD', '/at/','@') + '</a>')}catch(e){}
                                //--></script></div><div class="fm-footnote"><div class="p">Re-use of this article is permitted in accordance with the Creative Commons Deed, Attribution 2&#x000b7;5, which does not permit commercial exploitation.</div></div><div class="fm-pubdate">Received March 15, 2006; Accepted May 10, 2006.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id632779" class="sidefm-pmclink" style="text-transform: none;">Introduction</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561446" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id446756" class="sidefm-pmclink" style="text-transform: none;">Experimental procedures</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id446917" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id632752">Abstract</div><div class="section-content"><!--article-meta-->Several components of honeybee venom are known to cause allergenic responses in humans and other vertebrates. One such component, the minor allergen Api m 6, has been known to show amino acid variation but the genetic mechanism for this variation is unknown. Here we show that Api m 6 is derived from a single locus, and that substantial protein-level variation has a simple genome-level cause, without the need to invoke multiple loci or alternatively spliced exons. Api m 6 sits near a misassembled section of the honeybee genome sequence, and we propose that a substantial number of indels at and near Api m 6 might be the root cause of this misassembly. We suggest that genes such as Api m 6 with coding-region or untranslated region indels might have had a strong effect on the assembly of this draft of the honeybee genome.<div class="p"><span class="kwd-label">Keywords: </span><span class="kwd-text">allelic variation, allergen, Api m 6, bee venom, signal peptide</span></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id632752" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Introduction</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561446" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id446756" class="sidefm-pmclink" style="text-transform: none;">Experimental procedures</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id446917" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id632779">Introduction</div><div class="section-content"><p>Dangerous allergenic responses can be caused in humans by components of the venom of stinging social insects (<a href="#b7" class="cite-reflink bibr">Hoffman, 2003</a>). A major research goal remains to predict which fraction of the human population will be likely to react adversely to specific venom components received through incidental or occupation-related stinging. Further, an understanding of the components of venoms that induce allergic responses can be used to better tune therapeutic treatments of allergic responses. Allergens from insects and other sources are designated with the terms &#x02018;major&#x02019; or &#x02018;minor&#x02019; depending on whether more or less than 50% of the hypersensitive patients develops a specific IgE response against a given allergen (<a href="#b11" class="cite-reflink bibr">Larsen &#x00026; Lowenstein, 1996</a>). Honeybee venom contains the major allergens Api m 1 (phospholipase A2), Api m 2 (hyaluronidase), Api m 3 (acid phosphatase), Api m 4 (melittin) and Api m 7 (CUB serine protease), along with at least one minor allergen, Api m 6, which shows 42% IgE responsiveness (<a href="#b8" class="cite-reflink bibr">Hoffman, 2006</a>).</p><p>Here we describe the genomic basis for observed protein-level heterogeneity found for Api m 6. Api m 6 migrates as an 8-kDa band in sodium dodecyl sulphate&#x02013;polyacrylamide gel electrophoresis and its amino acid sequence was determined on high-performance liquid chromatography-purified preparations (<a href="#b9" class="cite-reflink bibr">Kettner <em>et al</em>., 2001</a>). It exists as four isoforms of 7190, 7400, 7598 and 7808 Da, respectively, differing in their primary structure at the amino and carboxy terminus by a maximum of six amino acids (<a href="#b9" class="cite-reflink bibr">Kettner <em>et al</em>., 2001</a>). Allergen heterogeneity can be due to allelic variation at a single allergen gene (<a href="#b5" class="cite-reflink bibr">Gao <em>et al</em>., 2005</a>), the occurrence of multiple genes of allergens encoding highly homologous proteins (<a href="#b14" class="cite-reflink bibr">Piersma <em>et al</em>., 2005</a>), or by alternative splicing of a single transcript (<a href="#b13" class="cite-reflink bibr">Mykles <em>et al</em>., 1998</a>). Some bee venom components are already known to have a rather peculiar gene organization. Indeed, the precursors of bee venom apamin and MCD peptide are encoded by two genes in tandem, which share the same 3&#x02032;-exon (<a href="#b6" class="cite-reflink bibr">Gmachl &#x00026; Kreil, 1995</a>). Here we use new genome-level data for honeybees to determine the cause behind multiple isoforms of Api m 6. We present the complete cDNA sequences for two Api m 6 variants. We then use the latest honeybee assembly to show that these variants arise from a single polymorphic locus. Interestingly, high sequence-level variation at and around this locus appears to be responsible for a break in the honeybee genome assembly.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id632752" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id632779" class="sidefm-pmclink" style="text-transform: none;">Introduction</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Results and discussion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id446756" class="sidefm-pmclink" style="text-transform: none;">Experimental procedures</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id446917" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id561446">Results and discussion</div><div class="section-content"><p><div class="head2 head-separate">Cloning and sequencing of cDNA fragments encoding Api m 6</div>5&#x02032;-Rapid amplification of cDNA ends (5&#x02032;-RACE) using gene-specific primers resulted in an amplicon of approximately 350 bp in size that was cloned in the pCR&#x000ae;4 vector. Transformation to chemically competent TOP-10 <em>Escherichia coli</em> yielded a large number of transformants, 10 of which were retained for further analysis. DNA sequences were found to be identical (represented by <em>clones 5.1</em>) and consisted of a putative 5&#x02032; untranslated region (UTR) and a coding region for which the predicted protein was identical to that known for Api m 6. 3&#x02032;-RACE resulted in a fragment of approximately the same size, which was again cloned and sequenced. Here the sequenced clones showed two transcript variants, represented by the inserts of <em>clone 3.3</em> and <em>clone 3.9</em>. Subsequently, we amplified the complete Api m 6 transcript from a venom gland cDNA preparation using primers designed, respectively, at the extreme 5&#x02032;- and 3&#x02032;-ends of <em>clone 5.1</em> and <em>clone 3.9</em>. Sequencing of the corresponding 528 bp fragment revealed two additional minor transcript differences, this time at the 5&#x02032;-end, 38 and 117 bp upstream of the forward primer, when compared with the insert of <em>clone 5.1</em>. This cDNA sequence was deposited to GenBank (accession number <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0nCHOolIGrrBz159cuBEfakLAvF0IMQn3Tw6Hn5QP&amp;reftype=extlink-entrez-nucleotide&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=88770351" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">DQ384991</a></span>), as was a sequence assembly of the inserts of <em>clones 5.1</em> and <em>3.9</em> (accession number <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=021Y3EQhIt-H7rcY13dSuuASNZrmd7HZwZSDHaQvu&amp;reftype=extlink-entrez-nucleotide&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=88770349" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">DQ384990</a></span>). Because the latter matched perfectly well the latest honeybee genome assembly (see further), it was named variant 1, whereas <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0bK_UhHK_oae_OdI62n-L1ChocA3UgSJwhEVgUkQI&amp;reftype=extlink-entrez-nucleotide&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=88770351" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">DQ384991</a></span> became variant 2.</p><p><div class="head2 head-separate">Deduced amino acid sequence and protein heterogeneity</div>The deduced amino acid sequences of both transcripts are given in <a href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig01" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">Fig. 1</a>. At the protein level differences were noticed at position 14 (Val against Ile) and at the carboxy terminal region, with two additional residues Leu and Pro. This corresponds with the differences found at the C-terminal ends of the four Api m 6 variants described by <a href="#b9" class="cite-reflink bibr">Kettner <em>et al</em>. (2001)</a>: Api m 6.01 and Api m 6.03 having the same two additional residues, in contrast to the variants Api m 6.02 and Api m 6.04. Further we noticed that the deduced amino acid sequences were 21 residues longer at the N-terminus when compared with the earlier described variants Api m 6.03 and Api m 6.04 (<a href="#b9" class="cite-reflink bibr">Kettner <em>et al</em>., 2001</a>). However, based on an <em>in silico</em> SigP-NN prediction (<a href="#b3" class="cite-reflink bibr">Bendtsen <em>et al</em>., 2004</a>), a signal peptide cleavage site was identified between position 21 and 22, resulting in a mature protein that starts at exactly the same residue. Depending on the cDNA variant that was translated this mature protein will correspond to Api m 6.03 or Api m 6.04. The other two variants described by <a href="#b9" class="cite-reflink bibr">Kettner <em>et al</em>. (2001)</a> lack the first four N-terminal amino acids Phe-Gly-Gly-Phe of the mature protein. We have found no indication that this was the result of a new transcript or alternative splicing variant. Nor could we evidence that the mature proteins of these variants are shortened by enzymatic activity similar to step-wise cleavage of the pro part from promelittin by dipeptidylpeptidase IV (<a href="#b10" class="cite-reflink bibr">Kreil <em>et al</em>., 1980</a>). Further we notice that amino acid replacement at position 14 in the deduced amino acid sequence is located in the signal peptide and has no consequences for the mature allergen.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig01" name="fig01"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig01" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1761135&amp;blobname=imb0015-0577-f1.gif" border="1" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig01" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span>Nucleotide and deduced amino acid sequences of Api m 6 transcript variant 1 (A) and 2 (B). The numbers on the left denote the nucleotide numbers. The start and stop codon are underlined. Residues &#x02212;1 to &#x02212;21 are from the signal peptide.</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig01" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><div class="head2 head-separate">Sequence homology and conserved domains</div>Conserved domain search (<a href="#b12" class="cite-reflink bibr">Marchler-Bauer &#x00026; Bryant, 2004</a>) of the deduced amino acid sequence from transcript variant 1 revealed a trypsin inhibitor like cysteine rich (TIL) domain from residue 37&#x02013;91 (score: 37.3; <em>E</em>-value: 5e-04). This family contains trypsin inhibitors as well as a domain found in many extracellular proteins. The domain typically contains 10 cysteine residues that form five disulphide bonds in the combination 1&#x02013;7, 2&#x02013;6, 3&#x02013;5, 4&#x02013;10 and 8&#x02013;9. The assumption that Api m 6 represents a trypsin inhibitor was already made by <a href="#b2" class="cite-reflink bibr">Banks &#x00026; Shipolini (1986)</a>, although the protein was at that time hardly characterized and certainly not yet named as such. In fact, they described two peptides H1 (17 residues) and H3 (35 residues), which later were found to correspond with the N-terminal ends of Kettner's isoforms of Api m 6, differing from each other only in that one lacks the first four residues. The amino acid compositions were similar to a protease inhibitor that was earlier purified from bee venom by <a href="#b16" class="cite-reflink bibr">Shkenderov (1973)</a>. Although no proteolytic activity could be assigned to the peptides H1 and H3, it appears now that Api m 6 has a molecular weight quite near that of Shkenderov's protease inhibitor, i.e. 9000 Da.</p><p><div class="head2 head-separate">Noncoding sequence variation and the genome assembly</div>A BLASTN search (<a href="#b1" class="cite-reflink bibr">Altschul <em>et al</em>., 1997</a>) of the complete cDNA against honeybee genome assembly 4.0 (Amel_4.0&#x02013;20061003) revealed two hits with very high scores/<em>E</em>-values: scaffold 16.18 (Contig5539), score: 486; <em>E</em>-value: e-136 and unmapped scaffold GroupUn.6096 (Contig14926), score: 371; <em>E</em>-value: e-101. DNA sequence comparison demonstrated a 100% match between Contig5539 (between position 37212 and 36406, introns excluded) and an assembly of the RACE fragments from <em>clone 5.1</em> and <em>clone 3.9</em> (Api m 6 variant 1, <a href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig02" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">Fig. 2A</a>). On the other hand, Contig14926 (between position 16221 and 17048, introns excluded) was identical to the coding region of the full size Api m 6 transcript and the insert of <em>clone 3.3</em> (Api m 6 variant 2, <a href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig02" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">Fig. 2B</a>). However, the UTR of the full size cDNA shows an indel of two consecutive thymines, found upstream from position 41 in <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=08W8OBoup2YiWCNTmks4qR6nH30Nvl8zCtnQW-kFK&amp;reftype=extlink-entrez-nucleotide&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=88770351" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">DQ384991</a></span>. There are also eight indels (at four locations) and one G to A transition in the 3&#x02032; UTR of variant 1 when aligned with variant 2. These results demonstrate the existence of several different transcript variants of the bee venom allergen Api m 6, originating from genome-level variation at a single locus.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig02" name="fig02"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig02" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1761135&amp;blobname=imb0015-0577-f2.gif" border="1" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig02" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span>Alignment of two <em>in silico</em> spliced honeybee genome sequences with different cDNA fragments of the bee venom allergen Api m 6. (A) Api m 6 was present in mapped scaffold 16.18 from genome assembly 4.0 (positions marked above the bar, along with coordinates</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig02" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>Api m 6 shows a substantial amount of sequence variation (nearly all haplotypes sequenced to date differ in at least one place) as well as sections of repetitive simple sequences (monobasic A or T runs, as well as an AT dinucleotide repeat; <a href="articlerender.fcgi?artid=1761135&amp;rendertype=figure&amp;id=fig01" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">Fig. 1</a>). It is conceivable that this variation was the root cause for misassembly at this section of the draft honeybee genome sequence. First, it is evident that assembly scaffolds 16.18 and GroupUn.6095 are homologous, and that GroupUn.6095 spans the gap between assembly scaffolds 16.18 and 16.19. These genome sections apparently assembled separately because of sequence variation found in the two haplotypes sequenced for the Honeybee Genome Project. This effect is probably widespread in assemblies of this and other draft genome sequences. In fact, this effect is seen further down chromosome 16 (scaffolds 16.11 and 16.12), where the hypervariable immune effector apidaecin (<a href="#b4" class="cite-reflink bibr">Casteels <em>et al</em>., 1994</a>) has apparently disrupted the genome assembly. An understanding of allelic variation at specific genes might help unite unassembled parts of the bee genome. As a corollary, misassembled sections of the bee genome might indicate biologically important variation in genes adjacent to these gaps.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id632752" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id632779" class="sidefm-pmclink" style="text-transform: none;">Introduction</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561446" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Experimental procedures</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id446917" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id446756">Experimental procedures</div><div class="section-content"><p><div class="head2 head-separate">Bee venom glands</div>Honeybees (<em>Apis mellifera carnica</em>) were all taken at the hive entrance of a single colony from the apiary of the Ghent University (Belgium). The venom glands of 320 bees were dissected under anaesthesia by chilling. First, the whole sting apparatus was removed from the abdomen and submerged in RNALater&#x000ae; (Ambion, Austin, TX, USA). Subsequently, the glands were separated from the reservoir and collected all together in 100 &#x000b5;l fresh solution. Homogenization and mRNA isolation was done using the Micro-FastTrack&#x02122; 2.0 Kit (Invitrogen, Carlsbad, CA, USA) following the protocol for fresh and frozen tissue. After spectrophotometric yield determination, mRNA was stored in elution buffer at &#x02212;80 &#x000b0;C until ready for use.</p><p><div class="head2 head-separate">cDNA preparation and primer development</div>Venom gland cDNA was prepared using AMV reverse transcriptase and the oligo dT primer from the cDNA Cycle&#x000ae; Kit (Invitrogen) according to the manufacturer's instructions. To find the corresponding genome sequence for primer development, the Api m 6 amino acid sequence (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0lwOboV9lanfEDhv_i3dBYkzBWczUrcXJ80HA6QFw&amp;reftype=extlink-entrez-protein&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CProtein&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=protein&amp;dopt=GenPept&amp;list_uids=34921475" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">P83563</a></span>) was B<span class="small-caps">last</span>-searched against an early release of the honeybee genome (Amel_1.2) at <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0xAuY_7FeBh3orpLjleVP1kMQc32lKwDvSTTauQnk&amp;reftype=extlink&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.hgsc.bcm.tmc.edu/projects/honeybee" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.hgsc.bcm.tmc.edu/projects/honeybee</a></span>. The target domain was located on group 16.4 and extended from position 258118&#x02013;258415 (total length of the group: 806 207 bp). Subsequently Api m 6 gene-specific primers (GSP) for rapid amplification of cDNA ends (RACE; see further) were developed using the Primer3 software on the world wide web (<a href="#b15" class="cite-reflink bibr">Rozen &#x00026; Skaletsky, 2000</a>). This gave the following result: GSP forward primer (GSP-fw) 5&#x02032;-TTGGAGGATTTGGAGGCTTGGAGGA-3&#x02032; and GSP reverse primer (GSP-rv) 5&#x02032;-GCATTTAGATCGCGGAACGCATACCT-3&#x02032;. Their suitability for further analysis was tested by a simple polymerase chain reaction (PCR) using the venom gland cDNA preparation (see above) as template. The resulting amplicon was sequenced for confirmation.</p><p><div class="head2 head-separate">5&#x02032;-Rapid amplification of cDNA ends (5&#x02032;-RACE) and 3&#x02032;-RACE</div>RACE ready cDNA was prepared by following the protocol described in the GeneRacer&#x02122; Kit (Invitrogen). Briefly, 100 ng of bee venom gland mRNA was subsequently treated with calf intestinal phosphatase and tobacco acid pyrophosphatase, to be ligated at its 5&#x02032;-end with the GeneRacer&#x02122; RNA oligo. This ligated mRNA was reverse transcribed using the SuperScript&#x02122; III RT and the GeneRacer&#x02122; oligo dT primer to create RACE-ready first-strand cDNA with known priming sites at the 5&#x02032;- and 3&#x02032;-ends. Generation of 5&#x02032;-RACE fragment was done by PCR, using a combination of GeneRacer&#x02122; 5&#x02032;-primer and GSP-rv, whereas the 3&#x02032;-amplification needed a combination of GSP-fw and GeneRacer&#x02122; 3&#x02032; primer. Both reactions were done in an Eppe&#x0039b;dorf Mastercycler using a touchdown protocol.</p><p><div class="head2 head-separate">Cloning and DNA sequencing</div>PCR products were cloned in the pCR&#x000ae;4-TOPO&#x000ae; vector. Individual transformants were picked and analysed for the presence of insert by PCR. The corresponding amplicons were used for sequence analysis.<p>DNA sequencing was performed using a Perkin Elmer ABI Prism 377 (Perkin Elmer, Wellesley, MA, USA) automated DNA sequencer. PCR product was treated with shrimp alkaline phosphatase (1 U/&#x000b5;l, Amersham E70092Y, Amersham Biosciences, Buckingham, UK) and exonuclease I (20 U/&#x000b5;l, Epicentre Biotechniologies &#x000d7;40505k, Madison, WI, USA) for 15 min at 37 &#x000b0;C, followed by 15 min at 80 &#x000b0;C to inactivate the enzymes. This material was then used for cycle sequencing without any further purification, using the ABI Prism BigDye V 3.1 Terminator Cycle Sequencing kit. The sequencing conditions were 30 s at 96 &#x000b0;C, 15 s at 50 &#x000b0;C and 4 min at 60 &#x000b0;C for 27 cycles. Primers used for sequencing were GSP-fw, GSP-rv or GeneRacer&#x02122; 5&#x02032;-primer. Cycle sequence products were precipitated by adding 25 &#x000b5;l of 95% ethanol and 1 &#x000b5;l 3 <span class="small-caps">m</span> sodium acetate, pH 4.6 to each cycle sequencing reaction (10 &#x000b5;l). The samples were placed at &#x02212;20 &#x000b0;C for 15 min and centrifuged at 14 000 r.p.m. (12 225 <em>g</em>) for 15 min. After precipitation, an additional wash of the pellet was performed with 125 &#x000b5;l of 70% ethanol and centrifuged at 14 000 r.p.m. (12 225 <em>g</em>) for 5 min. The pellet was dried in a Speedvac concentrator, redissolved in loading buffer and run on a 48 cm 4.25% acrylamide/bisacrylamide (29 : 1) gel.</p></p><p><div class="head2 head-separate">Analysis of sequence data</div>Partial cDNA clones and the complete cDNA sequence generated above were aligned with honeybee genome assembly 4.0 (Amel_4.0&#x02013;20061003) using BLASTN (<a href="#b1" class="cite-reflink bibr">Altschul <em>et al</em>., 1997</a>), without filters. Sequences were also compared by BLASTN with unscaffolded contigs. Several thousand base pairs of flanking DNA on either side of Api m 6 (e.g. on Contig5539) was used to confirm that this contig and unassembled Contig14926 were in fact derived from the same genome location, despite showing substantial genome sequence variation at and near Api m 6. Unscaffolded Contig14926 was used, via BLASTN, to span the gap between assembled scaffolds 16.18 and 16.19. Sequence variation 2 5&#x02032;- and 3&#x02032; UTR regions of Api m 6 was characterized by alignment of <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0VSPRXoNrlucNgja6YplL6jxug2qqmi0aFd156Q7c&amp;reftype=extlink-entrez-nucleotide&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=88770351" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">DQ384991</a></span> and Contigs14926 and 5539.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id632752" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id632779" class="sidefm-pmclink" style="text-transform: none;">Introduction</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561446" class="sidefm-pmclink" style="text-transform: none;">Results and discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id446756" class="sidefm-pmclink" style="text-transform: none;">Experimental procedures</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id446917">References</div><div class="section-content"><div class="back-matter-section"><ul class="back-ref-list" style="list-style-type: disc;"><li id="b1">Altschul, SF. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. <span><span class="ref-journal">Nucleic Acids Res. </span>1997;<span class="ref-vol">25</span>:3389–3402.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=08CddW9xcJsWCdzaQU3y9a-x-qNCD0K5A2J1jDkBw&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9254694">PubMed</a>]</span></li><li id="b2">Banks, BEC.  Chemistry and pharmacology of honey bee venom. In: Pick T. ,  editor. <span class="ref-journal">Venoms of the Hymenoptera.</span> London: Academic Press; 1986. pp. 329–416.</li><li id="b3">Bendtsen, JD. Improved prediction of signal peptides: SignalP 3.0. <span><span class="ref-journal">J Mol Biol. </span>2004;<span class="ref-vol">340</span>:783–795.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0LoR6DbIAJAHMM9H-i9fxesPy7f9wiFFRaIkj6QOM&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15223320">PubMed</a>]</span></li><li id="b4">Casteels, P. Biodiversity of apidaecin-type peptide antibiotics &#x02013; prospects of manipulating the antibacterial spectrum and combating acquired-resistance. <span><span class="ref-journal">J Biol Chem. </span>1994;<span class="ref-vol">269</span>:26107–26115.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=08CujoE08aJSNGYgkcDZ6LBAblEtDf1xIBRpr8kF1&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7929322">PubMed</a>]</span></li><li id="b5">Gao, ZS, et al.  Linkage map positions and allelic diversity of two <em>Mal d 3</em> (non-specific lipid transfer protein) genes in the cultivated apple (<em>Malus domestica</em>). <span><span class="ref-journal">Theor Appl Gen. </span>2005;<span class="ref-vol">110</span>:479–491.</span></li><li id="b6">Gmachl, M. The precursors of the bee venom constituents apamin and MCD peptide are encoded by 2 genes in tandem which share the same 3&#x02032;-exon. <span><span class="ref-journal">J Biol Chem. </span>1995;<span class="ref-vol">270</span>:12704–12708.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=02O2BLEc_3AcojB7_MdKkq_cIqBcyaVVCCjGe7k9g&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7759523">PubMed</a>]</span></li><li id="b7">Hoffman, DR. Fatal reactions to hymenoptera stings. <span><span class="ref-journal">Allergy Asthma Proc. </span>2003;<span class="ref-vol">24</span>:123–127.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0nC978PypvmQGYmznA5JwUdLqCjNSEngmKTlHm5tw&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12776446">PubMed</a>]</span></li><li id="b8">Hoffman, DR. Hymenoptera venom allergens. <span><span class="ref-journal">Clin Rev Allergy Immunol. </span>2006;<span class="ref-vol">30</span>:109–128.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0deYkIKsKEpoa4FYR_eRqCuiw-8zWOe5I09DuAkcY&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16645223">PubMed</a>]</span></li><li id="b9">Kettner, A. Api m 6: a new bee venom allergen. <span><span class="ref-journal">J Allergy Clin Immunol. </span>2001;<span class="ref-vol">107</span>:914–920.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0dEQs2OZ6BTeUyNos0KYA_ZRw1O_c85FBxJuadQZ_&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11344362">PubMed</a>]</span></li><li id="b10">Kreil, G. Stepwise cleavage of the pro part of promelittin by dipeptidylpeptidase-IV &#x02013; evidence for a new type of precursor-product conversion. <span><span class="ref-journal">Eur J Biochem. </span>1980;<span class="ref-vol">111</span>:49–58.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=08wdF1RiSD9yViNw4he3RpMMoY7NbLXumnfblZAYJ&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7002560">PubMed</a>]</span></li><li id="b11">Larsen, JN. Allergen nomenclature. <span><span class="ref-journal">J Allergy Clin Immunol. </span>1996;<span class="ref-vol">97</span>:577–578.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0L90vvdw3mvdVG6SoN_f3fJ6hnK-DAXgytoKPQAlz&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8621841">PubMed</a>]</span></li><li id="b12">Marchler-Bauer, A. CD-search: protein domain annotations on the fly. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:W327–W331.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0ZOqxBkYg0FcB73nu0xZZd3D1iYbuNj5Kwwk1rkha&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15215404">PubMed</a>]</span></li><li id="b13">Mykles, DL. Cloning of tropomyosins from lobster (<em>Homarus americanus</em>) striated muscles: fast and slow isoforms may be generated from the same transcript. <span><span class="ref-journal">J Muscle Res Cell Motil. </span>1998;<span class="ref-vol">19</span>:105–115.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0dsOC1A2BbVpTQbfXL2_SnPRBLMjvmdYrZeMN65X-&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9536438">PubMed</a>]</span></li><li id="b14">Piersma, SR. Proteolytic processing of the peanut allergen Ara h 3. <span><span class="ref-journal">Mol Nutr Food Res. </span>2005;<span class="ref-vol">49</span>:744–755.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0TsC8Y1DmjkrzWeKn46Kisjbga4R7H8XJevxxKkzO&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15995984">PubMed</a>]</span></li><li id="b15">Rozen, S.  Primers on the WWW for general users and for Biologist programmers. In: Krawetc S. ,  editor. <span class="ref-journal">Bioinformatics, Methods, and Protocols: Methods in Molecular, Biology.</span> Totowa, NJ: Humana Press; 2000. pp. 365–386.</li><li id="b16">Shkenderov, S. A protease inhibitor in bee venom: identification, partial-purification and some properties. <span><span class="ref-journal">FEBS Lett. </span>1973;<span class="ref-vol">33</span>:343–347.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0raw5DBMgsdDoaDeqhkdZ_189Gr22UewPydNMb58a&amp;reftype=pubmed&amp;artid=1761135&amp;article-id=1761135&amp;iid=129405&amp;issue-id=129405&amp;jid=379&amp;journal-id=379&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/4728973">PubMed</a>]</span></li></ul></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">Wiley-Blackwell Online Open</span> are provided here courtesy of <br><b>Wiley-Blackwell, John Wiley &amp; Sons</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc1-->

