<html>
<head>
<title>FaST: Project Descriptions - Biosciences (BIOS) Division</title>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
<style type='text/css'>
  #myChaser {
    position:absolute;
    left:0px;
    top:300px;
    width:75px
  }
</style>
<link rel="stylesheet" href="../../../Templates/normal_links.css" type="text/css">
</head>
<body bgcolor="#ffffff" vlink="#CC00CC" alink="#9900FF" text="#000000" link="#9900CC" LEFTMARGIN='0' TOPMARGIN='0' MARGINWIDTH='0' MARGINHEIGHT='0'>
<div id='myChaser'> <a href="../../sci_ed.htm"><img name="home" src="../../../images/home.gif" width="75" height="30" border="0"
  ></a><br>
  <a href="../faq.html"><img name="faq" src="../../../images/faq.gif" width="75" height="22" border="0"
  ></a><br>
  <a href="https://applicationlink.labworks.org/applicationlink/default.htm"><img name="apply" src="../../../images/sci_ed_apply.gif" width="75" height="22" border="0"
  ></a><br>
  <a href="../contacts.htm"><img name="contact" src="../../../images/contact.gif" width="75" height="34" border="0"></a> 
</div>
<script language='JavaScript' src='../../../applicationlink/9-14-01/applicationLink/Blue%20Floater/chaser.js'></script>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td><img name="doe_seal" src="../../../images/ofc_scnc_r1_c1.gif" width="137" height="99" border="0" usemap="#doe_sealMap"
><img name="banner" src="../../../images/ofc_scnc_r1_c2.gif" width="503" height="99" border="0"
></td>
  </tr>
</table>
<table width="720" border="0" cellpadding="0">
  <tr> 
    <td bgcolor="#000000" width="11%" height="18"> 
      <div align="center"><font face="Arial, Helvetica, sans-serif" color="#FFFFFF" size="2"><b><a href="../about.html" class="white">About</a></b></font></div>
    </td>
    <td bgcolor="#000000" width="13%" height="18"> 
      <div align="center"><font face="Arial, Helvetica, sans-serif" color="#FFFFFF" size="2"><b><a href="../eligibility.html" class="white">Eligibility</a></b></font></div>
    </td>
    <td bgcolor="#000000" width="20%" height="18"> 
      <div align="center"><font face="Arial, Helvetica, sans-serif" color="#FFFF00" size="2"><b><a href="../project_desc.html" class="white">Project Descriptions</a></b></font></div>
    </td>
    <td bgcolor="#000000" width="17%" height="18"> 
      <div align="center"><font face="Arial, Helvetica, sans-serif" color="#FFFFFF" size="2"><b><a href="../finance.html" class="white">Financial Info</a></b></font></div>
    </td>
    <td bgcolor="#000000" width="15%" height="18"> 
      <div align="center"><font face="Arial, Helvetica, sans-serif" color="#FFFFFF" size="2"><b><a href="../dates.htm" class="white">Key Dates</a></b></font></div>
      </td>
    <td bgcolor="#000000" width="15%" height="18"> 
      <div align="center"><font face="Arial, Helvetica, sans-serif" color="#FFFFFF" size="2"><b><a href="../apply.html" class="white">How To Apply</a></b></font></div>
    </td>
  </tr>
</table>
<br>
<table width="723" border="0" cellpadding = "0" nowrap>
  <tr> 
    <td width="85"> <img src="../../shim.gif" width="75"> 
    </td>
    <td width="638" valign = "top"> 
      <table width="105%" border="0">
        <tr>
          <td width="31%"><img src="../../../images/header/Fast_RGB.gif" alt="FaST Logo"></td>
          <td width="69%">
            <h2 align="left"><font face="Arial, Helvetica, sans-serif">Faculty
            and Student Teams Program</font></h2>
          </td>
        </tr>
      </table>
      <h2><font face="Avian, Avant Garde, Arial, Helvetica, sans-serif"><b><img src="../../../images/choose/question.gif" width="35" height="35" align="middle" alt="questioning"></b> 
      </font><font face="Times New Roman">Project Descriptions<br>
      </font>
      </h2>
      <p><font face="Times New Roman" size="5"><font face="Times New Roman"><b>Argonne 
        National Laboratory</b></font><b><br>
      </b></font><b><font face="Times New Roman" size="4">
      The Biosciences (BIOS) Division</font></b></p>
      <p><font face="Times New Roman" size="3">Requesting applications from science or engineering faculty members
        at institutions serving students underrepresented in science, engineering,
      mathematics and technology to work on the following projects.</font></p>
      <p><b><font face="Times New Roman" size="3">Project Description</font></b></p>
      <p><font size="3" face="Times New Roman">Research in this division is aimed at defining the biological and medical
        hazards to humans from energy technologies and new energy options. Health-related
        studies are supported by fundamental research in scientific disciplines,
        including molecular and cellular biology, crystallography, biophysics,
        genetics, radiobiology, biochemistry, chemistry, and environmental toxicology.
        The research involves the integration of findings from investigations
        at the molecular, cellular, tissue, organ, and whole-animal levels, with
        the ultimate aim of applying these findings to problems of human health.
        The Division is organized into two scientific sections (Biophysics, and
        Functional Genomics), plus a Structural Biology Center that operates
        two beamlines at the Advanced Photon Source. Each section comprises several
        research groups with considerable interaction occurring among all groups.
        Divisional support facilities include an editorial office, a computer
        center, a biomedical library, and an instrument design and maintenance
      shop.</font></p>
      <p><font size="3" face="Times New Roman">Some representative examples of the research projects available for
        FaST teams are listed below. More detailed information regarding the
        myriad of research projects occurring at Argonne National Laboratory
        can be found online in our <em>Research Participation Catalog</em> (<a href="http://www.dep.anl.gov/catalog/catalog.htm">http://www.dep.anl.gov/catalog/catalog.htm</a>).
        The catalog contains the titles and descriptions of almost all research
        projects underway at Argonne National Laboratory. The catalog is available
        for downloading OR one can browse the catalog online.</font></p>
      <p><strong><font face="Times New Roman" size="3">Research Opportunities in the Biosciences Division</font></strong></p>
      <font size="3"><span style="mso-bidi-font-family: Times New Roman">
      <font face="Times New Roman">The
      basic mission of the Biosciences Division is to conduct multidisciplinary
      research that further increases understanding of the fundamental
      mechanisms of life and enables valuable advances in health protection,
      environmental restoration, energy production, industrial processing, and
      other applications.<span style="mso-spacerun:yes">&nbsp; </span>Research
      projects in the Division range from fundamental studies of DNA sequences
      using molecular biology and computational strategies to practical
      applications of genomic information, including selection of targets for
      protein structure determinations and protein engineering to elucidate
      relationships between protein structure and function.<span style="mso-spacerun:yes">&nbsp;
      </span>Within the broad goal of understanding biological mechanisms
      relevant to bioremediation, energy production, and protection of human
      health, the major objectives for </font>
 </span>
      <font face="Times New Roman"><span style="mso-bidi-font-size: 12.0pt; mso-bidi-font-family: Times New Roman; color: black"><a href="http://www.bio.anl.gov/research/"><span style="color:
black">research</span></a></span><span style="mso-bidi-font-size: 12.0pt; mso-bidi-font-family: Times New Roman">
      </span><span style="mso-bidi-font-family: Times New Roman">projects in the
      Division are: <span style="mso-spacerun:yes">&nbsp;</span><o:p>
      </o:p>
      </span></font>
      </font>
      <p class="MsoBodyText" style="line-height:normal;tab-stops:.5in"><span style="mso-bidi-font-family: Times New Roman"><font size="3"><span style="mso-tab-count:1"><font face="Times New Roman">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      </font>
      </span><font face="Times New Roman">(a) To characterize the mechanisms involved in regulation of gene
      expression through the development and use of tools for analysis of DNA,
      RNA, and proteins; <o:p>
      </o:p>
      </font></font></span></p>
      <p class="MsoBodyText" style="line-height:normal;tab-stops:.5in"><span style="mso-bidi-font-family: Times New Roman"><font size="3"><span style="mso-tab-count:1"><font face="Times New Roman">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      </font>
      </span><font face="Times New Roman">(b) To deepen understanding of protein stability and
      protein-protein interaction by using protein engineering techniques; <o:p>
      </o:p>
      </font></font></span></p>
      <p class="MsoBodyText" style="line-height:normal;tab-stops:.5in"><span style="mso-bidi-font-family: Times New Roman"><font size="3"><span style="mso-tab-count:1"><font face="Times New Roman">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      </font>
      </span><font face="Times New Roman"><span style="mso-spacerun:yes">&nbsp;</span>(c) To predict protein
      structure and function from gene sequence data and then use the
      predictions to select protein targets for structure determination;<o:p>
      </o:p>
      </font></font></span></p>
      <p class="MsoBodyText" style="line-height:normal;tab-stops:.5in"><span style="mso-bidi-font-family: Times New Roman"><font size="3"><span style="mso-tab-count:1"><font face="Times New Roman">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      </font>
      </span><font face="Times New Roman">(d) To probe genome sequences with both laboratory and
      computational techniques in order to discover regulatory pathways relevant
      to human health, environmental restoration, and energy production;<o:p>
      </o:p>
      </font></font></span></p>
      <p class="MsoBodyText" style="line-height:normal;tab-stops:.5in"><span style="mso-bidi-font-family: Times New Roman"><font size="3"><span style="mso-tab-count:1"><font face="Times New Roman">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      </font>
      </span><font face="Times New Roman">(e) To develop the analytical approaches and software required to
      exploit massively parallel computers for exploration of the
      three-dimensional structures of proteins; and <o:p>
      </o:p>
      </font></font></span></p>
      <p class="MsoBodyText" style="line-height:normal;tab-stops:.5in"><font size="3"><span style="mso-bidi-font-family: Times New Roman"><span style="mso-tab-count:1"><font face="Times New Roman">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      </font>
      </span><font face="Times New Roman">(f) To exploit fully the unique capabilities of
      </font> </span><font face="Times New Roman"><st1:place>
      <span style="mso-bidi-font-family: Times New Roman">Argonne</span></st1:place>
      <span style="mso-bidi-font-family: Times New Roman"> 's </span><st1:place>
      <st1:PlaceName>
      <span style="mso-bidi-font-family: Times New Roman">Structural</span></st1:PlaceName>
      <st1:PlaceName>
      <span style="mso-bidi-font-family: Times New Roman">Biology</span></st1:PlaceName>
      <st1:PlaceType>
      <span style="mso-bidi-font-family: Times New Roman">Center</span></st1:PlaceType>
      </st1:place>
      <span style="mso-bidi-font-family: Times New Roman"> (SBC) at the APS in
      exploring currently intractable questions involving reaction mechanisms
      and molecular structure<span style="mso-spacerun: yes">.</span><span style="mso-spacerun:yes">
      </span><o:p>
      </o:p>
      </span></font></font></p>
      <font face="Times New Roman" size="3"><span style="mso-bidi-font-family: Times New Roman"><span style="mso-tab-count:1">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      </span>The primary sponsors of work conducted within the Division are the </span><st1:country-region>
      <st1:place>
      <span style="mso-bidi-font-family: Times New Roman">U.S.</span></st1:place>
      </st1:country-region>
      <span style="mso-bidi-font-family: Times New Roman"> Department of Energy
      (DOE) and the National Institutes of Health (NIH).<span style="mso-spacerun:yes">&nbsp;
      </span>One of the facilities in the Division (the </span><st1:place>
      <st1:PlaceName>
      <span style="mso-bidi-font-family: Times New Roman">Structural</span></st1:PlaceName>
      <st1:PlaceName>
      <span style="mso-bidi-font-family: Times New Roman">Biology</span></st1:PlaceName>
      <st1:PlaceType>
      <span style="mso-bidi-font-family: Times New Roman">Center</span></st1:PlaceType>
      </st1:place>
      <span style="mso-bidi-font-family: Times New Roman"> at the Advanced
      Photon Source) is a user facility with customers (users) from other
      National Laboratories, universities, and industry. GM/CA (General Medicine
      and CAncer Institutes Collaborative Access Team) will also be a national
      user facility when up and running in 2 years. Descriptions of research
      programs within the Biosciences Division follow. <o:p>
      </o:p>
 </span></font>
      <p class="MsoNormal" align="center" style="text-align:center"><b style="mso-bidi-font-weight:
normal"><i style="mso-bidi-font-style:normal"><font face="Times New Roman" size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">BIOPHYSICS
      SECTION<o:p>
      </o:p>
      </span></font></i></b></p>
      <u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">
     <font face="Times New Roman" size="3">MACROMOLECULAR
      INTERACTIONS<o:p>
      </font>
      </span></u>
      <p>
      <font face="Times New Roman" size="3">A
      major research goal in biological science is to understand the
      relationship between the amino acid sequence of a protein and its
      three-dimensional structure, stability, and function. Because the
      interactions between the amino acids within a protein obey the same laws
      of physics that control interactions between proteins, study of the
      self-association properties of immunoglobulin light chains is relevant to
      the fundamental properties of all proteins. Antibody light chains are
      produced in large quantities by patients who have myeloma, a neoplasm.
      Because the proteins produced by two patients will be similar in
      three-dimensional structure but will differ in amino-acid sequence,
      differences in self-association (under various conditions of pH, ionic
      strength, and temperature) can be related to the physics that determines
      the protein structure and function. In addition, these studies provide
      increased understanding of the biophysical properties of these proteins
      that lead to disease complications in many patients and provide a model
      system for other, structurally related, protein-based diseases. We are
      using site-specific mutagenesis, molecular dynamics simulations, and novel
      bioinformatic approaches to help analyze experimental results.<span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"></o:p>
      </span></font></p>
      <p class="MsoNormal" style="tab-stops:45.0pt"><font size="3"><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman; text-transform: uppercase"><font face="Times New Roman">Protein
      Crystallography and Molecular Modeling</font></span></u><font face="Times New Roman"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><o:p>
      </o:p>
      </o:p>
      </span></font></font></p>
      <font face="Times New Roman" size="3">
     The
      principal aims of this program are the expression, isolation and
      characterization of biologically important macromolecules, the
      determination of their detailed three-dimensional structures in
      crystalline state, and the correlation of structure with biological
      function. The biomolecules under study include cytochromes, bacterial
      photosynthetic reaction center. The techniques used in this program are
      taken from a variety of disciplines including molecular biology, protein
      chemistry, chromatography, protein crystallography, and computer modeling
      of protein structures. Major equipment includes a rotating anode X-ray
      generator with an R-axis#2 data collection system and interactive computer
      color graphics terminals for manipulating macromolecules in three
      dimensions.</font>
      <p class="MsoBodyText3" style="line-height:normal"><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3">SBC/APS
      USER FACILITY<o:p>
      </o:p>
      </font>
      </span></u></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p></font></span><font face="Times New Roman">This program applies modern crystallographic methods to rapidly
      determine structures of biological macromolecules-proteins and nucleic
      acids-as single molecules, as multicomponent complexes, and complexed with
      bound ligands. A significant effort in this program is directed toward
      improving the methods for crystallographic investigation of macromolecular
      structure, by developing new and better methods and instruments to
      measure, process, and analyze diffraction data using cryocrystallography.
      The program operates two advanced x-ray beamlines at the Advanced Photon
      Source, for tuned, high-throughput, monochromatic x-ray diffraction data
      collection that is used to determine crystal structures. <st1:City>
      <st1:place>
      Crystal</st1:place>
      </st1:City>
      structures are being studied of chaperone proteins which direct the
      folding of protein receptors, important enzymes from pathogenic and
      thermolphilic organisms, and nucleic acids. Structures of proteins derived
      from genomic analysis as part of our structural genomic initiative are
      being determined at this facility. Major equipment includes undulator and
      bending magnet beamlines, rotating-anode x-ray generator, with imaging
      plate detector, modern workstations with large capacity data-storage
      disks, interactive graphics workstations for molecular modeling, HPLC,
      FPLC, and ectrophoresis equipment, and all necessary facilities and
      equipment for molecular biology, molecular genetic manipulations of DNA,
      protein purification and crystallization, and activity assays.</font></font></p>
      <p class="MsoNormal" style="tab-stops:.6in"><font size="3"><span style="mso-bidi-font-size: 10.0pt"><font face="Times New Roman"><o:p></font></span><u><font face="Times New Roman"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">PROTEIN
      ENGINEERING<o:p>
      </o:p>
      </span></font></u></font></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3"><o:p>
      This program is aimed at a generalized understanding of how the
      three-dimensional structure of a membrane protein defines its function.
      Historically, our work has focused on the bacterial photosynthetic
      reaction center, a transmembrane protein complex that functions in the
      process by which light energy is converted into chemical energy. More
      recently, our research focus has expanded to the development of a
      generalized system for the heterologous expression, purification, and
      crystallization of any membrane protein for structure determination or
      functional characterization. Techniques consistently exercised include
      gene cloning with plasmid vectors, PCR amplification of DNA sequences, DNA
      sequencing, protein and DNA gel electrophoresis, protein expression and
      purification, spectroscopy, bioassay of mutant phenotypes, and protein
      crystallization. We are in the process of automating many of our routine
      methodologies using robots recently installed within the division.<o:p>
      </o:p>
      </font>
      </span></p>
      <p class="MsoNormal" style="text-align:justify"><font size="3"><span style="mso-bidi-font-size: 10.0pt"><font face="Times New Roman"><o:p></font></span><u><font face="Times New Roman"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">STRUCTURAL
      STUDIES OF MACROMOLECULAR ASSEMBLIES<o:p>
      </o:p>
      </span></font></u></font></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p>
      </font>
      </span><font face="Times New Roman">Recognition of biological macromolecules and their interaction and
      assembly into larger supermacromolecular structures are at the heart of
      many important processes in molecular and cellular biology. For example,
      macromolecular assembly occurs in protein biosynthesis, in the recognition
      of receptors by protein hormones, in the folding of proteins, and in the
      recognition of and binding to nucleic acids by proteins that regulate the
      expression of genetic information. We are studying macromolecular
      assemblies at the atomic and molecular levels by x-ray crystallography, in
      particular the protein-protein interactions of molecular chaperones of the
      hsp60 and hsp70 classes, and large oligomeric enzymes, and protein-nucleic
      acid complexes. Because the crystals of macromolecular assemblies are
      usually small and fragile and have large unit cell dimensions, they
      diffract weakly. Furthermore, these crystals have large, complex
      structures and their structure determination is experimentally demanding.
      These studies take advantage of the Advanced Photon Source and the <st1:place>
      <st1:PlaceName>
      Structural</st1:PlaceName>
      <st1:PlaceName>
      Biology</st1:PlaceName>
      <st1:PlaceType>
      Center</st1:PlaceType>
      </st1:place>
      and the <st1:place>
      <st1:PlaceName>
      Midwest</st1:PlaceName>
      <st1:PlaceType>
      Center</st1:PlaceType>
      </st1:place>
      for Structural Genomics facilities at <st1:place>
      Argonne</st1:place>
      . The techniques being used include molecular biology and biochemistry,
      liquid chromatography and electrophoresis, and synchrotron-based x-ray
      crystallography. Robotic workstations support purification,
      crystallization, data collection and structure determination.</font></font></p>
      <p class="MsoNormal" style="tab-stops:.6in"><font size="3"><b style="mso-bidi-font-weight:normal"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p></font></span></b><u><font face="Times New Roman"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">HIGH
      THROUGHPUT APPROACHES TO STUDY PROTEIN FUNCTION<o:p>
      </o:p>
      </span></font></u></font></p>
      <p class="MsoNormal" style="text-align: justify; tab-stops: .6in; margin-left: 0"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p>
      </font>
      </span><font face="Times New Roman">The abundance of genomic sequence data from different organisms
      provides an opportunity to accelerate our understanding of protein
      structure and function. However, optimal utilization of this information
      requires the development of high throughput methods for the generation of
      expression clones and the evaluation of protein function. We are
      developing automated methods for high throughput gene cloning and
      expression, site-specific mutagenesis, and the study of protein function.
      A Beckman Coulter Core System with integrated liquid handling stations and
      supporting devices provides the capability<span style="mso-spacerun:yes">&nbsp;
      </span>for high throughput production of expression clones for structural
      genomics and other large-scale programs that aim to characterize protein
      structure and function. This comprehensive strategy provides an
      alternative to the single protein approach that has previously dominated
      cell biology. The current cloning and analysis process spans four days
      with a maximum linear throughput of 400 targets per production run. The
      output generated from the expression cloning process is a 96-well plate
      map that specifies the location of soluble expression clone plasmids.
      Although developed for structural genomics, the experience gained by
      implementation of these initial protocols will provide a platform for
      extension of the system capabilities for application in other growth areas
      of high throughput molecular biology including site-specific mutagenesis,
      phage display, and protein interaction studies.</font></font></p>
      <p class="MsoNormal" style="text-indent: -.5in; margin-left: 0.5" align="left"><font face="Times New Roman" size="3"><b style="mso-bidi-font-weight:normal"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><o:p></span></b><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><u>BRIDG</u></span><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">I
      BRIDGING
      BIOSCIENCE AND NANO-SCIENCE:<span style="mso-spacerun: yes; mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">&nbsp;
      </span>DEVELOPING NOVEL TOOLS FOR
      UNDERSTANDING PROTEIN-LIGAND INTERACTIONS FOR SYSTEMS BIOLOGY<o:p>
      </o:p>
      </span></u></font></p>
      <p class="MsoNormal" style="text-indent: -.5in; margin-left: 0.5" align="left"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><u><font face="Times New Roman">HPUT</font></u>
      <font face="Times New Roman">
      AA</font></span> <font face="Times New Roman"> key aspect of the functional characterization of genomes is the
      understanding and consequent mapping of interactions among proteins and
      between proteins and ligands at systems level. This project is involved in
      the development of novel methods to make this mapping possible by engaging
      the combination of bioscience and nanotechnology. Particular research
      efforts for this project emphasize the use of single molecule manipulation
      and detection methods (such as scanning fluorescence correlation
      spectroscopy) to select ligands directly from combinatorial libraries.
      Since unveiling the mechanism of protein-ligand interactions plays an
      important role in elucidating protein functionality, we will also develop
      new tools to solve the protein binding domain structure using surface
      enhanced Raman spectroscopy.</font></font></p>
      <p class="MsoNormal"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p>
      </font>
      </span><font face="Times New Roman"><b style="mso-bidi-font-weight:
normal"><i style="mso-bidi-font-style:normal"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">FUNCTIONAL
      GENOMICS SECTION</span></i></b><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><o:p>
      </o:p>
      </span></font></font></p>
      <p class="MsoNormal" style="tab-stops:.6in"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p></font></span><font face="Times New Roman"><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">PROTEOMICS</span></u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><o:p>
      </o:p>
      </span></font></font></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p>
      </font>
      </span><font face="Times New Roman">Two-dimensional gel electrophoresis coupled with computerized image
      and data analysis is being used to characterize the normal protein
      composition of cells and to detect changes in response to environmental
      pressures. Current studies are focused on the analysis and identification
      of proteins produced by microorganisms. In addition to two-dimensional gel
      electrophoresis of proteins (isoelectric focusing combined with sodium
      dodecyl sulfate polyacrylamide gel electrophoresis), this project involves
      the use of image and data analysis algorithms, World Wide Web databases,
      and mass spectrometry. The construction and maintenance of interactive
      Internet databases is an important part of the data presentation for this
      project.</font></font></p>
      <font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-fareast-font-family: Times New Roman; mso-bidi-font-family: Times New Roman; mso-ansi-language: EN-US; mso-fareast-language: EN-US; mso-bidi-language: AR-SA">
      <font face="Times New Roman"><br clear="all" style="mso-special-character:line-break;page-break-before:always">
      </font>
      </span>
      <font face="Times New Roman"><st1:place><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman; text-transform: uppercase">meta</span></u></st1:place><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman; text-transform: uppercase">lloproteomics</span></u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><o:p>
      </o:p>
      </span></font>
      </font>
      <p class="MsoBodyText3" style="line-height:normal"><font face="Times New Roman" size="3"><o:p>
      Our laboratory is conducting research into the interaction of trace metals
      such as copper and zinc with proteins and how these interactions change
      during cellular processes to affect protein activity. It is our contention
      that metal:protein are regulated dynamically during numerous cellular
      processes and such regulations may act as a form of post-translational
      regulation of a protein’s structure and function. As a system of study,
      we are using several models of angiogenesis, the process by which new
      blood capillaries are formed. It is well-known that angiogenesis is high
      sensitive, both <i style="mso-bidi-font-style:normal">in vitro</i> and <i style="mso-bidi-font-style:normal">in
      vivo</i>, to bioavailable copper. We are employing and developing numerous
      methodologies to investigate dynamism in the metalloproteome during
      endothelial cell angiogenesis. These include use of the Advanced Photon
      Source for high resolution x-ray fluorescence metal mapping <i style="mso-bidi-font-style:normal">in
      situ</i>, development of separative methodologies for metalloproteins, and
      isolation and identification of novel metalloproteins used by these cells
      as they undergo morphogenetic differentiation.</font></p>
      <p class="MsoNormal" style="tab-stops:45.0pt"><font face="Times New Roman" size="3"><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman; text-transform: uppercase">biochemical
      toxicology</span></u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><o:p>
      </o:p>
      </span></font></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p>
      </font>
      </span><font face="Times New Roman">This research program is designed to investigate health effects of
      toxic metals to which humans may be environmentally or occupationally
      exposed. One research area focuses on the role of pregnancy, lactation, or
      ovariectomy in the susceptibility of animals to bone loss after cadmium
      exposure. Mechanisms of cadmium action on bone are studied in isolated
      bone cells in culture. Molecular pathways of cadmium action are
      investigated for specific genes known to influence bone resorption and by
      gene expression microarray to identify unknown genes. Another research
      area focuses on the biochemical pathways for toxic heavy metals, including
      their uptake and tissue deposition. The role of metallothionein, a
      metal-binding protein, is studied using normal and metallothionein-deficient
      mice. Measurements of calcium and cadmium content in tissues are performed
      using atomic absorption spectroscopy.</font></font></p>
      <p class="MsoNormal" style="tab-stops:45.0pt"><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3">ANTIBODY
      ENGINEERING<o:p>
      </o:p>
      </font>
      </span></u></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3"><o:p>
      With the recent completion of two hundred sequenced bacterial and six
      eukaryotic genomes, the scientific community is entering a
      “post-genomics era”. To add value to this accomplishment, the
      community’s attention is now directed at determining the function of the
      thousands of gene products, proteins, in each cell. Traditionally, one
      valuable type of reagent that is widely used to probe cells and learn when
      the protein is synthesized, where it is localized, and what it is
      associated with in the cell is the antibody. However, it typically takes
      two to three months to generate rabbit or mouse antibodies to each
      individual protein, and there is limited control by the investigator on
      the quality of the antibodies generated by the immunized animals. To
      overcome the limitations of antibody generation and to meet the need for
      thousands of antibodies, we utilize phage-display to isolate high-affinity
      and selectively generate designer antibodies to any protein. Such
      antibodies will be used to 1) affinity purify the target proteins from
      cells, and then identify interacting proteins through gel electrophoresis
      and mass spectrometry, 2) promote crystallization of proteins for x-ray
      diffraction studies at the Advanced Photon Source (APS), and 3) format the
      antibodies as arrays onto glass slides, with which one can measure the
      concentrations of many proteins simultaneously in cells, as they respond
      to stimuli or become diseased. <o:p>
      </o:p>
      </font>
      </span></p>
      <font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-fareast-font-family: Times New Roman; mso-bidi-font-family: Times New Roman; mso-ansi-language: EN-US; mso-fareast-language: EN-US; mso-bidi-language: AR-SA">
      <font face="Times New Roman"><br clear="all" style="mso-special-character:line-break;page-break-before:always">
      </font>
      </span>
      <font face="Times New Roman"><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman">COMBINATORIAL
      BIOLOGY</span></u><span style="mso-bidi-font-family: Times New Roman"><o:p>
      </o:p>
      </span></font>
      </font>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3">A
      major project within this laboratory is analysis of the mechanism by which
      small molecules bind to proteins. In spite of the fact that multiple three
      dimensional pictures have been obtained of protein-ligand pairings, the
      pace of rational drug design has been hindered by a lack of global
      coherent rules underlying small molecule-protein interactions. By studying
      the pattern of virally-presented combinatorial peptides binding to common
      metabolites such as ATP and glucose, and correlation those sequences with
      three dimensional structures of known metabolite/protein pairs, we aim to
      create a database of peptide sequences which are predictive for metabolite
      binding in known protein sequences. Information derived from this work can
      eventually be extended to combinatorial chemistry-derived drugs predict
      potential targets within the human body prior to clinical trials.<o:p>
      </o:p>
      </font>
      </span></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3"><o:p>
      Mortality in over 90% of cancer patients is the result not of the effects
      of the primary lesion, but the crowding out of normal cells by metastatic
      tumor cells at secondary sites within the body. Cellular migration of
      tumors is dependent upon both the successful disruption of cell-cell
      contacts at the primary site and the erection of proper scaffolding at the
      secondary site(s). A major step in scaffolding construction must include
      the attraction of new blood vessels (or angiogenesis) to feed and
      oxygenate the new tumor. These new vessels are primarily built with a
      class of cells called endothelial cells, which are one of the very few
      cell types in mammals that have the ability to migrate post-embryonically.
      Our group is using a systems biology approach to identify the major
      protein players involved in endothelial cell migration, differentiation,
      and morphogenesis in order to develop a new class of side effect-free
      anti-cancer drugs.<o:p>
      </o:p>
      </font>
      </span></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3">Our
      recently acquired ability to decode the genomes of both uni- and
      multi-cellular organisms (including man) has given us the blueprint for
      understanding biological processes at a level not before possible. The
      conversion of sequence information into the who, what, where, when, how
      and why of any one biological process, however, will require the
      simultaneous application of a number of intersecting methodologies.
      Knowledge of the three dimensional structure of a protein will in turn
      frequently translate into functional information, allowing for the
      elucidation of unexpected links in biological pathways not as amenable to
      discovery by the more traditional hypothesis-driven research methods.<o:p>
      </o:p>
      </font>
      </span></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3">Small
      angle x-ray scattering (SAXS) of proteins and macromolecular complexes in
      solution has been shown to reliably yield information about the size and
      shape of proteins. More recently it has been demonstrated by our group
      that wide angle scattering patterns (WAXS) obtained at high flux third
      generation synchrotron beam lines are not only sensitive to protein
      conformation states, but the scattering patterns generated can be
      quantitatively compared to detailed structural models. This data, made
      possible by third generation synchrotron sources, provides a rich source
      of structural information that has not yet been exploited. Given the broad
      range of conditions and particle sizes amenable to solution x-ray
      scattering, a combination of HTP SAXS and WAXS analysis downstream from a
      large-scale protein production facility has the potential to generate
      information on the size, shape and structural class (i.e. fold) of every
      expressed protein.<o:p>
      </o:p>
      </font>
      </span></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3">This
      project proposes to establish and standardize a SAXS/WAXS protocol to
      collect structural data on a large number of proteins; construct a
      database of small and wide angle scattering patterns; and to use this data
      to assign a structural class to each of the proteins studied. The long
      term goal is a high throughput method for experimentally assigning the
      structural class of every protein that can be expressed from a genome.
      This work will require the high brilliance focused undulator beams at the
      Advanced Photon Source (APS) that generate a flux density 100-fold greater
      than those of second generation sources.<o:p>
      </o:p>
      </font>
      </span></p>
      <p class="MsoNormal" style="tab-stops:45.0pt"><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman; text-transform: uppercase"><font face="Times New Roman" size="3">microbiology<o:p>
      </o:p>
      </font>
      </span></u></p>
      <p class="MsoNormal" style="text-align:justify;mso-layout-grid-align:none;
text-autospace:none"><font face="Times New Roman" size="3">Microbes can convert
      agricultural feedstocks, such as sugars derived from corn, into valuable
      chemicals, and have long been exploited for this ability. Efforts are now
      in progress to expand their use to include the production of
      larger-volume, less-expensive chemicals through new &quot;green&quot;
      processes. At Argonne, modern techniques in microbiology, genetic
      engineering, and enzymology are applied in &quot;metabolic
      engineering&quot;, in which metabolic pathways are altered by adjusting
      gene expression or introducing new enzymes, thereby channeling metabolites
      to the desired end products. Another major activity of the Microbiology
      Group involves a collaboration with structural biologists in determining
      the structures of diverse microbial proteins. This project, funded by NIH,
      seeks to accelerate greatly the rate at which protein structures can be
      solved, as well as provide a complete library of potential protein
      structures. In that effort, we have developed new expression vectors and
      novel cell culture methods that facilitiate high-throughput production of
      proteins. Additional vectors under development promise to improve the
      solubility of expressed proteins and to allow their production in host
      cells other than E. coli.<o:p>
      </o:p>
      </font></p>
      <p class="MsoBodyText3" style="line-height:normal"><font face="Times New Roman" size="3">Microbes
      can convert cheap, renewable resources to valuable products and have long
      been exploited for this ability. Efforts are now in progress to expand
      their use to include the production of larger-volume, less-expensive
      chemicals. New processes and products to be developed will reduce both
      dependency on petroleum and the environmental liabilities of some
      industrial chemicals. At <st1:place>
      Argonne</st1:place>
      , modern techniques in microbiology, genetic engineering, and enzymology,
      as well as classical approaches, are being applied in this effort.
      Addition of foreign genes or alteration of gene expression, called
      &quot;metabolic engineering&quot;, attempts to alter the metabolic
      pathways of the microbes to produce different chemicals. Site-specific
      mutagenesis of proteins attempts to change the specificity or stability of
      enzymes to create novel catalysts that will carry out useful reactions not
      performed by naturally occurring enzymes. Strains developed by these
      approaches are evaluated in laboratory scale fermentations, which are then
      optimized for production of the desired metabolites.</font></p>
      <p class="MsoNormal"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p></font></span><u><font face="Times New Roman"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman; text-transform: uppercase">terrestrial
      ecology<o:p>
      </o:p>
      </span></font></u></font></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3">The
      ability of plants to adapt or respond to a changing environment is
      dependent on homeostatic capacities that minimize the cost of growth and
      biomass allocation. Plants’ responses to environmental stresses, such as
      nutrient limitation or anthropogenic effects such as elevated CO<sub>2</sub>,
      suggest that they have a centralized system of stress response involving
      changes in nutrient and water use, carbon allocation, hormonal balances,
      and reliance on mycorrhizal fungi. Our research addresses mechanisms
      controlling plants with obligate and facultative dependency on the
      mycorrhizal symbiosis and the relative importance of these mechanisms in
      various plant life forms. Our overall objective is to determine whether a
      major mechanism of control is the balance between photo-assimilate supply
      to the roots and the host’s need for nutrients. To address this
      objective, two general questions will be investigated:<span style="mso-spacerun:yes">&nbsp;
      </span>(1) what are the mechanisms controlling photo-assimilate allocation
      to the fungus and nutrient inflow to the plant? (2) Will the host’s
      dependence on supplied nutrients influence its ability to adjust to a
      changing environment? (3) Does a relationship exist between net C gain of
      the host and the amount and activity of mycorrhizal fungi?<i style="mso-bidi-font-style:normal"><o:p>
      </o:p>
      </i></font></span></p>
      <p class="MsoNormal" style="tab-stops:45.0pt"><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman; text-transform: uppercase"><font face="Times New Roman" size="3">terrestrial
      CARBON PROCESSES<o:p>
      </o:p>
      </font>
      </span></u></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman" size="3">Concerns
      over rising concentrations of atmospheric CO<sub>2</sub> have increased
      interest in the capacity of soil to serve as a carbon sink. The amount of
      carbon stored in world soils is estimated at more than twice the carbon in
      vegetation or in the atmosphere. Thus, even relatively small changes in
      soil carbon storage per unit area could have a significant impact on the
      global carbon balance. Soil carbon may be stabilized because of its
      biochemical recalcitrance or by incorporation into organomineral complexes
      with clays. Soil structure also plays a dominant role in controlling
      microbial access to substrates and, thus, relatively labile organic
      material can be physically protected from decomposition by incorporation
      into soil aggregates. This project is researching the biological
      mechanisms involved in the formation, stabilization, and degradation of
      aggregates and how the aggregation process, in turn, influences soil
      carbon dynamics. This information is necessary (1) to identify management
      practices that maximize soil carbon sequestration and (2) to determine the
      potential for terrestrial ecosystems to serve as a sink for elevated
      concentrations of atmospheric CO<sub>2</sub>. This project involves
      application of soil physical and biological fractionation techniques and
      stable isotope measurements to samples obtained from elevated CO<sub>2 </sub>experiments
      and sites with long-term plots representing alternative land management
      strategies.<i style="mso-bidi-font-style:normal"><o:p>
      </o:p>
      </i></font></span></p>
      <p class="MsoNormal" style="tab-stops:.6in"><u><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman; text-transform: uppercase"><font face="Times New Roman" size="3"><o:p>
      synchrotron-based environmental research<o:p>
      </o:p>
      </font>
      </span></u></p>
      <p class="MsoNormal" style="text-align:justify;tab-stops:.6in"><font size="3"><span style="mso-bidi-font-size: 10.0pt; mso-bidi-font-family: Times New Roman"><font face="Times New Roman"><o:p>
      </font>
      </span><font face="Times New Roman">This research program in synchrotron-based environmental research
      is aimed at exploring applications of new advances in x-ray physics to
      understanding problems in environmental science. The principal goal of
      this program is to address general issues concerning the bioavailability
      of contaminants in the environment, with a particular emphasis on the
      mobility, uptake mechanisms, transformations, and toxicity of metals and
      organic chemicals in natural soils. Research projects include both the
      study of bulk samples by using x-ray absorption spectroscopy and the study
      of microscopic samples and spatial variations on the micron length scale
      by using x-ray fluorescence imaging, phase contrast imaging, and x-ray
      absorption spectroscopy with micron-sized spots. Particular research
      interests within the group include:<span style="mso-spacerun:yes">&nbsp; </span>(1)
      the investigation of mineral-microbe interactions so as to better
      understand the role of these interactions in the fate and transport of
      heavy metal contaminants; and (2) the study of newly created,
      highly-reactive materials for the selective removal of radioactive
      contaminants from DOE waste storage tasks.</font></font></p>
      <p><strong><font face="Times New Roman" size="3">Applicants Responsibilities and Relationship to Project</font></strong></p>
      <p><font face="Times New Roman" size="3">Applicants will receive support under the Department of Energy Faculty
        and Student Team (FaST) Research Program to work collaboratively with
        the project research team at the Laboratory for up to 10 weeks during
        the year starting during the summer of 2006. Summer and academic year
        visits to Argonne Lab will be scheduled by mutual agreement between the
        Research Project Directors at Argonne National Laboratory (ANL) and the
        successful applicant. Faculty will be expected to identify students from
        their campus to participate in the Undergraduate Research Participation
        programs offered by the Department of Energy at ANL. Ideally, faculty
        will provide some mentorship and/or advising support to students during
        the summer research activities. It is expected that the faculty member
        will become an integral part of the research team working on this project
        and will support the project through the academic year on her or his
        campus.</font></p>
      <p><b><font face="Times New Roman" size="3">Qualifications of Ideal Candidate</font></b></p>
      <table width="90%" border="0" cellspacing="0" cellpadding="5">
        <tr valign="top"> 
          <td><font face="Times New Roman" size="3">Faculty:</font></td>
          <td><font face="Times New Roman" size="3"><span style="mso-fareast-font-family: Times New Roman; mso-ansi-language: EN-US; mso-fareast-language: EN-US; mso-bidi-language: AR-SA">Ph.D. in <span style="color: blue; mso-bidi-language: AR-SA; mso-fareast-language: EN-US; mso-ansi-language: EN-US; mso-fareast-font-family: Times New Roman; font-family: Arial; font-size: 10.0pt"><font color="#000000">biological
            sciences, biophysics, bioinformatics</font></span><span style="color:black">,&nbsp;biochemistry,
            or related field. Proven experimental abilities. Established record
            of publication in field is preferred. Works well in a collaborative
            environment with students and other researchers. Currently teaches
            and collaborates with students in his/her field. Willing to work at
            ANL for an extended period (2-3 summer months, or longer).</span></span></font></td>
        </tr>
        <tr valign="top"> 
          <td><font face="Times New Roman" size="3">Student:</font></td>
          <td><font face="Times New Roman" size="3">Working towards a BS in physics, materials science, biological
            sciences with strong interests in experimental research. Works well
            in collaboration with faculty, other students, and researchers. Willing
          to work at ANL for an extended period.</font></td>
        </tr>
      </table>
      <p><b><font face="Times New Roman" size="3">Support and Financial Commitments</font></b></p>
      <p><font face="Times New Roman" size="3">See <a href="../finance.html">Financial Information</a>.</font></p>
      <p><b><font face="Times New Roman" size="3">For More Information contact:</font></b>      </p>
      <table width="90%" border="0" cellpadding="0" cellspacing="0">
        <tr valign="top"> 
          <td width="50%"> 
            <p><font face="Times New Roman" size="3">Harold W. Myron<br>
              Director<br>
              Division of Educational Programs <br>
              Argonne National Laboratory<br>
              E-mail: <a href="mailto:hmyron@dep.anl.gov">hmyron@dep.anl.gov</a><br>
              Phone: 630-252-4114</font></p>
          </td>
          <td width="50%"><font face="Times New Roman" size="3">Linda Phaire-Washington<br>
            Senior Program Leader<br>
            Division of Educational Programs <br>
            Argonne National Laboratory <br>
            E-mail: <a href="mailto:washington@dep.anl.gov">washington@dep.anl.gov</a><br>
          Phone: 630-252-1751</font></td>
        </tr>
      </table>
      <p><a href="http://www.dep.anl.gov"><font face="Times New Roman" size="3"><br>
        http://www.dep.anl.gov</font></a></p>
      <p>&nbsp;</p>
    </td>
  </tr>
</table>
<map name="doe_sealMap"> 
  <area shape="circle" coords="38,37,27" href="http://www.energy.gov">
</map>
</body>
</html>
