<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Genome Research"><meta name="citation_title" content="Draft Sequencing and Comparative Genomics of Xylella fastidiosa Strains Reveal Novel Biological Insights"><meta name="citation_authors" content="Anamitra Bhattacharyya, Stephanie Stilwagen, Gary Reznik, Helene Feil, William S. Feil, Iain Anderson, Axel Bernal, Mark D'Souza, Natalia Ivanova, Vinayak Kapatral, Niels Larsen, Tamara Los, Athanasios Lykidis, Eugene Selkov, Jr., Theresa L. Walunas, Alexander Purcell, Rob A. Edwards, Trevor Hawkins, Robert Haselkorn, Ross Overbeek, Nikos C. Kyrpides, and  Paul F. Predki"><meta name="citation_date" content="2002 October"><meta name="citation_issue" content="10"><meta name="citation_volume" content="12"><meta name="citation_firstpage" content="1556"><meta name="citation_doi" content="10.1101/gr.370702"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=187531&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=187531&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=187531"><meta name="citation_pmid" content="12368248"><meta name="DC.Title" content="Draft Sequencing and Comparative Genomics of Xylella fastidiosa Strains Reveal Novel Biological Insights"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Cold Spring Harbor Laboratory Press"><meta name="DC.Contributor" content="Anamitra Bhattacharyya"><meta name="DC.Contributor" content="Stephanie Stilwagen"><meta name="DC.Contributor" content="Gary Reznik"><meta name="DC.Contributor" content="Helene Feil"><meta name="DC.Contributor" content="William S. Feil"><meta name="DC.Contributor" content="Iain Anderson"><meta name="DC.Contributor" content="Axel Bernal"><meta name="DC.Contributor" content="Mark D'Souza"><meta name="DC.Contributor" content="Natalia Ivanova"><meta name="DC.Contributor" content="Vinayak Kapatral"><meta name="DC.Contributor" content="Niels Larsen"><meta name="DC.Contributor" content="Tamara Los"><meta name="DC.Contributor" content="Athanasios Lykidis"><meta name="DC.Contributor" content="Eugene Selkov, Jr."><meta name="DC.Contributor" content="Theresa L. Walunas"><meta name="DC.Contributor" content="Alexander Purcell"><meta name="DC.Contributor" content="Rob A. Edwards"><meta name="DC.Contributor" content="Trevor Hawkins"><meta name="DC.Contributor" content="Robert Haselkorn"><meta name="DC.Contributor" content="Ross Overbeek"><meta name="DC.Contributor" content="Nikos C. Kyrpides"><meta name="DC.Contributor" content="Paul F. Predki"><meta name="DC.Date" content="2002 October"><meta name="DC.Identifier" content="10.1101/gr.370702"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody3.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs3.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Draft Sequencing and Comparative Genomics of Xylella fastidiosa Strains Reveal Novel Biological Insights</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=00iHpc0TbjDzuflp5TrT-zANDLGgMAqtaGmzg7kpc&amp;reftype=other&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-genores.gif" alt="Logo of genores" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" coords="0,58,94,74" href="redirect3.cgi?&amp;&amp;auth=0tW30r6IaXfSQh4HNSKNo-_KqnbIyNNr1XlYms5zP&amp;reftype=publisher&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.cshlpress.com" alt="CSHL Press" title="CSHL Press" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="rect" coords="95,58,196,74" href="redirect3.cgi?&amp;&amp;auth=0lGEJ1nPf6zO2Le4UaxdIpWq3qrUgPxTBVoSRnQbO&amp;reftype=publisher&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.genome.org" alt="Journal Home" title="Journal Home" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="rect" coords="197,58,298,74" href="redirect3.cgi?&amp;&amp;auth=0V3vG_5fXTyLTechj4D2TT-1JxIEmQwybh5m5c5Wl&amp;reftype=publisher&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.genome.org/subscriptions" alt="Subscriptions" title="Subscriptions" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="rect" coords="299,58,390,74" href="redirect3.cgi?&amp;&amp;auth=0zOOK8KkSchzJRrVF1FRjALxgs7eZclzHUGzgTQse&amp;reftype=publisher&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://genome.cshlp.org/misc/etoc.dtl" alt="eTOC Alerts" title="eTOC Alerts" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="rect" coords="391,58,499,74" href="redirect3.cgi?&amp;&amp;auth=0toe5sw6YSa2pwrLUqxqTci-c8wElsCTPDtJlTAoJ&amp;reftype=publisher&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biosupplynet.com" alt="BioSupplyNet" title="BioSupplyNet" target="pmc_ext" onclick="focuswin('pmc_ext')"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=102&amp;action=archive">Genome Res</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=6010">v.12(10); Oct 2002</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=187531&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=187531&amp;blobtype=pdf" class="sidefm-pmclink">PDF (430K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=6010" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=102&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=00GeNpAtD3_IhKVR3wE5PmKeunJCvo-A6ia6EiQBt&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12368248">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=0LA0sUMG2Ovbgjzq41OBLs-egeyDgOmGP9ebvOAWk&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=12368248">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0ZasInKwqWRtWzcEEKGn-AZ0uA52YPHZk07ffYkv-&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=12368248">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0Zi3cpZxnRjvz2g9qayHd39ILVYXcOEYnUS1DW5XZ&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=187531&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=02WlH9Ghbg70Skr3VAAP8HYb7TtRqAMU_fK4QuQxw&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=187531">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0DQETkCRaEfaWT2yP0ab2bNBbWOkC4sP0BBN0y5mX&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Bhattacharyya+A%5Bauth%5D" class="sidefm-pmclink"> Bhattacharyya, A.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=06MLVkUHh8H-V0rJV55dabIHUubw0jOnAq0jh5Afw&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Stilwagen+S%5Bauth%5D" class="sidefm-pmclink"> Stilwagen, S.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0NEVhczXOmOEExc4BXREKUBDpOzAu7VZUV-CxNAdi&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Reznik+G%5Bauth%5D" class="sidefm-pmclink"> Reznik, G.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0XYVcQqpP2ub1Pc_BWLO6EOZQm7AN77-S1AoTZkru&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Predki+PF%5Bauth%5D" class="sidefm-pmclink"> Predki, P.</a></div></div></div></td><td rowspan="103" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-genores.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">Genome Res. </span><span class="citation-publication-date">2002 October; </span><span class="citation-volume">12</span><span class="citation-issue">(10)</span><span class="citation-flpages">: 1556–1563. </span></div><div><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1101/gr.370702.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC187531</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 2002, Cold Spring Harbor Laboratory Press</div><div class="fm-title">Draft Sequencing and Comparative Genomics of <em>Xylella fastidiosa</em> Strains Reveal Novel Biological Insights</div><div class="fm-author contrib-group">Anamitra Bhattacharyya,<sup>1,</sup><sup>8</sup> Stephanie Stilwagen,<sup>2</sup> Gary Reznik,<sup>1</sup> Helene Feil,<sup>3</sup> William S. Feil,<sup>4</sup> Iain Anderson,<sup>1</sup> Axel Bernal,<sup>1</sup> Mark D'Souza,<sup>1</sup> Natalia Ivanova,<sup>1</sup> Vinayak Kapatral,<sup>1</sup> Niels Larsen,<sup>1</sup> Tamara Los,<sup>1</sup> Athanasios Lykidis,<sup>1</sup> Eugene Selkov, Jr.,<sup>1</sup> Theresa L. Walunas,<sup>1</sup> Alexander Purcell,<sup>4</sup> Rob A. Edwards,<sup>5</sup> Trevor Hawkins,<sup>2</sup> Robert Haselkorn,<sup>6</sup> Ross Overbeek,<sup>1</sup> Nikos C. Kyrpides,<sup>1</sup> and  Paul F. Predki<sup>2,</sup><sup>7</sup><div class="fm-affl"><sup>1</sup>Integrated Genomics, Inc., Chicago, Illinois 60612, USA; <sup>2</sup>Department of Energy, Joint Genome Institute, Walnut Creek, California 94598, USA; <sup>3</sup>Department of Plant and Microbial Biology, University of California, Berkeley, California 94720, USA; <sup>4</sup>Department of Environmental Science, Policy, and Management, University of California, Berkeley, California 94720, USA; <sup>5</sup>Department of Microbiology, University of Tennessee, Memphis, Tennessee 38163, USA; <sup>6</sup>Departments of Molecular Genetics &#x00026; Cell Biology and Biochemistry &#x00026; Molecular Biology, Chicago, Illinois 60637, USA; <sup>7</sup>Protometrix, Inc., Guilford, Connecticut 06437, USA</div></div><div class="fm-footnote"></div><div class="fm-footnote" id="FN8"><sup>8</sup>Corresponding author.</div><div class="fm-pubdate">Received April 19, 2002; Accepted July 30, 2002.</div><div class="links-box"><div class="fm-footnote"><img src="corehtml/pmc/pmcgifs/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="tocrender.fcgi?artid=187531&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id497520" class="sidefm-pmclink" style="text-transform: none;">RESULTS AND DISCUSSION</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id498399" class="sidefm-pmclink" style="text-transform: none;">METHODS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id498628" class="sidefm-pmclink" style="text-transform: none;">WEB SITE REFERENCES</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id499709" class="sidefm-pmclink" style="text-transform: none;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id497270">Abstract</div><div class="section-content"><!--article-meta-->Draft sequencing is a rapid and efficient method for determining the near-complete sequence of microbial genomes. Here we report a comparative analysis of one complete and two draft genome sequences of the phytopathogenic bacterium, <em>Xylella fastidiosa</em>, which causes serious disease in plants, including citrus, almond, and oleander. We present highlights of an <em>in silico</em> analysis based on a comparison of reconstructions of core biological subsystems. Cellular pathway reconstructions have been used to identify a small number of genes, which are likely to reside within the draft genomes but are not captured in the draft assembly. These represented only a small fraction of all genes and were predominantly large and small ribosomal subunit protein components. By using this approach, some of the inherent limitations of draft sequence can be significantly reduced. Despite the incomplete nature of the draft genomes, it is possible to identify several phage-related genes, which appear to be absent from the draft genomes and not the result of insufficient sequence sampling. This region may therefore identify potential host-specific functions. Based on this first functional reconstruction of a phytopathogenic microbe, we spotlight an unusual respiration machinery as a potential target for biological control. We also predicted and developed a new defined growth medium for <em>Xylella</em>.<p>&#x0005b;The sequence data from this study have been submitted to GenBank under accession nos. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0JkKpP3ZBncgIPdNgMLge5KVxSTj5mhGF-2mts524&amp;reftype=extlink-entrez-nucleotide&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CFront%20Matter&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=20070516" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">NC_002723</a></span> (<em>X. fastidiosa</em> Almond &#x0005b;Dixon&#x0005d;) and <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0Ti92_MpkIQL66KkwYYwqGymXKDeHpNYYOSNfJQVA&amp;reftype=extlink-entrez-nucleotide&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CFront%20Matter&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=20070517" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">NC_002722</a></span> (<em>X. fastidiosa</em> Oleander &#x0005b;Ann-1&#x0005d;).</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id497270" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id497520" class="sidefm-pmclink" style="text-transform: none;">RESULTS AND DISCUSSION</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id498399" class="sidefm-pmclink" style="text-transform: none;">METHODS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id498628" class="sidefm-pmclink" style="text-transform: none;">WEB SITE REFERENCES</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id499709" class="sidefm-pmclink" style="text-transform: none;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id497331"> </div><div class="section-content"><em>Xylella fastidiosa</em> is a Gram-negative bacterium belonging to the gamma subgroup of the proteobacteria (<a href="#B24" class="cite-reflink bibr">Wells et al. 1987</a>). This microorganism is an important plant pathogen causing many economically important crop diseases, such as Pierce's disease (PD) of grapevine and citrus variegated chlorosis in citrus (<a href="#B17" class="cite-reflink bibr">Moller et al. 1974</a>; <a href="#B19" class="cite-reflink bibr">Purcell 1997</a>). Almond leaf scorch was first described in 1974 (<a href="#B17" class="cite-reflink bibr">Moller et al. 1974</a>), and electron microscope studies indicated the presence in almond of the same bacterium previously associated with PD of grapevines (<a href="#B16" class="cite-reflink bibr">Mircetich et al. 1976</a>). Cross-inoculation studies showed that PD and the almond leaf scorch strains were pathogenic to both almond and grape (<a href="#B2" class="cite-reflink bibr">Davis and Thompson 1980</a>). Oleander leaf scorch was first noticed in 1994 in southern California. The glassy-winged sharpshooter, <em>Homalodisca coagulata, </em>was found to be the vector for this bacterium. It is possible that several other varieties of sharpshooters can transmit <em>X. fastidiosa</em> between oleander plants (<a href="#B20" class="cite-reflink bibr">Purcell et al. 1999</a>). The oleander leaf scorch strain could not be reisolated from grapevine in a greenhouse after inoculation using needle puncture (<a href="#B20" class="cite-reflink bibr">Purcell et al. 1999</a>). This result provides evidence for host specificity among strains of <em>X. fastidiosa</em>. Previous studies classified <em>X. fastidiosa</em> as a single species, but differentiated members of the species, depending on such criteria as host specificity and pathogenicity (<a href="#B13" class="cite-reflink bibr">Hendson et al. 2001</a> and references therein). We define each strain of <em>X. fastidiosa</em> to be a pathovar for a specific plant, as suggested previously (<a href="#B13" class="cite-reflink bibr">Hendson et al. 2001</a>). We will therefore refer throughout to the citrus, almond, and oleander strains as <em>Xf pv citrus </em>(XFA), <em>Xf pv almond</em> (XFX), and <em>Xf pv oleander </em>(XFY).<p>The complete genome sequence of the <em>X. fastidiosa</em> 9a5c strain (XFA), which causes citrus variegated chlorosis , was recently published (<a href="#B23" class="cite-reflink bibr">Simpson et al. 2000</a>). In this study, we report the gapped-genome sequences of <em>Xf pv almond</em> (Dixon strain) (XFX) and <em>Xf pv oleander</em> (Ann-1 strain) (XFY) and compare them to the complete genomic sequence of XFA. Here we present highlights of the first functional reconstruction of a phytopathogenic microbe based on the three sequenced genomes. This approach has led us to the identification of a putative drug target in the aerobic respiratory chain of this organism, as well as to a prediction of the specific growth requirements of this bacterium, which were used to develop a defined medium for optimal growth. Furthermore, based on a global alignment of the proteomes for these genomes, we identified a region in XFA that is absent in the other two genomes that potentially encodes host-specific functions. Comparisons of curated pathways and their components present in the complete XFA genome relative to the gapped XFY and XFX genomes allows the prediction of functions potentially missed during gapped-genome sequencing. Such an approach provides an added level of quality control to the analysis of sequenced genomes.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id497270" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>RESULTS AND DISCUSSION</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id498399" class="sidefm-pmclink" style="text-transform: none;">METHODS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id498628" class="sidefm-pmclink" style="text-transform: none;">WEB SITE REFERENCES</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id499709" class="sidefm-pmclink" style="text-transform: none;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id497520">RESULTS AND DISCUSSION</div><div class="section-content"><p><div class="head2 head-separate">Gapped-Genome Sequencing of New <em>Xf</em><img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />Strains</div>The capacity to produce draft genomes based on a random shotgun approach (<a href="#B8" class="cite-reflink bibr">Fleischmann et al. 1995</a>) far exceeds our capacity to produce completely finished genomes. The draft-sequencing process is faster (two to three weeks compared with months or years for finished sequence) and much less expensive. The gapped-genomic sequence of the two new <em>Xylella</em> strains (XFX and XFY) represents more than 95&#x00025; of the complete sequence, with 9.4- and 8.1-fold sequence coverage, respectively (Table <a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a>). As we hereby show, the microbial draft sequences (high-quality annotated assemblies from &#x0003e;8-fold coverage) offers an extremely useful form of information, leading to an effective sampling of &#x0003e;95&#x00025; of the gene repertoire of a given microbial genome. The genome size of both almond (XFX) and oleander (XFY) strains is very similar to the completely sequenced XFA genome, with 2.4 Mb (XFX) and 2.6 Mb (XFY), compared with 2.7 Mb for the XFA genome. All three strains display a similar GC content. We identified 2681 (XFX) and 2870 (XFY) ORFs distributed over 121 and 93 contigs, respectively, compared with the 2985 ORFs of the XFA genome. Using the ERGO bioinformatics suite, we assigned functions to 62&#x00025; of the genes in each of the two new <em>Xylella</em> genomes, compared with the 58&#x00025; of genes in the XFA genome. Of the remaining 38&#x00025; of the ORFs in the XFX and XFY genomes, most show sequence similarity to other proteins of unknown function (hypothetical proteins), and only 3&#x00025;&#x02013;4&#x00025; of the total ORFs in the two genomes did not show any sequence similarity to other known proteins (unique ORFs).  <div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T1" name="T1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 1" title="Table 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 1</strong></a><div class="figure-table-caption-in-article"><span>Comparative Statistics of Sequenced Genomes of <em>Xylella fastidiosa</em><img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />Strains</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>Nearly 59&#x00025; of the ORFs in any of the three <em>Xylella</em> genomes can be grouped in ortholog clusters and 22&#x00025; in paralog clusters. Interestingly, the number of paralog clusters in XFY is less than that in XFA, but there are more ORFs in the XFY paralog clusters compared with the XFA strain. This result indicates that there is greater functional redundancy in the oleander strain, XFY.</p></p><p><div class="head2 head-separate">Identification of <em>Xf</em> Oleander (XFY)<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />Episome</div>The sequencing and assembly of the XFY genome revealed that it possesses a putative 30-kb episome (Table <a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a>). The plasmid bears 36 ORFs, of which three encode proteins involved in replication (RXFY0974, RXFY0979) and plasmid maintenance and stability (RXFY0987). A Type IV secretion pathway (chromosomal cluster) that includes VirB2 (RXFY02430), B4 (RXFY00976), B5 (RXFY00992), B6 (RXFY00989), B8 (RXFY00988), B9 (RXFY00984), B10 (RXFY00977), B11 (RXFY00991), and VirD4 (RXFY00973), as well as a putative transcriptional regulator (RXFY00983) are encoded on the 30-kb replicon. The VirB system is designed to mediate the plasmid transfer from a donor to a recipient bacterium. Two members of the DNA conjugative transfer pathway (TraM, RXFY02499 and TrbN, RXFY00996) have also been found along with a &#x0201c;nickase&#x0201d; (RXFY0985), which is responsible for initiating transposition by introducing single-stranded nicks in the circular plasmid DNA.  <div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T2" name="T2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T2" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 2" title="Table 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 2</strong></a><div class="figure-table-caption-in-article"><span>Comparative Statistics of the <em>Xf</em> pv oleander (XFY) and <em>Xf</em> pv <em>citrus</em> (XFA; p51)<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />Eplsomes</span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><div class="head2 head-separate">Pathway Analysis</div>As part of the <em>X. fastidiosa</em> functional reconstruction (described later), ORFs were assigned to the appropriate metabolic and nonmetabolic pathways (<a href="#B15" class="cite-reflink bibr">Kyrpides et al. 2000</a>; <a href="#B22" class="cite-reflink bibr">Selkov et al. 2000</a>). Figure <a href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1</a> shows the statistics of functional roles in all three organisms according to subsystem in more detail. Table <a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a> indicates that a similar number of pathways are present in all three <em>Xf</em> strains. Because the citrus (XFA) genome is complete, the percent of functions that appear missing from pathways (Table <a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>) is likely to represent instances of nonorthologous gene replacement, that is, the replacement of a particular protein function by unrelated protein(s) with the same cellular function (<a href="#B9" class="cite-reflink bibr">Forterre 1999</a>; <a href="#B10" class="cite-reflink bibr">Galperin and Koonin 1999</a>). Therefore, the number of functions missing from pathways for XFA represents the relative baseline of functions that appear as &#x0201c;absent&#x0201d;. Given that ERGO integrates a database of cellular pathways, it is possible to predict which of the functional roles that appear as missing from the XFX/XFY pathways are likely to reside in their nonsequenced regions. We define a functional role as a step in a pathway, and one functional role may have more than one biochemical function. For example, there are 40 and 97 functional roles missing from the genomes of XFY and XFX, respectively, but present in XFA (Table <a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>). A comparison of these genomes shows a similar number of functional roles associated with each subsystem. Most of the missing functions in almond and oleander strains are present in informational processing pathways (Table <a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>). Some of these missing functions represent components of the core functional machinery (for example, subunits of the large and small ribosomal complexes), and therefore we would expect them to be present in the nonsequenced gaps. More rigorous investigation of these cases indicated that although these functions are indeed present in these genomes, some are associated with ORFs residing on contigs with &#x0003c;20 reads, which have lower coverage and sequence quality (see Methods). We speculate that the functional absences present in the secretion subsystem in XFX (Table <a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>) are likely due to the fact that a plasmid-borne &#x02018;pathogenicity island&#x02019; was not clearly identified in XFX, in contrast to the XFA and XFY. Thus, the pathway analysis approach introduces an additional quality control step in the reconstruction of a partial genome and allows a greater degree of functionality to be assigned to sequenced genomes.    <div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F1" name="F1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=187531&amp;blobname=51916-10f1_L4TT.gif" border="1" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span>Comparative disposition of functional roles in cellular subsystems in <em>Xf</em> genomes. The number of functional roles present in the each of the described subsystems was plotted for each of the <em>Xylella</em> genomes.</span></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T3" name="T3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T3" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 3" title="Table 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=187531&amp;rendertype=table&amp;id=T3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 3</strong></a><div class="figure-table-caption-in-article"><span>Prediction of Functions Missing from Pathways in Gapped <em>Xf</em><img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />Genomes</span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><div class="head2 head-separate">Phage Integration Region Identifies Potential Host-Specific<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />Functions</div>During our comparative analysis of the three <em>Xylella</em> genomes, we noticed that the XFA genome contains two tRNA genes (encoding glycine and threonine, at 1,638,644 and 1,707,587 bp) that flank a region marked by the presence of prophage genes absent from the XFX and XFY genomes. This region was identified in the <em>XFA</em> genome using a comparative genome/proteome alignment tool called GenomeWalk, the visual rendering of which is shown in Figure <a href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F2" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">2</a>. The putative phage encoded in this region is most similar to the Siphophage group of double-stranded DNA phage based on whole-genome level and taxonomic comparisons (<a href="#B21" class="cite-reflink bibr">Rohwer and Edwards 2002</a>). The DNA sequence in this region has a mean GC content of 65.9&#x00025;, significantly greater than that found in the XFA genome as a whole (52.6&#x00025;), indicating that it has been inherited relatively recently in the evolution of the <em>Xf</em> genome. This region also contains genes encoding integrase enzymes and DNA replication machinery, as well as genes encoding structural proteins such as capsid and tail proteins.  <div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F2" name="F2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=187531&amp;blobname=51916-10f2_C4TT.gif" border="1" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span>Graphical alignment of <em>Xylella fastidiosa</em> (<em>Xf</em>) proteomes using GenomeWalk. A unique 65-kb phage insertion region (red rectangle, brown shading) in the <em>Xf</em> citrus strain (XFA) is absent in the <em>Xf</em> almond (XFX) and oleander (XFY) strains (arrow).</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>Closer inspection of this prophage region reveals that it contains two unusual &#x0201c;operons&#x0201d;. These operons are striking by the presence of oxidoreductase, dehydrogenase, esterase, hydrolase, and isomerase enzymes, as well as outer membrane permease/transporter proteins, indicating that they are involved in carbon-utilization pathways. Both operons are completed by LysR-family transcriptional regulators that presumably control transcription of the genes in response to environmental signals. It is unclear which carbon-utilization pathways these enzymes are involved in, although the limited distribution (no close orthologs previously identified) indicates that these operons may provide host-specific functions to XFA that are not present in other <em>Xf</em> strains. The completeness of the proteins encoded in this region, the lack of similar prophage in related <em>Xf</em> genomes, and the similarity between this phage and prophage found in the enteric <em>Salmonella typhi</em> genome recently sequenced (<a href="#B18" class="cite-reflink bibr">Parkhill 2000</a>) might indicate that this is still a viable phage and not a cryptic remnant of an invading parasite.</p></p><p><div class="head2 head-separate">Functional Reconstruction</div>We embarked on a functional reconstruction of the <em>X. fastidiosa </em>(<em>Xf</em>) genomes based on the genomic DNA sequences. A more detailed description of how this is done has been published previously (<a href="#B15" class="cite-reflink bibr">Kyrpides et al. 2000</a>; <a href="#B22" class="cite-reflink bibr">Selkov et al. 2000</a>; <a href="#B6" class="cite-reflink bibr">DelVecchio et al. 2002</a>; <a href="#B14" class="cite-reflink bibr">Kapatral et al. 2002</a>). Here, we present selected highlights of this analysis specifically pertaining to respiratory mechanisms and some of the core metabolism that is relevant to the growth of the bacteria.<p>Reconstruction of electron transport in <em>Xylella</em> spp. was performed by analyzing the respiratory complexes, both anaerobic and aerobic, as well as proton transport and ATPase. In contrast to previous interpretations (<a href="#B23" class="cite-reflink bibr">Simpson et al. 2000</a>), the resulting overview indicates that the <em>Xf </em>strains possess quite a simple and unusual aerobic respiratory complex (Fig. <a href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F3" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">3</a>A,B) that does not have the adaptive capabilities of, for instance, <em>Escherichia coli</em> or <em>Bacillus subtilis</em>. Functional analysis <em>in silico </em>shows the presence of the least energy-efficient type of aerobic respiration of any known organism reported to date. The ORF numbering corresponds to the XFA genome, but orthologs in the XFX and XFY strains are present. All three <em>Xf</em> possess cytochrome o (bo) ubiquinol oxidase (EC 1.10.3.-) as the only terminal oxidase (RXFA01387&#x02013;1390). Such a limited system of aerobic respiration indicates that this type of energy metabolism is not a prevalent pathway for this bacterial family. In addition, it indicates that <em>Xf</em> are capable of aerobic respiration at high aeration levels but not under oxygen-limiting conditions, because of the absence of cytochromes with high oxygen affinity. Surprisingly, there is no trace of a cytochrome c oxidase (or other type of quinol oxidase as the terminal oxidase) despite the availability of the entire operon encoding the ubiquinol-cytochrome c reductase complex (EC <span class="ext-reflink">1.10.2.2</span>; complex III: Rieske iron-sulfur protein, cytochrome b and cytochrome c1 precursor &#x0005b;RXFA00908&#x02013;910&#x0005d;, respectively). Thus, we can suggest that, because the cytochrome o (bo) ubiquinone oxidase is the only available terminal oxidase, it provides a strong potential control point and drug target for these phytopathogens. Although soluble periplasmic cytochrome c has not been found in the <em>Xf </em>genomes, another protein, the copper-containing protein azurin (RXFA00507), could perform electron transport in the periplasm between ubiquinol-cytochrome c reductase and the missing cytochrome c oxidase.  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F3" name="F3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=187531&amp;blobname=51916-10f3a_F1TT.gif" border="1" class="icon-reflink" alt="Figure 3" title="Figure 3"><img src="picrender.fcgi?artid=187531&amp;blobname=51916-10f3b_F1TT.gif" border="1" class="icon-reflink" alt="Figure 3" title="Figure 3"><img src="picrender.fcgi?artid=187531&amp;blobname=51916-10f3c_F1TT.gif" border="1" class="icon-reflink" alt="Figure 3" title="Figure 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 3</strong></a><div class="figure-table-caption-in-article"><span>Graphical visualization of the functional reconstruction of <em>Xylella fastidiosa</em> aerobic and anaerobic respiration. (<em>A</em>) Diagrammatic representation of aerobic respiration. Three-dimensional boxes show the presence of functional components and filled arrows</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>Anaerobic respiration in <em>Xf </em>is also quite limited (Fig. <a href="articlerender.fcgi?artid=187531&amp;rendertype=figure&amp;id=F3" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">3</a>C). The entire system for nitrate utilization to dinitrogen is absent. Additionally, the enzymes converting nitrate into ammonia appear to be absent. <em>Xf </em>clearly prefers anaerobic respiration based on sulfur metabolism. The entire operon that encodes enzymes for the use and reduction of sulfate to sulfide is present (sulfate adenylyltransferase &#x0005b;EC <span class="ext-reflink">2.7.7.4</span>; RXFA01500&#x02013;1501&#x0005d;, adenylylsulfate kinase &#x0005b;EC2.7.1.25; RXFA01501&#x0005d;, phosphoadenosine phosphosulfate reductase &#x0005b;EC <span class="ext-reflink">1.8.99.4</span>; RXFA01497&#x0005d;, and sulfite reductase &#x0005b;EC <span class="ext-reflink">1.8.1.2</span>; RXFA01498&#x02013;1499&#x0005d;). Indication that another type of anaerobic respiration may be present is suggested by the presence of arsenate reductase (RXFA00115). The simplicity and the type of anaerobic respiratory complex indicate that the enzymes of the sulfate&#x02013;sulfide reduction pathway could also serve as a potential drug target.</p><p><em>Xf </em>has a well-defined operon encoding H<sup>&#x0002b;</sup>-transporting ATP synthase (EC <span class="ext-reflink">3.6.1.34</span>; RXFA01142&#x02013;1149). There is no indication of any other type of ATP synthase. Proton balance seems to be maintained mainly by NADH-quinone oxidoreductase (EC <span class="ext-reflink">1.6.5.3</span>; RXFA00305&#x02013;318), cytochrome o ubiquinol oxidase, quinone pool, and ubiquinol-cytochrome c reductase that pump protons out and the reverse catalyzed by the H<sup>&#x0002b;</sup>-transporting ATP synthase.</p><p><em>X. fastidiosa</em> is able to use very few sugars (glucose, fructose, mannose, ribose, glycerol, N-acetylglucosamine) and cellulose. Paradoxically, despite its ability to use these sugars, it was not apparent from previous studies (<a href="#B23" class="cite-reflink bibr">Simpson et al. 2000</a>) how these sugars are transported into the cell. We have identified candidates for some of the sugar transport systems, which have orthologs in <em>Xf pv almond</em> and <em>Xf pv oleander</em>. For example, based on the chromosomal neighborhood of the XFA ORF, RXFA01462, which is positioned at the beginning of a N-acetylglucosamine utilization operon, this ORF is likely to be the N-acetylglucosamine and glucosamine transporter. The <em>Xylella</em> genomes contain one family 1 ABC transporter, the subunits of which are dispersed throughout the genome. The sugar transport permease (RXFA02446, RXFA02467) and the periplasmic binding protein (RXFA02448) are found adjacent to an enzyme involved in glucose metabolism. The ATPase subunit (RXFA01067) is found in a distant region of the genome. We propose that this sugar transporter is likely to be involved in glucose uptake.</p><p>The range of amino acids that might be used by <em>Xylella</em> as a sole carbon source seems to be limited to <span class="small-caps">d</span>- and <span class="small-caps">l</span>-alanine (via alanine racemase and <span class="small-caps">d</span>-amino acid dehydrogenase), glycine (via glycine cleavage system) and <span class="small-caps">l</span>-glutamate (via glutamate dehydrogenase). Thus, some carbon and energy source, other than amino acids and an oligopeptide mixture, should be provided for efficient growth of <em>Xylella</em>. This carbon source might be glycerol (all genes for glycerol utilization were found, including glycerol uptake protein, glycerol kinase, and glycerol-3-phosphate dehydrogenases), malate or oxaloacetate (C4-dicarboxylate transporter and NADP-dependent malic enzyme are clustered together), glucose, fructose, or N-acetylglucosamine. Most likely, an optimal growth medium for <em>Xf</em> would be predicted to contain glycerol as a carbon and energy source and <span class="small-caps">l</span>-glutamate as both a nitrogen and carbon source. In addition, the clear preference of <em>Xf </em>for sulfate reduction-based anaerobic respiration could indicate that the best medium for the <em>Xf </em>family would contain an increased concentration of sulfur-containing compounds.</p></p><p><div class="head2 head-separate">Growth Medium</div>Culture media for <em>Xf </em>were first developed for both grape and almond strains (<a href="#B3" class="cite-reflink bibr">Davis et al. 1978</a>) and eventually for other <em>X. fastidiosa</em> strains such as those from peach and periwinkle (<a href="#B4" class="cite-reflink bibr">Davis et al. 1981a</a>). Determination of the minimum metabolic requirements for this organism would thus allow improvement in medium composition for optimal growth and make this organism more tractable for study. Based on our functional reconstruction of the sequenced <em>Xf</em> genomes, we generated predictions of the nutritional profile of these phytopathogens, and we designed growth experiments to test and develop a defined medium. A commonly used growth medium was modified by adding one or more of the predicted compounds. The standard growth medium used was PW (<a href="#B5" class="cite-reflink bibr">Davis et al. 1981b</a>). Medium components that were changed included those that are putative sources for carbon, nitrogen, and iron. The bacterial strain used for the experiments was <em>Xf pv almond</em> (Dixon). Numbers of colony-forming units (CFU) of <em>Xf</em> were determined after 7 days growth at 28&#x000b0;C.<p>We compared the growth of <em>Xf pv almond</em> on standard PW as the growth medium (1.13 &#x000d7; 10<sup>9 </sup>CFU ml<sup>-1</sup>), and determined that the addition of fructose (8.40 &#x000d7; 10<sup>8 </sup>CFU ml<sup>-1</sup>), oxaloacetate (5.80 &#x000d7; 10<sup>8</sup> CFU ml<sup>-1</sup>), N-acetylglucosamine (2.90 &#x000d7; 10<sup>8</sup> CFU ml<sup>-1</sup>), or &#x003b1;-keto-glutarate (7.00 &#x000d7; 10<sup>8</sup> CFU ml<sup>-1</sup>) could substitute for the Bovine serum albumin (BSA) constituent of the PW medium when added separately. In comparison, glycerol (2.93 &#x000d7; 10<sup>7 </sup>CFU ml<sup>-1</sup>) and glucose (5.70 &#x000d7; 10<sup>6</sup> CFU ml<sup>-1</sup>) could not substitute as well for BSA in the PW-based medium. Sucrose did not substitute for BSA, as was mentioned in previous studies (<a href="#B5" class="cite-reflink bibr">Davis et al. 1981b</a>). The different iron sources used were hemin chloride (1.13 &#x000d7; 10<sup>9</sup> CFU ml<sup>-1</sup>), ferric pyrophosphate (4.60 &#x000d7; 10<sup>8</sup> CFU ml<sup>-1</sup>), or iron sulfate (3.10 &#x000d7; 10<sup>8</sup> CFU ml<sup>-1</sup>). These iron sources affected the growth of <em>Xf</em> similarly. The addition of the amino acids glycine (4.40 &#x000d7; 10<sup>8</sup> and 2.9 &#x000d7; 10<sup>8</sup> CFU ml<sup>-1</sup>) or glycine plus <span class="small-caps">l</span>-alanine (9.50 &#x000d7; 10<sup>8</sup> and 6.83 &#x000d7; 10<sup>8</sup> CFU ml<sup>-1</sup>) to the PW base did not appreciably improve growth, although <em>Xylella</em> approached the growth properties with the PW-based medium. Indeed, when glycine was omitted from the modified medium, less growth was observed (data not shown). <em>Xylella</em> appear to grow better in the presence of glycine, although the exact reason for this is unclear. When <span class="small-caps">l</span>-cysteine was added to PW, no growth was observed. Our results indicate that when BSA is removed from the PW-based medium, the best growth conditions are provided by the addition of fructose, glutamine, glycine, and iron sulfate. Thus, <em>Xf</em> was found to grow well on media components predicted by the metabolic reconstruction when they were added as substitutes for BSA in the basal medium.</p><p>The aspects of the genome comparisons and functional reconstructions presented here represent the relative highlights of the biological analysis discerned from the genome analysis of the phytopathogenic <em>X. fastidiosa</em> (<em>Xf</em>) species. A more complete description of the genome analyses and reconstructions are presented elsewhere (Bhattacharyya et al., in press). However, there are two intriguing questions posed from this study. First, does the unique phage insertion sequence in the <em>Xf</em> citrus strain indeed bear potential host-specific functions? Second, can the strong potential control point identified in the aerobic respiratory chain provide a potential drug target in these phytopathogens? We envision that microarray, genetic, proteomic, and pharmacological approaches will provide the tools to address these new challenges identified from this <em>in silico</em> analysis of the genome sequences of these organisms.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id497270" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id497520" class="sidefm-pmclink" style="text-transform: none;">RESULTS AND DISCUSSION</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>METHODS</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id498628" class="sidefm-pmclink" style="text-transform: none;">WEB SITE REFERENCES</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id499709" class="sidefm-pmclink" style="text-transform: none;">REFERENCES</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id498399">METHODS</div><div class="section-content"><p><div class="head2 head-separate">Sequencing and Assembly</div>Genomic DNA for strains of both <em>Xf pv almond (Dixon) </em>and <em>oleander</em> (Ann-1) were obtained from the collection of isolates from A.H.P, at the University of California, Berkeley. Using a whole genome shotgun approach, purified DNA was fragmented using a GeneMachines Hydroshear, end repaired and size fractionated on agarose gels. Fragments in the 3- to 4-kb size range were excised, eluted, and blunt-end ligated to <em>Sma</em>I-linearized pUC18 DNA. DNA from overnight cultures (180 &#x003bc;L) of these libraries was purified using an automated 96-well SPRI protocol (<a href="#B12" class="cite-reflink bibr">Hawkins et al. 1994</a>). These templates were cycle-sequenced using both universal M13-forward and M13-reverse primers with dye-terminator chemistry (AP Biotech). The resulting products were sequenced on MegaBACE 1000 capillary sequencers. Raw traces were preprocessed using Cimarron software version 2.1905 (AP Biotech) and base-called using Phred (<a href="#B7" class="cite-reflink bibr">Ewing et al. 1998</a>). Read lengths in libraries typically averaged 500&#x02013;550 bases with Phred &#x02265;20. Individual sequence reads were assembled using Phrap (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0PoKthyje6il8wUzZ6WHVe057WtitDgzt7XFDxAAp&amp;reftype=extlink&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://bozeman.mbt.washington.edu/phrap.docs/phrap.html" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://bozeman.mbt.washington.edu/phrap.docs/phrap.html</a></span>), and contig order/orientation was established by using technology developed at the Joint Genome Institute (JGI) (e.g., readsincontig, phrap2gapspan) in conjunction with the graphical sequence editor, Consed (<a href="#B11" class="cite-reflink bibr">Gordon et al. 1998</a>). Libraries were sequenced to at least 8&#x000d7; depth of coverage. The DNA sequences of the two <em>Xylella</em> strains (almond and oleander strains) are available at the JGI Web server (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0zqgsROAYg6qxqyZ_5ZG94wa_Fjks9Op6vXxBIAoH&amp;reftype=extlink&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.jgi.doe.gov/JGI_microbial/html/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.jgi.doe.gov/JGI_microbial/html/</a></span>).</p><p><div class="head2 head-separate">ORF Prediction</div>ORFs were predicted with a proprietary ORF-calling software system developed at Integrated Genomics. The system automatically combines its own statistically predicted ORFs with ORFs derived from external sources (if available) and BLAST and FASTA similarities. ORFs were predicted using this approach for all three <em>Xf</em> genomes including the previously published <em>Xf pv citrus</em> (<a href="#B23" class="cite-reflink bibr">Simpson et al. 2000</a>) whose DNA sequence was extracted from GenBank. The ORF prediction tools were not run on contigs with Phred &#x0003c;20 reads.</p><p><div class="head2 head-separate">Genome Analysis: Annotations, Functional Reconstructions, and Comparative Genomics</div>We used the ERGO bioinformatics suite (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0HcXVYZLe4ySmmUMTvZxnYAizDnVHql15ZsE2LkHt&amp;reftype=extlink&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.integratedgenomics.com" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.integratedgenomics.com</a></span>) (<a href="#B15" class="cite-reflink bibr">Kyrpides et al. 2000</a>; <a href="#B22" class="cite-reflink bibr">Selkov et al. 2000</a>; <a href="#B6" class="cite-reflink bibr">DelVecchio et al. 2002</a>; <a href="#B14" class="cite-reflink bibr">Kapatral et al. 2002</a>). ERGO contains an integration of over 450 genomes (including complete and partial, public, and proprietary from all kingdoms), with an extensive manually curated set of functional annotations and over 5000 manually curated cellular pathways, which is part of the Integrated Genomics proprietary Pathway Database. The analysis of the three <em>Xf</em> strains was performed as previously described (<a href="#B15" class="cite-reflink bibr">Kyrpides et al. 2000</a>; <a href="#B22" class="cite-reflink bibr">Selkov et al. 2000</a>; <a href="#B6" class="cite-reflink bibr">DelVecchio et al. 2002</a>; <a href="#B14" class="cite-reflink bibr">Kapatral et al. 2002</a>). The gene clustering based on sequence (i.e., ortholog and paralog clusters), as well as the one based on chromosomal context (i.e., chromosomal and fusion clusters), were computed on the basis of proprietary technology developed at Integrated Genomics. Comparative genomics technology such as GenomeWalk (available through ERGO suite) was also used. GenomeWalk provides a graphical whole genome comparison environment that facilitates the identification of unique chromosomal regions between closely related genomes. Workbench allows the identification of the common and unique clusters of genes between genomes, by using clustering algorithms to calculate protein clusters between a set of organisms.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id498628">WEB SITE REFERENCES</div><div class="section-content"><p><span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0PiaFsPGYInFgr6Y8I2iRLWurDDeJ70taRWsiz5hC&amp;reftype=extlink&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.jgi.doe.gov/JGI_microbial/html/index.html" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.jgi.doe.gov/JGI_microbial/html/index.html</a></span>; JGI Web server.</p><p><span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0jkAjCePMX3CigRK9vTS3DqyaLbyOdXRRYHO-LQrh&amp;reftype=extlink&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.integratedgenomics.com" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.integratedgenomics.com</a></span>; ERGO bioinformatics suite</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id499695">Acknowledgments</div><div class="section-content"><p><p>We thank members of the bioinformatics and genome analysis group at Integrated Genomics, specifically, Gordon Pusch, Lynn Jablonski, Olga Vassieva, Allen Bartman, and Warren Gardner for assistance during preliminary stages of the genome analysis. We would also like to thank Ken Frankel (JGI) for his contribution in the genomic sequence analysis. Funding for this work was supported by the Integrated Genomics research and development program, and by the U.S. Department of Energy research grants, from the Office of Biological and Environmental Research, by the University of California, Lawrence Livermore National Laboratory under Contract No. W-7405-Eng-48, Lawrence Berkeley National Laboratory under contract No. DE-AC03 76SF00098, and Los Alamos National Laboratory under contract No. W-7405-ENG-36.</p><p>The publication costs of this article were defrayed in part by payment of page charges. This article must therefore be hereby marked &#x0201c;advertisement&#x0201d; in accordance with 18 USC section 1734 solely to indicate this fact.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="footnotes">Footnotes</div><div class="section-content"><div class="fm-footnote"><div class="p">E-MAIL <span class="e_id359290">anamitra/at/integratedgenomics.com</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id359290", '<a class="ext-reflink" href="' + reverseAndReplaceString('moc.scimonegdetargetni/ta/artimana:otliam', '/at/', '@') + '">' + reverseAndReplaceString('moc.scimonegdetargetni/ta/artimana', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; FAX (312) 226-9446.</div></div><div class="fm-footnote"><div class="p">Article and publication are at http://www.genome.org/cgi/doi/10.1101/gr.370702.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id497270" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id497520" class="sidefm-pmclink" style="text-transform: none;">RESULTS AND DISCUSSION</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id498399" class="sidefm-pmclink" style="text-transform: none;">METHODS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id498628" class="sidefm-pmclink" style="text-transform: none;">WEB SITE REFERENCES</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>REFERENCES</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id499709">REFERENCES</div><div class="section-content"><div class="back-matter-section"><ul class="back-ref-list-1st-line-shifted"><li id="B1">Bhattacharyya, A; Stilwagen, S; Ivanova, N; D'Souza, M; Bernal, A; Lykidis, A; Kapatral, V; Anderson, I; Larsen, N; Los, T, et al.  <span class="ref-journal">Proc. Natl. Acad. Sci.</span> 2002.   in press.</li><li id="B2">Davis, MJ; Thompson, SV. Etiological role of a xylem-limited bacterium causing Pierce's disease in almond leaf scorch. <span><span class="ref-journal">Phytopathology. </span>1980;<span class="ref-vol">70</span>:472–475.</span></li><li id="B3">Davis, MJ; Purcell, AH; Thompson, SV. Pierce's disease of grapevines: Isolation of the causal bacterium. <span><span class="ref-journal">Science. </span>1978;<span class="ref-vol">199</span>:75–77.</span></li><li id="B4">Davis, MJ; Whitcomb, RF; Gillespie, AG., Jr . Fastidious bacteria of plant vascular tissue and invertebrates (including so called rickettsia-like bacteria). In: Starr MP, Stolp H, Truper HG, Balows A, Schlegel HG. ,  editors. <span class="ref-journal">The prokaryotes: A handbook on habits, isolation, and identification of bacteria.</span> Heidelberg: Springer-Verlag; 1981a. pp. 2172–2188.</li><li id="B5">Davis, MJ; French, WJ; Schaad, NW. Axenic culture of the bacteria associated with phony disease of peach and plum leaf scald. <span><span class="ref-journal">Curr Microbiol. </span>1981b;<span class="ref-vol">6</span>:309–314.</span></li><li id="B6">DelVecchio, VG; Kapatral, V; Redkar, RJ; Patra, G; Mujer, C; Los, T; Ivanova, N; Anderson, I; Bhattacharyya, A; Lykidis, A, et al.  The genome sequence of the facultative intracellular pathogen <em>Brucella melitensis.</em> <span><span class="ref-journal">Proc Natl Acad Sci. </span>2002;<span class="ref-vol">99</span>:443–448.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=003ZiRMjv1CwZZbKc10gpKP3SuxcO_n5CC4-JQ5fI&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11756688">PubMed</a>]</span></li><li id="B7">Ewing, B; Hillier, L; Wendl, MC; Green, P. Base-calling of automated sequencer traces using phred. I. Accuracy assessment. <span><span class="ref-journal">Genome Res. </span>1998;<span class="ref-vol">8</span>:175–185.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=06u1B5PUG7CbvG5msZR01861MZcZCwOCdRmDjNAqx&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9521921">PubMed</a>]</span></li><li id="B8">Fleischmann, RD; Adams, MD; White, O; Clayton, RA; Kirkness, EF; Kerlavage, AR; Bult, CJ; Tomb, JF; Dougherty, BA; Merrick, JM, et al.  Whole-genome random sequencing and assembly of <em>Haemophilus influenzae Rd.</em> <span><span class="ref-journal">Science. </span>1995;<span class="ref-vol">269</span>:496–512.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=08W6KD3566bIDooAPyXpCHy0owU9lI5Eiad6HWQuz&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7542800">PubMed</a>]</span></li><li id="B9">Forterre, P. Displacement of cellular proteins by functional analogues from plasmids or viruses could explain puzzling phylogenies of many DNA informational proteins. <span><span class="ref-journal">Mol Microbiol. </span>1999;<span class="ref-vol">33</span>:457–465.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0bAUba9gVScopfhA0MHMEN-aNcFUyQfKEH-0Xw5jv&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10417637">PubMed</a>]</span></li><li id="B10">Galperin, MY; Koonin, EV. Functional genomics and enzyme evolution. Homologous and analogous enzymes encoded in microbial genomes. <span><span class="ref-journal">Genetica. </span>1999;<span class="ref-vol">106</span>:159–170.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0huwE6E6_lI2u2htgkPic4RTy_MABCJUxUceliAvb&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10710722">PubMed</a>]</span></li><li id="B11">Gordon, D; Abajian, C; Green, P. Consed: A graphical tool for sequence finishing. <span><span class="ref-journal">Genome Res. </span>1998;<span class="ref-vol">8</span>:195–202.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Tm5PIazSfmcpkfOQwKL6A8thECozgo_IhkVuj5W4&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9521923">PubMed</a>]</span></li><li id="B12">Hawkins, TL; O'Connor-Morin, T; Roy, A; Santillan, C. DNA purification and isolation using a solid-phase. <span><span class="ref-journal">Nucleic Acids Res. </span>1994;<span class="ref-vol">22</span>:4543–4544.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hAgo_E_CAMMbhZpZ96wNpRDZJYsOj_8TGSAXtQhq&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7971285">PubMed</a>]</span></li><li id="B13">Hendson, M; Purcell, AH; Chen, D; Smart, C; Guilhabert, M; Kirkpatrick, B. Genetic diversity of Pierce's disease strains and other pathotypes of <em>Xylella fastidiosa.</em> <span><span class="ref-journal">Appl Environ Microbiol. </span>2001;<span class="ref-vol">67</span>:895–903.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0fACiCr0wawRmrho3PSCrIzFQHRt9huev-Siuv5yC&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11157260">PubMed</a>]</span></li><li id="B14">Kapatral, V; Anderson, I; Ivanova, N; Reznik, G; Los, T; Lykidis, A; Bhattacharyya, A; Bartman, A; Gardner, W; Grechkin, G, et al.  Genome sequence and analysis of the oral bacterium <em>Fusobacterium nucleatum</em> strain ATCC 25586. <span><span class="ref-journal">J Bacteriol. </span>2002;<span class="ref-vol">184</span>:2005–2018.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0PG8qpBVSA1NqVKDsKK4BAOMszuC-ueY9YFgUGAf7&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11889109">PubMed</a>]</span></li><li id="B15">Kyrpides, NC; Ouzounis, CA; Iliopoulos, I; Vonstein, V; Overbeek, R. Analysis of the <em>Thermotoga maritima</em> genome combining a variety of sequence similarity and genome context tools. <span><span class="ref-journal">Nucleic Acids Res. </span>2000;<span class="ref-vol">28</span>:4573–4576.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0rmMxVlWHK5rAjbw4dw9gNP7ztTdKbcdOftyl6AgQ&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11071948">PubMed</a>]</span></li><li id="B16">Mircetich, SM; Lowe, SK; Moller, JW; Nyland, G. Etiology of almond leaf scorch disease and transmission of the causal agent. <span><span class="ref-journal">Phytopathology. </span>1976;<span class="ref-vol">66</span>:17–24.</span></li><li id="B17">Moller, WJ; Sanborn, RR; Mircetich, SM; Williams, HE; Beutel, JA. A newly recognized leaf scorch disease of almond. <span><span class="ref-journal">Plant Dis Rep. </span>1974;<span class="ref-vol">58</span>:99–101.</span></li><li id="B18">Parkhill, J. In defense of complete genomes. <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">18</span>:493–494.</span></li><li id="B19">Purcell, AH. <em>Xylella fastidiosa</em>, a regional problem or global threat? <span><span class="ref-journal">Journal of Plant Pathology. </span>1997;<span class="ref-vol">79</span>:99–105.</span></li><li id="B20">Purcell, AH; Saunders, SR; Hendson, M; Grebus, ME; Henry, MJ. Causal role of <em>Xylella fastidiosa</em> in oleander leaf scorch disease. <span><span class="ref-journal">Phytopathology. </span>1999;<span class="ref-vol">89</span>:53–58.</span></li><li id="B21">Rohwer, F; Edwards, R. <em>The phage proteomic tree: A genome-based taxonomy for phage.</em> <span><span class="ref-journal">J Bacteriol. </span>2002;<span class="ref-vol">184</span>:4529–4535.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0VULUaSJZ5_GMuDWYZ2nO-t-ldVvzmwJYTcBGuQMw&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12142423">PubMed</a>]</span></li><li id="B22">Selkov, E; Overbeek, R; Kogan, Y; Chu, L; Vonstein, V; Holmes, D; Silver, S; Haselkorn, R; Fonstein, M. Functional analysis of gapped microbial genomes: Amino acid metabolism of <em>Thiobacillus ferrooxidans.</em> <span><span class="ref-journal">Proc Natl Acad Sci. </span>2000;<span class="ref-vol">97</span>:3509–3514.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0PInCutnCvyVFOxsF4TjiH2mivrwedSRkFEi7NkuB&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10737802">PubMed</a>]</span></li><li id="B23">Simpson, AJ; Reinach, FC; Arruda, P; Abreu, FA; Acencio, M; Alvarenga, R; Alves, IM; Araya, JE; Baia, GS; Baptista, CS, et al.  The genome sequence of the plant pathogen <em>Xylella fastidiosa.</em> <span><span class="ref-journal">Nature. </span>2000;<span class="ref-vol">406</span>:151–157.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=02McTTUVMbDV2HNwfG0z3A-73VYxfH3KfPx9aoQ-N&amp;reftype=pubmed&amp;artid=187531&amp;article-id=187531&amp;iid=6010&amp;issue-id=6010&amp;jid=102&amp;journal-id=102&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10910347">PubMed</a>]</span></li><li id="B24">Wells, JM; Raju, BC; Hung, HY; Weisberg, WG; Mandelco-Paul, L; Brenner, DJ. <em>Xylella fastidiosa</em> new-genus new-species gram-negative xylem-limited fastidious plant bacteria related to <em>Xanthomonas</em> spp. <span><span class="ref-journal">Int J Syst Bacteriol. </span>1987;<span class="ref-vol">37</span>:136–143.</span></li></ul></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">Genome Research</span> are provided here courtesy of <br><b>Cold Spring Harbor Laboratory Press</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc1-->

