<html>

<head>
 <meta name="author" content="Books">

<meta name="keywords" content="national center for biotechnology information, ncbi, bookshelf, free access, book, online, textbook, development, embryo, molecular embryology, clinical genetics, evolution, bioethics, morphogenesis, differentiation">

<meta name="description" content="Developmental Biology provides the fundamental set of techniques, paradigms, and models necessary for a student to understand the core of developmental biology---differentiation and morphogenesis. Gilbert; Sinauer Associates, Inc.">

 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <!-- Google indexing --> <meta name="robots" content="noindex,nofollow,NOARCHIVE"> 
 <title>The Origins of Anterior-Posterior Polarity</title>
 <link rel="stylesheet" href="/books/bookres.fcgi/dbio/books.css">
 <!-- link type="text/css" rel="stylesheet" href="/corehtml/ncbi_test.css" -->
 <LINK REL="SHORTCUT ICON" HREF="favicon.ico">
</head>
<body bgcolor="#FFFFFF" topmargin=0 leftmargin=0 marginwidth=0 marginheight=0>
 <a name="top"></a>

<script type="text/javascript" language="JavaScript1.2">
var PopUpMenu2_GlobalConfig = [  ["UseThisGlobalConfig","yes"] ];
</script>
<!-- script language="JavaScript1.2" src="http://www.ncbi.nlm.nih.gov/coreweb/javascript/popupmenu2/popupmenu2_5loader.js"></script -->
<script language="JavaScript1.2" src="http://www.ncbi.nlm.nih.gov/coreweb/javascript/popupmenu2/popupmenu2_7loader.js"></script>

 <!-- logo -->
<table width="100%" border="0" cellpadding="0" cellspacing="0">
 <tr>
  <td width="100%" bgcolor="#0C2230" align="left"><img
  src="/books/bookres.fcgi/dbio/logo.gif" USEMAP="#map" ALIGN="left" BORDER="0"><MAP NAME="map"><AREA SHAPE="rect" coords="0,0 600,100" HREF="http://www.sinauer.com/"></MAP>
</td>
 </tr> 
</table>
   


<table width="100%" border="0" cellpadding="0" cellspacing="0">
<tr>
<td bgcolor="#ffcc66">&nbsp;&nbsp;&nbsp;
<a class="toolbar-link" href="/books/bv.fcgi?call=bv.View..ShowTOC&rid=dbio.TOC&depth=2">Short Contents</a> | 
<a class="toolbar-link" href="/books/bv.fcgi?call=bv.View..ShowTOC&rid=dbio.TOC&depth=10">Full Contents</a></td>
<td bgcolor="#ffcc66">
<a class="toolbar-link" href="/entrez/query.fcgi?db=Books">Other books</a> <span class="toolbar-nolink">@</span> 
<a class="toolbar-link" href="http://www.ncbi.nlm.nih.gov">NCBI</a></td>
</tr>
</tr>
</table>

<br>
        <table width="100%">
         <tr>          
          <td width="25%" valign="top"><table width="100%" class="sidebar-table">
<tr><th class="sidebar-header">Navigation</th></tr>
<tr><td><div class="toc-sidebar-text"><table cellspacing="2" cellpadding="2">
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-0" href="javascript:BVShow('dbio')">About this book</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-1">Part 2. Early embryonic development</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">9. The genetics of axis specification in Drosophila</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-3">Early Drosophila Development</a></td>
</tr>
<tr>
<td width="8">
<img src="/books/bookres.fcgi/system_default/click.gif" border="0"> </td>
<td><a class="toc-sidebar-active-3">The Origins of Anterior-Posterior Polarity</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-3">The Generation of Dorsal-Ventral Polarity</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-3" href="javascript:BVShow('dbio.biblist.2076')">References</a></td>
</tr>
</table></div></td></tr>
</table>
<table width="100%" class="sidebar-table">
<tr><th class="sidebar-header">Figures</th></tr>
<tr><td><span class="sidebar-text"><table cellspacing="2" cellpadding="2">
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1972">Figure 9.8. <link_caption>Generalized model of Drosophila anterior-posterior...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1975">Figure 9.9. <link_caption>Normal and irradiated embryos of...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1977">Figure 9.10. <link_caption>Three independent genetic pathways interact...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1981">Figure 9.11. <link_caption>A model of anterior-posterior pattern...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1987">Figure 9.12. <link_caption>Phenotype of a strongly affected...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1988">Figure 9.13. <link_caption>Gradient of Bicoid protein in...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1989">Figure 9.14. <link_caption>Schematic representation of the experiments...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1990">Figure 9.15. <link_caption>Gradient of Caudal protein in...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1992">Figure 9.16. <link_caption>Control of hunchback mRNA translation...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1994">Figure 9.17. <link_caption>Formation of the unsegmented poles...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1996">Figure 9.18. <link_caption>Segments and parasegments. A and...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1997">Figure 9.19. <link_caption>Three types of segmentation gene...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.1998">Figure 9.20. <link_caption>Defects seen in the fushi...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2000">Figure 9.21. <link_caption>Conversion of maternal protein gradients...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2002">Figure 9.22. <link_caption>Specific promoter regions of the...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2004">Figure 9.23. <link_caption>Hypothesis for the formation of...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2005">Figure 9.24. <link_caption>Transcription of the fushi tarazu...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2007">Figure 9.25. <link_caption>Model for the transcription of...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2008">Figure 9.26. <link_caption>Cell specification by the Wingless/Hedgehog...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2011">Figure 9.27. <link_caption><span style='background-color:#ffffCC'>Homeotic</span> <span style='background-color:#ffffCC'>gene</span> expression in Drosophila....</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2012">Figure 9.28. <link_caption>This four-winged fruit fly was...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2013">Figure 9.29. <link_caption>(A) Head of a wild-type...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2017">Figure 9.30. <link_caption>Antibody staining of the Ultrabithorax...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.figgrp.2018">Figure 9.31. <link_caption>Schematic representation of the differences...</link_caption></a></td>
</tr>
</table></span></td></tr>
</table><table width="100%" class="sidebar-table">
<tr><th class="sidebar-header">Tables</th></tr>
<tr><td><span class="sidebar-text"><table cellspacing="2" cellpadding="2">
<tr>
<td><img src="/books/bookres.fcgi/system_default/table.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.table.1978">Table 9.1. <link_caption>Maternal effect genes that effect...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/table.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.table.2003">Table 9.2. <link_caption>Major genes affecting segmentation pattern...</link_caption></a></td>
</tr>
</table></span></td></tr>
</table><table width="100%" class="sidebar-table">
<tr><th class="sidebar-header">Boxes</th></tr>
<tr><td><span class="sidebar-text"><table cellspacing="2" cellpadding="2"><tr>
<td><img src="/books/bookres.fcgi/system_default/box.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=dbio.box.2019"> <link_caption>The Homeodomain Proteins</link_caption></a></td>
</tr></table></span></td></tr>
</table>

	     <br><script type="text/javascript" src="/books/bookres.fcgi/system_default/search.js"></script>

<form name="frmGo" method="get" action="/entrez/query.fcgi">
 <input type="hidden" name="db" value="Books">
 <input type="hidden" name="cmd" value="Search">
 <input type="hidden" name="term" value=".">
 <input type="hidden" name="doptcmdl" value="Books">
</form>

<form name="frmSearch" onSubmit="return Process('dbio')">
<table class="sidebar-table" width="100%">
<tr><th class="sidebar-header">Search</th></tr>
<tr><td>
	<input type="text" name="__term" size="12" 
	      onKeyPress="KeyPress('dbio',event)">
	&nbsp;
	<a href="javascript:Process('dbio');"><img src="/books/bookres.fcgi/system_default/searchbtn.gif" border="0"></a>
	<br><small>
  <input type="radio" name="__sbook" checked> This book&nbsp; &nbsp;
  <input type="radio" name="__sbook"> All books<br>
  <input type="radio" name="__sbook"> PubMed</small>
</td></tr></table>
</form>
</td>
          <td valign="top"><a name="dbio.section.1971"></a><a class="toc-sidebar-passive-0" href="javascript:BVShow('dbio')">Developmental Biology</a> <img src="/books/bookres.fcgi/system_default/clickpar.gif" border="0"> <span class="toc-sidebar-passive-1"> Part 2. </span><a class="toc-sidebar-passive-1" href="javascript:BVShow('dbio.part.1333')">Early embryonic development</a> <img src="/books/bookres.fcgi/system_default/clickpar.gif" border="0"> <span class="toc-sidebar-passive-2"> 9. </span><a class="toc-sidebar-passive-2" href="javascript:BVShow('dbio.chapter.1955')">The genetics of axis specification in Drosophila</a><br><br><span class="section-header">The Origins of Anterior-Posterior Polarity</span><br><br><p>The anterior-posterior polarity of the embryo, larva, and adult has its origin in the anterior-posterior polarity of the egg (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1972">Figure 9.8</a>). The <b>maternal effect genes</b> expressed in the mother's ovaries produce messenger RNAs that are placed in different regions of the egg. These messages encode transcriptional and translational regulatory proteins that diffuse through the syncytial blastoderm and activate or repress the expression of certain zygotic genes. Two of these proteins, <b>Bicoid</b> and <b>Hunchback</b>, regulate the production of anterior structures, while another pair of maternally specified proteins, <b>Nanos</b> and <b>Caudal</b>, regulates the formation of the posterior parts of the embryo. Next, the zygotic genes regulated by these maternal factors are expressed in certain broad (about three segments wide), partially overlapping domains. These genes are called <b>gap genes</b> (because mutations in them cause gaps in the segmentation pattern), and they are among the first genes transcribed in the embryo. Differing concentrations of the gap gene proteins cause the transcription of <b>pair-rule genes</b>, which divide the embryo into periodic units. The transcription of the different pair-rule genes results in a striped pattern of seven vertical bands perpendicular to the anterior-posterior axis. The pair-rule gene proteins activate the transcription of the <b>segment polarity genes</b>, whose mRNA and protein products divide the embryo into 14 segment-wide units, establishing the periodicity of the embryo. At the same time, the protein products of the gap, pair-rule, and segment polarity genes interact to regulate another class of genes, the homeotic selector genes, whose transcription determines the developmental fate of each segment.</p><a name="1973"></a><span class="subsect1-header">The Maternal Effect Genes</span><br><br><a name="1974"></a><span class="subsect2-header">Embryological evidence of polarity regulation by oocyte cytoplasm</span><br><br><p>Classic embryological experiments demonstrated that there are at least two "organizing centers" in the insect egg, one in the anterior of the egg and one in the posterior. For instance, Klaus <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2212">Sander (1975)</a> found that if he ligated the egg early in development, separating the anterior from the posterior region, one half developed into an anterior embryo and one half developed into a posterior embryo, but neither half contained the middle segments of the embryo. The later in development the ligature was made, the fewer middle segments were missing. Thus, it appeared that there were indeed gradients emanating from the two poles during cleavage, and that these gradients interacted to produce the positional information determining the identity of each segment. Moreover, when the RNA of the anterior of insect eggs was destroyed (by either ultraviolet light or RNase), the resulting embryos lacked a head and thorax. Instead, these embryos developed two abdomens and telsons (tails) with mirror-image symmetry: telson-abdomen-abdomen-telson (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1975">Figure 9.9</a>; <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2151">Kalthoff and Sander 1968</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1068484">Kandler-Singer and Kalthoff 1976</a>). Thus, Sander's laboratory postulated the existence of a gradient at both ends of the egg, and hypothesized that the egg sequesters an RNA that generates a gradient of anterior-forming material.</p><center><table width="400" class="message" cellpadding="3" cellspacing="0" border="0">
<tr><td class="bq-header" align="center"><span class="bq-header">WEBSITE</span></td></tr>
<tr><td class="message"><p><b>9.4 Evidence for gradients in insect development.</b> The original evidence for gradients in insect development came from studies providing evidence for two "organization centers" in the egg, one located anteriorly and one located posteriorly. <a href="http://www.devbio.com/chap09/link0904.shtml" target="new">http://www.devbio.com/chap09/link0904.shtml</a></p></td></tr>
</table></center><br><a name="1976"></a><span class="subsect2-header">The molecular model: protein gradients in the early embryo</span><br><br><p>In the late 1980s, the gradient hypothesis was united with a genetic approach to the study of <i>Drosophila</i> embryogenesis. If there were gradients, what were the morphogens whose concentrations changed over space? What were the genes that shaped these gradients? And did these morphogens act by activating or inhibiting certain genes in the areas where they were concentrated? Christiane Nüsslein-Volhard led a research program that addressed these questions. The researchers found found that one set of genes encoded gradient morphogens for the anterior part of the embryo, another set of genes encoded morphogens responsible for organizing the posterior region of the embryo, and a third set of genes encoded proteins that produced the terminal regions at both ends of the embryo (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1977">Figure 9.10</a>; <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.table.1978">Table 9.1</a>). This work resulted in a Nobel Prize for Nüsslein-Volhard and her colleague, Eric Wieschaus, in 1995.</p><center><table width="400" class="message" cellpadding="3" cellspacing="0" border="0">
<tr><td class="bq-header" align="center"><span class="bq-header">WEBSITE</span></td></tr>
<tr><td class="message"><p><b>9.5 Christiane Nüsslein-Volhard and the molecular approach to development.</b> The research that revolutionized developmental biology had to wait for someone to synthesize molecular biology, embryology, and <i>Drosophila</i> genetics. <a href="http://www.devbio.com/chap09/link0905.shtml" target="new">http://www.devbio.com/chap09/link0905.shtml</a></p></td></tr>
</table></center><br><p>The anterior-posterior axis of the <i>Drosophila</i> embryo appears to be patterned before the nuclei even begin to function. The nurse cells of the ovary deposit mRNAs in the developing oocyte, and these mRNAs are apportioned to different regions of the cell. In particular, four maternal messenger RNAs are critical to the formation of the anterior-posterior axis:</p><br>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*"><b><i>bicoid</i></b> and <b><i>hunchback</i></b> mRNAs, whose protein products are critical for head and thorax formation<br><br>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*"><b><i>nanos</i></b> and <b><i>caudal</i></b> mRNAs, whose protein products are critical for the formation of the abdominal segments<br><br><p>The <i>bicoid</i> mRNAs are located in the anterior portion of the unfertilized egg, and are tethered to the anterior microtubules. The <i>nanos</i> messages are bound to the cytoskeleton in the posterior region of the unfertilized egg. The <i>hunchback</i> and <i>caudal</i> mRNAs are distributed throughout the oocyte. Upon fertilization, these mRNAs can be translated into proteins. At the anterior pole, the <i>bicoid</i> RNA is translated into Bicoid protein, which forms a gradient highest at the anterior. At the posterior pole, the <i>nanos</i> message is translated into Nanos protein, which forms a gradient highest at the posterior. Bicoid protein inhibits the translation of the <i>caudal</i> RNA, allowing Caudal protein to be synthesized only in the posterior of the cell. Conversely, Nanos protein, in conjunction with Pumilio protein, binds to <i>hunchback</i> RNA, preventing its translation in the posterior portion of the embryo. Bicoid also elevates the level of Hunchback protein in the anterior of the embryo by binding to the enhancers of the <i>hunchback</i> gene and stimulating its transcription. The result of these interactions is the creation of four protein gradients in the early embryo (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1981">Figure 9.11</a>):</p><br>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*"> An anterior-to-posterior gradient of Bicoid protein<br><br>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*"> An anterior-to-posterior gradient of Hunchback protein<br><br>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*">  A posterior-to-anterior gradient of Nanos protein<br><br>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*"> A posterior-to-anterior gradient of Caudal protein<br><br><p>The Bicoid, Hunchback, and Caudal proteins are transcription factors whose relative concentrations can activate or repress particular zygotic genes. The stage is now set for the activation of zygotic genes in those nuclei that were busily dividing while this gradient was being established.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1986"></a><span class="subsect2-header">Evidence that the bicoid gradient constitutes the anterior organizing center</span><br><br><p>In <i>Drosophila,</i> the phenotype of the bicoid mutant provides valuable information about the function of gradients. Instead of having anterior structures (acron, head, and thorax) followed by abdominal structures and a telson, the structure of the bicoid mutant is telson-abdomen-abdomen-telson (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1987">Figure 9.12</a>). It would appear that these embryos lack whatever substances are needed for the formation of anterior structures. Moreover, one could hypothesize that the substance that these mutants lack is the one postulated by Sander and Kalthoff to turn on genes for the anterior structures and turn off genes for the telson structures (compare <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1975">Figures 9.9</a> and <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1987">9.12</a>).</p><p>Further studies have strengthened the view that the product of the wild-type <i>bicoid</i> gene is the morphogen that controls anterior development. First, <i>bicoid</i> is a maternal effect gene. Messenger RNA from the mother's <i>bicoid</i> genes is placed in the embryo by the mother's ovarian cells (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1988">Figure 9.13A</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2877746">Frigerio et al. 1986</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2901954">Berleth et al. 1988</a>). The <i>bicoid</i> RNA is strictly localized in the anterior portion of the oocyte (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1988">Figure 9.13B</a>), where the anterior cytoskeleton anchors it through the message's 3´ untranslated region (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=9130719">Ferrandon et al. 1997</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=9632762">Macdonald and Kerr 1998</a>). This mRNA is dormant until fertilization, at which time it receives a longer polyadenylate tail and can be translated. <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3383244">Driever and Nüsslein-Volhard (1988b)</a> have shown that when Bicoid protein is translated from this RNA during early cleavage, it forms a gradient, with the highest concentration in the anterior of the egg and the lowest in the posterior third of the egg. Moreover, this protein soon becomes concentrated in the embryonic nuclei in the anterior portion of the embryo (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1988">Figure 9.13C</a>-<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1988">D</a>; see also Figure 5.35).</p><center><table width="400" class="message" cellpadding="3" cellspacing="0" border="0">
<tr><td class="bq-header" align="center"><span class="bq-header">WEBSITE</span></td></tr>
<tr><td class="message"><p><b>9.6 Mechanism of</b><b><i>bicoid</i></b><b>mRNA localization.</b> One of the most critical steps in <i>Drosophila</i> pattern formation is the binding of the <i>bicoid</i> mRNA to the anterior microtubules. Several genes are involved in this process, wherein the <i>bicoid</i> message forms a complex with several proteins. <a href="http://www.devbio.com/chap09/link0906.shtml" target="new">http://www.devbio.com/chap09/link0906.shtml</a></p></td></tr>
</table></center><br><p>Further evidence that Bicoid protein is the anterior morphogen came from experiments that altered the steepness of the gradient. Two genes, <i>exuperantia</i> and <i>swallow,</i> are responsible for keeping the <i>bicoid</i> message at the anterior pole of the egg. In their absence, the <i>bicoid</i> message diffuses farther into the posterior of the egg, and the gradient of Bicoid protein is less steep (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3383245">Driever and Nüsslein-Volhard 1988a</a>). The phenotype produced by these two mutants is similar to that of <i>bicoid</i>-deficient embryos, but less severe. These embryos lack their most anterior structures and have an extended mouth and thoracic region. Thus, by altering the gradient of Bicoid protein, one correspondingly alters the fate of the embryonic regions.</p><p>Confirmation that the Bicoid protein is crucial for initiating head and thorax formation came from experiments in which purified <i>bicoid</i> RNA was injected into early-cleavage embryos (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1989">Figure 9.14</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2226200">Driever et al. 1990</a>). When injected into the anterior of <i>bicoid</i>-deficient embryos (whose mothers lacked <i>bicoid</i> genes), the <i>bicoid</i> RNA rescued the embryos and caused them to have normal anterior-posterior polarity. Moreover, any location in an embryo where the <i>bicoid</i> message was injected became the head. If <i>bicoid</i> RNA was injected into the center of an embryo, that middle region became the head, and the regions on either side of it became thorax structures. If a large amount of <i>bicoid</i> RNA was injected into the posterior end of a wild-type embryo (with its own endogenous <i>bicoid</i> message in its anterior pole), two heads emerged, one at either end.</p><p>The next question then emerged: How might a gradient in Bicoid protein control the determination of the anterior-posterior axis? Recent evidence suggests that Bicoid acts in two ways to specify the anterior of the <i>Drosophila</i> embryo. First, it acts as a repressor of posterior formation. It does this by binding to and suppressing the translation of <i>caudal</i> RNA, which is found throughout the egg and early embryo. The Caudal protein is critical in specifying the posterior domains of the embryo, and it activates the genes responsible for the invagination of the hindgut (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=9609826">Wu and Lengyel 1998</a>). The Bicoid protein binds to a specific region of the <i>caudal</i> message's 3´ untranslated region, thereby preventing the translation of this message in the anterior section of the embryo (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1990">Figure 9.15</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8602214">Dubnau and Struhl 1996</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8602224">Rivera-Pomar et al. 1996</a>). This suppression is necessary, for if Caudal protein is made in the anterior, the head and thorax are not properly formed.</p><p>Second, the Bicoid protein functions as a transcription factor. Bicoid protein enters the nuclei of early-cleavage embryos, where it activates the <i>hunchback</i> gene. The transcription of <i>hunchback</i> is seen only in the anterior half of the embryo<img border="0" src="/books/bookres.fcgi/system_default/mdash.gif" alt=" ---">the region where Bicoid protein is found. Mutants deficient in maternal and zygotic Hunchback protein lack mouthparts and thorax structures. In the late 1980s, two laboratories independently demonstrated that Bicoid protein binds to and activates the <i>hunchback</i> gene (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2911348">Driever and Nüsslein-Volhard 1989</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2567637">Struhl et al. 1989</a>). The Hunchback protein derived from the synthesis of new <i>hunchback</i> mRNA joins the Hunchback protein synthesized by the translation of maternal messages in the anterior of the embryo. The Hunchback protein, also a transcription factor, represses abdominal-specific genes, thereby allowing the region of <i>hunchback</i> expression to form the head and thorax.</p><p>The Hunchback protein also works with Bicoid in generating the anterior pattern of the embryo. Based on two pieces of evidence, <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2502714">Driever and co-workers (1989)</a> predicted that at least one other anterior gene besides <i>hunchback</i> must be activated by Bicoid. First, deletions of <i>hunchback</i> produce only some of the defects seen in the bicoid mutant phenotype. Second, as we saw in the <i>swallow</i> and <i>exuperantia</i> experiments, only moderate levels of Bicoid protein are needed to activate thorax formation (i.e., <i>hunchback</i> gene expression), but head formation requires higher concentrations. <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2502714">Driever and co-workers (1989)</a> predicted that the promoters of such a head-specific gap gene would have low-affinity binding sites for Bicoid protein. This gene would be activated only at extremely high concentrations of Bicoid protein<img border="0" src="/books/bookres.fcgi/system_default/mdash.gif" alt=" ---">that is, near the anterior tip of the embryo. Since then, three gap genes of the head have been discovered that are dependent on very high concentrations of Bicoid protein for their expression (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1974035">Cohen and Jürgens 1990</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1974036">Finkelstein and Perrimon 1990</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=7720559">Grossniklaus et al. 1994</a>). The <i>buttonhead,</i><i>empty spiracles,</i> and <i>orthodenticle</i> genes are needed to specify the progressively anterior regions of the head. In addition to needing high Bicoid levels for activation, these genes also require the presence of Hunchback protein to be transcribed (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8087852">Simpson-Brose et al. 1994</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=7852948">Reinitz et al. 1995</a>). The Bicoid and Hunchback proteins act synergistically at the enhancers of these "head genes" to promote their transcription.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1991"></a><span class="subsect2-header">The posterior organizing center: localizing and activating nanos</span><br><br><p>The posterior organizing center is defined by the activities of the <i>nanos</i> gene (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1935684">Lehmann and Nüsslein-Volhard 1991</a>;<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1908748">Wang and Lehmann 1991</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1720354">Wharton and Struhl 1991</a>). The <i>nanos</i> RNA is produced by the ovarian nurse cells and is transported into the posterior region of the egg (farthest away from the nurse cells). The <i>nanos</i> message is bound to the cytoskeleton in the posterior region of the egg through its 3´ UTR and its association with the products of several other genes (<i>oskar, valois, vasa, staufen,</i> and <i>tudor</i>).<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.section.1971#2021">*</a> If <i>nanos</i> or any other of these maternal effect genes are absent in the mother, no embryonic abdomen forms (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3093084">Lehmann and Nüsslein-Volhard 1986</a>; <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2217">Schüpbach and Wieschaus 1986</a>).</p><p>The <i>nanos</i> message is dormant in the unfertilized egg, as it is repressed by the binding of the Smaug protein to its 3´ UTR (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8895661">Smibert et al. 1996</a>). At fertilization, this repression is removed, and Nanos protein can be synthesized. The Nanos protein forms a gradient that is highest at the posterior end. Nanos functions by inactivating <i>hunchback</i> mRNA translation (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1992">Figure 9.16</a>, see also <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1981">Figure 9.11</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2450283">Tautz 1988</a>). In the anterior of the cleavage-stage embryo, the <i>hunchback</i> message is bound in its 3´ UTR by the Pumilio protein, and the message can be translated into Hunchback protein. In the posterior of the early embryo, however, the bound Pumilio can be joined by the Nanos protein. Nanos binds to Pumilio and deadenylates the <i>hunchback</i> mRNA, preventing its translation (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1459455">Barker et al. 1992</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=9247343">Wreden et al. 1997</a>). The <i>hunchback</i> RNA is initially present throughout the embryo, although more can be made from zygotic nuclei if they are activated by Bicoid protein. Thus, the combination of Bicoid and Nanos proteins causes a gradient of Hunchback protein across the egg. The Bicoid protein activates <i>hunchback</i> gene transcription in the anterior part of the embryo, while the Nanos protein inhibits the translation of <i>hunchback</i> RNA in the posterior part of the embryo.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1993"></a><span class="subsect2-header">The terminal gene group</span><br><br><p>In addition to the anterior and posterior morphogens, there is third set of maternal genes whose proteins generate the extremes of the anterior-posterior axis. Mutations in these terminal genes result in the loss of the unsegmented extremities of the organism: the <b>acron</b> and the most anterior head segments and the <b>telson</b> (tail) and the most posterior abdominal segments (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3709971">Degelmann et al. 1986</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3412488">Klingler et al. 1988</a>). A critical gene here appears to be <b><i>torso</i></b>, a gene encoding a receptor tyrosine kinase. The embryos of mothers with mutations of the <i>torso</i> gene have neither acron nor telson, suggesting that the two termini of the embryo are formed through the same pathway. The <i>torso</i> RNA is synthesized by the ovarian cells, deposited in the oocyte, and translated after fertilization. The transmembrane Torso protein is not spatially restricted to the ends of the egg, but is evenly distributed throughout the plasma membrane (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2560750">Casanova and Struhl 1989</a>). Indeed, a dominant mutation of <i>torso,</i> which imparts constitutive activity to the receptor, converts the entire anterior half of the embryo into an acron and the entire posterior half into a telson. Thus, Torso must normally be activated only at the ends of the egg.</p><p><a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2385293">Stevens and her colleagues (1990)</a> have shown that this is the case. Torso protein is activated by the follicle cells only at the two poles of the oocyte. Two pieces of evidence suggest that the activator of the Torso protein is probably the <b>Torso-like</b> protein: first, loss-of-function mutations in the <i>torso-like</i> gene create a phenotype almost identical to that produced by torso mutants, and second, ectopic expression of Torso-like causes the activation of the Torso protein in the new location. The <i>torso-like</i> gene is usually expressed only in the anterior and posterior follicle cells, and the secreted Torso-like protein can cross the perivitelline space to activate the Torso protein in the egg membrane (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8107870">Martin et al. 1994</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=9510028">Furriols et al. 1998</a>). In this manner, the Torso-like protein activates the Torso protein in the anterior and posterior regions of the oocyte membrane. The end products of the RTK-kinase cascade activated by the Torso protein diffuse into the cytoplasm at both ends of the embryo (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1994">Figure 9.17</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=9342046">Gabay et al. 1997</a>; see <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.chapter.1011">Chapter 6</a>). These kinases are thought to inactivate a transcriptional inhibitor of the <i>tailless</i> and <i>huckebein</i> gap genes (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=9367438">Paroush et al. 1997</a>). These two genes then specify the termini of the embryo. The distinction between the anterior and posterior termini depends on the presence of Bicoid. If the terminal genes act alone, the terminal regions differentiate into telsons. However, if Bicoid is also present, the region forms an acron (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1638983">Pignoni et al. 1992</a>).</p><p>The anterior-posterior axis of the embryo is therefore specified by three sets of genes: those that define the anterior organizing center, those that define the posterior organizing center, and those that define the terminal boundary region. The anterior organizing center is located at the anterior end of the embryo and acts through a gradient of Bicoid protein that functions as a transcription factor to activate anterior-specific gap genes and as a translational repressor to suppresses posterior-specific gap genes. The posterior organizing center is located at the posterior pole and acts translationally through the Nanos protein to inhibit anterior formation and transcriptionally through the Caudal protein to activate those genes that form the abdomen. The boundaries of the acron and telson are defined by the product of the torso gene, which is activated at the tips of the embryo. The activation of those genes responsible for constructing the posterior is performed by Caudal, a protein whose synthesis (as we have seen above) is inhibited in the anterior portion of the embryo. The next step in development will be to use these gradients of transcription factors to activate specific genes along the anterior-posterior axis.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1995"></a><span class="subsect1-header">The Segmentation Genes</span><br><br><p>The process of cell fate commitment in <i>Drosophila</i> appears to have two steps: specification and determination (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2227">Slack 1983</a>). Early in development, the fate of a cell depends on environmental cues, such as those provided by the protein gradients mentioned above. This specification of cell fate is flexible and can still be altered in response to signals from other cells. Eventually, the cells undergo a transition from this loose type of commitment to an irreversible determination. At this point, the fate of a cell becomes cell-intrinsic.<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.section.1971#2022"><sup><img border="0" src="/books/bookres.fcgi/system_default/dagger.gif" alt="[dagger]"></sup></a> The transition from specification to determination in <i>Drosophila</i> is mediated by the <b>segmentation genes</b>. These genes divide the early embryo into a repeating series of segmental primordia along the anterior-posterior axis. Mutations in segmentation genes cause the embryo to lack certain segments or parts of segments. Often these mutations affect <b>parasegments</b>, regions of the embryo that are separated by mesodermal thickenings and ectodermal grooves. The segmentation genes divide the embryo into 14 parasegments (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3919303">Martinez-Arias and Lawrence 1985</a>). The parasegments of the embryo do not become the segments of the larva or adult; rather, they include the posterior part of an anterior segment and the anterior portion of the segment behind it (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1996">Figure 9.18</a>). While the segments are the major anatomical divisions of the larval and adult body plan, they are built according to rules that use the parasegment as the basic unit of construction.</p><p>There are three classes of segmentation genes, which are expressed sequentially (see <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1972">Figure 9.8</a>). The transition from an embryo characterized by gradients of morphogens to an embryo with distinct units is accomplished by the products of the <b>gap genes</b>. The gap genes are activated or repressed by the maternal effect genes, and they divide the embryo into broad regions, each containing several parasegment primordia. The <i>Krüppel</i> gene, for example, is expressed primarily in parasegments 4<img border="0" src="/books/bookres.fcgi/system_default/ndash.gif" alt=" --">6, in the center of the <i>Drosophila</i> embryo (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1997">Figures 9.19A</a>; <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1972">9.8B</a>); the absence of the Krüppel protein causes the embryo to lack these regions. The protein products of the gap genes interact with neighboring gap gene proteins to activate the transcription of the <b>pair-rule genes</b>. The products of these genes subdivide the broad gap gene regions into parasegments. Mutations of pair-rule genes, such as <i>fushi tarazu</i> (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1972">Figures 9.8C</a>, <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1997">9.19B</a>, <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1998">9.20</a>), usually delete portions of alternate segments. Finally, the <b>segment polarity</b> genes are responsible for maintaining certain repeated structures within each segment. Mutations in these genes cause a portion of each segment to be deleted and replaced by a mirror-image structure of another portion of the segment. For instance, in <i>engrailed</i> mutants, portions of the posterior part of each segment are replaced by duplications of the anterior region of the subsequent segment (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1997">Figures 9.19C</a>, <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1972">9.8D</a>). Thus, the segmentation genes are transcription factors that use the gradients of the early-cleavage embryo to transform the embryo into a periodic, parasegmental structure.</p><p>After the parasegmental boundaries are set, the pair-rule and gap genes interact to regulate the homeotic selector genes, which determine the identity of each segment. By the end of the cellular blastoderm stage, each segment primordium has been given an individual identity by its unique constellation of gap, pair-rule, and <span style='background-color:#ffffCC'>homeotic</span> <span style='background-color:#ffffCC'>gene</span> products (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2167">Levine and Harding 1989</a>).</p><a name="1999"></a><span class="subsect2-header">The gap genes</span><br><br><p>The gap genes were originally discovered through a series of mutant embryos that lacked groups of consecutive segments (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2000">Figure 9.21</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=6776413">Nüsslein-Volhard and Wieschaus 1980</a>). Deletions caused by mutations of the <i>hunchback,</i><i>Krüppel,</i> and <i>knirps</i> genes span the entire segmented region of the <i>Drosophila</i> embryo. The <i>giant</i> gap gene overlaps with these three, and mutations of the <i>tailless</i> and <i>huckebein</i> genes delete portions of the unsegmented termini of the embryo.</p><p>The expression of the gap genes is dynamic. There is usually a low level of transcriptional activity across the entire embryo that becomes defined into discrete regions of high activity as cleavage continues (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2146">Jäckle et al. 1986</a>). The critical element appears to be the expression of the Hunchback protein, which by the end of nuclear division cycle 12 is found at high levels across the anterior part of the embryo, and then forms a steep gradient through about 15 nuclei. The last third of the embryo has undetectable Hunchback levels. The transcription patterns of the anterior gap genes are initiated by the different concentrations of the Hunchback and Bicoid proteins. High levels of Hunchback protein induce the expression of <i>giant,</i> while the <i>Krüppel</i> transcript appears over the region where Hunchback begins to decline. High levels of Hunchback protein also prevent the transcription of the posterior gap genes (such as <i>knirps</i>) in the anterior part of the embryo (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1568245">Struhl et al. 1992</a>). It is thought that a gradient of the Caudal protein, highest at the posterior pole, is responsible for activating the abdominal gap genes <i>knirps</i> and <i>giant.</i> The <i>giant</i> gene has two methods for its activation, one for its anterior expression band and one for its posterior expression band (Rivera-Pomar 1995; Schulz and Tautz 1995).</p><p>After the establishment of these patterns by the maternal effect genes and Hunchback, the expression of each gap gene becomes stabilized and maintained by interactions between the different gap gene products themselves (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2000">Figure 9.21B</a>).<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.section.1971#2023"><sup><img border="0" src="/books/bookres.fcgi/system_default/ddagger.gif" alt="[double dagger]"></sup></a> For instance, <i>Krüppel</i> gene expression is negatively regulated on its anterior boundary by the Hunchback and Giant proteins and on its posterior boundary by the Knirps and Tailless proteins (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2146">Jäckle et al. 1986</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2896123">Harding and Levine 1988</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1348871">Hoch et al. 1992</a>). If Hunchback activity is lacking, the domain of <i>Krüppel</i> expression extends anteriorly. If Knirps activity is lacking, <i>Krüppel</i> gene expression extends more posteriorly. The boundaries between the regions of gap gene transcription are probably created by mutual repression. Just as the Giant and Hunchback proteins can control the anterior boundary of <i>Krüppel</i> transcription, so Krüppel protein can determine the posterior boundaries of <i>giant</i> and <i>hunchback</i> transcription. If an embryo lacks the <i>Krüppel</i> gene, <i>hunchback</i> transcription continues into the area usually allotted to <i>Krüppel</i> (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2146">Jäckle et al. 1986</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1893878">Kraut and Levine 1991</a>). These boundary-forming inhibitions are thought to be directly mediated by the gap gene products, because all four major gap genes (<i>hunchback,</i><i>giant,</i><i>Krüppel,</i> and <i>knirps</i>) encode DNA-binding proteins that can activate or repress the transcription of other gap genes (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2412131">Knipple et al. 1985</a>; <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2125">Gaul and Jäckle 1990</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1576969">Capovilla et al. 1992</a>).<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="2001"></a><span class="subsect2-header">The pair-rule genes</span><br><br><p>The first indication of segmentation in the fly embryo comes when the pair-rule genes are expressed during the thirteenth division cycle. The transcription patterns of these genes are striking in that they divide the embryo into the areas that are the precursors of the segmental body plan. As can be seen in <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2002">Figure 9.22B</a>-<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2002">E</a>and <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1972">Figure 9.8C</a>, one vertical band of nuclei (the cells are just beginning to form) expresses a pair-rule gene, then another band of nuclei does not express it, and then another band of nuclei expresses it again. The result is a "zebra stripe" pattern along the anterior-posterior axis, dividing the embryo into 15 subunits (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=6420705">Hafen et al. 1984</a>). Eight genes are currently known to be capable of dividing the early embryo in this fashion; they are listed in <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.table.2003">Table 9.2</a>. It is important to note that not all nuclei express the same pair-rule genes. In fact, within each parasegment, each row of nuclei has its own constellation of pair-rule gene expression that distinguishes it from any other row.</p><p>How are some nuclei of the <i>Drosophila</i> embryo told to transcribe a particular gene while their neighbors are told not to transcribe it? The answer appears to come from the distribution of the protein products of the gap genes. Whereas the RNA of each of the gap genes has a very discrete distribution that defines abutting or slightly overlapping regions of expression, the <i>protein</i> products of these genes extend more broadly. In fact, they overlap by at least 8<img border="0" src="/books/bookres.fcgi/system_default/ndash.gif" alt=" --">10 nuclei (which at this stage accounts for about two to three segment primordia). This was demonstrated in a striking manner by <img border="0" src="/books/bookres.fcgi/system_default/xscaron.gif" alt="S">tanojevíc and co-workers (1989). They fixed cellularizing blastoderms (i.e., the stage when cells are beginning to form at the rim of the syncytial embryo), stained the Hunchback protein with an antibody carrying a red dye, and simultaneously stained the Krüppel protein with an antibody carrying a green dye. Cellularizing regions that contained both proteins bound both antibodies and were stained bright yellow (see <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1972">Figure 9.8B</a>). Krüppel protein overlaps with Knirps protein in a similar manner in the posterior region of the embryo (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2331752">Pankratz et al. 1990</a>).</p><p>Three genes are known to be the primary pair-rule genes. These genes<img border="0" src="/books/bookres.fcgi/system_default/mdash.gif" alt=" ---"><i>hairy,</i><i>even-skipped,</i> and <i>runt</i><img border="0" src="/books/bookres.fcgi/system_default/mdash.gif" alt=" ---">are essential for the formation of the periodic pattern, and they are directly controlled by the gap gene proteins. The enhancers of the primary pair-rule genes are recognized by gap gene proteins, and it is thought that the different concentrations of gap gene proteins determine whether a pair-rule gene is transcribed or not. The enhancers of the primary pair-rule genes are often modular: the control over each stripe is located in a discrete region of the DNA. One of the best-studied enhancers is that for the <i>even-skipped</i> gene. The structure of this enhancer is shown in <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2002">Figure 9.22A</a>. It is composed of modular units arranged in such a way that each unit regulates a separate stripe. For instance, the second <i>even-skipped</i> stripe is repressed by both Giant and Krüppel proteins and is activated by Hunchback protein and low concentrations of Bicoid (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2004">Figure 9.23</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2026328">Small et al. 1991</a>, <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1327756">1992</a>; <img border="0" src="/books/bookres.fcgi/system_default/xscaron.gif" alt="S">tanojevíc et al. 1991). DNase I footprinting (see <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.chapter.728">Chapter 5</a>) showed that the enhancer region for this stripe contains six binding sites for Krüppel protein, three for Hunchback protein, three for Giant protein, and five for Bicoid protein. Similarly, <i>even-skipped</i> stripe 5 is regulated negatively by Krüppel protein (on its anterior border) and by Giant protein (on its posterior border) (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8626035">Small et al. 1996</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=10226011">Fujioka 1999</a>).</p><p>The importance of these enhancers can be shown by both genetic and biochemical means. First, a mutation in a particular enhancer can delete its particular stripe and no other. Second, if a reporter gene such as <i>lacZ</i> (encoding <font face="symbol">b</font>-galactosidase) is fused to one of these enhancer elements, the <i>lacZ</i> gene is expressed only in that particular stripe (see <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2002">Figure 9.22</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=10226011">Fujioka et al. 1999</a>). Third, the placement of the stripes can be altered by deleting the gap genes that regulate them. Thus, the placement of the stripes of pair-rule gene expression is a result of (1) the modular <i>cis</i>-regulatory enhancer elements of the pair-rule genes and (2) the <i>trans</i>-regulatory gap gene proteins that bind to these enhancer sites.</p><p>Once initiated by the gap gene proteins, the transcription pattern of the primary pair-rule genes becomes stabilized by their interactions among themselves (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2167">Levine and Harding 1989</a>). The primary pair-rule genes also form the context that allows or inhibits the expression of the later-acting secondary pair-rule genes. One such secondary pair-rule gene is <b><i>fushi tarazu</i></b> (<b><i>ftz</i></b>; Japanese, "too few segments;" <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1972">Figures 9.8</a>, <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1997">9.19</a>, <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1998">9.20</a>). Early in cycle 14, <i>ftz</i> RNA and protein are seen throughout the segmented portion of the embryo. However, as the proteins from the primary pair-rule genes begin to interact with the <i>ftz</i> enhancer, the <i>ftz</i> gene is repressed in certain bands of nuclei to create interstripe regions. Meanwhile, the Ftz protein interacts with its own promoter to stimulate more transcription of the <i>ftz</i> gene (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2005">Figure 9.24</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3096577">Edgar et al. 1986b</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2697548">Karr and Kornberg 1989</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1574120">Schier and Gehring 1992</a>).</p><p>The expression of the each pair-rule gene in seven stripes divides the embryo into fourteen parasegments, with each pair-rule gene being expressed in alternate parasegments. Moreover, each row of nuclei within each parasegment expresses a particular and unique combination of pair-rule products. These products will activate the next level of segmentation genes, the segment polarity genes.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="2006"></a><span class="subsect2-header">The segment polarity genes</span><br><br><p>So far, our discussion has described interactions between molecules within the syncytial embryo. But once cells form, interactions take place between the cells. These intercellular interactions are mediated by the segment polarity genes, and they accomplish two important tasks. First, they reinforce the parasegmental periodicity established by the earlier transcription factors. Second, through this cell-to-cell signaling, cell fates are established within each parasegment.</p><p>The segment polarity genes encode proteins that are constituents of the Wingless and Hedgehog signal transduction pathways (see <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.chapter.1011">Chapter 6</a>). Mutations in these genes lead to defects in segmentation and in gene expression pattern across each parasegment. The development of the normal pattern relies on the fact only one row of cells in each parasegment is permitted to express the Hedgehog protein, and only one row of cells in each parasegment is permitted to express the Wingless protein. The key to this pattern is the activation of the <i>engrailed</i> gene in those cells that are going to express the Hedgehog protein. The <i>engrailed</i> gene is activated when cells have high levels of the Even-skipped, Fushi tarazu, or Paired transcription factors. Moreover, it is repressed in those cells that receive high levels of Odd-skipped, Runt, or Sloppy-paired proteins. As a result, Engrailed is expressed in fourteen stripes across the anterior-posterior axis of the embryo (see <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.1972">Figure 9.8D</a>). (Indeed, in mutations that cause the embryo to be deficient in Fushi tarazu, only seven bands of Engrailed are expressed.) These stripes of <i>engrailed</i> transcription mark the anterior boundary of each parasegment (and the posterior border of each segment). The <i>wingless</i> gene is activated in those bands of cells that receive little or no Even-skipped or Fushi tarazu proteins, but which do contain the Sloppy-paired protein. This causes <i>wingless</i> to be transcribed solely in the row of cells directly anterior to the cells where <i>engrailed</i> is transcribed (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2007">Figure 9.25</a>).</p><p>Once <i>wingless</i> and <i>engrailed</i> expression is established in adjacent cells, this pattern must be maintained to retain the parasegmental periodicity of the body plan established by the pair-rule genes. It should be remembered that the mRNAs and proteins involved in initiating these patterns are short-lived, and that the patterns must be maintained after their initiators are no longer being synthesized. The maintenance of these patterns is regulated by interactions between cells expressing <i>wingless</i> and those expressing <i>engrailed.</i> The Wingless protein, secreted from the <i>wingless</i>-expressing cells, diffuses to adjacent cells. The cells expressing <i>engrailed</i> can bind this protein because they contain the <i>Drosophila</i> membrane receptor protein for Wingless, D-Frizzled-2 (see Figure 6.23; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8717036">Bhanot et al. 1996</a>). This receptor activates the Wnt signal transduction pathway, resulting in the continued expression of <i>engrailed</i> (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8107779">Siegfried et al. 1994</a>).</p><p>Moreover, this activation starts another portion of this reciprocal pathway. The Engrailed protein activates the transcription of the <i>hedgehog</i> gene in the <i>engrailed</i>-expressing cells. The Hedgehog protein can bind to the Hedgehog receptor (the Patched protein) on neighboring cells. When it binds to the adjacent posterior cells, it stimulates the expression of the <i>wingless</i> gene. The result is a reciprocal loop wherein the Engrailed-synthesizing cells secrete the Hedgehog protein, which maintains the expression of the <i>wingless</i> gene in the neighboring cells, while the Wingless-secreting cells maintain the expression of the <i>engrailed</i> and <i>hedgehog</i> genes in their neighbors in turn (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1861720">Heemskerk et al. 1991</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1653906">Ingham et al. 1991</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1280560">Mohler and Vani 1992</a>). In this way, the transcription pattern of these two types of cells is stabilized. This interaction creates a stable boundary, as well as a signaling center from which Hedgehog and Wingless proteins diffuse across the parasegment.</p><p>The diffusion of these proteins is thought to provide the gradients by which the cells of the parasegment acquire their identities. This process can be seen in the dorsal epidermis, where the rows of larval cells produce different cuticular structures depending on their position within the segment. The 1° row consists of large, pigmented spikes called denticles. Posterior to these cells, the 2° row produces a smooth epidermal cuticle. The next two cell rows have a 3° fate, making small, thick hairs, and these are followed by several rows of cells that adopt the 4° fate, producing fine hairs (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2008">Figure 9.26</a>).</p><p>The <i>wingless</i>-expressing cells lie within the region producing the fine hairs, while the <i>hedgehog</i>-expressing cells are near the 1° row of cells. The fates of the cells can be altered by experimentally increasing or decreasing the levels of Hedgehog or Wingless protein (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8313468">Heemskerk and DiNardo 1994</a>;<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8620835">Bokor and DiNardo 1996</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=8689684">Porter et al. 1996</a>). For example, if the <i>hedgehog</i> gene is fused to a heat shock promoter and the embryos are grown at a temperature that activates the gene, more Hedgehog protein is made, and the cells normally showing 3° fates will become 2° cells. The rows of 4° cells farthest from the Wingless-secreting cells may also become 3° or 2° cells. It seems that the cells closest to the Wingless secreters cannot respond to Hedgehog, and Hedgehog cannot, by itself, specify the 1° fate (which may require the expression of certain pair-rule gene products). Thus, Hedgehog and Wingless appear necessary for elaborating the entire pattern of cell types across the parasegment. However, the mechanism by which they accomplish this specification is not clear. Either these signals act in a graded fashion, as morphogens, or they act locally to initiate a cascade of local signaling events, in which each interaction uses a different ligand and receptor (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2008">Figure 9.26</a>). The resulting pattern of cell fates also changes the focus of patterning from parasegment to segment. There are now external markers, as the <i>engrailed</i>-expressing cells become the most posterior cells of each segment.</p><center><table width="400" class="message" cellpadding="3" cellspacing="0" border="0">
<tr><td class="bq-header" align="center"><span class="bq-header">WEBSITE</span></td></tr>
<tr><td class="message"><p><b>9.7 Asymmetrical spread of morphogens.</b> It is unlikely that morphogens such as Wingless spread by free diffusion. The asymmetry of Wingless diffusion suggests that neighboring cells play a crucial role in moving the protein. <a href="http://www.devbio.com/chap09/link0907.shtml" target="new">http://www.devbio.com/chap09/link0907.shtml</a></p></td></tr>
</table></center><br><center><table width="400" class="message" cellpadding="3" cellspacing="0" border="0">
<tr><td class="bq-header" align="center"><span class="bq-header">WEBSITE</span></td></tr>
<tr><td class="message"><p><b>9.8 Getting a head in the fly.</b> The segment polarity genes may act differently in the head than in the trunk. Indeed, the formation of the <i>Drosophila</i> head may differ significantly from the way the rest of the body is formed. <a href="http://www.devbio.com/chap09/link0908.shtml" target="new">http://www.devbio.com/chap09/link0908.shtml</a></p></td></tr>
</table></center><br><a name="2009"></a><span class="subsect1-header">The Homeotic Selector Genes</span><br><br><a name="2010"></a><span class="subsect2-header">Patterns of <span style='background-color:#ffffCC'>homeotic</span> <span style='background-color:#ffffCC'>gene</span> expression</span><br><br><p>After the segmental boundaries have been established, the characteristic structures of each segment are specified. This specification is accomplished by the <b>homeotic selector genes</b> (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=103000">Lewis 1978</a>). There are two regions of <i>Drosophila</i> chromosome 3 that contain most of these homeotic genes (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2011">Figure 9.27</a>). One region, the <b>Antennapedia complex</b>, contains the homeotic genes <i>labial</i> (<i>lab</i>), <i>Antennapedia</i> (<i>Antp</i>), <i>sex combs reduced</i> (<i>scr</i>), <i>deformed</i> (<i>dfd</i>), and <i>proboscipedia</i> (<i>pb</i>). The <i>labial</i> and <i>deformed</i> genes specify the head segments, while <i>sex combs reduced</i> and <i>Antennapedia</i> contribute to giving the thoracic segments their identities. The <i>proboscipedia</i> gene appears to act only in adults, but in its absence, the labial palps of the mouth are transformed into legs (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=6421639">Wakimoto et al. 1984</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1971986">Kaufman et al. 1990</a>). The second region of homeotic genes is the <b>bithorax complex</b> (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=103000">Lewis 1978</a>). There are three protein-coding genes found in this complex: <i>ultrabithorax</i> (<i>ubx</i>), which is required for the identity of the third thoracic segment; and the <i>abdominal A</i> (<i>abdA</i>) and <i>Abdominal B</i> (<i>AbdB</i>) genes, which are responsible for the segmental identities of the abdominal segments (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3917555">Sánchez-Herrero et al. 1985</a>). The lethal phenotype of the triple-point mutant Ubx<sup>-</sup>, abdA<sup>-</sup>, AbdB<sup>-</sup> is identical to that resulting from a deletion of the entire bithorax complex (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=14650432">Casanova et al. 1987</a>). The chromosome region containing both the Antennapedia complex and the bithorax complex is often referred to as the <b>homeotic complex (Hom-C)</b>.</p><p>Because these genes are responsible for the specification of fly body parts, mutations in them lead to bizarre phenotypes. In 1894, William Bateson called these organisms "homeotic mutants," and they have fascinated developmental biologists for decades.<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.section.1971#2024"><sup><img border="0" src="/books/bookres.fcgi/system_default/sect.gif" alt="[section]"></sup></a> For example, the body of the normal adult fly contains three thoracic segments, all of which produce a pair of legs. The first thoracic segment does not produce any further appendages, but the second thoracic segment produces both a set of legs and a set of wings. The third thoracic segment produces a set of wings and a set of balancers known as <b>halteres</b>. In homeotic mutants, these specific segmental identities can be changed. When the <i>ultrabithorax</i> gene is deleted, the third thoracic segment (which is characterized by halteres) becomes transformed into another second thoracic segment. The result (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2012">Figure 9.28</a>) is a fly with four wings<img border="0" src="/books/bookres.fcgi/system_default/mdash.gif" alt=" ---">an embarrassing situation for a classic dipteran. <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.section.1971#2025"><sup><img border="0" src="/books/bookres.fcgi/system_default/para.gif" alt="[paragraph]"></sup></a> Similarly, the Antennapedia protein is usually used to specify the second thoracic segment of the fly. But when flies have a mutation wherein the <i>Antennapedia</i> gene is expressed in the head (as well as in the thorax), legs rather than antennae grow out of the head sockets (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2013">Figure 9.29</a>). In the recessive mutant of <i>Antennapedia,</i> the gene fails to be expressed in the second thoracic segment, and antennae sprout out of the leg positions (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=7254358">Struhl 1981</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2430723">Frischer et al. 1986</a>; <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2215">Schneuwly et al. 1987</a>).</p><p>These major homeotic selector genes have been cloned and their expression analyzed by in situ hybridization (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=3898362">Harding et al. 1985</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2896587">Akam 1987</a>). Transcripts from each gene can be detected in specific regions of the embryo and are especially prominent in the central nervous system (see <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2011">Figure 9.27</a>).<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="2014"></a><span class="subsect2-header">Initiating the patterns of <span style='background-color:#ffffCC'>homeotic</span> <span style='background-color:#ffffCC'>gene</span> expression</span><br><br><p>The initial domains of <span style='background-color:#ffffCC'>homeotic</span> <span style='background-color:#ffffCC'>gene</span> expression are influenced by the gap genes and pair-rule genes. For instance, the expression of the <i>abdA</i> and <i>AbdB</i> genes is repressed by the gap gene proteins Hunchback and Krüppel. This inhibition prevents these abdomen-specifying genes from being expressed in the head and thorax (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=7600999">Casares and Sánchez-Herrero 1995</a>). Conversely, the <i>Ultrabithorax</i> gene is activated by certain levels of the Hunchback protein, so that it is originally transcribed in a broad band in the middle of the embryo, and the transcription of <i>Antennapedia</i> is activated by Krüppel (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2896123">Harding and Levine 1988</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1568245">Struhl et al. 1992</a>). The boundaries of <span style='background-color:#ffffCC'>homeotic</span> <span style='background-color:#ffffCC'>gene</span> expression are soon confined to the parasegments defined by the Fushi tarazu and Even-skipped proteins (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=2878371">Ingham and Martinez-Arias 1986</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1356761">Müller and Bienz 1992</a>).<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="2015"></a><span class="subsect2-header">Maintaining the patterns of <span style='background-color:#ffffCC'>homeotic</span> <span style='background-color:#ffffCC'>gene</span> expression</span><br><br><p>The expression of homeotic genes is a dynamic process. The <i>Antennapedia</i> gene (<i>Antp</i>), for instance, although initially expressed in presumptive parasegment 4, soon appears in parasegment 5. As the germ band expands, <i>Antp</i> expression is seen in the presumptive neural tube as far posterior as parasegment 12. During further development, the domain of <i>Antp</i> expression contracts again, and <i>Antp</i> transcripts are localized strongly to parasegments 4 and 5. Like that of other homeotic genes, <i>Antp</i> expression is negatively regulated by all the <span style='background-color:#ffffCC'>homeotic</span> <span style='background-color:#ffffCC'>gene</span> products expressed posterior to it (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2134">Harding and Levine 1989</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1970762">González-Reyes and Morata 1990</a>). In other words, each of the bithorax complex genes represses the expression of <i>Antennapedia.</i> If the <i>Ultrabithorax</i> gene is deleted, <i>Antp</i> activity extends through the region that would normally have expressed <i>Ubx</i> and stops where the <i>Abd</i> region begins. (This allows the third thoracic segment to form wings like the second thoracic segment, as in <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2013">Figure 9.29</a>.) If the entire bithorax complex is deleted, <i>Antp</i> expression extends throughout the abdomen. (Such a larva does not survive, but the cuticle pattern throughout the abdomen is that of the second thoracic segment.)</p><p>As we saw above, the gap gene and pair-rule gene proteins are transient, but the identities of the segments must be stabilized so that differentiation can occur. Thus, once the transcription patterns of the homeotic genes have become stabilized, they are "locked" into place by alteration of the chromatin conformation in these genes. The repression of homeotic genes appears to be maintained by the Polycomb family of proteins, while the active chromatin conformation appears to be maintained by the Trithorax proteins (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2143">Ingham and Whittle 1980</a>; <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.biblist.2076#2175">McKeon and Brock 1991</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1350533">Simon et al. 1992</a>).<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="2016"></a><span class="subsect2-header">Realisator genes</span><br><br><p>The search is now on for "realisator genes," those genes that are the targets of the <span style='background-color:#ffffCC'>homeotic</span> <span style='background-color:#ffffCC'>gene</span> proteins and which function to form the specified tissue or organ primordia. In the formation of the second thoracic segment, for example, <i>Antennapedia</i> is expressed. <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=9565034">Casares and Mann (1998)</a> have shown that Antennapedia protein binds to the enhancer of the <i>homothorax</i> gene and prevent its expression. Homothorax is necessary for producing a transcription factor critical for antenna formation. Therefore, one of Antennapedia's functions is to suppress those genes necessary for antenna development.</p><p>The Ultrabithorax protein is able to repress the expression of the <i>Wingless</i> gene in those cells that will become the halteres of the fly. One of the major differences between the appendage-forming cells of the second and the third thoracic segments is that Wingless expression occurs in the appendage-forming cells of the second thoracic segment, but not in those of the third thoracic segment. Wingless acts as a growth promoter and morphogen in these tissues. In the third thoracic segment, Ubx protein is found in these cells, and it prevents the expression of the <i>Wingless</i> gene (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2017">Figure 9.30</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=9585507">Weatherbee et al. 1998</a>). Thus, one of the ways in which Ubx protein specifies the third thoracic segment is by preventing the expression of those genes that would generate the wing tissue.</p><p>Another target of the homeotic proteins, the <i>distal-less</i> gene (itself a homeobox-containing gene: see Sidelights and Speculations), is necessary for limb development and is active solely in the thorax. Distal-less expression is repressed in the abdomen, probably by a combination of Ubx and AbdA proteins that can bind to its enhancer and block transcription (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1358457">Vachon et al. 1992</a>; <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=7555723">Castelli-Gair and Akam 1995</a>). This presents a paradox, since parasegment 5 (entirely thoracic and leg-producing) and parasegment 6 (which includes most of the legless first abdominal segment) both expressUbx. How can these two very different segments be specified by the same gene? <a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=7555723">Castelli-Gair and Akam (1995)</a> have shown that the mere presence of Ubx protein in a group of cells is not sufficient for specification. Rather, the time and place of its expression within the parasegment can be critical. Before <i>Ubx</i> expression, parasegments 4<img border="0" src="/books/bookres.fcgi/system_default/ndash.gif" alt=" --">6 have similar potentials. At division cycle 10, <i>Ubx</i> expression in the anterior parts of parasegments 5 and 6 prevents those parasegments from forming structures (such as the anterior spiracle) characteristic of parasegment 4. Moreover, in the posterior compartment of parasegment 6 (but not parasegment 5), Ubx protein blocks the formation of the limb primordium by repressing the <i>distal-less</i> gene. At division cycle 11, by which time Ubx has pervaded all of parasegment 6, the <i>distal-less</i> gene has become self-regulatory and cannot be repressed by Ubx (<a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.figgrp.2018">Figure 9.31</a>).<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><hr size="1"><span class="footnote"><a name="2021"></a><p>*Like the placement of the <i>bicoid</i> message, the location of the <i>nanos</i> message is determined by its 3´ untranslated region. If the <i>bicoid</i> 3´ UTR is experimentally placed on the protein-encoding region of <i>nanos</i> RNA, the <i>nanos</i> message gets placed in the anterior of the egg. When the RNA is translated, the Nanos protein inhibits the translation of <i>hunchback</i> and <i>bicoid</i> mRNAs, and the embryo forms two abdomens<img border="0" src="/books/bookres.fcgi/system_default/mdash.gif" alt=" ---">one in the anterior of the embryo and one in the posterior (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1423595">Gavis and Lehmann 1992</a>). The localization of <i>nanos</i> RNA is ultimately dependent on interactions between the oocyte and the neighboring follicle cells that localize the <i>oskar</i> message to the posterior pole.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p>
<a name="2022"></a><p><sup><img border="0" src="/books/bookres.fcgi/system_default/dagger.gif" alt="[dagger]"></sup>Aficionados of information theory will recognize that the process by which the anterior-posterior information in morphogenetic gradients is transferred to discrete and different parasegments represents a transition from analog to digital specification. Specification is analog, determination digital. This process enables the transient information of the gradients in the syncytial blastoderm to be stabilized so that it can be utilized much later in development (<a class="text-ref" href="/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=1982920">Baumgartner and Noll 1990</a>).<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p>
<a name="2023"></a><p><sup><img border="0" src="/books/bookres.fcgi/system_default/ddagger.gif" alt="[double dagger]"></sup>The interactions between genes and gene products are facilitated by the fact that these reactions occur within a syncytium, in which the cell membranes have not yet formed.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p>
<a name="2024"></a><p><sup><img border="0" src="/books/bookres.fcgi/system_default/sect.gif" alt="[section]"></sup><i>Homeo</i> means "similar." <i>Homeotic mutants</i> are mutants in which one structure is replaced by another (as where an antenna is replaced by a leg). <i>Homeotic genes</i> are genes whose mutation can cause such transformations; thus, they are genes that specify the identity of a particular body segment. The <i>homeobox</i> is a conserved DNA sequence of about 180 base pairs that is shared by many homeotic genes. This sequence encodes the 60-amino acid <i>homeodomain,</i> which recognizes specific DNA sequences. The homeodomain is an important region of the transcription factors encoded by homeotic genes (see Sidelights &amp; Speculations). Not all genes with homeoboxes are homeotic genes, however.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p>
<a name="2025"></a><p><sup><img border="0" src="/books/bookres.fcgi/system_default/para.gif" alt="[paragraph]"></sup>Dipterans (two-winged insects such as flies) are thought to have evolved from four-winged insects; it is possible that this change arose via alterations in the bithorax complex. <a class="text-ref" href="/books/bv.fcgi?call=bv.View..ShowSection&amp;rid=dbio.chapter.5398">Chapter 22</a> includes more speculation on the relationship between the homeotic complex and evolution.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p></span>
</td>
        </tr>
        </table> 

 <br><center><small>&copy; 2000 <a href="http://www.sinauer.com/">by Sinauer Associates, Inc.</a></small></center><br>

 <br>

<!-- redirect empty query -->
<script type="text/javascript">
  var s = document.location.href;
  if(document.forms.length>0 && document.forms[0].name=='redirect_form')
  {
      var rs = document.forms[0].redirect.value;
      if(rs.indexOf('google')>-1 && (rs.indexOf('medline')>-1 || rs.indexOf('pubmed')>-1))
      {
//          document.write('Redirect to PubMed site: '+s.substr(0, s.indexOf('books/bv.fcgi'))+'entrez/');
          document.location.href = s.substr(0, s.indexOf('books/bv.fcgi'))+'entrez/';

       }
       else
      {
//         document.write('Redirect to Bookshelf site: '+s.substr(0, s.indexOf('bv.fcgi')));
         document.location.href = s.substr(0, s.indexOf('bv.fcgi'));
       }
  }  
</script>


</body>
</html>
