<html>
<head>
<title>Distinctive Biology of Forest Trees Highlighted at Sixth 
International Meeting</title>
<!--file distinct.html-->
</head> 
<body>
<h2>Distinctive Biology of Forest Trees Highlighted at Sixth International
Meeting</h2>
<p>
Published in Probe Volume 4(1-2): July 1993-July 1994
<hr>
<p>
<strong>Claire Kinlaw</strong> and <strong>David Harry</strong><br>
Institute of Forest Genetics, USDA, Forest Service<br>
Berkeley, CA 94701<p>

<p>
Forest trees and their molecular genetics were the focus of a recent
conference (Prouts Neck, Maine, USA, May 20-23, 1994) organized by
Michael Greenwood and Keith Hutchison (University of Maine, USA). A
group of 70 researchers gathered to discuss progress toward
understanding and manipulating molecular processes within this diverse
group of economically and ecologically important species. 

<p>
The biology of forest trees provides both distinct challenges and
unique opportunities. Compared to previous meetings of this research
community, significant progress has been made in several areas.
Research teams continue to isolate and characterize new genes, while
transgenic plants, especially in <i>Populus</i>, are increasingly being used
to study gene function <i>in vivo</i>. Genome mapping has also matured. In
earlier meetings, presentations described the construction of genetic
linkage maps, while at this meeting maps were presented as tools to
identify and dissect quantitative trait loci (QTL). 

<p>
Advantages of the haploid genetics offered by conifer gametophytes
continue to be exploited for mapping work and for population surveys.
Recent advances in model organisms continue to influence studies in
forest trees. Homeotic genes, for example, are being sought for flower
and cone development. Other research focuses on processes that either
are unique to trees or are simply more important in trees than in other
organisms. For example, lignin and its related biosynthetic pathways
are affected by wounding and stress in trees as in other plants, while
in trees alone, lignin also is a major component of wood. 

<p>
<strong>Gene Expression Patterns</strong>

<p>
As highlighted by Olof Olsson (Swedish University of Agricultural
Sciences, Sweden), woody angiosperms show fluctuations in gene
expression during annual cycles of quiescence and "reesence" in
addition to changes observed during development and in response to
environmental stresses. With the goal of increasing pulp production
efficiency, Wout Boerjan's laboratory (Universiteit Gent, Belgium) has
produced transgenic <i>Populus</i> plants containing antisense constructs for
the lignin biosynthetic enzymes O-methyltransferase (OMT) and cinnamyl
alcohol dehydrogenase (CAD). Work on conifer lignin and its role in
development continues despite being hindered by the lack of reliable
transformation and regeneration methods. Working on <i>Pinus taeda</i>, John
Mackay (North Carolina State Biotechnology Group, USA, directed by Ron
Sederoff) described the isolation and characterization of several genes
encoding lignin biosynthetic enzymes including phenylalanine ammonia
lyase (PAL), cinnamyl alcohol dehydrogenase (CAD), and 4-coumarate:CoA
ligase (4-CL). 

<p>
A slightly different tack toward understanding xylogenesis is being
taken by Mackay's colleague, Malcolm Campbell. Because myb-like genes
play an important role in signal transduction pathways in other
organisms, such genes may also control conifer xylem development.
Campbell has initiated cloning of <i>Pinus taeda</i> homologues. A similar
rationale underlies the strategy of Sharon Regan and Bob Rutledge
(Petawawa National Forestry Institute, Canada) in their efforts to
characterize MADS box homeotic genes controlling cone development in
<i>Picea mariana</i>.

<p>
With the goal of understanding the role of flavonoids in root
formation, Lise Jouanin's laboratory (INRA, Versailles Cedex, France)
has produced transgenic <i>Populus</i> and <i>Juglans</i> containing altered 
levels
of chalcone synthase (CHS). Carmen Diaz-Sala, Keith Hutchison, and Mike
Greenwood (University of Maine, USA) are investigating the cellular and
molecular changes associated with the organization of root primordia in
<i>Pinus taeda</i>. In particular, they are addressing how the cytoskeleton
orients the plane of cellular divisions and nuclear reorganization.

<p>
Engineering for pest resistance using proteinase inhibitors is being
done by several groups including those of Ned Klopfenstein (USDA-FS
Center for Semiarid Agroforestry, USA), Lise Jouanin, and Seguin Armand
(Universite Laval, Canada). Jouanin has found a cysteine protease
inhibitor to be particularly effective against pests which contain high
levels of cysteine proteases.

<p>
Genes responding to major environmental stresses have been identified
in stress-specific cDNA libraries. Genes induced by drought stress in
<i>Pinus taeda</i> (Shujun Chang et al., Texas A&amp;M University, USA) 
include
caffeoyl CoA, SAM synthetase, chitinase, and a protein similar to an
animal skin matrix component. Genes induced by ozone (Dieter Ernst,
Institut fur Biochemische Pflanzenpathologie, Germany) in <i>Pinus
sylvestris</i> include CAD, stilbene synthase,
hydroxymethylglutaryl-CoA-synthase, and polyubiquitin. As found in
angiosperms, certain conifer genes appear to be induced by a number of
environmental stresses. For example chitinase is induced by wounding,
fungal infection, and drought (Haiguo Wu, Craig Echt, and John Davis,
University of Florida, USA). 

<p>
To expand upon the identification of new conifer genes, Claire Kinlaw
(Institute of Forest Genetics, USA) has initiated "single-pass"
sequencing efforts of <i>Pinus taeda</i>  seedling cDNAs used as markers by
David Neale and co-workers (Institute of Forest Genetics, USA) for
genetic maps. These identified sequences will provide molecular tools
for studying conifer genome organization and evolution. Early results
have been encouraging in that a variety of genes have been identified
including those encoding photosynthetic proteins, translation factors,
glycolytic enzymes, and stress-response proteins.

<p>
With a systemic point of view, Gary Coleman (Oregon State University,
USA) proposed a model to explain how trees regulate autumn nitrogen
storage and spring remobilization in response to nitrogen availability
and photoperiod. In this model, bark and leaves communicate with each
other using a bark storage protein (BSP) and a leaf protein encoded by
Win4. During short days or high levels of nitrogen, BSP accumulates in
bark parenchyma while the Win4-encoded protein is repressed. During
long day shoot growth, BSP is degraded while the Win4-encoded protein
accumulates.

<p>
<strong>Genetic Maps and Quantitative Trait Loci</strong>

<p>
Genetic maps using two alternative approaches are being used to
identify QTLs. Lively discussions of the merits and disadvantages of
these two alternate approaches accompanied formal presentations. Mitch
Sewell (Institute of Forest Genetics, USA) described the integration of
restriction fragment length polymorphism (RFLP)-based linkage maps from
two <i>Pinus taeda</i>  pedigrees. This work will further efforts by 
Neale and
his co-workers to dissect wood quality traits and to understand conifer
genome organization and evolution. Several members of the Forest
Biotechnology Group at North Carolina State University (USA) presented
RAPD-based maps including those from <i>Eucalyptus</i>  (Dario Grattapaglia)
for the identification of QTL controlling sprouting and rooting.

<p>
<strong>New Markers</strong>

<p>
Several laboratories are exploring the use of length polymorphism among
simple sequence repeats (SSRs). Craig Echt, (USDA Forest Service,
Rhinelander, USA) has observed that approximately 0.7% of the <i>Pinus
strobus</i> genome is comprised of SSRs. Of the primer pairs tested from
the flanking sequences of <i>Pinus strobus</i> SSR loci, approximately 65%
reliably amplify DNA. A high proportion show size polymorphisms, and a
significant number amplify DNA from other conifer taxa. In apparent
contrast to these observations, Keith Hutchison (University of Maine,
USA) has observed a low level of size polymorphisms among SSR alleles
in <i>Larix laricina</i>.

<p>
With a similar goal of developing co-dominant PCR-based markers, but
using a different approach, David Harry (Institute of Forest Genetics,
USA) is designing and testing primers based upon sequences of specific
<i>Pinus taeda</i> cDNAs. Approximately 75% of the primer pairs reliably
amplify genomic DNA, with a high proportion revealing Mendelian
polymorphisms following restriction enzyme digestion. Some primers
amplify only hard pines, others amplify all pines, and still others
amplify DNA from other conifer taxa. Hisato Okuizumi presented an
application of restriction landmark genomic scanning (RLGS) to large
genomes by including a restriction trapper. High-speed scanning of
entire genomes and the construction of genetic maps of individual trees
from a single run with several hundred loci are made possible. As an
example, a profile of <i>Pinus koraiensis</i> was shown.

<p>
<strong>Describing Genome Flux and Evolution</strong>

<p>
Because seed plants represent an ancient lineage, and because woody
plants have long generation times, mechanisms of genetic mutation and
genome evolution, as well as rates of species evolution, continue to be
important areas of study. In seeming contrast to low levels of observed
SSR polymorphism in <i>Larix laricina</i>, Hutchison and coworkers have found
relatively high levels of sequence variation within genomic regions
encoding proteins. <p>
In addition, they have observed segregation distortion of alleles under
different environments. The apparent contrast between the high levels
of polymorphism among coding regions and low levels of polymorphism
among SSR regions may indicate that conifers have a relatively
efficient mismatch repair mechanism and may thus partially account for
the stability of conifer karyotypes. 

<p>
Jean Bousquet (Universite Laval, Canada) and his co-workers are
investigating ancient events during the evolution of seed plants. After
carefully calibrating a molecular clock, they established that modern
gymnosperms derived from a single lineage, and they estimated
divergence times to have occurred as follows:<p>

liverworts from vascular plants<br>
440 mya<p>

ferns from seed bearing plants<br>
400 mya<p>

flowering from cone bearing plants<br>
290 mya<p>

monocots from dicots<br>
200 mya<p>

Pinus  from Pseudotsuga<br>
140 mya

<p>
Ross Whetten (North Carolina State University, USA) is exploiting the
idea that a tree's crown represents a common lineage of shoots with
known separation times. Using visible phenotypes in peach, Whetten
estimated the somatic excision rate of a transposable element. The rate
is relatively higher than rates reported for annual species and varies
among meristematic layers. This notion that different shoots within the
crown of a tree can be genetically distinct might help explain how
long-lived trees endure pathogens and insect pests with shorter
generation times. 

<p>
<strong>Genetic Diversity and Population Structure</strong>

<p>
In addition to their use in mapping, RAPDs have been used by a number
of laboratories for estimating genetic diversity and describing
population structure. Natalie Isabel (Universite Laval, Canada)
compared estimates of genetic variation within and among populations of
<i>Picea mariana</i> using RAPDs and allozymes. Results from these two types
of markers were similar. Linda DeVerno (Petawawa National Forestry
Institute, Canada) surveyed <i>Pinus resinosa</i>  using 400 RAPD primers and
found no polymorphism. Again this data supports earlier conclusions
based upon allozymes. 

<p>
<strong>1995 Meeting</strong>

<p>
The next tree molecular geneticists meeting will take place at
Universiteit Gent, Belgium, in combination with the IUFRO Somatic Cell
Genetics Working Party. Those wishing more information are encouraged
to contact Wout Boerjan<br>
(Boerjan%research%RUG.genetica@genwet1.rug.ac.be).

</body>
</html>
