<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America"><meta name="citation_title" content="Origin of the metazoan phyla: Molecular clocks confirm paleontological&#x02009;estimates"><meta name="citation_authors" content="Francisco Jos&#x000e9; Ayala, Andrey Rzhetsky, and  Francisco J. Ayala"><meta name="citation_date" content="1998 January 20"><meta name="citation_issue" content="2"><meta name="citation_volume" content="95"><meta name="citation_firstpage" content="606"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=18467&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=18467&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=18467"><meta name="citation_pmid" content="9435239"><meta name="DC.Title" content="Origin of the metazoan phyla: Molecular clocks confirm paleontological&#x02009;estimates"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="National Academy of Sciences"><meta name="DC.Contributor" content="Francisco Jos&#x000e9; Ayala"><meta name="DC.Contributor" content="Andrey Rzhetsky"><meta name="DC.Contributor" content="Francisco J. Ayala"><meta name="DC.Date" content="1998 January 20"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-navy.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody2.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs2.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Origin of the metazoan phyla: Molecular clocks confirm paleontological<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />estimates</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0n1zzhe9bIDiCffVrV-5IdelQb8WGAo6sjZ11D512&amp;reftype=other&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-pnas.gif" alt="Logo of pnas" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="PNAS Home page." coords="306,2,479,64" href="redirect3.cgi?&amp;&amp;auth=08ChHgEOxZ8OrlPV7UBnv65_llJiraCW0HGCJMAV8&amp;reftype=publisher&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.pnas.org" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="rect" alt="Reference to the article." coords="0,34,57,50" onclick="focuswin('pmc_ext') " href="redirect3.cgi?&amp;&amp;auth=0hUJ02O3dUI9XqIv0biiwT1U-suG_9MwE6rx115XH&amp;reftype=publisher&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.pnas.org/cgi/pmidlookup?view=long&amp;pmid=9435239" target="pmc_ext"><area shape="rect" alt="PNAS Info for Authors" coords="59,34,142,50" href="redirect3.cgi?&amp;&amp;auth=0p_gmPnRZjjgjFYZqsBR4UB9Iwe49NWGI6W7L_QmE&amp;reftype=publisher&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.pnas.org/misc/iforc.shtml" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="rect" alt="PNAS Subscriptions" coords="143,34,200,50" href="redirect3.cgi?&amp;&amp;auth=0RCg3QOhHEIvGRgUU06SGXPkI-iXIF_szpDD1o5H8&amp;reftype=publisher&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.pnas.org/subscriptions/" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="rect" alt="PNAS About" coords="201,34,239,50" href="redirect3.cgi?&amp;&amp;auth=0PAkgseMi6lbYn4X77ZYscOX3zePn97W6mcCqsknc&amp;reftype=publisher&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.pnas.org/misc/about.shtml" target="pmc_ext" onclick="focuswin('pmc_ext')"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-ltblue"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=2&amp;action=archive">Proc Natl Acad Sci U S A</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=522">v.95(2); Jan 20, 1998</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=18467&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=18467&amp;blobtype=pdf" class="sidefm-pmclink">PDF (165K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=522" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=2&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0-QivkVDtGvx_oUv0OJEsNkH4UiQHEsYsjNnY2QQZ&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9435239">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=00C7mY8sQqPMKR9CCaoXGDhlFCeYDSNLlakhcskLm&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=9435239">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0le3jvlj5WrN_1eqyuj_TpC-SCaUT17sDir5ZFAae&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=9435239">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0vKvRrIm4FtOGnPh8muqBHX8DeN9wOy1DrIU-ekzi&amp;reftype=Substance&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|PubChem%20Substance&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=18467&amp;dopt=pmc_pcsubstance">Substance</option><option value="redirect3.cgi?&amp;&amp;auth=0BIldU_ShHhnRhDUKzHZ7xKz6uD3JlSsfhqnv45JV&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=18467&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0T7q-tjo4mFruRXL00kd-aprsmcxvSDDuYqYR_5sU&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=18467">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0vEPMZek8y7CiXrSCBgidxq6W-UoQEjkFXugdOkyR&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Ayala+FJ%5Bauth%5D" class="sidefm-pmclink"> Ayala, F.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0RIsNf4z5NZQpZ-b3mcRtdzu0eHrACWjvuIKMD5DU&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Rzhetsky+A%5Bauth%5D" class="sidefm-pmclink"> Rzhetsky, A.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0XweSQLwXsm66uSqBz1ZaU2aQqGo1aRcI_lpJO5rd&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Ayala+FJ%5Bauth%5D" class="sidefm-pmclink"> Ayala, F.</a></div></div></div></td><td rowspan="103" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-pnas.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">Proc Natl Acad Sci U S A. </span><span class="citation-publication-date">1998 January 20; </span><span class="citation-volume">95</span><span class="citation-issue">(2)</span><span class="citation-flpages">: 606–611. </span></div><div><span class="fm-vol-iss-date"> </span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC18467</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 1998, The National Academy of Sciences</div><div class="fm-doctopic">Evolution</div><div class="fm-title">Origin of the metazoan phyla: Molecular clocks confirm paleontological<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />estimates</div><div class="fm-author contrib-group">Francisco Jos&#x000e9; Ayala,<sup>&#x0002a;</sup> Andrey Rzhetsky,<sup>&#x02020;</sup> and  Francisco J. Ayala<sup>&#x02021;</sup><div class="fm-affl"><sup>&#x0002a;</sup>Institute of Molecular Evolutionary Genetics, Pennsylvania State University, University Park, PA 16802; <sup>&#x02020;</sup>Columbia Genome Center, Columbia University, New York, NY 10032; and <sup>&#x02021;</sup>Department of Ecology and Evolutionary Biology, University of California, Irvine, CA 92697</div></div><div class="fm-footnote"></div><div class="fm-footnote"><div class="p">Contributed by Francisco J. Ayala</div></div><div class="fm-pubdate">Accepted November 19, 1997.</div><div class="links-box"><div class="fm-footnote"><img src="corehtml/pmc/pmcgifs/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="tocrender.fcgi?artid=18467&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id406657" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666699" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666906" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id511005" class="sidefm-pmclink" style="text-transform: uppercase;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id663308">Abstract</div><div class="section-content"><!--article-meta-->The time of origin of the animal phyla is controversial. Abundant fossils from the major animal phyla are found in the Cambrian, starting 544 million years ago. Many paleontologists hold that these phyla originated in the late Neoproterozoic, during the 160 million years preceding the Cambrian fossil explosion. We have analyzed 18 protein-coding gene loci and estimated that protostomes (arthropods, annelids, and mollusks) diverged from deuterostomes (echinoderms and chordates) about 670 million years ago, and chordates from echinoderms about 600 million years ago. Both estimates are consistent with paleontological estimates. A published analysis of seven gene loci that concludes that the corresponding divergence times are 1,200 and 1,000 million years ago is shown to be flawed because it extrapolates from slow-evolving vertebrate rates to faster-evolving invertebrate rates, as well as in other ways.</div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id663308" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id406657" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666699" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666906" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id511005" class="sidefm-pmclink" style="text-transform: uppercase;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id663312"> </div><div class="section-content">The time of origin of the metazoan phyla is controversial. A common view is that the first coelomates appeared in the late Neoproterozoic, some 700 million years (My) ago, and the divergence between protostomes and deuterostomes occurred about 600 My ago. The divergence between the deuterostome phyla (echinoderms and chordates) may have occurred during the Vendian, before the beginning of the Cambrian 544 My ago (<a href="#B1" class="cite-reflink bibr">1</a>&#x02013;<a href="#B8" class="cite-reflink bibr">8</a>). Fossil remains of nearly all readily fossilizable animal phyla have been recovered from Cambrian rocks (<a href="#B4" class="cite-reflink bibr">4</a>).<p>This interpretation has been challenged on the grounds that it relies on negative evidence, namely the scarcity of fossil remains preceding the Cambrian, followed by the relatively sudden appearance during the Cambrian of many diverse phyla, classes, and orders. This &#x0201c;Cambrian explosion&#x0201d; might simply reflect the difficulty of preservation and discovery of soft-bodied and perhaps tiny animals (<a href="#B9" class="cite-reflink bibr">9</a>&#x02013;<a href="#B11" class="cite-reflink bibr">11</a>). Resolution of this controversy has been sought in DNA sequence data and the theory of the molecular clock. An early study of cytochrome <em>c</em> sequences yielded results consistent with the Cambrian explosion view, although with slightly earlier dates, placing the origin of two protostome phyla, the annelids and arthropods, at 750 My ago, and the divergence between protostomes and deuterostomes at 720 My ago (<a href="#B12" class="cite-reflink bibr">12</a>&#x02013;<a href="#B16" class="cite-reflink bibr">16</a>). Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>) have, however, recently concluded from the analysis of seven genes that the divergence of protostomes and deuterostomes occurred nearly twice as early as the Cambrian&#x02014;i.e., about 1,200 My ago&#x02014;and that chordates diverged from the echinoderms about 1,000 My ago.</p><p>Crucial to these conclusions and others that rely on molecular data is the hypothesis of the molecular clock that molecular evolutionary rates for a particular gene are constant through time and across taxa. Yet we know that genes can evolve at disparate rates at different times or in different taxa (<a href="#B18" class="cite-reflink bibr">18</a>&#x02013;<a href="#B21" class="cite-reflink bibr">21</a>). In this paper, we examine 18 different genes. Our results are consistent with the Cambrian explosion view&#x02014;namely, they suggest that the divergence of protostomes and deuterostomes occurred in the late Neoproterozoic, around 544&#x02013;700 My ago, and that the divergence between echinoderms and chordates preceded the Cambrian, but not very much. The genes that we have analyzed include six that were also studied by Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>), but we have eliminated by a statistical procedure those branches of the evolutionary tree that are evolving at rates significantly different from the average. Examination of ref. <a href="#B17" class="cite-reflink bibr">17</a> manifests a variety of methodological problems and indicates that the data depart importantly from the assumption of constant evolutionary rates.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id663308" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>MATERIALS AND METHODS</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666699" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666906" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id511005" class="sidefm-pmclink" style="text-transform: uppercase;">References</a></div></span></div></td><td class="content-cell"><span id="Materials"></span><div class="head1 section-title" style="text-transform: uppercase;" id="id406657"><a id="Materials"></a>MATERIALS AND METHODS</div><div class="section-content"><span id="Materials"></span><p><span class="head3">Genes and DNA Sequences. </span> The 18 protein-coding genes are listed in Tables <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> and <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a>. The &#x003b1;- and &#x003b2;-globin genes result from a duplication that occurred during the evolution of higher fishes (gnathostomes); a previous gene duplication, which may have slightly preceded the origin of chordates, yielded the vertebrate hemoglobin and myoglobin genes. The &#x003b1;- and &#x003b2;-globin genes are listed separately in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a>, but only one globin has been analyzed in animals other than the gnathostome vertebrates. The genes listed in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> and the DNA sequences analyzed are the same used by Wray <em>et al.</em> (ref. <a href="#B17" class="cite-reflink bibr">17</a>; and <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0NaP90HOz-D4LQRskRyx2Ylo_d1CV8BQvwOZK_5tl&amp;reftype=extlink&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://life.bio.sunysb.edu/ee/precambrian" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://life.bio.sunysb.edu/ee/precambrian</a></span>). We have omitted the 18S rRNA gene, which they included, because we were unable to obtain reliable alignment of the sequences across the diverse taxa. Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a> lists 12 additional loci for which sequences are available in the electronic databases for several vertebrate taxa and at least one invertebrate taxon, and for which linear trees could be constructed (see below; list of sequences, accession numbers, and alignments are available from the first author upon request).    <div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T1" name="T1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 1" title="Table 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 1</strong></a><div class="figure-table-caption-in-article"><span>Divergence times between chordates and invertebrate phyla, derived from six protein-encoding gene<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />loci</span></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T2" name="T2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T2" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 2" title="Table 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 2</strong></a><div class="figure-table-caption-in-article"><span>Divergence times between chordates and arthropods, derived from 12 protein-encoding<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />genes</span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span class="head3">Sequence Alignment and Genetic Distances. </span> Sequences were aligned by using the <span class="small-caps">clustal w</span> computer program (<a href="#B22" class="cite-reflink bibr">22</a>) with adjustments made by eye. Genetic distances were calculated in two ways: with the gamma correction for multiple amino acid replacements; and with the Poisson correction (<a href="#B23" class="cite-reflink bibr">23</a>). We set the gamma parameter to 2, and thus the gamma method yields results virtually identical to those obtained with Dayhoff&#x02019;s (<a href="#B24" class="cite-reflink bibr">24</a>) PAM (accepted point mutation) matrix, used by ref. <a href="#B17" class="cite-reflink bibr">17</a>. The Poisson model assumes equal rates of substitution across all sites within a given sequence, whereas the gamma model allows for violations of this assumption in the calculation of genetic distances (<a href="#B25" class="cite-reflink bibr">25</a>&#x02013;<a href="#B28" class="cite-reflink bibr">28</a>). Reference times from the fossil record for the divergence between vertebrate groups used in the rate calibrations are the same as in ref. <a href="#B17" class="cite-reflink bibr">17</a>.</p><p><span class="head3">Statistical Methods. </span> We use the <em>Z</em> statistic of the &#x0201c;two-cluster test&#x0201d; (<a href="#B29" class="cite-reflink bibr">29</a>) to ascertain whether the average amino acid substitution rates are statistically similar between the invertebrate and the vertebrate sets of sequences. In our test, the groups compared are not required to be monophyletic. To map divergence times onto our phylogenetic tree, we compute the height of the ancestral node as half the genetic distance between two reference sequence clusters, and determine the ratio of divergence time to node height. The substitution rate is the average of this ratio for all reference nodes. We multiply this average by the estimated heights of deeper nodes to obtain estimates of divergence times.<p>We apply the &#x0201c;branch-length&#x0201d; test of ref. <a href="#B29" class="cite-reflink bibr">29</a> to a phylogenetic tree of the gene sequences, reconstructed by using the neighbor joining method of ref. <a href="#B30" class="cite-reflink bibr">30</a> and the Poisson correction for multiple replacements (<a href="#B31" class="cite-reflink bibr">31</a>). This test calculates the genetic distance from root to tip for each lineage, and it determines for each taxon whether or not it has evolved at a rate significantly different from the average. A &#x003c7;<sup>2</sup> statistic is computed that evaluates the extent to which the entire tree (excluding in this case the outgroup nonmetazoan sequences) conforms to a molecular clock, and the aberrant sequences (<em>P</em> &#x0003c; 0.05) are removed from the tree. The test is then reapplied to the remaining sequences, and the process is repeated until the remaining &#x0201c;linearized tree&#x0201d; consists only of lineages that are all evolving at a uniform rate.</p></p><p><span class="head3">Estimation of Divergence Time in Linear Trees. </span> Calibration of substitution rates is performed by mapping divergence times from the fossil record onto specific nodes of the linear tree, allowing a direct extrapolation to deeper divergence times. Under the assumptions of the molecular clock model, the number of amino acid substitutions separating two protein sequences is on average proportional to the time elapsed since the divergence of these sequences. If at least one time estimate can be assigned to a node of a phylogenetic tree of a set of protein sequences, then time estimates can be obtained for the remaining nodes. By using the phylogenetic tree to reconstruct divergence times, the covariance between pairwise genetic distances can be directly measured as the variance of the estimated branch lengths shared by the pair of distances (<a href="#B23" class="cite-reflink bibr">23</a>).<p>Denote by <em>t</em><sub><em>u</em></sub> the value of an unknown divergence time and an estimate of this value by <em><img src="corehtml/pmc/pmcents/x0074x0302.gif" border="0" alt="t" /></em><sub><em>u</em></sub>. Given a reconstructed phylogenetic tree, which we assume is correct, binary, and obeys a molecular clock, an estimate of the unknown divergence time can be computed as <a id="E1" name="E1"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=18467&amp;blobtype=equ&amp;blobname=M1" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M1" title="equation M1"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">1</span></div></td></tr></table> where <em>&#x00125;</em><sub><em>u</em></sub> estimates the height of the interior node (measured in terms of the number of amino acid substitutions per site) that corresponds to time <em>t</em><sub><em>u</em></sub>; <em>&#x00125;</em><sub><em>ri</em></sub>, <em><img src="corehtml/pmc/pmcents/x0074x0302.gif" border="0" alt="t" /></em><sub><em>ri</em></sub>, and <em><img src="corehtml/pmc/pmcents/x0072x0302.gif" border="0" alt="[r with circumflex]" /></em><sub><em>ri</em></sub> denote the estimated height, divergence time and their ratio, respectively, for the <em>i</em>th reference node that can be assigned a known estimate of the absolute time.</p><p>With data sets for which more than one reference node is available, multiple <em><img src="corehtml/pmc/pmcents/x0072x0302.gif" border="0" alt="[r with circumflex]" /></em><sub><em>ri</em></sub> values are estimated; under the assumptions of the molecular-clock theory, all <em><img src="corehtml/pmc/pmcents/x0072x0302.gif" border="0" alt="[r with circumflex]" /></em><sub><em>ri</em></sub>s have the same expected value. Although the error associated with the estimate of the interior-node height, <em>&#x00125;</em><sub><em>ri</em></sub>, can be easily calculated, the corresponding error associated with absolute time estimate, <em><img src="corehtml/pmc/pmcents/x0074x0302.gif" border="0" alt="t" /></em><sub><em>ri</em></sub>, is not known. An unweighted average of the different estimates of <em>r</em> is calculated as <a id="E2" name="E2"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=18467&amp;blobtype=equ&amp;blobname=M2" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M2" title="equation M2"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">2</span></div></td></tr></table> where <em>n</em><sub><em>r</em></sub> is the total number of available reference nodes. By using the ordinary least-squares method, the node heights <em>h</em><sub><em>u</em></sub> and <em>h</em><sub><em>r</em></sub> can be estimated from the pairwise distances, <em>d</em><sub><em>ij</em></sub>s, between protein sequences as <a id="E3" name="E3"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=18467&amp;blobtype=equ&amp;blobname=M3" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M3" title="equation M3"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">3</span></div></td></tr></table> where A and B are two sequence clusters joined together by node <em>u</em>, and C and D are the two corresponding clusters for node <em>r</em>; &#x0007c;<em>A</em>&#x0007c; represents the number of sequences in cluster A. Using the delta-technique (<a href="#B32" class="cite-reflink bibr">32</a>), we can express the variance of the time estimate <em>t</em><sub><em>u</em></sub> as <a id="E4" name="E4"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=18467&amp;blobtype=equ&amp;blobname=M4" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M4" title="equation M4"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">4</span></div></td></tr></table> In recent studies involving phylogenetic estimation of the divergence time, the variance of the reference paleontological time estimate, Var(<em><img src="corehtml/pmc/pmcents/x0074x0302.gif" border="0" alt="t" /></em><sub><em>r</em></sub>) is implicitly assumed to be zero (e.g., refs. <a href="#B17" class="cite-reflink bibr">17</a> and <a href="#B33" class="cite-reflink bibr">33</a>). Although a rigorous estimate for Var(<em><img src="corehtml/pmc/pmcents/x0074x0302.gif" border="0" alt="t" /></em><sub><em>r</em></sub>) is not currently available, it is likely to be nonzero and may considerably inflate the resulting value of Var(<em><img src="corehtml/pmc/pmcents/x0074x0302.gif" border="0" alt="t" /></em><sub><em>u</em></sub>).</p><p>The values of Var(<em><img src="corehtml/pmc/pmcents/x0072x0302.gif" border="0" alt="[r with circumflex]" /></em><sub><em>r</em></sub>) and Cov(<em>&#x00125;</em><sub><em>u</em></sub>, <em><img src="corehtml/pmc/pmcents/x0072x0302.gif" border="0" alt="[r with circumflex]" /></em><sub><em>r</em></sub>) can be approximated with the delta-technique as <a id="E5" name="E5"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=18467&amp;blobtype=equ&amp;blobname=M5" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M5" title="equation M5"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">5</span></div></td></tr></table> and <a id="E6" name="E6"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=18467&amp;blobtype=equ&amp;blobname=M6" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M6" title="equation M6"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">6</span></div></td></tr></table> Estimates of the variances and covariances of <em>&#x00125;</em><sub><em>i</em></sub> and <em>&#x00125;</em><sub><em>j</em></sub> are calculated by the method of ref. <a href="#B23" class="cite-reflink bibr">23</a>, and the covariance between two evolutionary distances is calculated as the variance of the longest path shared by the distances in the true tree. With this approach and the first-order approximation Var(<em><img src="corehtml/pmc/pmcents/x0064x0302.gif" border="0" alt="d" /></em><sub><em>ij</em></sub>) &#x02248; <em>d</em><sub><em>ij</em></sub>/<em>s</em>, where <em>s</em> is the number of sequence sites used for the calculation of distances, <a id="E7" name="E7"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=18467&amp;blobtype=equ&amp;blobname=M7" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M7" title="equation M7"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">7</span></div></td></tr></table> where &#x003bb;<sub><em>ij,kl</em></sub> is the length of the longest route in the true tree that includes both the path from sequence <em>i</em> to sequence <em>j</em> and the path from sequence <em>k</em> to sequence <em>l</em>, and A<sub><em>x</em></sub> and B<sub><em>x</em></sub>, C<sub><em>y</em></sub> and D<sub><em>y</em></sub> are pairs of unique clusters corresponding to the interior nodes <em>x</em> and <em>y</em>, respectively.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id663308" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id406657" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>RESULTS</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666906" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id511005" class="sidefm-pmclink" style="text-transform: uppercase;">References</a></div></span></div></td><td class="content-cell"><span id="Results"></span><div class="head1 section-title" style="text-transform: uppercase;" id="id666699"><a id="Results"></a>RESULTS</div><div class="section-content"><span id="Results"></span><p>Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> gives the time estimates (in millions of years) for the divergence between the chordates and various invertebrate phyla. Chordates and echinoderms are deuterostomes, more closely related to each other than they are to the protostomes, represented in the table by three phyla: Arthropoda, Annelida, and Mollusca. The order of branching among these three phyla is uncertain, although annelids and mollusks appear to be closer to each other than either is to arthropods (<a href="#B3" class="cite-reflink bibr">3</a>, <a href="#B34" class="cite-reflink bibr">34</a>). For the present purposes, we shall assume that the three protostome phyla are equally divergent from the vertebrates.</p><p>The time estimates in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> are derived from genetic distances estimated with two different methods for correcting for multiple substitutions. The Poisson method assumes that the rate of substitution for a particular sequence is identical for all sites, whereas the gamma method does not make this assumption. The six genes (the &#x003b1;- and &#x003b2;-globin genes present in the vertebrates are represented by only one gene in the invertebrates) in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> are the same genes analyzed by Wray <em>et al</em>. (<a href="#B17" class="cite-reflink bibr">17</a>). The 18S rRNA gene analyzed by these authors is not included in our analysis because we could not obtain alignments that would be unambiguous and robust&#x02014;i.e., that could be extended from one to another set of taxa comparisons. The taxa represented in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> are a subset of the taxa analyzed by Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>). We eliminated by the branch-length method those lineages showing rates statistically different from the average for the particular gene locus. Wray <em>et al.</em> also eliminated &#x0201c;invertebrate sequences that showed consistently faster rates&#x0201d; (ref. <a href="#B17" class="cite-reflink bibr">17</a>, p. 571), although they do not say whether statistical significance was used for this elimination.</p><p>The average time estimates derived from the loci shown in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> are consistent with the common view that the animal phyla that appear in the Cambrian fossil record, but not earlier, diverged before the Cambrian (&#x02248;700&#x02013;540 My ago) but not much earlier as proposed by Wray <em>et al</em>. (<a href="#B17" class="cite-reflink bibr">17</a>). Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a> gives time estimates for the protostome&#x02013;deuterostome divergence obtained by analysis of 12 additional gene loci. The estimated time of divergence is again somewhat greater with the gamma than with the Poisson correction but consistent with the hypothesis that it occurred during the late Neoproterozoic. The combined results from all 18 genes are summarized in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>. Fig. <a href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1</a> shows the phylogeny of the phyla on geological and time scales, using the average of the Poisson and gamma estimates for the two divergence points (protostomes&#x02013;deuterostomes and echinoderms&#x02013;chordates) with shading indicating the range between the means obtained by the two methods. The time estimates displayed in Fig. <a href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1</a> are consistent with commonly accepted paleontological interpretations.    </p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T3" name="T3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T3" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 3" title="Table 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 3</strong></a><div class="figure-table-caption-in-article"><span>Estimated divergence times between echinoderms and chordates and between them and three deuterostome phyla (arthropods, annelids, and<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />mollusks)</span></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F1" name="F1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=18467&amp;blobname=pq0283883001.gif" border="1" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span>Estimated divergence times for selected animal phyla. Mean divergence times are the averages based on 18 gene loci: 673 million years for the protostome&#x02013;deuterostome divergence and 595 million years for the echinoderm&#x02013;chordate divergence.</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id663308" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id406657" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666699" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>DISCUSSION</a></div><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id511005" class="sidefm-pmclink" style="text-transform: uppercase;">References</a></div></span></div></td><td class="content-cell"><span id="Discussion"></span><div class="head1 section-title" style="text-transform: uppercase;" id="id666906"><a id="Discussion"></a>DISCUSSION</div><div class="section-content"><span id="Discussion"></span><p>The theory of the molecular clock has provided useful, sometimes definitive, information toward settling matters of phylogenetic topology and the time of remote evolutionary events. The theory takes into account that each particular gene or protein evolves at a distinct rate and thus may serve as an independent molecular clock. Yet, heterogeneity across taxa and/or through time is often the case, and particular molecular clocks may have very erratic behavior (e.g., see refs. <a href="#B18" class="cite-reflink bibr">18</a>&#x02013;<a href="#B21" class="cite-reflink bibr">21</a>, <a href="#B35" class="cite-reflink bibr">35</a>).</p><p>The time of origin of animal phyla remains unsettled. The abundant appearance of most readily fossilizable animal phyla in the Cambrian fossil record, but not earlier, is frequently taken as indication that most animal phyla evolved shortly before the Cambrian (<a href="#B1" class="cite-reflink bibr">1</a>&#x02013;<a href="#B7" class="cite-reflink bibr">7</a>, <a href="#B36" class="cite-reflink bibr">36</a>). Others, however, think it likely that animal phyla may have originated much earlier and that absence of their fossil remains before the Cambrian is due to one or more of the following conditions: smallness of the early metazoa, lack of hard body parts, and unsuitable geological circumstances for fossilization and preservation. Thus, it has been proposed that the protostomes and deuterostomes diverged around 1,200 My ago and the two deuterostome phyla, echinoderms and chordates, around 1,000 My ago (<a href="#B17" class="cite-reflink bibr">17</a>).</p><p>We have sought evidence on this matter by analyzing 18 gene loci coding for proteins that have been sequenced in numerous relevant taxa. Extrapolation from known divergence times determined by the fossil record depends on &#x0201c;linear&#x0201d; trees&#x02014;i.e., phylogenies in which the lineages are all evolving at the same rate, which rate is to be extrapolated to determine the unknown dates. We have, therefore, tested our trees for linearity and excluded all branches that could be shown to evolve at rates significantly different from the average for the tree.</p><p>The branch-length test (<a href="#B29" class="cite-reflink bibr">29</a>) provides a statistical method to determine whether a particular taxon has evolved at a significantly faster or slower rate than the average, as determined by a &#x003c7;<sup>2</sup> statistic that evaluates the extent to which the entire tree conforms to a molecular clock. Calibration of substitution rates is performed by mapping directly all available divergence times from the fossil record onto specific nodes of the linear tree.</p><p>The variances of the time estimates are often quite large (see Tables <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> and <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a>) because of the reduced number of taxa, but also because our calculations take into account the nonindependence of the phylogenetically correlated protein sequences, rather than treating distance estimates as independent observations, as done in ref. <a href="#B17" class="cite-reflink bibr">17</a>. The gamma method gives somewhat greater time estimates of divergence than the Poisson method, but the two sets of estimates are fairly similar, with the notable exception of the globins. Higher vertebrates possess &#x003b1;- and &#x003b2;-globin gene families, each with several members. Numerous duplications have occurred that become entangled through a &#x0201c;birth-and-death&#x0201d; process, by which a gene in one species but not in others is lost and replaced by a paralogous one within the same genome (<a href="#B37" class="cite-reflink bibr">37</a>&#x02013;<a href="#B39" class="cite-reflink bibr">39</a>). If paralogous and orthologous genes are intermingled, the mean and variance will increase because the divergence time of paralogous proteins corresponds to the time of gene duplication rather than to the time of speciation. It seems likely that the particular history of the globin genes may account for the large variances and discrepancies obtained for them. We could have eliminated the globins from our analysis, but we have left them, with the understanding that they may be inflating the overall average estimates of divergence time. Removing the globins virtually does not change the average estimates obtained with the Poisson method: 568 &#x000b1; 52 and 602 &#x000b1; 54 for the vertebrate&#x02013;echinoderm and protostome&#x02013;deuterostome divergences, respectively. But it reduces the averages obtained with the gamma method: 560 &#x000b1; 43 and 666 &#x000b1; 64 (628 &#x000b1; 76 and 736 &#x000b1; 65 in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>) and brings them closer to the Poisson estimates.</p><p>The results summarized in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a> and Fig. <a href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1</a> are consistent with the Cambrian explosion theory proposing that the animal phyla originated not much before the Cambrian, during the late Neoproterozoic, some 544&#x02013;700 My ago. This conclusion contrasts with the estimates obtained by Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>), who propose that the protostome&#x02013;deuterostome divergence is twice or more as old as the Cambrian, having occurred about 1,200 My ago, and that the echinoderm&#x02013;chordate divergence is about 1,000 My old. They investigated the same set of six protein-encoding genes shown in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> plus the 18S rRNA gene. The taxa in their study include the taxa used in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> as well as taxa that we excluded because of lineages evolving significantly faster or slower than the average. We obtained the gene sequences from their web site. On the whole, they include (see Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T4" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">4</a>) about twice as many taxa as used in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a>.  </p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T4" name="T4"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T4" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 4" title="Table 4"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 4</strong></a><div class="figure-table-caption-in-article"><span>Tests for homogeneity of sequence divergence rates in the data of Wray <em>et al.</em><img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />(<a href="#B17" class="cite-reflink bibr">17</a>)</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>) argue that their results are robust because (<em>i</em>) genetic distances and divergence times are highly correlated (p. 570 and their table 1) and (<em>ii</em>) the relative rate test indicates low rate variation, with standard errors ranging from 0.5&#x00025; to 2.0&#x00025; of the mean (p. 571 and their table 3). However, the relative rate test has low statistical power, and it has long ago been shown that apparently insignificant variation in average rates may hide differences in rate of evolution along the branches of the star phylogeny by 200&#x00025; and more (e.g., ref. <a href="#B1" class="cite-reflink bibr">1</a>, see figure 9-23). Statistically more powerful tests have been developed, such as the branch-length test. High correlation and significant regression between distances and times are not, either, convincing evidence of uniform rates. To put it plainly, a time-dependent process such as molecular evolution will provide positive correlation with, and significant regression on, time without implication that the rate of change is constant. If we record the average time taken by travelers between Los Angeles and each of San Diego, San Francisco, and New York, we would likely find strong correlation between distance and time, even though different travelers may be going by car, rail, or plane. It would be folly to extrapolate the average time&#x02013;distance rate observed between Los Angeles and these three cities and use it for estimating the distance between Los Angeles and London. The following sources of evidence further manifest that the results of Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>) are not robust.</p><p>First, their regression methods introduce a host of statistical problems. Genetic distances between taxa are very highly correlated, yet all possible pairwise combinations of genetic distance measurements are computed and treated as independent observations in the regression calculations. Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>) attempt to account for this nonindependence by assuming that the total number of degrees of freedom in the data is equal to the number of nodes in the corresponding binary tree. Unfortunately, while their approach, including the bootstrap and Mantel test, can detect the stochastic component of the total error, it is powerless to correct for the systematic bias in the computation of the mean slope caused by the dominance of distances that involve large clusters of nearly identical sequences. As a result, differences in substitution rates between vertebrates and invertebrates are amplified by the nonindependence of distance measurements and by the disproportionate representation of the vertebrate taxa. A statistically rigorous method of extrapolating distance times and estimating variances must take into account the phylogenetic relationships underlying the molecular sequences.</p><p>Second, we have applied the &#x0201c;two-cluster test&#x0201d; (<a href="#B29" class="cite-reflink bibr">29</a>) to test for each locus in the data set of ref. <a href="#B17" class="cite-reflink bibr">17</a> whether the average substitution rates differ between the vertebrate and invertebrate groups. The <em>Z</em> statistic indicates that the vertebrate rate of evolution is significantly slower than the invertebrate rate at two loci (Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T4" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">4</a>). A slower rate in the vertebrate sequences (indicated by the negative sign of <em>Z</em>; see below about hemoglobin), which were used for calibrating divergence times, introduces a systematic bias in the extrapolation, exaggerating the time divergence estimates between the vertebrate and invertebrate phyla.</p><p>Third, we have applied the &#x0201c;branch-length test&#x0201d; (<a href="#B29" class="cite-reflink bibr">29</a>) to the phylogenetic tree reconstructed for each locus with the neighbor-joining method (<a href="#B30" class="cite-reflink bibr">30</a>) using the Poisson correction for multiple replacements (<a href="#B23" class="cite-reflink bibr">23</a>). This test calculates the genetic distance from root to tip for each lineage and determines for each taxon whether it has evolved at a rate significantly different from the average. A &#x003c7;<sup>2</sup> statistic evaluates the extent to which the entire tree (excluding the outgroup nonmetazoan sequences) conforms to a molecular clock. Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T4" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">4</a> shows a significant departure from a molecular clock at <em>P</em> &#x0003c; 0.001 for every locus in the data set of ref. <a href="#B17" class="cite-reflink bibr">17</a>.</p><p>Fourth, we notice in figure 1 of Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>) that the data points at each of four loci consist of two discrete sets, approximately 0&#x02013;150 My and 300&#x02013;450 My. We have calculated the rate of evolution by the regression of genetic distance on known divergence time separately for the two sets of data points (without imposing the restriction that the regression line pass through the origin, similarly as done by Wray <em>et al.</em> in ref. <a href="#B17" class="cite-reflink bibr">17</a>). If the rate of evolution is approximately constant through time, one would expect that the rates obtained for the two sets of data points should be fairly similar. The two rates are quite disparate and significantly so (Fig. <a href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F2" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">2</a>).  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F2" name="F2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=18467&amp;blobname=pq0283883002.gif" border="1" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span>Rates of molecular evolution (genetic distance versus time) obtained with the data points of Wray <em>et al</em>. (<a href="#B17" class="cite-reflink bibr">17</a>), but separately for those 0&#x02013;150 and 300&#x02013;450 My old. The regression slopes for each of the four loci follow (given in parentheses</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>Fifth, we notice that only vertebrate data are used to calibrate the protein-coding genes, but echinoderms and mollusks are also used for calibrating 18S rRNA (ref. <a href="#B17" class="cite-reflink bibr">17</a>, legend for table 1). We ask whether the 18S rRNA rate would remain the same if only vertebrate data are used, as for the other genes. The regression slope obtained for the vertebrate data alone is 0.77 &#x000d7; 10<sup>&#x02212;4</sup>, but it is twice as large, 1.5 &#x000d7; 10<sup>&#x02212;4</sup> when mollusks and echinoderms are added (ref. <a href="#B17" class="cite-reflink bibr">17</a>, table 1). If only vertebrates had been used for calibrating the clock, as done by Wray <em>et al.</em> for the other genes, the protostome&#x02013;deuterostome divergence times would be 2,600&#x02013;3,200 My, surely much too ancient. In any case, we see that the vertebrates are evolving slower than the invertebrates for the 18S rRNA gene, as noted above for protein-coding genes.</p><p>Sixth, we notice in table 3 of Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>) that the mean genetic distances between animals and other multicellular kingdoms do not seem conspicuously different from the distances between bacteria and animals. We show in Table <a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T5" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">5</a> the mean distances obtained by averaging over loci the distances provided by Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>). The average distance between the eubacteria and animals is 1.51 &#x000b1; 0.50, not significantly different from the distance between plants and animals of 1.40 &#x000b1; 0.62, or between fungi (protist for hemoglobin) and animals, 1.68 &#x000b1; 0.81. We have used these data to estimate <em>a</em> and <em>b</em> in Fig. <a href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F3" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">3</a>. Using the mean genetic distances that Wray <em>et al.</em> give in their table 3, we conclude that <em>b</em> &#x02248; 0, even though the time span encompasses the evolution from the prokaryote to the eukaryote cell, the proliferation of the protist phyla, and the origin of multicellularity. If we exclude the hemoglobin gene, which shows the most heterogeneous rates in Wray <em>et al.</em> (ref. <a href="#B17" class="cite-reflink bibr">17</a>, table 3), the mean genetic distances from metazoa become 0.792 &#x000b1; 0.217 for plant, 0.892 &#x000b1; 0.235 for fungus/yeast, and 1.044 &#x000b1; 0.211 for the bacterium; and the <em>b</em>/<em>a</em> ratio becomes 0.24. If we accept Wray <em>et al.</em>&#x02019;s value of <em>a</em> = 1,200 My, <em>b</em> would be 288 My rather than a typical estimate of &#x02248;2,000 My.    </p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T5" name="T5"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T5" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 5" title="Table 5"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=18467&amp;rendertype=table&amp;id=T5" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 5</strong></a><div class="figure-table-caption-in-article"><span>Mean genetic distances between metazoa and nonmetazoa calculated as the averages between the values given by Wray <em>et al.</em> (ref. <a href="#B17" class="cite-reflink bibr">17</a>, table 3) for six gene loci, and time parameters derived from the<img src="corehtml/pmc/pmcents/x2009.gif" border="0" alt=" " />distances</span></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F3" name="F3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=18467&amp;blobname=pq0283883003.gif" border="1" class="icon-reflink" alt="Figure 3" title="Figure 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 3</strong></a><div class="figure-table-caption-in-article"><span>Phylogeny of eubacteria and multicellular kingdoms, with branch lengths arbitrary. The averages for the genetic distances given in table 3 of ref. <a href="#B17" class="cite-reflink bibr">17</a> are 1.50 &#x000b1; 0.50 between bacteria and animals, which is not greater than 1.40 &#x000b1; 0.62 between</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=18467&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>It seems warranted to conclude that the estimates of invertebrate&#x02013;vertebrate divergence times obtained by Wray <em>et al.</em> (<a href="#B17" class="cite-reflink bibr">17</a>) are invalid, owing to methodological problems and violations of the molecular clock. Extrapolations to distant times from molecular evolutionary rates estimated within confined data-sets are fraught with danger (<a href="#B18" class="cite-reflink bibr">18</a>&#x02013;<a href="#B21" class="cite-reflink bibr">21</a>). Nevertheless, our time estimates, obtained by systematic elimination of erratic rates, are consistent with the common interpretation that protostomes and deuterostomes originated in the late Neoproterozoic, during the 160 My preceding the Cambrian.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id510987">Acknowledgments</div><div class="section-content"><p><p>We thank W. M. Fitch, R. R. Hudson, R. K. Selander, and J. W. Valentine for discussions. This research was supported by National Institutes of Health Grant GM42397.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id510992">ABBREVIATION</div><div class="section-content"><p><div class="def-list"><table border="0" cellspacing="4"><tr><td>My</td><td>million years</td></tr></table></div><br style="clear:both;"></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: uppercase;">Top</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id663308" class="sidefm-pmclink" style="text-transform: uppercase;">Abstract</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id406657" class="sidefm-pmclink" style="text-transform: uppercase;">MATERIALS AND METHODS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666699" class="sidefm-pmclink" style="text-transform: uppercase;">RESULTS</a></div></span><span style="text-transform: uppercase;"><div class="sidefm-pmclink-item"><a href="#id666906" class="sidefm-pmclink" style="text-transform: uppercase;">DISCUSSION</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: uppercase;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: uppercase;" id="id511005">References</div><div class="section-content"><div class="back-matter-section"><div class="ref-cit-blk" id="B1"><div class="ref-label">1.</div><div class="ref-cit">Dobzhansky, T; Ayala, F J; Stebbins, G L; Valentine, J W. <span class="ref-journal">Evolution.</span> San Francisco: Freeman; 1977. </div></div><div class="ref-cit-blk" id="B2"><div class="ref-label">2.</div><div class="ref-cit">Valentine, J W. <span><span class="ref-journal">Syst Zool. </span>1973;<span class="ref-vol">22</span>:97–102.</span></div></div><div class="ref-cit-blk" id="B3"><div class="ref-label">3.</div><div class="ref-cit">Valentine, J W. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1989;<span class="ref-vol">86</span>:2272–2275.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0DwgXBO1k1C65m02u_4ZTxAzkOoK_cBo208hMMAMS&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16594022">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B4"><div class="ref-label">4.</div><div class="ref-cit">Valentine, J W; Awramik, S M; Signor, P W; Sadler, P M. <span><span class="ref-journal">Evol Biol. </span>1991;<span class="ref-vol">25</span>:279–356.</span></div></div><div class="ref-cit-blk" id="B5"><div class="ref-label">5.</div><div class="ref-cit">Lipps J H, Signor P W. ,  editors. <span class="ref-journal">Origin and Early Evolution of Metazoa.</span> New York: Plenum; 1992. </div></div><div class="ref-cit-blk" id="B6"><div class="ref-label">6.</div><div class="ref-cit">Gould, S J. <span class="ref-journal">Wonderful Life.</span> New York: Norton; 1989. </div></div><div class="ref-cit-blk" id="B7"><div class="ref-label">7.</div><div class="ref-cit">Schopf J W, Klein C. ,  editors. <span class="ref-journal">The Proterozoic Biosphere. A Multidisciplinary Study.</span> Cambridge, U.K.: Cambridge Univ. Press; 1992. </div></div><div class="ref-cit-blk" id="B8"><div class="ref-label">8.</div><div class="ref-cit">Grotzinger, J P; Bowring, S A; Saylor, B Z; Kaufman, A J. <span><span class="ref-journal">Science. </span>1995;<span class="ref-vol">270</span>:598–604.</span></div></div><div class="ref-cit-blk" id="B9"><div class="ref-label">9.</div><div class="ref-cit">Boaden, P J S. <span><span class="ref-journal">Zool J Linn Soc. </span>1989;<span class="ref-vol">96</span>:217–227.</span></div></div><div class="ref-cit-blk" id="B10"><div class="ref-label">10.</div><div class="ref-cit">Conway Morris, S. <span><span class="ref-journal">Nature (London). </span>1993;<span class="ref-vol">361</span>:219–225.</span></div></div><div class="ref-cit-blk" id="B11"><div class="ref-label">11.</div><div class="ref-cit">Glaessner, M F. <span class="ref-journal">The Dawn of Animal Life.</span> Cambridge, U.K.: Cambridge Univ. Press; 1984. </div></div><div class="ref-cit-blk" id="B12"><div class="ref-label">12.</div><div class="ref-cit">Brown, R H; Richardson, M; Boulter, D; Ramshaw, J A M; Jeffries, R P S. <span><span class="ref-journal">Biochem J. </span>1972;<span class="ref-vol">128</span>:971–974.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0TI96z4n_E80UtJPHZ3uHX6pDdRrR_cM95-q9RQ5i&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/4344699">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B13"><div class="ref-label">13.</div><div class="ref-cit">Phillipe, H; Chenuil, A; Adoutte, A. <span><span class="ref-journal">Development (Cambridge, UK) Suppl. </span>1994;<span class="ref-vol">1994</span>:15–25.</span></div></div><div class="ref-cit-blk" id="B14"><div class="ref-label">14.</div><div class="ref-cit">Runnegar, B. <span><span class="ref-journal">Lethaia. </span>1982;<span class="ref-vol">15</span>:199–205.</span></div></div><div class="ref-cit-blk" id="B15"><div class="ref-label">15.</div><div class="ref-cit">Runnegar, B. <span><span class="ref-journal">Paleontology. </span>1986;<span class="ref-vol">29</span>:1–24.</span></div></div><div class="ref-cit-blk" id="B16"><div class="ref-label">16.</div><div class="ref-cit">Erwin, D H. <span><span class="ref-journal">Lethaia. </span>1989;<span class="ref-vol">22</span>:251–257.</span></div></div><div class="ref-cit-blk" id="B17"><div class="ref-label">17.</div><div class="ref-cit">Wray, G A; Levinton, J S; Shapiro, L H. <span><span class="ref-journal">Science. </span>1996;<span class="ref-vol">274</span>:568–573.</span></div></div><div class="ref-cit-blk" id="B18"><div class="ref-label">18.</div><div class="ref-cit">Ayala, F J. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1997;<span class="ref-vol">94</span>:7776–7783.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0TWiH6btcvQHoYnxxeZVcBcQDe6tsDMbJJ2m1m5h6&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9223263">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B19"><div class="ref-label">19.</div><div class="ref-cit">Ayala, F J; Barrio, E; Kwiatowski, J. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1996;<span class="ref-vol">93</span>:11729–11734.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Pyq1emCOzPTBnVUR1hFcfZ3477vux-WUkqhqukux&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8876205">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B20"><div class="ref-label">20.</div><div class="ref-cit">Ayala, F J. <span><span class="ref-journal">J Heredity. </span>1986;<span class="ref-vol">77</span>:226–235.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0PM-NNSkP7eECz3PRPrc66tgJJtrSTWzAWDfyfQpH&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/3020121">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B21"><div class="ref-label">21.</div><div class="ref-cit">Gillespie, J H. <span class="ref-journal">The Causes of Molecular Evolution.</span> New York: Oxford Univ. Press; 1991. </div></div><div class="ref-cit-blk" id="B22"><div class="ref-label">22.</div><div class="ref-cit">Thompson, J D; Higgins, D G; Gibson, T J. <span><span class="ref-journal">Nucleic Acids Res. </span>1994;<span class="ref-vol">22</span>:4673–4680.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Z3t_kUnSWUggkThJx9Pa6N-CEf7xiKOk0IcQVkx4&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7984417">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B23"><div class="ref-label">23.</div><div class="ref-cit">Nei, M; Stephens, J C; Saitou, N. <span><span class="ref-journal">Mol Biol Evol. </span>1985;<span class="ref-vol">2</span>:66–85.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0ZYE62L8R5PS1utXli45ts3Yj3pFgd94uQbrE0kPH&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/2897060">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B24"><div class="ref-label">24.</div><div class="ref-cit">Dayhoff, M O. <span class="ref-journal">Atlas of Protein Sequence and Structure.</span> Vol. 5. Washington, DC: National Biomedical Research Foundation; 1978.  , Suppl. 3.</div></div><div class="ref-cit-blk" id="B25"><div class="ref-label">25.</div><div class="ref-cit">Golding, G B. <span><span class="ref-journal">Mol Biol Evol. </span>1983;<span class="ref-vol">1</span>:125–142.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0borl--lUDoP0St1MSDMB-mPTiPe_SJel2iczw5LY&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/6599960">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B26"><div class="ref-label">26.</div><div class="ref-cit">Jin, L; Nei, M. <span><span class="ref-journal">Mol Biol Evol. </span>1990;<span class="ref-vol">10</span>:1396–1402.</span></div></div><div class="ref-cit-blk" id="B27"><div class="ref-label">27.</div><div class="ref-cit">Yang, Z. <span><span class="ref-journal">Mol Biol Evol. </span>1993;<span class="ref-vol">10</span>:1396–1402.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=04A9ke6wdY5C5A18iaicuhv0G6F4JvFuiZhJoQA4A&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8277861">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B28"><div class="ref-label">28.</div><div class="ref-cit">Takahata, N. <span><span class="ref-journal">Proc R Soc London Ser B. </span>1991;<span class="ref-vol">243</span>:13–18.</span></div></div><div class="ref-cit-blk" id="B29"><div class="ref-label">29.</div><div class="ref-cit">Takezaki, N; Rzhetsky, A; Nei, M. <span><span class="ref-journal">Mol Biol Evol. </span>1995;<span class="ref-vol">12</span>:823–833.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FMVJ51FeI0PwLUxuXUHJMYWcMDQbkFPRtg9RP5mN&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7476128">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B30"><div class="ref-label">30.</div><div class="ref-cit">Saitou, N; Nei, M. <span><span class="ref-journal">Mol Biol Evol. </span>1987;<span class="ref-vol">4</span>:406–425.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FA9A3UwS1L2lKOnopqO9EQGbIKyIjVGTdl6wQ5bx&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/3447015">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B31"><div class="ref-label">31.</div><div class="ref-cit">Zuckerkandl, E; Pauling, L. <span class="ref-journal">Evolving Genes and Proteins.</span> Bryson V, Vogel H J. ,  editors. New York: Academic; 1965. pp. 97–166.</div></div><div class="ref-cit-blk" id="B32"><div class="ref-label">32.</div><div class="ref-cit">Kendall, M B. <span class="ref-journal">The Advanced Theory of Statistics.</span> New York: Hafner; 1956. </div></div><div class="ref-cit-blk" id="B33"><div class="ref-label">33.</div><div class="ref-cit">Hedges, S B; Parker, P H; Sibley, C G; Kumar, S. <span><span class="ref-journal">Nature (London). </span>1996;<span class="ref-vol">381</span>:226–229.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0HYbLKLMr0HwkWiWAcwaGBoFZ-vZoxiMcIxBgYQem&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8622763">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B34"><div class="ref-label">34.</div><div class="ref-cit">Eernisse, F J; Albert, J S; Anderson, F E. <span><span class="ref-journal">Syst Biol. </span>1992;<span class="ref-vol">41</span>:305–330.</span></div></div><div class="ref-cit-blk" id="B35"><div class="ref-label">35.</div><div class="ref-cit">Li, W-H. <span class="ref-journal">Molecular Evolution.</span> Sunderland, MA: Sinauer; 1997. </div></div><div class="ref-cit-blk" id="B36"><div class="ref-label">36.</div><div class="ref-cit">Valentine, J W. <span class="ref-journal">Patterns of Evolution.</span> Hallam A. ,  editor. Amsterdam: Elsevier; 1977. pp. 27–58.</div></div><div class="ref-cit-blk" id="B37"><div class="ref-label">37.</div><div class="ref-cit">Ohita, T; Nei, M. <span><span class="ref-journal">Mol Biol Evol. </span>1994;<span class="ref-vol">11</span>:469–482.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=08U_ZlfAy4xxD7X_pWX9ZaKFoGX5fUSxXxz_dokrR&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8015440">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B38"><div class="ref-label">38.</div><div class="ref-cit">Koonin, E V; Mushegian, A R. <span><span class="ref-journal">Curr Opin Genet Dev. </span>1996;<span class="ref-vol">6</span>:757–762.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hCg063x-xD7E5AS2QsNlDXtYAVYIwZTQLTQDJkgK&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8994848">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B39"><div class="ref-label">39.</div><div class="ref-cit">Nei, M; Gu, X; Sitnikova, T. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1997;<span class="ref-vol">94</span>:7799–7806.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0x_qOiZhyGtY7X4Vq7LR-bCH6fRXjhD9YAYV74kkv&amp;reftype=pubmed&amp;artid=18467&amp;article-id=18467&amp;iid=522&amp;issue-id=522&amp;jid=2&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9223266">PubMed</a>]</span></div></div><div class="ref-cit-blk" id="B40"><div class="ref-label">40.</div><div class="ref-cit">Sokal, R R; Rohlf, F J. <span class="ref-journal">Biometry.</span> 2nd Ed. New York: Freeman; 1981. </div></div></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <br><b>National Academy of Sciences</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc2-->

