<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="BMC Evolutionary Biology"><meta name="citation_title" content="Paths of lateral gene transfer of lysyl-aminoacyl-tRNA synthetases with a unique evolutionary transition stage of prokaryotes coding for class I and II varieties by the same organisms"><meta name="citation_authors" content="Shaul Shaul, Ruth Nussinov, and  Tal Pupko"><meta name="citation_date" content="2006"><meta name="citation_volume" content="6"><meta name="citation_firstpage" content="22"><meta name="citation_doi" content="10.1186/1471-2148-6-22"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1475646&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1475646&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1475646"><meta name="citation_pmid" content="16529662"><meta name="DC.Title" content="Paths of lateral gene transfer of lysyl-aminoacyl-tRNA synthetases with a unique evolutionary transition stage of prokaryotes coding for class I and II varieties by the same organisms"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="BioMed Central"><meta name="DC.Contributor" content="Shaul Shaul"><meta name="DC.Contributor" content="Ruth Nussinov"><meta name="DC.Contributor" content="Tal Pupko"><meta name="DC.Date" content="2006"><meta name="DC.Identifier" content="10.1186/1471-2148-6-22"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody4.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs3.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Paths of lateral gene transfer of lysyl-aminoacyl-tRNA synthetases with a unique evolutionary transition stage of prokaryotes coding for class I and II varieties by the same organisms</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0DwqXBMfMO2z2uvw_Tut3mLwClEB8ra23UwAL8AbB&amp;reftype=other&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-bmceb.gif" alt="Logo of bmceb" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="BioMed Central web site" coords="0,0,99,74" onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0vMv2--cgodRhU21-MvgDGittR7posXSCjCJJoQxK&amp;reftype=publisher&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com" target="pmc_ext"><area shape="rect" coords="100,54,172,74" alt="Reference to the article." onclick="focuswin('pmc_ext') " href="redirect3.cgi?&amp;&amp;auth=0zAJ6PmQYqy4qBjq-PRweaM8V0j0iLT_PlHrmo5u5&amp;reftype=publisher&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7CArticle%7CFullText&amp;rendering-type=normal&amp;&amp;http://dx.doi.org/10.1186/1471-2148-6-22" target="pmc_ext"><area shape="rect" coords="173,54,223,74" alt="Search." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0Fi5faXYZwvfpxGYeaRj_W2v-sVC9LWiK431EJQJw&amp;reftype=publisher&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/search/bmcevolbiol" target="pmc_ext"><area shape="rect" coords="224,55,342,74" alt="Manuscript submission." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0fELjvElrUgIORdbr8yBC-ZDkModY7dSzGxvfUAvD&amp;reftype=publisher&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/manuscript/bmcevolbiol" target="pmc_ext"><area shape="rect" coords="343,53,480,74" alt="Registration." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0TIjKVtJ9SWnHQZ9RPjlO8ajyAzM52H5Pnkf_jksb&amp;reftype=publisher&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/registration" target="pmc_ext"><area shape="rect" coords="0,0,480,74" alt="Journal front page." onclick="focuswin('pmc_ext') " href="redirect3.cgi?&amp;&amp;auth=0dIhyhr8S0pBDrtrZs8seG9T7NgpgT27yr14iZkrz&amp;reftype=publisher&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/bmcevolbiol" target="pmc_ext"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=28&amp;action=archive">BMC Evol Biol</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=126576">v.6; 2006</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=1475646&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="#supplementary-material-sec" class="sidefm-supplink">Supplementary Material</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=1475646&amp;blobtype=pdf" class="sidefm-pmclink">PDF (554K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=126576" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=28&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0JKL54DKAfdFzN-GesHSPa3xybMZc8qaZkxsRzAXe&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16529662">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=0daP6AfM_xR99reyQ2ApDLRj8XuCqDfIBr717JAxL&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=16529662">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0fSHY73D4H7xIFzoqP3h3OAFjDvMyK7ldeb8xiQs4&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=16529662">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0nirhTwzlnO_mAAb45u7k5xSBeO4vQz62_od6J5_c&amp;reftype=Protein&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Protein&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1475646&amp;dopt=pmc_protein">Protein</option><option value="redirect3.cgi?&amp;&amp;auth=0rkmGbFsQc4aTotEO6CnychRmpxhybhJneimy-QDm&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1475646&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0VsBPGNQO8Nx9nc2q-g-_eqbOcBqB2FnhITMYCQWg&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=1475646">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0lAqzp9hmMDH8fas1MN-hL2tCXSegs2A7y71FAAYa&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Shaul+S%5Bauth%5D" class="sidefm-pmclink"> Shaul, S.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0Nu3L9kkbzdXsWYRCUjYEO9qcNbWkLczokdxJkAWz&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Nussinov+R%5Bauth%5D" class="sidefm-pmclink"> Nussinov, R.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0RO-TeE78_ZmMjanwKZw3fxGK_i1nb3Eq8-4grAdz&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Pupko+T%5Bauth%5D" class="sidefm-pmclink"> Pupko, T.</a></div></div></div></td><td rowspan="107" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-bmceb.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">BMC Evol Biol. </span><span class="citation-publication-date">2006; </span><span class="citation-volume">6</span><span class="citation-issue"></span><span class="citation-flpages">: 22. </span></div><div><span class="fm-vol-iss-date">Published online 2006 March 12. </span><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1186/1471-2148-6-22.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC1475646</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 2006 Shaul et al; licensee BioMed Central Ltd.</div><div class="fm-title">Paths of lateral gene transfer of lysyl-aminoacyl-tRNA synthetases with a unique evolutionary transition stage of prokaryotes coding for class I and II varieties by the same organisms</div><div class="fm-author contrib-group">Shaul Shaul,<sup>1</sup> Ruth Nussinov,<sup>2,</sup><sup>3</sup> and  Tal Pupko<sup><img src="corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>4</sup><div class="fm-affl"><sup>1</sup>Department of Zoology, George S. Wise Faculty of Life Sciences, Tel Aviv University, Ramat Aviv 69978, Israel</div><div class="fm-affl"><sup>2</sup>Basic Research Program, SAIC-Frederick, Inc. Center for Cancer Research, Nanobiology Program, NCI-Frederick Frederick, MD 21702, USA</div><div class="fm-affl"><sup>3</sup>Sackler Inst. of Molecular Medicine, Department of Human Genetics and Molecular Medicine, Sackler School of Medicine, Tel Aviv University, Tel Aviv 69978, Israel</div><div class="fm-affl"><sup>4</sup>Department of Cell Research and Immunology, George S. Wise Faculty of Life Sciences, Tel Aviv University, Tel Aviv 69978, Israel</div></div><div class="fm-footnote"><sup><img src="corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div class="fm-footnote"><span style="white-space: nowrap;" id="id458494">Shaul Shaul: <span class="e_id458494">sshaul/at/netvision.net.il</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id458494", '<a class="ext-reflink" href="' + reverseAndReplaceString('li.ten.noisivten/ta/luahss:otliam', '/at/', '@') + '">' + reverseAndReplaceString('li.ten.noisivten/ta/luahss', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id545614">Ruth Nussinov: <span class="e_id545614">ruthn/at/ncifcrf.gov</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id545614", '<a class="ext-reflink" href="' + reverseAndReplaceString('vog.frcficn/ta/nhtur:otliam', '/at/', '@') + '">' + reverseAndReplaceString('vog.frcficn/ta/nhtur', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id628269">Tal Pupko: <span class="e_id628269">talp/at/post.tau.ac.il</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id628269", '<a class="ext-reflink" href="' + reverseAndReplaceString('li.ca.uat.tsop/ta/plat:otliam', '/at/', '@') + '">' + reverseAndReplaceString('li.ca.uat.tsop/ta/plat', '/at/','@') + '</a>')}catch(e){}
                                //--></script></span> </div><div class="fm-pubdate">Received October 17, 2005; Accepted March 12, 2006.</div><div class="fm-copyright"><p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0zs8SGpnOIQqmcLFA5pZ7BGhJBj7NahVgUP1Mx5CE&amp;reftype=extlink&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://creativecommons.org/licenses/by/2.0" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://creativecommons.org/licenses/by/2.0</a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561380" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id661591" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id662487" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663136" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663146" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663571" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663579" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663727" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id561328">Abstract</div><div class="section-content"><!--article-meta--><p><div class="abs-head2 head-separate">Background</div>While the premise that lateral gene transfer (LGT) is a dominant evolutionary force is still in considerable dispute, the case for widespread LGT in the family of aminoacyl-tRNA synthetases (aaRS) is no longer contentious. aaRSs are ancient enzymes, guarding the fidelity of the genetic code. They are clustered in two structurally unrelated classes. Only lysine aminoacyl-tRNA synthetase (LysRS) is found both as a class 1 and a class 2 enzyme (LysRS1-2). Remarkably, in several extant prokaryotes both classes of the enzyme coexist, a unique phenomenon that has yet to receive its due attention.</p><p><div class="abs-head2 head-separate">Results</div>We applied a phylogenetic approach for determining the extent and origin of LGT in prokaryotic LysRS. Reconstructing species trees for Archaea and Bacteria, and inferring that their last common ancestors encoded LysRS1 and LysRS2, respectively, we studied the gains and losses of both classes. A complex pattern of LGT events emerged. In specific groups of organisms LysRS1 was replaced by LysRS2 (and vice versa). In one occasion, within the alpha proteobacteria, a LysRS2 to LysRS1 LGT was followed by reversal to LysRS2. After establishing the most likely LGT paths, we studied the possible origins of the laterally transferred genes. To this end, we reconstructed LysRS gene trees and evaluated the likely origins of the laterally transferred genes. While the sources of LysRS1 LGTs were readily identified, those for LysRS2 remain, for now, uncertain. The replacement of one LysRS by another apparently transits through a stage simultaneously coding for both synthetases, probably conferring a selective advantage to the affected organisms.</p><p><div class="abs-head2 head-separate">Conclusion</div>The family of LysRSs features complex LGT events. The currently available data were sufficient for identifying unambiguously the origins of LysRS1 but not of LysRS2 gene transfers. A selective advantage is suggested to organisms encoding simultaneously LysRS1-2.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561328" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Background</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id661591" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id662487" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663136" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663146" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663571" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663579" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663727" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id561380">Background</div><div class="section-content"><p>In protein synthesis the rules of the genetic code are established through catalytic aminoacylation of tRNAs by their cognate synthetases. With some notable exceptions, each aaRS enzyme acylates a specific amino acid to its cognate tRNA. Throughout the three Domains of life synthetases are partitioned into two structurally and evolutionary unrelated classes (class 1 and class 2 aaRS) [<a href="#B1" class="cite-reflink">1</a>]. These differ in their secondary structure arrangements, in their conserved sequence motifs composing the active site, and in the side of the tRNA acceptor stem to which they dock [<a href="#B2" class="cite-reflink">2</a>]. LysRS is the only known exception to this classification, aminoacylating tRNA(Lys) by two rather than by one enzyme: LysRS1 featuring the distinct structure and characteristics of a class 1 aaRS and LysRS2 with the distinct structure and characteristics of a class 2 aaRS [<a href="#B3" class="cite-reflink">3</a>]. Structural studies of LysRS1 and LysRS2 complexed with lysine indicated that in addition to the canonical aaRS class distinctions, the amino acid binding site of LysRS1 is more compact than that of LysRS2 [<a href="#B4" class="cite-reflink">4</a>,<a href="#B5" class="cite-reflink">5</a>].</p><p>All known Eukaryotae apparently code only for LysRS2. Most Bacteria code for LysRS2, but some taxa, predominantly within the class of alpha-proteobacteria, code for LysRS1. Archaea mostly code for LysRS1, with some exceptions coding for LysRS2. R.F. Doolittle and J. Handy [<a href="#B6" class="cite-reflink">6</a>] predicted that prokaryotes will be found coding for both types of LysRS by the same organism. For a while this prediction was not accepted [<a href="#B7" class="cite-reflink">7</a>,<a href="#B8" class="cite-reflink">8</a>]. Recently, it was confirmed: the genomes of several mesophilic prokaryotes were shown to encode both LysRS1 (<em>lysK</em>) and LysRS2 <em>(lysS</em>). Already five such organisms have been identified: <em>Methanosarcina mazei </em>[<a href="#B9" class="cite-reflink">9</a>], <em>Methanosarcina acetivorans </em>[<a href="#B10" class="cite-reflink">10</a>], <em>Methanosarcina barkeri </em>[<a href="#B11" class="cite-reflink">11</a>], <em>Bacillus cereus </em>[<a href="#B12" class="cite-reflink">12</a>] and <em>Treponema palladium </em>[<a href="#B13" class="cite-reflink">13</a>].</p><p>With continuing increase of complete genome sequencing, there is no compelling reason to doubt that the number of prokaryotae discovered to code for both classes of LysRS will rise. The growing database of LysRS1 and LysRS2 synthesized by various archaeal and bacterial phyla and sometimes by the same organism motivated us to address two issues: (1) what are the incidences, patterns and sources of LysRS1 and LysRS2 LGTs between prokaryotes found in current databases of completely sequenced genomes; (2) what is the likely explanation for the phenomenon of organisms retaining both classes of LysRS? To clarify these issues, we reconstructed the relevant archaeal and bacterial species trees, made the most parsimonious assignments of LysRS classes to the ancestral nodes of the trees, reconstructed LysRS1-2 gene trees in order to determine the probable origins of the transferred genes, reviewed the literature for organisms encoding simultaneously two varieties of aaRSs, albeit of the same class, and evaluated the significance of the experimentally determined distinction between the amino acid binding sites of LysRS1-2 in the context of the phenomenon of Archaea and Bacteria encoding both classes of synthetases.</p><p>Our analysis of the collected data confirmed that the extant distribution of LysRS1 and LysRS2 reflects a wide-spread LGT &#x02013; characteristic for the entire aaRS family of enzymes [<a href="#B14" class="cite-reflink">14</a>,<a href="#B15" class="cite-reflink">15</a>]. It enabled us to determine some of the most likely paths and several of the origins of these LGT events, and to elucidate the probable selective advantage to several prokaryotes encoding simultaneously both enzyme classes in the presence of environmentally dependent LysRS inhibitors.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561328" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561380" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Results</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id662487" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663136" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663146" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663571" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663579" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663727" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id661591">Results</div><div class="section-content"><p><div class="head2 head-separate">The evolutionary position of organisms coding for LysRS1, LysRS2 and both enzymes simultaneously</div>Except for the Archaeon <em>Cenarchaeum symbiosum</em> whose genome is still being sequenced, all organisms analyzed in this study for the reconstruction of the species trees have had their entire genomes sequenced and annotated. Thus, there is a reliable assignment for each organism whether it codes for LysRS1 and/or LysRS2. The assignments for Archaea and Bacteria are presented in figs. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a> and <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a> respectively. The occurrences of LGT events during the evolution of both Bacteria and Archaea are evident from the fact that both LysRSs are found in the two Domains (figs <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>, <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>).<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F1" name="F1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1475646&amp;blobname=1471-2148-6-22-1.gif" border="1" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span>Archaeal tree. Species tree for 19 Archaea encoding LysRS1 and LysRS2. Organisms coding for LysRS1 are colored blue, those coding for LysRS2 are in red, and the three Archaea that code both are in purple. Arrows indicate inferred LGT events.</span></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F2" name="F2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1475646&amp;blobname=1471-2148-6-22-2.gif" border="1" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span>Bacterial tree. Species tree for 43 Bacteria encoding LysRS2 and LysRS1. <em>Mycoplasma</em> in the figure refers to both <em>Mycoplasma pneumoniae </em>and <em>Mycoplasma genitalium</em>. <em>Gamma-proteobacteria (5)</em> refers to the following five species: <em>Escherichia coli</em>, <em>Haemophilus</em></span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><div class="head2 head-separate">LGTs in Archaea and the LysRS of the last common ancestor of Archaea</div>The most parsimonious assignment of LysRS types to the ancestral nodes in fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a> requires a minimum of two LGT events (marked by arrows in the figure). The majority of Archaea only code for LysRS1. Indeed, according to the most parsimonious reconstruction, their last common ancestor coded for LysRS1. The alternative scenario, in which the ancestor codes for LysRS2, requires two additional LGT events (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>). The three Methanosarcinales &#x02013; <em>M. barkeri, M. acetivorans</em>, and <em>M. mazei </em>&#x02013; code for both enzyme classes. Thus, it seems that their last common ancestor received the gene for LysRS2 via LGT. Since they are monophyletic, only a single LGT is needed to explain the presence of both enzyme classes in this group.<p>The situation is more complex in Crenarchaeota (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>). In this clade, two families code for LysRS1 (<em>C. symbiosum </em>and <em>Aeropyrum pernix</em>), and two for LysRS2 (<em>Pyrobaculum aerophilum </em>and <em>Sulfolobus solfataricus</em>). There are several possible scenarios that can explain this phenomenon:</p><p>1. The common ancestor of <em>P. aerophilum, S. solfataricus </em>and <em>A. pernix </em>first received a copy of LysRS2, and then lost its copy of LysRS1. Subsequently, <em>A. pernix </em>received a copy of LysRS1, and lost its copy of LysRS2 (two gains and two losses).</p><p><em>2. P. aerophilum </em>after its divergence, gained a copy of LysRS2, and lost its copy of LysRS1. Similarly, after its divergence <em>S. solfataricus </em>gained a copy of LysRS2, and lost its copy of LysRS1 (two gains and two losses).</p><p>3. The common ancestor of <em>P. aerophilum, S. solfataticus </em>and <em>A. pernix </em>received a copy of LysRS2. After the divergence of <em>P. aerophilum</em>, this organism lost its class 1 copy. The same loss occurred again, after the divergence of <em>S. solfataricus</em>. A third loss, of class 2 occurred in the lineage leading to <em>A. pernix</em>. This alternative requires one gain, and three losses.</p><p>Among these three scenarios, the third requires the least number of LGTs. When an organism codes for both classes of LysRS, a loss event of one copy may be sustainable. Assuming that a loss of one copy out of two is more likely than a LGT, the third scenario is the most probable one. It should be noted that all these alternatives rely on the correctness of the species tree. However, the phylogeny among <em>P. aerophilum, S. solfataricus </em>and <em>A. pernix </em>(fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>) was reconstructed with 100% bootstrap support [<a href="#B16" class="cite-reflink">16</a>]. See section 'Are the species-trees correct?' below for further discussion on the robustness of the species tree.</p></p><p><div class="head2 head-separate">LGTs in Bacteria and the LysRS of the last common ancestor of Bacteria</div>The most parsimonious assignment of LysRS types to the ancestral nodes in fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a> requires a minimum of five LGT events (marked by arrows in the figure). The majority of Bacteria only code for LysRS2. It is most parsimonious to assume that the bacterial ancestor coded for LysRS2 (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>). Among Actinobacteria, some species code for LysRS2 (<em>Mycobacterium leprae </em>and <em>Mycobacterium tuberculosis</em>), and some code for LysRS1 (<em>Streptomyces coelicolor </em>and <em>Streptomyces avermitilis</em>). The two species coding for LysRS1 are monophyletic [<a href="#B17" class="cite-reflink">17</a>]. Thus, it seems that a LysRS1 was laterally transferred to the common ancestor of <em>Streptomycetes </em>followed by a LysRS2 loss. Within the Firmicutes, <em>B. cereus </em>codes for both LysRS1 and LysRS2. Since this is the only known Firmicutae that codes for both types, we conclude that the LysRS1 was transferred to this species. Within the Spirochetes clade, <em>Leptospira integrans </em>codes for LysRS2, while in the second group, including the <em>T. pallidum</em>, <em>Treponema denticola </em>and <em>Borrelia burgdorferi </em>species, the last two species code for LysRS1 only, while <em>T. pallidum </em>codes for both LysRS types. This can be explained by a single LGT event (gain of LysRS1) in the common ancestor of <em>T. pallidum</em>, <em>T. denticola</em>, and <em>B. burgdorferi</em>, followed by a LysRS2 loss in <em>T. denticola </em>and <em>B. burgdorferi</em>. Within the proteobacteria, the beta, gamma, and epsilon clades, all code for LysRS2. Only within the alpha-proteobacteria most species code for LysRS1 (<em>Rickettsia conorii</em>, <em>Rickettsia prowasekii</em>, <em>Wolbachia </em>sp., <em>Sphingomonas aromaticivorans</em>, <em>Magnetospirillum magnetotacticum</em>, <em>Rhodobacter sphaeroides</em>, <em>Caulobacter crescentus</em>, <em>Rhodopseudomonas palustris</em>, <em>Brusella malitensis</em>, <em>Brucella suis</em>, and <em>Mesorhizobium loti</em>). Following the branching pattern of the Proteobacteria (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>) it seems that the last common ancestor of the alpha-proteobacteria had gained a copy of LysRS1, and lost its LysRS2. However, the clade including the rhizobiales <em>Sinorhizobium meliloti </em>and <em>Agrobacterium tumefaciens </em>codes for LysRS2. Thus, it seems that in the common ancestor of these species, LysRS2 was regained, and LysRS1 was lost. This scenario calls for two LGT events and two losses.</p><p><div class="head2 head-separate">Gene trees</div>Gene trees summarize our current estimation of the evolutionary relationships among the LysRS sequences. Combined with the species tree, gene trees are a valuable source of information concerning the origins of the laterally transferred genes.</p><p><div class="head2 head-separate">The origin of the inferred LGTs in Archaea</div>The maximum parsimony analysis has indicated that the ancestor of the Archaea most likely coded for LysRS1. We reconstructed a LysRS2 gene tree (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">3</a>) to track the origin of the genes that were laterally transferred to the Archaea. Since our species tree (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>) does not contain all the known LysRS2 sequences, we used blastp [<a href="#B18" class="cite-reflink">18</a>] to enlarge our bacterial LysRS2 database by choosing the first 177 non-redundant sequences &#x02013; 131 from complete genomes and 46 encoded by bacteria whose entire genome has not been sequenced yet. AspRS sequences were used to root the tree.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F3" name="F3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1475646&amp;blobname=1471-2148-6-22-3.gif" border="1" class="icon-reflink" alt="Figure 3" title="Figure 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 3</strong></a><div class="figure-table-caption-in-article"><span>LysRS2 gene tree. Maximum likelihood phylogenetic tree for 177 bacterial and 6 archaeal LysRS2 sequences, colored green and black, respectively. The tree has been rooted using four AspRS sequences (<em>B. aphidicoda, E. coli, C. jejuni, T. maritima</em>). Bootstrap</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>The LysRS2 genes of <em>P. aerophilum, S. solfataricus</em>, and <em>Sulfolobus tokodaii</em>, cluster together in the gene tree with a very high bootstrap value (99%) supporting our conclusion of a single LysRS2 LGT to the Crenarchaeota clade. This clade has only a low bootstrap value (36%) with respect to genes located on other branches of the tree. Two alternatives can explain such a position: (1) The LGT from Bacteria to Crenarchaeota is from an ancient ancestor of the bacterial Domain or from an extinct bacterial lineage that is an outgroup to most extant Bacteria, or from a yet unidentified bacterium. (2) There is not enough information to resolve the location of this clade within the LysRS2 gene tree, as is evident by the low bootstrap value. In the first alternative, it is not very likely that the LGT is from the bacterial ancestor, as we know that the LysRS2 LGT to Crenarchaeota occurred after the divergence of <em>C. symbiosum </em>(fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>). All these possibilities are likely scenarios and further bacterial genome sequencing has the potential to settle this issue.</p><p><em>M. barkeri</em>, <em>M. acetivorans </em>and <em>M. mazei </em>code for both classes of LysRS. The origin of their LysRS2 gene is in doubt. The group clusters with the bacterial lineage <em>A. aquifex </em>with a low bootstrap value (60%). Again, additional genomic bacterial sequencing might shed light on the history of this LGT event.</p></p><p><div class="head2 head-separate">The origin of the inferred LGTs in Bacteria</div>Five LGT events were inferred in the bacterial tree (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>). A gene tree of LysRS1 sequences encoded by Archaea and Bacteria was reconstructed in order to infer the origin of the laterally transferred genes (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">4</a>). Several LysRS1 sequences encoded by bacteria whose complete genomes have not been determined yet &#x02013; e.g., <em>Bradyrhizobium sp, Rickettsia sibirica, Borella afzelii </em>&#x02013; were excluded from this study due to high percentage identity, 61, 88 and 95%, between their sequences and sequences of bacterial LysRS1 utilized for the reconstruction of the gene tree (from <em>M. loti</em>, <em>R. prowazekii </em>and <em>B. burgdorferi </em>respectively). GluRS sequences were used to root the tree. The bacterial sequences of the <em>S. coelicolor, S. avermitilis, B. cereus, B. burgdorferi, T. pallidum</em>, and <em>T. denticola</em>, together with the archaeal <em>Thermococcaceae </em>clade (<em>P. horikoshii</em>, <em>P. abyssi </em>and <em>P</em>. <em>furiosus</em>) cluster together with a very high bootstrap support (99%). Such clustering is indicative of a Thermococcaceal source for the LysRS1 found in all bacterial sequences excluding the alpha proteobacteria.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F4" name="F4"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F4" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1475646&amp;blobname=1471-2148-6-22-4.gif" border="1" class="icon-reflink" alt="Figure 4" title="Figure 4"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 4</strong></a><div class="figure-table-caption-in-article"><span>LysRS1 gene tree. Maximum likelihood phylogenetic tree for 17 archaeal and 17 bacterial LysRS1 sequences, colored black and green, respectively. The tree has been rooted using four GluRS sequences (<em>B. aphidicoda, E. coli, C. jejuni, T. maritima</em>). Bootstrap</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F4" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>The LysRS1 sequences in alpha proteobacteria cluster with <em>C. symbiosum </em>with a very high bootstrap support (100%). Thus, <em>C. symbiosum </em>or a related yet undetermined Archaea is the most likely source for the LGT to alpha proteobacteria excluding the two species <em>A. tumerfaciens</em>, and <em>S. meliloti</em>. Within the alpha proteobacteria, these two species reversed to a LysRS2 class gene. To infer the origin of this reversed LGT, we refer to fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">3</a>. See Discussion, section 'The possible origins of laterally transferred LysRS genes' below, for details.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561328" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561380" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id661591" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Discussion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663136" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663146" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663571" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663579" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663727" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id662487">Discussion</div><div class="section-content"><p><div class="head2 head-separate">Most likely scenarios for LysRS LGT</div>In this study we analyzed the pattern of LysRSs LGT based on organisms with fully sequenced genomes, coding for LysRS1, LysRS2, or both. Analyzing this information and taking into account the evolutionary relationships among the organisms (the species tree) made it possible to infer the most likely LGT scenarios. As previously determined by other researchers for the entire family of aaRS enzymes [<a href="#B14" class="cite-reflink">14</a>], we also found that LGT for a particular synthetase &#x02013; LysRS &#x02013; is quite common in both Bacteria and Archaea. Inferring that the last common ancestors of Bacteria and Archaea most likely coded for LysRS2 and LysRS1, respectively, a complex pattern of LGT events emerged: LysRS1 was replaced by LysRS2 (and vice versa) in a specific group of organisms. In one occasion, a LysRS2 to LysRS1 LGT was followed by a reversed LGT to LysRS2, within the same group. It should be noted that a transition from one LysRS to another most probably occurred through an intermediate evolutionary stage in which organisms coded for both LysRSs [<a href="#B15" class="cite-reflink">15</a>]. Examples of extant species embodying such a stage are the three <em>Methanosarcinales</em>, <em>B. cereus</em>, and <em>T. pallidum</em><p>The LysRS1 and LysRS2 genes coded by <em>T. pallidum </em>probably illustrate an advanced phase of such a transitional stage: (i) the LysRS2 gene only codes for 351 residues [<a href="#B19" class="cite-reflink">19</a>]. This region shows a high similarity to the 376 residues of the <em>E. coli </em>LysRS2 catalytic domain located in the COOH-terminal region. However, the 144 residues at the NH2-terminal region in the <em>E. coli </em>enzyme, which includes the 80 residues of the tRNA (Lys) anticodon binding domain that are critical for the enzyme's acylation activity, are not coded <em>in T. palladium </em>[<a href="#B13" class="cite-reflink">13</a>]. The observed lack of the LysRS2 anticodon binding domain is the result of a LysRS1 gene entering the common ancestor of <em>T. palladium </em>and <em>B. burgdorferi </em>by LGT from an archaeal lineage [<a href="#B20" class="cite-reflink">20</a>]. LysRS1 proved by some measure more advantageous to <em>Treponema </em>than LysRS2. The latter became non-functional, subject in the course of evolution to gradual elimination from the genome accompanying the loss of function.</p></p><p><div class="head2 head-separate">Are the species-trees correct?</div>Our results depend on inferred species trees that might not be the true ones. Nevertheless, they do not rely on the existence of clades with low statistical support. For the archaeal tree (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>), <em>M. barkeri</em>, <em>M. acetivorans</em>, and <em>M. mazei </em>which code for both LysRS1 and LysRS2 are monophyletic [<a href="#B21" class="cite-reflink">21</a>]. Further, their clustering with <em>Hallobacterium </em>is supported with high bootstrap values [<a href="#B22" class="cite-reflink">22</a>]. The phylogenetic position of <em>P. aerophilum, S. solfataricus </em>and <em>A. pernix </em>is also generally accepted [<a href="#B16" class="cite-reflink">16</a>]. For the bacterial tree (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>), there is wide agreement regarding the monophyly of <em>alpha-proteobacteria </em>and the monophyly of <em>Spirochetes </em>[<a href="#B23" class="cite-reflink">23</a>].</p><p><div class="head2 head-separate">The possible origins of laterally transferred LysRS genes</div>We determined seven LGT events &#x02013; two in Archaea and five in Bacteria. One of the main difficulties in the inference of the origins of LGTs is that such inference heavily relies on a gene tree. A gene tree is always reconstructed from a single gene, and hence, based on a limited amount of data. Thus, the bootstrap values for various bifurcations in the tree are usually not very high. It is well known that increased taxonomical sampling improves such inference [<a href="#B24" class="cite-reflink">24</a>]. To this end, we reconstructed the LysRS gene trees from an extensive database of extant Bacteria and Archaea. Not surprisingly, we could not reliably infer the origins of the two LysRS2 LGTs to Archaea (figs. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a> and <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">3</a>). Encouragingly, the possible origins of four of the bacterial LGTs (<em>Actinobacteria</em>, <em>B</em>. <em>cereus</em>, <em>alpha-proteobacteria </em>and <em>Spirochetae</em>) were determined with a high degree of confidence (figs. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a> and <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">4</a>). For example the archeal <em>Pyrococci </em>clade seems to contain the closest LysRS1 sequences to those of bacterial species (fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">4</a>). Yet, the details of the LGT events are still unknown: These species are hyperthermophiles, inhabiting environments with extremely high temperatures such as undersea hot vents, whereas all the above mentioned bacteria are mesophiles. The physiological and biochemical conditions that promoted such an evolutionary event remain an enigma.<p>The last intriguing question concerns the LGT reversal of two alpha-proteobacterial species (<em>A. tumefaciens </em>and <em>S. meliloti</em>) to code for LysRS2. The bootstrap value clustering them with other bacteria is very low (39% with <em>Dehalococcus </em><em>ethenogenes</em>, see SM, <a href="#S3" class="side-supplink">Additional file 3</a>). As both species are capable of nitrogen-fixing [<a href="#B25" class="cite-reflink">25</a>,<a href="#B26" class="cite-reflink">26</a>], we speculate that an extinct nitrogen fixing bacteria may have been the origin of the LysRS2 LGT.</p><p>Additional sequences of bacterial LysRS2 genes are likely to shed new light on the evolution of the LysRS2 LGT events for which the origin remains uncertain. It is remarkable that the sources of LysRS1 LGTs are readily identifiable while those for LysRS2 remain, for now, shrouded in uncertainty.</p></p><p><div class="head2 head-separate">Possible advantages for organisms coding for both LysRS classes</div>Long before the discovery of Archaea and Bacteria coding for both LysRSs, it was found that some prokaryotes code for two paralogous genes for some synthetases: <em>lysS </em>and <em>lysU </em>in <em>E.coli</em>&nbsp;<a href="#B33" class="cite-reflink">33</a>, <em>thrSv </em>and <em>thrS2 </em>and <em>tyrS </em>and <em>tyrZ</em>, in <em>B. subtilis </em>[<a href="#B34" class="cite-reflink">34</a>,<a href="#B35" class="cite-reflink">35</a>]. Recently, co-existing forms were published for SerRS1/SerRS2 and TrpRS1/TrpRS2 in <em>C. acetobutylicum</em>, CysRS1/CysRS2 in <em>M. tuberculosis</em>, TrpRS1/TrpRS2 in <em>E. faecalis </em>[<a href="#B14" class="cite-reflink">14</a>] and GluRS1/GluRS2 in more than 30 bacterial genomes [<a href="#B14" class="cite-reflink">14</a>,<a href="#B27" class="cite-reflink">27</a>-<a href="#B31" class="cite-reflink">31</a>]. While only some of the functions of the observed redundancies have been determined, it is noteworthy that in some cases it was found that the aaRS duplications render a selective advantage to the affected organisms providing protection against potentially detrimental effects on protein synthesis caused by amino acid competitors [<a href="#B32" class="cite-reflink">32</a>].<p>One example is the <em>Streptococcus pneumoniae </em>coding for two distantly related MetRS genes. It was found that one of them proves necessary and sufficient for resistance to MetRS inhibitors [<a href="#B33" class="cite-reflink">33</a>]. Another example is the existence of two IleRS variants in <em>Pseudomonas fluorescens. </em>This gamma proteobacterium produces the anti-bacterial agent pseudomonic acid (mupirocin), which if not neutralized, competitively inhibits the acylation of tRNA(Ile) with isoleucine, thereby shutting off protein synthesis and arresting cell growth. <em>P. fluorescens </em>avoids self destruction by one of its IleRS variants binding preferentially to isoleucine, with a remarkably high insensitivity to mupirocin [<a href="#B34" class="cite-reflink">34</a>].</p><p>A related selective advantage is surmised for the prokaryotes coding for both classes of LysRS by the same organism. In the case of LysRS1 and LysRS2, there is evidence that the former is less sensitive to inhibitors, due to the active site of LysRS1 being more compact than that of LysRS2 [<a href="#B4" class="cite-reflink">4</a>], i.e., LysRS1 is less accommodating to lysine analogues with backbone substitutions compared with LysRS2 [<a href="#B5" class="cite-reflink">5</a>]. This bequeathed a possible selective advantage to <em>B. cereus </em>and <em>T. pallidum </em>after acquiring a copy of LysRS1: harmful lysine-analogues to LysRS in the environment bind preferentially to LysRS2, leaving LysRS1 available for unimpeded acylation of lysine to cognate tRNAs. What could be the possible selective advantage for the <em>Methanosarcinales </em>acquiring LysRS2? Comparing the rate constants (<em>k<sub>cat</sub></em>) of LysRS1 and LysRS2 reveals that LysRS2 has a substrate turnover speed more than 15 times greater than that of LysRS1, while their Michaelis constants (<em>Km</em>) values are practically the same [<a href="#B35" class="cite-reflink">35</a>]. Therefore, we hypothesize that the selective advantage for coding LysRS2 is in their enhanced ability for protein synthesis. Thus, it is possible that Bacteria and Archaea coding for the two types of LysRS, in fact, developed a "safety net": in the absence of LysRS inhibitors, LysRS2 is expected to be the dominant active form. In the presence of inhibitors, LysRS1 provides a means for continuing protein synthesis.</p><p>Noteworthy, recently it was determined that in <em>B. cereus </em>LysRS1 and LysRS2 aminoacylate two tRNA species: the canonical tRNA(Lys) and a smaller RNA annotated tRNA(Other), which features a tryptophan anticodon (CCA) with a non-canonical secondary structure. tRNA(Other) was found to be synthesized only in the presence of both LysRSs, which act together during tRNA(Other) aminoacylation. This process is confined to the stationary phase, suggesting a role in growth-phase-specific protein synthesis [<a href="#B47" class="cite-reflink">47</a>].</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id663136">Conclusion</div><div class="section-content"><p>The LysRS family of enzymes has undergone several complex LGT events. The currently available data were sufficient for unambiguously identifying the origins of LysRS1 but not of LysRS2 gene transfers. The LGT transition stage of simultaneous encoding LysRS1-2 by several Archaea and Bacteria may confer a selective advantage in the presence of environmentally dependent LysRS inhibitors.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561328" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561380" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id661591" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id662487" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663136" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Methods</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663571" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663579" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663727" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id663146">Methods</div><div class="section-content"><p><div class="head2 head-separate">Data collection</div>LysRS1-2 sequences were retrieved from public databases: the Aminoacyl-tRNA synthetases database (aaRSDB) [<a href="#B19" class="cite-reflink">19</a>], the National Center for Biotechnology Information (NCBI) [<a href="#B36" class="cite-reflink">36</a>], the Swiss-Prot Protein knowledgebase/TrEMBL Computer-annotated supplement to Swiss-Prot [<a href="#B37" class="cite-reflink">37</a>]. 16S rRNA and 23S rRNA sequences were retrieved from the same public databases, and in addition from the Joint Genome Institute, Microbial Genomes (JGI) [<a href="#B38" class="cite-reflink">38</a>], the Ribosomal Database Project II [<a href="#B39" class="cite-reflink">39</a>] and the European Ribosomal RNA Database [<a href="#B40" class="cite-reflink">40</a>]. Additional bacterial LysRS2 sequences were obtained using NCBI Protein-protein BLAST (blastp) [<a href="#B41" class="cite-reflink">41</a>], seeded by <em>A. tumefaciens </em>LysRS2 [NCBI: <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0deo44j4-TE5r6a_TKgec41UnR4c9t1CQ7xNwjAC_&amp;reftype=extlink-entrez-protein&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CProtein&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=protein&amp;dopt=GenPept&amp;list_uids=17938162" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">NP_534951</a></span>]. <a href="#S1" class="side-supplink">Additional file 1</a> provides a listing of the bacterial and archaeal phyla, classes and species, with corresponding LysRS2 accession numbers and their sources. <a href="#S2" class="side-supplink">Additional file 2</a> provides a listing of the archaeal and bacterial phyla, classes and species, with corresponding LysRS1 accession numbers and their sources.</p><p><div class="head2 head-separate">Reconstruction of the Archaea species tree</div>Species tree for 19 Archaea was based on [<a href="#B21" class="cite-reflink">21</a>,<a href="#B22" class="cite-reflink">22</a>]. It incorporates the two major phyla of the Kingdom &#x02013; Crenarchaota and Euryarchaeota - and most of the representative genera in each phylum. The conspicuous exception was the absence of the psychrophilic crenarchaeon <em>Cenarchaeum symbiosum</em>. Its phylogenetic position was obtained from [<a href="#B42" class="cite-reflink">42</a>]. The tree is given in fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>.</p><p><div class="head2 head-separate">Reconstruction of the Bacteria species tree</div>Species tree for 43 Bacteria was based on [<a href="#B43" class="cite-reflink">43</a>], which includes the major phylogenetic relationships among phyla of the Kingdom. The phylogenetic position of most genera was obtained from the 16S rRNA based reconstruction provided in [<a href="#B44" class="cite-reflink">44</a>]. Of special interest for us were the positions of the genera within the alpha proteobacteria, because they include the site for the putative LGT event involving <em>A. tumeficiens </em>and <em>S. meliloti</em>. Specifically, the phylogenetic relationships among <em>A. tumeficiens</em>, <em>S. meliloti</em>, <em>B. Suis</em>, and <em>M. Loti </em>inferred in [<a href="#B44" class="cite-reflink">44</a>] were different depending on the gene used for the reconstruction (16S rRNA or HSP70). We therefore utilized the 23S rRNA database [<a href="#B39" class="cite-reflink">39</a>] to reconstruct a neighbor joining [<a href="#B46" class="cite-reflink">46</a>] tree of alpha-proteobacteria (with 100 bootstrap replicates) and compared the results with those given in [<a href="#B44" class="cite-reflink">44</a>]. In this reconstruction <em>A. tumeficiens </em>and <em>S. meliloti </em>clustered together with very high bootstrap support (in agreement with fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2b</a> of [<a href="#B44" class="cite-reflink">44</a>]), and hence they are grouped together in fig. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>. We also utilized the 16S rRNA gene database [<a href="#B40" class="cite-reflink">40</a>] to reconstruct a neighbor joining proteobacterial phylogenetic tree with 100 replicates bootstrap, and compared our results with the trees in [<a href="#B23" class="cite-reflink">23</a>,<a href="#B43" class="cite-reflink">43</a>]; The referenced and obtained trees were in agreement (not shown).</p><p><div class="head2 head-separate">Reconstruction of the gene trees</div>Gene trees for LysRS2 and LysRS1 with bootstrap support values (100 replicates) were reconstructed using maximum likelihood (ML) as implemented in the PHYML software [<a href="#B48" class="cite-reflink">48</a>]. Among site rate variation was modeled using a gamma distribution with 4 discrete categories. Similar results were obtained using the neighbor joining reconstruction method [<a href="#B43" class="cite-reflink">43</a>] (data not shown). ML trees with bootstrap value support are presented in figs. <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">3</a> and <a href="articlerender.fcgi?artid=1475646&amp;rendertype=figure&amp;id=F4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">4</a>, respectively. To enhance the presentation of the entire (voluminous) data, in these two figures many Bacteria and Archaea are grouped under common headings, in conformity with the presentation in the complete (unabbreviated) LysRS2 and LysRS1 ML trees, given as SM, <a href="#S3" class="side-supplink">additional files 3</a> and <a href="#S4" class="side-supplink">4</a> respectively.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id663571">Authors' contributions</div><div class="section-content"><p>SS and TP analyzed the data, prepared the figures and contributed to writing the manuscript, RN initiated the study and contributed to the writing of the manuscript.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id663579"><a id="supplementary-material-sec"></a>Supplementary Material</div><div class="section-content"><!--body--><div id="S1"><div><strong>Additional file 1</strong></div><p>Listing of the bacterial and archaeal phyla, classes and species, with the corresponding LysRS2 accession numbers and their sources</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1475646&amp;blobname=1471-2148-6-22-S1.doc">Click here for file</a><sup> (94K)</sup></blockquote></div></div><div id="S2"><div><strong>Additional file 2</strong></div><p>Listing of the archaeal and bacterial phyla, classes and species, with corresponding LysRS1 accession numbers and their sources.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1475646&amp;blobname=1471-2148-6-22-S2.doc">Click here for file</a><sup> (41K)</sup></blockquote></div></div><div id="S3"><div><strong>Additional file 3</strong></div><p>Complete PHYML LysRS2 Maximum Likelihood gene tree in Newick format.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1475646&amp;blobname=1471-2148-6-22-S3.txt">Click here for file</a><sup> (8.1K)</sup></blockquote></div></div><div id="S4"><div><strong>Additional file 4</strong></div><p>Complete PHYML LysRS1 Maximum Likelihood gene tree in Newick format.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=1475646&amp;blobname=1471-2148-6-22-S4.txt">Click here for file</a><sup> (1.6K)</sup></blockquote></div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id663706">Acknowledgements</div><div class="section-content"><p><p><p>We thank Prof. W. Ford Doolittle for his critical reading of an early draft of the manuscript and for his comments and Adi Stern for her assistance in the analyses. TP was supported by an Israeli Science Foundation grant number 1208/04 and by a grant in Complexity Science from the Yeshaia Horvitz Association. The research of RN in Israel has been supported in part by the "Center of Excellence in Geometric Computing and its Applications" funded by the Israel Science Foundation (administered by the <em>Israel Academy of Sciences</em>). This project has been funded in whole or in part with Federal funds from the National Cancer Institute, National Institutes of Health, under contract number NO1-CO-12400. The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government. This research was supported (in part) by the Intramural Research Program of the NIH, National Cancer Institute, Center for Cancer Research.</p></p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561328" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id561380" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id661591" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id662487" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663136" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663146" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663571" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id663579" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id663727">References</div><div class="section-content"><div class="back-matter-section"><ul class="back-ref-list" style="list-style-type: decimal;"><li id="B1">Eriani, G; Delarue, M; Poch, O; Gangloff, J; Moras, D. Partition of tRNA synthetases into two classes based on mutually exclusive sets of sequence motifs. <span><span class="ref-journal">Nature. </span>1990;<span class="ref-vol">347</span>:203–206. doi: 10.1038/347203a0.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0DsOhD7VzxP1y5RtR_5146yHTrJO1xA1A-rPfjQKS&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/2203971">PubMed</a>]</span></li><li id="B2">Cavarelli, J; Moras, D. Recognition of tRNAs by aminoacyl-tRNA synthetases. <span><span class="ref-journal">FASEB J. </span>1993;<span class="ref-vol">7</span>:79–86.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0dW9NHueS8d58Ta2bCIFu84PO5dsc8sjmRmTTbQhl&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8422978">PubMed</a>]</span></li><li id="B3">Ibba, M; Bono, JL; Rosa, PA; S&#x000f6;ll, D. Archaeal-type lysyl-tRNA synthetase in the Lyme disease spirochete <em>Borrelia burgdorferi</em>. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1997;<span class="ref-vol">94</span>:14383–14388. doi: 10.1073/pnas.94.26.14383.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0l36Ou7352h7royGSImAbXiV5jKKKuiF3PKDe7QxU&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9405621">PubMed</a>]</span></li><li id="B4">Terada, T; Nureki, O; Ishitani, R; Ambrogelly, A; Ibba, M; S&#x000f6;ll, D; Yokoyama, S. Functional convergence of two lysyl-tRNA synthetases with unrelated topologies. <span><span class="ref-journal">Nat Struct Biol. </span>2002;<span class="ref-vol">9</span>:257–262. doi: 10.1038/nsb777.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0t5u8DuWhOhdrmXJP6R-ZHdSs4bmjqV3hOEhvTQ28&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11887185">PubMed</a>]</span></li><li id="B5">Jester, BC; Levengood, JD; Roy, H; Ibba, M; Devine, KM. Nonorthologous replacement of lysyl-tRNA synthetase prevents addition of lysine analogues to the genetic code. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2003;<span class="ref-vol">100</span>:14351–14356. doi: 10.1073/pnas.2036253100.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hWXH0YH2ykbtSn98V4XPoRfu3PlGecMGL-YIM53v&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14623972">PubMed</a>]</span></li><li id="B6">Doolittle, RF; Handy, J. Evolutionary anomalies among the aminoacyl-tRNA synthetases. <span><span class="ref-journal">Curr Op Gen Dev. </span>1998;<span class="ref-vol">8</span>:630–636. doi: 10.1016/S0959-437X(98)80030-0.</span></li><li id="B7">Martinis, SA; Plateau, P; Cavarelli, J; Florentz, C. Aminoacyl-tRNA synthetases: a new image for a classical family. <span><span class="ref-journal">Biochimie. </span>1999;<span class="ref-vol">81</span>:683–700. doi: 10.1016/S0300-9084(99)80126-6.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0PwaFtyPycQHmVmVk3cFrSs58BJTYONLQIDJJTAxN&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10492015">PubMed</a>]</span></li><li id="B8">Francklyn, C; Perona, JJ; Puetz, J; Hou, Y-M. Amynoacyl-tRNA synthetases: Versatile players in the changing theater of translation. <span><span class="ref-journal">RNA. </span>2002;<span class="ref-vol">8</span>:1363–1372. doi: 10.1017/S1355838202021180.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FU6xEz4cvOKCEUCW5sal6_M73VCotv5LiiN2AAMH&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12458790">PubMed</a>]</span></li><li id="B9">Deppenmeier, U; Johann, A; Hartsch, T; Merkl, R; Schmitz, RA; Martinez-Arias, R; Henne, A; Wiezer, A; B&#x000e4;umer, S; Jacobi, C; Br&#x000fc;ggemann, H; Lienard, T; Christmann, A; B&#x000f6;meke, M; Steckel, S; Bhattacharyya, A; Lykidis, A; Overbeek, R; Klenk, H-P; Gunsalus, RP; Fritz, H-J; Gottschalk, G. The genome of <em>Methanosarcina mazei </em>: evidence for LGT between bacteria and archaea. <span><span class="ref-journal">J Mol Microbiol Biotechnol. </span>2002;<span class="ref-vol">4</span>:435–461.</span></li><li id="B10">Galagan, JE; Nusbaum, C; Roy, A; Endrizzi, MG; Macdonald, P; FitzHugh, W; Calvo, S; Engels, R; Smirnov, S; Atnoor, D; Brown, A; Allen, N; Naylor, J; Stange-Thomann, N; DeArellano, K; Johnson, R; Linton, L; McEwan, P; McKernan, K; Talamas, J; Tirrell, A; Ye, W; Zimmer, A; Barber, RD; Cann, I; Graham, DE; Grahame, DA; Guss, AM; Hedderich, R; Ingram-Smith, C; Kuettner, HG; Krzycki, JA; Leigh, JA; Li, W; Liu, J; Mukhopadhyay, B; Reeve, JN; Smith, K; Springer, TA; Umayam, LA; White, O; White, RH; de Macario, EC; Ferry, JG; Jarrell, KF; Jing, H; Macario, AJL; Paulsen, I; Pritchett, M; Sowers, KR; Swanson, RV; Zinder, SH; Lander, E; Metcalf, WW; Birren, B. The genome of <em>Methanosarcina acetivorans </em>reveals extensive metabolic and physiological diversity. <span><span class="ref-journal">Genome Res. </span>2002;<span class="ref-vol">12</span>:532–542. doi: 10.1101/gr.223902.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0TYwuWFXQaU9-pFbJSnAmyuA5GI8qIzRlVQ5MRQER&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11932238">PubMed</a>]</span></li><li id="B11">Brill, J. <em>Methanosarna barkeri </em><em>fusaro</em> analysis files. <span class="ref-journal">JGI Microbial Sequencing Program.</span> 2003. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0-InfdBk8NYwG_67cvkLWTd7bukDDCRyoMZPq0kQA&amp;reftype=extlink&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://genome.jgi-psf.org/mic_cur1.html" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://genome.jgi-psf.org/mic_cur1.html</a></span></li><li id="B12">Ivanova, N; Sorokin, A; Anderson, I; Galleron, N; Candelon, B; Kapatral, V; Bhattacharyya, A; Reznik, G; Mikhailova, N; Lapidus, A; Chu, L; Mazur, M; Goltsman, E; Larsen, N; D'Souza, M; Walunas, T; Grechkin, Y; Pusch, G; Haselkorn, R; Fonstein, ; Ehrlich, SD; Overbeek, R; Kyrpides, N. Genome sequence of <em>Bacillus cereus </em>and comparative analysis with <em>Bacillus anthracis</em>. <span><span class="ref-journal">Nature. </span>2003;<span class="ref-vol">423</span>:87–91. doi: 10.1038/nature01582.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0-mCttv6i-KTN9blX8VyMP5S3pqsN65M95XqQfke3&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12721630">PubMed</a>]</span></li><li id="B13">Fraser, CM; Norris, SJ; Weinstock, GM; White, O; Sutton, GG; Dodson, R; Gwinn, M; Hickey, EK; Clayton, R; Ketchum, KA; Sodergren, E; Hardham, JM; McLeod, MP; Salzberg, S; Peterson, J; Khalak, H; Richardson, D; Howell, JK; Chidambaram, M; Utterback, T; McDonald, L; Artiach, P; Bowman, C; Cotton, MD; Fujii, C; Garland, S; Hatch, B; Horst, K; Roberts, K; Sandusky, M; Weidman, J; Smith, HO; Venter, JC. Complete genome sequence of <em>Treponema pallidum</em>, the syphilis spirochete. <span><span class="ref-journal">Science. </span>1998;<span class="ref-vol">281</span>:375–388. doi: 10.1126/science.281.5375.375.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0bayDgaHlOkIBr3P067KqyORnW75vsqiyKDZwUAbt&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9665876">PubMed</a>]</span></li><li id="B14">Woese, CR; Olsen, GJ; Ibba, M; S&#x000f6;ll, D. Aminoacyl-tRNA synthetases, the genetic code, and the evolutionary process. <span><span class="ref-journal">Microbiol Mol Biol Rev. </span>2000;<span class="ref-vol">64</span>:202–236. doi: 10.1128/MMBR.64.1.202-236.2000.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pI2Jpb1qct_c6hzcLXp7YclIw9sFdxmpoZXUjAdb&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10704480">PubMed</a>]</span></li><li id="B15">Doolittle, WF; Boucher, Y; Nesbe, CL; Douady, CJ; Anderson, JO; Roger, AJ. How big is the iceberg on which organellar genes in nuclear genomes are but the tip? <span><span class="ref-journal">Phil Trans R Soc B. </span>2003;<span class="ref-vol">358</span>:39–58. doi: 10.1098/rstb.2002.1185.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=02gLhejyBagmXhACDyzxh9Iy87hMmHpGMoCioXktx&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12594917">PubMed</a>]</span></li><li id="B16">Waters, E; Hohn, MJ; Ahel, I; Graham, DE; Adams, MD; Barnstead, M; Beeson, KY; Bibbs, L; Bolanos, R; Keller, M; Kretz, K; Lin, X; Mathur, E; Ni, J; Podar, M; Richardson, T; Sutton, GG; Simon, M; Soll, D; Stetter, KO; Short, JM; Noordewier, M. The genome of <em>Nanoarchaeum equitans</em>: Insights into early archaeal evolution and derived parasitism. <span><span class="ref-journal">Proc Natl Acad Sci US. </span>2003;<span class="ref-vol">100</span>:12984–12988. doi: 10.1073/pnas.1735403100.</span></li><li id="B17">Ikeda, H; Ishikawa, J; Hanamoto, A; Shinose, M; Kikuchi, H; Shiba, T; Sakaki, Y; Hattori, M; Omura, S. Complete genome sequence and comparative analysis of the industrial microorganism <em>Streptomyces avermitilis</em>. <span><span class="ref-journal">Nat Biotechnol. </span>2003;<span class="ref-vol">21</span>:526–531. doi: 10.1038/nbt820.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0zWzxJX-xZLiePhvQx1DxpZ4WCmd-uNucCIdXr5RV&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12692562">PubMed</a>]</span></li><li id="B18">Altschul, SF; Gish, W; Miller, W; Myers, EW; Lipman, DJ. Basic local alignment search tool. <span><span class="ref-journal">J Mol Biol. </span>1990;<span class="ref-vol">215</span>:403–410. doi: 10.1006/jmbi.1990.9999.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0vKWVs7-hGG0knJBZGU63B0XIG_HbjNhxiY93jAi4&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/2231712">PubMed</a>]</span></li><li id="B19">Szymanski, M; Deniziak, MA; d Barciszewski, J. Aminoacyl-tRNA synthetases database. <span><span class="ref-journal">Nucleic Acids Res. </span>2001;<span class="ref-vol">29</span>:288–290. doi: 10.1093/nar/29.1.288. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0H5Y1yukqDcJR0lys1ZNoOSnizdeUzKQPdyY4NQ8N&amp;reftype=extlink&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://rose.man.poznan.pl/aars" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://rose.man.poznan.pl/aars</a></span></span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Jk13J13yZucEQzyRp2UuNogkSDy564NmYO5oJQd5&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11125115">PubMed</a>]</span></li><li id="B20">Subramanian, G; Koonin, EV; Aravind, L. Comparative genome analysis of the pathogenic spirochetes <em>Borrelia burgdorferi </em>and <em>Treponema pallidum</em>. <span><span class="ref-journal">Infect Immun. </span>2000;<span class="ref-vol">68</span>:1633–1648. doi: 10.1128/IAI.68.3.1633-1648.2000.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Z1eXp9hxCOzWynDVHQrxOcWTWoJkcIwpRx8pNkst&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10678983">PubMed</a>]</span></li><li id="B21">Allers, T; Mevarech, M. Archaeal genetics &#x02013; the third way. <span><span class="ref-journal">Nat Rev Genet. </span>2005;<span class="ref-vol">6</span>:58–73. doi: 10.1038/nrg1504.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0jyHsKYAsc6aC1v_ekXLL2hgzUOW88oFzwUEK7Ali&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15630422">PubMed</a>]</span></li><li id="B22">Brochier, C; Forterre, P; Gribaldo, S. An emerging phylogenetic core of Archaea: phylogenies of transcription and translation machineries converge following addition of new genome sequences. <span><span class="ref-journal">BMC Evol Biol. </span>2005;<span class="ref-vol">5</span>:36. doi: 10.1186/1471-2148-5-36.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hOjlRXad3C4D03_ukBTuLaYlIregUsZXR4d2YQbH&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15932645">PubMed</a>]</span></li><li id="B23">Madigan, MT; Martinko, JM; Parker, J. <span class="ref-journal">Biology of microorganisms.</span> 10. Prentice Hall; 2003. </li><li id="B24">Zwickl, DJ; Hillis, DM. Increased taxon sampling greatly reduces phylogenetic error. <span><span class="ref-journal">Sys Biol. </span>2002;<span class="ref-vol">51</span>:588–598. doi: 10.1080/10635150290102339.</span></li><li id="B25">Wood, DW; Setubal, JC; Kaul, R; Monks, DE; Kitajima, JP; Okura, VK; Zhou, Y; Chen, L; Wood, GE; Almeida, NF, Jr; Woo, L; Chen, Y; Paulsen, IT; Eisen, JA; Karp, PD; Bovee, D, Sr; Chapman, P; Clendenning, J; Deatherage, G; Gillet, W; Grant, C; Kutyavin, T; Levy, R; Li, MJ; McClelland, E; Palmieri, A; Raymond, C; Rouse, G; Saenphimmachak, C; Wu, Z; Romero, P; Gordon, D; Zhang, S; Yoo, H; Tao, Y; Biddle, P; Jung, M; Krespan, W; Perry, M; Gordon-Kamm, B; Liao, L; Kim, S; Hendrick, C; Zhao, ZY; Dolan, M; Chumley, F; Tingey, SV; Tomb, JF; Gordon, MP; Olson, MV; Nester, EW. The Genome of the Natural Genetic Engineer <em>Agrobacterium tumefaciens </em>C58. <span><span class="ref-journal">Science. </span>2001;<span class="ref-vol">294</span>:2317–2323. doi: 10.1126/science.1066804.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=04_x__FOajrdbQJyAx_xnu00hC5qD5MXxaQ6Ukknm&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11743193">PubMed</a>]</span></li><li id="B26">Capela, D; Barloy-Hubler, F; Gouzy, J; Bothe, G; Ampe, F; Batut, J; Boistard, P; Becker, A; Boutry, M; Cadieu, E; Dreano, S; Gloux, S; Godrie, T; Goffeau, A; Kahn, D; Kiss, E; Lelaure, V; Masuy, D; Pohl, T; Portetelle, D; Puhler, A; Purnelle, B; Ramsperger, U; Renard, C; Thebault, P; Vandenbol, M; Weidner, S; Galibert, F. Analysis of the chromosome sequence of the legume symbiont <em>Sinorhizobium meliloti </em>strain 1021. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>2001;<span class="ref-vol">98</span>:9877–9882. doi: 10.1073/pnas.161294398.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0DQnXg4jjppOczEFVLIt6SJiUw1-VJ89VjspRD59a&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11481430">PubMed</a>]</span></li><li id="B27">Clark, RL; Neidhardt, FC. Roles of the two lysyl-tRNA synthetases of <em>Escherichia coli </em>: analysis of nucleotide sequences and mutant behavior. <span><span class="ref-journal">J Bacteriol. </span>1990;<span class="ref-vol">172</span>:3237–3243.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0DiYHh7Wbt1-IRbkA2fOg8vGYJsC7-J06F7JA-58K&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/2188953">PubMed</a>]</span></li><li id="B28">Putzer, H; Brackhage, AA; Grunberg-Manago, M. Independent genes for two threonyl-tRNA synthetases in <em>Bacillus subtilis </em>. <span><span class="ref-journal">J Bacteriol. </span>1990;<span class="ref-vol">172</span>:4593–4602.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0HqkpJGOVeK4iqS8iqwHvTh80kE8jAi5b3g8tsku3&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/2115870">PubMed</a>]</span></li><li id="B29">Henkin, TM; Glass, BL; Grundy, FJ. Analysis of <em>Bacillus subtilis tyrS </em>gene: conservation of a regulatory sequence in multiple tRNA synthetase genes. <span><span class="ref-journal">J Bacteriol. </span>1992;<span class="ref-vol">174</span>:1299–1306.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0PUG5HaFigs1CRzHNrOaL07jsJfDn6BJP0i8zZQyp&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/1735721">PubMed</a>]</span></li><li id="B30">Salazar, JC; Ahel, I; Orellana, O; Tumbula-Hansen, D; Krieger, R; Daniels, L; S&#x000f6;ll, D. Coevolution of an aminoacyl-tRNA synthetase with its tRNA sustrates. <span><span class="ref-journal">Proc Natl. </span>2003;<span class="ref-vol">100</span>:13863–13868. doi: 10.1073/pnas.1936123100.</span></li><li id="B31">Lee, J; Hendrickson, TL. Divergent anticodon recognition in contrasting glutamyl-tRNA synthetases. <span><span class="ref-journal">J Mol Biol. </span>2004;<span class="ref-vol">344</span>:1167–1174. doi: 10.1016/j.jmb.2004.10.013.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=08_rmidWChfUVuqQ71nJAPnIZrA-LSWtZL_DVY5JV&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15561136">PubMed</a>]</span></li><li id="B32">Brown, JR; Gentry, D; Becker, JA; Ingraham, K; Holmes, DJ; Stanhope, MJ. Horizontal transfer of drug-resistant aminoacyl-transfer-RNA synthetases of anthrax and Gram-positive pathogens. <span><span class="ref-journal">EMBO Rep. </span>2003;<span class="ref-vol">4</span>:692–698. doi: 10.1038/sj.embor.embor881.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0NKG_GI-gJe0VA7puSL4uJGA15MNp5zMmGi7IvA57&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12792655">PubMed</a>]</span></li><li id="B33">Gentry, DR; Ingraham, KA; Stanhope, MJ; Rittenhouse, S; Jarvest, RL; O'Hanlon, PJ; Brown, JR; Holmes, DJ. Variable sensitivity to bacterial methionyl-tRNA synthetase inhibitors reveals subpopulations of <em>Streptococcus pneumoniae </em>with two distinct methionyl-tRNA synthetase genes. <span><span class="ref-journal">Antimicrob Agents Chemother. </span>2003;<span class="ref-vol">47</span>:1784–1789. doi: 10.1128/AAC.47.6.1784-1789.2003.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0B1hpG9fQiMaYmWRnXIj5EEZbWXgWzCWDvBk63QKK&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12760849">PubMed</a>]</span></li><li id="B34">Yanagisawa, T; Kawakami, M. How does <em>Pseudomonas fluorescens </em>avoid suicide from its antibiotic pseudomonic acid? Evidence for two evolutionarily distinct isoleucyl-tRNA synthetases conferring self-defense. <span><span class="ref-journal">J Biol Chem. </span>2003;<span class="ref-vol">278</span>:25887–25894. doi: 10.1074/jbc.M302633200.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0JUe-nvC0-ZwIERSrW8ve1VUg-5HVTPvdu3idxkAR&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12672810">PubMed</a>]</span></li><li id="B35">Ibba, M; Losey, HC; Kawarabayasi, Y; Kikuchi, H; Bunjun, S; S&#x000f6;ll, D. Substrate recognition by class I lysyl-tRNA synthetases: a molecular basis for gene displacement. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1999;<span class="ref-vol">96</span>:418–423. doi: 10.1073/pnas.96.2.418.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0lqtJMf5-WTFXMTpCx61mKUD3zFn10UTLvVTcZAzo&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9892648">PubMed</a>]</span></li><li id="B36">National Center for Biotechnology Information. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0bo1_LZBJ6qKxMlyKiF-6hsNfi-2YDXvwmzr7u5Ep&amp;reftype=extlink&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.ncbi.nlm.nih.gov</a></span></li><li id="B37">Swiss-Prot Protein knowledgebase / TrEMBL Computer annotated supplement to Swiss-Prot. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0HCfswfnBulbC6L_tSDDBjl-jsMK7zVMQIrTWh5nX&amp;reftype=extlink&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.expasy.org" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.expasy.org</a></span></li><li id="B38">Joint Genome Institute, Microbial Genomes. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0V_zRV-FyVKSWcmJiNUtMJlBWtj7PrMuuse_NokcC&amp;reftype=extlink&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://genome.jgi-psf.org/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://genome.jgi-psf.org/</a></span></li><li id="B39">Ribosomal Database Project II. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0hGn2QbYx6MoSMGqTFtRqm-zQUpbuGFa-OJKdC5wh&amp;reftype=extlink&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://rdp.cme.msu.edu" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://rdp.cme.msu.edu</a></span></li><li id="B40">European Ribosomal RNA Database. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0Lk_pX6hGUbZT7nb6z6G3TCEBiCrh0GR8orduJQHW&amp;reftype=extlink&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.psb.ugent.be/rRNA" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.psb.ugent.be/rRNA</a></span></li><li id="B41">NCBI Protein-protein BLAST. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0PwXFzWoSxxapqRsducxmYey4os681CcfVgi5c5Yh&amp;reftype=extlink&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/BLAST/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.ncbi.nlm.nih.gov/BLAST/</a></span></li><li id="B42">Preston, CM; Wu, KY; Molinski, TF; DeLong, EF. A psychrophilic crenarchaeon inhabits a marine sponge: <em>Cenarchaeum symbiosum </em>gen. nov., sp. nov. <span><span class="ref-journal">Proc Natl Acad Sci USA. </span>1996;<span class="ref-vol">93</span>:6241–6246. doi: 10.1073/pnas.93.13.6241.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0bk050veR_W4lPFIp2tilunQcZmPfXXaCwc2woAUQ&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8692799">PubMed</a>]</span></li><li id="B43">Mira, A; Pushker, R; Legault, BA; Moreira, D; Rodr&#x000ed;guez-Valera, F. Evolutionary relationships of <em>Fusobacterium nucleatum </em>based on phylogenetic analysis and comparative genomics. <span><span class="ref-journal">BMC Evol Biol. </span>2004;<span class="ref-vol">4</span>:50. doi: 10.1186/1471-2148-4-50.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FYbAraCcNs88K0HnY6N1kewn_dRDhAeQB5t8gQNq&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15566569">PubMed</a>]</span></li><li id="B44">Badger, JH; Eisen, JA; Ward, NL. Genomic analysis of <em>Hyphomonas neptunium </em>contradicts 16S rRNA-based phylogenetic analysis; implications for the taxonomy of the orders <em>Rhodobacterales </em>and <em>Caulobacterales</em>. <span><span class="ref-journal">Intl J Evol Syst Microb. </span>2005;<span class="ref-vol">55</span>:1021–1026. doi: 10.1099/ijs.0.63510-0.</span></li><li id="B45">Thompson, JD; Gibson, TJ; Plewniak, F; Jeanmougin, F; Higgins, DG. The CLUSTAL_X windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. <span><span class="ref-journal">Nucleic Acids Res. </span>1997;<span class="ref-vol">24</span>:4876–4882. doi: 10.1093/nar/25.24.4876.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0VIiOfvAhEJBTLZHHYA_uet-ZSyZkMyUgSoVf25GH&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9396791">PubMed</a>]</span></li><li id="B46">Saitou, N; Nei, M. The neighbor-joining method: a new method for reconstructing phylogenetic trees. <span><span class="ref-journal">Mol Biol Evol. </span>1987;<span class="ref-vol">4</span>:406–425.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0RONtAzPGW4zQwM_BhhsRapwyPQacxIyFV467AA_I&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/3447015">PubMed</a>]</span></li><li id="B47">Ataide, SF; Jester, BC; Devine, KM; Ibba, M. Stationary-phase expression and aminoacylation of a transfer-RNA-like small RNA. <span><span class="ref-journal">EMBO Rep. </span>2005;<span class="ref-vol">6</span>:742–747. doi: 10.1038/sj.embor.7400474.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0p3wH0XntrxXCo8-n7Q1wNQY8m_oncynM4Rq7hQEC&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16065067">PubMed</a>]</span></li><li id="B48">Guindon, S; Gascuel, O. A simple, fast and accurate algorithm to estimate large phylogenies by maximum likelihood. <span><span class="ref-journal">Syst Biol. </span>2003;<span class="ref-vol">52</span>:696–704. doi: 10.1080/10635150390235520.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0HwaLgUp8Z05sfkQvwRMfGgg5rIiGxQ1hoc0eDQoV&amp;reftype=pubmed&amp;artid=1475646&amp;article-id=1475646&amp;iid=126576&amp;issue-id=126576&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14530136">PubMed</a>]</span></li></ul></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">BMC Evolutionary Biology</span> are provided here courtesy of <br><b>BioMed Central</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc2-->

