<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="American Journal of Human Genetics"><meta name="citation_title" content="Linkage Disequilibrium Analysis of Biallelic DNA Markers, Human Quantitative Trait Loci, and Threshold-Defined Case and Control Subjects"><meta name="citation_authors" content="Nicholas J. Schork, Swapan K. Nath, Daniele Fallin, and  Aravinda Chakravarti"><meta name="citation_date" content="2000 November"><meta name="citation_issue" content="5"><meta name="citation_volume" content="67"><meta name="citation_firstpage" content="1208"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1288563&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1288563&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1288563"><meta name="citation_pmid" content="11032785"><meta name="DC.Title" content="Linkage Disequilibrium Analysis of Biallelic DNA Markers, Human Quantitative Trait Loci, and Threshold-Defined Case and Control Subjects"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Society of Human Genetics"><meta name="DC.Contributor" content="Nicholas J. Schork"><meta name="DC.Contributor" content="Swapan K. Nath"><meta name="DC.Contributor" content="Daniele Fallin"><meta name="DC.Contributor" content="Aravinda Chakravarti"><meta name="DC.Date" content="2000 November"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-green.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody10.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs5.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Linkage Disequilibrium Analysis of Biallelic DNA Markers, Human Quantitative Trait Loci, and Threshold-Defined Case and Control Subjects</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0PG-q4uFfyoiw3YtwHPAiwwX_Xl55kYE4GfDsDkbO&amp;reftype=other&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-ajhg.gif" alt="Logo of ajhg" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Journal URL: redirect3.cgi?&amp;&amp;auth=06Is_4OB7n5wo-ZtB9sqJf_nCZW8anWWYrKhO5Awd&amp;reftype=publisher&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article|Banner&amp;TO=Publisher|Other|N%2FA&amp;rendering-type=normal&amp;&amp;http://www.journals.uchicago.edu/AJHG/index.html" shape="rect" coords="0,0,500,100" href="redirect3.cgi?&amp;&amp;auth=0lEZVOxgy_EYdBVAFLFs5yKmI96xLkKR--XdS2Asb&amp;reftype=publisher&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.journals.uchicago.edu/AJHG/index.html" onclick="focuswin('pmc_ext')" target="pmc_ext"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-green"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=203&amp;action=archive">Am J Hum Genet</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=124318">v.67(5); Nov 2000</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=1288563&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=1288563&amp;blobtype=pdf" class="sidefm-pmclink">PDF (282K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=124318" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=203&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0v9Af6WJxikCPtkh06H7zG7Cbl17CmfsgksW4XkvL&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11032785">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=0h_ioTX6c338EuocfJq5rre8NgyOw9ATOHLek450Y&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=11032785">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0HwiaHNWCVCE29-GfLLkPLf8IYrbbloWBUT8IkkhK&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=11032785">PubMed LinkOut</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0ZgJ3dh2HOsUpxUs1HVHCg_sM4ATXKoqONpyE1AnF&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Schork+NJ%5Bauth%5D" class="sidefm-pmclink"> Schork, N.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0-kNtZnyAeju8oDvk-_TKv09i6nFr6gWkZjkHDQ0W&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Nath+SK%5Bauth%5D" class="sidefm-pmclink"> Nath, S.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0PifHxb1iYHlYaoowDhxGuh5WVpnP8narDmToSk66&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Fallin+D%5Bauth%5D" class="sidefm-pmclink"> Fallin, D.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0-gmVYcYC5maf2y2KLsjak3iwvw8P9TJ0KIKeM5kb&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Chakravarti+A%5Bauth%5D" class="sidefm-pmclink"> Chakravarti, A.</a></div></div></div></td><td rowspan="104" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-ajhg.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">Am J Hum Genet. </span><span class="citation-publication-date">2000 November; </span><span class="citation-volume">67</span><span class="citation-issue">(5)</span><span class="citation-flpages">: 1208–1218. </span></div><div><span class="fm-vol-iss-date">Published online 2000 October 13. </span><span class="fm-vol-iss-date"> </span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC1288563</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a> &#x000a9; 2000 by The American Society of Human Genetics. All rights reserved.</div><div class="fm-title">Linkage Disequilibrium Analysis of Biallelic DNA Markers, Human Quantitative Trait Loci, and Threshold-Defined Case and Control Subjects</div><div class="fm-author contrib-group">Nicholas J. Schork,<sup>1,</sup><sup>*</sup> Swapan K. Nath,<sup>2</sup> Daniele Fallin,<sup>2</sup> and  Aravinda Chakravarti<sup>3</sup><div class="fm-affl"><sup>1</sup>Department of Statistical Genomics, The Genset Corporation, La Jolla, CA; Departments of <sup>2</sup>Epidemiology and Biostatistics and <sup>3</sup>Genetics, Case Western Reserve University, Cleveland</div></div><div class="fm-footnote"></div><div class="fm-footnote">Address for correspondence and reprints: Dr. Nicholas J. Schork, Department of Epidemiology and Biostatistics, Case Western Reserve University, R215 Rammelkamp Building, MetroHealth Medical Center, 2500 MetroHealth Drive, Cleveland, OH 44109-1998. E-mail: <span class="e_id323087">njs2/at/po.cwru.edu</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id323087", '<a class="ext-reflink" href="' + reverseAndReplaceString('ude.urwc.op/ta/2sjn:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ude.urwc.op/ta/2sjn', '/at/','@') + '</a>')}catch(e){}
                                //--></script></div><div class="fm-footnote" id="FN1"><sup>*</sup>N.J.S. is on leave from the Department of Epidemiology and Biostatistics, Case Western Reserve University, Cleveland, OH; the Program for Population Genetics and the Department of Biostatistics, Harvard University School of Public Health, Boston, MA; and the Jackson Laboratory, Bar Harbor, ME.</div><div class="fm-pubdate">Received May 1, 2000; Accepted September 8, 2000.</div><div class="links-box"><div class="fm-footnote"><img src="corehtml/pmc/pmcgifs/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="tocrender.fcgi?artid=1288563&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337490" class="sidefm-pmclink" style="text-transform: none;">Introduction</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337607" class="sidefm-pmclink" style="text-transform: none;">Materials and Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338598" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338964" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id339896" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id337475">Abstract</div><div class="section-content"><!--article-meta-->Linkage disequilibrium (LD) mapping has been applied to many simple, monogenic, overtly Mendelian human traits, with great success. However, extensions and applications of LD mapping approaches to more complex human quantitative traits have not been straightforward. In this article, we consider the analysis of biallelic DNA marker loci and human quantitative trait loci in settings that involve sampling individuals from opposite ends of the trait distribution. The purpose of this sampling strategy is to enrich samples for individuals likely to possess (and not possess) trait-influencing alleles. Simple statistical models for detecting LD between a trait-influencing allele and neighboring marker alleles are derived that make use of this sampling scheme. The power of the proposed method is investigated analytically for some hypothetical gene-effect scenarios. Our studies indicate that LD mapping of loci influencing human quantitative trait variation should be possible in certain settings. Finally, we consider possible extensions of the proposed methods, as well as areas for further consideration and improvement.</div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337475" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Introduction</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337607" class="sidefm-pmclink" style="text-transform: none;">Materials and Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338598" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338964" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id339896" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><span id="SC1"></span><div class="head1 section-title" style="text-transform: none;" id="id337490"><a id="SC1"></a>Introduction</div><div class="section-content"><span id="SC1"></span><p>Recent technological advances in molecular genetics have provided researchers with extremely powerful tools that they can use to probe the genetic basis of traits and diseases. Although there are many different strategies for exploiting these technologies, one that has been receiving considerable recent attention is the association study. The association study involves a simple comparison of the frequencies of an allele or haplotype between individuals with and without a trait of interest. If compelling evidence for frequency differences exist, then either the locus (or loci) in question harbors alleles that causally or directly influence the trait, or the alleles are in linkage disequilibrium (LD) with alleles at a neighboring locus that directly influence the trait in question.</p><p>When testing a particular locus and its alleles for association with a trait or disease, one will rarely know, in the absence of ancillary information, whether or not an association is likely to arise from causality or LD. In fact, mapping trait-influencing loci under the assumption that LD patterns can reveal the approximate location of a trait-influencing gene has become an important and commonly used strategy in positional cloning and association mapping in general (see, e.g., Jorde <a href="#RF12" class="cite-reflink">1995</a>). Unfortunately, the success of LD mapping has been confined largely to monogenic, overtly Mendelian traits. Part of the reason for the lack of success in exploitation of LD mapping strategies for more-complex traits is the lack of analysis methods and study designs that can accommodate their multifactorial nature or can extract as much association information as possible from a sample. This is particularly true for quantitative or metrical traits, such as cholesterol or blood pressure level&#x02014;that is, those traits that vary continuously in the population at large, since they are typically influenced by a number of genetic and nongenetic factors, the individual effects of which are often obscured by the effects of the others. An additional issue that plagues LD-mapping studies of quantitative traits is population stratification and cryptic heterogeneity, which, fortunately, can be dealt with for analyses involving quantitative traits in a manner analogous to the methods used for qualitative traits (see, e.g., Devlin and Roeder [<a href="#RF8" class="cite-reflink">1999</a>], Pritchard and Rosenberg [<a href="#RF19" class="cite-reflink">1999</a>], and Pritchard et al. [<a href="#RF20" class="cite-reflink">2000</a>] for discussion).</p><p>One strategy for extracting as much information from a sample for a quantitative trait as possible is to derive that sample from the extremes of the trait distribution in question (see, e.g., Gu et al. <a href="#RF11" class="cite-reflink">1997</a>, Risch and Zhang <a href="#RF22" class="cite-reflink">1995</a>, and Xu et al. <a href="#RF32" class="cite-reflink">1999</a>). In this study, we consider conducting a single-locus association analysis between a biallelic marker locus and individuals sampled at opposite ends of a quantitative trait distribution. We derive equations for assessment of the power of the proposed method as a function of many factors. An appendix offers a description of the notation used. Our results suggest that single-locus LD mapping using the proposed sampling strategy can be quite powerful in certain settings. We also discuss possible extensions of the proposed method, as well as some of its limitations.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337475" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337490" class="sidefm-pmclink" style="text-transform: none;">Introduction</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Materials and Methods</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338598" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338964" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id339896" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><span id="SC2"></span><div class="head1 section-title" style="text-transform: none;" id="id337607"><a id="SC2"></a>Materials and Methods</div><div class="section-content"><span id="SC2"></span><p><span id="SC2.1"></span><div class="head2 head-separate">Basic Mixture Model</div>Consider a locus with two alleles, denoted as &#x0201c;+&#x0201d; and &#x0201c;&#x02212;,&#x0201d; that influence a quantitative phenotype. There are three possible (diploid) genotypes, ++, +&#x02212;, and &#x02212;&#x02212;. Associated with each of these genotypes is a phenotypic mean effect, &#x003bc;<sub><em>g</em></sub>, and variance &#x003c3;<sup>2</sup><sub><em>g</em></sub> (where <img src="picrender.fcgi?artid=1288563&amp;blobtype=equ&amp;blobname=M1" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M1" title="equation M1">). For simplicity&#x02019;s sake, we assume that &#x003c3;<sup>2</sup><sub>--</sub>=&#x003c3;<sup>2</sup><sub>-+</sub>=&#x003c3;<sup>2</sup><sub>++</sub>=&#x003c3;<sup>2</sup><sub><em>r</em></sub>, although this assumption may be problematic for marker-locus alleles merely linked to a quantitative trait locus (QTL), as discussed later. The variation in trait values, <em>x,</em> among individuals with the same genotype, <em>g,</em> is assumed to be characterized by the normal density function, denoted <img src="corehtml/pmc/pmcents/x03C6.gif" border="0" alt="[var phi]" />&#x00028;<em>x</em>&#x0007c;&#x003bc;<sub><em>g</em></sub>,&#x003c3;<sup>2</sup><sub><em>g</em></sub>&#x00029; .<p>Let <em>p</em> be the frequency of the &#x0201c;+&#x0201d; allele and <em>q</em>=&#x00028;1-<em>p</em>&#x00029; be the frequency of the &#x0201c;&#x02212;&#x0201d; allele. If Hardy-Weinberg equilibrium of the alleles is assumed, the frequencies of the three genotypes are as follows: <em>f</em><sub>++</sub>=<em>p</em><sup>2</sup>, <em>f</em><sub>-+</sub>=2<em>pq</em>, and <em>f</em><sub>--</sub>=<em>q</em><sup>2</sup>. The population-level variation of <em>x,</em> then, can be described as a mixture distribution, denoted as &#x0201c;&#x003c1;&#x00028;&#x02022;&#x00029;.&#x0201d; Assume equality of genotype-specific trait variances, let <img src="picrender.fcgi?artid=1288563&amp;blobtype=equ&amp;blobname=M2" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M2" title="equation M2">, and then let<a id="DF12" name="DF12"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><div><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208df12.jpg" alt="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow. The name of referred object is AJHGv67p1208df12.jpg" border="0" title="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow."></div><div style="clear:both;"></div></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>See MacLean et al. (<a href="#RF15" class="cite-reflink">1976</a>) and Schork et al. (<a href="#RF27" class="cite-reflink">1996</a>) for discussions. To simplify things even further, consider assigning &#x003bc;<sub>--</sub>=-<em>a</em>, &#x003bc;<sub>-+</sub>=<em>d</em>, &#x003bc;<sub>++</sub>=<em>a</em>, and &#x003c3;<sup>2</sup><sub><em>r</em></sub>=1. Then, a model for the dominance of the + allele over the &#x02212; allele would assume <em>d</em>=<em>a</em>, a model for recessivity of the + allele would assume <em>d</em>=-<em>a</em>, a purely additive model would assume <em>d</em>=0, and assumptions whereby -<em>a</em>&#x0003c;<em>d</em>&#x0003c;<em>a</em> would provide models suggestive of semidominance. The additive genetic variance attributable to the locus for any model can thus be calculated as &#x003c3;<sup>2</sup><sub><em>a</em></sub>=2<em>pq</em>&#x0005b;<em>a</em>-<em>d</em>&#x00028;<em>p</em>-<em>q</em>&#x00029;&#x0005d;<sup>2</sup>, the dominance variance can be computed as &#x003c3;<sup>2</sup><sub><em>d</em></sub>=&#x0005b;2<em>pqd</em>&#x0005d;<sup>2</sup>, and the total genetic variance can be computed as &#x003c3;<sup>2</sup><sub><em>G</em></sub>=&#x003c3;<sup>2</sup><sub><em>a</em></sub>+&#x003c3;<sup>2</sup><sub><em>d</em></sub>. Given that &#x003c3;<sup>2</sup><sub><em>r</em></sub>=1, the broad-sense heritability attributable to the locus is <em>H</em><sub><em>B</em></sub>=&#x003c3;<sup>2</sup><sub><em>G</em></sub>/&#x00028;&#x003c3;<sup>2</sup><sub><em>G</em></sub>+1&#x00029;, whereas the narrow sense heritability is <em>H</em><sub><em>N</em></sub>=&#x003c3;<sup>2</sup><sub><em>a</em></sub>/&#x00028;&#x003c3;<sup>2</sup><sub><em>G</em></sub>+1&#x00029;. Note that the total variance for the trait, for arbitrary &#x003c3;<sup>2</sup><sub><em>r</em></sub>, is &#x003c3;<sup>2</sup><sub><em>t</em></sub>=&#x003c3;<sup>2</sup><sub><em>G</em></sub>+&#x003c3;<sup>2</sup><sub><em>r</em></sub>.</p></p><p><span id="SC2.2"></span><div class="head2 head-separate">Sampling Extremes</div>We will now consider the sampling of individuals from the ends of the trait distribution to maximize the probability of obtaining individuals with and without the + (&#x02212;) allele. For the sake of convenience, assume that interest is in the allele associated with higher trait values. We consider thresholds for sampling that will define &#x0201c;case subjects&#x0201d; (i.e., individuals in the upper end of the trait distribution) and &#x0201c;control subjects&#x0201d; (i.e., individuals in the lower end of the trait distribution). To define case subjects, we consider individuals whose trait value is in the upper &#x003b1;<sub><em>u</em></sub> percentile of the trait distribution. Control subjects are considered individuals with trait values in the lower &#x003b1;<sub><em>l</em></sub> percentile of the trait distribution. Trait values that must either be surpassed, &#x003c4;<sub><em>u</em></sub>, for case-subject assignment or not surpassed, &#x003c4;<sub><em>l</em></sub>, for control-subject assignment can be obtained by solving the integrals<a id="DF400" name="DF400"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><div><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208df400.jpg" alt="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow. The name of referred object is AJHGv67p1208df400.jpg" border="0" title="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow."></div><div style="clear:both;"></div></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>and<a id="DF300" name="DF300"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><div><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208df300.jpg" alt="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow. The name of referred object is AJHGv67p1208df300.jpg" border="0" title="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow."></div><div style="clear:both;"></div></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>Let <em>P</em> denote a probability, such that <em>P</em><sub><em>v</em></sub> with a subscript denotes a specific probability, and <em>P()</em> denotes a probability function that can be evaluated at a certain point. From equation <a href="#DF12" class="cite-reflink">(1)</a>, the conditional probability of possessing the + allele, given that an individual is a case subject (i.e., has a trait value that surpasses the threshold &#x003c4;<sub><em>u</em></sub>), can be computed from Bayes&#x02019; rule:<a id="DF36" name="DF36"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><div><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208df36.jpg" alt="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow. The name of referred object is AJHGv67p1208df36.jpg" border="0" title="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow."></div><div style="clear:both;"></div></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>Given control-subject status, similar conditional probabilities can be computed for the possession of the + allele.</p><p><span id="SC2.100"></span><div class="head2 head-separate">Conditional Marker Frequencies: Single-Locus</div>Consider a marker locus with two alleles, <em>M</em> and <em>m,</em> that is linked to a locus that influences a quantitative trait for which the sampling strategy described in the previous section has been applied. Let the <em>M</em> allele be in disequilibrium with the + allele at the trait locus. Further, let <em>s</em> be the frequency of the <em>M</em> allele and <em>t</em>=&#x00028;1-<em>s</em>&#x00029; be the frequency of the <em>m</em> allele. Using standard equations, the frequency of the four possible two-locus haplotypes across the trait and marker loci are given by<a id="DF38" name="DF38"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><div><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208df38.jpg" alt="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow. The name of referred object is AJHGv67p1208df38.jpg" border="0" title="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow."></div><div style="clear:both;"></div></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>where &#x003b4; is the disequilibrium strength value between alleles at the two loci. After some simple algebra, it can be shown that the frequency of the <em>M</em> allele among individuals sampled from the upper end of the trait distribution is given by<a id="DF40" name="DF40"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><div><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208df40.jpg" alt="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow. The name of referred object is AJHGv67p1208df40.jpg" border="0" title="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow."></div><div style="clear:both;"></div></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>Similar equations can be derived for the frequency with which an individual sampled from the lower end of the trait distribution will carry the <em>M</em> allele, <em>P</em><sub><em>M</em>&#x0007c;<em>l</em></sub>. Equations of this type have also been derived by Slatkin (<a href="#RF29" class="cite-reflink">1999</a>) and Neilsen and colleagues (Nielsen et al. <a href="#RF17" class="cite-reflink">1998</a>; Nielsen and Weir <a href="#RF18" class="cite-reflink">1999</a>), in slightly different contexts.</p><p><span id="SC2.3"></span><div class="head2 head-separate">Testing LD: Power Considerations</div>The derivations above make it relatively easy to pursue power studies for test settings involving different marker and trait allele frequencies, interlocus distances, and LD strengths. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a> depicts a simple 2 &#x000d7; 2 contingency table that can be set up to assess the association between the marker-locus alleles and case-/control-subject status.<div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="TB1" name="TB1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 1" title="Table 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 1</strong></a><div class="figure-table-caption-in-article"><span>Basic Design for Investigating the Association Between a Marker Allele (or Haplotype) and Threshold-Defined Case and Control Subjects</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>For present purposes, the statistic of interest that can be derived from the 2 &#x000d7; 2 table is the odds ratio<a id="DF42" name="DF42"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><div><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208df42.jpg" alt="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow. The name of referred object is AJHGv67p1208df42.jpg" border="0" title="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow."></div><div style="clear:both;"></div></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>Schlesselman (<a href="#RF25" class="cite-reflink">1982</a>) discusses calculations for assessing the power of tests of the hypothesis <em>H</em><sub>0</sub>:<em>OR</em>=1 (i.e., the marker locus is not in LD with the trait locus, or the locus being tested for LD does not have an effect on the trait of interest). In particular, if <em>n</em><sub><em>u</em></sub> is the number of case subjects in the study, <em>n</em><sub><em>l</em></sub>=<em>cn</em><sub><em>u</em></sub> is the number of control subjects (i.e., <em>c</em> is the control-subject:case-subject ratio and the total number of subjects is <em>N</em>=<em>n</em><sub><em>u</em></sub>+<em>n</em><sub><em>l</em></sub>), <img src="picrender.fcgi?artid=1288563&amp;blobtype=equ&amp;blobname=M3" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M3" title="equation M3">, <img src="picrender.fcgi?artid=1288563&amp;blobtype=equ&amp;blobname=M4" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M4" title="equation M4">, <em>z</em><sub>&#x003b1;</sub> is the quantile associated with a standard normal distribution for the (type I error) probability &#x003b1;, and<a id="DF51" name="DF51"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><div><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208df51.jpg" alt="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow. The name of referred object is AJHGv67p1208df51.jpg" border="0" title="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow."></div><div style="clear:both;"></div></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>then, assuming relevant parameters have been set to hypothesized values, &#x003a9;=&#x0007b;<em>z</em><sub>&#x003b1;</sub>,<em>n</em><sub><em>u</em></sub>,<em>c</em>,&#x003bc;<sub>--</sub>,&#x003bc;<sub>-+</sub>,&#x003bc;<sub>++</sub>,&#x003c3;<sup>2</sup>,<em>p</em>,<em>s</em>,&#x003b4;&#x0007d;, power can be calculated as<a id="DF200" name="DF200"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><div><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208df200.jpg" alt="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow. The name of referred object is AJHGv67p1208df200.jpg" border="0" title="A mathematical equation, expression, or formula that is to be displayed as a block (callout) within the narrative flow."></div><div style="clear:both;"></div></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>where <img src="corehtml/pmc/pmcents/x03C6.gif" border="0" alt="[var phi]" />&#x00028;<em>x</em>&#x0007c;0,1&#x00029; is the standard normal-density function evaluated at value <em>x.</em> Thus, given assumptions about a number of parameters&#x02014;the trait-locus allele frequencies, locus-specific heritability and dominance effects, LD strength, marker-locus allele frequencies, thresholds for defining case-/control-subject status, number of case subjects, ratio of control to case subjects, and type I&#x02013;error rate&#x02014;one can compute the power to detect an LD-induced association between a biallelic marker locus and a QTL through tests of <em>OR.</em></p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337475" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337490" class="sidefm-pmclink" style="text-transform: none;">Introduction</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337607" class="sidefm-pmclink" style="text-transform: none;">Materials and Methods</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Results</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338964" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id339896" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><span id="SC3"></span><div class="head1 section-title" style="text-transform: none;" id="id338598"><a id="SC3"></a>Results</div><div class="section-content"><span id="SC3"></span><p>In this section, we consider some computations involving the equations and derivations described in the previous section. We also showcase the effects of various parameters and assumptions on power to detect a locus effect. We perform the calculations in a few hypothetical situations, with the understanding that the reader may be interested in some unique or specific situations not overtly addressed in this paper. A computer program that can carry out the relevant calculations is available from the authors.</p><p><span id="SC3.1"></span><div class="head2 head-separate">Conditional Marker Allele Probabilities</div><a href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB2" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 2</a> offers some examples of calculations using equations 1&#x02013;6 and ultimately gives the conditional probability that an individual sampled from the upper and lower ends of the trait distribution possesses an allele in LD with a trait-influencing allele. Note that when there is no dominance and the alleles are equally frequent (i.e., the value of <em>p</em>&#x02014;that is, the column listed as <em>p</em>&#x02014;is 0.5, and the value of <em>d</em> is 0), the quantiles used to define the upper and lower percentiles of the trait distribution are equal in absolute value, as expected.<div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="TB2" name="TB2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB2" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 2" title="Table 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 2</strong></a><div class="figure-table-caption-in-article"><span>Conditional Probability that an Individual Possesses a Trait-Value-Increasing Allele, Given that His or Her Trait Value Is in the Lower and Upper Percentiles of the Trait Distribution and under the Assumption of Different Trait-Locus Allele Effects<sup>[Note]</sup></span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span id="SC3.2"></span><div class="head2 head-separate">Basic Sample-Size-Requirement Studies</div><a href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB3" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 3</a> offers some examples of sample-size-requirement calculations for some select assumptions about the trait and marker-locus allele frequencies, dominance and locus effects, and LD strength. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB3" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 3</a> makes it clear that if the trait-locus effect is modest (e.g., ~20% of the variation explained) and/or the trait-locus allele frequency matches the associated marker-locus allele frequency, then it might be possible to detect an effect with realistic sample sizes even at genomewide rates. These are meant as examples only, since there are an infinite number of situations one might want to consider in terms of power. It is important, however, to consider the impact that assumptions about the potential locus effect&#x02014;and, more importantly, the sampling scheme&#x02014;have on power. We focus on the effects of some of these parameters in isolation in the sections that follow.<div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="TB3" name="TB3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB3" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 3" title="Table 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 3</strong></a><div class="figure-table-caption-in-article"><span>Sample Sizes Necessary to Detect an Association between a Marker Locus and a Trait-Influencing Locus Assuming Different Values for the Trait-Locus Allele Frequencies, Locus Effect Sizes, and LD Strength<sup>[Note]</sup></span></div></td></tr></table></div></div><div style="clear:both;"></div><p><span id="SC3.2.1"></span><span class="head3">Influence of the definition of &#x0201c;extremes&#x0201d; </span> By sampling more and more extreme individuals, one can increase the power of an association study in certain instances. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1</a> depicts power curves with the assumption that individuals have been sampled in a symmetrical way from the upper and lower percentiles of a trait distribution. Four different settings were studied with respect to sample size and locus effect (i.e., sample sizes of 100 and 50 case and control subjects, respectively, and locus-specific heritabilities of .1 and .25, respectively). It was assumed that the trait-locus effect was dominant, with the dominant allele having a frequency of .25 and an associated marker-locus allele frequency of .25. The marker and trait alleles were assumed to be in LD at 75% of the maximum for loci with the specified allele frequencies (assuming Lewontin&#x02019;s D&#x02032; as scaled to a maximum achievable LD given allele frequencies [Lewontin <a href="#RF13" class="cite-reflink">1988</a>]). A type I&#x02013;error rate of .05 was also assumed. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1</a> clearly shows that sampling more extreme individuals results in greater power. However, for the case subjects examined, the drop-off in power is not large if the sample is large or the locus effect is pronounced.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="FG1" name="FG1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208fg1.gif" border="1" class="icon-reflink" alt="Figure  1" title="Figure  1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure  1</strong></a><div class="figure-table-caption-in-article"><span>Effect of sampling thresholds on the power to identify a QTL via association mapping. Simulating conditions were: <em>D</em>&#x02032; between trait/marker loci = 0.75, <em>p</em><sub>M</sub> = 0.25, and <em>p</em><sub>+</sub> =.025, when the dominant model is assumed. Type I&#x02013;error rate was set</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span id="SC3.2.2"></span><span class="head3">Influence of the locus effect </span> Obviously, the larger the trait-influencing locus effect is, the easier it will be to detect an association between that locus and a marker-locus allele. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2</a> depicts power curves for trait-locus&#x02013;specific heritabilities ranging from .0 to .5 for samples of sizes 25, 50, 100, and 250 case and control subjects. The trait-locus allele was assumed to be dominant with a .25 frequency, and the associated marker-locus allele was assumed to have a frequency of .25. The marker and trait alleles were assumed to be in LD at 75% of the maximum for loci with the specified allele frequencies. Case and control subjects were assumed to be drawn from the upper and lower 25% of the trait distribution, respectively. A type I&#x02013;error rate of .05 was also assumed. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2</a> clearly shows that as the locus effect increases, the power increases greatly. Thus, even with relatively small sample size (~50 case and control subjects), the power to detect a locus with a moderate effect, given the assumed sampling scheme, is quite good.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="FG2" name="FG2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208fg2.gif" border="1" class="icon-reflink" alt="Figure  2" title="Figure  2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure  2</strong></a><div class="figure-table-caption-in-article"><span>Effect of the heritability of a QTL on the power to identify that locus via association mapping</span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span id="SC3.2.3"></span><span class="head3">Influence of LD strength </span> Clearly, if a marker locus and a trait-influencing locus do not have alleles in LD, then the marker-locus alleles will not act as good &#x0201c;surrogates&#x0201d; for the trait-influencing alleles. Thus, the strength of the LD between the marker and the trait-locus alleles is of extreme importance in detecting an association. It is important, therefore, to consider just how strong LD has to be before one can detect an association. Since the frequency of the alleles can have an impact on both LD strength and one&#x02019;s ability to detect it, we have chosen to exemplify the influence of LD strength on mapping power by fixing the sampling strategy and locus effect parameters and varying LD strength and heritability. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 3</a> depicts the expected increase in power with increasing LD between the marker and trait loci. The trait-locus allele was assumed to be dominant with a .25 frequency, and the associated marker-locus allele was assumed to have a frequency of .25. Case and control subjects were assumed to be drawn from the upper and lower 25% of the trait distribution. A type I&#x02013;error rate of .05 was assumed.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="FG3" name="FG3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208fg3.gif" border="1" class="icon-reflink" alt="Figure  3" title="Figure  3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure  3</strong></a><div class="figure-table-caption-in-article"><span>Effect of LD strength between marker and trait loci on the power to identify a QTL via association mapping</span></div></td></tr></table></div></div><div style="clear:both;"></div></p></p><p><span id="SC3.3"></span><div class="head2 head-separate">Influence of Trait and Marker Loci Allele Frequencies</div>The single-locus test described here relies on the LD between the marker and trait loci, as shown in <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">figure 3</a>. Because LD strength is dependent on allele frequencies, it is also of interest to measure the simultaneous effects of marker and trait allele frequencies on power of the extreme sampling method. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 4</a> shows power as a function of SNP marker allele frequency for trait allele frequencies of .1, .2, .3, .4, and .5, when the disequilibrium between them is held constant at 75%, the maximum possible for the given frequencies. One hundred case subjects and 100 control subjects sampled from the upper and lower 10th percentiles of the trait distribution were assumed, as was a locus-specific heritability of 20%. The type I&#x02013;error rate was set to .05. Although this plot demonstrates reasonable power for all of the trait allele frequencies, because of the high level of LD, it can be seen that the maximum power is achieved when the trait and marker allele frequencies are equal. This is intuitive and has implications for association studies, as described in the <a href="#SC4" class="cite-reflink">Discussion</a> section. In addition, this issue has been addressed by others, in slightly different contexts (Schaid and Sommer <a href="#RF24" class="cite-reflink">1994</a>; McGinnis <a href="#RF16" class="cite-reflink">1998</a>; Schaid and Rowland <a href="#RF23" class="cite-reflink">1998</a>).<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="FG4" name="FG4"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG4" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208fg4.gif" border="1" class="icon-reflink" alt="Figure  4" title="Figure  4"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure  4</strong></a><div class="figure-table-caption-in-article"><span>Effect of quantitative trait and marker-locus allele frequencies on the power to identify the QTL via association mapping.</span></div></td></tr></table></div></div><div style="clear:both;"></div><p><span id="SC3.3.1"></span><span class="head3">Influence of the definition of &#x0201c;control subject&#x0201d; </span> Often, in case/control studies, a researcher will sample individuals with a disease and then merely define control subjects as individuals without the disease. This definition of &#x0201c;control subject&#x0201d; can create a very heterogeneous group. As has been emphasized throughout this paper, for a quantitative phenotype, one can select control subjects who may lead to increases in mapping power, because they are less similar to the case subjects with respect to phenotype (i.e., their trait values are further removed from the case subjects&#x02019; trait values, making it less likely that they share genes for that trait). <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG5" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 5</a> displays, in two ways, the effect of varying definitions of &#x0201c;control subject.&#x0201d; <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG5" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 5</a><em>A</em> shows the increase in power with increasing number of control subjects, while keeping the other parameters constant. In this scheme, 100 case subjects were taken from the upper 25% of the trait distribution, and the varying number of control subjects was assumed to be sampled from the lower 25%. Under the conditions shown, reasonable power can be obtained from a control-subject/case-subject ratio &#x0003c;1, even for low heritability values. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG5" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 5</a><em>B</em> demonstrates the effect of varying the lower threshold of the trait distribution used for control subject sampling, while keeping the number of case subjects to control subjects constant at 100. One hundred case subjects were assumed to be sampled from the top 25% of trait distribution, the trait-locus allele frequency was assumed to be .25, the associated marker-locus allele frequency was also assumed to be .25, and the LD between the alleles was assumed to be 75% of its maximum and the trait-locus allele was assumed to be dominant. The type I&#x02013;error rate was set to .05. <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG5" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 5</a> shows that power decreases as less extreme control-subject&#x02013;sampling thresholds are used. However, even for low heritability, the power for sampling control subjects from the lower half of the trait distribution is still quite good under the conditions studied.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="FG5" name="FG5"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG5" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1288563&amp;blobname=AJHGv67p1208fg5.gif" border="1" class="icon-reflink" alt="Figure  5" title="Figure  5"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG5" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure  5</strong></a><div class="figure-table-caption-in-article"><span>Effect of the definition of a &#x0201c;control subject&#x0201d; on the power to identify a QTL via association mapping. <em>A,</em> Power as a function of the number of control subjects sampled while keeping case-subject sample size and sampling percentiles constant</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG5" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337475" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337490" class="sidefm-pmclink" style="text-transform: none;">Introduction</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337607" class="sidefm-pmclink" style="text-transform: none;">Materials and Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338598" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Discussion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id339896" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><span id="SC4"></span><div class="head1 section-title" style="text-transform: none;" id="id338964"><a id="SC4"></a>Discussion</div><div class="section-content"><span id="SC4"></span><p>Association mapping, although not without its problems, is enjoying a tremendous resurgence of interest because of the availability of polymorphic marker databases, high-throughput genotyping equipment, and a recognition of the limits of conventional linkage analysis mapping strategies for identifying the determinants of common complex diseases (Risch and Merikangas <a href="#RF21" class="cite-reflink">1996</a>; Collins et al. <a href="#RF6" class="cite-reflink">1997</a>, <a href="#RF7" class="cite-reflink">1998</a>). A predominant issue that has arisen in the wake of this intense interest concerns the manner in which one should conduct an association study, especially with respect to quantitative traits. For example, many researchers have devised analogs of the standard transmission-disequilibrium test (TDT) for quantitative traits (see, e.g., Allison <a href="#RF1" class="cite-reflink">1997</a>) with the hope that the advantages associated with the TDT (i.e., a control for population stratification) could be exploited (Spielman et al. <a href="#RF30" class="cite-reflink">1993</a>; Ewens and Spielman <a href="#RF9" class="cite-reflink">1995</a>; Spielman and Ewens <a href="#RF31" class="cite-reflink">1996</a>). Others have developed models for association analysis of quantitative traits that make use of fixed and random effects models for family-based collections (Boerwinkle et al. <a href="#RF4" class="cite-reflink">1986</a>; George and Elston <a href="#RF10" class="cite-reflink">1987</a>; Amos et al. <a href="#RF3" class="cite-reflink">1996</a>). These models have the clear advantage of allowing for the control or accommodation of residual genetic and familial effects on the trait. In addition, the sampling units&#x02014;families or pedigrees&#x02014;can be used in complementary linkage analysis studies as well (Schork <a href="#RF26" class="cite-reflink">2000</a>).</p><p>Unfortunately, the problem with both the TDT and random-effects&#x02013;models approach is that they require family or (at least) parental information. Such information may be difficult or impossible to collect. Although one conceivably could collect a random sample of unrelated individuals and examine associations between a quantitative trait and marker-locus alleles, using analysis of variance and related statistical procedures, these strategies would not be optimal. The proposed sampling scheme, involving unrelated individuals sampled from thresholds defined by the trait distribution, is intuitive and can result in substantial power increases. In addition, many of the problems thought to plague case/control samples, such as stratification and cryptic heterogeneity, can be overcome with an appropriate use of DNA markers and analysis strategies (Pritchard et al. <a href="#RF20" class="cite-reflink">2000</a>; Schork et al., <a href="#RF28" class="cite-reflink">in press</a>). Unfortunately, there are some drawbacks, both with the proposed method and with our derivations concerning its power. First, our derivations require knowledge of the trait distribution, so that relevant sampling thresholds can be defined. Rarely will one know the actual distribution of trait values in the population. However, large epidemiological studies often can estimate such distributions and therefore can provide approximate thresholds.</p><p>Second, our calculations assume that the trait values were distributed as normal variates, with constant variances across the genotype categories. It is unlikely that a trait will exhibit such homoscedasticity across genotype categories. It is also unlikely that a trait will exhibit perfect normality. This is especially true if multiple loci influence the trait of interest. Such multilocus influences can easily affect the power to detect a locus effect with simple sampling frameworks (Allison et al. <a href="#RF2" class="cite-reflink">1998</a>). We view our assumptions of normality and homoscedasticity as working assumptions. We encourage others to investigate alternatives.</p><p>Third, although we concentrated on single-locus associations, we recognize that haplotype and multilocus analyses might be more powerful. Haplotype and multilocus analyses may be able to exploit LD relationships among multiple markers and thereby make up for weak LD between any marker allele and the trait-influencing allele (Long et al. <a href="#RF14" class="cite-reflink">1995</a>; Clark et al. <a href="#RF5" class="cite-reflink">1998</a>; Schork et al., <a href="#RF28" class="cite-reflink">in press</a>). Unfortunately, power calculations assuming multiple haplotypes would be complicated to pursue. We therefore leave the details of such studies to further research.</p><p>One interesting facet of our study results concerns the effect of marker and trait-locus allele frequencies (e.g., <a href="articlerender.fcgi?artid=1288563&amp;rendertype=table&amp;id=TB3" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">table 3</a> and <a href="articlerender.fcgi?artid=1288563&amp;rendertype=figure&amp;id=FG4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">fig. 4</a>). It is clear that, in certain situations, power increases in mapping will occur if the trait-influencing allele and associated marker-locus allele have the same frequency. This is likely due to the fact that if, for example, the associated marker-locus allele frequency is much greater than the trait-locus allele frequency, there will be many control subjects possessing the associated allele. This will obviously weaken evidence for an association, especially if the LD is weak between the trait and marker-locus alleles. The implications of this phenomenon for mapping studies are far-reaching. Consider the development and use of a map of markers that have similar allele frequencies. Such a map might not be ideal for detecting associations with alleles that have different allele frequencies from those of the markers. Overcoming this problem may be possible by studying haplotypic associations, which might provide greater specificity and matching of disease-allele frequencies. Obviously, greater research into this and related issues are needed.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id339229">Acknowledgments</div><div class="section-content"><p><p>The authors would like to thank Hemant Tiwari for reading the manuscript. D.F. and S.K.N. are supported, in part, by National Institutes of Health (NIH) grants HL54998-01 and RR03655-11, awarded to N.J.S. A.C. is also supported in part by NIH grant HL54998-01.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><span id="AP1"></span><div class="head1 section-title" style="text-transform: none;" id="id339239"><a id="AP1"></a>Appendix: Notation</div><div class="section-content"><p><div><div class="p"><strong></strong></div><div class="p"> <div class="p"><p></p></div></div></div><div><table cellspacing="3" cellpadding="2" frame="hsides" rules="groups" class="rendered"><tbody><tr><td align="left" valign="top"><em>x</em></td><td align="left" valign="top">Trait value</td></tr><tr><td align="left" valign="top">&#x02212;, +</td><td align="left" valign="top"> Alleles at the trait-influencing locus</td></tr><tr><td align="left" valign="top">&#x003bc;<sub><em>g</em></sub></td><td align="left" valign="top">Mean genotype effect: <img src="picrender.fcgi?artid=1288563&amp;blobtype=equ&amp;blobname=M5" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M5" title="equation M5"></td></tr><tr><td align="left" valign="top">&#x003c3;<sup>2</sup><sub><em>g</em></sub></td><td align="left" valign="top">Variance of trait values associated with genotype <em>g</em></td></tr><tr><td align="left" valign="top">&#x003c3;<sup>2</sup><sub><em>r</em></sub></td><td align="left" valign="top">Variance of trait values, assuming equality of genotype variances</td></tr><tr><td align="left" valign="top">&#x003c3;<sup>2</sup><sub><em>G</em></sub></td><td align="left" valign="top">Total trait variance caused by genetic effects at single locus</td></tr><tr><td align="left" valign="top">&#x003c3;<sup>2</sup><sub><em>a</em></sub></td><td align="left" valign="top">Total trait variance caused by additive genetic effects at a single locus</td></tr><tr><td align="left" valign="top">&#x003c3;<sup>2</sup><sub><em>d</em></sub></td><td align="left" valign="top">Total trait variance caused by dominance genetic effects at a single locus</td></tr><tr><td align="left" valign="top"><em>p</em>, <em>q</em></td><td align="left" valign="top">Frequencies of the + and &#x02212; alleles, respectively</td></tr><tr><td align="left" valign="top"><em>f</em><sub><em>g</em></sub></td><td align="left" valign="top">Frequency of trait-influencing genotype <em>g</em></td></tr><tr><td align="left" valign="top">&#x003c1;&#x00028;&#x02022;&#x00029;</td><td align="left" valign="top">Trait distribution in the population</td></tr><tr><td align="left" valign="top"><em>a</em></td><td align="left" valign="top">+ allele effect</td></tr><tr><td align="left" valign="top"><em>d</em></td><td align="left" valign="top">Dominance effect</td></tr><tr><td align="left" valign="top"><em>H</em><sub><em>B</em></sub></td><td align="left" valign="top">Broad-sense heritability</td></tr><tr><td align="left" valign="top"><em>H</em><sub><em>N</em></sub></td><td align="left" valign="top">Narrow-sense heritability</td></tr><tr><td align="left" valign="top">&#x003c4;<sub><em>u</em></sub>, &#x003c4;<sub><em>l</em></sub></td><td align="left" valign="top">Threshold values for defining case and control subjects, respectively</td></tr><tr><td align="left" valign="top">&#x003b1;<sub><em>u</em></sub>, &#x003b1;<sub><em>l</em></sub></td><td align="left" valign="top">Percentiles for sampling case and control subjects, respectively</td></tr><tr><td align="left" valign="top"><em>M, m</em></td><td align="left" valign="top">Alleles at the marker locus</td></tr><tr><td align="left" valign="top"><em>s, t</em></td><td align="left" valign="top">Frequencies of the <em>M</em> and <em>m</em> alleles, respectively</td></tr><tr><td align="left" valign="top">&#x003b4;</td><td align="left" valign="top">LD strength between trait and marker alleles.</td></tr><tr><td align="left" valign="top"><em>P</em><sub><em>M</em>&#x0007c;<em>u</em></sub></td><td align="left" valign="top">Probability of possessing a marker allele given case-subject status</td></tr><tr><td align="left" valign="top"><em>n</em><sub><em>u</em></sub>, <em>n</em><sub><em>l</em></sub></td><td align="left" valign="top">Number of case and control subjects, respectively</td></tr><tr><td align="left" valign="top"><em>N</em></td><td align="left" valign="top">Total number of case and control subjects</td></tr><tr><td align="left" valign="top"><em>c</em></td><td align="left" valign="top">Ratio of case subjects and control subjects</td></tr><tr><td align="left" valign="top">OR </td><td align="left" valign="top">Odds ratio for a 2 &#x000d7; 2 table assessing the marker and case/control status relationship</td></tr><tr><td align="left" valign="top">&#x003b1;, &#x003b2;</td><td align="left" valign="top">Type I&#x02013; and type II&#x02013;error rates</td></tr><tr><td align="left" valign="top"><em>z</em><sub>&#x003b1;</sub>, <em>z</em><sub>&#x003b2;</sub></td><td align="left" valign="top"> &#x003b1;, &#x003b2; quantiles of the standard normal distribution.</td></tr></tbody></table></div></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337475" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337490" class="sidefm-pmclink" style="text-transform: none;">Introduction</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id337607" class="sidefm-pmclink" style="text-transform: none;">Materials and Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338598" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id338964" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id339896">References</div><div class="section-content"><div class="back-matter-section"><div class="ref-cit-blk" id="RF1"><div class="ref-cit">Allison D (1997) Transmission disequilibrium tests for quantitative traits. Am J Hum Genet 60:676&#x02013;690  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0f5Kh8ULCUizwXjvS_jIGi2h7uXR_9SDGgjdenQdt&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9042929">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF2"><div class="ref-cit">Allison DB, Schork NJ, Wong SL, Elston RC (1998) Extreme selection strategies in gene mapping studies of oligogenic quantitative traits do not always increase power. Hum Hered 15:261&#x02013;267  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0DuZxDJJU3G7clZ3E4wI_qEz6ddIBYervuoS0m5T0&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9526169">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF3"><div class="ref-cit">Amos CI, Zhu DK, Boerwinkle E (1996) Assessing genetic linkage and association with robust components of variance approaches. Ann Hum Genet 60:143&#x02013;160  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0-sZxkFzdHTGcXf_trMum7agupibIrcSkfIJIkQcy&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8839128">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF4"><div class="ref-cit">Boerwinkle E, Charkraborty R, Sing CF (1986) The use of measured genotype information in the analysis of quantitative phenotypes in man. I. Models and analytical methods. Ann Hum Genet 50:181&#x02013;194  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0vSKKcrqVW_KB6S2etGqSlcQAlvbukAvPjCQ2G5W-&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/3435047">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF5"><div class="ref-cit">Clark AG, Weiss KM, Nickerson DA, Taylor SL, Buchanan A, Stengard J, Salomaa V, Vartiainen E, Perola M, Boerwinkle E, Sing CF (1998) Haplotype structure and population-genetics inferences from nucleotide-sequence variation in human lipoprotein lipase. Am J Hum Genet 63:595&#x02013;612  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0fQ2D5AizDRpyVQ3WFAxJNYYO-yQ_WGPRwt2QsQ-d&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9683608">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF6"><div class="ref-cit">Collins FS, Guyer MS, Chakravarti A (1997) Variations on a theme: cataloging human DNA sequence variation. Science 278:1580&#x02013;1581  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0R5Xto2AK8lMtLmuFdXS0ZeQXMHi9rTUB3d80tkrl&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9411782">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF7"><div class="ref-cit">Collins FS, Patrinos A, Jordan E, Chakravarti A, Gesteland R, Walters L (1998) New goals for the U.S. Human Genome Projects: 1998&#x02013;2003. Science 282:682&#x02013;689  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0xS0fD4XtttZTwK9lUuL2nJImHvbicvYCHzNeDk77&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9784121">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF8"><div class="ref-cit">Devlin B, Roeder K (1999) Genomic control for association studies. Am J Hum Genet Suppl 65:A83.</div></div><div class="ref-cit-blk" id="RF9"><div class="ref-cit">Ewens WJ, Spielman RS (1995) The transmission/disequilibrium test: history, subdivision, and admixture. Am J Hum Genet 57:455&#x02013;464  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FQtSKzVfB3aAJ7YBFAXip3-KnqlXgYv-YCtVqAF-&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7668272">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF10"><div class="ref-cit">George VT, Elston RC (1987) Testing the association between polymorphic markers and quantitative traits in pedigrees. Genet Epidemiol 4:193&#x02013;201  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Z15QZXvKQ7dKyQyP2z0_OEQnIQnfUXHkz2B_QAqO&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/3609719">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF11"><div class="ref-cit">Gu C, Todorov AA, Rao DC (1997) Genome screening using extremely discordant and extremely concordant sib pairs. Genet Epidemiol 14:791&#x02013;796  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0HCvuxh_RFfLik2OcNkkz0FJ2iWVZnusV263mNkSV&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9433579">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF12"><div class="ref-cit">Jorde LB (1995) Linkage disequilibirum as a gene-mapping tool. Am J Hum Genet 56:11&#x02013;14  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0DOnsYJG8QTsmWT08NAfX1e08UBUyaXey1juY-ATs&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7825565">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF13"><div class="ref-cit">Lewontin RC (1988) On measures of gametic disequilibrium. Genetics 120:849&#x02013;852  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0nmuVtck_eLW0q87O20oZo95-ZJVZdx6bzbyKAAgN&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/3224810">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF14"><div class="ref-cit">Long JC, Williams RC, Urbanek M (1995) An E-M algorithm and testing strategy for multiple locus haplotypes. Am J Hum Genet 56:799&#x02013;810  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=02Azb-O09GhhFxNC-K-XxHsAXGVb2Sr_hfYwjQk3l&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7887436">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF15"><div class="ref-cit">MacLean C, Morton N, Elston R, Yee S (1976) Skewness in commingled distributions. Biometrics 32:695&#x02013;699  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0TMLlvDfzksgNrn6HYTr8TMADb0hBTnoF_W-S7Qmm&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/963179">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF16"><div class="ref-cit">McGinnis RE (1998) Hidden linkage: a comparison of the affected sibpair (ASP) test and transmission/disequilibrium test (TDT). Ann Hum Genet 62:159&#x02013;179  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0j7ejdeXUPbSenGGZnjaIDdM_2lm3zXB_3UyxBk_O&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9759477">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF17"><div class="ref-cit">Nielsen DM, Ehm MG, Weir BS (1998) Detecting marker-disease association by testing for Hardy-Weinberg disequilibrium at a marker locus. Am J Hum Genet 63:1531&#x02013;1540  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0tW2mao30S129YBNjbC77Qpmpg-YjwdjSHDApXQDG&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9867708">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF18"><div class="ref-cit">Nielsen DM, Weir BS (1999) A classical setting for associations between markers and loci affecting quantitative traits. Genet Res 74:271&#x02013;277  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0P7XA1S4veQz8tSlGhPfzjY6R9AsAo8smtE4WN5r7&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10689804">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF19"><div class="ref-cit">Pritchard JK, Rosenberg NA (1999) Use of unlinked genetic markers to detect population stratification in association studies. Am J Hum Genet 65:220&#x02013;228  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0xyBCYvi55kAluohF_4dgf7gwawG1yP-RhB6b4k2i&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10364535">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF20"><div class="ref-cit">Pritchard JK, Stephens M, Rosenberg NA, Donnelly P (2000) Association mapping in structured populations. Am J Hum Genet 67:170&#x02013;181  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0ZI5BwpUq7aC0C_dGN8zGsdI5sq3rVkXCFUQUlAxd&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10827107">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF21"><div class="ref-cit">Risch N, Merikangas K (1996) The future of genetic studies of complex human diseases. Science 273:1516&#x02013;1517  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0d7JqLJyuKhXWcHUtR5k4WdwbjU16GzSgw9RmV5F7&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8801636">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF22"><div class="ref-cit">Risch N, Zhang H (1995) Extreme discordant sib pairs for mapping quantitative trait loci in humans. Science 268:1584&#x02013;1589  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pQ1AWUGAP4Fbn1dquN6YORVlsFPkWD1kXU75aQ-W&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7777857">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF23"><div class="ref-cit">Schaid DJ, Rowland C (1998) Use of parents, sibs, and unrelated controls for detection of associations between genetic markers and disease. Am J Hum Genet 63:1492&#x02013;1506  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=02_zWonfiKKh_9sq2V4f8akAOecG5C89KxfMUy5Dx&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9792877">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF24"><div class="ref-cit">Schaid DJ, Sommer SS (1994) Comparison of statistics for candidate gene association studies using cases and parents. Am J Hum Genet 55:402&#x02013;408  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FaUYF-g27_6kCIjykPO8OuTVrdf6nyeuf0T0EQCf&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8037216">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF25"><div class="ref-cit">Schlesselman JJ (1982) Case-control studies. Oxford University Press, New York.</div></div><div class="ref-cit-blk" id="RF26"><div class="ref-cit">Schork NJ (2000) Genome partitioning and whole genome analysis. In: Rao DC, Province MA (eds) Advances in genetics. Academic Press, New York.</div></div><div class="ref-cit-blk" id="RF27"><div class="ref-cit">Schork NJ, Allison DB, Thiel B (1996) Mixture distributions in human genetics research. Stat Methods Med Res 5:155&#x02013;178  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0l7GOPuhqohZAzG_drqvfDqYpu2vwsX7gTNoj1Aqe&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8817796">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF28"><div class="ref-cit">Schork NJ, Fallin D, Thiel B, Xu X, Broeckel U, Jacob HJ, Cohen D (2000) The future of genetic case/control studies. In: Rao DC, Province MA (eds) Advances in human genetics. Academic Press, New York.</div></div><div class="ref-cit-blk" id="RF29"><div class="ref-cit">Slatkin M (1999) Disequilibrium mapping of a quantitative trait locus in an expanding population. Am J Hum Genet 64:1764&#x02013;1772  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0NK1mELKFVm2VA6o8xii95cCH0GeGq-R5doXF9krp&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10330364">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF30"><div class="ref-cit">Spielman RC, McGinnis RE, Ewens WJ (1993) Transmission test for linkage disequilibrium: The insulin gene region and insulin depedent diabetes mellitus (IDDM). Am J Hum Genet 52:506&#x02013;516  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0zqiRX4vS-0kxqETPdrU7lRHQKHkmj7Y34SqYBAwg&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8447318">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF31"><div class="ref-cit">Spielman RS, Ewens WJ (1996) The TDT and other family-based tests for linkage disequilibrium and association. Am J Hum Genet 59:983&#x02013;989  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0NUfc5EauwAl4xMj15Rb3X4_1inMx3zzWwqAqNQJR&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8900224">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="RF32"><div class="ref-cit">Xu X, Rogus JJ, Terwedow HA, Yang J, Wang Z, Chen C, Niu T, Wang B, Xu H, Weiss S, Schork NJ, Fang Z (1999) An extreme-sib-pair genome scan for genes regulating blood pressure. Am J Hum Genet 64:1694&#x02013;701  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Rgm_qWtaZnDLp5pAl8Cv_wbd8leXqWvRAPTCmQjL&amp;reftype=pubmed&amp;artid=1288563&amp;article-id=1288563&amp;iid=124318&amp;issue-id=124318&amp;jid=203&amp;journal-id=203&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10330357">PubMed</a>]</span>.</div></div></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">American Journal of Human Genetics</span> are provided here courtesy of <br><b>American Society of Human Genetics</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc1-->

