<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Genetics"><meta name="citation_title" content="Statistical Power of Expression Quantitative Trait Loci for Mapping of Complex Trait Loci in Natural Populations"><meta name="citation_authors" content="Paul Schliekelman"><meta name="citation_date" content="2008 April"><meta name="citation_issue" content="4"><meta name="citation_volume" content="178"><meta name="citation_firstpage" content="2201"><meta name="citation_doi" content="10.1534/genetics.107.076687"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2323809&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2323809&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2323809"><meta name="citation_pmid" content="18245851"><meta name="DC.Title" content="Statistical Power of Expression Quantitative Trait Loci for Mapping of Complex Trait Loci in Natural Populations"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="Genetics Society of America"><meta name="DC.Contributor" content="Paul Schliekelman"><meta name="DC.Date" content="2008 April"><meta name="DC.Identifier" content="10.1534/genetics.107.076687"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody9.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs2.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Statistical Power of Expression Quantitative Trait Loci for Mapping of Complex Trait Loci in Natural Populations</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0H7iCfR-VFHPl9DvdCvTIOjJ41nQrAPH1RvVG358B&amp;reftype=other&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-genetics.gif" alt="Logo of genetics" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area alt="Journal URL: redirect3.cgi?&amp;&amp;auth=061s1Gg14W05QJX8-Q87T-FfAHsa3HgCvx_k085k8&amp;reftype=publisher&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article|Banner&amp;TO=Publisher|Other|N%2FA&amp;rendering-type=normal&amp;&amp;http://www.genetics.org/" shape="rect" coords="0,0,500,100" href="redirect3.cgi?&amp;&amp;auth=0VQ6BtHu-PaH2QReR3_xkf-i-oUf7aWTTIorjWAAk&amp;reftype=publisher&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.genetics.org/" onclick="focuswin('pmc_ext')" target="pmc_ext"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-slateblue"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=301&amp;action=archive">Genetics</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=164313">v.178(4); Apr 2008</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=2323809&amp;blobtype=pdf" class="sidefm-pmclink">PDF (307K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=164313" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=301&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0jIMDuv65HV4zTqd_ahtxlFhPTaUC0ytWschv75Sw&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/18245851">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=0FiHBmMhGWxAjkNBfHDI1rmyxzbYmBP9vtHboYQLs&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=18245851">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0RGO7coVgPmM-MIbgX-exx_-nzbHnVoSq1PG7MAZ5&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=18245851">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0JuguZ-GQQgmKIi30fmoyz4rG0vRNW__HAobcNQol&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=2323809&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0tkb2CjQVOSvF9gff8iCmzAl3d1phRCJ9hAxG_5fL&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=2323809">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0VuVECew3yj-ET0v-CpxW_8OW4H0ZwNR7g37Fv5S7&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Schliekelman+P%5Bauth%5D" class="sidefm-pmclink"> Schliekelman, P.</a></div></div></div></td><td rowspan="104" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-genetics.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">Genetics. </span><span class="citation-publication-date">2008 April; </span><span class="citation-volume">178</span><span class="citation-issue">(4)</span><span class="citation-flpages">: 2201–2216. </span></div><div><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1534/genetics.107.076687.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC2323809</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 2008 by the Genetics Society of America</div><div class="fm-title">Statistical Power of Expression Quantitative Trait Loci for Mapping of Complex Trait Loci in Natural Populations</div><div class="fm-author contrib-group">Paul Schliekelman<sup>1</sup><div class="fm-affl">Department of Statistics, University of Georgia, Athens, Georgia 30602</div></div><div class="fm-footnote"></div><div class="fm-footnote" id="cor1"><sup>1</sup><em>Author e-mail:</em><span class="e_id489784">pdschlie/at/stat.uga.edu</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id489784", '<a class="ext-reflink" href="' + reverseAndReplaceString('ude.agu.tats/ta/eilhcsdp:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ude.agu.tats/ta/eilhcsdp', '/at/','@') + '</a>')}catch(e){}
                                //--></script></div><div class="fm-footnote"><div class="p">Communicating editor: M. W. F<span class="small-caps">eldman</span></div></div><div class="fm-pubdate">Received May 29, 2007; Accepted January 11, 2008.</div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id549642" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: METHODS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id551833" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: RESULTS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554252" class="sidefm-pmclink" style="text-transform: none;">POWER TO MAP CAUSATIVE LOCI USING DIFFERENTIALLY EXPRESSED TRANSCRIPTS AS QUANTITATIVE TRAITS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554522" class="sidefm-pmclink" style="text-transform: none;">DISCUSSION</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id555194" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id144025">Abstract</div><div class="section-content"><!--article-meta-->A number of recent genomewide surveys have found numerous QTL for gene expression, often with intermediate to high heritability values. As a result, there is currently a great deal of interest in genetical genomics&#x02014;that is, the combination of genomewide expression data and molecular marker data to elucidate the genetics of complex traits. To date, most genetical genomics studies have focused on generating candidate genes for previously known trait loci or have otherwise leveraged existing knowledge about trait-related genes. The purpose of this study is to explore the potential for genetical genomics approaches in the context of genomewide scans for complex trait loci. I explore the expected strength of association between expression-level traits and a clinical trait, as a function of the underlying genetic model in natural populations. I give calculations of statistical power for detecting differential expression between affected and unaffected individuals. I model both reactive and causative expression-level traits with both additive and multiplicative multilocus models for the relationship between phenotype and genotype and explore a variety of assumptions about dominance, number of segregating loci, and other parameters. There are two key results. If a transcript is causative for the disease (in the sense that disease risk depends directly on transcript level), then the power to detect association between transcript and disease is quite good. Sample sizes on the order of 100 are sufficient for 80% power. On the other hand, if the transcript is reactive to a disease locus, then the correlation between expression-level traits and disease is low unless the expression-level trait shares several causative loci with the disease&#x02014;that is, the expression-level trait itself is a complex trait. Thus, there is a trade-off between the power to show association between a reactive expression-level trait and the clinical trait of interest and the power to map expression-level QTL (eQTL) for that expression-level trait. Gene expression-level traits that are most strongly correlated with the clinical trait will themselves be complex traits and therefore often hard to map. Likewise, the expression-level traits that are easiest to map will tend to have a low correlation with the clinical trait. These results show some fundamental principles for understanding power in eQTL-based mapping studies.</div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id144025" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id549642" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: METHODS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id551833" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: RESULTS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554252" class="sidefm-pmclink" style="text-transform: none;">POWER TO MAP CAUSATIVE LOCI USING DIFFERENTIALLY EXPRESSED TRANSCRIPTS AS QUANTITATIVE TRAITS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554522" class="sidefm-pmclink" style="text-transform: none;">DISCUSSION</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id555194" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id144049"> </div><div class="section-content">IT has become apparent that mapping of complex trait loci is a much more difficult task than once envisioned. <a href="#bib12" class="cite-reflink">G<span class="small-caps">lazier</span> <em>et al.</em> (2002)</a> reviewed the literature for major experimental species and found only a relative handful of complex traits whose genetic basis is well understood. <a href="#bib1" class="cite-reflink">A<span class="small-caps">ltmuller</span> <em>et al.</em> (2001)</a> conducted a comprehensive review of 101 whole-genome scan studies of complex diseases in humans and found that only one-third produced significant linkages. Furthermore, few of the linkages that were significant have been reproduced in other studies. In contrast, geneticists have been very successful at mapping single-locus traits. Hundreds of single-locus traits have been mapped in humans alone.<p>The reason that single-locus traits are relatively easy to map is that there is a strong correlation between genotype at the causative locus and the trait. If a qualitative trait is present, then there is usually or always a corresponding genotype at the causative locus. By contrast, with a multilocus trait it is possible to exhibit the trait without there being a &#x0201c;trait genotype&#x0201d; at a particular locus or to not exhibit the trait when there is a trait genotype at that causative locus. Thus, there is not a consistent genotypic difference between those individuals exhibiting the trait and those who are not (or between different levels of the trait for a quantitative trait) and power to map complex trait loci is low.</p><p>In the past few years, the new field of &#x0201c;genetical genomics&#x0201d; has emerged&#x02014;that is, the combination of molecular marker data and genomewide expression data to elucidate the genetic basis of gene expression. Evidence has emerged of substantial genetic variation in gene expression (<em>e.g.</em>, <a href="#bib5" class="cite-reflink">B<span class="small-caps">rem</span> <em>et al.</em> 2002</a>; <a href="#bib27" class="cite-reflink">O<span class="small-caps">leksiak</span> <em>et al.</em> 2002</a>; <a href="#bib36" class="cite-reflink">Y<span class="small-caps">an</span> <em>et al.</em> 2002</a>; <a href="#bib8" class="cite-reflink">C<span class="small-caps">heung</span> <em>et al.</em> 2003</a>; <a href="#bib31" class="cite-reflink">S<span class="small-caps">chadt</span> <em>et al.</em> 2003</a>; <a href="#bib26" class="cite-reflink">M<span class="small-caps">orley</span> <em>et al.</em> 2004</a>). For example, <a href="#bib5" class="cite-reflink">B<span class="small-caps">rem</span> <em>et al.</em> (2002)</a> conducted linkage analysis of expression levels in a cross between two yeast strains. They found 1528 genes with differential expression. Among those genes they found a median heritability of expression of 84%. Expression levels for 308 of these genes showed linkage to at least one marker. <a href="#bib25" class="cite-reflink">M<span class="small-caps">onks</span> <em>et al.</em> (2004)</a> measured expression in 23,499 genes in lymphoblastoid cell lines from members of 15 CEPH families. A total of 2340 were found to be differentially expressed and 762 (31%) of these had heritability significantly different than zero. Median heritability was 0.34 for these genes and 25% had heritability &#x0003e;0.44. A genome scan was performed and 55 expression-level QTL (eQTL) for 33 genes were found. Twenty of these QTL explained &#x0003e;90% of the variance and &#x0003e;45 explained &#x0003e;70% of the variance.</p><p>These and other studies show that there may be many expression-level traits with a relatively simple and tractable genetic basis. This indicates that microarrays might be a powerful tool for mapping loci underlying complex traits. The premise is that certain gene expression-level traits might have a stronger correlation with the underlying causative loci than the trait does itself. That is, (1) many expression-level traits are highly heritable relative to complex clinical traits and appear to have a relatively simple genetic basis, (2) there is evidence of multiple expression-level traits mapping to the same chromosomal region (and perhaps the same locus), and (3) if genetic polymorphisms that affect disease risk also affect expression levels of other genes, these expression-level traits will give high power for mapping the polymorphisms. By causative locus, I mean a locus at which there is genotypic variation responsible for variation in a clinical trait of interest. By clinical trait, I mean an organismal trait of interest, such as disease status, body weight, etc. Of course, the word &#x0201c;clinical&#x0201d; is used just for convenience&#x02014;the trait need not solely be of clinical interest.</p><p>The potential of this type of approach was demonstrated by <a href="#bib31" class="cite-reflink">S<span class="small-caps">chadt</span> <em>et al.</em> (2003)</a>. They performed an F<sub>2</sub> cross between two inbred mouse strains and performed a genomewide scan for linkage for expression levels of 23,574 genes. The mice were on a high-fat diet for 4 months and thus developed a spectrum of obesity. They were classified according to the subcutaneous fat-pad-mass (FPM) trait and expression levels were compared between the upper 25% and the lower 25% for this trait. They examined the 280 genes most differentially expressed between these two groups and found that clustering on this set of genes divided the mice into two high-FPM groups and one low-FPM group, revealing genetic heterogeneity in the high-FPM mice. They then performed a genome scan for the FPM trait and showed that LOD scores of QTL for this trait were substantially increased when only one of the high-FPM groups was included. They also showed that a number of genes had expression-level traits mapping to the same region as a QTL for the FPM trait.</p><p>To apply genetical genomics approaches to a clinical trait (<em>e.g.</em>, a disease), it is necessary to establish a connection between expression-level traits and the clinical trait. To date, most of the focus in genetical genomics has been in exploring the quantitative genetics of gene expression and in candidate gene studies. Substantial progress has been made, on both experimental (<em>e.g.</em>, the articles cited above) and statistical (see <a href="#bib18" class="cite-reflink">K<span class="small-caps">endziorski</span> and W<span class="small-caps">ang</span> 2006</a> for review) fronts. However, little is still known about how transcriptional variation among genotypes relates to variation in clinical traits (<a href="#bib11" class="cite-reflink">G<span class="small-caps">ibson</span> and W<span class="small-caps">eir</span> 2005</a>). Until this is better understood, it will be difficult to apply genetical genomics approaches to understanding complex traits. The goal of this article is to explore how transcriptional variation among genotypes should be expected to relate to variation in a qualitative trait.</p><p>The ideal expression-level trait for mapping would be one whose variation is entirely determined by the genotype at a single one of the <em>L</em> causative loci. If such expression-level traits existed for each of the causative loci, then they could all be mapped with relative ease. However, such expression-level traits would not correlate strongly with the disease and might be difficult to detect as differentially expressed between disease affected and unaffected individuals. Expression-level traits that correlate strongly with the disease would be easiest to detect as differentially expressed, but would not be any better than the disease itself as a response variable for mapping. <a href="#bib31" class="cite-reflink">S<span class="small-caps">chadt</span> <em>et al</em>. (2003)</a> and others have found a very large number of eQTL mapping to locations throughout the genome, irrespective of trait status. Determining which of these eQTL are relevant requires establishing a correlation between the expression-level trait and the disease. Thus, the use of eQTL for mapping complex traits may just shift the obstacle of low power for establishing a correlation between trait and genotype to a new obstacle of low power for establishing a correlation between disease and expression level. Thus, it is unclear how powerful we should expect eQTL-based strategies to be.</p><p>A number of studies have used microarrays for identifying candidate genes (<em>e.g.</em>, <a href="#bib21" class="cite-reflink">L<span class="small-caps">awn</span> <em>et al.</em> 1999</a>; <a href="#bib3" class="cite-reflink">B<span class="small-caps">erge</span> <em>et al.</em> 2000</a>; <a href="#bib17" class="cite-reflink">K<span class="small-caps">arp</span> <em>et al.</em> 2000</a>; <a href="#bib35" class="cite-reflink">W<span class="small-caps">ayne</span> and M<span class="small-caps">c</span>I<span class="small-caps">ntyre</span> 2002</a>; <a href="#bib19" class="cite-reflink">K<span class="small-caps">irst</span> <em>et al.</em> 2004</a>; <a href="#bib6" class="cite-reflink">B<span class="small-caps">ystrykh</span> <em>et al.</em> 2005</a>; <a href="#bib7" class="cite-reflink">C<span class="small-caps">hesler</span> <em>et al.</em> 2005</a>; <a href="#bib16" class="cite-reflink">H<span class="small-caps">ubner</span> <em>et al.</em> 2005</a>; <a href="#bib22" class="cite-reflink">L<span class="small-caps">i</span> <em>et al.</em> 2005</a>; <a href="#bib32" class="cite-reflink">S<span class="small-caps">chadt</span> <em>et al.</em> 2005</a>). For example, <a href="#bib6" class="cite-reflink">B<span class="small-caps">ystrykh</span> <em>et al.</em> (2005)</a> mapped eQTL from mouse stem cells. Those that mapped to the region of a known QTL for stem cell turnover were then identified as candidate genes. Such approaches are promising when there is prior knowledge of the location of a causative locus. The primary focus of this article is genome scans&#x02014;that is, whole-genome linkage or association studies when there is no candidate region previously identified.</p><p>There are two basic tasks that must be accomplished to use genomewide expression information for a genome scan: (1) an association must be established between the transcript and the clinical trait and (2) association must be established between the transcript and a marker. These tasks might be done in a two-stage analysis or as some combined analysis, but they must be accomplished somehow. First, I examine the statistical power to detect expression-level changes as being associated with the clinical trait. Then, I examine the power to detect eQTL for expression-level traits that have been identified as differentially expressed.</p><p>I consider both transcripts that are disease causative (that is, variation in transcript level contributes directly to disease risk) and those that are controlled by a disease locus but are not causative. These latter transcripts are termed &#x0201c;reactive.&#x0201d; Note that this term refers to transcripts reactive to a disease locus and not to the disease itself (thus the usage is somewhat different from that in some previous eQTL articles). Power is shown to be quite good for causative transcripts, with sample sizes on the order of 100 being sufficient for reliable detection of association with the clinical trait. The story is more complicated for reactive transcripts. There is a trade-off in power between the two tasks laid out in the previous paragraph. The sample sizes needed for reasonable power to detect expression-level differences are often very high unless the expression-level trait and disease share multiple causative loci (thus creating a higher correlation between disease and expression level). Unfortunately, the power to map these causative loci via the expression-level trait plummets as the number of influencing loci increases. The result of this trade-off is that, for natural populations, there may be no expression-level traits that will give good power for detecting association both with the clinical trait and with a marker (although the exact power varies greatly depending on model assumptions and parameter values). It should be stressed that I am considering only associations involving single expression-level traits and there is potentially a great deal of information contained in the joint behavior of expression-level traits. Furthermore, improved experimental designs are possible. Thus, it is not the goal of this article to make absolute statements about statistical power. Rather, the goal is to determine the fundamental issues and to provide a baseline for understanding power in such studies.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id144025" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>POWER TO DETECT EXPRESSION-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: METHODS</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id551833" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: RESULTS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554252" class="sidefm-pmclink" style="text-transform: none;">POWER TO MAP CAUSATIVE LOCI USING DIFFERENTIALLY EXPRESSED TRANSCRIPTS AS QUANTITATIVE TRAITS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554522" class="sidefm-pmclink" style="text-transform: none;">DISCUSSION</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id555194" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id549642">POWER TO DETECT EXPRESSION-LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: METHODS</div><div class="section-content"><p><span class="head3">Trait and expression-level models: </span> The focus of this article is on qualitative traits. I assume that the trait is binary and is controlled by genotype at <em>L</em> causative loci. For ease of description I refer to the trait as a disease. However, the results are equally valid for any binary trait. I further assume that a total of <em>N</em> expression-level traits <em>X</em><sub>1</sub>, <em>X</em><sub>2</sub>, <em>X</em><sub>3</sub>, <em>&#x02026;</em>, <em>X<sub>N</sub></em> are being tested. Each expression-level trait (ELT) depends on some subset of the <em>L</em> causative loci, with the subset being empty for the many expression-level traits that are independent of all of the causative loci. ELTs that are controlled by disease loci are assumed to have distributions that are mixtures of normal distributions. That is, the expression distribution is assumed to be normal distribution on some scale (<em>e.g</em>., log scale) with mean and variance dependent on the genotype at the controlling loci. It is presently uncertain how well this assumption holds for most expression levels, but this seems reasonable as a first approximation. <a href="#bib15" class="cite-reflink">H<span class="small-caps">sieh</span> <em>et al.</em> (2007)</a> found evidence of multimodality in transcription in 7&#x02013;10% of genes in two <em>Drosophila melanogaster</em> lines. Other genes may have had either little within-population variation or genetic control too complicated to be detected with the available sample sizes.<p>Consider an ELT <em>X</em><sub>1</sub> that is either causative for the disease or reactive to a disease locus. We wish to find the distribution for <em>X</em><sub>1</sub> as a function of an individual's disease-affected status. If <em>T</em> is an indicator variable for disease status (<em>T</em> &#x0003d; 1 indicating disease and <em>T</em> &#x0003d; 0 indicating no disease), then we have<a id="fd1" name="fd1"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M1" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M1" title="equation M1"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(1)</span></div></td></tr></table>where <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M2" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M2" title="equation M2"> is a vector of genotypes at the causative loci and the disease prevalence is <em>K</em> &#x0003d; <em>P</em>(<em>T</em> &#x0003d; 1).</p><p>Similarly,<a id="fd2" name="fd2"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M3" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M3" title="equation M3"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(2)</span></div></td></tr></table>We can see that expression-level distributions will be mixtures of normal distributions, with the weights determined by the trait value and the population genetic parameters.</p><p><span class="head4">Multiplicative trait model: </span> To proceed, we need to further specify the trait penetrance model. The first example model is a multiplicative one. If the ELT is reactive only, we have<a id="fd3" name="fd3"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M4" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M4" title="equation M4"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(3)</span></div></td></tr></table>where <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M5" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M5" title="equation M5"> is the contribution to disease risk from the one-locus genotype <em>i<sub>j</sub></em> on locus <em>j</em>. Thus, <em>G</em> is a vector &#x0005b;<em>g</em><sub>1</sub>, <em>g</em><sub>2</sub>, &#x02026;, <em>g<sub>L</sub></em>&#x0005d;. The mechanistic basis for a multiplicative model is that each locus affects one of a chain of <em>L</em> disease events that must all occur for the disease to occur. By &#x0201c;disease event&#x0201d; I mean some physiological event associated with the locus that contributes to the occurrence of the disease.<p>Substituting (3) for <em>P</em>(<em>T</em> &#x0003d; 1 &#x0007c; <em>G</em>) in (1),<a id="fd4" name="fd4"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M6" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M6" title="equation M6"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(4)</span></div></td></tr></table>where <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M7" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M7" title="equation M7"> denotes the multiple summations over the individual-locus genotypes <em>g<sub>i</sub></em> and I have suppressed the <em>X</em><sub>1</sub> &#x0003d; <em>x</em> and <em>G</em> &#x0003d; <em>g</em> notation to simplify the appearance of the equations. I assume no gametic disequilibrium so that <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M8" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M8" title="equation M8">. In this case <em>K</em> &#x0003d; <em>K</em><sub>1</sub><em>K</em><sub>2</sub> &#x02026; <em>K<sub>L</sub></em>, where<a id="fd5" name="fd5"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M9" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M9" title="equation M9"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(5)</span></div></td></tr></table>Thus (4) can be simplified to<a id="fd6" name="fd6"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M10" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M10" title="equation M10"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(6)</span></div></td></tr></table>where I assume, without loss of generality, that the loci have been indexed such that the loci 1&#x02013;<em>c</em> are the ones that affect the expression level <em>X</em><sub>1</sub>. Thus, the expression-level distribution for affected individuals depends only on genotypes at the loci influencing that expression-level trait. On the other hand, the factor 1 &#x02212; <em>K</em> in the denominator of (2) cannot be separated by genotype and thus no cancellation occurs and the expression-level distribution is dependent on genotype at all <em>L</em> causative loci. Thus, the distribution (1) will have a strong dependence on the genotype at the loci 1, &#x02026;, <em>c</em>, while this dependence will be obscured by effects from the other loci in the distribution (2). This reflects the fact that under a multiplicative model an affected individual has always experienced a disease event at all <em>L</em> causative loci while an unaffected individual may have disease alleles at particular loci (he/she just does not have disease genotypes at all disease loci).</p><p>If the ELT is causative, the disease risk depends both on genotype and on the value of the ELT <em>X</em><sub>1</sub>,<a id="fd7" name="fd7"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M11" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M11" title="equation M11"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(7)</span></div></td></tr></table>where <em>u</em>(<em>X</em><sub>1</sub>) is the contribution to disease risk from the ELT <em>X</em><sub>1</sub>. I assume that loci 1&#x02013;<em>c</em> affect disease risk only through their impact on <em>X</em><sub>1</sub>. <a href="#fd2" class="cite-reflink">Equation 2</a> becomes<a id="fd8" name="fd8"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M12" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M12" title="equation M12"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(8)</span></div></td></tr></table></p><p>The functions <em>u<sub>c&#x0002b;1</sub></em>(<em>g<sub>c</sub></em><sub>&#x0002b;1</sub>) &#x02026; <em>u<sub>L</sub></em>(<em>g<sub>L</sub></em>) are of the same type as those for the reactive model. Note that it is not specified whether there are causative ELTs associated with the loci <em>c</em> &#x0002b; 1&#x02013;<em>L</em>. The effects of these loci are contained in the functions <em>u<sub>j</sub></em>(<em>g<sub>j</sub></em>). The effect could be via a causative ELT or via some other mechanism.</p></p><p><span class="head4">Additive trait model: </span> Under an additive model with a reactive ELT, the disease probability is given by<a id="fd9" name="fd9"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M13" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M13" title="equation M13"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(9)</span></div></td></tr></table><p>Under this model the disease event associated with any causative locus is sufficient to cause the disease, but the more that such disease events occur the higher the chance of disease. <a href="#fd1" class="cite-reflink">Equation 1</a> becomes<a id="fd10" name="fd10"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M14" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M14" title="equation M14"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(10)</span></div></td></tr></table>and (2) becomes<a id="fd11" name="fd11"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M15" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M15" title="equation M15"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(11)</span></div></td></tr></table>where <em>K &#x0003d; K</em><sub>1</sub> &#x0002b; <em>K</em><sub>2</sub> &#x0002b; &#x02026; <em>K<sub>L</sub></em> under an additive model. Similar expressions apply for a causative ELT.</p></p></p><p><span class="head3">Expression-level distribution: </span> I use the above relationships to calculate the power to detect expression-level differences due to genotypic differences between affected and unaffected individuals. The premise is that there should be a large number of changes in expression-level traits associated with the occurrence of a disease or other trait.<p>Suppose that some individual has mutations at some set of disease causative loci and that each such mutation initiates some sequence of events. These different sequences of events will then eventually interact in some way to cause the disease (or, more generally, to increase disease probability). ELTs that are &#x0201c;upstream&#x0201d; in the sequence of events originating from a mutation will have a strong association with that mutation, but a weak association with the disease. That is, if the mutation is weakly associated with the disease (as is the case for complex diseases), then ELTs strongly associated with that mutation must also be weakly associated with the disease. On the other hand, ELTs that occur &#x0201c;downstream&#x0201d; of the interactions between mutations will have stronger association with the disease but weaker association with the disease loci (because these ELTs are themselves complex traits). I assume that no reactive ELTs are simultaneously upstream and downstream&#x02014;that is, for example, that no reactive ELT depends directly both on genotype at a causative locus and on disease status (although there would be indirect dependencies in both cases). The precise meaning of this becomes clear in the derivations that follow.</p><p>My focus is on the statistical power to map a specific one of the <em>L</em> causative loci. I call this locus the target causative locus (TCL). I assume that the genotype at this causative locus affects the expression level of <em>M</em> genes (possibly including itself). If there is a causative ELT, then <em>M</em> is always taken as one. The genotype at loci besides the <em>L</em> causative loci may also affect an expression-level trait, but such loci are uncorrelated with the disease and their effect is assumed to be included in the normal distributions. When I refer to a causative locus as influencing or controlling an expression level, I mean specifically that genetic variation at the causative locus affects the expression-level trait.</p><p>Under the assumptions stated earlier, the expression-level distribution is a mixture of normal distributions, where the mean and the variance of those normals are genotype dependent. For <em>c</em> &#x0003d; 1, and a multiplicative model with a reactive ELT, we have<a id="fd12" name="fd12"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M16" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M16" title="equation M16"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(12)</span></div></td></tr></table>and<a id="fd13" name="fd13"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M17" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M17" title="equation M17"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(13)</span></div></td></tr></table>where <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M18" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M18" title="equation M18"> denotes a normal probability density distribution with mean <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M19" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M19" title="equation M19"> and variance <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M20" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M20" title="equation M20">.</p><p>Standard statistical theory states that if the probability density function is<a id="" name=""></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M21" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M21" title="equation M21"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>where the <em>p<sub>i</sub></em> sum to 1, then the mean and variance are given by<a id="" name=""></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M22" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M22" title="equation M22"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table>and<a id="fd14" name="fd14"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M23" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M23" title="equation M23"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(14)</span></div></td></tr></table>The <em>p<sub>i</sub></em>'s in this case refer to the conditional genotype probabilities like those occurring in <a href="#fd12" class="cite-reflink">Equations 12</a> and <a href="#fd13" class="cite-reflink">13</a>: that is, <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M24" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M24" title="equation M24"> and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M25" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M25" title="equation M25">. The resulting distribution is not normal. However, I consider sample sizes large enough that the central limit theorem applies and the distribution of means is normal. Power is then calculated using standard theory for the comparison of two groups.</p><p>If there are multiple expression-level traits dependent on the TCL (<em>i.e.</em>, <em>M</em> &#x0003e; 1), then <em>X</em><sub>1</sub> becomes a vector of expression levels and the normal distribution is replaced by a multivariate normal. In this case a simulation approach is used for power calculations (described later).</p><p>We next need to specify how the expression mean and variance depend on genotype. I take the variance as a constant for all genotypes. I use either a multiplicative or an additive model for the dependence of the mean on genotype. If expression is multiplicative and controlled by loci 1&#x02013;<em>c</em>, then the expression mean is given by<a id="fd15" name="fd15"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M26" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M26" title="equation M26"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(15)</span></div></td></tr></table>The value <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M27" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M27" title="equation M27"> is the contribution to the mean from locus <em>i</em>.</p><p>In the additive case, the expression mean is given by<a id="fd16" name="fd16"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M28" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M28" title="equation M28"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(16)</span></div></td></tr></table></p><p>Except where noted otherwise, the form of genetic control (multiplicative or additive) is the same for expression level and disease.</p></p><p><span class="head3">Population genetic parameters: </span> I assume that all alleles at the TCL can be classified either as normal or as disease causing. All disease-causing alleles at the TCL are assumed to have equal effect. The frequency of disease alleles (denoted by <em>D</em>) is <em>p</em> and that of normal alleles (denoted by <em>d</em>) is 1 &#x02212; <em>p</em>. The contribution to disease risk <em>u</em>(<em>g<sub>i</sub></em>) is <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M29" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M29" title="equation M29"> for a normal homozygote (genotype <em>dd</em>),<img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M30" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M30" title="equation M30"> for a disease homozygote (genotype <em>DD</em>), and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M31" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M31" title="equation M31"> for heterozygotes (genotype <em>Dd</em>). These are the probabilities that the disease event associated with that locus occurs, given genotype. The amount of disease risk associated with the TCL is given by a parameter <em>L</em>. The single-locus contribution to disease risk <em>K<sub>L</sub></em> is taken as equal to <em>K<sup>1/L</sup></em> for a multiplicative model and <em>K</em>/<em>L</em> for an additive one. Thus, this is the contribution that locus would have if there were <em>L</em> loci all of equal effect. Note that this does not assume that there are actually <em>L</em> loci of equal effect, just that the effect attributable to the TCL is the same is if there were. Note that the role of <em>L</em> in the previous derivations is no different when we interpret <em>L</em> this way. For the target causative locus I take <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M32" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M32" title="equation M32"> and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M33" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M33" title="equation M33"> for a multiplicative penetrance model and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M34" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M34" title="equation M34"> and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M35" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M35" title="equation M35"> for an additive model.<p>Very little is known about allele frequencies or other parameters for complex traits. Because the prevalence <em>K</em> is typically known, I take it as given and solve for the allele frequency <em>p</em> using (5) after specifying the parameters <em>h</em>, &#x003c0;, &#x003b4;, and <em>L</em>.</p><p>The parameters <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M36" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M36" title="equation M36"> and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M37" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M37" title="equation M37"> are the mean expression levels for individuals with two disease alleles at all <em>c</em> loci controlling the expression level and the genotype with 0 disease alleles at those loci, respectively. If the expression genetics are multiplicative, then the single-locus expression mean <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M38" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M38" title="equation M38"> is given by <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M39" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M39" title="equation M39">, <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M40" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M40" title="equation M40">, and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M41" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M41" title="equation M41"> for individuals with zero, one, and two disease alleles at locus <em>i</em>, respectively. If expression is controlled additively, then these values become <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M42" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M42" title="equation M42">, <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M43" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M43" title="equation M43">, and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M44" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M44" title="equation M44">.</p><p>The parameter <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M45" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M45" title="equation M45"> is the standard deviation in expression (assumed the same for all genotypes) and I take <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M46" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M46" title="equation M46">. Values for &#x003c4; were chosen on the basis of calculations given as supplemental material using the data of <a href="#bib14" class="cite-reflink">H<span class="small-caps">edenfalk</span> <em>et al.</em> (2001)</a>, who measured expression-level differences between patients carrying different breast cancer mutations. Briefly, I calculated <em>t</em>-statistics for differences in expression in 3326 genes between women carrying and not carrying BRCA1 mutations. I used an approach similar to that used by <a href="#bib4" class="cite-reflink">B<span class="small-caps">rem</span> and K<span class="small-caps">ruglyak</span> (2005)</a> for correcting for multiple testing. That is, I randomly divided the data set in two and used half of the data to calculate <em>t</em>-statistics for all 3226 genes. Then, I used the other half of the data to recalculate the <em>t</em>-statistics for those genes with the 50 highest values. The 10th highest <em>t</em>-statistic from this set was ~3 (the maximum was 6 and the median was 2). On the basis of this, I take a baseline of &#x003c4; &#x0003d; 3. This was chosen as an extreme but not the most extreme value: that is, we are specifically interested in the expression-level traits that are most correlated with the disease mutation, but want to avoid possible outliers. See the supplemental material for more details. <a href="articlerender.fcgi?artid=2323809&amp;rendertype=table&amp;id=tbl1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a> gives a description of parameters used in this article.</p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="tbl1" name="tbl1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=table&amp;id=tbl1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="TABLE 1" title="TABLE 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=table&amp;id=tbl1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>TABLE 1</strong></a><div class="figure-table-caption-in-article"><span><strong>Description of parameters</strong></span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span class="head3">Calculations of statistical power: </span> Statistical power is defined as the probability of rejecting the null hypothesis given that it is false. In this case, the null hypothesis is that there is no difference in mean expression level between affected and unaffected individuals for the gene in question. The alternate hypothesis is that there is a difference. The goal here is to calculate, for a given set of parameter values, the power to detect expression-level difference for at least one gene whose expression-level trait is controlled by the TCL. I assume a typical setup for microarray experiments. There are <em>n</em><sub>1</sub> affected individuals and <em>n</em><sub>2</sub> unaffected individuals. Expression-level traits are measured for a large number of genes to find ones that are differentially expressed between the two treatments. The population means and variances for affected and unaffected individuals are calculated from (15). For sufficiently large sample sizes, the treatment means will have a normal distribution with these means and variances. Using standard theory, the power to detect differences between treatments can be calculated. I assume a Bonferroni correction for multiple testing. Note that in standard false discovery rate approaches (<a href="#bib2" class="cite-reflink">B<span class="small-caps">enjamini</span> and H<span class="small-caps">ochberg</span> 1995</a>) no tests are rejected unless the most significant one reaches the Bonferroni-corrected significance threshold. Thus, there is no difference between a Bonferroni correction and a false discovery rate approach because I am calculating the power to detect <em>any</em> differences.<p>In the case of reactive ELTs, I assume that the <em>M</em> expression-level traits are independent given the genotype at their causative loci. This is most likely not true in many cases. This assumption will increase the calculated power because correlations between ELTs would decrease the effective value of <em>M</em>.</p><p>Note that I consider detection of an association between a reactive ELT and the disease as a &#x0201c;success,&#x0201d; because that reactive ELT can be used to map the controlling disease locus even if the ELT itself is not disease causative. However, in another context a researcher might be interested in detecting only ELTs that are disease causative. In this case, the statistical power calculated here for reactive ELTs would actually be the probability of type I error.</p><p>In this article I strictly focus on expression-level traits directly controlled by disease-causative loci. There may also be expression levels whose controlling locus is linked to a disease locus. These expression levels will also be correlated with the disease, but this correlation will be weakened by recombination. To avoid complications, I do not consider such genes.</p><p>I use a simulation approach to calculate approximate power for <em>M</em> &#x0003e; 1. For each repetition, expression levels for treatment and control groups are randomly generated using distributions (1) and (2). A <em>t</em>-test is conducted for each of the <em>M</em> expression-level traits. If at least one expression-level trait is significantly different between treatment and controls, then that repetition is declared a success. The power is then approximately equal to the fraction of success in a large number of trials.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id144025" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id549642" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: METHODS</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>POWER TO DETECT EXPRESSION LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: RESULTS</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554252" class="sidefm-pmclink" style="text-transform: none;">POWER TO MAP CAUSATIVE LOCI USING DIFFERENTIALLY EXPRESSED TRANSCRIPTS AS QUANTITATIVE TRAITS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554522" class="sidefm-pmclink" style="text-transform: none;">DISCUSSION</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id555194" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id551833">POWER TO DETECT EXPRESSION LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: RESULTS</div><div class="section-content"><p><span class="head3">Reactive expression-level traits&#x02014;multiplicative model: </span> <p><span class="head4">Dependence on h and L: </span> <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1</a> shows surface plots of power to detect expression-level trait differences <em>vs. L</em> (effective number of causative loci) and <em>h</em> (dominance coefficient) for a multiplicative model. For each plot, disease prevalence <em>K</em> &#x0003d; 0.01, expression-level trait standard deviation <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M54" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M54" title="equation M54">, sample size is 200 (100 for each treatment), the penetrance parameters &#x003c0; and &#x003b4; are 0.0 and 1, respectively (indicating perfect penetrance with respect to the full multilocus genotype), and the expression-level trait depends only on a single TCL. Thus, there are three expression-level trait means. In both plots <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M55" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M55" title="equation M55"> and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M56" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M56" title="equation M56">, where <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M57" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M57" title="equation M57"> is the mean expression level for individuals with <em>i</em> disease alleles at the TCL. These values correspond to &#x003c4; &#x0003d; 3. The dominance in expression is <em>h</em><sub>E</sub> &#x0003d; 0 (completely recessive) in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1a</a> and <em>h</em><sub>E</sub> &#x0003d; 0.5 (no dominance) in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1b</a>. I assume that there are 20,000 genes tested on the microarray. These are default parameter values (with <em>h</em><sub>E</sub> &#x0003d; 0.5) throughout this article unless otherwise specified.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig1" name="fig1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2323809&amp;blobname=GEN17842201f1.gif" border="1" class="icon-reflink" alt="Figure 1. " title="Figure 1. "></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>F<span class="small-caps">igure</span> 1.&#x02014;</strong></a><div class="figure-table-caption-in-article"><span>Plot of power <em>vs. L</em> (number of disease loci) and <em>h</em> (the dominance coefficient). The power is the probability of rejecting the null hypothesis of no difference in expression between the treatments. Parameter values are &#x003c0; &#x0003d; 0, &#x003b4;</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>The determinant of power is the amount of contrast between the treatment means. In <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1a</a>, the expression-level trait is recessive and there is no mean difference between individuals with <em>dd</em> and <em>Dd</em> genotype at the causative locus. When <em>h</em> &#x0003d; 0 (disease risk is recessive with respect to the TCL), then all affected individuals are <em>DD</em> at the TCL. When <em>L</em> &#x0003d; 1, then all unaffected individuals are <em>dd</em> or <em>Dd</em> at the TCL and there is a strong contrast in expression level between the affected and unaffected treatments. The power is near 1. When <em>h</em> moves away from 0, then some <em>Dd</em> individuals are also affected. Because the disease allele is rare when <em>L</em> &#x0003d; 1, most affected individuals are <em>Dd</em> when <em>h</em> &#x0003e; 0 and thus there is little contrast between treatments. Thus, we see that the power quickly drops to near 0 as <em>h</em> is increased for <em>L</em> &#x0003d; 1.</p><p>When <em>L</em> &#x0003e; 1, then some nondisease individuals will also have <em>DD</em> genotypes at the TCL (see the description of the multiplicative trait model). As <em>L</em> is increased, the disease-allele frequency at the TCL also increases. As the number of causative loci is increased, then the probability of the disease events associated with those loci must also increase to maintain the constraint that the disease prevalence is 0.01. For example, the disease-allele frequency is 0.68 for <em>L</em> &#x0003d; 6 and 0.75 for <em>L</em> &#x0003d; 10 at <em>h</em> &#x0003d; 0. See <a href="#bib33" class="cite-reflink">S<span class="small-caps">chliekelman</span> and S<span class="small-caps">latkin</span> (2002)</a> for further details and plots of allele frequencies. As <em>L</em> increases, the proportion of unaffected individuals with a <em>DD</em> genotype at the TCL also increases. Thus, the contrast between the two treatments also decreases and power drops quickly. At <em>h</em> &#x0003d; 0.1, power is near 97% for <em>L</em> &#x0003d; 6, but drops to ~46% for <em>L</em> &#x0003d; 11. The power also drops quickly as <em>h</em> is increased from 0. For example, at <em>h</em> &#x0003d; 0.3, the power is &#x0003c;50% for all values of <em>L</em> and for <em>h</em> &#x02265; 0.5 the maximum power is ~0.05. One interesting aspect of <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1a</a> is that power actually increases when <em>L</em> is increased from 1 when <em>h</em> is low but &#x0003e;0. This occurs because allele frequency increases as <em>L</em> is increased. As discussed above, the disease-allele frequency is low for <em>L</em> &#x0003d; 1 and when <em>h</em> &#x0003e; 0 most affected individuals are genotype <em>Dd</em>. The allele frequency increases as <em>L</em> does and thus more affected individuals are of genotype <em>DD</em>. Only genotype <em>DD</em> produces any contrast in expression level, so power is increased.</p><p><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1b</a> has identical parameter values except that <em>h</em><sub>E</sub> &#x0003d; 0.5 (no dominance in expression). In this case, the power to detect differential expression is less sensitive to <em>h</em> and does not drop to zero as <em>h</em> is increased. Power is lowest for intermediate <em>h</em> and as above is highest for <em>h</em> &#x0003d; 0. However, the power in the best case is substantially worse for larger <em>L</em> than in the previous example. The maximum power is 0.81 for <em>L</em> &#x0003d; 6 and 0.61 for <em>L</em> &#x0003d; 5. As discussed above, disease-allele frequency is quite high for larger <em>L</em> when <em>h</em> &#x0003d; 0. Thus, the difference between treatment groups is primarily due to differences between <em>Dd</em> and <em>DD</em> individuals. Because the expression-level means are closer together, the difference is less. The closer <em>h</em> is to 0.5, the more affected individuals are of <em>Dd</em> genotype. Again, the contrast between treatments is lower and power decreases.</p><p>Supplemental material available at my website shows a similar plot for the case with <em>h</em><sub>E</sub> &#x0003d; 1. In this case the power is highest at <em>h</em> &#x0003d; 1 and drops off quickly as <em>h</em> is decreased from 1 (although less quickly than in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 1a</a>).</p></p><p><span class="head4">Dependence on c: </span> Next, I explore how the number of loci (<em>c</em>) controlling the expression-level trait affects the power to detect an expression-level difference. The more similar the genotypic dependence of the expression-level trait is to that of the disease, the more contrast in expression-level trait we expect there to be between affecteds and unaffecteds. Thus, increasing the number of loci controlling the expression-level trait should increase the power (since all loci controlling the expression-level trait are assumed to be controlling disease status also). There is no question that increasing the range of variation in an expression-level trait will increase power. The unknown is how changing the control of expression level affects power. Therefore, the maximum and minimum expression levels are kept constant as <em>c</em> is varied. That is, the expression level for the genotype with all <em>c</em> loci as <em>DD</em> is constant and the expression level for the genotype with all <em>c</em> loci as <em>dd</em> is constant. <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2</a> shows plots for several parameter values. In <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2, a and b</a>, the expression-level trait has an additive dependence on the TCLs while it has a multiplicative dependence in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2, c and d</a>. In all cases the disease has a multiplicative dependence. We see that the power increases only slowly with <em>c</em> in the additive case. The advantage of increasing <em>c</em> is that it increases the concordance of genotypic dependence between expression-level trait and disease status. However, if disease and expression-level traits depend in fundamentally different ways on the multilocus genotype, this concordance does not increase substantially with <em>c</em>.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig2" name="fig2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2323809&amp;blobname=GEN17842201f2.gif" border="1" class="icon-reflink" alt="Figure 2. " title="Figure 2. "></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>F<span class="small-caps">igure</span> 2.&#x02014;</strong></a><div class="figure-table-caption-in-article"><span>Plot of power <em>vs. c</em> (number of loci controlling expression-level trait). The <em>y</em>-axis is the probability of detecting a significant expression-level difference between affected and unaffecteds. The different curves correspond to the value of <em>L</em> as shown</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>In <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2, c and d</a>, we see the unexpected result that the power initially increases with <em>c</em> but then decreases. The increase is because of the increased concordance of control of disease and genotype. That is, it is quite likely that an unaffected individual will have one or two disease alleles at a given locus (just not at all loci). Thus, if <em>c</em> &#x0003d; 1 then the ELT will often be differently regulated in unaffected individuals. It is less likely that the unaffected individual will have disease alleles at all of some set of multiple loci. Thus, the ELT is less likely to be differently regulated in unaffecteds when <em>c</em> &#x0003e; 1 and thus there is stronger association between disease and ELT and power initially increases with <em>c</em> (except for <em>L</em> &#x0003d; 3). The decrease occurs because of how the disease and expression models are defined. As <em>c</em> increases, it becomes likely that multiple controlling loci for the ELT in affecteds will be heterozygotic. In <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2c</a>, the contributions to the expression means for <em>dd</em>, <em>Dd</em>, and <em>DD</em> individuals are 0, 0.5, and 1, respectively. Because these values are multiplied to get the mean, then the mean moves much closer to 0 than to 1 if there are more than one to two heterozygote loci. Thus, when <em>c</em> is large, then even affected individuals often have low expression means and this diminishes the contrast in expression between affected and unaffected. In <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2d</a>, <em>h</em> is increased from 0.5 to 0.9. This leads to a decreased disease allele frequency (for constant <em>K</em>) and thus more heterozygote loci for affecteds. This in turn leads to more rapid loss of power as <em>c</em> increases. It is difficult to say whether this behavior is reasonable or should be considered an artifact of the model.</p><p>We see in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2, c and d</a>, that there is good power to detect expression-level differences for <em>L &#x0003d;</em> 3 and <em>c</em> &#x0003d; 1&#x02013;2 and for <em>L</em> &#x0003d; 6 cases and <em>c</em> &#x0003d; 2&#x02013;3. For higher values of <em>L</em> and <em>h</em> &#x0003d; 0.5 (<a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2c</a>) the power is maximized in the 50&#x02013;60% range for <em>c</em> &#x0003d; 4&#x02013;5. For the <em>h</em> &#x0003d; 0.9 case (<a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2d</a>) power is substantially lower.</p></p><p><span class="head4">Dependence on M (number of expression-level traits controlled by the TCL): </span> If the difference in genotype distribution between affecteds and unaffecteds means is not large at the TCLs, then the probability of the sample means being different enough to achieve significance may be small. In this case, the probability of achieving significance is increased if there are multiple expression-level traits dependent on the TCLs. <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 3</a> shows a surface plot of power <em>vs. c</em> and <em>M</em> for <em>L</em> &#x0003d; 9, <em>h</em><sub>E</sub> &#x0003d; <em>h &#x0003d;</em> 0.5, and a sample size of 200. The power here is defined as the probability that at least one of the <em>M</em> ELTs is detected as differentially expressed. When <em>c &#x0003d;</em> 1, we see the power increasing from 5% for <em>M</em> &#x0003d; 1 to 50% for <em>M</em> &#x0003d; 40. In this case, there is little distinction between affecteds and unaffecteds at the TCL. The disease-allele frequency is 0.6 for these parameter values. Approximately 84% (that is, 1 &#x02212; 0.4<sup>2</sup>) of unaffecteds carry a disease allele at the TCL and there is little difference in TCL genotype between affecteds and unaffecteds. With so little difference in genotype distribution, each additional ELT dependent on the TCL makes only a small difference. However, a large number of such ELTs do make a substantial difference. For the case <em>c</em> &#x0003d; 2, small changes in <em>M</em> have a substantial affect on power. When we jointly consider two loci, the genotypic difference between treatments is greater. Power is still poor (36%) for <em>M</em> &#x0003d; 1, but improves to &#x0003e;80% for <em>M</em> &#x0003d; 8. For <em>c</em> &#x0003d; 3, power increases from ~60 to ~90% when <em>M</em> is increased from 1 to 4. Thus, increased <em>M</em> can help power substantially in cases where the genotypic distinction between treatments is intermediate.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig3" name="fig3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2323809&amp;blobname=GEN17842201f3.gif" border="1" class="icon-reflink" alt="Figure 3. " title="Figure 3. "></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>F<span class="small-caps">igure</span> 3.&#x02014;</strong></a><div class="figure-table-caption-in-article"><span>Effect of <em>M</em>. The <em>z</em>-axis is the probability of detecting a significant expression-level difference between affecteds and unaffecteds for at least one of the <em>M</em> expression-level traits plotted against the parameters <em>c</em> and <em>M</em>. Parameter values are otherwise</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig3" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span class="head4">Dependence on sample size: </span> <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 4</a> shows plots of power <em>vs.</em> sample size for various values of <em>M</em> and <em>c</em> for <em>L</em> &#x0003d; 9. <em>M</em> &#x0003d; 1 for all curves in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 4a</a>, but varies in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 4b</a> as shown. When <em>M</em> &#x0003d; 1, there is no case for which the power reaches 80% for sample sizes &#x0003c;250. A sample size of 600 is required for 80% power when <em>c</em> &#x0003d; 1. The best case is 80% power for a sample size of 250 when <em>c</em> &#x0003d; 3. The sample sizes become more reasonable (although still rather large) for higher <em>M</em>. Power is 85% and 92%, respectively, for <em>c &#x0003d;</em> 3 and <em>M</em> &#x0003d; 4 and 8 and a sample size of 160. However, a sample size of 350 is required to get 80% power even when <em>M</em> &#x0003d; 8 for <em>c</em> &#x0003d; 1 or 8.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig4" name="fig4"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig4" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2323809&amp;blobname=GEN17842201f4.gif" border="1" class="icon-reflink" alt="Figure 4. " title="Figure 4. "></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>F<span class="small-caps">igure</span> 4.&#x02014;</strong></a><div class="figure-table-caption-in-article"><span>Plot of power <em>vs.</em> sample size. The <em>y</em>-axis is the probability of detecting a significant expression-level difference between affecteds and unaffecteds. The different curves in a correspond to different values of <em>c</em> as shown. The curves in b correspond to</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig4" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span class="head4">Dependence on penetrance parameters: </span> All of the above plots had perfect penetrance for the full multilocus genotype (that is, <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M65" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M65" title="equation M65"> and &#x003b4; &#x0003d; 1). <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig5" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 5</a> shows that varying &#x003c0; (the probability of an individual with no disease alleles being affected) from zero seriously decreases power, while decreasing &#x003b4; from 1 has negligible effect. For &#x003c0; &#x02265; 0.00005, the power is near zero even for a sample size of 1500. When &#x003c0; &#x0003d; 0 it is certain that an affected individual has at least one disease allele at each locus. When &#x003c0; &#x0003e; 0, then some affected individuals do not have any disease alleles at some loci. For the parameters in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig5" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 5</a> (<em>L</em> &#x0003d; 9 and prevalence &#x0003d; 0.01) and &#x003c0; &#x0003d; 0.0005, it can be shown that most affected individuals have at least one locus with no disease alleles. Thus, the contrast in expression levels between affecteds and unaffecteds is diminished. On the other hand, many loci in unaffected individuals have a disease alleles regardless of the value of &#x003b4; and changing it therefore has less effect. The sharp drop in power as <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M66" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M66" title="equation M66"> is increased is largely due to the particular form of model used. The parameter &#x003c0; is the probability of an individual with no disease alleles being affected. The parameter <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M67" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M67" title="equation M67">, the contribution to disease risk of the TCL, is <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M68" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M68" title="equation M68">. When <em>L</em> is large, <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M69" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M69" title="equation M69"> increases very quickly as &#x003c0; does. For example, when <em>L</em> &#x0003d; 9 and &#x003c0; &#x0003d; 0.0005, then <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M70" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M70" title="equation M70">. Thus, the probability that a locus with no disease alleles contributes its disease event is quite high. The effect of varying &#x003b4; is small for the same reason. For &#x003b4; &#x0003d; 0.5 and <em>L</em> &#x0003d; 9, the value of <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M71" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M71" title="equation M71"> is 0.5<sup>1/9</sup> &#x0003d; 0.93. It is, of course, unknown whether this form for <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M72" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M72" title="equation M72"> is appropriate. However, if we assume a multiplicative model and that individuals with no disease alleles can have the disease, then it is necessary.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig5" name="fig5"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig5" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2323809&amp;blobname=GEN17842201f5.gif" border="1" class="icon-reflink" alt="Figure 5. " title="Figure 5. "></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig5" onclick="startTarget(this, 'figure', 1024, 800)"><strong>F<span class="small-caps">igure</span> 5.&#x02014;</strong></a><div class="figure-table-caption-in-article"><span>The effect of penetrance parameters. The <em>y</em>-axis is the probability of detecting a significant expression-level difference between affecteds and unaffecteds. The curves correspond to different values of the penetrance parameters &#x003c0; and &#x003b4;</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig5" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><span class="head4">Dependence on the heritability of expression-level trait: </span> As the heritability of the expression-level trait with respect to the TCL increases, so does the power to detect that TCL. In the above results, I assumed <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M73" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M73" title="equation M73">, where <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M74" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M74" title="equation M74"> and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M75" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M75" title="equation M75"> are the expression-level means for individuals with all 2<em>c</em> disease alleles and 0 disease alleles, respectively, and &#x003c3; is the standard deviation in gene expression. This corresponds to 45% of variance explained by genotype. Referring to supplemental Figure 1, we see that a value of three is among the highest values calculated for the Hedenfalk data. However, there are genes with values ranging as high as six, and the power to detect expression level differences for these genes will be greater. <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig6" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 6</a> shows a plot of power <em>vs.</em> the parameter <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M76" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M76" title="equation M76"> for various values of <em>L</em> and <em>c</em>. In this figure <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M77" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M77" title="equation M77"> and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M78" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M78" title="equation M78"> are held constant at 1 and 0, respectively, while the expression-level variance &#x003c3; is varied. The sample size is 200, <em>h</em> &#x0003d; 5, <em>h</em><sub>E</sub> &#x0003d; 0.5, and all other parameters are as in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2</a>. As expected, the power increases steadily with &#x003c4;. For example, the power in the <em>L</em> &#x0003d; 6, <em>c</em> &#x0003d; 1 case increases from ~7% at &#x003c4; &#x0003d; 2 to 86% for &#x003c4; &#x0003d; 8. On the other hand, the power in the <em>L</em> &#x0003d; 9, <em>c</em> &#x0003d; 1 case reached only 37% at &#x003c4; &#x0003d; 8 because it started from only 1%.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig6" name="fig6"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig6" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2323809&amp;blobname=GEN17842201f6.gif" border="1" class="icon-reflink" alt="Figure 6. " title="Figure 6. "></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig6" onclick="startTarget(this, 'figure', 1024, 800)"><strong>F<span class="small-caps">igure</span> 6.&#x02014;</strong></a><div class="figure-table-caption-in-article"><span>The effect of the heritability of expression level. The <em>y</em>-axis is the probability of detecting a significant expression-level difference between affecteds and unaffecteds. The <em>x</em>-axis is the value of the parameter <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M79" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M79" title="equation M79">. The curves correspond to different values</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig6" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p></p><p><span class="head3">Natural population&#x02014;additive model: </span> Power under an additive penetrance model is substantially worse than that under a multiplicative model. <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig7" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 7a</a> shows a plot of power to detect expression-level differences <em>vs.</em> sample size for <em>L</em> &#x0003d; 4 and <em>L</em> &#x0003d; 6 and a range of <em>c</em> values for each. The power is very poor. With just four disease loci, sample sizes of &#x0003e;800 are required to reach 80% power. When <em>L</em> &#x0003d; 6, power is just 10% with an 800-sample size.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig7" name="fig7"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig7" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2323809&amp;blobname=GEN17842201f7.gif" border="1" class="icon-reflink" alt="Figure 7. " title="Figure 7. "></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig7" onclick="startTarget(this, 'figure', 1024, 800)"><strong>F<span class="small-caps">igure</span> 7.&#x02014;</strong></a><div class="figure-table-caption-in-article"><span>Power for the additive penetrance model. The <em>y</em>-axis is the probability of detecting a significant expression-level difference between affecteds and unaffecteds with an additive penetrance model. The <em>x</em>-axis in a is the sample size and the <em>x</em>-axis in b is</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig7" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>Crucially, the power is not affected much by <em>c</em>. This is in contrast to the multiplicative model, where increasing <em>c</em> has a large effect on power. Under the additive model, the mean expression levels for affecteds and unaffecteds do not change as <em>c</em> is varied. A proof of this is given in the supplemental results. Power increases with <em>c</em> under a multiplicative model (for small <em>c</em>) because increasing <em>c</em> increases the concordance between expression-level trait and disease. Under an additive model, each locus contributes independently and increasing <em>c</em> does not increase the concordance.</p><p>Power under a multiplicative penetrance is much higher than that under additive penetrance even for <em>c</em> &#x0003d; 1. This is due to the very different genotype distributions under the two models. With multiplicative penetrance, affected individuals have at least one disease allele at all loci (or at most loci if the penetrance parameter <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M80" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M80" title="equation M80">). In contrast, only one locus need have a disease allele to cause disease with additive penetrance. With <em>L</em> &#x0003d; 6 and other parameters as in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig7" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 7a</a>, ~90% of affected individuals carry only a single disease allele and most of the remaining 10% carry single disease alleles at two loci. Thus, only (90%/6 &#x0002b; 10%/3) &#x0003d; 18% of affected individuals carry a disease allele on a given disease locus. Assuming no dominance as in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig7" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 7</a>, the average expression level for a gene controlled by this locus is 0.5 and thus the average expression level among affecteds is 0.5 &#x000d7; 0.18 &#x0003d; 0.09. Most unaffecteds carry no disease alleles, so the mean difference in expression level between affected and unaffected is ~0.09. With the same parameters and a multiplicative model, affecteds on average have three loci that are homozygous for the disease allele and three loci that are heterozygous. The average expression level in affecteds for a gene controlled by the disease locus would be ~0.5 &#x000d7; 1 &#x0002b; 0.5 &#x000d7; 0.5 &#x0003d; 0.75. Now, as discussed earlier, disease-allele frequencies are high under a multiplicative model (in strong contrast to the case with an additive model). The disease-allele frequency would be 0.46 for the parameters in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig7" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 7</a> and genotype frequencies in unaffecteds would be ~0.21 disease homozygote, 0.50 heterozygote, and 0.29 normal homozygote. Thus, the mean expression level in unaffecteds would be 0.21 &#x000d7; 1 &#x0002b; 0.5 &#x000d7; 0.5 &#x0002b; 0.29 &#x000d7; 0 &#x0003d; 0.46 and the mean difference between affecteds and unaffecteds would be ~0.75 &#x02212; 0.46 &#x0003d; 0.29. The mean difference in expression level is 3.5 times higher for the multiplicative model than for the additive one and power to detect expression-level differences is correspondingly higher under the multiplicative model.</p><p><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig7" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 7b</a> shows a plot of power <em>vs. M</em> for sample size 200, <em>L</em> &#x0003d; 4, and <em>c</em> &#x0003d; 1, 3, and 5. We see increasing <em>M</em> has minor impact. Even for this low value of <em>L</em> the power is still low for <em>M</em> up to 20.</p></p><p><span class="head3">Causative expression-level trait: </span> There are many possibilities for how disease risk depends on the expression level of a causative transcript. I use a simple threshold model with <em>u</em>(<em>X</em><sub>1</sub>) &#x0003d; 1 if <em>X</em><sub>1</sub> &#x02265; <em>Q</em> and <em>u</em>(<em>X</em><sub>1</sub>) &#x0003d; 0 if <em>X</em><sub>1</sub> &#x0003c; <em>Q</em>. The expression means and variances for affected and unaffected individuals are calculated using <a href="#fd15" class="cite-reflink">Equation 15</a> in a similar manner to that done for the reactive ELT. For example, the expected value among affecteds for the multiplicative model is<a id="fd17" name="fd17"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M81" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M81" title="equation M81"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;">(17)</span></div></td></tr></table><p><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig8" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 8a</a> shows a plot of power to detect a difference in <em>X</em><sub>1</sub> between affecteds and unaffecteds as a function of sample size. The parameter <em>Q</em> &#x0003d; 0.5, <em>c</em> &#x0003d; 1, and the expression means are <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M82" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M82" title="equation M82">, <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M83" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M83" title="equation M83">, and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M84" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M84" title="equation M84">. The different curves correspond to the value of <em>L</em> as shown. Other parameters are as in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 2</a>. The power is greatly improved compared to a reactive ELT. For example, we see that 80% power is achieved at a sample size of ~100 when <em>L</em> &#x0003d; 9. This compares to a sample size of 600 for 80% power for similar parameter values with a reactive ELT. This increased power results from a combination of higher expression means and lower variances for affected individuals when the ELT is causative relative to the case when it is reactive. Both of these effects result primarily from the heterozygote expression distribution being truncated in affected individuals. The value of the ELT must exceed <em>Q</em> in affected individuals. With the parameter values in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig8" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 8</a> (<em>Q</em> &#x0003d; 0.5 and <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M85" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M85" title="equation M85">), half of the expression distribution is truncated in affected heterozygotes.</p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig8" name="fig8"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig8" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2323809&amp;blobname=GEN17842201f8.gif" border="1" class="icon-reflink" alt="Figure 8. " title="Figure 8. "></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig8" onclick="startTarget(this, 'figure', 1024, 800)"><strong>F<span class="small-caps">igure</span> 8.&#x02014;</strong></a><div class="figure-table-caption-in-article"><span>Power for causative ELT. The <em>y</em>-axis is the probability of detecting a significant expression-level difference between affecteds and unaffecteds for a causative ELT as described in the text. The <em>x</em>-axis in a is the sample size and the <em>x</em>-axis in b is the</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig8" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig8" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 8b</a> shows the effect of the threshold <em>Q</em>. We see that for these parameter values the power is at a peak at ~<em>Q</em> &#x0003d; 0.6, but that <em>Q</em> has a small impact on power. The effects of varying <em>Q</em> are complicated. As <em>Q</em> increases, fewer individuals will have an ELT value that exceeds it. Thus, the disease-allele frequency must increase to keep disease prevalence fixed. Increasing <em>Q</em> has four major effects. First, it increases the expression mean in affected individuals, which tends to increase power. Second, it increases the expression mean in unaffected individuals (because the disease-allele frequency increases), which tends to decrease power. Third, increasing <em>Q</em> reduces the portion of each genotype-specific expression distribution that exceeds the threshold, tending to decrease variance. Fourth, when <em>Q</em> reaches extreme values, then the disease allele frequency does also. This tends to reduce expression variance by reducing genetic variance in the population. The interaction between these effects is complex, but the net effect on power is small because the various effects are often in opposition to each other.</p><p>Figures available as supplemental material show the effects of various other parameters on the power for the causative ELT. Power increases very substantially with <em>c</em> as the proportion of the risk explained by the ELT increases. The degree of dominance in the ELT (that is, the mean expression level for heterozygotes) has a roughly inverse effect to that of <em>Q</em>. That is, power is at a peak for intermediate dominance and the various effects discussed above for <em>Q</em> work in roughly the opposite direction as <em>h</em> is increased.</p><p>Allowing <em>M</em> to vary for causative ELTs would require specifying a model for how these ELTs interact to determine disease risk. Thus, I keep <em>M</em> fixed at one. Other parameters have effects similar to those observed for reactive ELTs.</p><p>This model for a causative ELT is not compatible with additive disease risk for the standard parameter values used in this article. In the additive case, the disease risk associated with the causative locus is <em>K/L</em> &#x0003d; 0.01/9 &#x0003d; 0.0011. Under the standard parameters, the ELT has a probability of 0.07 of exceeding the threshold <em>Q</em> &#x0003d; 0.5 when its controlling locus has no disease alleles. This means that an individual with no disease alleles still has a probably 0.07 of being affected, which far exceeds the &#x0201c;allowed&#x0201d; disease risk for the locus. This can be corrected by lowering the genotype-specific ELT standard deviation <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M86" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M86" title="equation M86"> to a level that makes the probability of exceeding <em>Q</em> &#x0003c; 0.011. This requires halving the standard deviation to ~0.15 for the standard parameter values. Not surprisingly, power is quite good with this low ELT variance (comparable to that seen in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig8" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 8</a>). However, the analysis of <img src="picrender.fcgi?artid=2323809&amp;blobtype=equ&amp;blobname=M87" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M87" title="equation M87"> values described in the <span class="small-caps">methods</span> section indicates that ELTs with values this low may be very rare.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id144025" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id549642" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: METHODS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id551833" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: RESULTS</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>POWER TO MAP CAUSATIVE LOCI USING DIFFERENTIALLY EXPRESSED TRANSCRIPTS AS QUANTITATIVE TRAITS</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554522" class="sidefm-pmclink" style="text-transform: none;">DISCUSSION</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id555194" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id554252">POWER TO MAP CAUSATIVE LOCI USING DIFFERENTIALLY EXPRESSED TRANSCRIPTS AS QUANTITATIVE TRAITS</div><div class="section-content"><p>Above, I considered the power to detect differences in expression-level traits between disease-affected and -unaffected individuals. For causative ELTs, good power to detect association with the disease is achieved with <em>c</em> &#x0003d; 1 and sample sizes of ~100. With <em>c</em> &#x0003d; 1 (meaning that the ELT has a single controlling locus), the power to map the controlling locus will be good. Thus, in this case, eQTL-based strategies for genome scans should be powerful.</p><p>The situation is more complicated for reactive ELTs. Power was shown to be very poor if an additive penetrance model holds. Power for multiplicative penetrance depends strongly on the parameter <em>c</em>. If there is only a single TCL controlling an expression-level trait (<em>c</em> &#x0003d; 1), then power to detect an association between that gene and disease status is low unless <em>L</em> is small. If multiple TCL control the gene (<em>i.e.</em>, <em>c</em> &#x0003e; 1), then the power to detect associations with the disease is higher. However, this expression-level trait is then a multilocus trait and it will have lower power in use as a quantitative trait for detecting the TCL(s).</p><p>A number of methods for mapping QTL in natural populations have been proposed (see, <em>e.g.</em>, <a href="#bib24" class="cite-reflink">L<span class="small-caps">ynch</span> and W<span class="small-caps">alsh</span> 1998</a>). I will focus on humans and assume that mapping is conducted using Haseman&#x02013;Elston (HE) regression (<a href="#bib13" class="cite-reflink">H<span class="small-caps">aseman</span> and E<span class="small-caps">lston</span> 1972</a>). HE regression uses sib pairs and detects linkage to a QTL by regression of the squared difference in trait values on the number of alleles identical-by-descent (IBD) between the sibs at each marker. If a marker is linked to a QTL, then there should be a negative correlation between the squared trait difference and the IBD status at the marker.</p><p>I used a Monte Carlo simulation to estimate power for HE regression with an eQTL. I assumed a multiplicative model for gene expression with equally contributing loci. <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig9" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 9a</a> shows a plot of power <em>vs.</em> the parameter <em>c</em> for sample sizes of 100, 300, and 500 sib pairs. <em>M</em> &#x0003d; 1 in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig9" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 9a</a>. That is, there is one expression-level trait controlled by the eQTL. This expression-level trait being tested is assumed to be completely linked to one of the <em>c</em> controlling loci. A Bonferroni correction for 50 tests was applied (<em>i.e.</em>, &#x003b1;&#x0003d; 0.05/50 &#x0003d; 0.001). This is very a modest correction considering that there typically might be tens or hundreds of differentially expressed genes tested against each of hundreds of markers (note that a correction for 20,000 tests was used in previous sections of this article). We see that power drops quickly as <em>c</em> increases, with &#x0003c;10% power for <em>c</em> &#x0003d; 2 and 500 sib pairs.</p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig9" name="fig9"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig9" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2323809&amp;blobname=GEN17842201f9.gif" border="1" class="icon-reflink" alt="Figure 9. " title="Figure 9. "></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig9" onclick="startTarget(this, 'figure', 1024, 800)"><strong>F<span class="small-caps">igure</span> 9.&#x02014;</strong></a><div class="figure-table-caption-in-article"><span>Power to map eQTL using Haseman&#x02013;Elston regression. Power was calculated using a Monte Carlo simulation with 500 replicates. The marker being tested is assumed to be completely linked to one of the <em>c</em> controlling loci for the expression-level trait.</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig9" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>The power to detect a causative locus is increased if it is an eQTL for multiple expression-level traits. That is, if there are <em>M</em> expression-level traits controlled by the locus then there are <em>M</em> chances to detect it. However, these are not independent chances because the sample individuals are the same for each expression-level test. Thus, genotypes are not replicated. The expression-level traits are only independent conditioned on genotype.</p><p>The power curves in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig9" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 9b</a> correspond to <em>M</em> &#x0003d; 10, 20, 50, and 100 as shown. The sample size is 500 sib pairs in each case. The power is defined as the probability that the target causative locus is detected as an eQTL for at least one of the expression-level traits. Each of the <em>M</em> expression-level traits is assumed to be controlled by the same set of <em>c</em> loci. We see that the power increases strongly with <em>M</em>. However, <em>M</em> &#x0003d; 100 is required to achieve 80% power when <em>c</em> &#x0003d; 2 and the power is only 20% when <em>c</em> &#x0003d; 3.</p><p>The basic HE regression method assumed here is not the most powerful method available for detecting QTL in natural populations. However, other methods are not dramatically better and it is not expected that the qualitative results will be much different. Thus, we see that the power for detecting an eQTL is expected to be poor when the expression-level traits have more than one controlling locus, unless that locus is an eQTL for many expression-level traits.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id144025" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id549642" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: METHODS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id551833" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: RESULTS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554252" class="sidefm-pmclink" style="text-transform: none;">POWER TO MAP CAUSATIVE LOCI USING DIFFERENTIALLY EXPRESSED TRANSCRIPTS AS QUANTITATIVE TRAITS</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>DISCUSSION</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id555194" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id554522">DISCUSSION</div><div class="section-content"><p>There are too many unknowns to draw any firm conclusions about the likely effectiveness of mapping complex trait loci using gene expression levels as quantitative traits. There is strong evidence that a substantial number of gene expression-level traits in many organisms have a relatively simple genetic basis and high heritability relative to that seen for complex clinical traits. Furthermore, <a href="#bib31" class="cite-reflink">S<span class="small-caps">chadt</span> <em>et al.</em> (2003)</a>, B<span class="small-caps">rystrykh</span> <em>et al.</em> (2005), C<span class="small-caps">hessler</span> <em>et al.</em> (2005), <a href="#bib16" class="cite-reflink">H<span class="small-caps">ubner</span> <em>et al.</em> (2005)</a>, and others have identified probable associations of eQTL with clinical traits in mice or rats. Expression-level traits with high heritability will give good power for mapping their associated eQTL. However, I have shown here that establishing that these eQTL are also associated with the trait of interest may be difficult. If such expression-level traits are causative, in the sense of disease risk depending directly on expression level, then sample sizes on the order of 100 are sufficient for good power. On the other hand, if the ELT is reactive to a causative locus, then there is a trade-off between the power to show an association between expression-level trait and disease and the power to map eQTL for that expression-level trait. The simpler the genotypic dependence of an expression-level trait is on a disease-causative eQTL, the easier it is to map that eQTL. From a mapping perspective, the ideal case is that the expression-level trait depends on a single causative locus. However, in this case the correlation with the disease will be small. Most of the results in this article with <em>c</em> &#x0003d; 1 and <em>L</em> &#x0003e; 3 for a reactive ELT show poor power to detect differential expression between affecteds and unaffecteds unless sample size is well over 500 individuals. This power increases quickly as <em>c</em> does, although the sample sizes required for 80% power are all rather high compared to most current microarray experiments. Unfortunately, the power to map causative loci drops quickly with <em>c</em>. <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig9" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 9</a> shows that for Haseman&#x02013;Elston regression the power to detect linkage is &#x0003c;10% for <em>c</em> &#x0003e; 1 even with 500 sib pairs. This indicates that for humans there may be no expression-level traits that give good power both to detect reactive expression-level differences between affecteds and unaffecteds and to map their underlying causative loci. The underlying problem is that power is poor for mapping QTL in natural populations. This is somewhat ameliorated if the disease locus is an eQTL for many expression-level traits. However, there must be on the order of hundreds of controlled expression-level traits before power becomes reasonable. I stress again, however, that these power calculations are not meant to be taken too literally because there are many other possible methods for doing genome scans using eQTL. Thus, the results are intended more as a guide to intuition.</p><p>Results that will appear in a future publication (P. S<span class="small-caps">chliekelman</span>, unpublished results) show that the situation is better for crosses with inbred lines. The power to detect expression-level differences is not substantially higher, but the power to detect linkage drops off more slowly as the number of loci (<em>c</em>) controlling the expression-level trait increases.</p><p>These results do not imply that no reactive expression-level changes should be detected with natural populations. On the contrary, the power to detect such differences is quite reasonable for expression-level traits influenced by multiple causative loci (<em>i.e.</em>, expression-level traits with <em>c</em> &#x0003e; 1). The point is that the reactive expression-level traits that are found to be most differentially expressed are likely to be poor candidates for use as quantitative traits for linkage detection.</p><p><span class="head3">Model assumptions: </span> The results of this article depend on a large number of assumptions about how disease risk and expression levels depend on genotype. This is unavoidable given our lack of knowledge of the genetics of complex traits. I have used two very different disease-penetrance models and a wide range of parameter values to give a sense of what the range of possibilities is. However, in the end the results of a model are only as good as the assumptions that go into it and it is possible that these assumptions are badly off.<p>In a sense, the additive and multiplicative models are at extreme ends of a continuum. A multiplicative model requires that each of the causative loci have a disease event for the disease to occur. The additive model requires only a single locus to have a disease event for disease to occur. As shown in this article, this difference results in very different genotype distributions. I have focused on the multiplicative model because the power to detect reactive expression-level differences is very poor under an additive model and the additive model does not appear consistent with a causative ELT. If an additive model is correct, then there appears to be little hope for mapping complex disease genes using eQTL. Fortunately, additive disease-penetrance models do not appear to be consistent with family history data in humans. <a href="#bib29" class="cite-reflink">R<span class="small-caps">isch</span> (1990)</a> showed that inheritance patterns do not change as the number of causative loci increases under an additive model. Thus, if the inheritance patterns for a disease indicate a multilocus trait, then it cannot be additive. Multiplicative models typically do fit family history data well (<em>e.g.</em>, <a href="#bib33" class="cite-reflink">S<span class="small-caps">chliekelman</span> and S<span class="small-caps">latkin</span> 2002</a>). However, it is likely that a great many other models would fit equally well and there is no way to tell if the &#x0201c;true&#x0201d; model behaves more like an additive model, like a multiplicative model, or like neither model.</p><p>The values of <em>M</em> (the number of conditionally independent expression-level traits influenced by the TCL) assumed here are also highly speculative. Surveys of eQTL have found great variability in the number of gene expression-level traits mapping to individual loci. <a href="#bib31" class="cite-reflink">S<span class="small-caps">chadt</span> <em>et al.</em> (2003)</a> found several hotspots with hundreds of eQTL mapping to 4-cM regions. <a href="#bib7" class="cite-reflink">C<span class="small-caps">hesler</span> <em>et al.</em> (2005)</a> found a single locus modulating the expression of 1650 transcripts in addition to numerous loci modulating hundreds of transcripts. On the other hand, <a href="#bib25" class="cite-reflink">M<span class="small-caps">onks</span> <em>et al.</em> (2004)</a> did not find evidence of such hotspots. Their study was conducted with randomly chosen human families whereas studies that have found hotspots were conducted with inbred lines. One possible explanation for the difference is that there is little variation at hotspot loci in natural populations, but selection for divergent phenotypes (or perhaps chance) in inbred lines leads to genotypic differences between strains. Another possibility is that expression-level traits with multilocus inheritance in natural populations become monogenic between inbred lines and thus are easier to detect. In any event, an increase in <em>M</em> increases the power to detect at least one expression-level trait associated with the TCL and a causative locus with very high <em>M</em> would be detectable as differentially expressed with reasonable sample sizes even for <em>c</em> &#x0003d; 1 (see <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 3</a>). However, <em>M</em> is the effective number of <em>independent</em> (given TCL genotype) expression-level traits influenced by the TCL. The amount of correlation between expression-level traits in the above studies is unknown. Most likely they are not all independent and the effective number of independent expression-level traits may be substantially lower than the observed number.</p></p><p><span class="head3">Comparisons to standard methods for genome scans: </span> The required sample sizes shown here are large by the standards of typical microarray experiments, but not outrageous on the scale of human linkage studies. Power studies (<em>e.g.</em>, <a href="#bib30" class="cite-reflink">R<span class="small-caps">isch</span> and M<span class="small-caps">erikangas</span> 1996</a>; <a href="#bib34" class="cite-reflink">S<span class="small-caps">lager</span> <em>et al.</em> 2000</a>) have shown that sample sizes of many thousands might be required to detect complex trait loci in genome scans. There are too many unknowns to say whether eQTL will provide a more effective method for mapping complex trait loci than do conventional mapping strategies. If there are causative ELTs for many complex traits, then the results here indicate that eQTL-based strategies may be very effective. The central point of this article with respect to reactive eQTL is that high power in mapping of eQTL is traded for low power in correlating the eQTL with traits of interest. It remains to be determined exactly how low. If we assume that transcripts with high heritability relative to causative loci (and thus linkage or association is easy to detect) exist, then the fundamental question is how the power to detect correlations between expression-level traits and disease compares to the power to detect linkage or association between the disease and the causative loci. There is reason for optimism. If there are typically many genes with expression levels mapping to single eQTL (<em>i.e.</em>, high <em>M</em>) and if &#x003c4;-values are typically very high, then required sample sizes drop down to the range of &#x02264;200. Furthermore, the experimental design and method of data analysis that I have assumed can certainly be improved (see the next section).<p>The results here show that the power to detect association of a causative ELT with a disease locus compares very favorably with the power to detect complex trait loci using traditional mapping methods. Why should this be so? That is, the locus controlling the ELT in <a href="articlerender.fcgi?artid=2323809&amp;rendertype=figure&amp;id=fig8" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 8</a> accounts for one-ninth of the disease risk. Numerous gene-mapping power studies show that the power to detect such a locus is poor. Why is the power apparently better to detect disease association with an ELT that accounts only for one-ninth of the risk? The answer to this can be understood by analogy with association studies for complex traits. It is well established (<em>e.g.</em>, <a href="#bib30" class="cite-reflink">R<span class="small-caps">isch</span> and M<span class="small-caps">erikangas</span> 1996</a>) that association studies are powerful for detecting common disease alleles when there are only two alleles in the population. However, power in association studies plummets when there are multiple alleles (<a href="#bib34" class="cite-reflink">S<span class="small-caps">lager</span> <em>et al.</em> 2000</a>). In detecting association of a causative eQTL with a complex trait, the ELT is a proxy for genotype. If the genotype-specific expression means were widely enough separated, then we could unambiguously assign individuals to groups corresponding to genotype and conduct an association test directly. We do a similar thing when we do a <em>t</em>-test of an ELT determined by a mixture of expression distributions. We will tend to find a significant difference in expression between affecteds and unaffecteds if the allele frequency at the locus controlling the transcript is substantially different between affecteds and unaffecteds, which is the same requirement for an association test. It can be shown (P. S<span class="small-caps">chliekelman</span>, unpublished data) that there are strong similarities between the functional forms of the noncentrality parameters that determine power for a case&#x02013;control test and for a <em>t</em>-test of an ELT determined by a mixture of genotype-specific expression distributions and that these tests have similar dependence on population genetic parameters. A key advantage of an ELT approach is that there are effectively only two allele types. That is, all alleles that increase expression (and therefore disease risk in this study) are collapsed into a single disease-risk-increasing group and all alleles that decrease expression are grouped into a risk-decreasing group. Thus, testing for an association between an ELT and a clinical trait has a strong similarity to association testing with just two alleles.</p><p>A key advantage of ELT-based methods is that they identify direct correlations to genes and not to markers. Even if the identified gene is reactive only to a disease locus and not disease causative, there is still is no recombination with a marker to diminish power. It appears that establishing association between transcripts and trait will be the low-power stage in genome scans using eQTL, while establishing linkage between a transcript and a marker will be relatively easy. Having no recombination involved in the low-power stage may be a major advantage for eQTL genome scans. Of course, there will also likely be genes whose expression level is controlled by a locus linked to a disease locus and where recombination is a factor. Such genes will also be correlated with the disease, but more weakly if all else is the same.</p></p><p><span class="head3">Improvements in power: </span> There appears to be substantial potential for increasing the power of genome scans using eQTL beyond what I have shown here. <a href="#bib31" class="cite-reflink">S<span class="small-caps">chadt</span> <em>et al.</em> (2003)</a> stressed the importance of combining information from all transcripts instead of focusing on transcripts individually as I have done here. Several studies (<a href="#bib9" class="cite-reflink">G<span class="small-caps">hazalpour</span> <em>et al.</em> 2005</a>, <a href="#bib10" class="cite-reflink">2006</a>; <a href="#bib23" class="cite-reflink">L<span class="small-caps">i</span> <em>et al.</em> 2006</a>) have used approaches that combine information from many transcripts in what are essentially genome scans. For example, <a href="#bib10" class="cite-reflink">G<span class="small-caps">hazalpour</span> <em>et al</em>. (2006)</a> identified 12 transcription modules using gene coexpression networks constructed from expression data for 3421 genes in an F<sub>2</sub> cross between inbred mice strains. They then looked for correlations between these transcription modules and obesity-related clinical traits by calculating the average correlation between the traits and genes in the module. Finally, they mapped eQTL for all genes within the modules and looked for genomic regions that were enriched for eQTL for each module. <em>cis-</em>eQTL in these regions were then taken as candidate genes for traits correlated with the corresponding transcription module. For example, they found one module significantly correlated with body weight and nine genomic regions enriched for this module. One of the major advantages of this approach relative to the assumptions of the analysis presented here is that the gene module approach greatly reduces the number of tests for correlation (that is, from thousands down to 12). This greatly increases the power to detect a correlation with the trait. A potential drawback is that there is no way to tell whether a particular candidate region is causative for the trait or is causative only for the module. Still, this type of approach seems very promising.<p>Experimental designs utilizing family structure will reduce the number of segregating loci between affected and unaffected subjects and this can be used to increase power for detecting expression-level differences associated with causative loci. <a href="#bib20" class="cite-reflink">K<span class="small-caps">raft</span> <em>et al.</em> (2003)</a> proposed a family-based test for correlation between gene expression and a quantitative trait. They presented power simulations showing very good power for their proposed test, but did not model the genetics of the trait or expression level. Instead they assumed correlations of 33 or 67% between trait and expression level. The results here indicate that this assumed correlation is far too high for complex traits. While their method is promising, the power cannot be expected to be as high as their simulations show.</p><p><a href="#bib28" class="cite-reflink">P<span class="small-caps">erez</span>-E<span class="small-caps">nciso</span> <em>et al.</em> (2003)</a> proposed a partial least-squares regression approach to construct &#x0201c;supergenes&#x0201d; composed of linear combinations of transcripts chosen to maximize correlation with the trait of interest. This linear combination of expression levels is then used as a quantitative trait for mapping QTL. Simulations showed that this method gives substantial improvements in power over using the trait directly for QTL mapping. <a href="#bib28" class="cite-reflink">P<span class="small-caps">erez</span>-E<span class="small-caps">nciso</span> <em>et al.</em> (2003)</a> assumed a large QTL effect compared to the effects for causative loci shown in this article and as a result showed good power to detect the QTL for many scenarios. It is not clear that the increase in power for their method would be sufficient to make much difference if the QTL effect is small. The results of this article indicate that we might expect large groups of transcripts controlled by individual causative loci to have a weak association with the trait. This suggests that methods of analysis that use joint association of sets of genes with the disease might be an effective strategy.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id555190">Acknowledgments</div><div class="section-content"><p><p>I thank Mark Schliekelman and several anonymous reviewers for helpful comments on this manuscript.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id144025" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id549642" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: METHODS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id551833" class="sidefm-pmclink" style="text-transform: none;">POWER TO DETECT EXPRESSION LEVEL DIFFERENCES DUE TO POLYMORPHISMS BETWEEN AFFECTED AND UNAFFECTED INDIVIDUALS: RESULTS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554252" class="sidefm-pmclink" style="text-transform: none;">POWER TO MAP CAUSATIVE LOCI USING DIFFERENTIALLY EXPRESSED TRANSCRIPTS AS QUANTITATIVE TRAITS</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id554522" class="sidefm-pmclink" style="text-transform: none;">DISCUSSION</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id555194">References</div><div class="section-content"><div class="back-matter-section"><ul class="back-ref-list-1st-line-shifted"><li id="bib1">Altmuller, J., L. J. Palmer, G. Fischer, H. Scherb and M. Wjst, 2001 Genomewide scans of complex human diseases: true linkage is hard to find. <span class="ref-journal">Am. J. Hum. Genet.</span> <span class="ref-vol">69:</span> 936&#x02013;950.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0zWnzRWkPJlRTsYg0MArTKnq6L8ZXTrWd7vr7GQam&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11565063">PubMed</a>]</span>.</li><li id="bib2">Benjamini, Y., and Y. Hochberg, 1995 Controlling the false discovery rate - a practical and powerful approach to multiple testing. <span class="ref-journal">J. R. Stat. Soc. Ser. B Methodol.</span> <span class="ref-vol">57:</span> 289&#x02013;300.</li><li id="bib3">Berge, K. E., H. Tian, G. A. Graf, L. Yu, N. V. Grishin <em>et al.</em>, 2000 Accumulation of dietary cholesterol in sitosterolemia caused by mutations in adjacent ABC transporters. <span class="ref-journal">Science</span> <span class="ref-vol">290:</span> 1771&#x02013;1775.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0xcjwTPhax_9GbYIEBcTKr2rTEcFz2DiMbuzbKkaZ&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11099417">PubMed</a>]</span>.</li><li id="bib4">Brem, R. B., and L. Kruglyak, 2005 The landscape of genetic complexity across 5,700 gene expression traits in yeast. <span class="ref-journal">Proc. Natl. Acad. Sci. USA</span> <span class="ref-vol">102:</span> 1572&#x02013;1577.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0dcNMwPKfg58RIu9bKINUUDF02YdVtr6gqHf2uA2j&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15659551">PubMed</a>]</span>.</li><li id="bib5">Brem, R. B., G. Yvert, R. Clinton and L. Kruglyak, 2002 Genetic dissection of transcriptional regulation in budding yeast. <span class="ref-journal">Science</span> <span class="ref-vol">296:</span> 752&#x02013;755.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0lcJzdrDdohCQyVxFK30OHEOzUBz8WLyE_YvPGA71&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11923494">PubMed</a>]</span>.</li><li id="bib6">Bystrykh, L., E. Weersing, B. Dontje, S. Sutton, M. T. Pletcher <em>et al.</em>, 2005 Uncovering regulatory pathways that affect hematopoietic stem cell function using &#x02018;genetical genomics&#x02019;. <span class="ref-journal">Nat. Genet.</span> <span class="ref-vol">37:</span> 225&#x02013;232.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0fMZ7BM-G28HLMOQ2E9iEMv4dmVmm00cSPF2g4QQF&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15711547">PubMed</a>]</span>.</li><li id="bib7">Chesler, E. J., L. Lu, S. Shou, Y. Qu, J. Gu <em>et al.</em>, 2005 Complex trait analysis of gene expression uncovers polygenic and pleiotropic networks that modulate nervous system function. <span class="ref-journal">Nat. Genet.</span> <span class="ref-vol">37:</span> 233&#x02013;242.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0B1t2EYEIzhqe7rbDgtQ0uIuSq14o27SP4E0LP58R&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15711545">PubMed</a>]</span>.</li><li id="bib8">Cheung, V. G., L. K. Conlin, T. M. Weber, M. Arcaro, K. Y. Jen <em>et al.</em>, 2003 Natural variation in human gene expression assessed in lymphoblastoid cells. <span class="ref-journal">Nat. Genet.</span> <span class="ref-vol">33:</span> 422&#x02013;425.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0lozRx0Rnok_vsG_8k_2phF5m6JwRhEJy6lni_Qhe&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12567189">PubMed</a>]</span>.</li><li id="bib9">Ghazalpour, A., S. Doss, S. S. Sheth, L. A. Ingram-Drake, E. E. Schadt <em>et al.</em>, 2005 Genomic analysis of metabolic pathway gene expression in mice. <span class="ref-journal">Genome Biol.</span> <span class="ref-vol">6:</span> R59.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hGO1tUNnPO-JXUH6kbqKvixavMjrD5dEc2GmEAxG&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15998448">PubMed</a>]</span>.</li><li id="bib10">Ghazalpour, A., S. Doss, B. Zhang, S. Wang, C. Plaisier <em>et al.</em>, 2006 Integrating genetic and network analysis to characterize genes related to mouse weight. <span class="ref-journal">PLoS Genet.</span> <span class="ref-vol">2:</span> 1182&#x02013;1192.</li><li id="bib11">Gibson, G., and B. Weir, 2005 The quantitative genetics of transcription. <span class="ref-journal">Trends Genet.</span> <span class="ref-vol">21:</span> 616&#x02013;623.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0bsbMxO-IbvL2pztF-3jiyO3rSZZCX1xsromDcQh7&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16154229">PubMed</a>]</span>.</li><li id="bib12">Glazier, A. M., J. H. Nadeau and T. J. Aitman, 2002 Finding genes that underlie complex traits. <span class="ref-journal">Science</span> <span class="ref-vol">298:</span> 2345&#x02013;2349.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0ZAAdJSt9JeoaJg4k4jN7IVRf2b_nOJhSlbVckkFp&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12493905">PubMed</a>]</span>.</li><li id="bib13">Haseman, J. K., and R. C. Elston, 1972 The investigation of linkage between a quantitative trait and a marker locus. <span class="ref-journal">Behav. Genet.</span> <span class="ref-vol">2:</span> 3&#x02013;19.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0x1RHZCqMxFvWYy_KU2DEilkSNYQ1jgkSfsp3Rk87&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/4157472">PubMed</a>]</span>.</li><li id="bib14">Hedenfalk, I., D. Duggan, Y. Chen, M. Radmacher, M. Bittner <em>et al.</em>, 2001 Gene-expression profiles in hereditary breast cancer. <span class="ref-journal">N. Engl. J. Med.</span> <span class="ref-vol">344:</span> 539&#x02013;548.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=06oqlHNI3OwzbXIQrINmRF4cJmTYBlE84j1LyC5lH&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11207349">PubMed</a>]</span>.</li><li id="bib15">Hsieh, W. P., G. Passador-Gurgel, E. A. Stone and G. Gibson, 2007 Mixture modeling of transcript abundance classes in natural populations. <span class="ref-journal">Genome Biol.</span> <span class="ref-vol">8:</span> R98.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0LKm56um79Uvdatl_XtZ9lKETqMZ41tyNQeitvQDr&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17547747">PubMed</a>]</span>.</li><li id="bib16">Hubner, N., C. A. Wallace, H. Zimdahl, E. Petretto, H. Schulz <em>et al.</em>, 2005 Integrated transcriptional profiling and linkage analysis for identification of genes underlying disease. <span class="ref-journal">Nat. Genet.</span> <span class="ref-vol">37:</span> 243&#x02013;253.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=00ABqPwqp9j3ieAvwaUlvYSJKzZcE6yhCWY1hfklg&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15711544">PubMed</a>]</span>.</li><li id="bib17">Karp, C. L., A. Grupe, E. Schadt, S. L. Ewart, M. Keane-Moore <em>et al.</em>, 2000 Identification of complement factor 5 as a susceptibility locus for experimental allergic asthma. <span class="ref-journal">Nat. Immunol.</span> <span class="ref-vol">1:</span> 221&#x02013;226.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0R39bbpvF-Cm3fFTivT3T4gi7QY78xMtrUe7dLkry&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10973279">PubMed</a>]</span>.</li><li id="bib18">Kendziorski, C., and P. Wang, 2006 A review of statistical methods for expression quantitative trait loci mapping. <span class="ref-journal">Mamm. Genome</span> <span class="ref-vol">17:</span> 509&#x02013;517.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0teNFKG7ijOrzH7PkmliJZ6rydYOSHJJsDHLm6kF6&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16783633">PubMed</a>]</span>.</li><li id="bib19">Kirst, M., A. A. Myburg, J. P. G. De Leon, M. E. Kirst, J. Scott <em>et al.</em>, 2004 Coordinated genetic regulation of growth and lignin revealed by quantitative trait locus analysis of cDNA microarray data in an interspecific backcross of eucalyptus. <span class="ref-journal">Plant Physiol.</span> <span class="ref-vol">135:</span> 2368&#x02013;2378.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0DMwC7QgWs4ta7z32xSWZNC9Rn4JeiXxkAXzjO5KR&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15299141">PubMed</a>]</span>.</li><li id="bib20">Kraft, P., E. Schadt, J. Aten and S. Horvath, 2003 A family-based test for correlation between gene expression and trait values. <span class="ref-journal">Am. J. Hum. Genet.</span> <span class="ref-vol">72:</span> 1323&#x02013;1330.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0JQW4YFMXq9gfKI1Snbljb6uCV468hSuwlloHF5we&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12687497">PubMed</a>]</span>.</li><li id="bib21">Lawn, R. M., D. P. Wade, M. R. Garvin, X. Wang, K. Schwartz <em>et al.</em>, 1999 The Tangier disease gene product ABC1 controls the cellular apolipoprotein-mediated lipid removal pathway. <span class="ref-journal">J. Clin. Invest.</span> <span class="ref-vol">104:</span> R25&#x02013;R31.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0-QjtNi1ClUu_YsYoVOxWXzi4amuPpgzPDki3jADx&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10525055">PubMed</a>]</span>.</li><li id="bib22">Li, H., L. Lu, K. F. Manly, E. J. Chesler, L. Bao <em>et al.</em>, 2005 Inferring gene transcriptional modulatory relations: a genetical genomics approach. <span class="ref-journal">Hum. Mol. Genet.</span> <span class="ref-vol">14:</span> 1119&#x02013;1125.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0ReZDDFOjvNlMj8uSrDSj7U9HR9jGgPI1FrbKMQUp&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15772094">PubMed</a>]</span>.</li><li id="bib23">Li, H. Q., H. Chen, L. Bao, K. F. Manly, E. J. Chesler <em>et al.</em>, 2006 Integrative genetic analysis of transcription modules: towards filling the gap between genetic loci and inherited traits. <span class="ref-journal">Hum. Mol. Genet.</span> <span class="ref-vol">15:</span> 481&#x02013;492.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0RgDZ2BFRAlcguyoaQeTmiMOgQLRIhZQQCJ0RwAm3&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16371421">PubMed</a>]</span>.</li><li id="bib24">Lynch, M., and B. Walsh, 1998 <em>Genetics and Analysis of Quantitative Traits.</em> Sinauer, Sunderland, MA.</li><li id="bib25">Monks, S. A., A. Leonardson, H. Zhu, P. Cundiff, P. Pietrusiak <em>et al.</em>, 2004 Genetic inheritance of gene expression in human cell lines. <span class="ref-journal">Am. J. Hum. Genet.</span> <span class="ref-vol">75:</span> 1094&#x02013;1105.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=02oWk2o6byTZzeADleOHEsOAG0SiUyXJ89wHvVA-O&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15514893">PubMed</a>]</span>.</li><li id="bib26">Morley, M., C. M. Molony, T. M. Weber, J. L. Devlin, K. G. Ewens <em>et al.</em>, 2004 Genetic analysis of genome-wide variation in human gene expression. <span class="ref-journal">Nature</span> <span class="ref-vol">430:</span> 743&#x02013;747.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=08ChhmHc0n12_K9gS1brSqDxCxp5fyKKQiUgoOAeV&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15269782">PubMed</a>]</span>.</li><li id="bib27">Oleksiak, M. F., G. A. Churchill and D. L. Crawford, 2002 Variation in gene expression within and among natural populations. <span class="ref-journal">Nat. Genet.</span> <span class="ref-vol">32:</span> 261&#x02013;266.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0NoehjcnLAzgi_pn9pFMOI4MBWlNdzX-ZHgCo0A4e&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12219088">PubMed</a>]</span>.</li><li id="bib28">Perez-Enciso, M., M. A. Toro, M. Tenenhaus and D. Gianola, 2003 Combining gene expression and molecular marker information for mapping complex trait genes: a simulation study. <span class="ref-journal">Genetics</span> <span class="ref-vol">164:</span> 1597&#x02013;1606.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0zQR4FjapX6fy7TK-YGTS4BW79UGU3LVJhn69c56g&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12930763">PubMed</a>]</span>.</li><li id="bib29">Risch, N., 1990 Linkage strategies for genetically complex traits. I. Multilocus models. <span class="ref-journal">Am. J. Hum. Genet.</span> <span class="ref-vol">46:</span> 222&#x02013;228.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0HG_FxQCL4WBGhJ5sqpaIK1fDPjvgor8qwN3A_A1L&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/2301392">PubMed</a>]</span>.</li><li id="bib30">Risch, N., and K. Merikangas, 1996 The future of genetic studies of complex human diseases. <span class="ref-journal">Science</span> <span class="ref-vol">273:</span> 1516&#x02013;1517.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=04c289jq8sMSpogWODMgd19C_8Hl9z2zYztqSB5Ih&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8801636">PubMed</a>]</span>.</li><li id="bib31">Schadt, E. E., S. A. Monks, T. A. Drake, A. J. Lusis, N. Che <em>et al.</em>, 2003 Genetics of gene expression surveyed in maize, mouse and man. <span class="ref-journal">Nature</span> <span class="ref-vol">422:</span> 297&#x02013;302.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=007nMfNLkcwZrggXRj7SFU1dk98C_Dkp5TYJzeQSZ&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12646919">PubMed</a>]</span>.</li><li id="bib32">Schadt, E. E., J. Lamb, X. Yang, J. Zhu, S. Edwards <em>et al.</em>, 2005 An integrative genomics approach to infer causal associations between gene expression and disease. <span class="ref-journal">Nat. Genet.</span> <span class="ref-vol">37:</span> 710&#x02013;717.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=00oyoc6C8LxMJexUe60HncOOP87fBS8UC3nXDl5mm&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15965475">PubMed</a>]</span>.</li><li id="bib33">Schliekelman, P., and M. Slatkin, 2002 Multiplex relative risk and estimation of the number of loci underlying an inherited disease. <span class="ref-journal">Am. J. Hum. Genet.</span> <span class="ref-vol">71:</span> 1369&#x02013;1385.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0NUSC59NT3gEvS1sfCwNuwrmoFBQT9WWYkJqIm5dt&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12454800">PubMed</a>]</span>.</li><li id="bib34">Slager, S. L., J. Huang and V. J. Vieland, 2000 Effect of allelic heterogeneity on the power of the transmission disequilibrium test. <span class="ref-journal">Genet. Epidemiol.</span> <span class="ref-vol">18:</span> 143&#x02013;156.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0-SAMIvPTTERzunQGoVTjU9ewoO8nLTVXjQv9D5xz&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10642427">PubMed</a>]</span>.</li><li id="bib35">Wayne, M. L., and L. M. McIntyre, 2002 Combining mapping and arraying: an approach to candidate gene identification. <span class="ref-journal">Proc. Natl. Acad. Sci. USA</span> <span class="ref-vol">99:</span> 14903&#x02013;14906.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0PQWWolpfp-KVVrQid0tob84mL1Wet_EdPmnkRQP4&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12415114">PubMed</a>]</span>.</li><li id="bib36">Yan, H., W. Yuan, V. E. Velculescu, B. Vogelstein and K. W. Kinzler, 2002 Allelic variation in human gene expression. <span class="ref-journal">Science</span> <span class="ref-vol">297:</span> 1143.  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pKOMz0mX7h2xOeVIzNyBGq24lT4kMvcvdDqCq5fe&amp;reftype=pubmed&amp;artid=2323809&amp;article-id=2323809&amp;iid=164313&amp;issue-id=164313&amp;jid=301&amp;journal-id=301&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12183620">PubMed</a>]</span>.</li></ul></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">Genetics</span> are provided here courtesy of <br><b>Genetics Society of America</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc1-->

