<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
        "http://www.w3.org/TR/html4/loose.dtd"> 
<HTML LANG="EN">
<!-- #BeginTemplate "/Templates/Graphic-Version.dwt" --> 
<HEAD>
<!-- #BeginEditable "doctitle" --> <TITLE>NIAAA Publications</TITLE> <LINK REL="stylesheet" HREF="journal.css" TYPE="text/css"><!-- #EndEditable --> 
<meta NAME="keywords" CONTENT="Alcohol Alerts; Alcohol Research Health; Newsletters; pamphlets; brochures; professional education; research monographs; reports; Publicaciones en Español; Delivery Treatment Prevention Services; Tenth Report Congress; Epidemiologic Data Directory; Data Reference manuals; Surveillance; Pamphlets, Brochures; AgePage Aging; Alcoholism: Getting Facts; Family History; Drinking Your Pregnancy; Harmful Interactions Mixing Alcohol Medicine; Health Sciences Education Curricula; Disparities Materials; Cut Down; What You Don't Know Can Harm You; Affect Child; Kids Alcohol Free; Talk Child About Alcohol; Science Education; Parents Help Teens Party Right Graduation; Underage Drinking Prevention; Intimate Relationships; COMBINE; Helping Patients Who Drink Too Much; Clinicians guide; Pocket Guide; economic costs; physician training; MATCH; assessing problems guide; sentencing DUI offenders; thesaurus; residence program New York; improving treatment practice North Carolina; Social Work Curriculum; State of Science Moderate Drinking; NASADAD Issue Brief one; Un tema de salud de la mujer ; Hable con sus Hijos Sobre el Uso del Alcohol ; Antecedentes de alcoholismo en la familia; La bebida y su embarazo; Mantengamos A Los Niño Libres De Alcohol; Mezclando bebidas alcohólicas con medicamentos"> 
<META http-equiv=Content-Type content="text/html; charset=iso-8859-1"><LINK 
href="../../style.css" type=text/css rel=stylesheet> <LINK 
href="file:///C|/Documents%20and%20Settings/zahral/newsletter.css" type=text/css rel=stylesheet> 
<META content="MSHTML 6.00.2800.1400" name=GENERATOR> 
</HEAD>
<BODY leftMargin=0 topMargin=0 marginwidth="0" marginheight="0" vlink="#666666" alink="#6666FF">
<!-- BEGIN HEADER --> <br> <TABLE cellSpacing=0 cellPadding=0 width="100%" border=0 Summary="This is a layout table"> 
<TBODY> <TR> <TD width="50%" rowspan="2">&nbsp;&nbsp;&nbsp;<img src="../../images/NIAAA_newest_logo.gif" width="457" height="70" alt="National Institute on Alcohol Abuse and Alcoholism banner" border="0"></TD></TR> 
<TR> <TD width="50%" valign="top" align="center"> <form method="get" action="http://search2.google.cit.nih.gov/search" name="search"> 
<input tabindex="1" type="hidden" name="site" value="NIAAA"/> <input tabindex="2" type="hidden" name="client" value="NIAAA_frontend"/> 
<input tabindex="3" type="hidden" name="proxystylesheet" value="NIAAA_frontend"/> 
<input tabindex="4" type="hidden" name="output" value="xml_no_dtd"/> <input tabindex="5" type="hidden" name="filter" value="0"/> 
<input tabindex="6" type="hidden" name="getfields" value="*"/> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
<br> &nbsp;&nbsp;<font face="Arial, Helvetica, sans-serif" size="1"><b>SITE SEARCH</b></font>&nbsp; 
<input tabindex="7" id="searchbox" type="text" name="q" size="19" maxlength="255" value="  "/> 
<input tabindex="8" id=btnGo 
            onClick="RedirectPopUp('http://search.google.cit.nih.gov/search',this.form);" 
            type=image height=26 width=49 src="../../images/goBtn.gif" 
            value=Go name="image22"> <br> <a tabindex="9" 
            href="http://www.niaaa.nih.gov/SearchTips.htm"><img 
            height=12 alt="Advanced Search Page" 
            src="../../images/advSearchPage.gif" width=112 
            border=0></a> </form></TD></TR> </TBODY> </TABLE><!-- END HEADER --> 
<TABLE cellSpacing=0 cellPadding=0 width="100%" bgColor=#003a91 border=0 Summary="This is a layout table"> 
<TBODY> <TR> <TD><img src="../../images/Publications.jpg" width="760" height="55" alt="Publications" border="0"></TD></TR></TBODY></TABLE><!-- BEGING MAIN CONTENT --> 
<font size="2" face="Arial, Helvetica, sans-serif">Back to:</font><font size="1" face="Arial, Helvetica, sans-serif"> 
</font><font size="1"><font face="Arial, Helvetica, sans-serif"><a tabindex="10" href="http://www.niaaa.nih.gov">NIAAA 
Home</a></font></font><font size="1" face="Arial, Helvetica, sans-serif"> &gt;<a tabindex="11" href="http://www.niaaa.nih.gov/Publications/"> 
Publications </a></font> <table width="100%" border="0" Summary="This is a layout table"> 
<tr> <td width="74%" valign="top"> <div align="center"><!-- #BeginEditable "Insert%20a%20heading%20title%20here" --> 
<h2>&nbsp;</h2><!-- #EndEditable --></div></td><td width="26%" valign="top"><!-- #BeginEditable "Put%20the%20PDF%20link%20here" --><!-- #EndEditable --></td></tr> 
</table><font face="Arial, Helvetica, sans-serif" size="2"><!-- #BeginEditable "Please%20insert%20the%20body%20text%20here" --> 
<h3><em>Technologies from the Field</em></h3><h1>QUANTITATIVE TRAIT LOCUS ANALYSIS: 
<em>MULTIPLE CROSS AND HETEROGENEOUS STOCK MAPPING</em></h1><h3><em>Robert Hitzemann, 
Ph.D.; John K. Belknap, Ph.D.; and Shannon K. McWeeney, Ph.D.</em></h3><p><em></em>ROBERT 
HITZEMANN, PH.D., is a professor in and chair of the Department of Behavioral 
Neuroscience, Oregon Health &amp; Science University, and a research pharmacologist 
in the Research Service, Portland VA Medical Center, both in Portland, Oregon. 
</p><p>JOHN K. BELKNAP, PH.D., is a professor in the Department of Behavioral 
Neuroscience, Oregon Health &amp; Science University and a senior career scientist 
in the Research Service, Portland VA Medical Center, both in Portland, Oregon. 
</p><p>SHANNON K. MCWEENEY, PH.D., is an associate professor in the Division of 
Biostatistics in the Department of Public Health and Preventative Medicine and 
an associate professor in the Department of Medical Informatics and Clinical Epidemiology, 
Oregon Health &amp; Science University, Portland, Oregon.</p><p><em>KEY WORDS: 
Genetic theory of alcohol and other drug use; genetic factors; environmental factors; 
behavioral phenotype; behavioral trait; quantitative trait gene (QTG); inbred 
animal strains; recombinant inbred (RI) mouse strains; quantitative traits; quantitative 
trait locus (QTL) mapping; multiple cross mapping (MCM); heterogeneous stock (HS) 
mapping; microsatellite mapping; animal models</em></p><p> Until well into the 
1990s, both preclinical and clinical research focused on finding &#8220;the&#8221; 
gene for human diseases, including alcoholism. This focus was reinforced by the 
emergence of technologies to either inactivate (i.e., knock out) a gene or add 
extra copies of an existing gene in a living organism, which clearly demonstrated 
that over- or underexpressing a single gene could have a profound effect on behavior. 
However, a small but vocal group of scientists, including many alcohol researchers, 
argued that behaviors, including alcohol-related behaviors, were complex traits 
and therefore no one gene likely would have a large effect. This view was consistent 
with a large body of genetic research conducted in plants and fruit flies (e.g., 
Paterson et al. 1988) indicating that, for example, even a presumably simple characteristic, 
such as the size of a tomato, was determined by several genes. However, it was 
difficult to convince the scientific community that, in terms of its genetic determination, 
behavior was similar to the size of a tomato. Only with the advent of new genetic 
tools did it become possible to prove that many different genes contribute to 
complex behavioral characteristics. These tools included the following (see Phillips 
2002): </p><ul> <li>Panels of recombinant inbred (RI) mouse strains. RI strains 
generally are generated by repeatedly inbreeding brother&#8211;sister pairs from 
the second-generation (F<sub>2</sub>) offspring of two genetically distinct parent 
inbred strains. Each F<sub>2</sub> animal has a slightly different combination 
of the parental genes. By repeated inbreeding of brother&#8211; sister pairs, 
researchers can generate numerous distinct inbred animal strains. </li><li>Quantitative 
trait locus (QTL) mapping. Quantitative traits are characteristics such as height 
or sensitivity to alcohol that differ in the extent to which an individual possesses 
that characteristic. The variation in these traits is determined by both genetic 
and environmental factors. As noted above, the genetic contribution typically 
involves multiple genes, and each of these genes may exist in several variants 
(i.e., alleles). QTL analysis allows one to map, with some precision, the genomic 
position of these alleles. </li></ul><p>For many researchers in the alcohol field, 
the breakthrough with respect to the genetic determination of alcoholrelated behaviors 
occurred when Plomin and colleagues (1991) made the seminal observation that a 
specific panel of RI mice (i.e., the BXD panel) could be used to identify the 
physical location of (i.e., to map) QTLs for behavioral phenotypes. Because the 
phenotypes of the different strains in this panel had been determined for many 
alcoholrelated traits, researchers could readily apply the strategy of RI&#8211;QTL 
mapping (Gora-Maslak et al. 1991). Although investigators recognized early on 
that this panel was not extensive enough to answer all questions, the emerging 
data illustrated the rich genetic complexity of alcoholrelated phenotypes (Belknap 
1992; Plomin and McClearn 1993). </p><p>The next advance came with the development 
of microsatellite maps (Dietrich et al. 1992, 1996). Microsatellites are short 
pieces of DNA characterized by the repetition of short (i.e., two to four nucleotide) 
sequences.<sup>1</sup>(<sup>1</sup>Nucleotides are the building blocks of DNA. 
There are four different nucleotides called adenosine [A], cytosine [C], guanosine 
[G], and thymidine [T]. Microsatellites are characterized by the repetition of 
two-to-four nucleotide pattens, such as CACACACA.) The number of repetitions of 
some microsatellites differs among individuals or inbred strains and therefore 
can be used as a marker, allowing researchers to track how specific microsatellite 
sequences are inherited. Researchers have mapped the locations, of thousands of 
such microsatellites in the mouse as well as human genome. Tracking microsatellite 
markers at specific known sites in the genome is useful because one can simultaneously 
track the gene variants linked to these markers. With these tools available, the 
first QTL study mapping a behavioral trait (i.e., activity in a novel environment) 
in F<sub>2</sub> offspring of two genetically distinct inbred mouse strains was 
published by Flint and colleagues (1995). This study detected numerous QTLs that 
were significantly associated with the behavior under investigation (see Lander 
and Kruglyak [1995] for a discussion of how a QTL is determined to be significant). 
Subsequently, there was an explosion of behavioral QTL mapping studies, including 
studies that focused on alcoholrelated traits. In a summary of the behavioral 
mapping data in mice and rats, Flint (2003) reported that hundreds of QTLs had 
been detected and that, as expected, most of these had very small effects (i.e., 
accounted for less than 5 percent of the phenotypic variance). Although there 
has been no detailed summary of behavioral QTL mapping data since 2003, it is 
reasonable to assume that the number of QTLs detected just in animal models has 
increased by an order of magnitude. </p><p>Of course the easiest, most convenient 
strategy to map QTLs in mice would be to cross animals from two inbred strains 
that differ in the behavior under investigation (e.g., sensitivity to alcohol) 
and then study the offspring to identify relevant QTLs and eventually determine 
which gene located in the vicinity of the QTL actually is responsible for the 
observed effect. The main problem with mapping QTLs in such simple intercrosses 
is that the DNA region, in which the QTL most likely is located (i.e., the 95 
percent confidence interval [CI]<sup>2</sup> [<sup>2</sup>As with other parameters, 
the location of a QTL cannot be determined exactly based on measurements in just 
a few animals. Instead, a CI is calculated using statistical models that gives 
an estimated range of values which likely includes the unknown parameter. A 95 
percent CI means that the likelihood of the estimated range including the actual 
value is 95 percent. The CI is calculated from a given set of sample data, and 
the more data are available, the more accurate the estimate will be and the smaller 
the range of values for the CI will be. A very wide CI indicates that more data 
should be collected before the parameter can be determined accurately.] of the 
QTL), frequently is very large and may, in some cases, include an entire chromosome. 
Darvasi and Soller (1995) provided a simple equation<sup>3</sup> (<sup>3</sup>According 
to this equation, CI = 1,500/Nd<sup>2</sup>, where d = the standardized gene effect; 
the constant 1,500 was determined empirically from computer simulations and is 
not related to genome size.) to calculate the 95 percent CI. Based on this equation, 
if researchers used 600 F<sub>2</sub> animals to map a QTL with an effect size 
of 5 percent, the DNA region that would contain the QTL with 95 percent certainty 
would encompass 25 centiMorgan (cM) or, for most chromosomes, between 35 and 50 
million nucleotides&#8212;a region that typically contains hundreds of genes. 
To reduce this interval to a size that can be analyzed more easily (i.e., to about 
1 cM), one would have to study 15,000 animals, which obviously is not feasible. 
It therefore seems safe to say that the issue of reducing the QTL interval (given 
the generally modest effect size of most behavioral QTLs) has been the biggest 
impediment in moving from identifying QTLs to identifying the actual quantitative 
trait gene(s) (QTGs) and eventually even the relevant nucleotides in those genes 
(i.e., the quantitative trait nucleotide[s] [QTNs]). Accordingly, relatively few 
QTGs have been identified unambiguously that contribute to behavioral phenotypes 
(e.g., Yalcin et al. 2004), and only one of these&#8212;a gene called <em>Mpdz</em>&#8212; 
is associated with an alcohol-related trait (i.e., acute alcohol withdrawal) (Fehr 
et al. 2002; Shirley et al. 2004). </p><p>In recent years, however, several strategies 
have emerged that may help reduce the QTL interval and thereby facilitate the 
identification of QTGs. This article briefly describes two approaches&#8212;multiple 
cross and heterogeneous stock mapping. Additional approaches are described in 
the following article by Denmark and colleagues (pp. 266&#8211;269). </p><h2>Multiple 
Cross Mapping </h2><p>The concept of combining (i.e., integrating) data obtained 
from intercrosses of several inbred strains (i.e., multiple crosses) is being 
used widely to improve QTL characterization for traits of agricultural value (see, 
for example, Christiansen et al. 2006; Khatkar et al. 2004). The application of 
this approach, which has been termed multiple cross mapping (MCM), to traits of 
physiological and behavioral interest also is becoming more frequent (e.g., Hitzemann 
et al. 2000, 2002, 2003; Jagodic and Olsson 2006; Li et al. 2005; Malmanger et 
al. 2006; Park et al. 2003; Wergedal et al. 2007; Wittenburg et al. 2005). Our 
interest in MCM was triggered by the observation that QTL data generated by three 
different mouse F<sub>2</sub> intercrosses in three different laboratories<sup>4</sup> 
(<sup>4</sup>The studies involved crosses of C57BL/6J mice with BALB/cJ mice [Flint 
et al. 1995], A/J mice [Gershenfeld et al. 1997], and DBA/2J mice [Koyner et al. 
2000].) apparently all detected the same QTL on a part of mouse chromosome 1 that 
was associated with open-field activity (Flint et al. 1995; Gershenfeld et al. 
1997; Koyner et al. 2000); however, the QTL was not detected in a cross of two 
other mouse strains<sup>5</sup> (<sup>5</sup>This intercross involved BALB/cJ 
mice and LP/J mice [Hitzemann et al. 2000].) (Hitzemann et al. 2000). Hitzemann 
and colleagues (2000) proposed that the information obtained with multiple crosses 
could be used to develop an empirical algorithm for sorting microsatellite markers 
in order to detect chromosomal regions with the highest probability of containing 
QTLs. </p><p>The principle underlying this theory was that since the inbred mouse 
strains used actually are closely related, the data described above suggests that 
there must be a region or regions on chromosome 1 where three strains (i.e., DBA/2J, 
BALB/cJ and A/J strains) are identical and different from the fourth strain (i.e., 
C57BL/6J strain). It is perhaps easiest to visualize this in binary terms, where 
0 and 1 represent different nucleotides; in a region of interest, the three similar 
strains could have the structure &#8220;0100011100&#8221; while the C57BL/6J strain 
would have the structure &#8220;1011100011.&#8221; These different patterns are 
termed differences in haplotype structure. Accordingly, the three strains carry 
one unit of haplotype structure and the C57BL/6J strain carries a different unit. 
The haplotype difference could involve a single nucleotide polymorphism (SNP) 
or, as in the example above, multiple SNPs. Knowing the regions where the strains 
are similar and where they differ enhances QTL analyses because it provides additional 
information and thus greater statistical power. Even more details of this haplotype 
structure became available when researchers developed dense maps that showed the 
location of SNPs in multiple mouse strains (e.g., Wade et al. 2002). These maps 
confirmed that some regions of the genome contain very few SNPs, whereas others 
contain many SNPs. A QTL was presumed to have a greater likelihood of being associated 
with the SNP-dense region than with the SNP-poor region where there is very little 
genetic variation. </p><p>When conducting MCM analyses, researchers often use 
&#8220;crosses of convenience&#8221;&#8212;that is, they draw on data obtained 
in studies that they and other groups have conducted with the strains they were 
using to address specific research questions. One problem associated with this 
approach, however, is that often there are missing data. For example, consider 
the data that originally led to our development of MCM. The three studies on which 
the analysis was based involved four different inbred mouse strains, but only 
three different crosses of these animals were analyzed; data for the remaining 
possible crosses were not available. Without this information, however, the true 
haplotype structure of the QTL cannot be determined. To address this issue, we 
created a balanced panel of crosses from four inbred strains in which every strain 
was crossed with every other strain and have used this panel to map QTLs for open-field 
activity and alcoholinduced locomotion (Hitzemann et al. 2003; Malmanger et al. 
2006). With this approach, the MCM algorithm markedly reduced the QTL CIs and 
correctly predicted QTL position and haplotype structure as determined by heterogeneous 
stock (HS) mapping, which is described in the following section. </p><h2>Heterogeneous 
Stock Mapping </h2><p>The problem associated with conducting de novo MCM rather 
than using a convenience sample of already available, but incomplete, crosses 
is that it requires a lot of work and many animals. Assuming that, as described 
above, at least about 600 animals are needed to identify a QTL using an intercross 
of two inbred strains, then the genetic makeup (i.e., genotype) and relevant behavioral 
and physical characteristics (i.e., the phenotype) of 2,400 animals would have 
to be determined to obtain a balanced panel for four inbred strains. Although 
genotyping has become much easier with the availability of high-throughput devices 
to map SNPs, the overall effort is still considerable and costly. These considerations 
have prompted the emergence of HS mapping as an alternative strategy that is precise 
and provides good information on haplotype structure. </p><p>In heterogeneous 
populations, all individuals have diverse genetic backgrounds. For example, one 
commonly used heterogeneous mouse stock was generated by interbreeding animals 
from eight genetically diverse inbred strains (Phillips 2002). HS mapping was 
first described by Talbot and colleagues (1999) who used it to identify QTLs associated 
with the phenotype of open-field activity. The investigators were able to map 
numerous QTLs with high precision. However, the analyses did not detect QTLs associated 
with this phenotype that previously had been mapped in an F<sub>2</sub> intercross 
population. Mott and colleagues (2000) provided a solution to this problem by 
developing a mapping algorithm termed HAPPY, which was designed to map QTLs in 
any HS population derived from known inbred strains without requiring further 
pedigree information.<sup>6</sup> (<sup>6</sup>With this algorithm, the analysis 
basically occurs in two steps: [1] the ancestral haplotype is reconstructed using 
dynamic programming and then [2] QTLs are analyzed using linear regression.) The 
HAPPY algorithm found the previously detected QTLs in the HS mapping and also 
determined that the QTLs had the expected haplotype structure. Knowing the QTL 
signature is of considerable value when integrating QTL, gene expression, and 
gene sequence data. </p><p>There can be differences between the results achieved 
with HS mapping and those achieved with mapping in F<sub>2</sub> intercross populations 
(see figure 15). For example, when analyses of a QTL on chromosome 2 that is associated 
with alcohol-induced locomotor response were conducted using F<sub>2</sub> animals 
obtained by crossing C57BL/6J and DBA/2J mice, the resulting QTL interval was 
very broad (Demarest et al. 1999). Moreover, the investigators determined that 
those QTL alleles that the animals had inherited from the C57B6/6J mice were associated 
with a decreased response to alcohol. The same researchers then attempted to map 
the QTLs related to the ethanol response phenotype in an HS that was formed by 
crossing eight inbred mouse strains, including C57BL/6J and DBA/2J animals (Demarest 
et al. 2001). The analysis relied on microsatellite genotyping and simply classified 
alleles as either being similar to those found in C57BL/6J or being different 
from C57BL/6J alleles. This analysis detected multiple QTLs in the region of interest; 
furthermore, the C57BL/6J alleles were associated with both increased and decreased 
ethanol response. These findings suggest that the HR mapping approach is more 
sensitive than the F<sub>2</sub> intercross approach and generates a greater variety 
of QTLs because none of the data suggest that these multiple QTLs also were present 
in the F<sub>2</sub> intercross (although they also would have been invisible 
to the type of analysis used by Demarest and colleagues [1999]). Finally, Malmanger 
and colleagues (2006) performed QTL mapping for the ethanol response phenotype 
in an HS population generated by crossing four inbred strains (i.e., C57BL/6J, 
DBA/2J, LP/J, and BALB/cJ mice).<sup>7</sup> (<sup>7</sup>The mapping was done 
at generation 19, which represents an approximately 10-fold expansion of the genetic 
map because with each generation additional recombinations are added that allow 
for finer mapping. In addition, the researchers used a dense SNP panel.) This 
approach also detected a QTL peak in the region of interest that spanned a region 
of 1 to 2 million nucleotides. Moreover, the investigators determined the haplotype 
structure of the QTL and noted that the B6 allele was associated with decreased 
ethanol response. The integration of these data (i.e., position and haplotype 
of the QTL) with gene expression databases suggests a strong candidate QTG called 
<em>Scgn5</em> (also known as <em>7</em>B2 and <em>Sgne1</em>), which encodes 
a protein called secretogranin 5 (Malmanger et al. 2006). </p><table width="90%" border="0" cellpadding="0"  summary="layout only"> 
<tr> <td><p><img src="images/15_hitz.gif" alt="figure 15"></p><p><strong>Figure 
15.</strong> Three strategies for mapping a quantitative trait locus (QTL) on 
mouse chromosome 2 that is associated with acute ethanol locomotor response. The 
characteristic (i.e., phenotype) tested is the difference in activity between 
the administration of saline and the administration of 1.5 g/kg ethanol, measured 
in 5-minute intervals between 0 and 20 minutes after the injection. The top panel 
illustrates the result of a QTL mapping analysis in a C57BL/6J x DBA/2J F<sub>2</sub> 
intercross (<em>N</em> = 600) (Demarest et al. 1999). The second panel illustrates 
mapping of the same phenotype in heterogeneous stock [HS-NPT] mice (<em>N</em> 
= 500) at generation 32 (Demarest et al. 2001). Data were analyzed in a marker-by-marker 
design; all markers were microsatellites and were classified as C57BL/6J&#8211;like 
or different. A positive F value indicates that a non-C57 allele is associated 
with an increased ethanol response. The HS analysis detected several QTLs that 
were not found in the F2 intercross analysis. The bottom panel shows the results 
of mapping the same phenotype using heterogeneous stock [HS4] animals (<em>N</em> 
= 575) at generation 19 and using a panel of closely spaced SNPs as markers (Malmanger 
et al. 2006). The bar at the top shows the haplotype structure across the region 
of interest. </p><p>NOTE: The LOD (logarithm [base 10] of odds) is a measure of 
the degree of linkage between a given DNA region or gene and a specific trait.</p></td></tr> 
</table><p>Currently, four mouse HS populations are available to investigators. 
One of these, the HS/Ibg, which was formed by crossing eight inbred laboratory 
mouse strains,<sup>8</sup> (<sup>8</sup>Two of these strains are no longer available.) 
is available through the Institute for Behavioral Genetics. The other three populations 
are maintained by the first author and include the HS4 population described in 
the previous paragraph, the HS&#8211;NPT population (see Valdar et al. 2006), 
and the HS&#8211;CC population (an eight-way cross that contains three mouse strains 
derived from the wild). These HS populations are freely available.</p><h2>Conclusion 
</h2><p>QTL mapping has become an important aspect of efforts to determine the 
genetic basis of complex behaviors, such as alcohol-drinking behaviors. With new 
approaches to gene mapping, such as multicross mapping and HS mapping, which improve 
the accuracy with which QTLs can be located on the chromosomes, the identification 
of additional candidate QTGs likely is only a matter of time.</p><h2>Financial 
Disclosure</h2><p> The authors declare that they have no competing financial interests.</p><h2>REFERENCES</h2><p>Beck, 
J.A.; Lloyd, S.; Hafezparast, M.; et al. Genealogies of mouse inbred strains. 
<em>Nature Genetics</em> 24(1):23&#8211;25, 2000. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=10615122" tabindex="1">PMID: 
10615122 </a> <p> Belknap, J.D. Empirical estimates of Bonferroni corrections 
for use in chromosome mapping studies with the BXD recombinant inbred strains. 
<em>Behavior Genetics</em> 22:677&#8211;684, 1992. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=1290453" tabindex="2">PMID: 
1290453 </a> <p> Christiansen, M.J.; Feenstra, B.; Skovgaard, I.M.; and Andersen, 
S.B. Genetic analysis of resistance to yellow rust in hexaploid wheat using a 
mixture model for multiple crosses. TAG. <em>Theoretical and Applied Genetics</em> 
112:581&#8211;591, 2006. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=16395570" tabindex="3">PMID: 
16395570 </a> <p> Darvasi, A., and Soller, M. Advanced intercross lines, an experimental 
population for fine genetic mapping. <em>Genetics</em> 141:1199&#8211;1207, 1995. 
<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=8582624" tabindex="4">PMID: 
8582624</a> <p> Demarest, K.; McCaughran, J., Jr.; Mahjubi, E.; et al. Identification 
of an acute ethanol response quantitative trait locus on mouse chromosome 2. <em>Journal 
of Neuroscience</em> 19:549&#8211;561, 1999. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=9880575" tabindex="5">PMID: 
9880575 </a> <p> Demarest, K.; Koyner, J.; McCaughran, J., Jr.; et al. Further 
characterization and high-resolution mapping of quantitative trait loci for ethanol-induced 
locomotor activity. <em>Behavior Genetics</em> 31:79&#8211;91, 2001. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=11529277" tabindex="6">PMID: 
11529277 </a> <p> Dietrich, W.; Katz, H.; Lincoln, S.E.; et al. A genetic map 
of the mouse suitable for typing intraspecific crosses. <em>Genetics </em>131:423&#8211;447, 
1992. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=1353738" tabindex="7">PMID: 
1353738 </a> <p> Dietrich, W.F.; Miller, J.; Steen, R.; et al. A comprehensive 
genetic map of the mouse genome. <em>Nature</em> 380:149&#8211;152, 1996. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=8600386" tabindex="8">PMID: 
8600386 </a> <p> Fehr, C.; Shirley, R.L.; Belknap, J.K.; et al. Congenic mapping 
of alcohol and pentobarbital withdrawal liability loci to a &lt;1 centimorgan 
interval of murine chromosome 4: Identification of Mpdz as a candidate gene. <em>Journal 
of Neuroscience</em> 22(9):3730&#8211;3738, 2002. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=11978849" tabindex="9">PMID: 
11978849</a> <p> Flint, J. Analysis of quantitative trait loci that influence 
animal behavior.<em> Journal of Neurobiology</em> 54:46&#8211;77, 2003. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=12486698" tabindex="10">PMID: 
12486698 </a> <p> Flint, J.; Corley, R.; DeFries, J.C.; et al. A simple genetic 
basis for a complex psychological trait in laboratory mice. <em>Science </em>269:1432&#8211;1435, 
1995. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=7660127" tabindex="11">PMID: 
7660127</a> <p> Gershenfeld, H.K.; Neumann, P.E.; Mathis, C.; et al. Mapping quantitative 
trait loci for open-field behavior in mice. <em>Behavior Genetics</em> 27:201&#8211;210, 
1997. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=9210791" tabindex="13">PMID: 
9210791</a> <p> Gora-Maslak, G.; McClearn, G.E.; Crabbe, J.C.; et al. Use of recombinant 
inbred strains to identify quantitative trait loci in psychopharmacology. <em>Psychopharmacology</em> 
104:413&#8211;424, 1991. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=1780413" tabindex="14">PMID: 
1780413</a> <p> Hitzemann, R.; Demarest, K.; Koyner, J.; et al. Effects of genetic 
cross on the detection of quantitative trait loci and a novel approach to mapping 
QTLs. <em>Pharmacology, Biochemistry, and Behavior</em> 67:767&#8211;772, 2000. 
<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=11166067" tabindex="15">PMID: 
11166067</a> <p> Hitzemann, R.; Malmanger, B.; Cooper, S.; et al. Multiple cross 
mapping (MCM) markedly improves the localization of a QTL for ethanol-induced 
activation. <em>Genes, Brain, and Behavior</em> 1:214&#8211;222, 2002. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=12882366" tabindex="16">PMID: 
12882366 </a> <p> Hitzemann, R.; Malmanger, B.; Reed, C.; et al. A strategy for 
the integration of QTL, gene expression, and sequence analyses. <em>Mammalian 
Genome</em> 14:733&#8211;747, 2003. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=14722723" tabindex="17">PMID: 
14722723 </a> <p> Jagodic, M., and Olsson, T. Combined-cross analysis of genome-wide 
linkage scans for experimental autoimmune encephalomyelitis in rat. <em>Genomics</em> 
88:737&#8211;744, 2006.<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=17010567" tabindex="18"> 
PMID: 17010567</a> <p> Khatkar, M.S.; Thomson, P.C.; Tammen, I.; and Raadsma, 
H.W. Quantitative trait loci mapping in dairy cattle: Review and meta-analysis. 
<em>Genetics, Selection, Evolution</em> 26:163&#8211;190, 2004. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=15040897" tabindex="19">PMID: 
15040897 </a> <p> Koyner, J.; Demarest, K.; McCaughran, J., Jr; et al. Identification 
and time dependence of quantitative trait loci for basal locomotor activity in 
the BXD recombinant inbred series and a B6D2 F2 intercross. <em>Behavior Genetics</em> 
30:159&#8211;170, 2000. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=11105390" tabindex="20">PMID: 
11105390 </a> <p> Lander, E., and Kruglyak, L. Genetic dissection of complex traits: 
Guidelines for interpreting and reporting linkage results. <em>Nature Genetics</em> 
11:241&#8211;247, 1995. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=7581446" tabindex="21">PMID: 
7581446 </a> <p> Li, R.; Lyons, M.A.; Wittenburg, H.; et al. Combining data from 
multiple inbred line crosses improves the power and resolution of quantitative 
trait loci mapping. <em>Genetics </em>169:1699&#8211;1709, 2005. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=15654110" tabindex="22">PMID: 
15654110 </a> <p> Malmanger, B.; Lawler, M.; Coulombe, S.; et al. Further studies 
on using multiple-cross mapping (MCM) to map quantitative trait loci. <em>Mammalian 
Genome</em>&nbsp; 17:1193&#8211;1204, 2006. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=17143586" tabindex="23">PMID: 
17143586</a> <p> Mott, R.; Talbot, C.J.; Turri, M.G.; et al. A method for fine 
mapping quantitative trait loci in outbred animal stocks. <em>Proceedings of the 
National Academy of Sciences of the United States of America</em> 97:12649&#8211;12654, 
2000. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=11050180" tabindex="24">PMID: 
11050180 </a> <p> Park, Y.G.; Clifford, R.; Buetow, K.H.; and Hunter, K.W. Multiple 
cross and inbred strain haplotype mapping of complex-trait candidate genes. <em>Genome 
Research</em> 13:118&#8211;121, 2003. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=12529314" tabindex="25">PMID: 
12529314 </a> <p> Paterson, A.H.; Lander, E.S.; Hewitt, J.D.; et al. Resolution 
of quantitative traits into Mendelian factors by using a complete linkage map 
of restriction fragment length polymorphisms. <em>Nature</em> 335(6192):721&#8211;726, 
1988. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=11050180" tabindex="26">PMID: 
2902517</a> <p> Phillips, T. Animal models for the genetic study of human alcohol 
phenotypes. <em>Alcohol Research &amp; Health</em> 26:202&#8211;207, 2002. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=12875048" tabindex="27">PMID: 
12875048 </a> <p> Plomin, R., and McClearn, G.E. Quantitative trait loci (QTL) 
analyses and alcohol-related behaviors. <em>Behavior Genetics</em> 23:197&#8211;211, 
1993. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=8512533" tabindex="28">PMID: 
8512533 </a> <p> Plomin, R.; McClearn, G.E.; Gora-Maslak, G.; and Neiderhiser, 
J.M. Use of recombinant inbred strains to detect quantitative trait loci associated 
with behavior. <em>Behavior Genetics</em> 21:99&#8211;116, 1991. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=2049054" tabindex="29">PMID: 
2049054 </a> <p> Shirley, R.L.; Walter, N.A.; Reilly, M.T.; et al. Mpdz is a quantitative 
trait gene for drug withdrawal seizures. <em>Nature Neuroscience</em> 7:699&#8211;700, 
2004. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=15208631" tabindex="30">PMID: 
15208631</a> <p> Talbot, C.J.; Nicod, A.; Cherny, S.S.; et al. High-resolution 
mapping of quantitative trait loci in outbred mice. <em>Nature Genetics </em>21:305&#8211;308, 
1999.<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=10080185" tabindex="31"> 
PMID: 10080185</a> <p> Valdar, W.; Solberg, L.C.; Gauguier, D.; et al. Genome-wide 
genetic association of complex traits in heterogeneous stock mice. <em>Nature 
Genetics</em> 38:879&#8211;887, 2006. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=16832355" tabindex="32">PMID: 
16832355 </a> <p> Wade, C.M.; Kulbokas, E.J., 3rd; Kirby, A.W.; et al. The mosaic 
structure of variation in the laboratory mouse genome. <em>Nature</em> 420:574&#8211;578, 
2002. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=12466852" tabindex="33">PMID: 
12466852</a> <p> Wergedal, J.E.; Ackert-Bicknell, C.L.; Beamer, W.G.; et al. Mapping 
genetic loci that regulate lipid levels in a NZB/B1NJxRF/J intercross and a combined 
intercross involving NZB/B1NJ, RF/J, MRL/MpJ, and SJL/J mouse strains. <em>Journal 
of Lipid Research</em> 48:1724&#8211;1734, 2007. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=17496333" tabindex="34">PMID: 
17496333</a> <p> Wittenburg, H.; Lyons, M.A.; Li, R.; et al. Association of a 
lithogenic Abcg5/Abcg8 allele on chromosome 17 (Lith9) with cholesterol gallstone 
formation in PERA/EiJ mice. <em>Mammalian Genome</em> 16:495&#8211;504, 2005. 
<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=16151694" tabindex="35">PMID: 
16151694</a> <p> Yalcin, B.; Fullerton, J.; Miller, S.; et al. Unexpected complexity 
in the haplotypes of commonly used inbred strains of laboratory mice. <em>Proceedings 
of the National Academy of Sciences of the United States of America </em>101:9734&#8211;9739, 
2004. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=search&term=15210992" tabindex="36">PMID: 
15210992</a> <!-- #EndEditable --><br> </font> <p> <!-- END MAIN CONTENT --> </p><TABLE cellSpacing=0 cellPadding=0 width="100%" Summary="This is a layout table"> 
<TBODY> <TR> <TD bgColor=#003a91><IMG height=3 alt="" 
      src="../../images/1x1pixel.gif" width=3></TD></TR> <TR> <TD bgColor=#efefef> 
<TABLE cellSpacing=17 cellPadding=0 border=0 Summary="This is a layout table"> 
<TBODY> <TR vAlign=top> <TD class=footer colSpan=4><FONT color=#999999>[</FONT>&nbsp;<A tabindex="2"  
            href="http://www.niaaa.nih.gov/">NIAAA Home Page</A>&nbsp;<FONT 
            color=#999999>|</FONT>&nbsp;<A tabindex="3"  
            href="http://www.niaaa.nih.gov/SiteMap.htm">Site Map</A>&nbsp;<FONT 
            color=#999999>|</FONT>&nbsp;<A tabindex="4" 
            href="http://www.niaaa.nih.gov/Accessibility.htm">Accessibility</A>&nbsp;<FONT 
            color=#999999>|</FONT>&nbsp;<A tabindex="5" 
            href="http://www.niaaa.nih.gov/Privacy.htm">Privacy statement</A>&nbsp;<FONT color=#999999>|</FONT>&nbsp;<A tabindex="6" 
            href="http://www.niaaa.nih.gov/Disclaimer.htm">Disclaimer</A>&nbsp;<FONT 
            color=#999999>|</FONT>&nbsp;<A tabindex="7" 
            href="http://www.niaaa.nih.gov/ContactUs.htm">Contact Us</A>&nbsp;<FONT 
            color=#999999>]</FONT></TD></TR> <TR vAlign=top> <TD> <TABLE cellSpacing=0 cellPadding=0 border=0 Summary="This is a layout table"> 
<TBODY> <TR vAlign=top> <TD class=footer colSpan=2>National Institute on<BR>Alcohol 
Abuse and Alcoholism (NIAAA)<BR>5635 Fishers Lane, MSC 9304<BR>Bethesda, MD 20892-9304<BR>email: 
Web Sponsor (<A tabindex="8" 
                  href="mailto:niaaaweb-r@exchange.nih.gov">niaaaweb-r@exchange.nih.gov</A>)</TD></TR></TBODY></TABLE></TD><TD> 
<TABLE cellSpacing=0 cellPadding=0 border=0 Summary="This is a layout table"> 
<TBODY> <TR vAlign=top> <TD><A tabindex="9" href="http://www.nih.gov/"><IMG class=tableImage 
                  height=30 alt="N I H logo" 
                  src="../../images/footerlogo-nih.gif" width=35 
                  border=0></A></TD><TD class=footer>National Institutes of Health</TD></TR></TBODY></TABLE></TD><TD> 
<TABLE cellSpacing=0 cellPadding=0 border=0 Summary="This is a layout table"> 
<TBODY> <TR vAlign=top> <TD><A tabindex="10" href="http://www.os.dhhs.gov/"><IMG class=tableImage 
                  height=30 alt="H H S logo" 
                  src="../../images/footerlogo-hhs.gif" width=35 
                  border=0></A></TD><TD class=footer>Department of Health <BR>and 
Human Services</TD></TR></TBODY></TABLE></TD><TD><A tabindex="11" href="http://www.firstgov.com/"><IMG class=tableImage 
            height=30 alt=FirstGov 
            src="../../images/footerlogo_firstgov.gif" width=92 
            border=0></A></TD></TR></TBODY></TABLE></TD></TR> <TR> <TD bgColor=#efefef><IMG height=30 alt="" 
      src="file://///Niaaa-fcsnap/niaaawsnap2/Thuy/CMS/Welcome%20to%20NIAAA_files/1x1pixel.gif" 
width=1></TD></TR></TBODY></TABLE>
</BODY>
<!-- #EndTemplate -->
</HTML>
