



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" >
<!-- Begin Head tag-->
<head id="ctl00_Head1"><title>The Genetic Architecture of Complex Traits Workshop - National Institute of General Medical Sciences</title>
		<meta name="keywords" content="">
		<link rel="schema.dc" href="http://purl.org/dc/elements/1.1/">
		<meta name="dc.title" content="The Genetic Architecture of Complex Traits Workshop">
		<meta name="dc.description" content="January 29, 1998">
		<meta name="dc.creator" content="National Institute of General Medical Sciences">
		<meta name="dc.date.created" scheme="ISO8601" content="2005-09-27">
		<meta name="dc.date.modified" scheme="ISO8601" content="2008-11-19">
		<meta name="dc.language" scheme="ISO639-2" content="eng"><link rel="shortcut icon" type="image/x-icon" href="/nigms.nih.gov/favicon.ico" /><meta content="Microsoft Visual Studio .NET 8" name="GENERATOR" /><meta content="C#" name="CODE_LANGUAGE" /><meta content="JavaScript" name="vs_defaultClientScript" /><meta content="http://schemas.microsoft.com/intellisense/ie5" name="vs_targetSchema" /><meta name="ROBOTS" content="FOLLOW,INDEX">
<base href="http://www.nigms.nih.gov/nigms.nih.gov/Templates/CommonPage.aspx?NRMODE=Published&NRNODEGUID=%7b6A2E64C5-F838-4C17-83CA-9628B5BB6541%7d&NRORIGINALURL=%2fnews%2freports%2fgenetic_arch%2ehtml&NRCACHEHINT=Guest">
<meta name="description" content="January 29, 1998"><link href="/nigms.nih.gov/Style/Style.css" type="text/css" rel="stylesheet" />
        <script language="JavaScript" type="text/javascript" src="/nigms.nih.gov/Includes/NIGMSCMSJavaScriptFunctions.js"></script>
        <script language="JavaScript" type="text/javascript" src="/nigms.nih.gov/Includes/EmailPage.js"></script>
        <script language="JavaScript" type="text/javascript" src="/nigms.nih.gov/Includes/KeyPressHandler.js"></script>
        <script language="javascript" type="text/javascript">
	        function PrintThisPage() 
	        {
		        var sOption = "scrollbars,resizable"; 
		        var sWinHTMLHeading = "<div id='pageutilities'><a href='javascript:window.print()'><img src='/nigms.nih.gov/Images/printer_icon.gif' alt='Send to printer' width='16' height='13' border='0'></a>&nbsp;<a href='javascript:window.print()'>Send to printer</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;<a href='javascript:window.close()'>Close</a></div>";
			        sWinHTMLHeading += "<br><div id='hyperlinktoggle'>[<a href='javascript:hyperlinktoggle();'>Show Links</a>]</div>";
			        sWinHTMLHeading += "<br>";
        //			sWinHTMLHeading += "<br><br><h1>The Genetic Architecture of Complex Traits Workshop</h1>";
		        var sWinHTMLBody = document.getElementById('PrintContent1').innerHTML;
			        sWinHTMLBody += "<br>" + document.getElementById('PrintContent2').innerHTML;
		        var winprint=window.open("","",sOption);
			        winprint.document.open();
			        winprint.document.write('<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html><link href="/nigms.nih.gov/Style/PlainStyle.css" type="text/css" rel="stylesheet" /><link href="/nigms.nih.gov/Style/Print.css" rel="stylesheet" type="text/css" media="print" /><body>'); 
			        winprint.document.write('<scr'+'ipt language="javascript" type="text/javascript">function hyperlinktoggle() { for(i=0; i<document.links.length; i++) { if (document.links[i].innerText != "") { if (document.links[i].href.substring(0,4)!="java" && document.links[i].innerHTML.substring(0,4)!="http") { document.links[i].innerHTML = document.links[i].innerHTML + " &lt;" + document.links[i].href + "&gt;"}}} document.getElementById("hyperlinktoggle").innerHTML = "[<a href=javascript:document.location.reload()>Hide Links</a>]"}</scr'+'ipt>');
			        winprint.document.write(sWinHTMLHeading);
			        winprint.document.write(sWinHTMLBody);
			        winprint.document.write('</body></html>'); 
			        winprint.document.close();
	        }
        </script>
        <link href="/nigms.nih.gov/Style/Print.css" rel="stylesheet" type="text/css" media="print" />   
        <script language=JavaScript src="/nigms.nih.gov/Includes/RandomImage.js"></script>

        <SCRIPT LANGUAGE="javascript">
		<!--
		var url;
		url = '';	
		if (url != '')
		{	
			//Code to go back to previous page
			window.history.go(-1);	
			//Code that opens new window
			aWindow = window.open(url, 'newWindow');
		}
		//-->
		</SCRIPT>
<title>
	Untitled Page
</title></head>
<!-- End head tag -->
<body onkeydown="Search.handleKeypress(event);" onload="Search.clrSearch();randomImage();" >
<a name="top"></a>
<div id="skipnav"><a href="#main_content" title="Skip navigation links" id="main"><img alt="Skip navigation links" src="/nigms.nih.gov/Images/1x1space.gif" border="0" /></a> </div>
			<div id="bannerbkgnd">&nbsp;</div>
			<div id="container">
<div id="banner">
	<div id="bannerlogo">
		<a href="/"><img src="/nigms.nih.gov/Images/banner_nigms.gif" alt="NIGMS - National Institute of General Medical Sciences"
				width="310" height="58" border="0" /></a> <a href="http://www.nih.gov/" target="_blank"><img src="/nigms.nih.gov/Images/banner_nih.gif" alt="One of the National Institutes of Health"
				width="310" height="18" border="0" /></a>
	</div>
	<div id="bannercollage">
		<img id="theImage" name="theImage" src="/nigms.nih.gov/Images/banner_collage1.jpg" alt="" width="425" height="71" />
	</div>
	<div id="bannertabs">
	    <a href="/Initiatives/"><img src="/nigms.nih.gov/Images/tab_featuredareas.gif" alt="Featured Areas" width="102" height="19"
			border="0" class="bannertab" /></a><a href="/Research/"><img src="/nigms.nih.gov/Images/tab_funding2.gif" alt="Research Funding" width="117" height="19"
			border="0" class="bannertab" /></a><a href="/Training/"><img src="/nigms.nih.gov/Images/tab_training2.gif" alt="Training &amp; Careers" width="121"
			height="19" border="0" class="bannertab" /></a><a href="/Minority/"><img src="/nigms.nih.gov/Images/tab_minority2.gif" alt="Minority Programs" width="123"
			height="19" border="0" class="bannertab" /></a><a href="/News/"><img src="/nigms.nih.gov/Images/tab_news2.gif" alt="News &amp; Events" width="99" height="19"
			border="0" class="bannertab" /></a><a href="/Publications/"><img src="/nigms.nih.gov/Images/tab_publications2.gif" alt="Publications" width="85" height="19"
			border="0" class="bannertab" /></a><a href="/About/"><img src="/nigms.nih.gov/Images/tab_about2.gif" alt="About NIGMS" width="93" height="19"
			border="0" /></a>
	</div>	
</div><!-- End div banner -->

	<div id="bannerutilities"><a href="/">NIGMS Home</a> | <a href="/SiteMap.html">
			Site Map</a> | <a href="http://public.nigms.nih.gov/StaffContacts/">Staff Search
			</a>
	</div>
	<!-- Search-->
	<div id="bannersearch">
	<form method="post">

        <input type="hidden" name="site" value="NIGMS" />
		<input type="hidden" name="client" value="NIGMS_frontend" />
		<input type="hidden" name="proxystylesheet" value="NIGMS_frontend" />
		<input type="hidden" name="output" value="xml_no_dtd" />
		<input type="hidden" name="filter" value="0" />
		<input type="hidden" name="getfields" value="*" />
		<input type="hidden" name="num" value="20"/>
		<input type="hidden" name="numgm" value="5"/> 

        <p>
		<label for="search">Search:</label> <input name="search"  alt="Search" type="text" id="search" size="12" maxlength="255" /> 
		<input type="button" class="search" id="btnG" value=" "  alt="button" onclick="RedirectPopUp('http://search2.google.cit.nih.gov/search',this.form);" />
	    </p>
	</form><!--</form>-->
</div> <!-- End search -->


<form name="aspnetForm" method="post" action="CommonPage.aspx?NRMODE=Published&amp;NRNODEGUID=%7b6A2E64C5-F838-4C17-83CA-9628B5BB6541%7d&amp;NRORIGINALURL=%2fnews%2freports%2fgenetic_arch.html&amp;NRCACHEHINT=Guest" id="aspnetForm">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUJMjI4MjMyMjY1D2QWAmYPZBYCAgEPZBYEZg9kFhICAg8PFgQeBFRleHQFHFRhZ2xpbmUgKG9wdGlvbmFsKTogPGJyPjxicj4eB1Zpc2libGVoZGQCAw8PFgIfAWhkZAIEDw8WBB8ABRI8YnI+TWFpbiBDb250ZW50OiAfAWhkZAIGDw8WBB8ABQ08YnI+U3VtbWFyeTogHwFoZGQCBw8PFgIfAWhkZAIIDw8WAh8BaGRkAgkPDxYGHglGb3JlQ29sb3IKpAEeBF8hU0ICBB8BaGRkAgoPDxYCHwFoZGQCCw8PFgIfAWhkZAIBD2QWBGYPFgIfAWgWAgIDD2QWBAIBDxYCHwFoZAIfDxYCHwFoZAIBDxYCHwFoZGSiNpuOYSi5YkOKOIoib9V98loWTQ==" />


<script language="javascript" type="text/javascript">
<!--
   var __CMS_PostbackForm = document.forms['aspnetForm'];
   var __CMS_CurrentUrl = "/nigms.nih.gov/Templates/CommonPage.aspx?NRMODE=Published&NRNODEGUID=%7b6A2E64C5-F838-4C17-83CA-9628B5BB6541%7d&NRORIGINALURL=%2fnews%2freports%2fgenetic_arch%2ehtml&NRCACHEHINT=Guest";
// -->
</script>



<!-- Begin Content area -->


<div id="breadcrumbs">
Back to: <span id="ctl00_MainContent_Breadcrumb1_NIHCMSBreadcrumbControl1"><a href="/" class="breadcrumb">NIGMS Home</a>&nbsp;&gt;&nbsp;<a href="/News/" class="breadcrumb">News &amp; Events</a>&nbsp;&gt;&nbsp;<a href="/News/Reports/" class="breadcrumb">Reports</a><table border="0">

</table></span>
</div>
<a name="main_content"></a>
				<div id="maincontent">
					<table cellspacing="0" cellpadding="0" width="100%" border="0">
						<tr valign="top">
							<td>
								<div id="PrintContent1">
									
<h1>The Genetic Architecture of Complex Traits Workshop</h1>

									
									
									<p></p>
									
									
									<span id="ctl00_MainContent_BodyContentC" style="display:inline-block;"><H3>January 29, 1998</H3>
<P><I>This report summarizes the findings of a panel of experts who met at the National Institutes of Health on December 10-11, 1997 at a workshop entitled "The Genetic Architecture of Complex Traits." The report and recommendations were prepared for consideration by the National Advisory General Medical Sciences Council.</I> 
<P>
<H3>Executive Summary</H3>
<P>Most genetic traits of interest in populations of humans and other organisms are determined by many factors, including genetic and environmental components, which interact in often unpredictable ways. For such complex traits, the whole is not only greater than the sum of its parts, it may be different from the sum of its parts. Thus, complex traits have a genetic architecture that consists of all of the genetic and environmental factors that contribute to the trait, as well as their magnitude and their interactions. 
<P>The following recommendations are intended to increase the rate of progress and improve the quality of research on the analysis of complex traits. 
<P><B>Research</B> 
<P><I>Data</I> 
<UL>
<LI>NIH and the scientific community should focus on better acquisition of data, including larger samples and more refined phenotype definitions. 
<LI>There is no single simple sampling scheme that should be used to obtain the data for these studies. 
<LI>Given the reality that genetic architecture can and will vary as a function of population parameters, collection of data on population structure, including histories for different human populations and for model organisms, is essential and should be supported. </LI></UL>
<P><I>Methods</I> 
<UL>
<LI>The richness of current methods should be exploited to its fullest through creative applications to appropriate data sets. 
<LI>Every effort should be made to encourage the development of fundamentally new models and more sensitive methods of analysis. 
<LI>Because so little is known about genetic architecture, exploratory and observational studies should be encouraged. 
<LI>NIGMS should encourage collaborative studies among investigators in diverse disciplines. 
<LI>With respect to review and funding of research grant applications, it is important to emphasize that no single method or model for studying genetic architecture can be adopted universally. </LI></UL>
<P><I>Model Organisms</I> 
<UL>
<LI>The choice of organism and research design should be dictated by the complex trait of interest and the questions being asked. 
<LI>Studies of population structure and variability of organisms in natural populations are needed. </LI></UL>
<P><B>Resources</B> 
<UL>
<LI>Support is needed for new database structures for population data. 
<LI>The development of publicly available genetic data sets of genetic maps and haplotypes should be encouraged. </LI></UL>
<P><B>Training</B> 
<UL>
<LI>The expertise of computational scientists, including physicists, mathematicians, and engineers, is needed; however, most will need to be retrained in statistical genetics. 
<LI>NIH should support training in statistical genetics for scientists who intend to apply the tools of genetic analysis. 
<LI>Multidisciplinary training is essential. </LI></UL>
<P><B>Communications</B> 
<UL>
<LI>The analysis of complex traits does not lend itself to quick and easy solutions. 
<LI>At the same time, it is vitally important to communicate to the public and the scientific community the results of studies in accurate terminology. </LI></UL>
<P>
<H3>Introduction</H3>
<P>Most traits that vary in populations of humans and other organisms are determined by multiple factors. Most common diseases with a genetic component are such complex traits. The complexity arises from the fact that each factor contributes, at most, a modest amount to the total variation in the trait observed in the entire population. Complex traits may be continuous in distribution, like height or blood pressure, or they may be dichotomous, like "well" and "affected." Multiple genetic and environmental factors may interact with each other in unpredictable ways. Such unpredictable, nonlinear interactions mean that the expression of the trait may not be anticipated from knowledge of the individual effects of each of the component factors considered alone, no matter how well understood the separate components may be. Thus, the whole is not only greater than the sum of its parts, the whole may be different from the sum of its parts. 
<P>The genetic architecture of complex traits consists of a description of all of the genetic and environmental factors that affect the trait, along with the magnitude of their individual effects and the magnitudes of interactions among the factors. It is, in principle, possible to define the genetic components in terms of Mendelian segregation and location along a genetic map. Environmental factors are much less easily partitioned into separate factors whose individual effects and interactions can be sorted out. 
<P>It is critical to recognize that the genetic architecture is less a fundamental biological property of the trait than a characteristic of a trait in a particular population. The genetic architecture is a moving target that changes according to gene and genotype frequencies, the distributions of environmental factors, and such biological properties as age and gender. 
<P>The dependence on gene frequencies creates some seeming paradoxes of genetic architecture. For example, suppose a trait is completely determined by the interaction of two recessive alleles, one of which is rare and the other common. At the population level, the trait appears to be determined by the rare allele, because its presence limits the variation in the occurrence of the trait among individuals. If the allele frequencies were reversed, the other gene would appear to be the determining genetic cause. But in either instance, both recessive alleles contribute equally to the biological causation of the trait. 
<P>The implication of the population dependence is that the predominant genetic factors contributing to a complex trait may seem to differ from population to population. This is probably one reason for the apparent heterogeneity sometimes found in the results of genetic linkage studies in different populations. Insufficient statistical power in the linkage tests is also a possible explanation, and there is always the possibility that superficially identical complex traits in different populations may actually have different biological causes. 
<P>The existence of unpredictable, nonlinear interactions between the multiple factors affecting complex traits, as well as possible frequency-dependent differences in genetic architecture from one population to the next, emphasizes one of the principal conclusions of the December 10-11, 1997 workshop, "The Genetic Architecture of Complex Traits." The participants unanimously agreed that understanding the genetic and environmental basis of complex traits is not going to be easy and will not be achieved in a foreseeable time frame. Too little is known about the true nature of the complexity of such traits in any organism. 
<P>In an ideal case, when the factors are not numerous, when their main effects are quite large and their interaction effects quite small, and when interpopulation heterogeneity is minimal, very rapid progress can be made. It is by no means clear how widely actual complex traits in humans and other organisms depart from these ideal conditions. 
<P>Furthermore, while improved technology can be of tremendous importance, the challenges are not only technological. They are also conceptual (for example, how to identify nonlinear interactions, how to optimize computational algorithms); clinical (improved diagnostic criteria); and epidemiological (how to sample in such a way as to minimize spurious associations due to population structure and population history while maximizing the power to detect biologically significant associations). 
<P>Because the genetic architecture is a characteristic of a trait in a population, it is affected by population structure and population history--a fact that undermines the concept of a "disease gene." In a complex trait, there is no "disease gene" in the sense of a Mendelian factor that, by itself, causes a disease. Rather, the genetic and environmental factors underlying a complex trait must each be considered as contributing or predisposing rather than as determinative. Where diseases are concerned, genetic components may be regarded as risk factors. 
<P>In spite of these difficulties, the analysis of complex traits is fundamentally important to identifying the contributing genetic and environmental factors of traits and to understanding their underlying biology. The discussion and recommendations from "The Genetic Architecture of Complex Traits" workshop focused on opportunities for progress in four areas--research, resources, training, and communications--some of which can potentially be addressed by NIGMS. Other points will require discussion and action by other groups. 
<P>
<H3>Research</H3>
<P><I>Data</I> 
<P>The number of individuals that can be studied will ultimately determine the limit of resolution of analyses of complex traits. The sample sizes that researchers can collect and the quality of individual phenotype assignments (the ability to recognize and correctly assign trait values to individuals) are serious barriers to progress. Given the ongoing efforts to produce a dense, quality map of the human genome of single nucleotide polymorphisms (SNPs), sufficient numbers of markers to analyze the complex traits will be available soon. In human studies, therefore, the limitation will be correctly phenotyped individuals--that is, our ability to correctly diagnose disorders or completely describe traits. The situation is not quite the same for various model organisms where the development of new markers and new maps has lagged behind the effort for humans. 
<UL>
<LI><B>NIH and the scientific community should focus on better acquisition of data, including larger samples and more refined phenotype definitions.</B> This is especially true for studies of specific common diseases for which large samples of individuals who are assessed for disease according to the same criteria are very difficult to acquire. 
<P></P>
<LI><B>There is no single simple sampling scheme that should be used to obtain the data for these studies.</B> Data-rich does not necessarily mean information-rich. The questions being asked should dictate the analyses of choice, the types of data necessary, sampling design, and the final sample size. 
<P></P>
<LI><B>Given the reality that genetic architecture can and will vary as a function of population parameters, collection of data on population structure, including histories for different human populations and for model organisms, is essential and should be supported.</B> These data must include the definition of normal variation within these populations as well as the recording of disease phenotypes. Accumulation of these population data sets will also lead to studies of evolution within the human population and answers to questions about how these traits came to be and how some diseases achieve sufficient population frequency to be common within our population. Because the genetic architecture differs according to how closely the trait is related to survival and reproductive fitness, evolutionary forces can profoundly affect the genetic architecture of complex traits in humans and other organisms. </LI></UL>
<P><I>Methods</I> 
<P>The consensus of the participants at the workshop was that both current methods and fundamentally new approaches should be pursued aggressively. 
<UL>
<LI><B>The richness of the current methods should be exploited to its fullest through creative applications to appropriate data sets.</B> The current armamentarium of methods provides numerous ways to model traits, analyze data, and evaluate results for internal consistency and biological reality. However, just because these techniques and methods will allow us to learn a great deal, we should not hesitate to expand and develop them, to refine the models, and to improve them especially with respect to computational limitations. 
<P></P>
<LI><B>Every effort should be made to encourage the development of fundamentally new models and more sensitive methods of analysis.</B> Human geneticists have been slow to explore and adopt methods developed in other areas of science. For example, refinement of techniques for more rapid computation should be actively pursued. NIH, in its interaction with theoretical investigators and its review process, should encourage creative efforts even if they are not guaranteed to provide improvement. 
<P></P>
<LI><B>Because so little is known about genetic architecture, exploratory and observational studies should be encouraged.</B> Although the majority of NIH-funded grants are hypothesis-driven, science is, in fact, built on observations. We are sufficiently naïve about complex traits that exploratory studies must be supported. 
<P></P>
<LI><B>NIGMS should encourage collaborative studies among investigators in diverse disciplines.</B> Complex traits are not the province of any single discipline. The expertise of molecular biologists, biochemists, clinicians, evolutionary biologists, developmental biologists, mathematical and statistical geneticists, and many others is needed. 
<P></P>
<LI><B>With respect to review and funding of research grant applications, it is important to emphasize that no single method or model for studying genetic architecture can be adopted universally.</B> No single method can answer all, or even some, of the questions without being used in concert with additional approaches. It is a serious error to insist that all studies apply one method, such as association studies for linkage, when the full suite of methods is available to the investigators. In addition, it is inappropriate to abandon the candidate gene approach in favor of general genome searches for those traits where reasonable biological or physiological candidates can be identified. It is also inappropriate to insist on a candidate gene approach when a whole-genome search is justified by sample size and cost-effectiveness. Significant information has accrued through both the exploration of candidate gene regions and the rejection of candidate gene hypotheses. </LI></UL>
<P><I>Model Organisms</I> 
<P>The overriding advantage of model organisms is the ability to do both genetic and environmental manipulation that cannot be done with human beings. Studies using animal models to explore the genetic architecture of complex traits should be supported in order to identify general principles and pathways and to gain a broad understanding of the biology of complex traits. 
<UL>
<LI><B>The choice of organism and research design should be dictated by the complex trait of interest and the questions being asked.</B> There is no single, limited set of organisms that is sufficient for these studies. Studies of non-traditional organisms may have much to contribute. 
<P></P>
<LI><B>Studies of population structure and variability of organisms in natural populations are needed.</B> Because the genetic architecture of complex traits varies with context, measuring traits and identifying their variability in the wild is an important piece of the puzzle, and studies to do so have almost ceased to exist. </LI></UL>
<P>
<H3>Resources</H3>
<UL>
<LI><B>Support is needed for new database structures for population data.</B> A great deal of population data already exists, but most public databases, such as GenBank, are inadequate for handling population data, which must include the population source of the allele sequences and the frequency with which they occur. Many data currently exist on individual investigators' computers, but are not accessible to other scientists because there are no good mechanisms for sharing data. The study of population structure would be greatly enhanced by establishing one or more databases to make these data easily and reliably accessible. Pilot efforts might begin by augmenting the Genome Database (GDB) for human data or FlyBase for Drosophila data. 
<P></P>
<LI><B>The development of publicly available genetic data sets of genetic maps and haplotypes should be encouraged.</B> There are numerous NIH-supported studies that individually have low power to detect and map factors that contribute to complex traits. NIGMS should support the establishment of databases that enable data from such studies to be combined. Further, many successful gene mapping studies generate marker data that could be useful to other investigators and that could be made publicly available once the gene of interest is mapped. </LI></UL>
<P>
<H3>Training</H3>
<UL>
<LI><B>The expertise of computational scientists, including physicists, mathematicians, and engineers, is needed; however, most will need to be retrained in statistical genetics.</B> One of the impediments to such recruitment is the relatively low salary scale of entry-level postdoctoral students in biology. 
<P></P>
<LI><B>NIH should support training in statistical genetics for scientists who intend to apply the tools of genetic analysis.</B> Statistical methodologies must be applied knowledgeably, especially where human data are concerned. Many scientists, including clinicians, molecular geneticists, and others, are eager for basic training in analytical methods so that they can collaborate effectively with their statistical colleagues. 
<P></P>
<LI><B>Multidisciplinary training is essential.</B> Studies of complex traits are inherently multidisciplinary, requiring expertise in genetics, statistics, computational biology, and other areas. In human studies, a strong clinical component is often essential because of the need for careful and correct diagnoses. All too often there are structural or institutional barriers to multidisciplinary collaboration and training. Students must be prepared to cross these disciplinary boundaries if they are to succeed and contribute to future research studies. </LI></UL>
<P>
<H3>Communications</H3>
<P>The genes involved in complex traits are contributing factors rather than "disease genes." Any one of the genetic factors that contribute to a complex trait may actually account for a relatively small proportion of the total variation in the trait. Furthermore, by itself, the gene may not cause the disease, but rather may be one of many contributing genetic and environmental factors. The danger of oversimplification is to mislead the public into thinking that a disease has been conquered and effective new treatments and therapies are just around the corner. There is a great danger in raising false hopes among the public. 
<UL>
<LI><B>The analysis of complex traits does not lend itself to quick and easy solutions.</B> We do not yet know the true degree of complexity of complex traits. We hope some will approximate the simplest ideal case and be analyzed rather quickly. We expect others will be much more difficult. It is prudent to make no promises until we understand the nature and extent of complexity in genetic systems. 
<P></P>
<LI><B>At the same time, it is vitally important to communicate the results of studies in accurate terminology.</B> There is no gene for hypertension, depressive disorder, obesity, or any other complex trait. All genes that affect the trait are contributing factors that are more or less important only in relation to other contributing genetic and environmental factors in defined populations. Likewise, terms such as "heritability" are misleading because the common usage differs from the technical definition. In its common usage, "heritability" implies transmissibility, whereas its technical meaning is a ratio of variances. The public, as well as our scientific and medical colleagues, needs current and accurate information in order to understand the issues surrounding the study of complex traits. Scientists should take great care to communicate clearly in order to promote understanding of these difficult and very important issues. </LI></UL>
<P>
<H3>Conclusions</H3>
<P>The findings of the participants at "The Genetic Architecture of Complex Traits" workshop do not lend themselves to simplistic answers or quick fixes. The recommendations need to be considered thoughtfully and thoroughly, sometimes in collaboration with a broad spectrum of the scientific community. Success will depend on coordination among institutes and agencies and on increased understanding of the complexities of the scientific questions being asked. The participants note that NIGMS can address many issues; however, some recommendations are trans-NIH. The National Advisory General Medical Sciences Council may wish to consider broader dissemination of the report and recommendations. </P></span>
									
									
								</div>
								
								
								<div id="PrintContent2">
								
                                
                                
                                
                                
                                
                                
                                
                                
                                
								
								</div>
							</td>
						</tr>
					</table></div>
					<!-- BEGIN PAGE UTILITIES -->
				
<div id="pageutilities">
	
<a href="javascript:PrintThisPage();">
	<img src="/nigms.nih.gov/Images/printer_icon.gif" alt="" width="16" height="13" border="0" align="absMiddle" />
	Print this page
</a>
 
<a href="javascript:emailpage()"><img src="/nigms.nih.gov/Images/email_icon.gif" alt="" width="15" height="13" border="0" align="absMiddle" />
		E-mail this page</a>
<a href="http://publications.nigms.nih.gov/order/managecart.html"><img src="/nigms.nih.gov/images/cart_icon.gif" alt="My Cart" width="19" height="13" border="0" align="absMiddle" style="position:relative; top:-1px;" /> My Order</a>
</div>

<!-- END PAGE UTILITIES -->
				<!-- END MAIN CONTENT -->
	
<!-- End content area -->      

<div id="lastupdated">This page last updated
	November 19, 2008</div>
</div>        
<div id="footer">
	<p><a href="/">NIGMS Home</a> | <a href="http://search2.google.cit.nih.gov/search?q=&amp;site=NIGMS&amp;client=NIGMS_frontend&amp;proxystylesheet=NIGMS_frontend&amp;output=xml_no_dtd&amp;filter=undefined&amp;getfields=*&num=20&numgm=5">
			Site Search</a> | <a href="/SiteMap.html">Site Map</a> | <a href="/ContactUs.html">
			Contact Us</a> | <a href="/Privacy.html">Your Privacy</a> 
		| <a href="/Accessibility.html">Accessibility</a> | <a href="/Disclaimers.html">
			Disclaimers</a> | <a href="/FOIA.html">FOIA</a></p>
	<p>National Institute of General Medical Sciences&nbsp;•&nbsp;45 Center Drive MSC 
		6200&nbsp;•&nbsp;Bethesda, MD 20892-6200<br>
		Tel: 301-496-7301&nbsp;•&nbsp;E-mail: <a href="mailto:info@nigms.nih.gov">info@nigms.nih.gov</a></p>
	<p><a href="http://www.nih.gov/" target="_blank"><img src="/nigms.nih.gov/Images/footer_nihlogo.gif" alt="NIH - National Institutes of Health"
				width="37" height="37" border="0" class="footerlogo" /></a><a href="http://www.hhs.gov/" target="_blank"><img src="/nigms.nih.gov/Images/footer_hhslogo.gif" alt="DHHS - Department of Health and Human Services"
				width="37" height="37" border="0" class="footerlogo" /></a><a href="http://www.usa.gov/" target="_blank"><img src="/nigms.nih.gov/Images/footer_usagovlogo.gif" alt="USA.gov - Government Made Easy" width="92" height="37"
				border="0" class="footerlogo" /></a></p>
</div>

<script language="javascript" type="text/javascript" src="/nigms.nih.gov/CMS/NIHCMS/Includes/NIHCMSConsole.js"></script>





<!-- Generated by Microsoft.ContentManagement.WebControls.Console to reset form.action -->
<SCRIPT type="text/javascript">
<!--

    var __CMS_PostbackFormBeenReset = false;

    function __cmsRestFormAction()
    {
        if ( !__CMS_PostbackFormBeenReset )
        {
            // set form postback Url to be current translated CMS Url
            __CMS_PostbackForm.action = __CMS_CurrentUrl;
        }
    }

    var __cmsResetFormCachedPostBack = null;
    if( typeof window.__doPostBack != "undefined" )
    {
        __cmsResetFormCachedPostBack = __doPostBack;
        __doPostBack = __cmsResetFormOverridePostBack;
    }

    var __cmsResetFormCachedOnSubmit = null;
    if( typeof __CMS_PostbackForm.onsubmit != "undefined" )
    {
        __cmsResetFormCachedOnSubmit = __CMS_PostbackForm.onsubmit;
        __CMS_PostbackForm.onsubmit = __cmsResetFormOverrideOnSubmit;
    }
    else
    {
        __cmsResetFormCachedOnSubmit = null;
        __CMS_PostbackForm.onsubmit = __cmsResetFormOverrideOnSubmit;
    }

    function __cmsResetFormOverridePostBack( eventTarget, eventArgument )
    {
        __cmsRestFormAction();
        __cmsResetFormCachedPostBack( eventTarget, eventArgument );
    }

    function __cmsResetFormOverrideOnSubmit()
    {
        __cmsRestFormAction();
        if (__cmsResetFormCachedOnSubmit != null)
            return __cmsResetFormCachedOnSubmit();
        else
            return true;
    }
// -->
</SCRIPT>


</form>

</html>

