<HTML>

<HEAD>

<meta name="description" content="The Biometric Research Branch (BRB) is the statistical and biomathematical component of the Division of Cancer Treatment, Diagnosis and Centers (DCTDC). Its members provide statistical leadership for the national and international research programs of the division in developmental therapeutics, developmental diagnostics, diagnostic imaging and clinical trials.">

<meta name="keywords" content="biostatics, Clinical trials, Cancer Research, Statistics, BioInformatics, Computational Biology, Genomics, Biomathematics">

<title>NCI: DCTD: Biometric Research Branch</title>

<STYLE TYPE="text/css">

  BODY {background-color: #ffffff; font-family: Arial, Helvetica, sans-serif;}

  UL {font-family: Arial, Helvetica, sans-serif; font-size: 10pt;}

  P {font-family: Arial, Helvetica, sans-serif; font-size: 10pt;}

  .crumb {background-color: #ccccff; font-family: Arial, Helvetica, sans-serif;}

  .gNavBar {background-color: #993333;}

</STYLE>



<SCRIPT LANGUAGE="JavaScript">



<!--

function gotoFunction() 

{

  if(document.gotoCtl.siteList.options[document.gotoCtl.siteList.selectedIndex].value != "")

  {  

    self.location = document.gotoCtl.siteList.options[document.gotoCtl.siteList.selectedIndex].value;

  }

}

// -->



</SCRIPT>





</HEAD>

<BODY>
<table cellspacing=0 cellpadding=0 width="100%" border=0>
  <tbody> 
  <tr> 
    <td valign=center align=middle height=73> 
      <table height=73 cellspacing=0 cellpadding=0 width="149%" border=0>
        <tbody> 
        <tr> 
          <td valign=center align=left width=100><a 

            href="http://www.cancer.gov/index.html"><img height=55 

            src="nciLogo-rev.jpg" width=86 border=0></a></td>
          <td valign=center align=middle width=452 

          background=> 
            <center>
              <a href="http://linus.nci.nih.gov/~brb"><img height=128 

            src="BRBheader3.jpg" width=513 border=0></a> 
            </center>
          </td>
          <td valign=center align=right width=396 

      height=73></td>
        </tr>
        </tbody> 
      </table>
    </td>
  </tr>
  </tbody> 
</table>
<table class=crumb cellspacing=0 cellpadding=4 width="100%" border=0>
  <tbody> 
  <tr> 
    <td width="100%"> 
      <div class=crumb><!-- start Breadcrumbs --> 
        <p></p>
        <!-- end Breadcrumbs --></div>
    </td>
  </tr>
  </tbody> 
</table>
<div align=center> 
  <h4></h4>
</div>
<TABLE WIDTH="100%" ALIGN="CENTER" CELLPADDING="0" CELLSPACING="2" border="0">

  <TR>



    <TD VALIGN="top" height="2067"> 
      <div align="center"> 
        <p align="center"><font face="Verdana, Arial, Helvetica, sans-serif" size="2" color="#0000FF"><b>Development 
          of statistical methods and bioinformatic systems for decrypting the 
          peptide motifs recognized by T-cell clones</b></font> </p>
        </div>

      <P> Much research in the etiology and treatment of auto-immune diseases 
        is directed toward identifying the self antigens that are the target of 
        attack by T-cells and identifying the events that result in proliferation 
        of such T-cell clones. Identifying tumor antigens and peptide epitopes 
        that can elicit a T-cell response is also a key component in the development 
        of therapeutic cancer vaccines. Drs. Simon, Zhao, Sung, Koo and Mr. Grover 
        have been collaborating with Dr. Roland Martin's laboratory of neuroimmunology 
        in the National Institute of Neurological Diseases and with Dr. Clemencia 
        Pinilla of the Torry Pines Institute of Molecular Medicine on the use 
        of combinatorial peptide libraries to provide data that enables the decryption 
        of the motifs recognized by T-cell clones. We have developed statistical 
        models that use the data from T-cell stimulation assays using combinatorial 
        peptide libraries to estimate positional scoring matrices for predicting 
        the potential of any peptide for stimulating the T-cell clone in an MHC 
        dependent manner. More complex decision tree and neural network models 
        have also been developed to facilitate understanding the principles of 
        the peptide-MHC molecule-T cell receptor interactions. We have built <a href="http://linus.nci.nih.gov/TEST.html">a 
        web based bioinformatic analysis system</a> that enables the searching of 
        a translated version of Genbank (Genpept) to identify self peptides and 
        pathogen peptides that may have served as mimics in the initial stimulation 
        of the T-cell clone. In order to perform efficient searches, we maintain 
        the current version of Genpept in our MySql database on our four processor 
        Linux server. The models have been extended to use of combinatorial peptide 
        libraries with other types of assays. T-cell clones from a variety of 
        disease states, including cancers have been evaluated. 
      <P><b>Prediction of immune response based on TCR stimulation models constructed 
        from combinatorial peptide library assay and MHC binding</b> 
      <P>T cell receptors (TCR) recognize antigenic peptides in complex with the 
        major histocompatibility complex (MHC) molecules and this trimolecular 
        interaction initiates antigen-specific signaling pathways in the responding 
        T lymphocytes. For the study of autoimmune diseases and vaccine development 
        it is important to identify peptides (epitopes) that can stimulate a given 
        TCR. The use of combinatorial peptide libraries has recently been introduced 
        as a powerful tool for this purpose. A combinatorial library of n-mer 
        peptides is a set of complex mixtures each characterized by one position 
        fixed to be a specified amino acid and all other positions randomized. 
        A given TCR can be fingerprinted by screening a variety of combinatorial 
        peptide mixtures using a proliferation assay. We have developed statistical 
        models for elucidating the recognition profile of a TCR using combinatorial 
        library proliferation assay data and known MHC binding data. The model 
        can be used to predict potential molecular mimic peptides that are recognized 
        by the given TCR. 
      <P>
      <P><b>Prediction of MHC binding peptides with a model capturing their biophysical 
        properties from a database of known binders </b>
      <P>We have developed a new method for predicting MHC binding of peptides 
        using biophysical parameters of the constituent amino acids. Unlike conventional 
        matrix-based methods, our method does not assume independent binding of 
        the individual side chains and accommodates any inter-residue interactions. 
        The method discovers the most common 9-mer "property profile" within the 
        peptides in the publicly available database MHCPEP. The model can be used 
        to predict MHC binding regions in genomes of a pathogen such as HIV, which 
        may facilitate, for example, a rational development of vaccines. 
      <P><b>Degenerate nature of T-cell recognition of peptide-MHC complexes </b>
      <P>It is generally accepted that a high level of antigen specificity is 
        required for T-cell activation in order to avoid potentially pathogenic 
        immune responses to self antigens. On the other hand, T cells need to 
        be widely crossreactive in order for a comprehensive response to a diverse 
        spectrum of foreign antigens. We investigate how the immune system satisfies 
        these two albeit conflicting requirements. A simple probabilistic model 
        is extended in order to incorporate several known mechanisms of the immune 
        system. 
      <P>
      <P><b><font face="Times New Roman, Times, serif" size="3"><u>References:</u></font></b> 
      <P>Hemmer B, Gran B, Zhao Y, Marques A, Pascal J, Tzou A, Kondo T, Cortese 
        I, Bielekova B, Straus S, McFarland HF, Houghten R, Simon R, Pinilla C, 
        and Martin R. <b>Identifiction of candidate T cell epitopes and molecular 
        mimics in chronic Lyme disease</b>. <i>Nature Medicine</i> 5:1375-82,1999. 
      <P>Pinilla, C., Rubio-Godoy, V., Dutoit, V., Guillaume, P., Simon, R., Zhao, 
        Y., Houghten, R. A., Cerottini, J., Romero, P, and Valmori, D. <b>Combinatorial 
        peptide libraries as an alternative approach to the identification of 
        ligands for tumor reactive cytolytic T lymphocytes.</b> <i>Cancer Research</i> 
        61:5153-5160, 2001. 
      <P>Martin, R., Gran, B., Zhao, Y., Markovic-Plese, S., Bielekova, B., Marques, 
        A., Sung, M., Hemmer, B., Simon, R., MaFarlan, H. F., and Pinilla, C. 
        <b>Molecular mimicry and antigen-specific T-cell response in multiple 
        sclerosis and chronic CNS lyme disease.</b> <i>J. Autoimmunity</i> 16:187-192, 
        2001. 
      <P>Zhao, Y., Gran, B., Pinilla, C., Markovic-Plese, S., Hemmer, B., Tzou, 
        A., Whitney, L. W., Biddison, W. E., Martin, R., and Simon, R. <b>Combinatorial 
        peptide libraries and biometric score matrices permit the quantitative 
        analysis of specific and degenerate interactions between clonotypic T-cell 
        receptors and MHC-peptide ligands.</b> <i>J. Immunol.</i> 167:2130-3141, 
        2001. 
      <P>Zhao, Y., Grovel, L., and Simon, R. <b><a href="http://linus.nci.nih.gov/TEST.html">TEST: a web-based T-cell Epitope 
        Search Tool.</a> </b><i>Proceedings of the 14th IEEE symposium on computer-based 
        medical systems</i>, p.493- 497, 2001. 
      <P>Rubio-Godoy, V., Pinilla, C., Dutoit, V., Borras, E., Simon, R., Zhao, 
        Y., Cerottini, J., Romero, P, Houghten, R. A., and Valmori, D. <b>Towards 
        PS-SCL based identification of CD8+ tumor reactive T cell ligands: a comparative 
        analysis of PS-SCLs recognition by a single tumor-reactive CD8+CTL clone1. 
        </b><i>Cancer Research</i>, 2002, In press. 
      <P>Sung, M., Zhao, Y., Martin, R., and Simon, R. <b>T-cell epitope prediction 
        with Combinatorial Peptide Libraries. </b><i>J. of Computational Biology</i>, 
        2002, In press. 
    </TD> 
  </TR>    
</TABLE>
</BODY></HTML>

