<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD>
<TITLE>Computational Chemistry : Molecular Recognition Section : NIDDK</TITLE>



<BASE HREF="http://mgddk1dev2.niddk.nih.gov/">

<SCRIPT LANGUAGE="JavaScript">

<!-- Begin
function formHandler(form){
var URL = document.form.site.options[document.form.site.selectedIndex].value;
window.location.href = URL;
}
// End -->
</SCRIPT>






<script language="JavaScript" type="text/javascript">
<!--

if (navigator.appVersion.toLowerCase().indexOf("win")!=-1)
{

document.writeln("<link REL=stylesheet HREF=\"win.css\" type=\"text/css\">")
}
else {

document.writeln("<link REL=stylesheet HREF=\"mac.css\" type=\"text/css\">")
}

// --></script>



<STYLE type="text/css">
<!--
A:hover { color:#003A19 }
//-->
</STYLE>
</head>
<BODY>

<!-- cascaded nav bar -->
<script type='text/javascript'>

//HV Menu- by Ger Versluis (http://www.burmees.nl/)
//Submitted to Dynamic Drive (http://www.dynamicdrive.com)
//Visit http://www.dynamicdrive.com for this script and more

function Go(){return}

</script>
<!--<script type='text/javascript' src='exmplmenu_var.js'></script>
<script type='text/javascript' src='menu_com.js'></script>-->
<noscript>Your browser does not support script</noscript>

<table>
<tr><td colspan="2">



<!-- start header -->

	<!--  header row -->

<table width="610" border="0" cellspacing="0" cellpadding="0">
<TR><TD align="center">
<img src="/images/header.gif"  border="0" Alt="Molecular Recognition Section, Division of Intramural Research : NIDDK : National Institutes of Health"></TD>
</TR>

</table>

<!--end header-->

<!-- end  header row -->

</td></tr>
<tr><td>
<table><tr><td width="150" height="280" valign="top"><font color="white">NavigationINDEX</font></td>
<td valign="top">


<!-- start content -->

<H1>Computational Chemistry</H1>

<B>An Introduction to Modelling G Protein-Coupled Receptors</B>

<P>
This page describes the current research in our lab to better understand the
molecular mechanisms involved in receptor-ligand interactions.

<P>
For this purpose we use

<UL>

<LI>

<a href="/medchem.shtml">classical medicinal chemistry</A>,


<LI>

<A NAME="compchem">computational medicinal chemistry</A>,

and

<LI>

<a href="/molbio.shtml">molecular biology</A>

tools.

</UL>

<P>
This particular page is an excerpt from a publication:<BR>

<P>
<UL>
<LI>van Rhee, A.M.; Fischer, B.; Jacobson, K.A.<BR>
Modelling the P<sub>2Y</sub> purinoceptor using rhodopsin as template.<BR>
<I>Drug Design Discovery</I>, <B>1995</B>, <I>13</I>: 133-154.<BR>
</UL>

<P>
which describes how we approached the problem of visualizing the molecular
architecture of the G protein-coupled

<a href="/pharmacol.shtml">P2Y<sub>1</sub> purinoceptor</A>.

<P>
It is the intention to add more features (both classical medicinal chemistry
and computational chemistry) as we go along, but time is the limiting factor.
You could make a bookmark now, and check in every now and then.

<P>
A very characteristic feature of

<a href="/GPCR.shtml">G protein-coupled receptors</A>

(GPCRs) is their overall topology:

<P>
<IMG SRC="images/cartoon80.gif">

<P>
They all have similar

<A HREF="/compchem.shtml#KD">hydrophobicity profiles</A>

with seven stretches of increased hydrophobicity, most probably corresponding
to alpha-helical regions spanning the cell membrane.<BR>

<P>
Unfortunately, isolation, purification and crystallization of

<a href="/pharmaco2.shtml">P<sub>2</sub> purinoceptors</A>,

as for other GPCRs, remains to be demonstrated.

<P>
Not only is a crystallographic structure as yet unobtainable, but the very
limited quantities of receptor protein present in the membrane, and isolation
problems, prohibit the use of modern techniques, such as nuclear magnetic
resonance spectroscopy, towards structure elucidation.

<P>
Chemical modification<sup>

<A HREF="/compchem.shtml#26">26</A>

</sup> of the ligand binding site and

<a href="/MutationAnalysis.shtml">site-directed mutagenesis</A>

have been shown to be useful techniques in understanding receptor-ligand
interactions, but is particularly in the case of

<a href="/pharmaco2.shtml">P<sub>2</sub> purinoceptors</A>

restricted by the lack of suitable radioligands.

<P>
A possible further approach is to construct models of the receptors and their
ligand complexes using molecular modelling, as was first proposed by

<A HREF="/compchem.shtml#27">Hibert et al</A>.<BR>

<P>
On these pages we present models of the chick

<a href="/ATP.shtml">P<sub>2Y</sub> purinoceptor</A>,

and the human

<a href="/adenosine.shtml">A<sub>2A</sub> receptor</a>

based on the electron density map of rhodopsin<sup>

<A HREF="/compchem.shtml#28">28</A>

</sup>, rather than on the bacteriorhodopsin template<sup>

<A HREF="/compchem.shtml#29">29</A>

</sup> used by

<A HREF="/compchem.shtml#27">Hibert et al</A>.

<P>
The models are verified by incorporation of site-directed
mutagenesis<sup>

<A HREF="/compchem.shtml#41">41</A>,

<A HREF="/compchem.shtml#50">50</A>,

<A HREF="/compchem.shtml#51">51</A>

</sup> and ligand affinity data.<sup>

<a href="/compchem.shtml#14">14</A>,

<a href="/compchem.shtml#15">15</A>

</sup><BR>

<P>

<HR>

<B>Computational Methods.</B><BR>
<I>Software and Hardware.</I><BR>
Calculations and manipulations were performed using the<BR>
<UL>
<LI>

<A HREF="http://www.accelrys.com" TARGET="_top">Quanta</A>,<BR>

<LI>

<A HREF="http://www.accelrys.com" TARGET="_top">Insight II/Discover</A>, and

<LI>

<A HREF="http://cmm.info.nih.gov/modeling/guide_documents/tocs/computation_software.html" TARGET="_top">
MOPAC</A>

molecular modelling packages.<BR>
</UL>

<P>
<I>Model building.</I><BR>

<a href="/compchem.shtml#KD">Kyte-Doolittle hydrophobicity</A> and

<a href="/compchem.shtml#SP">Emini surface probability</A>

parameters were calculated using a 7 amino acid window. With these parameters,
the putative

<A HREF="concise.shtml">transmembrane domains</A>

(TMs) were identified.

<P>
Sequences of the putative transmembrane domains were aligned manually using
the alignment procedure described on our

<A HREF="/GPCR.shtml">&quot;GPCR page&quot;</A>.

<P>
Helices were built from the full length of the TM, using the Biopolymer module
in the Insight II program. The N terminus was capped with an acetamido group,
and the C terminus was capped with a carboxamido group. The secondary structure
was assumed to be a right-handed alpha helix (default phi and psi angles from
the Biopolymer module). After generation of the Amber atom type parameters and
template charges, the helices were energy minimized as described in the

<a href="/compchem.shtml#EM">Energy Minimization section</A>.<BR>

<P>

The energy minimized helices were converted to

<A HREF="http://www.pdb.bnl.gov" TARGET="_top">Protein Data Bank</A>

(PDB) format files and imported in the Quanta program. Coils (an alpha
carbon backbone representation of the helix) of the minimized helices were
drawn, that resemble the helical wheel representation to aid in orienting
the individual helices towards each other.<BR>

<P>
The following constraints were used constructing the helical bundle:

<P>
<UL>
<LI>1. The helical axes of the first, third, fifth and seventh helices were
quasi-antiparallel to those of the second, fourth and sixth helices.<BR>
<LI>2. The hydrophobic side of each helix was facing the lipid phase and the
hydrophilic side of each helix was facing either another helix or the pore
formed by the helical bundle.<BR>
<LI>3. Conserved residues, either identical throughout a subset or highly
homologous, determined the orientation of each helix relative to the other
helices.<BR>
<LI>4. Distances in the electron density map of
rhodopsin<sup>

<a href="/compchem.shtml#28">28</A>

</sup> correlated to atomic coordinates in the model. This was obtained through
triangulation of the electron density map and conversion of the coordinates to
a grid on the screen.<BR>
<LI>5. The assembly of helices maintained a clockwise order, when seen from
the intracellular side, as argued by
Baldwin.<sup>

<a href="/compchem.shtml#31">31</A>

</sup><BR>

<LI>6. None of the helices were intersecting.
</UL>

<P>

<A NAME="EM"><I>Energy Minimization.</I></A><BR>

The individual helices generated with Insight II's Biopolymer module were
energy minimized in Discover using a stepwise process. Initially, 200 steps of
Steepest Descent were performed, followed by minimization using the Conjugate
Gradient (CG) method until the gradient reached a value below 0.1
kcal/mol/&Aring;. Use of the Amber forcefield in Discover required
that in
all calculations the 1-4 nonbonded interactions were scaled by a factor 0.5,
and the dielectric constant was assumed to be distance independent with a
magnitude of 4. The bond, theta, phi, out-of-plane, nonbonded and Coulombic
interactions were all used to obtain the final energy, but were not expressly
scaled.<BR>

<P>
The helical bundles generated with Quanta were energy minimized in Discover in
a stepwise process. Initially, 500 steps CG were performed with the backbone
of the helices tethered with a force constant of 100 kcal/&Aring;.
In
consecutive runs (500 steps CG each), the force constant was reduced to 50
kcal/&Aring;, then 25 kcal/&Aring; and, finally, to 0
(no tethering).<BR>

<P>

<A NAME="LD"><I>Ligand Docking.</I></A><BR>

Selective and/or specific ligands for the receptors were fully minimized using
the MNDO hamiltonian of MOPAC. All ligands were rigidly docked into the helical
bundle using graphical manipulation coupled to continuous energy monitoring,
<I>i.e.</I> the ligand was manually docked into the binding site without
relaxing the atomic coordinates of either ligand or protein while continuously
calculating the energy of the whole. When a final position was reached,
consistent with a low local energy and known pharmacological data, the complex
of receptor and ligand was subjected to a minimization run of 4000 steps CG
(or until the gradient was &lt; 0.1 kcal/mol/&Aring;). Charges for the ligands
were imported from the MOPAC output files.<BR>

<P>

<HR>

<A NAME="discussion"><B>DISCUSSION.</B></A><BR>

<P>
We will now briefly discuss the results obtained through this approach,
but the information specific to either the

<A HREF="/ATP.shtml">P<sub>2Y</sub> purinoceptor</A>

or the

<A HREF="/adenosine.shtml">adenosine A<sub>2A</sub></A>

receptor is kept on separate pages.

<P>
<I>

<a href="/concise.shtml">Alignment of G protein-coupled receptors</A>

and

<a href="/dendrogram.shtml">dendrogram</A>.</I><BR>

In TM1, the motif <B>G</B>XXX<B>N</B> (P<sub>2Y</sub>) or <B>G</B>XX<B>GN</B>
(P<sub>2U</sub>) occurs in P<sub>2</sub> purinoceptors, rather than the
<B>GN</B> motif present in biogenic amine or adenosine receptors. The

<a href="/comprehensive.shtml">alternate motifs</A>,

however, are not exclusively used by P<sub>2</sub> receptors, but are shared
with, <I>e.g.</I>, the PGE<sub>3</sub>-II
(<B>G</B>XX<B>GN</B>) and the PAF (<B>G</B>XXX<B>N</B>) receptors. In
all sequences the last 5 C-terminal residues of this helix are frequently
occupied by lysine or arginine residues, indicating the end of the
transmembrane domain. Such basic residues may serve as 'membrane
anchors',<sup>

<a href="/compchem.shtml#34">34</A>,

<a href="/compchem.shtml#42">42</A>

</sup> and are useful in determining the position of the helix in the lipid
bilayer.<BR>

<P>
In TM2, the residue preceding the conserved leucine in the <B>L</B>XXX<B>D</B>
motif is a conserved serine for the biogenic amine receptors, but is either
an asparagine or a histidine in the P<sub>2Y</sub> and P<sub>2U</sub>
receptors, respectively. The asparagine is conserved among the P<sub>2Y</sub>,
AT<sub>1A</sub>, CKR1, IL-8A, PAF, NK2 receptors, the opsins and rhodopsin.
The histidine is shared between P<sub>2U</sub> and thrombin receptors.<BR>

<P>
Another difference between biogenic amine receptors and P<sub>2</sub> receptors
is the position of the conserved proline relative to the conserved aspartate.

<P>
Although absent in the muscarinic receptor, the proline is consistently spaced
by 8 residues from the aspartate in the other biogenic amine receptors, whereas
in many other receptors, the P<sub>2</sub> receptors included, only 7 residues
separate these two residues.

<P>
This last difference is possibly significant since the proline is located near
the luminal side of the receptor protein, which is thought to be important for
ligand binding.

<P>
The only exceptions, observed so far, to the conservation of the aspartate
occur in the substance P receptor where it is substituted with a glutamate,
and in the gonadotropin-releasing hormone receptor where an asparagine replaces
this residue.

<P>
The conserved aspartate in TM2 was shown by

<a href="/compchem.shtml#53">Horstman et al</A>

to be a sodium-dependent allosteric regulatory site in the
alpha<sub>2</sub>-adrenergic receptor.<BR>

<P>
The <B>DRY</B> motif, characteristic of the third transmembrane domain in
biogenic amine receptors, is replaced by a <B>HRY</B> motif in the
P2Y<sub>1</sub> receptor (<B>HRC</B> in the P<sub>2U</sub> receptor).
This motif has not been found in other receptors.
The only other substitution for the conserved aspartate, seems to be the D to
E in, e.g., the TXA<sub>2</sub>, PGE<sub>3</sub>-II, opsin and rhodopsin
sequences. The significance of this deviation is not clear, but is supposedly
important for coupling of the P<sub>2</sub> receptors to G proteins and not
for ligand binding, the subject of this study.<BR>

<P>
One or more of the first 6 positions in the N-terminal sequence of TM4 are
generally (37 out of 40

<a href="/extended.shtml">aligned sequences of family A GPCRs</A>

) occupied by lysine or arginine residues. Again these residues could
well serve as 'membrane anchors'. They seem to be occurring more frequently
towards the cytosolic side of helices than on the luminal side, thus reflecting
the polarity of the membrane. These 'membrane anchors' also occur much less
frequently in the C-terminal sequences of helices, than in the N-terminal
sequences, as is demonstrated by the alignment for TM5. The reason for this
disparity is believed to be the direction in which the side-chains are
pointing, <I>i.e.</I> opposite to the propagation direction of the helix,
allowing more rotational freedom for lysine and arginine residues near the
cytosolic N-terminus than for the cytosolic C-terminus.<BR>
There are no other marked differences between the
various receptor subfamilies for either TM4 or TM5.<BR>

<P>
The <B>C</B>XX<B>P</B> motif used for the alignment of TM6 could be substituted
by <B>W</B>X<B>P</B> for most receptors, but the P<sub>2</sub> receptors
deviate at this position. The P<sub>2Y</sub> receptor has a tyrosine at the
position of the conserved tryptophan, and this seems to be rather unique. The
phenylalanine in the P<sub>2U</sub> receptor at the same position occurs more
frequently, such as in the thrombin, and various orphan receptors.
Characteristic of the P<sub>2</sub> receptors is the presence of a lysine
(P<sub>2Y</sub>: K269) or an arginine (P<sub>2U</sub>: R265) at an otherwise
non-conserved position. It shares this feature only with the endothelin
receptors (both ET-A and ET-B subtypes) and the orphan receptor RSC338. The
proposed 'membrane anchors' occur quite frequently at the first position in
the alignment of this transmembrane domain.<BR>

<P>
TM7 is best aligned by means of the <B>NP</B>XX<B>Y</B> motif. However, the
P<sub>2</sub> receptors, the gonadotropin-releasing hormone receptor, the
TXA<sub>2</sub>, PGE<sub>3</sub>-II, and several orphan receptors constitute
an exception to this rule. The conserved asparagine is replaced by an
aspartate residue in the latter cases. A non-conserved aspartate (D352) in
the 5HT<sub>1B</sub> receptor aligns perfectly with aspartates in the IL-8A
receptor (D288), and in the D<sub>1B</sub> receptor (D342), a glutamine in
the P<sub>2Y</sub> receptor (Q296) and a lysine in the P<sub>2U</sub> receptor
(K289). Arginines 299 (P<sub>2Y</sub>) and 292 (P<sub>2U</sub>) align with
Y530 in the m3, E291 in the IL-8A and E287 in the CKR1 receptor, respectively.
Both positions are near the luminal side of the receptor and are likely
involved in ligand binding. The K and R residues occurring near the C-terminus
are probably better characterized as 'membrane anchors'.<BR>

<P>
If you are interested in the

<a href="/comprehensive.shtml">alignment of over 150 GPCRs</A>,

or in

<a href="/MutationAnalysis.shtml">mutagenesis studies of GPCRs</A>,

just follow these links. You might want to read our

<a href="/GPCR.shtml">introductory page</A>

first, though.

<P>
Bacteriorhodopsin, included in this study to facilitate comparison with other
modelling studies, <sup>[e.g.,

<a href="/compchem.shtml#27">27</A>,

<a href="/compchem.shtml#35">35</A>]

</sup> is clearly not related to any of the GPCR subfamilies. The degree of
relatedness between bacteriorhodopsin and GPCRs shown in the

<a href="/dendrogram.shtml">dendrogram</A>

is probably an overestimate of the actual distance, caused by the residue
alignment procedure of the GCG program.<BR>

<P>

<I><A NAME="KD">Kyte-Doolittle hydrophobicity</A>

 and Emini surface probability analysis.</I><BR>



At the portion of the sequence where proteins are supposed to
cross the lipid bilayer membrane, the hydrophobicity of that segment is usually
increased relative to the cytosolic and luminal portions. The procedure
developed by

<a href="/compchem.shtml#19">Kyte and Doolittle</A>

uses this phenomenon to identify transmembrane domains in protein sequences
with unknown structure. In the case of GPCRs, these transmembrane domains
are thought to be alpha helical, and more importantly amphiphilic. The
amphiphilicity of the TMs supposedly reflects the way a GPCR is built out of
seven of these helical transmembrane domains. Consequentially, the
hydrophobicity profile of these GPCRs is not as clear as one would wish.

<P>
An example of this effect is the poor separation of the sixth (VI) and
seventh (VII) TM in both the P2Y<sub>1</sub> receptor profile (below) and
the bacteriorhodopsin profile (not shown).

<P>

<CENTER>
<IMG SRC="images/P2Y-KD+SP.gif"><BR CLEAR=ALL>
</CENTER>

<P>
To facilitate identification of TMs, the Kyte-Doolittle hydrophobicity method
is often supplemented with methods describing other sequence dependent vectorial
parameters, such as the 'conservation moment' and
'hydrophilic moment',<sup>

<a href="/compchem.shtml#36">36</A>

</sup> for each helix or other computational methods<sup>

<a href="/compchem.shtml#43">43</A>

</sup>. Use of the 'conservation moment' method requires a particularly well
defined alignment of highly homologous sequences, and can not be applied to the
P<sub>2</sub> GPCRs because of the low sequence similarity of these receptors
with any other GPCR subfamily. The 'hydrophilic moment' method relies on a
database of partial hydrophilic factors for any given amino acid and is
dependent on the method with which these factors were measured. Furthermore,
the high incidence of basic residues in the P<sub>2Y</sub> receptor sequence
greatly influences the results obtained with this method.<BR>

<P>

In contrast, the <A NAME="SP">Emini surface probability</A>

 can be calculated from a given
sequence, indicating the propensity of a stretch of amino acids (the window)
to be at the surface of a protein. The combination of an increase in the

<a href="/compchem.shtml#KD">Kyte-Doolittle hydrophobicity index</A>

and a decrease in the Emini surface probability index was successfully applied
to predict the TMs in the P2Y<sub>1</sub> receptor (see figure above) and in
the reference protein bacteriorhodopsin. The start and end residue of each
helix was usually predicted within 3 residues of the TMs used by

<a href="/compchem.shtml#29">Henderson et al</A>.

The only major deviation in the P2Y<sub>1</sub> receptor prediction occurred
at the N-terminus of TM7 (VII). This particular sequence contains 3
hydrophilic residues (K, Q and R) that are possibly involved in ligand
binding, thus delaying the start of the predicted TM.<BR>

<P>
<I>Building the receptor model.</I><BR>
Modelling of

<A HREF="/GPCR.shtml">G protein-coupled receptors</A>

has become an important tool in understanding drug-receptor interactions
and in the development of new ligands for these receptors.<BR>

<P>
The first widely accepted method was the homology modeling method
by

<a href="/compchem.shtml#27">Hibert et al</A>.

This method involved the alignment of the receptor sequence with the sequence
of bacteriorhodopsin, and the subsequent mapping of the sequence onto the
structure of bacteriorhodopsin that was determined by

<a href="/compchem.shtml#29">Henderson et al</A>.<BR>

<P>
Bacteriorhodopsin is a proton pump in the outer membrane of <I>Halobacterium
halobium</I>, and lacks any functional or sequence homology with GPCRs.
Nevertheless, the procedure was based on the assumption that there would be
considerable structural homology. This structural homology was inferred by
the extraordinary similarity in the hydrophobicity plots, or

<a href="/compchem.shtml#KD">Kyte-Doolittle plots</A>,

of the biogenic amine subfamily of GPCRs and bacteriorhodopsin.<BR>

<P>
Recently, a low resolution electron density map of rhodopsin, a true member
of the GPCR superfamily, was published.<sup>

<a href="/compchem.shtml#28">28</A>

</sup> The low sequence homology with bacteriorhodopsin, the structural
differences that must arise from the different placement of proline residues
(causing bends in helices) in bacteriorhodopsin- and GPCR-sequences, and the
availability of an electron density map of a true member of the GPCR
superfamily prompted us to adapt a new method to build models of
GPCRs.<sup>

<a href="/compchem.shtml#34">34</A>

</sup> This novel method is based on a computational approach rather than strict
compliance with the atomic coordinates of a distantly related protein,
albeit with higher resolution.<BR>

<P>
To ascertain the viability of modelling transmembrane proteins (based on the
structure of rhodopsin<sup>

<a href="/compchem.shtml#34">34</A>

</sup>) by the methods described in the paper, we built models of
bacteriorhodopsin (data not shown) based on the electron density map of
bacteriorhodopsin,<sup>

<a href="/compchem.shtml#29">29</A>

</sup> and rhodopsin (data not shown) based on the electron density map of
bovine rhodopsin as published by

<a href="/compchem.shtml#28">Schertler et al</A>.<BR>

<P>
Root mean square (r.m.s.) distance calculations on superimposed structures
were performed to establish how well the models fitted the experimental data.
The bacteriorhodopsin model that was constructed, compared with the one
deposited in the PDB,<sup>

<a href="/compchem.shtml#29">29</A>

</sup> had an r.m.s. value of 2.15 &Aring; when measured on all backbone atoms
and 2.06 &Aring; on all C<sup>alpha</sup> atoms. Both values are lower than
the resolution of the model, i.e. 3.5 &Aring;. When the rhodopsin model was
compared with the bacteriorhodopsin model by

<a href="/compchem.shtml#29">Henderson et al</A>,

however, the r.m.s. value increased to 16.96 &Aring; when measured on all
backbone atoms and 16.86 &Aring; on all C<sup>alpha</sup> atoms. These
rather high values indicate that there are more structural differences between
bacteriorhodopsin and rhodopsin than have been assumed thus
far.<sup>

<a href="/compchem.shtml#44">44</A>

</sup><BR>

<P>
In our opinion, the position of the proline residues in the helices is a
major determinant in this matter. As can be seen from the

<a href="/extended.shtml">alignment for the helices</A>,

proline residues occur less frequently and at different positions in the
sequence of bacteriorhodopsin than in the sequence of rhodopsin. The
influence of proline residues in alpha-helices was extensively studied by

<a href="/compchem.shtml#37">Sankararamakrishnan and Vishveshwara</A> and

<a href="/compchem.shtml#45">von Heijne</A>.<BR>

<P>
We have found that applying the Amber forcefield to our calculations yields
results in agreement with

<a href="/compchem.shtml#37">Sankararamakrishnan and Vishveshwara</A> and

<a href="/compchem.shtml#45">von Heijne</A>,

and that those results are consistent with data
obtained from a crystallographic study of the transmembrane protein
photosynthetic reaction center (PDB:1prc), and globular proteins such as
phosphoglycerate kinase (PDB:3pgk), lysozyme (PDB:1l27) and alcohol
dehydrogenase (PDB:5adh). The differences between the structures derived
from the electron density maps of bacteriorhodopsin and rhodopsin also
illustrate why helical wheel models, widely used by molecular biologists,
are highly imprecise when applied to GPCRs.<sup>

<a href="/compchem.shtml#31">31</A>

</sup><BR>

<P>
The general picture looks like this:

<P>
<IMG SRC="images/compchem70.gif"><BR CLEAR=ALL>

<P>
For a detailed view of either our <BR>
<UL>
<LI>

<a href="/adenosine.shtml">A<sub>2A</sub> adenosine receptor model</A>,

or our <BR>
<LI>

<a href="/ATP.shtml">P2Y<sub>1</sub> purinoceptor model</A>, <BR>

just follow the appropriate links.
</UL>

<P>

<HR>

<B>REFERENCES.</B><BR>

<P>
<DIR>

<A NAME="1">

<LI>1.</A>	Hoyle, C.H.V. and Burnstock, G. (1991) ATP receptors
and their physiological roles. In
Adenosine in the nervous system. Stone, T.W., Ed., pp. 43-76. London: Academic
Press
Ltd.<BR>

<A NAME="2">

<LI>2.</A>	Hoyle, C.H.V. (1992) Transmission: purines. In
Autonomic neuroeffector mechanisms.
Burnstock, G. and Hoyle, C.H.V., Eds., pp. 367-407. Chur: Harwood Academic
Press.<BR>

<A NAME="3">

<LI>3.</A>	Edwards, F.A., Gibb, A.J., Colquhoun, D. (1992) ATP
receptor mediated synaptic
currents in the central nervous system. Nature , 359, 144-147.<BR>

<A NAME="4">

<LI>4.</A>	Burnstock, G. (1978) A basis for distinguishing two
types of purinergic receptor. In Cell
Membrane Receptors for Drugs and Hormones: a Multidisciplinary Approach.
Straub,
P.W. and Bolis, L., Eds., pp. 106-118. New York: Raven Press.<BR>

<A NAME="5">

<LI>5.</A>	Cusack, N.J. (1993) P2 receptors: subclassification
and structure-activity relationships.
Drug Dev. Res. 28, 244-252.<BR>

<A NAME="6">

<LI>6.</A>	Gordon, J.L. (1986) Extracellular ATP: effects,
sources and fate. J. Biochem. 233, 309-
319.<BR>

<A NAME="7">

<LI>7.</A>	Fredholm, B.B., Abbracchio, M.P., Burnstock, G.,
Daly, J.W., Harden, T.K.,
Jacobson, K.A., Leff, P., Williams, M. (1994) Nomenclature and classification
of
purinoceptors. Pharmacol. Rev. 46, 143-156.<BR>

<A NAME="8">

<LI>8.</A>	Abbracchio, M.P., Burnstock, G. (1994) Purinoceptors:
are there families of P2X and
P2Y purinoceptors? Pharmac. Ther. 64, 445-475.<BR>
<A NAME="9">

<LI>9.</A>	Dubyak, G.R. (1991) Signal transduction by

P2-purinergic receptors for extracellular ATP.
Am. J. Respir. Cell. Mol. Biol. 4, 295-300.<BR>
<A NAME="10">

<LI>10.</A>	Harden, T.K., Hawkins, P.T., Stephens, L., Boyer,

J.L., Downes, P. (1988)
Phosphoinositide hydrolysis by guanosine 5'-([g-thio]triphosphate)-activated
phospholipase C of turkey erythrocyte membranes. Biochem. J. 252, 583-593.<BR>

<A NAME="11">

<LI>11.</A>	Pirotton, S., Boeynaems, J.M. (1991) Transduction
mechanism of P2 purinergic receptors:
Role of phospholipase C and calcium. Nucleos. Nucleot. 10, 1009-1017.<BR>

<A NAME="12">

<LI>12.</A>	Benham, C.D., Tsien, R.W. (1987) A novel
receptor-operated Ca2+-permeable channel
activated by ATP in smooth muscle. Nature 328, 275-278.<BR>

<A NAME="13">

<LI>13.</A>	Bean, B.P. (1992) Pharmacology and electrophysiology
of ATP activated ion channels.
Trends Pharmacol. Sci. 13, 87-90.<BR>

<A NAME="14">

<LI>14.</A>	Fischer, B., Boyer, J.L., Hoyle, C.H.V., Ziganshin,
A.U., Brizzolara, A.L., Knight,
G.E., Zimmet, J., Burnstock, G., Harden, T.K., Jacobson, K.A. (1994)
Identification of
potent, selective P2Y-purinoceptor agonist structure-activity relationship
for 2-thioether
derivatives of adenosine 5'-triphosphate. J. Med. Chem. 36, 3937-3946.<BR>

<A NAME="15">

<LI>15.</A>	Burnstock, G., Fischer, B., Hoyle, C.H.V.,
Maillard, M., Ziganshin, A.U., Brizzolara,
A.L., von Isakovics, A., Boyer, J.L., Harden, T.K., Jacobson, K.A. (1994)
Structure
activity relationship for derivatives of adenosine 5'-triphosphate as
agonists at P2
purinoceptors: heterogeneity within P2X and P2Y subtypes. Drug Dev. Res.
31, 206-219.<BR>

<A NAME="16">

<LI>16.</A>	Windscheif, U., Ralevic, V., B&auml;umert, H.G.,
Mutschler, E., Lambrecht, G., Burnstock,
G. (1994) Vasoconstrictor and vasodilator responses to various agonists in
the rat perfused
mesenteric arterial bed - Selective-inhibition by PPADS of contractions
mediated via P2X-
purinoceptors. Br. J. Pharmacol. 113, 1015-1021.<BR>

<A NAME="17">

<LI>17.</A>	Humphries, R.G., Tomlinson, W., Ingall, A.H., Cage,
P.A., Leff, P. (1994)
FPL-66096 - A novel, highly potent and selective antagonist at human platelet
P2T-
purinoceptors. Br. J. Pharmacol. 113, 1057-1063.<BR>

<A NAME="18">

<LI>18.</A>	van Rhee, A.M., van der Heijden, M.P.A., Beukers,
M.W., IJzerman, A.P.,
Soudijn, W., Nickel, P. (1994) Novel competitive antagonists for P2
purinoceptors. Eur. J. Pharmacol. Mol. Pharmacol. Sect. 268, 1-7.<BR>

<A NAME="19">

<LI>19.</A>	Kyte, J., Doolittle, R.F. (1982) A simple method
for displaying the hydrophobic character
of a protein. J. Mol. Biol. 157, 105-132.<BR>

<A NAME="20">

<LI>20.</A>	Kubo, T., Fukuda, K., Mikama, A., Maeda, A.,
Takahashi, H., Mishina, M., Haga, Y.,
Ichiyama, A., Kangawa, K., Kajima, M., Matsuo, H. Hirose, T., Numa, T.
(1986)
Cloning, sequencing and expressing of complementary DNA coding the muscarinic
acetylcholine receptor. Nature 323, 411-416.<BR>

<A NAME="21">

<LI>21.</A>	Webb, T.E., Simon, J., Krishek, B.J., Bateson, A.N.,
Smart, T.G., King, B.F.,
Burnstock, G., Barnard, E.A. (1993) Cloning and functional expression of a
brain G-
protein-coupled ATP receptor. FEBS Lett. 324, 219-225.<BR>

<A NAME="22">

<LI>22.</A>	Filtz, T.M., Li, Q., Boyer, J.L., Nicholas, R.A.,
Harden, T.K. (1994) Expression of a
P2Y purinergic receptor that couples to phospholipase C. Mol. Pharmacol.
40, 8-14.<BR>

<A NAME="23">

<LI>23.</A>	Lustig, K.D., Shiau, A.K., Brake, A.J., Julius,
D. (1993) Expression cloning of an ATP
receptor from mouse neuroblastoma cells. Proc. Natl. Acad. Sci. USA 90,
5113-5117.<BR>

<A NAME="24">

<LI>24.</A>	Parr, C.E., Sullivan, D.M., Paradiso, A.M.,
Lazarowski, E.R., Burch, L.H., Olsen,
J.C., Erb, L., Weisman, G.A., Boucher, R.C., Turner, J.T. (1994)
Cloning and
expression of a human P2U nucleotide receptor, a target for cystic fibrosis
pharmacotherapy. Proc. Natl. Acad. Sci. USA 91, 3275-3279.<BR>

<A NAME="25">

<LI>25.</A>	Rice, W.R., Burton, F.M., Fiedeldey, D.T. (1995)
Cloning and expression of the alveolar
type II cell P2U-purinergic receptor. Am. J. Respir. Cell Mol. Biol. 12,
27-32.<BR>

<A NAME="26">

<LI>26.</A>	Jacobson, K.A., Stiles, G.L., Ji, X.D. (1992)
Chemical modification and irreversible
inhibition of striatal A2a adenosine receptors. Mol. Pharmacol. 42,
123-133.<BR>

<A NAME="27">

<LI>27.</A>	Hibert, M.F., Trumpp-Kallmeyer, S., Bruinvels,
A., Hoflack, J. (1991) Three-
dimensional models of neurotransmitter G-binding protein-coupled receptors.
Mol.
Pharmacol. 40, 8-15.<BR>

<A NAME="28">

<LI>28.</A>	Schertler, G.F., Villa, C., Henderson, R. (1993)
Projection structure of rhodopsin. Nature
362, 770-772.<BR>

<A NAME="29">

<LI>29.</A>	Henderson, R., Baldwin, J.M., Ceska, T.A., Zemlin,
F., Beckmann, E., Downing, K.H.
(1990) Model for the structure of bacteriorhodopsin based on high-resolution
electron cry-
microscopy. J. Mol. Biol. 213, 899-929.<BR>

<A NAME="30">

<LI>30.</A>	Oliveira, L., Paiva, A.C.M., Vriend, G. (1993) A
common motif in G-protein-coupled
seven transmembrane helix receptors. J. Comp.-Aided Mol. Design 7, 649-658.<BR>

<A NAME="31">

<LI>31.</A>	Baldwin, J.M. (1993) The probable arrangement of
the helices in G protein-coupled
receptors., EMBO J. 12, 1693-1703.<BR>

<A NAME="32">

<LI>32.</A>	Kov&aacute;cs, T. and &Ouml;tv&ouml;s, L. (1988)
Simple synthesis of 5-vinyl and 5-ethynyl-2'-
deoxyuridine-5'-triphosphates. Tetrahedron Lett. 29, 4525-4528.<BR>

<A NAME="33">

<LI>33.</A>	Moffat, J.G. (1964) A general synthesis of
nucleoside-5'-triphosphates. Can. J. Chem.
42, 599.<BR>

<A NAME="34">

<LI>34.</A>	Ballesteros, J.A., Weinstein, H. (1995) Integrated
methods for the construction of three
dimensional models and computational probing of structure-function
relations in G-protein
coupled receptors. Methods Neurosci. 25, 366-428.<BR>

<A NAME="35">

<LI>35.</A>	IJzerman, A.P., van Galen, P.J.M., Jacobson,
K.A. (1992) Molecular modeling
of adenosine receptors. I. The ligand binding site on the A1 receptor.
Drug Des.
Discov. 9, 49-67.<BR>

<A NAME="36">

<LI>36.</A>	Zhang, D., Weinstein, H. (1994) Polarity conserved
positions in transmembrane domains
of G-protein coupled receptors and bacteriorhodopsin. FEBS Lett. 337,
207-212.<BR>

<A NAME="37">

<LI>37.</A>	Sankararamakrishnan, R., Vishveshwara, S. (1992)
Geometry of proline-containing alpha-
helices in proteins. Int. J. Peptide Protein Res. 39, 356-363.<BR>

<A NAME="38">

<LI>38.</A>	Noel, J.P., Hamm, H.E., Sigler, P.B. (1993)
The 2.2 &Aring; crystal structure of transducin-a
complexed with GTPgS. Nature 366, 654.<BR>

<A NAME="39">

<LI>39.</A>	Brake, A. J., Wagenbach, M. J., Julius, D. (1994)
New structural motif for ligand-gated
ion channels defined by an ionotropic ATP receptor. Nature 371, 519-523.<BR>

<A NAME="40">

<LI>40.</A>	Valera, S., Hussy, N., Evans, R. J., Adami, N.,
North, R. A., Surprenant, A., Buell, G.
(1994) New class of ligand-gated ion-channel defined by P2X receptor for
extracellular
ATP. Nature 371, 516-519.<BR>

<A NAME="41">

<LI>41.</A>	Erb, L., Garrad, R., Wang, Y., Quinn, T., Turner,
J.T., Weisman, G.A. (1995) Site-
directed mutagenesis of P2U purinoceptors. J. Biol. Chem. 270, 4185-4188.<BR>

<A NAME="42">

<LI>42.</A>	von Heijne, G. and Manoil, C. (1990) Membrane
proteins: from sequence to structure.
Protein Eng. 4, 109-112.<BR>

<A NAME="43">

<LI>43.</A>	Sander, C. and Schneider, R. (1991) Database of
homology-derived protein structures.
Proteins 9, 56-68.<BR>

<A NAME="44">

<LI>44.</A>	Hoflack, J., Trumpp-Kallmeyer, S., Hibert, M. (1994)
Re-evaluation of bacteriorhodopsin
as a model for G protein-coupled receptors. Trends Pharmacol. Sci. 43,
348-350.<BR>

<A NAME="45">

<LI>45.</A>	von Heijne, G. (1991) Proline kinks in
transmembrane alpha-helices. J. Mol. Biol. 218, 499-
503.<BR>

<A NAME="46">

<LI>46.</A>	Wess, J., Gdula, D., Brann, M.R. (1991)
Site-directed mutagenesis of the m3 muscarinic
receptor: identification of a series of threonine and tyrosine residues
involved in agonist but
not antagonist binding. EMBO J. 10, 3729-3734.<BR>

<A NAME="47">

<LI>47.</A>	Suryanarayana, S., Daunt, D.A., von Zastrow, M.,
Kobilka, B.K. (1991) A point
mutation in the seventh hydrophobic domain of the a2 adrenergic receptor
increases its
affinity for a family of beta receptor antagonists. J. Biol. Chem. 266,
15488-15492.<BR>

<A NAME="48">

<LI>48.</A>	Oksenberg, D., Marsters, S.A., O'Dowd, B.F.,
Jin, H., Havlik, S., Peroutka, S.J.,
Ashkenazi, A. (1992) A single amino-acid difference confers major
pharmacological
variation between human and rodent 5-HT1B receptors. Nature  360, 161-163.<BR>

<A NAME="49">

<LI>49.</A>	Fraser, C.M., Wang, C.D., Robinson, D.A., Gocayne,
J.D., Venter, J.C. (1989) Site-
directed mutagenesis of m1 muscarinic acetylcholine receptors: conserved
aspartic acids
play important roles in receptor function. Mol. Pharmacol. 36, 840-847.<BR>

<A NAME="50">

<LI>50.</A>	Elling, C.E., M&oslash;ller Nielsen, S., Schwartz,
T.W. (1995) Conversion of antagonist
binding site to metal-ion site in the tachykinin NK-1 receptor. Nature
374, 74-77.<BR>

<A NAME="51">

<LI>51.</A>	Kim, J., Wess, J., van Rhee, A.M., Sch&ouml;neberg,
T., Jacobson, K.A. (1995) Site-directed
mutagenesis identifies residues involved in ligand recognition in the human
A2a adenosine
receptor. J. Biol. Chem. 270, 13987-13997.<BR>

<A NAME="52">

<LI>52.</A>	Zhou, W., Flanagan, C., Ballesteros, J.A., Konvicka,
K., Davidson, J.S., Weinstein,
H., Millar, R.P., Sealfon, S.C. (1994) A reciprocal mutation supports helix
2 and helix 7
proximity in the gonadotropin-releasing hormone receptor. Mol. Pharmacol. 2,
165-170.<BR>

<A NAME="53">

<LI>53.</A>	Horstman, D.A., Brandon, S., Wilson, A.L., Guyer,
C.A., Cragoe, E.J., Limbird, L.E.
(1990) An aspartate conserved among G-protein receptors confers allosteric
regulation of
a2-adrenergic receptors by sodium. J. Biol. Chem. 265, 21590-21595.<BR>

<A NAME="54">

<LI>54.</A>	IJzerman, A. P., van der Wenden, E. M., van Galen,
P.J.M., Jacobson, K. A. (1994)
Molecular modeling of adenosine receptors - the ligand-binding site on the
rat adenosine
A2a receptor. Eur. J. Pharmacol. Mol. Pharmacol. Sect. 268, 95-104.<BR>


<LI>55.Jacobson, K.A., Fischer, B., van Rhee, A.M. (1995)
Molecular probes for muscarinic
receptors: functionalized congeners of selective muscarinic antagonists.
Life Sci. 56, 823-
830.<BR>

</DIR>

<!-- end content -->

</td></tr></table>
</td></tr>
<tr><td colspan="2">
<!-- start footer -->

	<!--  footer row -->
<Script language="javascript">
if (!document.layers)
document.write('<div id="divStayTopLeft" style="position:absolute">')
</Script>

<layer id="divStayTopLeft">

<table border="1" width="160" cellspacing="0" cellpadding="0">

  <tr>

 <td width="100%" bgcolor="RGB(255,0,0)">
<!---      <p align="center"><b><font size="4">Menu</font></b></td>--->
  </tr>
  <tr>
    <td width="100%" bgcolor=white></td>

<tr>
<td width="100%" bgcolor="white"> 
      <p align="left"> <a href="http://mgddk1.niddk.nih.gov/MRS.shtml" STYLE="text-decoration: none"><font color="black">MRS Home Page</font></a><br> 

</td>
</tr>

<tr>
<td width="100%" bgcolor="white">
       <a href="http://mgddk1.niddk.nih.gov/staff.shtml" STYLE="text-decoration: none"><font color="black">Staff</font></a><br>
</td>
</tr>
<tr>
<td width="100%" bgcolor="white">
       <a href="http://mgddk1dev2.niddk.nih.gov/publica.shtml" STYLE="text-decoration: none"><font color="black">Selected Publications</font></a><br>
</td>
</tr>

<tr>
<td width="100%" bgcolor="white">
       <a href="http://mgddk1dev2.niddk.nih.gov/adenosine.shtml" STYLE="text-decoration: none"><font color="black">Adenosine Receptors</font></a><br>
</td>
</tr>
<tr>
<td width="100%" bgcolor="white">
       <a href="http://mgddk1dev2.niddk.nih.gov/ATP.shtml" STYLE="text-decoration: none"><font color="black">P2 Receptors</font></a><br>
</td>
</tr>
<tr>
<td width="100%" bgcolor="white">
       <a href="http://mgddk1dev2.niddk.nih.gov/medchem.shtml" STYLE="text-decoration: none"><font color="black">Medicinal Chemistry</font></a><br>

</td>
</tr>
<tr>
<td width="100%" bgcolor="white">
       <a href="http://mgddk1dev2.niddk.nih.gov/compchem.shtml" STYLE="text-decoration: none"><font color="black">Computational Chemistry</font></a><br>
</td>
</tr>
<tr>
<td width="100%" bgcolor="white">
       <a href="http://mgddk1dev2.niddk.nih.gov/PurineCLub.shtml" STYLE="text-decoration: none"><font color="black">The Purine Club</font></a><br>
</td>
</tr>
<tr>

<td width="100%" bgcolor="white">
       <a href="http://mgddk1dev2.niddk.nih.gov/compchem.shtml" STYLE="text-decoration: none"><font color="black">GPCRs</font></a><br>
</td>
</tr>
<tr>
<td width="100%" bgcolor="white">
       <a href="http://mgddk1dev2.niddk.nih.gov/hotlinks.shtml" STYLE="text-decoration: none"><font color="black">Links</font></a></td>
</td>
</tr>

  </tr>
</table>

<!--END OF EDIT-->

</layer>


<script type="text/javascript">

/*
Floating Menu script-  Roy Whittle (http://www.javascript-fx.com/)
Script featured on/available at http://www.dynamicdrive.com/
This notice must stay intact for use
*/

//Enter "frombottom" or "fromtop"
var verticalpos="fromtop"

if (!document.layers)
document.write('</div>')

function JSFX_FloatTopDiv()
{
	var startX = 3,
	startY = 150;
	var ns = (navigator.appName.indexOf("Netscape") != -1);
	var d = document;
	function ml(id)
	{
		var el=d.getElementById?d.getElementById(id):d.all?d.all[id]:d.layers[id];
		if(d.layers)el.style=el;
		el.sP=function(x,y){this.style.left=x;this.style.top=y;};
		el.x = startX;
		if (verticalpos=="fromtop")
		el.y = startY;
		else{
		el.y = ns ? pageYOffset + innerHeight : document.body.scrollTop + document.body.clientHeight;
		el.y -= startY;
		}
		return el;
	}
	window.stayTopLeft=function()
	{
		if (verticalpos=="fromtop"){
		var pY = ns ? pageYOffset : document.body.scrollTop;
		ftlObj.y += (pY + startY - ftlObj.y)/8;
		}
		else{
		var pY = ns ? pageYOffset + innerHeight : document.body.scrollTop + document.body.clientHeight;
		ftlObj.y += (pY - startY - ftlObj.y)/8;
		}
		ftlObj.sP(ftlObj.x, ftlObj.y);
		setTimeout("stayTopLeft()", 10);
	}
	ftlObj = ml("divStayTopLeft");
	stayTopLeft();
}
JSFX_FloatTopDiv();
</script>

<table width="610" border="0" cellspacing="0" cellpadding="0">

<tr><td bgcolor="#0F6060"><img src="/images/spacer.gif" height="1" vspace="0" hspace="0" alt=""></td></tr>


<tr><td align="left" bgcolor="#E7EFEF"><img src="/images/spacer.gif" height="5" vspace="0" hspace="0" alt=""></td></tr>

<tr><td bgcolor="#0F6060"><img src="/images/spacer.gif" height="1" vspace="0" hspace="0" alt=""></td></tr>

<tr><td bgcolor="#FFFFFF"><img src="/images/spacer.gif" height="5" vspace="0" hspace="0" alt=""></td></tr>

<tr><td>
<table width="750" border="0" cellspacing="0" cellpadding="3">
<TR><TD><font face="Arial, Helvetica, sans-serif" size=1>
<p class="small"><B>Chief: <a href="mailto:kajacobs@helix.nig.gov">Dr. Kenneth Jacobson</a></B><BR>
The Molecular Recognition Section (MRS) is in the <a href="http://www.niddk.nih.gov/intram/branchlb/lbc.htm">Laboratory of Bioorganic Chemistry</a> at the 
<a href="http://www.niddk.nih.gov">National Institute of Diabetes and Digestive and Kidney Diseases</a>
which is part of the <a href="http://www.nih.gov">National Institutes of Health</a>, Bethesda, MD, USA.  General inquiries may be addressed to NIDDK, NIH, Building 31, Room 9A04, 31 Center Drive, MSC 2560, Bethesda, MD 20892-2560, USA.  <br>
<a href="http://www.niddk.nih.gov/tools/privacy.htm">Privacy</a> || <a href="/copyright.shtml">Copyright, Disclaimers & Access Restrictions</a> ||  <a href="http://www.niddk.nih.gov/tools/accessibility.htm">Accessibility</a>  <BR>
</p></TD></TR></TABLE>
</font>
</TD></TR>

<tr><td bgcolor="#FFFFFF"><img src="/images/spacer.gif" height="5" vspace="0" hspace="0" alt=""></td></tr>

<tr><td bgcolor="#0F6060"><img src="/images/spacer.gif" height="1" vspace="0" hspace="0" alt=""></td></tr>

<tr><td bgcolor="#FFFFFF"><img src="/images/spacer.gif" height="3" vspace="0" hspace="0" alt=""></td></tr>

<TR><TD>
<table width="750" border="0" cellspacing="0" cellpadding="0">
<TR><td align="center"><a href="http://www.niddk.nih.gov/"><img src="../../images/niddk-small.gif" alt="National Institute of Diabetes, Digestive & Kidney Diseases" border="0"></a></td><TD align="center">
<a href="http://www.nih.gov"><img src="../../images/nih-bottom.gif" border="0" height="45" width="225" Alt="U. S. National Institutes of Health"></a></TD>
<TD align="center"><a href="http://www.hhs.gov"><img src="../../images/dhhs-bottom.gif" height="45" width="172" border="0" ALT="U.S. Department of Health &amp; Human Services"></a></TD>
<TD align="center"><a href="http://www.usa.gov"><img src="../../images/usagov.gif" border="0"  ALT="USA.gov Logo - link to the U.S. government's official web portal"></a></TD></TR></TABLE>

</TD></TR>


</TD>
</tr></table>



<!--end footer-->


<td width="100%" valign="top"><img src="/images/spacer.gif" width="100%" height="1" alt="">

</td>
</tr></table>

<map name="header2">
<area shape="rect" alt="National Institutes of Health" coords="6,59,206,79" href="http://www.nih.gov" title="National Institutes of Health">
</map>

<!-- end  footer row, NDF USAgov img last update 1-19-2007 -->

</td></tr>
</table>

</BODY>
</HTML>