<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="BMC Evolutionary Biology"><meta name="citation_title" content="Evaluation of the models handling heterotachy in phylogenetic inference"><meta name="citation_authors" content="Yan Zhou, Nicolas Rodrigue, Nicolas Lartillot, and  Herv&#x000e9; Philippe"><meta name="citation_date" content="2007"><meta name="citation_volume" content="7"><meta name="citation_firstpage" content="206"><meta name="citation_doi" content="10.1186/1471-2148-7-206"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=2248194&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2248194&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2248194"><meta name="citation_pmid" content="17974035"><meta name="DC.Title" content="Evaluation of the models handling heterotachy in phylogenetic inference"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="BioMed Central"><meta name="DC.Contributor" content="Yan Zhou"><meta name="DC.Contributor" content="Nicolas Rodrigue"><meta name="DC.Contributor" content="Nicolas Lartillot"><meta name="DC.Contributor" content="Herv&#x000e9; Philippe"><meta name="DC.Date" content="2007"><meta name="DC.Identifier" content="10.1186/1471-2148-7-206"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody4.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs3.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>Evaluation of the models handling heterotachy in phylogenetic inference</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0-MEWN7ymFK-OvO6PH2GdIdi72VdNyHfHUXsjKAKE&amp;reftype=other&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-bmceb.gif" alt="Logo of bmceb" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="BioMed Central web site" coords="0,0,99,74" onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0tyFN_BxKJkf3zoeljymAeSzh6kBISZJpATvbJQMm&amp;reftype=publisher&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com" target="pmc_ext"><area shape="rect" coords="100,54,172,74" alt="Reference to the article." onclick="focuswin('pmc_ext') " href="redirect3.cgi?&amp;&amp;auth=0t5P8dbbNL-jj7jWwDJPmSuwMaK1hmmBySA6kX5nI&amp;reftype=publisher&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7CArticle%7CFullText&amp;rendering-type=normal&amp;&amp;http://dx.doi.org/10.1186/1471-2148-7-206" target="pmc_ext"><area shape="rect" coords="173,54,223,74" alt="Search." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=04uv1eTUKmvHABqs3SeKhQsF3wvVW5fyLnGA_1Amg&amp;reftype=publisher&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/search/bmcevolbiol" target="pmc_ext"><area shape="rect" coords="224,55,342,74" alt="Manuscript submission." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0HiePqtkt8K0tV5Vy-dS9hl9NQgD_PqIzs8ivNkHv&amp;reftype=publisher&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/manuscript/bmcevolbiol" target="pmc_ext"><area shape="rect" coords="343,53,480,74" alt="Registration." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0D38qZ1rzxUYiWBs7H0OUYki0H9rVQLYt-AnF_5ef&amp;reftype=publisher&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/registration" target="pmc_ext"><area shape="rect" coords="0,0,480,74" alt="Journal front page." onclick="focuswin('pmc_ext') " href="redirect3.cgi?&amp;&amp;auth=0XolcAZQSwkhlzrFlqhenCeSkXPNXicDmUdrTx5ts&amp;reftype=publisher&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/bmcevolbiol" target="pmc_ext"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=28&amp;action=archive">BMC Evol Biol</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=141646">v.7; 2007</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="#supplementary-material-sec" class="sidefm-supplink">Supplementary Material</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=2248194&amp;blobtype=pdf" class="sidefm-pmclink">PDF (711K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=141646" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=28&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=0hg7tzUVy781oVDY-86n-bdAAXPC2Hl9fwaGT4Axq&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17974035">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=0lwwa_f4Lh140V4JwwhcvGr33wrq2ywOBnfX5eQP6&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=17974035">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0DY7Af6sLfQRA9_M95JXOXAdimbIWfMLAS-yMYk_L&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=17974035">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0Rypz3eYR9XPFKmbRX2Js-IPClVOofJHl1N7_cA_j&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=2248194&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0tSZa-CesVXzkxeu8uh1u9CWjVPeo_E-St2d_W54w&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=2248194">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=04kL9FNT3c3eQtYTPVyQNxCedv-Vpm5bUXVggHA-w&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Zhou+Y%5Bauth%5D" class="sidefm-pmclink"> Zhou, Y.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0PayxquNkEp6sKKh6YxYYNkP74Er5FSgl4RnCX55m&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Rodrigue+N%5Bauth%5D" class="sidefm-pmclink"> Rodrigue, N.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0HmP0sqoESCvrds7j2e0wUG4xLRfCR4hq3-AR_Qc1&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Lartillot+N%5Bauth%5D" class="sidefm-pmclink"> Lartillot, N.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=08_B2gVcMkEUztksoeK4YuDCz7ukHbG59zP9XNQW5&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Philippe+H%5Bauth%5D" class="sidefm-pmclink"> Philippe, H.</a></div></div></div></td><td rowspan="107" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-bmceb.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">BMC Evol Biol. </span><span class="citation-publication-date">2007; </span><span class="citation-volume">7</span><span class="citation-issue"></span><span class="citation-flpages">: 206. </span></div><div><span class="fm-vol-iss-date">Published online 2007 November 1. </span><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1186/1471-2148-7-206.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC2248194</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 2007 Zhou et al; licensee BioMed Central Ltd.</div><div class="fm-title">Evaluation of the models handling heterotachy in phylogenetic inference</div><div class="fm-author contrib-group">Yan Zhou,<sup>1</sup> Nicolas Rodrigue,<sup>1</sup> Nicolas Lartillot,<sup>2</sup> and  Herv&#x000e9; Philippe<sup><img src="corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1</sup><div class="fm-affl"><sup>1</sup>Canadian Institute for Advanced Research. D&#x000e9;partement de Biochimie, Universit&#x000e9; de Montr&#x000e9;al, Succursale Centre-Ville, Montr&#x000e9;al, Qu&#x000e9;bec H3C3J7, Canada</div><div class="fm-affl"><sup>2</sup>Laboratoire d'Informatique, de Robotique et de Micro&#x000e9;lectronique de Montpellier. CNRS &#x02013; Universit&#x000e9; de Montpellier 2. 161, rue Ada, 34392 Montpellier Cedex 5, France</div></div><div class="fm-footnote"><sup><img src="corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div class="fm-footnote"><span style="white-space: nowrap;" id="id476906">Yan Zhou: <span class="e_id476906">y.zhou/at/umontreal.ca</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id476906", '<a class="ext-reflink" href="' + reverseAndReplaceString('ac.laertnomu/ta/uohz.y:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ac.laertnomu/ta/uohz.y', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id424926">Nicolas Rodrigue: <span class="e_id424926">nicolas.rodrigue/at/umontreal.ca</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id424926", '<a class="ext-reflink" href="' + reverseAndReplaceString('ac.laertnomu/ta/eugirdor.salocin:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ac.laertnomu/ta/eugirdor.salocin', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id424953">Nicolas Lartillot: <span class="e_id424953">nicolas.lartillot/at/lirmm.fr</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id424953", '<a class="ext-reflink" href="' + reverseAndReplaceString('rf.mmril/ta/tollitral.salocin:otliam', '/at/', '@') + '">' + reverseAndReplaceString('rf.mmril/ta/tollitral.salocin', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id315559">Herv&#x000e9; Philippe: <span class="e_id315559">herve.philippe/at/umontreal.ca</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id315559", '<a class="ext-reflink" href="' + reverseAndReplaceString('ac.laertnomu/ta/eppilihp.evreh:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ac.laertnomu/ta/eppilihp.evreh', '/at/','@') + '</a>')}catch(e){}
                                //--></script></span> </div><div class="fm-pubdate">Received April 20, 2007; Accepted November 1, 2007.</div><div class="fm-copyright"><p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0xYA8SeMxxEBrRvujvfUGPwJKUHnydeO5QjxIf5zA&amp;reftype=extlink&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://creativecommons.org/licenses/by/2.0" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://creativecommons.org/licenses/by/2.0</a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639428" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id590721" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642382" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642822" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642882" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644716" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644726" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644837" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id639369">Abstract</div><div class="section-content"><!--article-meta--><p><div class="abs-head2 head-separate">Background</div>The evolutionary rate at a given homologous position varies across time. When sufficiently pronounced, this phenomenon &#x02013; called heterotachy &#x02013; may produce artefactual phylogenetic reconstructions under the commonly used models of sequence evolution. These observations have motivated the development of models that explicitly recognize heterotachy, with research directions proposed along two main axes: 1) the <em>covarion </em>approach, where sites switch from variable to invariable states; and 2) the <em>mixture of branch lengths </em>(MBL) approach, where alignment patterns are assumed to arise from one of several sets of branch lengths, under a given phylogeny.</p><p><div class="abs-head2 head-separate">Results</div>Here, we report the first statistical comparisons contrasting the performance of covarion and MBL modeling strategies. Using simulations under heterotachous conditions, we explore the properties of three model comparison methods: the Akaike information criterion, the Bayesian information criterion, and cross validation. Although more time consuming, cross validation appears more reliable than AIC and BIC as it directly measures the predictive power of a model on 'future' data. We also analyze three large datasets (nuclear proteins of animals, mitochondrial proteins of mammals, and plastid proteins of plants), and find the optimal number of components of the MBL model to be two for all datasets, indicating that this model is preferred over the standard homogeneous model. However, the covarion model is always favored over the optimal MBL model.</p><p><div class="abs-head2 head-separate">Conclusion</div>We demonstrated, using three large datasets, that the covarion model is more efficient at handling heterotachy than the MBL model. This is probably due to the fact that the MBL model requires a serious increase in the number of parameters, as compared to two supplementary parameters of the covarion approach. Further improvements of the both the mixture and the covarion approaches might be obtained by modeling heterogeneous behavior both along time and across sites.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639369" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Background</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id590721" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642382" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642822" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642882" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644716" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644726" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644837" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id639428">Background</div><div class="section-content"><p>Probabilistic methods for phylogenetic inference are based on mathematical models of sequence evolution [<a href="#B1" class="cite-reflink bibr">1</a>]. In the last 20 years, several approaches have been proposed for developing more sophisticated models, accounting for various properties of substitution processes [<a href="#B2" class="cite-reflink bibr">2</a>-<a href="#B8" class="cite-reflink bibr">8</a>]. One of the most well-characterized example of such an improvement is provided by the Rate Across Sites (RAS) model [<a href="#B2" class="cite-reflink bibr">2</a>], which relaxes the assumption that all sites of a protein or a nucleotide sequence evolve at the same rate. More specifically, the RAS model includes site-specific substitution rates, modeled as random variables following a gamma distribution. It generally has a better fit to the data, and it allows to circumvent certain artefacts in phylogenetic inference [<a href="#B9" class="cite-reflink bibr">9</a>]. It has been implemented in most maximum-likelihood and Bayesian phylogenetic software, and is now widely used for routine phylogenetic inference. More sophisticated distributions of substitution rates, such as mixtures of gamma distributions [<a href="#B10" class="cite-reflink bibr">10</a>], further increase the fit of the model to alignments, suggesting that improvements of the RAS model are still possible.</p><p>Functional and structural restrictions operating at a given residue may be subject to change over time [<a href="#B11" class="cite-reflink bibr">11</a>,<a href="#B12" class="cite-reflink bibr">12</a>], which should be reflected by substitution rates varying not only across sites, but also across time. In this line of thought, Fitch and Markowitz [<a href="#B13" class="cite-reflink bibr">13</a>] proposed the covarion hypothesis: due to functional restrictions, some codons (the <em>co</em>ncomitantly <em>vari</em>able cod<em>ons </em>or covarions) can accept substitutions at a given time, while others (invariant sites) cannot. Importantly a site can shift from being variable to being invariable (and vice versa) over time. More generally, Philippe and Lopez [<a href="#B14" class="cite-reflink bibr">14</a>] proposed, instead of covarion-like expression, the term heterotachy (from Greek, meaning "different speed") to describe the fact that sites evolve at different rates across time.</p><p>Heterotachy was shown to be frequent in both nucleotide and amino acid sequences [<a href="#B6" class="cite-reflink bibr">6</a>,<a href="#B15" class="cite-reflink bibr">15</a>-<a href="#B22" class="cite-reflink bibr">22</a>]. For instance, up to 95% of the variable sites of cytochrome b have a heterotachous behavior within vertebrates [<a href="#B23" class="cite-reflink bibr">23</a>]. Importantly, both simulation [<a href="#B24" class="cite-reflink bibr">24</a>,<a href="#B25" class="cite-reflink bibr">25</a>] and empirical [<a href="#B26" class="cite-reflink bibr">26</a>,<a href="#B22" class="cite-reflink bibr">22</a>,<a href="#B16" class="cite-reflink bibr">16</a>,<a href="#B28" class="cite-reflink bibr">28</a>] studies demonstrate that heterotachy may impede phylogenetic inference. This is expected because probabilistic methods are inconsistent when the underlying assumptions of their models are seriously violated. Models that handle heterotachy are thus of prime interest, particularly as larger and larger datasets are used [<a href="#B29" class="cite-reflink bibr">29</a>].</p><p>The initial covarion hypothesis, as formulated by [<a href="#B13" class="cite-reflink bibr">13</a>], makes an explicit link between site interdependencies and rate shifts, and for that reason, is not easy to implement. As a more tractable alternative, Tuffley and Steel [<a href="#B30" class="cite-reflink bibr">30</a>] proposed a site-independent mathematical version of the covarion idea, which was later implemented in a Bayesian framework [<a href="#B6" class="cite-reflink bibr">6</a>]. In Tuffley and Steel's covarion model, the substitution history at each site unfolds according to a doubly stochastic process: a classical first-order Markov process of substitution among the 4 nucleotide bases, or the 20 amino-acids, whose substitution rate is itself time-modulated in an on-off fashion. In Huelsenbeck's model, evolutionary rates of sites, when in the on state, are modeled by a gamma distribution. Galtier [<a href="#B5" class="cite-reflink bibr">5</a>] proposed a variant of this model, by merging the covarion-like random effects with the site-specific random-effects introduced by the RAS model: sites can take more than two rates ("on" and "off"), i.e. the off category plus, e.g., the four rates of a discretized gamma distribution. More recently, Wang et al. [<a href="#B31" class="cite-reflink bibr">31</a>] propose a more general model in which evolutionary rates can switch among different rate classes when they are in a variable state.</p><p>One merit of Tuffley and Steel's version of the covarion model is that it aims at capturing the dynamic heterotachous scenario by using only two additional global stationary parameters: s<sub>01</sub>, the switching rate from the off to the on state, and s<sub>10</sub>, the rate from on to off.  Note that these two parameters are both assumed to be stationary over time. On the other hand, this model assumes that rate-shifts occur in a strictly site-independent fashion, whereas, in principle, it is possible to imagine more general scenarios, in which groups of sites undergo collective rate shifts at very specific time-points, due to a sudden change of the selection pressure (this type of situation is precisely supposed to create the misplacement of microsporidia [<a href="#B28" class="cite-reflink bibr">28</a>,<a href="#B27" class="cite-reflink bibr">27</a>]).</p><p>Recently, Kolaczkowski and Thornton [<a href="#B24" class="cite-reflink bibr">24</a>] proposed a 'mixture of branch lengths' (MBL) model that could handle this kind of collective rate shifting. In this finite mixture model, which was later mathematically corrected [<a href="#B32" class="cite-reflink bibr">32</a>], each observation is assumed to arise from one of several components (the number of components being predefined), each specifying a distinct and independent set of branch lengths, onto the same topology. Loosely speaking, each site can "choose" among the available components that which best describes its pattern of changes along the tree. In practice, as there is no a priori knowledge of which site belongs to which component, the likelihood at each site is a weighted sum over all components [<a href="#B33" class="cite-reflink bibr">33</a>,<a href="#B32" class="cite-reflink bibr">32</a>]. The kind of heterotachy assumed in the MBL model [<a href="#B24" class="cite-reflink bibr">24</a>] can appear artificial at first sight, but is theoretically able to capture collective rate shifts, rather than the purely site independent on-off processes of the covarion model. In principle, the MBL model could thus provide a useful device for detecting singular and collective rate shift events.</p><p>However, the potential gain of the MBL over the covarion model is statistically expensive, because of the serious increase of the number of parameters implied (the number of additional branch lengths, (N<sub>c</sub>-1)*(2s-3), and the weights of the components, N<sub>c</sub>-1): (N<sub>c</sub>-1)*(2s-2), where N<sub>c </sub>is the number of components in the mixture, and s is the number of taxa. The MBL model poses practical challenges as well. For instance, in the Bayesian Markov chain Monte Carlo framework, the complicated structure of a single tree with several valuations (several sets of branch lengths) makes it difficult to propose update mechanisms that would be efficient for mixing in tree space, or, in a reversible-jump perspective, for averaging over the number of components. As a result, jointly estimating the phylogeny and the number of components will be a computational challenge.</p><p>A common statistical practice when facing computational difficulties is to make simplifying assumptions (e.g., a known phylogenetic tree), and to contrast the merit of different model configurations based on their statistical fit. Note that model comparisons based on likelihood ratio tests are not directly applicable here, as the set of models of interest do not all form a nested hierarchy. (Even evaluating the number of components would be difficult, because of the irregular parameter space in the mixture model [<a href="#B34" class="cite-reflink bibr">34</a>,<a href="#B35" class="cite-reflink bibr">35</a>], the logarithm of the likelihood follows a complicated mixture of chi-square distributions [<a href="#B36" class="cite-reflink bibr">36</a>]). An alternative is to use likelihood penalty methods, such as the Bayesian Information Criterion (BIC; [<a href="#B37" class="cite-reflink bibr">37</a>]), or Akaike Information Criterion (AIC; [<a href="#B38" class="cite-reflink bibr">38</a>]). When the number of observations (here aligned sites) is sufficiently large, BIC is asymptotically equivalent to the Bayes factor, and AIC to the expected relative Kullback-Leibler information [<a href="#B38" class="cite-reflink bibr">38</a>] Although easy to compute, these two measures rely on many assumptions to estimate the penalty for the increased number of parameters. Moreover, as for AIC, it further assumes that the models being tested are 'not too far' from the true model [<a href="#B38" class="cite-reflink bibr">38</a>]. In addition, AIC seems to overestimate the number of parameters when there are many parameters compared to the sample size [<a href="#B39" class="cite-reflink bibr">39</a>,<a href="#B40" class="cite-reflink bibr">40</a>]. Contrary to AIC, BIC has a tendency to under-estimation, given sparse data and results [<a href="#B41" class="cite-reflink bibr">41</a>]. Furthermore, in the context of mixture models, the regular assumptions for the AIC and BIC are no longer valid [<a href="#B42" class="cite-reflink bibr">42</a>,<a href="#B43" class="cite-reflink bibr">43</a>]. In any case, Djuric [<a href="#B44" class="cite-reflink bibr">44</a>] argued that the penalty for over-parameterization should strongly depend on the model structure, i.e., the types of unknown model parameters. Although BIC works reasonably well at the practical level [<a href="#B45" class="cite-reflink bibr">45</a>], Djuric [<a href="#B44" class="cite-reflink bibr">44</a>] suggested a careful examination before applying AIC/BIC.</p><p>Another evaluation of model fitness is the cross-validation (CV) method [<a href="#B46" class="cite-reflink bibr">46</a>]: it measures the predictive power of a model fitted to a first, randomly drawn, part of the dataset, when applied to the remaining (set aside) part of the data. Here, the portion of data set aside plays the role of 'future' observations. Accordingly, the best model is naturally the one that best predicts these future data. Compared to AIC and BIC, CV is computationally much more demanding, but also more reliable in principle: (1) this is an operational test, in which one measures the predictive power on data that have not been seen during the learning step, which guarantees the 'honesty' of the measure. In particular, it implies that there is no need to account for a dimensional penalty. (2) the expectation of cross-validated likelihood is an unbiased estimate of the Kullback-Leibler (KL) distance between the "true" distribution of column patterns, and the distribution implied by the model [<a href="#B47" class="cite-reflink bibr">47</a>], and (3) in fairly general settings (not including the leave-one out testing scheme), cross validation is asymptotically consistent, i.e. is able to choose the true model among identifiable alternatives [<a href="#B48" class="cite-reflink bibr">48</a>]. In addition to these theoretical guarantees, there is no specific requirement on the compared models (e.g. nested).</p><p>In this work, we explore the use of AIC, BIC and CV for the comparison of covarion and MBL models. We first validate and examine properties of the MBL model using simulations. Second, we contrast the conclusions of AIC, BIC and CV to the problem of determining the number of components of the MBL model, and to general comparisons with the covarion model. Third, we extend our model comparisons to three real data sets from nuclear, plastid and mitochondrial compartments, and show that the covarion model is always favored over the optimal MBL model.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639369" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639428" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Results</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642382" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642822" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642882" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644716" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644726" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644837" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id590721">Results</div><div class="section-content"><p><div class="head2 head-separate">Simulated data</div>We first implemented the mixture branch length model in the phylobayes package [<a href="#B49" class="cite-reflink bibr">49</a>]. Simulations allowed us to explore the performance of the MBL model when the true number of components as well as other parameters are known. Various levels of heterotachy can be easily obtained by tuning a single parameter, <em>&#x003c4;</em>, without affecting the average branch length (see Methods for details) of the tree topology displayed on Figure <a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1</a>. In addition, the degree of rate variation across sites was modulated by using several values of <em>&#x003b1;</em>, the shape parameter of the gamma distribution. A total of 16 data sets of 5,000 sites each were synthesized under the two-component MBL model and analyzed using the MBL model with number of components varying from one to four.<div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F1" name="F1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2248194&amp;blobname=1471-2148-7-206-1.gif" border="1" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span><strong>Topology used for computer simulations</strong>. The tree under the newick format is: ((((A:0.375, B:0.3):0.25, C:1):0.08, D:0.32):0.8,((E:0.42, F:0.31):0.24,(G:0.27,(H:0.2,(I:0.5, J:0.5):0.25):0.12):0.25):0.26). Scale bar indicates the expected number of changes</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>When the simulated data are analyzed with the exact number of components (two), the inferred values of the parameters are generally close to their true values (Table <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a>). For instance, the value of <em>&#x003b1; </em>is always inferred with an error smaller than 5%. The branch lengths and the weights are also well recovered, although only when the level of heterotachy is pronounced (<em>&#x003c4;</em>&#x0003e;= 0.4, Table <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a>). Interestingly, when weakly heterotachous datasets (<em>&#x003c4; </em>= 0.2) are analyzed under the two-component model, the weight for one of the two components shrinks to almost zero, and the corresponding branch lengths become meaningless, taking on extremely large or small values.</p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T1" name="T1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 1" title="Table 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 1</strong></a><div class="figure-table-caption-in-article"><span>Inferred values of <em>&#x003b1;</em>, the parameter of the discrete gamma distribution of the rates across sites, inferred weight of one of the two components (w) and Pearson correlation (r) of the inferred tree branch lengths with the true ones of their respective</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T1" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><p>Inferring the number of components followed a similar, but more complex, pattern (Table <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a>). When the dataset contains a strong heterotachous signal (<em>&#x003c4; </em>= 0.8), AIC, BIC and CV recover the expected number of components (two). In contrast, as the level of heterotachy gets weaker (<em>&#x003c4; </em>= 0.2), all criteria almost always choose the one-component model. The amount of heterotachous signal is simply insufficient in these 5,000 positions. Interestingly, under these conditions, when the MBL model with two components is used, the weight of one of them tends to be extremely small (Table <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a>), which is consistent with the higher fit of the one-component model. For intermediate level of heterotachy (<em>&#x003c4; </em>= 0.4 and 0.6), AIC supports 2 and 3 components and BIC 1 or 2, suggesting that AIC might tend to overestimate, and BIC might underestimate, the number of components, (Table <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a>). In contrast, in both cases, CV recovers the correct value.</p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T2" name="T2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T2" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 2" title="Table 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 2</strong></a><div class="figure-table-caption-in-article"><span>Optimal numbers of components determined by AIC, BIC or cross-validation (CV) on the simulated data with different levels of heterotachy (<em>&#x003c4;</em>) and with different rate across sites heterogeneity (<em>&#x003b1;</em>).</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>We next extended the comparisons by including the covarion model (Table <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T3" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">3</a>). As expected because sequences were simulated using an MBL model, the covarion model is never favored. However, under a low level of heterotachy (<em>&#x003c4; </em>= 0.2), the covarion model performs slightly better than the two-component model, in spite of the fact that the dataset is indeed a mixture of two components. This could be due to the fact that the covarion model requires less parameters than the 2-components MBL model.</p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T3" name="T3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T3" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 3" title="Table 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 3</strong></a><div class="figure-table-caption-in-article"><span>Cross-validation for the simulated datasets (<em>&#x003b1; </em>= 0.5)</span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><div class="head2 head-separate">Real data</div>When applied to three real datasets from nuclear, mitochondrial and plastid compartments, CV and BIC always supports the covarion model (Table <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T4" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">4</a>), while AIC favors parameter-rich MBL model. In the selection of the optimal number of components of the MBL model, CV always favors the two-component model (Table <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T4" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">4</a>). In contrast, BIC favors one component, except for mitochondrial alignment in which four or six components are virtually indistinguishable (44,416.88 versus 44,416.75), and AIC three or four components.<div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T4" name="T4"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T4" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 4" title="Table 4"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 4</strong></a><div class="figure-table-caption-in-article"><span>Comparison of the covarion model and MBL models with different number of components for three real datasets</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>We also studied the branch lengths of the two partitions detected by the MBL model (mitochondrial, Fig. <a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F2" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">2</a>; nuclear, see Additional File <a href="#S1" class="side-supplink supplementary-material">1</a>; plastid, see Additional File <a href="#S2" class="side-supplink supplementary-material">2</a>). Interestingly, in the case of mitochondrial alignment, the branch lengths of the two partitions mainly differ for catarrhinian primates, i.e. they evolved much faster in component I. To know whether particular genes are involved in this heterotachous behavior, we computed the posterior probability of each site belonging to either component (see Method, formula 9), and then averaged these posterior probabilities over the sites, separately for each gene. The sites belonging to the cytochrome oxidase (cox1-3) and cytochrome b (cytb) genes show a significantly different posterior probability of belonging to component I than the sites from other genes (P &#x0003c; 0.0001, Fig. <a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F3" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">3</a>). A chi-square test was also performed, showing that the two partitioning of the sites, into the cox/cytb or the non-cox/cytb gene groups, and into the 2 components of the model, are not independent (P &#x0003c; 0.001, Table <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T5" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">5</a>). Similarly, for plastid alignment, the two components are biologically relevant. The branch lengths of one component are relatively clock-like whereas for the other one all green plants except <em>Mesostigma </em>showed a highly accelerated rate. Interestingly, RNA polymerases show a significantly higher posterior probability of belonging to component II than the sites from ribosomal proteins (P &#x0003c; 0.0001, see Additional File <a href="#S3" class="side-supplink supplementary-material">3</a>) in agreement with recent studies [<a href="#B50" class="cite-reflink bibr">50</a>,<a href="#B22" class="cite-reflink bibr">22</a>].</p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F2" name="F2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2248194&amp;blobname=1471-2148-7-206-2.gif" border="1" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span><strong>Branch lengths for the two partitions in the case of the mitochondrial alignment of mammals (3591 sites, 17 species)</strong>. The shape parameter of the &#x00393; distribution was estimated to be 0.4. The weights are 0.40 for component I (B) and 0.60 for component</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F3" name="F3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2248194&amp;blobname=1471-2148-7-206-3.gif" border="1" class="icon-reflink" alt="Figure 3" title="Figure 3"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 3</strong></a><div class="figure-table-caption-in-article"><span><strong>Whiskers plot for the average posterior probabilities of component I for the two-component MBL model on the mitochondrial mammal dataset</strong>. A Kruskal-Wallis non-parametric test shows the means of posterior probabilities for genes are significantly different</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F3" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T5" name="T5"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T5" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 5" title="Table 5"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T5" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 5</strong></a><div class="figure-table-caption-in-article"><span>Contingency table for the mitochondrial alignment</span></div></td></tr></table></div></div><div style="clear:both;"></div></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639369" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639428" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id590721" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Discussion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642822" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642882" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644716" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644726" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644837" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id642382">Discussion</div><div class="section-content"><p><div class="head2 head-separate">Model comparisons: CV is more reliable than AIC and BIC</div>The maximum likelihood value is always improved when more parameters are added to the model. The widely used likelihood penalty information criteria, AIC and BIC, evaluate the fitness of models by heuristically adjusting the likelihood score. Based on asymptotic arguments [<a href="#B37" class="cite-reflink bibr">37</a>,<a href="#B38" class="cite-reflink bibr">38</a>], they compensate for the automatic increase of the likelihood merely due to the increase in number of parameters, using simple (and distinct) formulae for the dimensional penalty. By construction, AIC gives a milder dimensional penalty than BIC. In many practical cases, the difference may be overwhelmed by the difference in log-likelihood between the two models. However, in the present case, and on both real and simulated data sets, AIC and BIC do not always reach the same conclusions (Tables <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a> and <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T4" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">4</a>).<p>Cross-validation methods are much more expensive in terms of CPU time than these information criteria. However, they are conceptually more trustworthy, since they consist in a true blind test, i.e. instead of relying on a heuristic dimensional penalty, they measure the predictive power of the model on data that have not been seen during the parameter optimization step. In addition, they are valid even far from the asymptotic regime, i.e. when the number of sites is small. From comparisons among AIC, BIC and CV, we observe that BIC and CV generally agree, while AIC overestimates the fit of parameter-rich models. These observations are consistent with the reports that AIC seems to have an inherent bias in favor of overly parameterized models [<a href="#B51" class="cite-reflink bibr">51</a>-<a href="#B53" class="cite-reflink bibr">53</a>,<a href="#B41" class="cite-reflink bibr">41</a>,<a href="#B39" class="cite-reflink bibr">39</a>,<a href="#B40" class="cite-reflink bibr">40</a>],.</p></p><p><div class="head2 head-separate">Properties of the mixture branch length (MBL) model</div>The MBL model is able to detect heterotachous signals and recover the true number of components, sets of branch lengths, weights for the components, as well as the alpha parameter for the RAS gamma distribution, when datasets are simulated with a strong level of heterotachy (Tables <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T1" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">1</a> and <a href="articlerender.fcgi?artid=2248194&amp;rendertype=table&amp;id=T2" class="fig-table-link " onclick="startTarget(this, 'table', 1024, 800)">2</a>). In contrast, when the level of heterotachy is weak (e.g. <em>&#x003c4; </em>= 0.2) and the alignment size is in the order of magnitude of the currently used ones (5,000 amino acids), the homotachous (one component) model is preferred. This is consistent with the observations that the performance of the homotachous model is weakly affected under weakly heterotachous datasets (<em>&#x003c4; </em>= 0.2), and that it starts to get devastating only when the level of heterotachy gets higher (<em>&#x003c4; </em>= 0.4) [<a href="#B54" class="cite-reflink bibr">54</a>,<a href="#B32" class="cite-reflink bibr">32</a>,<a href="#B56" class="cite-reflink bibr">56</a>,<a href="#B24" class="cite-reflink bibr">24</a>]. It seems therefore that, at least on these simulated cases, when heterotachy does not impair phylogenetic inference, the classical non-mixture model is indeed found to be the optimal by standard model selection methods.<p>Estimating the adequate number of components can be viewed as a limitation of MBL models. On the one hand, we have shown that only the computationally demanding CV is able to provide an accurate estimate of the optimal number. On the other hand, it appears that, when the number of components is too high, the weights of these useless components are small (below 0.05, except for plastid -0.08- and nuclear -0.20- alignments). In other words, the over-parameterized model naturally reduces, but does not abolish, the effect of useless parameters, but is logically penalized in model comparison.</p><p>Interestingly, in the case of mitochondrial and plastid alignments, heterotachy detected by the MBL model is meaningful (Figs. <a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F2" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">2</a> and S2). For instance, the most important heterotachous signal detected by the MBL model on the mitochondrial data set consists in a collective rate-shift, preferentially concerning the positions of cox and cytb gene. This acceleration of the multisubunit respiratory complex cytochrome c oxidase in primates is well documented and co-evolution implies genes encoded in the nucleus and in the mitochondrion [<a href="#B57" class="cite-reflink bibr">57</a>]. Thus, the MBL model seems to be indeed able to detect collective behavior, corresponding to real biological events.</p></p><p><div class="head2 head-separate">How to model heterotachy?</div>However, and in spite of the considerable interest received by the MBL model recently [<a href="#B24" class="cite-reflink bibr">24</a>,<a href="#B22" class="cite-reflink bibr">22</a>,<a href="#B55" class="cite-reflink bibr">55</a>,<a href="#B56" class="cite-reflink bibr">56</a>,<a href="#B54" class="cite-reflink bibr">54</a>,<a href="#B58" class="cite-reflink bibr">58</a>,<a href="#B32" class="cite-reflink bibr">32</a>], both BIC and cross-validation indicate that the covarion model performs significantly better than the MBL model on all real data sets we have analyzed so far. This considerably reduces the relevance of Kolaczkowski and Thornton (2004) observations, concerning the failure of current models and methods, including covarion, to correctly infer phylogenetic trees under heterotachous conditions, as it further confirms how artificial the simulation conditions were.<p>An obvious explanation for MBL's failure is that it is too parameter-rich ((N<sub>c</sub>-1) *(2s-2), s is the number of species and N<sub>c </sub>the number of components). Indeed, a completely new set of branch lengths has to be inferred for each component, which may be too expensive, as heterotachy may manifest itself only on a subset of the branches. Accordingly, branch lengths of the two components are relatively well correlated (R between 0.57 and 0.63, Fig. <a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F4" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">4</a>), illustrating a parametric redundancy. The difference in the behavior of the covarion model and the MBL model on the real datasets and the simulation datasets implies that the real dataset might not have such global rate shifts (i.e. all the corresponding branch lengths in different categories would be drastically different) as designed in the simulation datasets.</p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F4" name="F4"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F4" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=2248194&amp;blobname=1471-2148-7-206-4.gif" border="1" class="icon-reflink" alt="Figure 4" title="Figure 4"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 4</strong></a><div class="figure-table-caption-in-article"><span><strong>Comparison of branch lengths from the two partitions for the nuclear (A), plastid (B) and mitochondrial (C) alignments</strong>. R = 0.63, 0.63 and 0.57 respectively.</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>When multiple genes are analyzed, a separate model [<a href="#B59" class="cite-reflink bibr">59</a>] is aimed at capturing heterotachous signal among genes. The only difference with the MBL model is that the number of components and their structures are defined a priori. The separate model may therefore probably suffer from the same weaknesses as the MBL model, an inherent over-parameterization due to the fact that branch lengths are well correlated among genes, with few exceptions [<a href="#B60" class="cite-reflink bibr">60</a>]. On the other hand, it may lead to more accurate phylogenetic inference, in case where the covarion model failed [<a href="#B50" class="cite-reflink bibr">50</a>]. This indicates that both the separate model and MBL-like approaches still deserve further studies.</p><p>Mixture models generally imply numerous additional parameters. Improved fitness is obtained only if most of these additional parameters are natural, i.e. have a great explanatory power. This is for example the case for the CAT model [<a href="#B7" class="cite-reflink bibr">7</a>] in which components reflect the amino acid spectrum allowed by structural and functional constraints. Unfortunately, the combinatorial effect is too important for MBL modeling to be efficient for instance, assuming only 2 independent collective rate shifts on two distinct branches, involving two intersecting groups of sites, will create 4 distinct site patterns, describing all possible ways a given site may have 'responded' to the first and/or to the second rate shift. In this situation, the MBL model will need 4 components to explain every site correctly. More generally, with S independent collective rate shifts, 2<sup>S </sup>components will be needed to describe all possible combinations that will all be likely to occur across the alignment. This combinatorial argument may explain the failure of the MBL model in practice, in spite of its ability to detect collective behaviors.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639369" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639428" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id590721" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642382" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Conclusion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642882" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644716" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644726" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644837" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id642822">Conclusion</div><div class="section-content"><p>The covarion model, in spite of its better fit, is a purely site-independent model. As such, it may not be optimally efficient at capturing collective rate shifts, such as those that we can detect using MBL, and may instead be meant for the background of "stationary" heterotachy present at every site. This suggests that an explicit model accounting for collective events, in the spirit of MBL, albeit more parsimonious in terms of parameterization, would be an interesting direction to take. A natural approach to do this would be a divergence point model [<a href="#B61" class="cite-reflink bibr">61</a>-<a href="#B63" class="cite-reflink bibr">63</a>], where, due to the functional and/or structural shift, some sites evolve differently from other sites in the different areas of the phylogeny defined by the divergence points.</p><p>In another direction, the covarion model, in the version that we test here [<a href="#B6" class="cite-reflink bibr">6</a>], can also be improved. Wang et al. [<a href="#B31" class="cite-reflink bibr">31</a>] introduced a more general model, in which rate can not only switch from on to off but also from a given rate to another and demonstrated a slight, but generally significant, improvement. Yet, this model remains homogeneous over positions, a constraint that could be released by considering a mixture model in which the parameters of the covarion process are component specific.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639369" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639428" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id590721" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642382" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642822" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Methods</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644716" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644726" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644837" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id642882">Methods</div><div class="section-content"><p><div class="head2 head-separate">The mixture branch length (MBL) model</div>The mixture model assumes several components with different sets of branch lengths. When sites are assumed to be independent, the likelihood for the data <em>D </em>in the mixture model is the product of <em>N </em>site-specific likelihoods, and each site's likelihood is the sum of likelihoods over all <em>Nc</em> components, weighted by the components' probabilities <em>w </em><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M1" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M1" title="equation M1"><p><a id="bmcM1" name="bmcM1"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M2" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M2" title="equation M2"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table></p><p>Where <em>l </em>is <em>Nc </em>sets of (2s-3) branch lengths (s is the number of species); <em>&#x003c4; </em>is the topology; <em>&#x003b8; </em>is the rest of parameters (such as rate matrix, stationary probability); and <em>C</em><sub><em>i </em></sub>is the alignment column at site <em>i</em>. The MBL model is implemented based on a homemade software, which uses a Bayesian Markov chain Monte Carlo (MCMC) sampler [<a href="#B7" class="cite-reflink bibr">7</a>]. Maximum likelihood was calculated via simulated annealing.</p></p><p><div class="head2 head-separate">The covarion model</div> The covarion model corresponds to a doubly stochastic process The   process of rate switching is described as:<p><a id="bmcM2" name="bmcM2"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M3" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M3" title="equation M3"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table></p><p>where s<sub>01 </sub>is the rate of switching from off to on; s<sub>10 </sub>is the rate of switching from on to off. Thus, two parameters are necessary for this process, the rates of switching between the two states, off and on. When a site is in the on state, it   undergoes substitutions among the 20 amino-acids according to a first   order Markov process, described by a rate matrix Q. Here, for both the   covarion and MBL models, this substitution process was described by a   JTT+&#x00393; model with four discrete categories..</p><p>The rate matrix can be</p><p><a id="bmcM3" name="bmcM3"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M4" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M4" title="equation M4"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table></p><p>where I is the identity matrix (r &#x000d7; r, r is the number of states, for a protein data set, r = 20). For more details on the implementation, see refs. [<a href="#B30" class="cite-reflink bibr">30</a>] and [<a href="#B6" class="cite-reflink bibr">6</a>].  Therefore, R is 40 &#x000d7; 40 rate matrix for the covarion in the Markov process. For both the MBL and covarion models, the substitution process was described by a JTT+&#x00393; model with four discrete categories.</p></p><p><div class="head2 head-separate">Maximum likelihood estimation using simulated annealing</div>We use simulated annealing, within our MCMC sampler, to obtain the maximum likelihood estimation. Simulated annealing is a straightforward generalization of the MCMC algorithm, especially for high-dimensional models such as MBL [<a href="#B64" class="cite-reflink bibr">64</a>]. In a normal MCMC run, at each cycle, a new parameter value (<em>x</em>'), slightly different from the current one (<em>x</em>), is proposed according to a stochastic kernel q(<em>x</em>, d<em>x</em>'), and accepted according to the Metropolis-Hastings rule, i.e. with probability<p><a id="bmcM4" name="bmcM4"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M5" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M5" title="equation M5"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table></p><p>where <em>L(x) </em>is the likelihood for the current state; <em>L(x') </em>is the likelihood for the proposed state; <em>q(x', dx) </em>is the probability of proposing from <em>x</em>' to <em>dx </em>state; <em>q(x, dx') </em>is the probability of proposing from <em>x </em>to <em>dx' </em>state. The only additional feature to be implemented for simulated annealing is to replace this Metropolis Hastings version by its thermal version:</p><p><a id="bmcM5" name="bmcM5"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M6" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M6" title="equation M6"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table></p><p>Here, <em>&#x003b2; </em>is analogous to an inverse temperature. If <em>&#x003b2;</em>&#x0003c;1, the Markov chain is heated up (the equilibrium distribution is flatter than the posterior distribution), and if <em>&#x003b2;</em>&#x0003e;1, it is cooled down (the equilibrium distribution is more peaked around its mode). At the reference temperature (<em>&#x003b2; </em>= 1), it reduces to the posterior distribution.</p><p>Based on this modification of the Metropolis principle, one can mimic the process of a thermodynamic annealing to obtain the maxima: we start at a high temperature (<em>&#x003b2; </em>= 1), whereby the posterior distributions are extensively visited; then, as the temperature decreases (as <em>&#x003b2; </em>increases), the distribution explored by the MCMC gets progressively more peaked around the mode, until, at a sufficiently low temperature, the Markov chain "freezes" at the ML estimate. Our cooling schedule consists in starting with <em>&#x003b2; </em>= 1, and increasing its value geometrically (i.e. <em>&#x003b2; </em>= 1.01* <em>&#x003b2;</em>), until <em>&#x003b2; </em>= 50000. To check whether the chain gets stuck in local maxima, several independent runs with random starting points are performed, and compared with each other. All the independent runs were found to converge at the same maximal point.</p></p><p><div class="head2 head-separate">Model evaluations</div>The BIC [<a href="#B37" class="cite-reflink bibr">37</a>] is defined as:<p><a id="bmcM6" name="bmcM6"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M7" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M7" title="equation M7"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table></p><p>where <img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M8" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M8" title="equation M8"> is now the overall set of parameters maximizing the log-likelihood ln<em>p</em>(<em>D</em>|<img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M9" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M9" title="equation M9">), <em>K </em>is the number of parameters that have been adjusted in <img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M10" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M10" title="equation M10">, and <em>N </em>is the number of sites. The penalty depends both on the number of parameters and on the number of sites; the smaller the BIC, the better the fitness of the model. Another criterion similar to the BIC, but less strict, is the Akaike Information Criteria (AIC; [<a href="#B38" class="cite-reflink bibr">38</a>]), for which the penalty only depends on the number of parameters:</p><p><a id="bmcM7" name="bmcM7"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M11" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M11" title="equation M11"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table></p><p>A second order correction for the AIC [<a href="#B65" class="cite-reflink bibr">65</a>] has a negligible impact in the present context, and so is not reported here.</p><p>We also compared models by the cross-validation (CV). Briefly, for a given model, we first optimize parameters on a portion of the dataset, i.e. the <em>learning set </em>(<em>D</em><sub><em>L</em></sub>), then use these parameters (<img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M12" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M12" title="equation M12"><sub><em>L</em></sub>) to compute the likelihood of the <em>testing set (D</em><sub><em>T</em></sub>). Thus, the CV score is obtained by sampling the <em>learning set </em>and the <em>testing set </em>several times, and taking the expectation of the likelihood over these replicates (parameters being inferred from the training tests):</p><p><a id="bmcM8" name="bmcM8"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M13" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M13" title="equation M13"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table></p><p>By averaging over replicates, one gets rid of sampling errors in the partitioning of the dataset into a learning set and a test set. In particular, one smoothes out possible (albeit unlikely) uneven repartitions in which sites corresponding to distinct components of the mixture would be partially segregated.</p><p>The <em>learning set </em>(<em>D</em><sub><em>L</em></sub>) and the <em>testing set (D</em><sub><em>T</em></sub>) can be created in various ways. One method is the so-called <em>v-fold cross-validation</em>. The original data set is partitioned into <em>v </em>disjoint subsets of equal size; then each partition is successively used as the <em>testing set (D</em><sub><em>T</em></sub>), the union of all other <em>v</em>-1 partitions being used as the <em>learning set </em>(<em>D</em><sub><em>L</em></sub>). The overall procedure is repeated until a total of <em>v </em>tests have been performed. In this work, we used the most currently used 2-fold cross-validation schemes. The random sampling of half data set was performed ten times, which yielded a precision of CV score sufficient to discriminate among the models under study. This small value is therefore a good compromise between computational time and accuracy.</p></p><p><div class="head2 head-separate">Identifying the optimal component for each site</div>Since we do not know exactly which component a given site belongs to, the likelihood for one site is the weighted sum of likelihoods conditional on each possible allocation of the site to the available components. We can, however, calculate the posterior probability of a site (i) belonging to a given component (k):<p><a id="bmcM9" name="bmcM9"></a><table width="100%" border="0" cellspacing="5" cellpadding="5"><tr><td width="95%"><div class="eqn-image"><img src="picrender.fcgi?artid=2248194&amp;blobtype=equ&amp;blobname=M14" border="0" style="vertical-align: middle; background: #F8F8F8;padding: 2pt;" alt="equation M14" title="equation M14"></div></td><td width="5%"><div class="eqn-id"><span style="white-space: nowrap;"></span></div></td></tr></table></p><p>These posterior probabilities were then averaged over the sites, for each gene of the alignment. Alternatively, each site was affiliated to the component of higher posterior probability, and a chi-square test of the independence between the affiliations to the component, and the affiliation to each of the genes, was performed.</p></p><p><div class="head2 head-separate">Simulations</div>All the simulations were done with the JTT replacement matrix, rate across site heterogeneity being modeled by a &#x00393; distribution (four discrete categories). Heterotachous data were simulated by concatenating two alignments generated under the same tree topology, but with different branch lengths [<a href="#B24" class="cite-reflink bibr">24</a>,<a href="#B54" class="cite-reflink bibr">54</a>]. Briefly, a reference tree, with branch lengths specified, is chosen (Fig. <a href="articlerender.fcgi?artid=2248194&amp;rendertype=figure&amp;id=F1" class="fig-table-link " onclick="startTarget(this, 'figure', 1024, 800)">1</a>). Next, each branch length of the two partitions is adjusted by multiplying the length of the reference tree either with (1 + <em>&#x003c4;</em>), or with (1 - <em>&#x003c4;</em>), where <em>&#x003c4; </em><img src="corehtml/pmc/pmcents/x2208.gif" border="0" alt="[set membership]" /> [0,1] is a parameter tuning the extent of heterotachy. The choice between the two opposite multipliers ((1 + <em>&#x003c4;</em>) and (1 - <em>&#x003c4;</em>)) is made at random, independently for each branch while under two constraints: a) the corresponding branch in the two partitions should be adjusted with opposite multipliers; b) in one partition, sister branches should be adjusted with opposite multipliers also; i.e., if one branch length in one partition is increased by a factor (1 + <em>&#x003c4;</em>), then the same branch in the other partition is decreased by a factor (1 - <em>&#x003c4;</em>) and also the sibling branch length in the same partition is decreased by a factor (1 - <em>&#x003c4;</em>). In this way, the average length over the alignment remains equal to the reference length [<a href="#B54" class="cite-reflink bibr">54</a>] and the branch length heterogeneity strictly followed the strategy by Kolaczkowski and Thornton [<a href="#B24" class="cite-reflink bibr">24</a>], i.e., the branch lengths in each component tend to behavior in a Felsenstein zone. Totally, 16 simulated datasets are generated with different discrete <em>&#x003b1; </em>(0.5,1,1.5,2) and different <em>&#x003c4;</em>(0.2,0.4,0.6,0.8).</p><p><div class="head2 head-separate">Real Datasets</div>Three protein datasets were used to examine the fitness of the covarion model, the mixture branch length models, and the homotachous model (one-component model):<p>&#x02022; Nuclear alignment: a subsample was obtained from the dataset of 133 nuclear genes and 57 animal species [<a href="#B66" class="cite-reflink bibr">66</a>]. The twenty most complete species were selected. For computing time reason, only the first 5000 sites were used.</p><p>&#x02022; Plastid alignment: the dataset was created by concatenating plastid ribosomal proteins (rpl14, rpl20, rpl2, rpl33, rps12, rps16, rpl16, rpl22, rpl32, rpl37, rps19, rps3, rps7, rps11, rps14, rps18, rps2, rps4 and rps8) and RNA polymerase proteins (rpolA, rpolBp, and rpolB) from green plants, glaucophytes, red algae, cryptophytes, stramenopiles and haptophytes. The ambiguously aligned regions were removed using Gblocks [<a href="#B67" class="cite-reflink bibr">67</a>]. The final alignment contains 22 species and 3754 sites.</p><p>&#x02022; Mitochondrial alignment: we used a concatenation of 12 mitochondrial genes (atp6, atp8, cox1, cox2, cox3, cytochrome b, nad1, nad2, nad3, nad4, nad4L and nad5) totally 3591 sites from 17 mammals.</p><p>The computing times for a CV replicate (on Pentium P4, 3.2 GHz) are approximately 80 and 190 (MBL 2 components and covarion), 40 and 110, and 35 and 80 hours for nuclear, plastid and mitochondrial datasets, respectively.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id644716">Authors' contributions</div><div class="section-content"><p>YZ implemented the covarion and MBL models into phylobayes, made all the computations, and wrote the first draft of the manuscript. NL and NR helped in the programming and MCMC settings. HP and NL conceived and supervised the study. All authors contributed to the analysis of the results and to the writing of the paper. They read and approved the final manuscript.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id644726"><a id="supplementary-material-sec"></a>Supplementary Material</div><div class="section-content"><!--body--><div id="S1"><div><strong>Additional file 1</strong></div><p>MBL model in the case of the nuclear alignment of opisthokonts.The branch lengths for the two partitions are provided.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=2248194&amp;blobname=1471-2148-7-206-S1.ppt">Click here for file</a><sup> (72K)</sup></blockquote></div></div><div id="S2"><div><strong>Additional file 2</strong></div><p>MBL model in the case of the plastid alignment of plants. The branch lengths for the two partitions are provided.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=2248194&amp;blobname=1471-2148-7-206-S2.ppt">Click here for file</a><sup> (73K)</sup></blockquote></div></div><div id="S3"><div><strong>Additional file 3</strong></div><p>MBL model and gene function in the case of the plastid alignment of plants. Average posterior probabilities of component I for the two-component MBL model are provided.</p><div><blockquote><a class="int-reflink" href="picrender.fcgi?artid=2248194&amp;blobname=1471-2148-7-206-S3.doc">Click here for file</a><sup> (24K)</sup></blockquote></div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id644824">Acknowledgements</div><div class="section-content"><p><p><p>We thank Henner Brinkmann for helpful comments and the R&#x000e9;seau Qu&#x000e9;b&#x000e9;cois de Calcul de Haute Performance for computational resources. This work was supported by G&#x000e9;nome Qu&#x000e9;bec, the Canadian Institute for Advanced Research, the Canadian Research Chair Program, the Centre National de la Recherche Scientifique (through the ACI-IMPBIO Model-Phylo funding program), and the Robert Cedergren Centre for bioinformatics and genomics. This work was financially supported in part by the "60&#x000e8;me commission franco-qu&#x000e9;b&#x000e9;coise de coop&#x000e9;ration scientifique".</p></p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639369" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id639428" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id590721" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642382" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642822" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id642882" class="sidefm-pmclink" style="text-transform: none;">Methods</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644716" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id644726" class="sidefm-pmclink" style="text-transform: none;">Supplementary Material</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id644837">References</div><div class="section-content"><div class="back-matter-section"><ul class="back-ref-list" style="list-style-type: decimal;"><li id="B1">Felsenstein, J. <span class="ref-journal">Inferring phylogenies.</span> Sunderland, MA, USA , Sinauer Associates, Inc.; 2004. p. 645.</li><li id="B2">Yang, Z. Maximum-likelihood estimation of phylogeny from DNA sequences when substitution rates differ over sites. <span><span class="ref-journal">Mol Biol Evol. </span>1993;<span class="ref-vol">10</span>:1396–1401.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0VAxoq6kSGcvkZ6upr9Kh1_5Fy4aNRin4U9EOSQ0e&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8277861">PubMed</a>]</span></li><li id="B3">Lanave, C; Preparata, G; Saccone, C; Serio, G. A new method for calculating evolutionary substitution rates. <span><span class="ref-journal">J Mol Evol. </span>1984;<span class="ref-vol">20</span>:86–93.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0x9dxuB9YB1Ubf5ma_b_fneyGqEyj31m_B_Fdt5Xl&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/6429346">PubMed</a>]</span></li><li id="B4">Galtier, N; Gouy, M. Inferring phylogenies from DNA sequences of unequal base compositions. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the USA. </span>1995;<span class="ref-vol">92</span>:11317–11321.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0XWwkBoBbhtcuYI9DTu4kXGdpu-nqG60KLeSb9ALU&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7479987">PubMed</a>]</span></li><li id="B5">Galtier, N. Maximum-likelihood phylogenetic analysis under a covarion-like model. <span><span class="ref-journal">Mol Biol Evol. </span>2001;<span class="ref-vol">18</span>:866–873.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=04MBcyoi9SVhIR72hq61EJ3ey4A6j-BGdyvpa15sH&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11319270">PubMed</a>]</span></li><li id="B6">Huelsenbeck, JP. Testing a covariotide model of DNA substitution. <span><span class="ref-journal">Mol Biol Evol. </span>2002;<span class="ref-vol">19</span>:698–707.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Dc32-WTA2Lq5UUCR3v5KkDUtCWBexgftCXfrSAUn&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11961103">PubMed</a>]</span></li><li id="B7">Lartillot, N; Philippe, H. A Bayesian mixture model for across-site heterogeneities in the amino-acid replacement process. <span><span class="ref-journal">Mol Biol Evol. </span>2004;<span class="ref-vol">21</span>:1095–1109.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0j_0tzxQz95XjyUiMaJ_Ut2nPAhxt88NgBWLhPAIB&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15014145">PubMed</a>]</span></li><li id="B8">Pagel, M; Meade, A. A phylogenetic mixture model for detecting pattern-heterogeneity in gene sequence or character-state data. <span><span class="ref-journal">Syst Biol. </span>2004;<span class="ref-vol">53</span>:571–581.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0X3G4Gm-ReR-qLeJdYfzIKE2WmWDeSIpNAnoRk56H&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15371247">PubMed</a>]</span></li><li id="B9">Yang, Z. Among-site rate variation and its impact on phylogenetic analyses. <span><span class="ref-journal">Trends Ecol Evol. </span>1996;<span class="ref-vol">11</span>:367–370.</span></li><li id="B10">Mayrose, I; Friedman, N; Pupko, T. A Gamma mixture model better accounts for among site rate heterogeneity. <span><span class="ref-journal">Bioinformatics. </span>2005;<span class="ref-vol">21 Suppl 2</span>:ii151–ii158.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0HYAKLJfZM5FTNkDyJFfNZtTss-OE8BGI9_jUGAuK&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16204095">PubMed</a>]</span></li><li id="B11">Fitch, WM. Rate of change of concomitantly variable codons. <span><span class="ref-journal">Journal of Molecular Evolution. </span>1971;<span class="ref-vol">1</span>:84–96.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pYzqAcoe32zZsxxjICVDZd7w2whhk1Ht0dFCrkYh&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/4377447">PubMed</a>]</span></li><li id="B12">Penny, D; McComish, BJ; Charleston, MA; Hendy, MD. Mathematical elegance with biochemical realism: the covarion model of molecular evolution. <span><span class="ref-journal">J Mol Evol. </span>2001;<span class="ref-vol">53</span>:711–723.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hmyxVv-Sjw-8wMeyJWqgRDWcZ-1wFaWsaDsWlQ-7&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11677631">PubMed</a>]</span></li><li id="B13">Fitch, WM; Markowitz, E. An improved method for determining codon variability in a gene and its application to the rate of fixation of mutations in evolution. <span><span class="ref-journal">Biochem Genet. </span>1970;<span class="ref-vol">4</span>:579–593.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0b1QtuZH038GSO1WgoMY5NnyxespjTFtS1BImSkvy&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/5489762">PubMed</a>]</span></li><li id="B14">Philippe, H; Lopez, P. On the conservation of protein sequences in evolution. <span><span class="ref-journal">Trends in Biochemical Sciences. </span>2001;<span class="ref-vol">26</span>:414–416.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0x3ZO2bQArmf3OBhMmXr6JA-K9rs0sZEtVkueg5L8&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11469218">PubMed</a>]</span></li><li id="B15">Ane, C; Burleigh, JG; McMahon, MM; Sanderson, MJ. Covarion structure in plastid genome evolution: a new statistical test. <span><span class="ref-journal">Mol Biol Evol. </span>2005;<span class="ref-vol">22</span>:914–924.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0jEUMe2fyF3DWN1FyIo3bw7XAaNhOKGPyy9bnlQRe&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15625184">PubMed</a>]</span></li><li id="B16">Lopez, P; Forterre, P; Philippe, H. The root of the tree of life in the light of the covarion model. <span><span class="ref-journal">Journal of Molecular Evolution. </span>1999;<span class="ref-vol">49</span>:496–508.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0vYOAN8PS1IUnEkKcWlysWBTbXS18PUEKjNGOX5BW&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10486007">PubMed</a>]</span></li><li id="B17">Misof, B; Anderson, CL; Buckley, TR; Erpenbeck, D; Rickert, A; Misof, K. An empirical analysis of mt 16S rRNA covarion-like evolution in insects: site-specific rate variation is clustered and frequently detected. <span><span class="ref-journal">J Mol Evol. </span>2002;<span class="ref-vol">55</span>:460–469.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0HumRG_Tjv8fmchfnGJF-gPoSdwP5unEnU4z28QoQ&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12355265">PubMed</a>]</span></li><li id="B18">Miyamoto, MM; Fitch, WM. Testing the covarion hypothesis of molecular evolution. <span><span class="ref-journal">Mol Biol Evol. </span>1995;<span class="ref-vol">12</span>:503–513.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FUuUF3FV9kQ_5JVOhuEm20n5VQBak7Tj6H7M_Q-W&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7739391">PubMed</a>]</span></li><li id="B19">Shalchian-Tabrizi, K; Skanseng, M; Ronquist, F; Klaveness, D; Bachvaroff, TR; Delwiche, CF; Botnen, A; Tengs, T; Jakobsen, KS. Heterotachy processes in rhodophyte-derived secondhand plastid genes: Implications for addressing the origin and evolution of dinoflagellate plastids. <span><span class="ref-journal">Mol Biol Evol. </span>2006;<span class="ref-vol">23</span>:1504–1515.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0XcX-7VTbjkwVcIR73qGDqAvfCcYE50DOAK58QQBn&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16699169">PubMed</a>]</span></li><li id="B20">Taylor, MS; Kai, C; Kawai, J; Carninci, P; Hayashizaki, Y; Semple, CA. Heterotachy in mammalian promoter evolution. <span><span class="ref-journal">PLoS Genet. </span>2006;<span class="ref-vol">2</span>:e30.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0BE9UJLnZn6BU-wT8RhFYnVhOigAnnWndleG2q5Ht&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16683025">PubMed</a>]</span></li><li id="B21">Baele, G; Raes, J; Van de Peer, Y; Vansteelandt, S. An improved statistical method for detecting heterotachy in nucleotide sequences. <span><span class="ref-journal">Mol Biol Evol. </span>2006;<span class="ref-vol">23</span>:1397–1405.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0dU9eTF_k_E6YZRWNust-L-z0W3zBWme0rkvv_AEk&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16672284">PubMed</a>]</span></li><li id="B22">Lockhart, P; Novis, P; Milligan, BG; Riden, J; Rambaut, A; Larkum, T. Heterotachy and tree building: a case study with plastids and eubacteria. <span><span class="ref-journal">Mol Biol Evol. </span>2006;<span class="ref-vol">23</span>:40–45.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pYXkf8sNk3A_jMExFh9tMyv34r0nflfwkxu-pQW8&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16151191">PubMed</a>]</span></li><li id="B23">Lopez, P; Casane, D; Philippe, H. Heterotachy, an important process of protein evolution. <span><span class="ref-journal">Mol Biol Evol. </span>2002;<span class="ref-vol">19</span>:1–7.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pcXy6qS4dfXVi_4mBjzxUpByZiZmT3dNO3rmbQtz&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11752184">PubMed</a>]</span></li><li id="B24">Kolaczkowski, B; Thornton, JW. Performance of maximum parsimony and likelihood phylogenetics when evolution is heterogeneous. <span><span class="ref-journal">Nature. </span>2004;<span class="ref-vol">431</span>:980–984.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0tQerq-vD86rhsbqrjaIbTvQDYWpcODqdgZgntQ_c&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15496922">PubMed</a>]</span></li><li id="B25">Lockhart, PJ; Larkum, AW; Steel, M; Waddell, PJ; Penny, D. Evolution of chlorophyll and bacteriochlorophyll: the problem of invariant sites in sequence analysis. <span><span class="ref-journal">Proceedings of the National Academy of Sciences of the USA. </span>1996;<span class="ref-vol">93</span>:1930–1934.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0L9FUNDiBVJ-tKPrRNybB9Cjoctu_17POFNxRS50y&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8700861">PubMed</a>]</span></li><li id="B26">Lockhart, PJ; Steel, MA; Barbrook, AC; Huson, D; Charleston, MA; Howe, CJ. A covariotide model explains apparent phylogenetic structure of oxygenic photosynthetic lineages. <span><span class="ref-journal">Mol Biol Evol. </span>1998;<span class="ref-vol">15</span>:1183–1188.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0VQ6HsCUdybwFc7Tce8SnJSVkPDqINQKiD2YUaA6T&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9729882">PubMed</a>]</span></li><li id="B27">Philippe, H; Germot, A. Phylogeny of eukaryotes based on ribosomal RNA: long-branch attraction and models of sequence evolution. <span><span class="ref-journal">Mol Biol Evol. </span>2000;<span class="ref-vol">17</span>:830–834.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0j5UHQjXHGH--elY4OALYP2g_1ddfyaev3dxEkkaK&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10917801">PubMed</a>]</span></li><li id="B28">Inagaki, Y; Susko, E; Fast, NM; Roger, AJ. Covarion shifts cause a long-branch attraction artifact that unites Microsporidia and Archaebacteria in EF-1a phylogenies. <span><span class="ref-journal">Mol Biol Evol. </span>2004;<span class="ref-vol">21</span>:1340–1349.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=06avHuCPzLMZHpOCGMEFPS_KWdcMF-tAB0LygeQJM&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15034136">PubMed</a>]</span></li><li id="B29">Philippe, H; Delsuc, F; Brinkmann, H; Lartillot, N. Phylogenomics. <span><span class="ref-journal">Annu Rev Ecol Evol Syst. </span>2005;<span class="ref-vol">36</span>:541–562.</span></li><li id="B30">Tuffley, C; Steel, M. Modeling the covarion hypothesis of nucleotide substitution. <span><span class="ref-journal">Math Biosci. </span>1998;<span class="ref-vol">147</span>:63–91.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0XmgnqDgHgLundMI-L7nFnNB5mfDBN83DDzVaRQqQ&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9401352">PubMed</a>]</span></li><li id="B31">Wang, HC; Spencer, M; Susko, E; Roger, AJ. Testing for covarion-like evolution in protein sequences. <span><span class="ref-journal">Mol Biol Evol. </span>2007;<span class="ref-vol">24</span>:294–305.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Zs2usqu28fJUaQXAmgY0DGPh9iRflQqnZYndWQY8&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17056642">PubMed</a>]</span></li><li id="B32">Spencer, M; Susko, E; Roger, AJ. Likelihood, parsimony, and heterogeneous evolution. <span><span class="ref-journal">Mol Biol Evol. </span>2005;<span class="ref-vol">22</span>:1161–1164.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0FK-uVcg8F-YC2XPWS2syAwesoqt-NOLKXIEerAkp&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15746012">PubMed</a>]</span></li><li id="B33">Gelman, A; Carlin, JB; Stern, HS; Rubin, DB. <span class="ref-journal">Bayesian data analysis.</span>  Chapman &#x00026; Hall/CRC; 2004. </li><li id="B34">Feng, Z; McCulloch, CE. Using bootstrap likelihood ratios in finite mixture models. <span><span class="ref-journal">J Roy Statist Soc Ser B. </span>1996;<span class="ref-vol">58</span>:609–617.</span></li><li id="B35">Wolfe, JH. <span class="ref-journal">A Monte Carlo study of the sampling distribution of the likelihood ratio for mixtures of multinomial distributions.</span> San Diego , US Naval personnel and Training Research Laboratory; 1971. </li><li id="B36">Self, SG; Liang, KY. Asymptotic Properties of Maximum Likelihood Estimators and Likelihood Ratio Tests Under Nonstandard Conditions. <span class="ref-journal">Journal of the American Statistical Association.</span> pp. 605–610.</li><li id="B37">Schwarz, G. Estimating the dimension of a model. <span><span class="ref-journal">Ann Stat. </span>1978;<span class="ref-vol">6</span>:461–464.</span></li><li id="B38">Akaike, H.  Information theory and an extension of the maximum likelihood principle. In: Petrov , Csaki . ,  editor. <span class="ref-journal">Proceedings 2nd International Symposium on Information Theory.</span> Budapest , Akademia Kiado; 1973. pp. 267–281.</li><li id="B39">Shono, H. Efficiency of the finite correction of Akaike&#x02019;s Information Criteria. <span><span class="ref-journal">Fisheries Science. </span>2000;<span class="ref-vol">66</span>:608–610.</span></li><li id="B40">Sakamoto, Y; Ishiguro, M; Kitagawa, G. <span class="ref-journal">Information Statistics.</span> Tokyo , Kyouritsu; 1983. </li><li id="B41">Xiang, T; Gong, S. Visual learning given spare data of unknown complexity. <span><span class="ref-journal">Proceedings of the Tenth IEEE International Conference on Computer Vision (ICCV&#x02019;05). </span>2005;<span class="ref-vol">1</span>:701–708.</span></li><li id="B42">Aitkin, M; Rubin, DB. Estimation and Hypothesis Testing in Finite Mixture Models. <span><span class="ref-journal">J Royal Statistical Soc B. </span>1985;<span class="ref-vol">47</span>:67–75.</span></li><li id="B43">Bozdogan, H.  Choosing the number of component clusters in the mixture-model using a new informational complexity criterion of the inverse-Fisher information matrix. In: Opitz O, Lausen B, Klar R. ,  editor. <span class="ref-journal">Information and classification, concepts, methods and applications.</span> Berlin , Springer; 1993. pp. 40–54.</li><li id="B44">Djuric, PM. Model Selection Based On Asymptotic Bayes Theory. <span class="ref-journal">IEEE Seventh SP Workshop on Statistical Signal and Array Processing.</span> 1994. pp. 7–10.</li><li id="B45">Keribin, C. Consistent estimation of the order of mixture models. <span><span class="ref-journal">Sankhya Ser A. </span>2000;<span class="ref-vol">62</span>:49–66.</span></li><li id="B46">Stone, M. Cross validatory choice and assessments of statistical predictions. <span><span class="ref-journal">J Roy Statist Soc Ser B. </span>1974;<span class="ref-vol">36</span>:111–117.</span></li><li id="B47">Smyth, P. Model selection for probabilistic clustering using cross-validated likelihood. <span><span class="ref-journal">Stat Comput. </span>2000;<span class="ref-vol">10</span>:63–72.</span></li><li id="B48">van der Laan, MJ; Dudoit, S; Keles, S. Asymptotic optimality of likelihood-based cross-validation. <span><span class="ref-journal">Statistical Applications in Genetics and Molecular Biology. </span>2004;<span class="ref-vol">3</span>:4.</span></li><li id="B49">Phylobayes.  <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0Vy0k5JkIc8h4lp6pH2FzH8hv5ICYTvgTrn2Zx5BQ&amp;reftype=extlink&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.lirmm.fr/mab/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.lirmm.fr/mab/</a></span></li><li id="B50">Rodriguez-Ezpeleta, N; Philippe, H; Brinkmann, H; Becker, B; Melkonian, M. Phylogenetic analyses of nuclear, mitochondrial, and plastid multigene data sets support the placement of mesostigma in the streptophyta. <span><span class="ref-journal">Mol Biol Evol. </span>2007;<span class="ref-vol">24</span>:723–731.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0v5wa5hAPu_YByiq65NW74JCoTtgKs0heD0sKc5RB&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17172635">PubMed</a>]</span></li><li id="B51">Alfaro, ME; Huelsenbeck, JP. Comparative performance of Bayesian and AIC-based measures of phylogenetic model uncertainty. <span><span class="ref-journal">Syst Biol. </span>2006;<span class="ref-vol">55</span>:89–96.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0xu0xWov7sRg_1KwEFxGphka1m5LLkY9EVTwPrAhz&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16507526">PubMed</a>]</span></li><li id="B52">Celeux, G; Soromenho, G. An entropy criterion for assessing the number of clusters in a mixture model. <span><span class="ref-journal">Journal of Classification. </span>1996;<span class="ref-vol">13</span>:195–212.</span></li><li id="B53">Soromenho, G. Comparing approaches for testing the number of components in a finite mixture model. <span><span class="ref-journal">Computational Statistics. </span>1994;<span class="ref-vol">9</span>:65–78.</span></li><li id="B54">Philippe, H; Zhou, Y; Brinkmann, H; Rodrigue, N; Delsuc, F. Heterotachy and long-branch attraction in phylogenetics. <span><span class="ref-journal">BMC Evol Biol. </span>2005;<span class="ref-vol">5</span>:50.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0rmEUPszVr4Sl4ykguGXV4KJi0MiJiRxbXavFhA1A&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16209710">PubMed</a>]</span></li><li id="B55">Gaucher, EA; Miyamoto, MM. A call for likelihood phylogenetics even when the process of sequence evolution is heterogeneous. <span><span class="ref-journal">Mol Phylogenet Evol. </span>2005;<span class="ref-vol">37</span>:928–931.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hapsDWda322imD3eoFKCdVY0Qx5pMyJhICc13Azn&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16291095">PubMed</a>]</span></li><li id="B56">Gadagkar, SR; Kumar, S. Maximum likelihood outperforms maximum parsimony even when evolutionary rates are heterotachous. <span><span class="ref-journal">Mol Biol Evol. </span>2005;<span class="ref-vol">22</span>:2139–2141.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0tYwgYghHbDN6amr_e9ljSLurSPAxTZXO9jlk1kfr&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16014870">PubMed</a>]</span></li><li id="B57">Schmidt, TR; Wu, W; Goodman, M; Grossman, LI. Evolution of nuclear- and mitochondrial-encoded subunit interaction in cytochrome c oxidase. <span><span class="ref-journal">Mol Biol Evol. </span>2001;<span class="ref-vol">18</span>:563–569.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0XKVD6Lk6Dt5rlIUPimU0sSlE9NF-GZKZzfFawAdO&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11264408">PubMed</a>]</span></li><li id="B58">Steel, M. Should phylogenetic models be trying to 'fit an elephant'? <span><span class="ref-journal">Trends Genet. </span>2005;<span class="ref-vol">21</span>:307–309.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0zQ4lwqROYPKWO_zSvV5yd71xDkapX2au0Ma4PQX4&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15922824">PubMed</a>]</span></li><li id="B59">Yang, Z. Maximum-likelihood models for combined analyses of multiple sequence data. <span><span class="ref-journal">Journal of Molecular Evolution. </span>1996;<span class="ref-vol">42</span>:587–596.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hImMbqCKOv8pwe0uDj74zmZ6zHWHGyNKND4cr5pa&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8662011">PubMed</a>]</span></li><li id="B60">Moreira, D; Kervestin, S; Jean-Jean, O; Philippe, H. Evolution of eukaryotic translation elongation and termination factors: variations of evolutionary rate and genetic code deviations. <span><span class="ref-journal">Mol Biol Evol. </span>2002;<span class="ref-vol">19</span>:189–200.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0tooIKwdxGaR7D5qeRMCLffOq_jhLFWPdWGpZHkDl&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11801747">PubMed</a>]</span></li><li id="B61">Huelsenbeck, JP; Larget, B; Swofford, D. A compound poisson process for relaxing the molecular clock. <span><span class="ref-journal">Genetics. </span>2000;<span class="ref-vol">154</span>:1879–1892.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0R_2S4AmDL_hWmRL0dsC3yiZ3XY8WHHpHrIg7skUI&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10747076">PubMed</a>]</span></li><li id="B62">Blanquart, S; Lartillot, N. A Bayesian compound stochastic process for modeling nonstationary and nonhomogeneous sequence evolution. <span><span class="ref-journal">Mol Biol Evol. </span>2006;<span class="ref-vol">23</span>:2058–2071.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0zeMHmDcXyVJ3qZdSRs5_Eq2t1nfD5nluLkeZdAzK&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16931538">PubMed</a>]</span></li><li id="B63">Dorman, KS. Identifying dramatic selection shifts in phylogenetic trees. <span><span class="ref-journal">BMC Evol Biol. </span>2007;<span class="ref-vol">7 Suppl 1</span>:S10.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0x79lY8tk3F8aEnEgGylGFEC5F8ujAJhl5g0umkXr&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17288568">PubMed</a>]</span></li><li id="B64">Kirkpatrick, S; Gelatt, CD; Vecchi, MP. Optimization by simulated annealing. <span><span class="ref-journal">Science. </span>1983;<span class="ref-vol">220</span>:671–680.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0lIBayMcXs_jEJ3wXeMXn0OcpMELFDP5R6KKNYAri&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17813860">PubMed</a>]</span></li><li id="B65">Posada, D; Buckley, TR. Model selection and model averaging in phylogenetics: advantages of akaike information criterion and bayesian approaches over likelihood ratio tests. <span><span class="ref-journal">Syst Biol. </span>2004;<span class="ref-vol">53</span>:793–808.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0LAetTML_hAwbrHGcwWLiVkXHVn17_d6ZWCqhKQuo&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15545256">PubMed</a>]</span></li><li id="B66">Baurain, D; Brinkmann, H; Philippe, H. Lack of resolution in the animal phylogeny: closely spaced cladogeneses or undetected systematic errors? <span><span class="ref-journal">Mol Biol Evol. </span>2007;<span class="ref-vol">24</span>:6–9.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=00eyTmhHycOy4rdS6ndmgSNjpls36x41eKms_5kWI&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/17012374">PubMed</a>]</span></li><li id="B67">Castresana, J. Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. <span><span class="ref-journal">Mol Biol Evol. </span>2000;<span class="ref-vol">17</span>:540–552.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0VUszsb3T2VtbSWdRkhhpFpqyr9Ih8i-ImGzsq5Ag&amp;reftype=pubmed&amp;artid=2248194&amp;article-id=2248194&amp;iid=141646&amp;issue-id=141646&amp;jid=28&amp;journal-id=28&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10742046">PubMed</a>]</span></li></ul></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">BMC Evolutionary Biology</span> are provided here courtesy of <br><b>BioMed Central</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc3-->

