<HTML>
<HEAD>
<link rel=stylesheet href=http://www.ncbi.nlm.nih.gov/corehtml/ncbi_test.css>
<TITLE>OMIM Gene Map Methods</TITLE>
</HEAD>
<BODY bgcolor=#ffffff text=#000000 link=#0000f0 vlink=#6000b0 alink=#f00000>

    <table border=0 cellpadding=0 cellspacing=0><tr>
      <td WIDTH=140><a href=http://www.ncbi.nlm.nih.gov/><img SRC=http://www.ncbi.nlm.nih.gov/corehtml/left.GIF BORDER=0 height=45 width=130></a></td>
      <td valign=BOTTOM width=*><a href=/entrez/query.fcgi?db=OMIM><img src=/entrez/query/static/gifs/entrez_omim.gif border=0 height=45 width=456></a></td>
    </tr></table>

    <table class=TEXT border=0 cellspacing=0 cellpadding=2 bgcolor=#000000 width=100%>
    <tr class=TEXT align=CENTER>
      <td width=12.5%><a href=/entrez/query.fcgi?db=PubMed class=GUTTER3><FONT COLOR=#FFFFFF>PubMed</FONT></a></td>
      <td width=12.5%><a href=/entrez/query.fcgi?db=Nucleotide class=GUTTER3><FONT COLOR=#FFFFFF>Nucleotide</FONT></a></td>
      <td width=12.5%><a href=/entrez/query.fcgi?db=Protein class=GUTTER3><FONT COLOR=#FFFFFF>Protein</FONT></a></td>
      <td width=12.5%><a href=/entrez/query.fcgi?db=Genome class=GUTTER3><FONT COLOR=#FFFFFF>Genome</FONT></a></td>
      <td width=12.5%><a href=/entrez/query.fcgi?db=Structure class=GUTTER3><FONT COLOR=#FFFFFF>Structure</FONT></a></td>
      <td width=12.5%><a href=/entrez/query.fcgi?db=Popset class=GUTTER3><FONT COLOR=#FFFFFF>PopSet</FONT></a></td>
      <td width=12.5%><a href=/entrez/query.fcgi?db=Taxonomy class=GUTTER3><FONT COLOR=#FFFFFF>Taxonomy</FONT></a></td>
      <td width=12.5%><a href=/entrez/query.fcgi?db=OMIM class=GUTTER3><FONT COLOR=#FFFFFF>OMIM</FONT></a></td>
    </tr></table>

<H2>OMIM Gene Map Methods</H2>
The methods for mapping genes (see Table 4 for
counts) are symbolized as follows:
<P><STRONG>A</STRONG> = <EM>in situ</EM> DNA-RNA or DNA-DNA annealing
(`hybridization'); e.g., ribosomal RNA genes to acrocentric chromosomes; kappa
light chain genes to chromosome 2.
<P><STRONG>AAS</STRONG> = deductions from the amino acid sequence of proteins;
e.g., linkage of delta and beta hemoglobin loci from study of hemoglobin Lepore.
(Includes deductions of hybrid protein structure by monoclonal antibodies; e.g.,
close linkage of MN and SS from study of Lepore-like MNSs blood group antigen.)
Also includes examples of hybrid genes as in one form of hypertrophic
cardiomyopathy and in apolipoprotein (Detroit).
<P><STRONG>C</STRONG> = chromosome mediated gene transfer (CMGT); e.g.,
cotransfer of galactokinase and thymidine kinase. (In conjunction with this
approach fluorescence-activated flow sorting can be used for transfer of
specific chromosomes.)
<P><STRONG>Ch</STRONG> = chromosomal change associated with particular phenotype
and not proved to represent linkage (Fc), deletion (D), or virus effect (V);
e.g., loss of 13q14 band in some cases of retinoblastoma. (`Fragile sites,'
observed in cultured cells with or without folate-deficient medium or BrdU
treatment, fall into this class of method; e.g., fragile site at Xq27.3 in one
form of X-linked mental retardation. Fragile sites have been used as markers in
family linkage studies; e.g., FS16q22 and haptoglobin.)
<P><STRONG>D</STRONG> = deletion or dosage mapping (concurrence of chromosomal
deletion and phenotypic evidence of hemizygosity), trisomy mapping (presence of
three alleles in the case of a highly polymorphic locus), or gene dosage effects
(correlation of trisomic state of part or all of a chromosome with 50% more gene
product). Includes "loss of heterozygosity" (loss of alleles) in malignancies.
Examples: glutathione reductase to chromosome 8. Includes DNA dosage; e.g.,
fibrinogen loci to 4q2. Dosage mapping also includes coamplification in tumor
cells.
<P><STRONG>EM</STRONG> = exclusion mapping, i.e., narrowing the possible
location of loci by exclusion of parts of the map by deletion mapping, extended
to include negative lod scores from families with marker chromosomes and
negative lod scores with other assigned loci; e.g., support for assignment of
MNSs to 4q.
<P><STRONG>F</STRONG> = linkage study in families; e.g., linkage of ABO blood
group and nail-patella syndrome. (When a chromosomal heteromorphism or
rearrangement is one trait, <STRONG>Fc</STRONG> is used; e.g., Duffy blood group
locus on chromosome 1. When 1 or both of the linked loci are identified by a DNA
polymorphism, <STRONG>Fd</STRONG> is used; e.g., Huntington disease on
chromosome 4. F = L in the HGM workshops.)
<P><STRONG>H</STRONG> = based on presumed homology; e.g., proposed assignment of
TF to 3q. Includes Ohno's law of evolutionary conservatism of X chromosome in
mammals. Mainly heuristic or confirmatory.
<P><STRONG>HS</STRONG> = DNA/cDNA molecular hybridization in solution (`Cot
analysis'); e.g., assignment of Hb beta to chromosome 11 in derivative hybrid
cells.
<P><STRONG>L</STRONG> = lyonization; e.g., OTC to X chromosome. (L = family
linkage study in the HGM workshops.)
<P><STRONG>LD</STRONG> = linkage disequilibrium; e.g., beta and delta globin
genes (HBB, HBD).
<P><STRONG>M</STRONG> = Microcell mediated gene transfer (MMGT); e.g., a
collagen gene (COL1A1) to chromosome l7.
<P><STRONG>OT</STRONG> = ovarian teratoma (centromere mapping); e.g., PGM3 and
centromere of chromosome 6.
<P><STRONG>Pcm</STRONG> = PCR of microdissected chromosome segments (see REl).
<P><STRONG>Psh</STRONG> = PCR of somatic cell hybrid DNA.
<P><STRONG>R</STRONG> = irradiation of cells followed by `rescue' through fusion
with nonirradiated (nonhuman) cells (Goss-Harris method of radiation-induced
gene segregation); e.g., order of genes on Xq. (Also called cotransference. The
complement of cotransference = recombination.)
<P><STRONG>RE</STRONG> = Restriction endonuclease techniques; e.g., fine
structure map of the beta-globin cluster (HBBC) on 11p; physical linkage of 3
fibrinogen genes (on 4q) and APOA1 and APOC3 (on 11p).
<DL>
  <DD><STRONG>REa</STRONG> = combined with somatic cell hybridization; e.g., NAG
  (HBBC) to 11p.
  <DD><STRONG>REb</STRONG> = combined with chromosome sorting; e.g., insulin to
  11p. Includes Lebo's adaptation (dual laser chromosome sorting and spot blot
  DNA analysis); e.g., MGP to 11q. (For this method, using flow sorted
  chromosomes, W is the symbol adopted by the HGM workshops.)
  <DD><STRONG>REc</STRONG> = hybridization of cDNA to genomic fragment (by YAC,
  PFGE, microdissection, etc.), e.g., A11 on Xq.
  <DD><STRONG>REf</STRONG> = isolation of gene from genomic DNA; includes 'exon
  trapping'
  <DD><STRONG>REl</STRONG> = isolation of gene from chromosome-specific genomic
  library (see Pcm).
  <DD><STRONG>REn</STRONG> = neighbor analysis in restriction fragments, e.g.,
  in PFGE. </DD></DL>
<P><STRONG>S</STRONG> = `segregation' (cosegregation) of human cellular traits
and human chromosomes (or segments of chromosomes) in particular clones from
interspecies somatic cell hybrids; e.g., thymidine kinase to chromosome 17. When
with restriction enzyme, REa; with hybridization in solution, HS.
<P><STRONG>T</STRONG> = TACT = telomere-associated chromosome fragmentation;
e.g., interferon-inducible protein 6-16.
<P><STRONG>V</STRONG> = induction of microscopically evident chromosomal change
by a virus; e.g., adenovirus 12 changes on chromosomes 1 and 17.
<P><STRONG>X/A</STRONG> = X-autosome translocation in female with X-linked
recessive disorder; e.g., assignment of Duchenne muscular dystrophy to Xp21.

<P>&nbsp;</P>
<div align=center class=medium1><p>
   <a href=/Omim/disclaimer.html>Disclaimer</a> |
   <a href=mailto:info@ncbi.nlm.nih.gov>Write to the Help Desk</a> |
   <a href=/entrez/query/static/privacy.html>Privacy Policy</a><br>
   <a href=http://www.ncbi.nlm.nih.gov>NCBI</a> |
   <a href=http://www.nlm.nih.gov>NLM</a> |
   <a href=http://www.nih.gov>NIH</a>
</p></div>

</BODY></HTML>
