<!DOCTYPE HTML PUBLIC "-//SQ//DTD HTML 2.0 HoTMetaL + extensions//EN">
<HTML><HEAD>
<TITLE>APPENDIX 3: LINKAGE TABLE FOR MUTANT GENES</TITLE>
</HEAD>
<BODY><HR>
<CENTER><H1>APPENDIX 3</H1></CENTER> 
<CENTER><H2>LINKAGE TABLE FOR MUTANT GENES</H2></CENTER>
<HR>
<P>
This Appendix lists mutant genes of <I>Arabidopsis</I> for which recombination
data have been obtained.  This information was extracted from research
publications and from responses to email surveys distributed to members of the
<I>Arabidopsis</I> community.  Additions and corrections should be reported to
David Meinke, curator, at: meinke@osuunx.ucc.okstate.edu.  Updated editions of
this table can be viewed through the WWW at:
<A HREF="http://mutant.lse.okstate.edu/meinke.html">
http://mutant.lse.okstate.edu/meinke.html</A>.
</P>
<HR>
<HR>
<P><B>
EXPLANATION OF COLUMNS:</B> 
</P>
<P>
<B>Locus: </B>
</P>
<P>
Name of gene identified by mutation, arranged in alphabetical order, and written
in CAPS and without italics to avoid confusion between numbers and letters. 
Genes marked with an asterisk are known by more than one name.  Synonyms for
these genes are included in the table but are marked with (=) in the second (#)
column.
</P>
<P>
<B>#:</B>
</P>
<P>
Chromosome where gene is located.  When two different numbers are included, the
mutant gene exhibits linkage to markers on different chromosomes.  An equal sign
(=) denotes synonyms as explained above.  
</P>
<P>
<B>Status:</B>
</P>
<P>
This summarizes the current status of linkage data:
</P>
<P>
 CM: Gene placed on classical map (Appendix 5).
</P>
<P>
 AV: Location approximated with visible markers.
</P>
<P>
 AM: Location approximated with molecular markers.
</P>
<P>
 AVM: Location approximated with visible and molecular markers. 
</P>
<P>
 LO: Gene assigned to linkage group only.
</P>
<P>
 U: Nature and extent of mapping data unresolved.
</P>
<P>
<B>Map:</B>
</P>
<P>
Estimated location (cM) of locus on classical genetic map (Appendix 5).  Numbers
marked with an asterisk refer to genes that were initially placed on the
recombinant inbred map based on data with molecular markers.  To compensate for
map expansion of RI chromosomes, the estimated location of a given locus on the
RI map was multiplied by the following constant:  (total length of classical
chromosome) / (total length of RI chromosome) in order to estimate the
corresponding location on the classical map.  These values for each chromosome
were: chromosome 1 (0.722); chromosome 2 (0.838); chromosome 3 (0.678);
chromosome 4 (0.624); chromosome 5 (0.609).  Other genes were localized only to
chromosomal regions: top; middle (mid); and bottom (bot).  
</P>
<P>
<B>Linked Marker:</B>
</P>
<P>
Linked visible or molecular marker with recombination data available.  In some
cases, data with more than one marker are included.  
</P>
<P>
<B>% Rec</B> 
</P>
<P>
Recombination percentage between locus and linked marker.  A = locus is above
linked marker; B = locus is below linked marker.  
</P>
<P>
<B>Lab:</B>
</P>
<P>
Reference laboratory that either provided linkage data or has worked extensively
with the mutant.  Names and email addresses for these reference laboratories are
provided in Appendix 4.
</P>
<P>
<B>Full Descriptive Name of Gene Symbol:</B>
</P>
<P>
Self explanatory.  Contact stock centers, databases, recent publications, or
reference laboratories for additional information
</P>
<HR>
<PRE WIDTH="80"><FONT SIZE="-1">
<B>Locus	#	Status	Map	Linked	%Rec	Lab	Full Descriptive Name of Gene Symbol
</B><B>				Marker</B>

<TT>ABA1	5	CM	98	MIN	7	1	ABA deficient
ABA2	1	CM	70	CH1	12B	1	ABA deficient
ABA3	1	CM	10	AN	10B	1	ABA deficient
ABI1	4	CM	55	CER2	3	1	ABA insensitive
ABI2	5	AVM	-	TZ	-	3	ABA insensitive
ABI3	3	CM	38	GL1	10	1	ABA insensitive
ABI4	2	CM	62	-	-	3	ABA insensitive
ABI5	2	CM	49	-	-	3	ABA insensitive
ACA1	5	U	-	-	-	4	Azetidine-2-carboxylic acid resistant
ACD1	3	CM	68	GL1	21	5	Accelerated cell death
ACD2	4	CM	68	AP2/CP3	&lt;1/4	&quot;5,86&quot;	Accelerated cell death
ACL1	4	CM	55	-	-	7	Acaulis (short floral stems)
ACL2	1	CM	83*	MI209/
				MI303	0/0	7	Acaulis (short floral stems)
ACL3	3	AV	-	GL1	10	7	Acaulis (short floral stems)
ACL5	5	AM	-	ASA1	5B	7	Acaulis (short floral stems)
ACT1	1	CM	35	AN	30	6	Cp glycerol-3-phosphate acyltrans. def.
ADD1	5	CM	23*	G4560	1	26	Arrested development
ADH1	1	CM	114	CLV1	6	8	Alcohol dehydrogenase deficient
AG	4	CM	42	CH42	3	1	Agamous
AGR	5	AM	-	LFY3	4	60	Agravitropic
AIM	4	U	BOT	-	-	27	Abnormal inflorescence meristem
AIN1	1	CM	70	-	-	67	Ethylene insensitive
ALB1	1	CM	12	DIS1	10	1	Albina
ALB2	5	CM	15	CO	1	1	Albina
ALB3	2	U	-	-	-	&quot;9,10&quot;	Albina
ALB4	4	CM	40	-	-	-	Albina
ALF1	=	HLS3	-	-	-	14	Aberrant lateral root formation
ALF3	5	CM	68*	DFR/
				NGA129	9/7	&quot;14,83&quot;	Aberrant lateral root formation
ALF4	5	CM	15*	NGA151/
				249	1/1	&quot;14,83&quot;	Aberrant lateral root formation
ALF5	3	U	MID	-	-	14	Aberrant lateral root formation
ALR1	1	AM	-	NGA280	33	72	Aluminum resistant
ALS1	5	CM	62*	NGA129	17A	72	Aluminum sensitive
ALS4	5	AM	-	NGA151	6	72	Aluminum sensitive
AMP1*	3	U	-	AP3-TT5	-	&quot;10,66&quot;	Altered meristem program
AMT1	5	CM	11	ASA-1	&lt;1	11	a-methyl tryptophan resistant
AN	1	CM	0	SUS1	&lt;1	&quot;1,2&quot;	Angustifolia (narrow leaf)
ANL1	5	CM	15	ALB2	0A	1	Anthocyaninless
ANT	4	AM	-	DHS1/AG	0/19	55	Aintegumenta (integument defect)
AP1	1	CM	99	CER6	2	1	Apetala
AP2*	4	CM	68	FD	1	1	Apetala
AP3	3	CM	81	TT5	0	1	Apetala
ARA1	4	CM	37	FCA	&lt;1	12	Arabinose sensitive
ARC6	5	U	BOT	-	-	62	&quot;Accumulation, replication chloroplast&quot;
AS1	2	CM	60	EMB18	&lt;1	&quot;1,2&quot;	Asymmetric leaves
AS2	1	CM	85	-	-	13	Asymmetric leaves
ATR	5	U	-	-	-	14	Altered tryptophan regulation
ATS	5	CM	64	EMB9	5B	1	Aberrant testa shape
AUX1*	2	CM	62	-	-	1	Auxin resistant
AXR1*	1	CM	12	AXR3	1	15	Auxin resistant
AXR2	3	CM	33	GL1	13	15	Auxin resistant
AXR3	1	CM	11	AXR1	1	15	Auxin resistant
AXR4	1	CM	62	-	-	-	Auxin resistant
BEL1	5	CM	50	-	-	-	Bell-shaped ovules; female sterile
BER	1	AM	-	RAP10-
				M253	-	57	Bicyclic ether resistant
BIO1	5	CM	74	TZ	4	2	Biotin auxotroph
BLU	=	HY4	-	-	-	71	Discontinued
BOT1	1	CM	115*	ADH	5B	59	Botero (short hypocotyl)
BP1	4	CM	15	EMB132	3	&quot;1,2&quot;	Brevipedicellus (downward siliques)
BUM	1	CM	89	AP1	9A	26	Bumbershoot (shoot meristem)
CAD1	5	CM	57	TT3	-	12	Cadmium sensitive
CAD2	4	U	-	-	-	12	Cadmium sensitive
CAL	1	CM	46	-	-	-	Cauliflower (defective inflorescence)
CAR1	1	CM	87*	NGA128	5B	72	CaMV resistant
CBB2	4	CM	76*	DHS1	3	41	Cabbage (extreme dwarf)
CBB3	5	CM	13*	NGA151/
				106	8/11	41	Cabbage (extreme dwarf)
CER1	1	CM	1	AN	1	1	Eceriferum (altered surface wax)
CER2	4	CM	52	EMB140	0	&quot;1,2&quot;	Eceriferum (altered surface wax)
CER3	5	CM	82	LFY	0	&quot;1,16&quot;	Eceriferum (altered surface wax)
CER4	4	CM	61	CP3	15	1	Eceriferum (altered surface wax)
CER5	1	CM	70	CLV2	16	1	Eceriferum (altered surface wax)
CER6*	1	CM	96	AP1	2	1	Eceriferum (altered surface wax)
CER7	3	CM	88	FUS6	&lt;1	&quot;1,2&quot;	Eceriferum (altered surface wax)
CER8	2	CM	72	EMB39	&lt;1	&quot;1,2&quot;	Eceriferum (altered surface wax)
CER9	4	CM	76	AP2	9	1	Eceriferum (altered surface wax)
CH1	1	CM	58	EMB22	4	&quot;1,2&quot;	Chlorina (yellow green)
CH5	5	CM	43	TT2	2	1	Chlorina (yellow green)
CH6	3	CM	84	TT5	3	1	Chlorina (yellow green)
CH12	4	U	-	-	-	17	Chlorina (yellow green)
CH13	4	U	-	-	-	17	Chlorina (yellow green)
CH17	3	U	-	-	-	17	Chlorina (yellow green)
CH42	4	CM	44	AG	3	1	Chlorina (yellow green)
CHvi3	3	U	-	-	-	17	Chlorina (yellow green)
CHL1	1	CM	14	DIS1	3	1	Chlorate resistant
CHL2	2	CM	32	ER	12	1	Chlorate resistant
CHL3*	1	CM	56	CH1	6	1	Chlorate resistant
CHL4	=	RGN	-	-	-	18	Discontinued
CHL5	2	U	-	-	-	18	Chlorate resistant
CHL6	5	CM	22	-	-	18	Chlorate resistant
CHM	3	CM	32	GL1	15	1	Chloroplast mutator
CHP7	5	CM	8	CO	6	1	Chlorophyll mutant
CHS1	1	CM	33	AN	29	6	Chilling sensitive
CKR1	=	EIN2	-	-	-	72	Cytokinin resistant
CLV1*	1	CM	110	EMB120	3	&quot;1,2&quot;	Clavata (extra carpels)
CLV2	1	CM	89	SUS3	2	&quot;1,2&quot;	Clavata (extra carpels)
CLV3	2	CM	43	ER	1	42	Clavata (extra carpels)
CN	4	CM	31	-	-	19	&quot;Convoluta (leaf shape, twisted stem)&quot;
CO*	5	CM	13	ALB2	1	1	Late flowering
COB	5	CM	80	LFY	1	43	Cobra (root defective)
COG1	=	RPP1	-	-	-	34	Altered recognition of Peronospora
COP1*	2	CM	56	AS	3	&quot;2,20&quot;	Constitutive photomorphogenic
COP2	=	AMP1	-	-	-	20	Constitutive photomorphogenic
COP3	=	HLS1	-	-	-	20	Constitutive photomorphogenic
COP8	4	AM	-	GA1	14A	20	Constitutive photomorphogenic
COP9*	4	CM	29	BP	10	&quot;2,20&quot;	Constitutive photomorphogenic
COP10	3	CM	13*	NGA162	2A	20	Constitutive photomorphogenic
COP11	=	FUS6	-	-	-	20	Constitutive photomorphogenic
COP12	=	FUS12	-	-	-	20	Constitutive photomorphogenic
COP13	=	FUS11	-	-	-	20	Constitutive photomorphogenic
COP14	=	FUS4	-	-	-	20	Constitutive photomorphogenic
COP15	=	FUS5	-	-	-	20	Constitutive photomorphogenic
CP1	4	CM	28	AG	13	1	Compacta (semi dwarf)
CP2	2	CM	34	FVE	6	1	Compacta (semi dwarf)
CP3	4	CM	73	AP2	5	&quot;1,86&quot;	Compacta (semi dwarf)
CPC	2	CM	73*	M336	3B	58	Caprice (root hair defective)
CRC	1	CM	100	AP1	1	21	Crabs claw (floral defective)
CRM1	3	LO	-	GL1	35	7	Corymbose inflorescence
CSI	4	CM	11	BP	-	22	Cesium resistant
CSR1	3	CM	70	TT5	9	1	Chlorsulfuron/imidazolinone resistant
CTR1	5	CM	0	TT4	11A	23	Constitutive triple response
CUD	4	CM	66*	DHS1	16A	43	Cudgel (root defective)
CUE1	5	CM	43*	TTG/-	11B/-	24	CAB underexpressed
CYR1	5	AM	-	DFR	13	44	Cytokinin resistant
DDM1	5	U	-	-	-	54	Decreased DNA methylation
DEI1	3	AM	-	PURM	2	59	&quot;Embryo, hypocotyl defective&quot;
DET1*	4	CM	20	BP	4	24	De-etiolated
DET2	2	CM	60	ER/M429	-/2A	24	De-etiolated
DET3	1	CM	12	CHL1	2	24	De-etiolated
DGD1	3	CM	11*	NGA162/
				172	6/18	85	Digalactosyl diacylglycerol deficient
DIM1	=	DWF1	-	-	-	73	Diminutia
DIS1	1	CM	19	GN	2	&quot;1,2&quot;	Distorted trichomes
DIS2	1	CM	45	EMB88	2	&quot;1,2&quot;	Distorted trichomes
DOC1	3	CM	4	-	-	24	Dark over-expression of CAB
DOC2	5	CM	36	-	-	24	Dark over-expression of CAB
DOC3	1	CM	90	-	-	24	Dark over-expression of CAB
DRG	4	CM	56*	PG11	3	55	Dragon (ovule defective)
DW1	1	CM	23	GAI	&lt;1	1	Dwarf
DWF1*	3	CM	26	-	-	25	Dwarf
E112	=	PRC1	-	-	-	59	Discontinued
ECO	1	AM	-	NGA280	10	26	&quot;Ectopic outgrowth (meristem, leaf)&quot;
ECR	=	RPW	-	-	-	&quot;64,79&quot;	Discontinued
EIN1	=	ETR	-	-	-	23	Ethylene insensitive
EIN2*	5	CM	1	-	-	23	Ethylene insensitive
EIR1	5	CM	70	TT3	12	23	Ethylene insensitive root
ELF3	2	CM	43	ER	1	26	Early flowering
ELG	4	CM	63	AP2	5A	1	Elongated hypocotyl
EMB1	3	CM	5	HY2	7	2	Embryo defective
EMB2	5	CM	27	TTG	1	2	Embryo defective
EMB7	2	LO	-	ER	26	2	Embryo defective
EMB9	5	CM	59	TT3	2	2	Embryo defective
EMB10	1	CM	27	DIS1	8	2	Embryo defective
EMB14	=	SUS2-2	-	-	-	2	Discontinued
EMB15	5	CM	85	YI	&lt;1	2	Embryo defective
EMB16	5	CM	90	YI	4	2	Embryo defective
EMB17	1	CM	111	GL2	7	2	Embryo defective
EMB18	2	CM	61	AS	1	2	Embryo defective
EMB19	1	LO	-	-	-	2	Embryo defective
EMB20	4	CM	56	CER2	2	2	Embryo defective
EMB22	1	CM	62	CH1	4	2	Embryo defective
EMB24	5	CM	1	LU	8	2	Embryo defective
EMB25	1	CM	100	CLV1	6	2	Embryo defective
EMB27	1	CM	66	CER5	4	2	Embryo defective
EMB28*	4	CM	58	CER2	5	2	Embryo defective
EMB29	3	CM	24	HY2	11	2	Embryo defective
EMB30*	1	CM	21	DIS1	2	2	Embryo defective 
EMB31	2	LO	-	-	-	2	Embryo defective
EMB33	=	SUS2-3	-	-	-	2	Discontinued
EMB34*	2	CM	0	STI	4	2	Embryo defective
EMB35	4	CM	58	CER2	4	2	Embryo defective
EMB36	=	EMB156
		-2	-	-	-	2	Discontinued
EMB38	2	CM	66	AS	6	2	Embryo defective
EMB39	2	CM	73	CER8	0	2	Embryo defective
EMB40	=	FS1	-	-	-	2	Embryo defective
EMB41	1	CM	92	CLV2	3	2	Embryo defective
EMB42	4	CM	60	AP2	11	2	Embryo defective
EMB43	1	CM	26	DIS2	16	2	Embryo defective
EMB49	2	CM	63	AS	8	2	Embryo defective
EMB51	3	CM	14	HY2	1	2	Embryo defective
EMB52	3	CM	3	HY2	9	2	Embryo defective
EMB53	2	CM	54	AS	9	2	Embryo defective
EMB54	1	CM	110	CLV2	19	2	Embryo defective
EMB55	=	EMB34-2	-	-	-	2	Discontinued
EMB56	4	CM	57	CER2	4	2	Embryo defective
EMB57	2	CM	18	CP2	14	2	Embryo defective
EMB58	2	CM	14	STI	9	2	Embryo defective
EMB59	5	LO	-	-	-	2	Embryo defective
EMB60	=	SUS1-3	-	-	-	2	Discontinued
EMB62	2	CM	11	STI	7	2	Embryo defective
EMB64	5	CM	71	TT3	13	2	Embryo defective
EMB65	3	CM	60	GL1	14	2	Embryo defective
EMB66	2	LO	-	-	-	2	Embryo defective
EMB67	5	CM	79	YI	5	2	Embryo defective
EMB68	5	CM	9	TTG	18	2	Embryo defective
EMB69	3	CM	73	TT5	8	2	Embryo defective
EMB70	4	AV	-	CER2	5	2	Embryo defective
EMB71	1	CM	84	CLV2	5	2	Embryo defective
EMB76	=	SUS1	-	-	-	2	Discontinued
EMB77	4	CM	59	CER2	11	2	Embryo defective
EMB78	=	FUS6	-	-	-	2	Discontinued
EMB79	1	LO	-	-	-	2	Embryo defective
EMB80	1	CM	76	CLV2	11	2	Embryo defective
EMB81	4	CM	39	CER2	15	2	Embryo defective
EMB83	1/4	LO	-	CH1/BP	5/11	2	Embryo defective
EMB84	3/4	LO	-	GL1/
				CER2	7/3	2	Embryo defective
EMB86*	5	CM	23	TTG	5	2	Embryo defective
EMB87	5	CM	85	YI	&lt;1	2	Embryo defective
EMB88	1	CM	42	DIS2	2	2	Embryo defective
EMB89	1	AV	-	DIS1	2	2	Embryo defective
EMB90	3	CM	28	HY2	17	2	Embryo defective
EMB91	2	AV	-	STI	11	2	Embryo defective
EMB93	2	CM	7	STI	2	2	Embryo defective
EMB94	2/5	LO	-	CP2/TTG	10/17	2	Embryo defective
EMB95	3	CM	24	HY2	9	2	Embryo defective
EMB101	2	CM	61	AS	2	2	Embryo defective
EMB104	4	CM	63	AP2	6	2	Embryo defective
EMB105	1	LO	-	-	-	2	Embryo defective
EMB106*	4	CM	55	CER2	2	2	Embryo defective
EMB107	4	CM	62	AP2	5	2	Embryo defective
EMB108	5	AV	-	TTG	2	2	Embryo defective
EMB109	4	CM	39	CER2	14	2	Embryo defective
EMB110	1	LO	-	-	-	2	Embryo defective
EMB111	2	CM	64	AS	3	2	Embryo defective
EMB115	4	LO	-	-	-	2	Embryo defective
EMB116	=	EMB256-
		2	-	-	-	2	Discontinued
EMB117	3	CM	84	CER7	3	2	Embryo defective
EMB118	1	CM	69	CH1	9	2	Embryo defective
EMB120	1	CM	113	CLV1	3	2	Embryo defective
EMB122	1/3	LO	-	CLV2/
				GL1	16/26	2	Embryo defective
EMB123	1/2	LO	-	CLV2/
				AS	8/2	2	Embryo defective
EMB125	3	CM	35	GL1	10	2	Embryo defective
EMB126	1	CM	49	DIS2	3	2	Embryo defective
EMB127	1	CM	29	DIS1	10	2	Embryo defective
EMB128	1	CM	76	CLV2	15	2	Embryo defective
EMB129	3/5	LO	-	GL1/YI	3/2	2	Embryo defective
EMB130	4	CM	46	CER2	7	2	Embryo defective
EMB131	1	CM	29	DIS1	10	2	Embryo defective
EMB132	4	CM	18	BP	3	2	Embryo defective
EMB133	3	CM	85	CER7	3	2	Embryo defective
EMB134	=	FUS8	-	-	-	2	Discontinued
EMB135	5	AV	-	TTG	18	2	Embryo defective
EMB137	4	CM	34	CER2	23	2	Embryo defective
EMB139	5	CM	51	TT3	6	2	Embryo defective
EMB140	4	CM	52	CER2	0	2	Embryo defective
EMB141	5	CM	56	TT3	1	2	Embryo defective
EMB142	1	CM	12	DIS1	4	2	Embryo defective
EMB143	=	COP9	-	-	-	2	Discontinued
EMB144	=	FUS9	-	-	-	2	Discontinued
EMB145	3	CM	0	HY2	12	2	Embryo defective
EMB146	2	CM	16	STI	11	2	Embryo defective
EMB148	4	CM	27	BP	12	2	Embryo defective
EMB149	3	CM	75	TT5	5	2	Embryo defective
EMB150	4	AV	-	BP	12	2	Embryo defective
EMB151	3/5	LO	-	HY2/YI	1/6	2	Embryo defective
EMB152	2	CM	47	ER	4	2	Embryo defective
EMB153	3	CM	17	HY2	5	2	Embryo defective
EMB154	3	CM	85	CER7	3	2	Embryo defective
EMB155	=	EMB106
		-2	-	-	-	2	Discontinued
EMB156*	1	CM	115	GL2	3	2	Embryo defective
EMB158	=	SUS3	-	-	-	2	Discontinued
EMB160	4	CM	20	BP	4	2	Embryo defective
EMB161*	5	CM	39	TTG	9	2	Embryo defective
EMB163	5	CM	9	TT4	3	2	Embryo defective
EMB166	5	CM	5	TTG	27	2	Embryo defective
EMB167	3	CM	63	GL1	12	2	Embryo defective
EMB168	=	COP1	-	-	-	2	Discontinued
EMB170*	5	CM	60	TT3	3	2	Embryo defective
EMB171	4	CM	27	BP	9	2	Embryo defective
EMB172	3	CM	74	TT5	4	2	Embryo defective
EMB173	1	CM	6	AN	2	2	Embryo defective
EMB175	5	CM	6	TTG	22	2	Embryo defective
EMB176	1	CM	11	DIS1	7	2	Embryo defective
EMB177	=	SUS2-1	-	-	-	2	Discontinued
EMB178	4	CM	22	BP	7	2	Embryo defective
EMB179	1	CM	99	CLV1	12	2	Embryo defective
EMB180	2/5	LO	-	ER/TT3	17/19	2	Embryo defective
EMB181	4	CM	44	CER2	8	2	Embryo defective
EMB201	3	CM	23	HY2	11	2	Embryo defective
EMB208	3	LO	-	GL1	25	2	Embryo defective
EMB209	5	CM	82	YI	5	2	Embryo defective
EMB210	4	LO	-	-	-	2	Embryo defective
EMB212	=	LEC1-1	-	-	-	2	Discontinued
EMB213	5	CM	20	TTG	8	2	Embryo defective
EMB214	=	TWN1	-	-	-	2	Discontinued
EMB215	5	CM	40	TTG	9	2	Embryo defective
EMB218	4	AV	-	CER2	10	2	Embryo defective
EMB222	5	CM	31	TTG	2	2	Embryo defective
EMB224	3	CM	30	GL1	12	2	Embryo defective
EMB225	2	CM	63	AS	5	2	Embryo defective
EMB230	4	CM	41	CER2	9	2	Embryo defective
EMB232	1	CM	27	DIS1	7	2	Embryo defective
EMB233	1	CM	3	DIS1	16	2	Embryo defective
EMB236	4	CM	33	BP	22	2	Embryo defective
EMB237	1	AV	-	DIS1	-	2	Embryo defective
EMB244	1	CM	71	CH1	13	2	Embryo defective
EMB246	5	LO	-	TT3	24	2	Embryo defective
EMB251	=	EMB161
		-2	-	-	-	2	Discontinued
EMB256*	5	CM	6	TT4	6	2	Embryo defective
EMB257	4	AV	-	CER2	5	2	Embryo defective
EMB260	1	CM	31	DIS1	14	2	Embryo defective
EMB262	5	CM	16	TT4	2	2	Embryo defective
EMB263	=	EMB86-2	-	-	-	2	Discontinued
EMB266	5	AV	-	TTG	3	2	Embryo defective
EMB270	1/4	LO	-	CH1/
				CER2	12/14	2	Embryo defective
EMB274	4	AV	-	CER2	15	2	Embryo defective
EMB277	2	AV	-	ER	6	2	Embryo defective
EMB286	3	CM	6	HY2	6	2	Embryo defective
EMB287	1	AV	-	CH1	13	2	Embryo defective
EMB293	3	AV	-	GL1	10	2	Embryo defective
EMBRG1	=	EMB28-2	-	-	-	&quot;2,74&quot;	Discontinued
EMBRG2	1	CM	97	CLV2	8	&quot;2,74&quot;	Embryo defective 
EMBRG3	1	CM	102	CLV1	6	&quot;2,74&quot;	Embryo defective 
EMBRG4	3	LO	-	GL1	23	&quot;2,74&quot;	Embryo defective 
EMBRG5	4	CM	50	CER2	2	&quot;2,74&quot;	Embryo defective 
EMBRG6	=	EMB170
		-2	-	-	-	&quot;2,74&quot;	Discontinued
EMBRG7	4	CM	9	BP	7	&quot;2,74&quot;	Embryo defective 
EMF1	5	CM	12*	G6830/
				6833	-	45	Embryonic flower
EMF2	5	CM	69*	M435/
				G4028	8/12	45	Embryonic flower
ER	2	CM	44	HY1	2	1	Erecta
ESD4	4	U	-	-	-	9	Early flowering in short days
ETO1	5	CM	83	YI	2	23	Ethylene overproducer
ETO2	5	U	-	-	-	23	Ethylene overproducer
ETR1*	1	CM	93	AP1	7	&quot;23,27&quot;	Ethylene insensitive
ETR2	3	U	TOP	-	-	27	Ethylene insensitive
FAD2	3	CM	20	G2488-9	2	6	Fatty acid desaturase deficient
FAD3	2	CM	54	ASA2	&lt;1	6	Fatty acid desaturase deficient
FAD4*	4	CM	69	CER2	17	6	Fatty acid desaturase deficient
FAD5*	3	CM	16	GL1	26	6	Fatty acid desaturase deficient
FAD6*	4	CM	54	AP2	13	6	Fatty acid desaturase deficient
FAD7*	3	CM	8	HY2	3	6	Fatty acid desaturase deficient
FADA	=	FAD4	-	-	-	6	Discontinued
FADB	=	FAD5	-	-	-	6	Discontinued
FADC	=	FAD6	-	-	-	6	Discontinued
FADD	=	FAD7	-	-	-	6	Discontinued
FAS1	1	CM	88	GL2	29	28	Fasciated (like clavata)
FAS2	5	CM	56	YI	33	28	Fasciated (like clavata)
FB	=	GI	-	-	-	-	Discontinued
FCA	4	CM	32	TH3	5	1	Late flowering
FD	4	CM	64	AP2	1	1	Late flowering
FE	1	CM	122	AP1	22	1	Late flowering
FG	=	CO	-	-	-	-	Late flowering
FHA	1	CM	12	GA4	11	1	Late flowering
FHY2	=	PHYA	-	-	-	-	Discontinued
FIL	2	CM	67*	LTP	1	58	Filamentous flower
FK	3	CM	77	TT5	4	29	Fackel (embryo defective)
FL	1	LO	-	AN	41	19	Folliculiformis (like clavata)
FLA	=	FRI	-	-	-	-	Late flowering
FLC	5	CM	8	-	-	-	Altered flower time
FLF	5	AV	TOP	-	-	46	Late flowering
FLO5	=	CLV1	-	-	-	30	Discontinued
FLO10	=	SUP	-	-	-	30	Floral defective
FLO2&gt;4	=	AP2	-	-	-	30	Discontinued
FLP	1	AM	-	NCC1/
				PVV4	3/26	78	Four lips (clustered stomata)
FPA	2	CM	83	CER8	17	1	Late flowering
FRE1	=	PHYA	-	-	-	-	Discontinued
FRI*	4	CM	0	-	-	&quot;1,31&quot;	Frigida (late flowering)
FS1*	5	CM	23	TTG	3	&quot;2,29&quot;	&quot;Fass (embryo, shoot defective)&quot;
FS2*	2	LO	-	-	-	59	&quot;Fass (embryo, shoot defective)&quot;
FT	1	CM	88	AP1	12	1	Late flowering
FUR1	4	CM	22	BP	7B	84	FUdR resistant
FUS1	=	COP1	-	-	-	32	Fusca (anthocyanin in cotyledons)
FUS2	=	DET1	-	-	-	32	Fusca (anthocyanin in cotyledons)
FUS3	3	CM	41	GL1	5	32	Fusca (anthocyanin in cotyledons)
FUS4*	5	CM	62*	DFR	0	20	Fusca (anthocyanin in cotyledons)
FUS5*	1	CM	4*	NGA59	4B	20	Fusca (anthocyanin in cotyledons)
FUS6*	3	CM	87	CER7	1	2	Fusca (anthocyanin in cotyledons)
FUS7	=	COP9	-	-	-	32	Fusca (anthocyanin in cotyledons)
FUS8*	5	CM	64	GL3	11	2	Fusca (anthocyanin in cotyledons)
FUS9*	3	CM	16	HY2	4	2	Fusca (anthocyanin in cotyledons)
FUS11*	5	CM	20*	NGA151	2B	20	Fusca (anthocyanin in cotyledons)
FUS12*	2	CM	46	ER/PY	2/5	&quot;20,32&quot;	Fusca (anthocyanin in cotyledons)
FVE	2	CM	28	PHYB	4	1	Late flowering
FWA	4	CM	54	GA5	2	1	Late flowering
FY	5	CM	12	TTG	20	1	Late flowering
GA1	4	CM	5	BP	10	1	GA requiring
GA2	1	CM	119	CLV1	9	1	GA requiring
GA3	5	CM	33	TTG	4	1	GA requiring
GA4	1	CM	22	GAI	&lt;1	1	GA requiring
GA5	4	CM	53	CER2	1	1	GA requiring
GAI	1	CM	22	GA4	&lt;1	1	GA insensitive
GF	2	LO	-	-	-	33	Female gametophytic factor
GI*	1	CM	33	GA4	11	1	Gigantea (late flowering)
GK	1	U	-	-	-	29	Gurke (embryo defective)
GL1	3	CM	46	SUP	9	1	Glabra (reduced trichomes)
GL2	1	CM	118	SUS2	2	&quot;1,2&quot;	Glabra (reduced trichomes)
GL3	5	CM	53	GA3	19	1	Glabra (reduced trichomes)
GLD1	2	U	-	-	-	6	Glycine decarboxylase deficient
GLD2	5	U	-	-	-	6	Glycine decarboxylase deficient
GN	=	EMB30	-	-	-	29	Gnom (embryo defective)
GR2	5	CM	71	GL3	17	19	Grisea (gray-green leaf color)
GRL	2	CM	60	AS	-	29	Gnarled (abnormal trichomes)
GUN1	2	CM	42*	GPA1	3	24	Genomes uncoupled
HCF107	3	AM	-	NGA162	-	94	High chlorophyll fluorescent
HCF109	5	CM	48*	PHYC	0	94	High chlorophyll fluorescent
HLS1*	4	AV	-	CER2	9	23	Hookless (ethylene response)
HLS2	4	AV	-	CER2	13	23	Hookless (ethylene response)
HLS3*	2	AM	-	M605	4A	23	Hookless (ethylene response)
HPT	=	AMP1	-	-	-	29	Hauptling (extra cotyledons)
HXC1	5	CM	9*	ASA1	&lt;1	&quot;80,81&quot;	Hypersens. Xanth. camp. pv. camp.
HY1	2	CM	44	ER	2	1	Elongated hypocotyl in light
HY2	3	CM	12	FAD7	3	6	Elongated hypocotyl in light
HY3	=	PHYB	-	-	-	1	Elongated hypocotyl in light
HY4*	4	CM	10	FCA	18	1	Elongated hypocotyl in light
HY5	5	CM	8	MS1	15	1	Elongated hypocotyl in light
HY8	=	PHYA	-	-	-	-	Discontinued
IAE1	2	CM	25*	PR1/
				MI398	0	31	Increased Ac excision
IAR1	1	CM	105*	NGA111	3A	47	IAA-alanine resistant
IAR2	5	AM	-	NGA76	-	47	IAA-alanine resistant
ICR1	3	AV	-	TT6-GL1	-	11	IAA conjugate resistant
ILR1	3	CM	1*	NGA172	1A	47	IAA-leucine resistant
IM	4	CM	48	CER2	4	1	Immutans (variegated)
INO	1	AM	-	NGA248/
				63	6/14	55	&quot;Inner, no outer (ovule defective)&quot;
KE21	3	U	TOP	-	-	63	Altered starch in leaves
KE33	1	U	TOP	-	-	63	Altered starch in leaves
KE81	3	U	BOT	-	-	63	Altered starch in leaves
KEU	1	CM	14	DIS1	5	29	Keule (embryo defective)
KIM	2	U	-	-	-	58	Kimagure (root hair defective)
KLK	5	LO	-	TTG	-	29	Klunker (embryo defective)
KN	1	AV	-	DIS1	10	29	Knolle (embryo defective)
LAD	5	CM	12*	-	-	95	Late anther dehiscence
LAN	2	CM	50	-	-	-	Lanceolate (narrow leaves)
LE*	1	CM	66	CH1	8	1	Lepida (dwarf)
LEC1*	1	CM	29	DIS1	9	2	Leafy cotyledon
LEC2	1	LO	-	-	-	2	Leafy cotyledon
LFY	5	CM	82	TZ	9	30	Leafy (floral defective)
LIT	5	CM	72*	DFR/LFY	19/20	43	Lions tail (root defective)
LU	5	CM	10	TT4	2	1	Lutescens (yellow-green)
MIN	5	CM	90	ABA1	7	1	Miniature (semi-dwarf)
MP	1	CM	22	AN	36	29	Monopteros (embryo defective)
MS1	5	CM	23	TTG	6	1	Male sterile
MS2	3	CM	27	-	-	-	Male sterile
MS31	1	U	-	-	-	48	Male sterile
MS32	1	U	-	-	-	48	Male sterile
MS33	2	U	-	-	-	48	Male sterile
MS34	2	U	-	-	-	48	Male sterile
MS35	3	U	-	-	-	48	Male sterile
MS36	3	U	-	-	-	48	Male sterile
MS37	5	U	-	-	-	48	Male sterile
MS38	5	U	-	-	-	48	Male sterile
MTO1	3	CM	19	HY2/
				ABI3	8/22	52	Methionine over accumulation
NDR1	3	AM	-	560B1	-	65	Non race-specific disease resistance
NIA1	1	CM	116	GL2	3	18	Nitrate reductase defective
NIA2	=	CHL3	-	-	-	18	Nitrate reductase defective
NIM1	1	CM	95*	NGA128/111	17/17	&quot;88,89&quot;	Non-inducible immunity
NPH1	3	CM	61	GL1	15B	91	Non-phototrophing hypocotyl
OMR1	3	CM	17	HY2	7	&quot;68,84&quot;	L-O-methylthreonine resistant
PA	=	LE	-	-	-	-	Discontinued
PDS1	1	CM	26	DIS1	7	2	Phytoene desaturation (ABA deficient)
PDS2	3	CM	4	HY2	8	2	Phytoene desaturation (ABA deficient)
PFL	1	CM	32	TH1	1	49	Pointed first leaves
PGM	5	CM	63	YI	22	1	Phosphoglucomutase deficient
PHO1	3	CM	27	G4708	3	6	Low inorganic phosphate
PHYA*	1	CM	16	-	-	-	Phytochrome defective
PHYB*	2	CM	31	FVE	4	1	Phytochrome defective
PI	5	CM	21	TTG	10	1	Pistillata (floral defective)
PID	2	CM	58	AS	-	21	Pinoid (meristem defective)
PIN1	1	CM	103	CRC	3B	21	Pin-formed (floral defective)
PLU	5	LO	-	-	-	82	Pluto (root defective)
POM1	1	CM	9*	NGA63	1	43	Pom-pon (root defective)
POM2	2	CM	45*	GPA1	4B	43	Pom-pon (root defective)
POP1	=	CER6	-	-	-	53	Pollen-pistil incompatibility
POP2	3	U	-	-	-	53	Pollen-pistil incompatibility
POP3	5	U	-	-	-	53	Pollen-pistil incompatibility
PRC1	5	CM	86	YI	&lt;1B	59	&quot;Procuste (short, deformed hypocotyl)&quot;
PT	=	AMP1	-	-	-	10	Primordia timing
PTL	5	CM	1	TFL1	1A	21	Petaless; petal loss
PY	2	CM	49	HY1	5	1	Pyrimidine requiring
QRT1	5	AM	-	DFR/LFY	-	53	Quartet
QRT2	3	CM	2*	GAPC	&lt;2	53	Quartet
QUI	1	AM	-	PVV4	8	43	Quill (root defective)
RAC1*	1	CM	51	M229-
				M335	-	34	Resistant to Albugo candida
RDS1	2	U	-	-	-	50	Regulation of desaturases in seeds
RE	2	CM	59	ER	19	1	Reticulata (dark veins)
REV	5	CM	85*	LFY/
				NGA129	6/21	56	Revolute leaf
RGN*	1	CM	1	AN	1	1	Chlorate resistant
RGR	=	AXR4	-	-	-	-	Reduced root gravitropism
RHD1	1	LO	-	AP1	-	35	Root hair defective
RHD2	5	LO	-	TT3	-	35	Root hair defective
RHD3	3	LO	-	GL1	-	35	Root hair defective
RHD4	3	LO	-	GL1	-	35	Root hair defective
RML1	4	CM	50*	MI422	&lt;1	45	Root meristemless
RML2	3	AM	-	M457	14	45	Root meristemless
RPM1	3	CM	7*	M583/
				G4523	2/2	36	Resistance to P. syringae pv maculicola
RPP1*	3	CM	52	GL1	-	34	Recognition of Peronospora parasitica
RPP2	4	CM	45	M600/
				M557	-	34	Recognition of Peronospora parasitica
RPP4	4	CM	38	-	-	34	Recognition of Peronospora parasitica
RPP5	4	CM	37	-	-	34	Recognition of Peronospora parasitica
RPP6	2	LO	TOP	-	-	34	Recognition of Peronospora parasitica
RPP7	1	CM	69	-	-	34	Recognition of Peronospora parasitica
RPP8	5	CM	58	-	-	34	Recognition of Peronospora parasitica
RPP9	=	RAC1	-	-	-	34	Recognition of Peronospora parasitica
RPP10	=	RPP1	-	-	-	34	Recognition of Peronospora parasitica
RPP11	3	CM	64	-	-	34	Recognition of Peronospora parasitica
RPP12	4	AM	-	AG/M557	-	34	Recognition of Peronospora parasitica
RPP13	3	CM	56*	M249	3A	34	Recognition of Peronospora parasitica
RPS2	4	CM	55	-	-	-	Resistant to P. syringae
RPS4	5	AM	-	DFR	-	65	Resistant to P. syringae
RPS5	1	CM	11*	ATTS
				0477	0	77	Resistant to P. syringae
RPT1	3	CM	62*	BGL1	19A	58	Root phototropism
RPW1	2	LO	-	-	-	79	Recognition of powdery mildew
RPW5	5	AM	-	NGA225	-	79	Recognition of powdery mildew
RPW6	5	AM	-	NGA158/
				139	-	79	Recognition of powdery mildew
RTN	2	CM	46*	ER-M220	-	51	Reduced trichome number
RTY	=	HLS3	-	-	-	76	Rooty
RTY3	=	HLS3	-	-	-	75	Rooty
RXC1	2	CM	56*	M323	5A	79	Resist.  to X. campestris pv amoraciae
SAB	1	CM	88*	G4026	4	43	Sabre (root defective)
SAR1	1	U	MID	-	-	15	Suppressor of auxin resistance
SEX1*	1	CM	13	AN	12	37	Starch excess
SHL	=	STM	-	-	-	69	Shootless
SHY2	1	AM	TOP	NCC1	-	61	Short hypocotyl
SMA4	1	CM	92*	NGA280	13B	77	Symptoms mult. avirulence genotypes
SOP	=	SEX1	-	-	-	37	Discontinued
SPT	4	CM	68	AP2	-	21	Spatula (floral defective)
SPY	3	CM	12	HY2	0	75	Spindly (altered GA transduction)
STI	2	CM	4	EMB93	2	2	Stichel (unbranced trichomes)
STM	1	CM	75	CH1	15B	69	Shoot meristemless
STP1	2	LO	-	NGA168	30A	87	Stunted plant
SU	5	CM	35	TTG	7	1	Suppressor of CHL4
SUL	2	CM	70	AS	10	1	Sulfurata
SUP*	3	CM	35	AXR2	1	&quot;8,30&quot;	Superman (also FLO10)
SUR	2	CM	31	PHYB	&lt;1	59	Super root
SUS1*	1	CM	0	AN	1	2	Abnormal suspensor
SUS2*	1	CM	116	GL2	2	2	Abnormal suspensor
SUS3*	1	CM	86	CLV2	2	2	Abnormal suspensor
TAG1	2	CM	24	CP2	8	50	Triacylglycerol biosynthesis defect
TED2	1	CM	45*	NGA248	5B	24	Suppressor of DET
TFL1	5	CM	2*	M447/
				TT4	1/15	26	Terminal flower
TFL2	5	CM	15*	TFL1/
				EMF1	-	26	Terminal flower
TH1	1	CM	33	GI	0	1	Thiamine requiring
TH2	5	CM	47	TT2	1	1	Thiamine requiring
TH3	4	CM	34	FCA	5	1	Thiamine requiring
TIR1	3	CM	87*	NGA112	2B	15	Auxin transport inhibitor resistant
TIR2	1	AM	BOT	-	-	15	Auxin transport inhibitor resistant
TIR3	3	AM	TOP	-	-	15	Auxin transport inhibitor resistant
TIR4	1	AM	BOT	-	-	15	Auxin transport inhibitor resistant
TIR5	5	AM	TOP	-	-	15	Auxin transport inhibitor resistant
TIR6	4	AM	TOP	-	-	15	Auxin transport inhibitor resistant
TIR7	5	AM	TOP	-	-	15	Auxin transport inhibitor resistant
TMM	1	AM	-	ADH/
				PAB5	9/18	78	Too many mouths (stomata)
TNY	5	CM	40*	TN139	0	9	Tiny
TOC1	5	CM	79*	EG7F2	5B	93	Timing of CAB1
TOM1	4	AM	BOT	JGB9	-	52	Tobamovirus multiplication
TON1	=	FS1	-	-	-	59	Tonneau 
TON2	=	FS2	-	-	-	59	Tonneau
TOR1	4	CM	58	-	-	13	Tortifolia (twisted petioles)
TRP1	5	LO	-	TT4	-	38	Tryptophan biosynthesis
TRP2	5	AM	-	TSB1	0	38	Tryptophan biosynthesis
TRP3	3	CM	90*	TSA1	0	38	Tryptophan biosynthesis
TRP4	1	CM	35	CHL1	20B	14	Tryptophan biosynthesis
TRP5	5	CM	10*	ASA1	0	14	Tryptophan biosynthesis
TSL	5	CM	20	-	-	-	Tousled (floral defective)
TT1	1	CM	55	CH1	4	1	Transparent testa
TT2	5	CM	44	TH2	1	1	Transparent testa
TT3	5	CM	57	EMB9	2	&quot;1,2&quot;	Transparent testa
TT4	5	CM	13	LU	2	1	Transparent testa
TT5	3	CM	81	CH6	3	1	Transparent testa
TT6	3	CM	96	-	-	&quot;1,2&quot;	Transparent testa
TT7	5	CM	6	ALB2	10	1	Transparent testa
TT8	4	CM	22	BP	7B	92	Transparent testa
TT9	3	AV	-	GL1	5	92	Transparent testa
TT10	5	CM	69	CH5	24B	92	Transparent testa
TTG	5	CM	28	GA3	4	1	&quot;Transparent testa, glabra&quot;
TWN1*	5	LO	-	YI-TT3	-	2	Twin (embryo defective)
TWN2	1	CM	19*	M219/
				G12080	-	6	Twin (embryo defective)
TZ	5	CM	77	BIO1	4	&quot;1,2&quot;	Thiazole requiring
UFO	1	CM	46	-	-	-	Unusual floral organs
UVH1	5	AM	-	NGA76	-	39	Ultraviolet hypersensitive
UVH3	3	AV	-	GL1	&lt;1	39	Ultraviolet hypersensitive
UVH6	1	AM	-	NGA59	-	39	Ultraviolet hypersensitive
UVR1	3	CM	53	-	-	70	UV repair defective
VAL1	3	CM	59	TT5	21A	84	Valine resistant
VAR1	5	CM	65	CER3	18	40	Variegated
VAR2	2	CM	71	ER	23	40	Variegated
VR2	2	LO	-	-	-	33	Virescens
VRN1	3	CM	17*	MI339	3	31	Reduced vernalization response
VRN2	4	AM	-	NGA8/
				MI422	-	31	Reduced vernalization response
WAM	1	CM	89	AP1	10A	90	Waldmeister (distorted shoot)
WAV5	=	AUX1	-	-	-	58	Wavy roots
XV	1	LO	-	-	-	33	Discontinued
YAK	1	U	-	-	-	21	Yakka (disrupted inflorescence)
YI	5	CM	85	EMB15	&lt;1	&quot;1,2&quot;	Yellow inflorescence
</TT></FONT><HR>
<TT></TT></PRE>
</BODY></HTML>
