<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="BMC Bioinformatics"><meta name="citation_title" content="SPdb &#x02013; a signal peptide database"><meta name="citation_authors" content="Khar Heng Choo, Tin Wee Tan, and  Shoba Ranganathan"><meta name="citation_date" content="2005"><meta name="citation_volume" content="6"><meta name="citation_firstpage" content="249"><meta name="citation_doi" content="10.1186/1471-2105-6-249"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1276010&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1276010&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1276010"><meta name="citation_pmid" content="16221310"><meta name="DC.Title" content="SPdb &#x02013; a signal peptide database"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="BioMed Central"><meta name="DC.Contributor" content="Khar Heng Choo"><meta name="DC.Contributor" content="Tin Wee Tan"><meta name="DC.Contributor" content="Shoba Ranganathan"><meta name="DC.Date" content="2005"><meta name="DC.Identifier" content="10.1186/1471-2105-6-249"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-slateblue.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody4.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs3.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>SPdb &#x02013; a signal peptide database</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0bg7Nq8vGoSAUeZS3mNCUv1T9MpzDlZ0HcKF1kkoc&amp;reftype=other&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/logo-bmcbioi.gif" alt="Logo of bmcbioi" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="rect" alt="BioMed Central web site" coords="0,0,99,74" onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0LqHJi58-bB_l9L0-QeCrZ7dIUdYikl3WYJYQdQd_&amp;reftype=publisher&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com" target="pmc_ext"><area shape="rect" coords="100,54,172,74" alt="Reference to the article." onclick="focuswin('pmc_ext') " href="redirect3.cgi?&amp;&amp;auth=06Q3BuQONMglQt-4xVhj2FwSsE8PWftCetp6QTA2I&amp;reftype=publisher&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7CArticle%7CFullText&amp;rendering-type=normal&amp;&amp;http://dx.doi.org/10.1186/1471-2105-6-249" target="pmc_ext"><area shape="rect" coords="173,54,223,74" alt="Search." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=02ku_k4wqtpJAYBe0avowqp_jZe47UJCSZ0WreQHI&amp;reftype=publisher&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/search/bmcbioinformatics" target="pmc_ext"><area shape="rect" coords="224,55,342,74" alt="Manuscript submission." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0LWWggLHDoPgSVWd1RPLNNYzgt5dzFLgKm_MBS5Uq&amp;reftype=publisher&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/manuscript/bmcbioinformatics" target="pmc_ext"><area shape="rect" coords="343,53,480,74" alt="Registration." onclick="focuswin('pmc_ext')" href="redirect3.cgi?&amp;&amp;auth=0hq1KQ44pJYOPsxBwamfPBJnLvVFIzHMUXqbd8k6Y&amp;reftype=publisher&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/registration" target="pmc_ext"><area shape="rect" coords="0,0,480,74" alt="Journal front page." onclick="focuswin('pmc_ext') " href="redirect3.cgi?&amp;&amp;auth=0fSUYJ1T6YPF8a1UxGJmjuwlpGQaptwfa-NjEWQ-X&amp;reftype=publisher&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.biomedcentral.com/bmcbioinformatics" target="pmc_ext"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=13&amp;action=archive">BMC Bioinformatics</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=18092">v.6; 2005</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=1276010&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=1276010&amp;blobtype=pdf" class="sidefm-pmclink">PDF (600K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=18092" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=13&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=083Ey4-_7KFmsKsLFS5L4WBbwfwJEQT80ZEED2kU-&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/16221310">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=0JYQxfZXUuXwDQ33lDs8sIsTywnWOc9qKS6DXtQMe&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=16221310">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0x9ky1fEsKdq6-aZOFaIaBwOMfmBpiYi17qlXj58A&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=16221310">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0TWsH7NvKnqF2N2iK4JO9v6IspOmrzR_sF9uCHAjN&amp;reftype=Nucleotide&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Nucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1276010&amp;dopt=pmc_nuccore">Nucleotide</option><option value="redirect3.cgi?&amp;&amp;auth=0hicONFyPoUDLohWvdK6pTx8z75ispW0EyoTbPAsO&amp;reftype=Substance&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|PubChem%20Substance&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1276010&amp;dopt=pmc_pcsubstance">Substance</option><option value="redirect3.cgi?&amp;&amp;auth=0V72-Iu7rIz1e5XDxN4ChdWdUq_JwbVSPaH1CiAVS&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=1276010&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0FcH2JIdmR_Jr8OVGVGUaLf6l_X9n_gMhnszxKAHJ&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=1276010">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0LKe55RrV6s2kazTBEYpVSPtB42fPF7fdzWEHZ5dE&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Choo+KH%5Bauth%5D" class="sidefm-pmclink"> Choo, K.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=06MIVZTqXXVd9ik9Owrp_s2tUhnXM4UL0HEBPjQI9&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Tan+TW%5Bauth%5D" class="sidefm-pmclink"> Tan, T.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0hkxhzQmh-nYEsI-djA8cUuleiRjA7kkPapChZA5e&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Ranganathan+S%5Bauth%5D" class="sidefm-pmclink"> Ranganathan, S.</a></div></div></div></td><td rowspan="106" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-bmcbioi.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">BMC Bioinformatics. </span><span class="citation-publication-date">2005; </span><span class="citation-volume">6</span><span class="citation-issue"></span><span class="citation-flpages">: 249. </span></div><div><span class="fm-vol-iss-date">Published online 2005 October 13. </span><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1186/1471-2105-6-249.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC1276010</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 2005 Choo et al; licensee BioMed Central Ltd.</div><div class="fm-title">SPdb &#x02013; a signal peptide database</div><div class="fm-author contrib-group">Khar Heng Choo,<sup>1</sup> Tin Wee Tan,<sup>1</sup> and  Shoba Ranganathan<sup><img src="corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup><sup>1,</sup><sup>2</sup><div class="fm-affl"><sup>1</sup>Department of Biochemistry, National University of Singapore, Singapore</div><div class="fm-affl"><sup>2</sup>Department of Chemistry and Biomolecular Sciences &#x00026; Biotechnology Research Institute, Macquarie University, Sydney, Australia</div></div><div class="fm-footnote"><sup><img src="corehtml/pmc/pmcgifs/corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><div class="fm-footnote"><span style="white-space: nowrap;" id="id500389">Khar Heng Choo: <span class="e_id500389">justin/at/bic.nus.edu.sg</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id500389", '<a class="ext-reflink" href="' + reverseAndReplaceString('gs.ude.sun.cib/ta/nitsuj:otliam', '/at/', '@') + '">' + reverseAndReplaceString('gs.ude.sun.cib/ta/nitsuj', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id561452">Tin Wee Tan: <span class="e_id561452">tinwee/at/bic.nus.edu.sg</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id561452", '<a class="ext-reflink" href="' + reverseAndReplaceString('gs.ude.sun.cib/ta/eewnit:otliam', '/at/', '@') + '">' + reverseAndReplaceString('gs.ude.sun.cib/ta/eewnit', '/at/','@') + '</a>')}catch(e){}
                                //--></script>; </span> <span style="white-space: nowrap;" id="id527887">Shoba Ranganathan: <span class="e_id527887">shoba/at/els.mq.edu.au</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id527887", '<a class="ext-reflink" href="' + reverseAndReplaceString('ua.ude.qm.sle/ta/abohs:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ua.ude.qm.sle/ta/abohs', '/at/','@') + '</a>')}catch(e){}
                                //--></script></span> </div><div class="fm-pubdate">Received April 15, 2005; Accepted October 13, 2005.</div><div class="fm-copyright"><p>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0HkokihommSLjQ1A0d0ZggzQOhEXULM3gb3-NkQwg&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://creativecommons.org/licenses/by/2.0" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://creativecommons.org/licenses/by/2.0</a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="links-box"><div class="fm-footnote"><img src="corehtml/pmc/pmcgifs/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="tocrender.fcgi?artid=1276010&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id539915" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id585752" class="sidefm-pmclink" style="text-transform: none;">Construction and content</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586423" class="sidefm-pmclink" style="text-transform: none;">Utility and discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586664" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586677" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586693" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586754" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id506740">Abstract</div><div class="section-content"><!--article-meta--><p><div class="abs-head2 head-separate">Background</div>The signal peptide plays an important role in protein targeting and protein translocation in both prokaryotic and eukaryotic cells. This transient, short peptide sequence functions like a postal address on an envelope by targeting proteins for secretion or for transfer to specific organelles for further processing. Understanding how signal peptides function is crucial in predicting where proteins are translocated. To support this understanding, we present SPdb signal peptide database <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0pe7gZN6npTYyVbrC7rD1MG1UIrQdK9QKC0z6jklb&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://proline.bic.nus.edu.sg/spdb" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://proline.bic.nus.edu.sg/spdb</a></span>, a repository of experimentally determined and computationally predicted signal peptides.</p><p><div class="abs-head2 head-separate">Results</div>SPdb integrates information from two sources (a) Swiss-Prot protein sequence database which is now part of UniProt and (b) EMBL nucleotide sequence database. The database update is semi-automated with human checking and verification of the data to ensure the correctness of the data stored. The latest release SPdb release 3.2 contains 18,146 entries of which 2,584 entries are experimentally verified signal sequences; the remaining 15,562 entries are either signal sequences that fail to meet our filtering criteria or entries that contain unverified signal sequences.</p><p><div class="abs-head2 head-separate">Conclusion</div>SPdb is a manually curated database constructed to support the understanding and analysis of signal peptides. SPdb tracks the major updates of the two underlying primary databases thereby ensuring that its information remains up-to-date.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id506740" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Background</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id585752" class="sidefm-pmclink" style="text-transform: none;">Construction and content</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586423" class="sidefm-pmclink" style="text-transform: none;">Utility and discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586664" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586677" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586693" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586754" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id539915">Background</div><div class="section-content"><p>G&#x000fc;nter Blobel discovered that "proteins have intrinsic signals that govern their transport and localization in the cell" [<a href="#B1" class="cite-reflink">1</a>]. Proteins synthesised at the ribosome (cytoplasm or rough endoplasmic reticulum), mitochondria or chloroplast are transported to their site of function. This process is known as protein targeting and it depends on targeting signals to direct the proteins to their specific locations.</p><p>There are many different classes of targeting signals. One of the commonly occurring signals is formed by short, transient peptides known as signal peptides or leader sequences, which are usually found at the amino terminus of secreted proteins. Signal peptides are present in both prokaryotic and eukaryotic cells, indicating its ancient universal origins. They function like a postal address label on an envelope by targeting the proteins for secretion or to specific organelle for further processing. The signal peptides are cleaved off and degraded upon reaching their targeted locations. Interestingly, not all proteins possess signal peptides [<a href="#B2" class="cite-reflink">2</a>,<a href="#B3" class="cite-reflink">3</a>], suggesting that other mechanisms for protein targeting exist.</p><p>Over the years, several prediction tools [<a href="#B4" class="cite-reflink">4</a>-<a href="#B10" class="cite-reflink">10</a>] have been developed to predict the cleavage sites of signal peptides. These prediction tools require training and testing datasets. As a preparatory step for prediction work, researchers often devote considerable time sifting through primary databases such as Swiss-Prot [<a href="#B11" class="cite-reflink">11</a>], EMBL [<a href="#B12" class="cite-reflink">12</a>] and other databases to collate and construct their own datasets. This repetitive process can and should be eliminated by creating a centralised repository of signal peptide sequences.</p><p>Searching through popular search engines and reviewing the Nucleic Acids Research database list [<a href="#B13" class="cite-reflink">13</a>] reveal several databases that provide information on protein subcellular localisation [<a href="#B14" class="cite-reflink">14</a>,<a href="#B15" class="cite-reflink">15</a>], nuclear proteins [<a href="#B16" class="cite-reflink">16</a>] and secreted proteins [<a href="#B17" class="cite-reflink">17</a>]. These databases do not provide signal peptide specific information except for SPD [<a href="#B17" class="cite-reflink">17</a>]. SPD or secreted protein database [<a href="#B17" class="cite-reflink">17</a>] is a collection of proteins from the human, mouse and rat proteomes originating from databases such as TrEMBL [<a href="#B18" class="cite-reflink">18</a>], Ensembl [<a href="#B19" class="cite-reflink">19</a>] and Refseq [<a href="#B20" class="cite-reflink">20</a>]. It also includes datasets from the Secreted Protein Discovery Initiative (SPDI) [<a href="#B21" class="cite-reflink">21</a>], a large-scale effort to identify novel human secreted and transmembrane proteins; the Riken mouse secretome and seven other related datasets [<a href="#B22" class="cite-reflink">22</a>]. SPD aims to be a comprehensive repository for secreted proteins, but it suffers from providing datasets that may still contain many erroneous annotations from its underlying data sources for example TrEMBL. TrEMBL is generated from an automated pipeline and has yet to undergo manual curation. In addition, the entries in SPD were not checked manually against the publications. Then, there is also the issue that the datasets are not being updated.</p><p>Besides the SPD which offer downloadable datasets, there are sites that offer downloadable datasets namely the SignalP datasets (1997) [<a href="#B10" class="cite-reflink">10</a>,<a href="#B23" class="cite-reflink">23</a>] and the datasets (2000) used by Meene <em>et al</em>. [<a href="#B24" class="cite-reflink">24</a>,<a href="#B25" class="cite-reflink">25</a>] in their evaluation of signal peptide prediction methods. More recently, there is the dataset consisting of 270 secreted recombinant human proteins with experimentally determined cleavage sites from Zhang and Henzel [<a href="#B26" class="cite-reflink">26</a>,<a href="#B27" class="cite-reflink">27</a>]. These datasets are often either limited in size or otherwise lacking in tools for querying the datasets. Moreover, these datasets although valuable but they are often outdated [<a href="#B28" class="cite-reflink">28</a>] especially when GenBank/EMBL, Swiss-Prot and other publicly accessible primary databases continue to churn out new entries or sequences.</p><p>Many researchers are confronted by similar obstacles in accessing up-to-date data, which are withheld from public access by method developers [<a href="#B29" class="cite-reflink">29</a>,<a href="#B30" class="cite-reflink">30</a>] and hence, we strongly believe that there is a urgent need to provide a publicly-accessible, manually curated and regularly updated database specialised for signal peptides. These datasets will not only be important for prediction work but they will also serve as the common datasets needed when researchers are performing benchmarking against each other methods or programs, without which we think it is difficult to perform proper or fair benchmarking of the multitudes of prediction methods.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id506740" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id539915" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Construction and content</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586423" class="sidefm-pmclink" style="text-transform: none;">Utility and discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586664" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586677" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586693" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586754" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id585752">Construction and content</div><div class="section-content"><p><div class="head2 head-separate">Construction and implementation of database</div>Recognizing the need for a curated, specialised and up-to-date database, we have developed a composite signal peptide database SPdb [<a href="#B31" class="cite-reflink">31</a>] that offers researchers a singular point for depositing their signal peptide annotations. SPdb integrates information from Swiss-Prot (part of UniProt) [<a href="#B18" class="cite-reflink">18</a>] and EMBL. It is updated when there is a major release of Swiss-Prot. First released in May 2004, SPdb was upgraded to release 3.2 recently to add on new features and to synchronise with the release of its underlying data sources.<p>SPdb is a relational database built using MySQL database management system [<a href="#B32" class="cite-reflink">32</a>] and using PERL/CGI [<a href="#B33" class="cite-reflink">33</a>] for processing web forms. An easy-to-navigate web interface was built to allow user to search through the database. Some of the web features were added in response to the requests by some of the users that have used our database since its inception. Through the web interface, users are able to download the returned results as FASTA formatted files or view the results as HTML web page. We have also provided a link from the search page to the Swiss-Prot ID tracker to verify whether an entry has been renamed e.g. ANL3_HUMAN from the Zhang and Henzel dataset [<a href="#B27" class="cite-reflink">27</a>], which is now ANGL3_HUMAN.</p><p>We deployed the bioinformatics pipeline shown in (Figure <a href="articlerender.fcgi?artid=1276010&amp;rendertype=figure&amp;id=F1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">1</a>) to construct SPdb. The pipeline to construct the database was semi-automated with specific checkpoints for manual checking of the results to minimise errors in the database.</p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F1" name="F1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1276010&amp;rendertype=figure&amp;id=F1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1276010&amp;blobname=1471-2105-6-249-1.gif" border="1" class="icon-reflink" alt="Figure 1" title="Figure 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1276010&amp;rendertype=figure&amp;id=F1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1</strong></a><div class="figure-table-caption-in-article"><span>Schematic diagram of the construction pipeline of SPdb.</span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><div class="head2 head-separate">Construction method</div>Signal sequences and coding sequences obtained from Swiss-Prot (TrEMBL entries were not taken into consideration) were filtered initially using the data extraction and redundancy reduction methodology proposed by Nielsen <em>et al</em>. [<a href="#B34" class="cite-reflink">34</a>] to segregate the dataset into two sets (a) the <em>preliminary filtered </em>set and (b) the <em>unverified sequences </em>set. The Nielsen <em>et al</em>. [<a href="#B34" class="cite-reflink">34</a>] methodology has been employed to generate the training and testing data used in SignalP [<a href="#B10" class="cite-reflink">10</a>,<a href="#B35" class="cite-reflink">35</a>]. We adapted and omitted some of the criteria proposed by the method since our goal is to build a repository of signal peptides with as many relevant and accurate entries as possible. We observed that the proposed methodology still renders many undesirable entries upon the filtering process. Thus, we have constructed SPdb by building on the strength of the proposed methodology [<a href="#B34" class="cite-reflink">34</a>] and improved it with our own criteria and filtering rules.<p>Any entries with the SIGNAL keyword indicated in the feature table FT field [<a href="#B36" class="cite-reflink">36</a>] of Swiss-Prot entries were presumed to contain information on signal sequence. This simple selection process yielded 18,146 entries out of the total 170,140 Swiss-Prot entries (Release 46.1). Entries that connoted uncertainty namely those with annotations like PROBABLE, POTENTIAL, BY SIMILARITY, HYPOTHETICAL and entries with ambiguous cleavage or signal peptide positions were tagged as <em>unverified sequences</em>. Then, entries with signal sequences length less than 11 were relegated to the <em>unverified sequences </em>set. Signal sequences are generally considered to be of length 15 to 40. This initial step filtered off 13,701 entries from the <em>preliminary filtered </em>set leaving behind 4,445 entries. These entries include type I signal peptides, type II signal peptides (lipoproteins) and TAT-containing signal peptides. Using the SIGNAL keyword, mitochondria and chloroplast transit peptides were excluded from the <em>preliminary set </em>since transit peptides are identified by the TRANSIT keyword in Swiss-Prot.</p><p>We proceeded to integrate information from the EMBL database. By integrating complementary information, besides providing extra information not found in Swiss-Prot, we could use the information from EMBL to cross check against Swiss-Prot, allowing us to discover erroneous annotations. This practice of using complementary information from other data sources has been found useful in data evaluation [<a href="#B37" class="cite-reflink">37</a>].</p><p>The first cross-reference entry to EMBL database was used for the respective Swiss-Prot entry. Based on the data categorisation of EMBL found in its release note [<a href="#B38" class="cite-reflink">38</a>], only sequences from the data groups fungi, human, invertebrate, mouse, organelle, bacteriophage, plant, prokaryote, rodent, viral, mammals and vertebrate were taken into consideration. Entries belonging to the data groups expressed sequence tags, genome survey sequences, high-throughput genome sequences, unfinished DNA sequences generated by high-throughput sequencing, patent sequences, synthetic sequences, contig sequences and unclassified were omitted. We extracted out relevant annotations from EMBL whenever available including coding sequence, signal sequence and its length, subcellular location, authors' notes and so on.</p><p>The annotations, specifically the <em>sig_region </em>and <em>misc </em>fields from the EMBL entry were utilised in the subsequent step to cross-check against the <em>preliminary filtered </em>entries. This step again filtered out many inconsistent entries where the positions are quoted wrongly by either source e.g. [Swiss-Prot:CD166_CHICK] where Swiss-Prot quoted cleavage position of 33 while EMBL provided 32. As a result, another 866 entries were eliminated to retain 3,579 entries in this newly filtered <em>Swiss-Prot/EMBL </em>set. It must be noted that there were some Swiss-Prot entries in the <em>Swiss-Prot/EMBL </em>set without any EMBL reference e.g. [Swiss-Prot:APOE_CAVPO]; or with insufficient annotations in the EMBL entries e.g. [Swiss-Prot:17KD_RICAU]; or their EMBL cross-references were indicated with annotation such as NOT_ANNOTATED_CDS e.g. [Swiss-Prot:2B31_HUMAN], ALT_TERM e.g. [Swiss-Prot:CD1E_HUMAN], ALT_INIT e.g. [Swiss-Prot:1A03_PANTR] and ALT_SEQ e.g. [Swiss-Prot:17KD_RICPR]. In these cases, all these entries were earmarked for manual curation. These terms "NOT_ANNOTATED_CDS", "ALT_TERM" and so on are known as <em>status identifiers </em>and they are found at the DR field in Swiss-Prot entries. The reader is referred to the detailed explanation found in the Swiss-Prot manual [<a href="#B39" class="cite-reflink">39</a>].</p><p>Following this step, all the entries in this <em>Swiss-Prot/EMBL </em>set are manually checked against the referred publications. We located numerous entries with discrepancies on signal peptide cleavage site between the Swiss-Prot annotations and the accompanying papers e.g. [Swiss-Prot:CECC_DROME, Swiss-Prot:AMCY_PARVE]. Entries that we do not have access to the accompanying papers e.g. [Swiss-Prot:ZEAL_MAIZE, Swiss-Prot:ZEA6_MAIZE] or those entries that we could not locate their cleavage site information in the papers e.g. [Swiss-Prot:GUX1_TRIRE]; these entries in addition to those entries which are inadequately labelled or entries with inconsistent positional information were all relegated to the <em>unverified sequences </em>set. In this manual curation step, we eliminated 995 entries from the <em>Swiss-Prot/EMBL </em>set of 3,579 entries. These 995 entries were entries that (a) both Swiss-Prot and the quoted papers provided the same putative position (b) we found differing positions quoted by Swiss-Prot as compared to the quoted papers (c) we did not have access to the quoted subscription-only papers or the papers referred to were old and in some cases there were no paper or no relevant paper quoted (d) we could not find or locate the cleavage site information (Table <a href="articlerender.fcgi?artid=1276010&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">1</a>).</p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T1" name="T1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1276010&amp;rendertype=table&amp;id=T1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 1" title="Table 1"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1276010&amp;rendertype=table&amp;id=T1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 1</strong></a><div class="figure-table-caption-in-article"><span>Distribution of the signal sequences filtered out in the manual curation step</span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><div class="head2 head-separate">Content of database</div>The result from filtering and manually curating the entries culminated in the latest release of SPdb release 3.2, with a total of 18,146 signal sequence entries, out of which 2,584 are filtered sequences (Table <a href="articlerender.fcgi?artid=1276010&amp;rendertype=table&amp;id=T2" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">2</a>). These filtered sequences, known as the <em>filtered sequences </em>set include the mature endogenous proteins that were sequenced on their N-terminal and have been checked against the accompanying reference paper to be considered as experimentally verified positions. The remaining 15,562 <em>unverified sequences </em>contain putative or experimentally unverified cleavage site signal sequences. This <em>unverified </em>set also contains entries with erroneous database annotations. It is worth noting that this <em>unverified </em>set might contain some experimentally verified signal sequences since we may not have access to the accompanying papers.<div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="T2" name="T2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1276010&amp;rendertype=table&amp;id=T2" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 2" title="Table 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1276010&amp;rendertype=table&amp;id=T2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 2</strong></a><div class="figure-table-caption-in-article"><span>Distribution of signal sequences in SPdb according to archaea (AR), bacteria (BA), viruses (VR) and eukaryotes (EU).</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>With the two primary databases integrated, SPdb contains four data groups namely archaea, bacteria, eukaryotes and viruses and (Table <a href="articlerender.fcgi?artid=1276010&amp;rendertype=table&amp;id=T2" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">2</a>). SPdb provides key extracted information (Figure <a href="articlerender.fcgi?artid=1276010&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>) such as organism source, organelle, subcellular location and other accompanying important notes. For full annotation, cross-referenced links to the originating database are provided. The signal peptide cleavage site is explicitly marked if such information is available. The signal peptide sequences and 30 residues [<a href="#B40" class="cite-reflink">40</a>] after the cleavage site are colour-coded using the convention as specified by RasMol amino acids colour scheme [<a href="#B41" class="cite-reflink">41</a>] which is based on the traditional amino acid properties. In the process of manually curating the 3,579 entries, we have added our own annotation for the 995 entries that were removed from this dataset later on.</p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="F2" name="F2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=1276010&amp;rendertype=figure&amp;id=F2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=1276010&amp;blobname=1471-2105-6-249-2.gif" border="1" class="icon-reflink" alt="Figure 2" title="Figure 2"></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=1276010&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2</strong></a><div class="figure-table-caption-in-article"><span>SPdb entry information includes a short description of the protein, the hydropathy plots and amino acids properties and more. (A) Each entry is marked as verified or unverified, with (B) a "report-error" link for users to inform us on any error or updated</span><a class="side-caption" style="font-size: 100%;" href="articlerender.fcgi?artid=1276010&amp;rendertype=figure&amp;id=F2" onclick="startTarget(this, 'figure', 1024, 800)"> (more ...)</a></div></td></tr></table></div></div><div style="clear:both;"></div></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id506740" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id539915" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id585752" class="sidefm-pmclink" style="text-transform: none;">Construction and content</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Utility and discussion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586664" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586677" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586693" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586754" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id586423">Utility and discussion</div><div class="section-content"><p>SPdb provides users with an easy-to-use web interface with flexibility to select for an entry or a collective set of entries matching users' criteria such as name of organism, data group, length of signal sequences, keyword searches and more importantly the option to choose between including or excluding certain entries. We have taken the approach to allow users to omit or filter any sequences since every user may have different requirements on the returned results. Each of the entry will be indicated whether it is verified or unverified (Figure <a href="articlerender.fcgi?artid=1276010&amp;rendertype=figure&amp;id=F2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">2</a>).</p><p>In the process of creating SPdb, we realise that although Swiss-Prot provides better quality annotation, it still contains erroneous or conflicting annotations as evident when we compare the positions or length of the signal sequences reported by Swiss-Prot with EMBL e.g. [Swiss-Prot:A2AP_HUMAN, Swiss-Prot:BTD_HUMAN]. We notice that the inconsistencies usually arise when there is more than one reference. The referred papers may quote different positions thus this may have caused the confusion. To help to resolve this issue, we have combined the annotations from -EMBL, and we managed to identify and filter off many such entries. The annotations on signal peptide found in EMBL were mostly accurate though there were cases when the information was reported incorrectly as well e.g. [EMBL:<span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0RQpD-DeX0Kb_r6cOEv2JhkSWg3dihV6dWwZPkQt1&amp;reftype=extlink-entrez-nucleotide&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBody&amp;TO=Entrez%7CTerm%7CNucleotide&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=nucleotide&amp;dopt=GenBank&amp;list_uids=155975" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">M19077</a></span>] in [Swiss-Prot:CHR1_BOMMO]. In this respect, we have included a link in each entry for users to report to us if they encounter any errors or discrepancies in SPdb.</p><p>Apart from the errors and inconsistencies just described, there was also the issue on experimental support from the journal publication. Many of the entries with annotation on signal sequences positions or length were predicted or deemed putative or potential (Table <a href="articlerender.fcgi?artid=1276010&amp;rendertype=table&amp;id=T1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">1</a>) by the researchers when they reported the positions in their papers. Nonetheless, the entries were not labelled with words like POTENTIAL, BY SIMILARITY and PROBABLE in the Swiss-Prot entry as previously assumed. We learnt that many of the referred papers were using prediction or sequence alignment software to identify or suggest the cleavage site of signal sequences. Therefore, we think it would be more appropriate and useful if the references to the papers were also indicated at the relevant fields so that any users of the entries can easily check and read up on the papers that mentioned about signal peptides or any other features.</p><p>All these issues and problems have made automation of the construction of SPdb immensely difficult if not remotely impossible. Prior to manually curating the entries, we have considered using text-mining approach but we forwent the method eventually when we discovered that many of abstracts did not contain the cleavage site information rather the information was found in the body of the paper, usually located under the results or discussion section. Moreover, the words or phrases used to express the positional information were also varying and difficult to express as extraction rules e.g. in the paper [<a href="#B42" class="cite-reflink">42</a>] quoted in entry [Swiss-Prot:PRRP_BOVIN], we encountered this sentence "... its N-terminal portion before Ser-23 showed the typical profile of a secretory signal peptide ...". Then there is also the problem where many of the papers require subscriptions, rendering the extraction program useless unless we can obtain the papers. Unless each of the paper submitted in future provides a short note on the features of the proteins described coupled with the improvement in text-mining accuracy, we will have to resort to manual curation.</p><p>In SPdb, datasets are classified into <em>filtered sequences </em>and <em>unverified sequences</em>. By classifying the entries into these two classifications, researchers can use them in the work of machine learning approaches, where datasets are sought after as training and testing sets in signal peptide cleavage site prediction.</p><p>Apart from facilitating test datasets, SPdb provides other information such as amino acid composition of the protein which have been suggested to correlate with the subcellular localisation of the protein [<a href="#B43" class="cite-reflink">43</a>]; amino acid residues properties (aromatic, non-polar, polar, charged and so on) are shown in graphical format to indicate which residues possess the properties visibly; also accompanying each entry are the hydropathy plots based on Kyte and Doolittle [<a href="#B44" class="cite-reflink">44</a>], Sweet and Eisenberg [<a href="#B45" class="cite-reflink">45</a>], Eisenberg <em>et al</em>. [<a href="#B46" class="cite-reflink">46</a>] of the signal sequences and the sequences downstream of the signal sequence cleavage. The plots are rendered using pepinfo within the computational analysis package of EMBOSS [<a href="#B47" class="cite-reflink">47</a>], an open source software suite for sequence analysis. Each signal peptide exhibits three distinct regions at the sequence level: the <em>n-region </em>(a positive charged region), the <em>h-region </em>(hydrophobic region) and the <em>c-region </em>(polar and neutral region) [<a href="#B9" class="cite-reflink">9</a>]. The hydropathy plots help to visualize and identify these regions.</p><p>De Gier <em>et al</em>. [<a href="#B48" class="cite-reflink">48</a>] showed that signal peptide processing by the signal recognition particle (SRP) requires certain contextual cues in the sequence downstream. SRP binds to N-terminus signal or signal-anchor sequence when the nascent polypeptide chain is synthesised by the ribosome up to ~60 amino acid residues. At this length, this segment is conveniently exposed and translation will resume upon dissociation of SRP from the nascent chain. In the effort to capture this information for the co-translational translocation mechanism, SPdb includes both the signal peptide sequences and 30 residues after the cleavage site.</p><p>For the future releases, we hope to include other information which maybe useful such as functional classification of signal peptides according to target destination, the profiles of signal peptides from various organisms and so on. As differentially targeted organelles or locations have variations on the general theme of signal peptide target proteins, we would like to include these different targeting signals for comparison and studies. Concerning secreted proteins that lack cleavable signal peptide [<a href="#B49" class="cite-reflink">49</a>] e.g. ovalbumin, a secreted glycoprotein and the major protein of egg-white which does not have a cleavable signal peptide [<a href="#B50" class="cite-reflink">50</a>], we would like to include this information and analyse how they differ from those proteins with cleavable signal peptide.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id506740" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id539915" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id585752" class="sidefm-pmclink" style="text-transform: none;">Construction and content</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586423" class="sidefm-pmclink" style="text-transform: none;">Utility and discussion</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Conclusion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586677" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586693" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586754" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id586664">Conclusion</div><div class="section-content"><p>Signal peptide plays an important role in the transport of secretory proteins. Understanding of signal peptide recognition and mechanisms of targeting, transport and translocation will unleash many applications in the area of drug design and medicine. We have provided a freely accessible, manually curated signal peptide database that is regularly updated and synchronised with the release of the two major primary databases, Swiss-Prot and EMBL. By integrating information from both databases, SPdb is able to eliminate some of the discrepancies and minimise the errors found in the sequence entries, thereby providing a better quality of the downloadable datasets that can be used by the research community for prediction work and other research.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id586677">Availability and requirements</div><div class="section-content"><p>SPdb is freely accessible through the website <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0tgolUdtwhs016NjxFQyFTxNi-V_ReHb55KervkeR&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://proline.bic.nus.edu.sg/spdb" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://proline.bic.nus.edu.sg/spdb</a></span>. We have made available a dedicated page to allow user to download the dataset in full based on certain criteria available to user.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id506740" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id539915" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id585752" class="sidefm-pmclink" style="text-transform: none;">Construction and content</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586423" class="sidefm-pmclink" style="text-transform: none;">Utility and discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586664" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586677" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Authors' contributions</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586754" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id586693">Authors' contributions</div><div class="section-content"><p>KHC built the database pipeline and the web interface. SR developed the signal peptide project and provided comments and suggestions on the features of the database while TWT provided assistance for the database design and the manuscript.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id586745">Acknowledgements</div><div class="section-content"><p><p><p>We would like to acknowledge Vivek Gopalan (while at the Dept. of Biochemistry, NUS) and the anonymous reviewers for their comments and suggestions. We also thank our users who have mailed us to provide their support, encouragement and comments.</p></p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id506740" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id539915" class="sidefm-pmclink" style="text-transform: none;">Background</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id585752" class="sidefm-pmclink" style="text-transform: none;">Construction and content</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586423" class="sidefm-pmclink" style="text-transform: none;">Utility and discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586664" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586677" class="sidefm-pmclink" style="text-transform: none;">Availability and requirements</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id586693" class="sidefm-pmclink" style="text-transform: none;">Authors' contributions</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id586754">References</div><div class="section-content"><div class="back-matter-section"><ul class="back-ref-list" style="list-style-type: decimal;"><li id="B1">Nobel Prize in Physiology or Medicine 1999. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0huHEIagpZW6e6PD9qUtnXOY9FTjyIEJsQbvszQQu&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://nobelprize.org/medicine/laureates/1999/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://nobelprize.org/medicine/laureates/1999/</a></span></li><li id="B2">Bowden, GA; Baneyx, F; Georgiou, G. Abnormal fractionation of beta-lactamase in Escherichia coli: evidence for an interaction with the inner membrane in the absence of a leader peptide. <span><span class="ref-journal">J Bacteriol. </span>1992;<span class="ref-vol">174</span>:3407–3410.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=06CSx39H1nxkd-6SVY1vf0BgA97DtLn_N5U8c-Q_k&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/1577708">PubMed</a>]</span></li><li id="B3">Flower, AM; Doebele, RC; Silhavy, TJ. PrlA and PrlG suppressors reduce the requirement for signal sequence recognition. <span><span class="ref-journal">J Bacteriol. </span>1994;<span class="ref-vol">176</span>:5607–5614.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0dOfA5widYxsd-Wab-dj6ILDee-KkcNG_PdLfJkI0&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8083155">PubMed</a>]</span></li><li id="B4">SIG-Pred: Signal Peptide Prediction. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0JQ-qluwK8I2Roy9uA2wsn4XEaybe3o13Wh9CwQhQ&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.bioinformatics.leeds.ac.uk/prot_analysis/Signal.html" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.bioinformatics.leeds.ac.uk/prot_analysis/Signal.html</a></span></li><li id="B5">SIGFIND - Signal Peptide Prediction Server (Eukaryotes). <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0-1MiH-v7g17iToK5zG3uqt-OLv9i5e2nqyhnGAu4&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://139.91.72.10/sigfind/sigfind.html" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://139.91.72.10/sigfind/sigfind.html</a></span></li><li id="B6">Hiller, K; Grote, A; Scheer, M; Munch, R; Jahn, D. PrediSi: prediction of signal peptides and their cleavage positions. <span><span class="ref-journal">Nucleic Acids Res. </span>2004;<span class="ref-vol">32</span>:W375–9.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0HEwvI63y5A3TqueoM2PKcW0mcy1rxXtEVWVRy5pA&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15215414">PubMed</a>]</span></li><li id="B7">Juncker, AS; Willenbrock, H; Von Heijne, G; Brunak, S; Nielsen, H; Krogh, A. Prediction of lipoprotein signal peptides in Gram-negative bacteria. <span><span class="ref-journal">Protein Sci. </span>2003;<span class="ref-vol">12</span>:1652–1662. doi: 10.1110/ps.0303703.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0xU9Yifdk60dxW4hGbhw0UXi0Nh8RYYWphsGe7A7G&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12876315">PubMed</a>]</span></li><li id="B8">Kall, L; Krogh, A; Sonnhammer, EL. A combined transmembrane topology and signal peptide prediction method. <span><span class="ref-journal">J Mol Biol. </span>2004;<span class="ref-vol">338</span>:1027–1036. doi: 10.1016/j.jmb.2004.03.016.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0R9ppx9xTgSJssk9YbsHeHsTlBg5h2FLs5X4a9ku1&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15111065">PubMed</a>]</span></li><li id="B9">Nielsen, H; Engelbrecht, J; Brunak, S; von Heijne, G. Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites. <span><span class="ref-journal">Protein Eng. </span>1997;<span class="ref-vol">10</span>:1–6. doi: 10.1093/protein/10.1.1.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0-uAi1ip3VUTr9c0BBLOmNYwRsw52KkUvjzpbUQfj&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9051728">PubMed</a>]</span></li><li id="B10">Bendtsen, JD; Nielsen, H; von Heijne, G; Brunak, S. Improved prediction of signal peptides: SignalP 3.0. <span><span class="ref-journal">J Mol Biol. </span>2004;<span class="ref-vol">340</span>:783–795. doi: 10.1016/j.jmb.2004.05.028.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0l56ZdkAXqNSIPYWz5azFjjhKKFtjXoCGIu3VI5cP&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15223320">PubMed</a>]</span></li><li id="B11">Bairoch, A; Boeckmann, B; Ferro, S; Gasteiger, E. Swiss-Prot: juggling between evolution and stability. <span><span class="ref-journal">Brief Bioinform. </span>2004;<span class="ref-vol">5</span>:39–55. doi: 10.1186/1471-2105-5-39.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0NaqaUdfyOIWtNp3ICO6hONsip6RgVzZ6tA06O5mv&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15153305">PubMed</a>]</span></li><li id="B12">Kanz, C; Aldebert, P; Althorpe, N; Baker, W; Baldwin, A; Bates, K; Browne, P; van den Broek, A; Castro, M; Cochrane, G; Duggan, K; Eberhardt, R; Faruque, N; Gamble, J; Diez, FG; Harte, N; Kulikova, T; Lin, Q; Lombard, V; Lopez, R; Mancuso, R; McHale, M; Nardone, F; Silventoinen, V; Sobhany, S; Stoehr, P; Tuli, MA; Tzouvara, K; Vaughan, R; Wu, D; Zhu, W; Apweiler, R. The EMBL Nucleotide Sequence Database. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33 Database Issue</span>:D29–33.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0-3ba7Lk7PCHezMWPOvOtFOo-IPZkYM4zzysYJAYl&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15608199">PubMed</a>]</span></li><li id="B13">NAR Database Category List - Protein localization and targeting. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0jaapTCli3OTrYRcEpJF90_5X1c7nT4G0CMiuLkDg&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www3.oup.co.uk/nar/database/subcat/3/7/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www3.oup.co.uk/nar/database/subcat/3/7/</a></span></li><li id="B14">PSORTdb. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0VKbSP5JtrFOzZZGEarE2s7KzW6l3zwX1sVvr359g&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://db.psort.org/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://db.psort.org/</a></span></li><li id="B15">DBSubLoc. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0-AQ0NlGP-SP95my1v1x-jy8dv96hhvXVX9DiTkOg&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.bioinfo.tsinghua.edu.cn/~guotao/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.bioinfo.tsinghua.edu.cn/~guotao/</a></span></li><li id="B16">Nuclear Protein Database (NPD). <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0HITcYP-nl_N6BMARp1EC04ks7ta5W2Xnq9JKrkl5&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://npd.hgu.mrc.ac.uk/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://npd.hgu.mrc.ac.uk/</a></span></li><li id="B17">Secreted Protein Database (SPD). <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0pKlax7lqsESVMSkNyK2HdS4Tz_n79dXHyRTz2ATU&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://spd.cbi.pku.edu.cn/spd_index.php" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://spd.cbi.pku.edu.cn/spd_index.php</a></span></li><li id="B18">Bairoch, A; Apweiler, R; Wu, CH; Barker, WC; Boeckmann, B; Ferro, S; Gasteiger, E; Huang, H; Lopez, R; Magrane, M; Martin, MJ; Natale, DA; O'Donovan, C; Redaschi, N; Yeh, LS. The Universal Protein Resource (UniProt). <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:D154–9. doi: 10.1093/nar/gki070.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0TMb0QMXYpP8HF91lOFITZpYZCwcawITsOC-T158I&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15608167">PubMed</a>]</span></li><li id="B19">Birney, E; Andrews, TD; Bevan, P; Caccamo, M; Chen, Y; Clarke, L; Coates, G; Cuff, J; Curwen, V; Cutts, T; Down, T; Eyras, E; Fernandez-Suarez, XM; Gane, P; Gibbins, B; Gilbert, J; Hammond, M; Hotz, HR; Iyer, V; Jekosch, K; Kahari, A; Kasprzyk, A; Keefe, D; Keenan, S; Lehvaslaiho, H; McVicker, G; Melsopp, C; Meidl, P; Mongin, E; Pettett, R; Potter, S; Proctor, G; Rae, M; Searle, S; Slater, G; Smedley, D; Smith, J; Spooner, W; Stabenau, A; Stalker, J; Storey, R; Ureta-Vidal, A; Woodwark, KC; Cameron, G; Durbin, R; Cox, A; Hubbard, T; Clamp, M. An overview of Ensembl. <span><span class="ref-journal">Genome Res. </span>2004;<span class="ref-vol">14</span>:925–928. doi: 10.1101/gr.1860604.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=00om1xrw4fk6nLXJCs5LXhPM7MpqU9wuXdoJMPkG1&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15078858">PubMed</a>]</span></li><li id="B20">Pruitt, KD; Tatusova, T; Maglott, DR. NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:D501–4. doi: 10.1093/nar/gki025.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0xC70PK7Tg_cCAO72jEMrBpokgtQeW8pzBrPQZA3b&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15608248">PubMed</a>]</span></li><li id="B21">Clark, HF; Gurney, AL; Abaya, E; Baker, K; Baldwin, D; Brush, J; Chen, J; Chow, B; Chui, C; Crowley, C; Currell, B; Deuel, B; Dowd, P; Eaton, D; Foster, J; Grimaldi, C; Gu, Q; Hass, PE; Heldens, S; Huang, A; Kim, HS; Klimowski, L; Jin, Y; Johnson, S; Lee, J; Lewis, L; Liao, D; Mark, M; Robbie, E; Sanchez, C; Schoenfeld, J; Seshagiri, S; Simmons, L; Singh, J; Smith, V; Stinson, J; Vagts, A; Vandlen, R; Watanabe, C; Wieand, D; Woods, K; Xie, MH; Yansura, D; Yi, S; Yu, G; Yuan, J; Zhang, M; Zhang, Z; Goddard, A; Wood, WI; Godowski, P; Gray, A. The secreted protein discovery initiative (SPDI), a large-scale effort to identify novel human secreted and transmembrane proteins: a bioinformatics assessment. <span><span class="ref-journal">Genome Res. </span>2003;<span class="ref-vol">13</span>:2265–2270. doi: 10.1101/gr.1293003.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0vqhsZDDsCeHbD3lOOB1G5sP0Xzm-A-DiwAyx45zK&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12975309">PubMed</a>]</span></li><li id="B22">SPD and its collated related datasets. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0xkiHHsDnh96c0jPrJ7IRNKzl5qrQqjk2GFozKAVs&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://spd.cbi.pku.edu.cn/help/spd_help.php#source" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://spd.cbi.pku.edu.cn/help/spd_help.php#source</a></span></li><li id="B23">SignalP Training Datasets (1997). <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=04SkqoaXTcJWCOyzRs-CToC4STuXHAsOGK8eQyQYO&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.cbs.dtu.dk/ftp/signalp/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.cbs.dtu.dk/ftp/signalp/</a></span></li><li id="B24">Test sets used in evaluating the different signal prediction methods. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0Lmg2C4nB_1ibHs8tVGxON81j_o8uWol1YElUIQFn&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;ftp://ftp.ebi.ac.uk/pub/contrib/swissprot/testsets/signal" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">ftp://ftp.ebi.ac.uk/pub/contrib/swissprot/testsets/signal</a></span></li><li id="B25">Menne, KM; Hermjakob, H; Apweiler, R. A comparison of signal sequence prediction methods using a test set of signal peptides. <span><span class="ref-journal">Bioinformatics. </span>2000;<span class="ref-vol">16</span>:741–742. doi: 10.1093/bioinformatics/16.8.741.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0ncG88tzvcZtmVNuT8gFPxB0owoXxJhlVwAPFP5iz&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/11099261">PubMed</a>]</span></li><li id="B26">Signal peptide prediction based on analysis of experimentally verified
cleavage sites - supplementary data. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0LMaqXGtfohent8Qs0WYSGJ4hwa-EjL_9CqBEJkD5&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://share.gene.com/share/cleavagesite/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://share.gene.com/share/cleavagesite/</a></span></li><li id="B27">Zhang, Z; Henzel, WJ. Signal peptide prediction based on analysis of experimentally verified cleavage sites. <span><span class="ref-journal">Protein Sci. </span>2004;<span class="ref-vol">13</span>:2819–2824. doi: 10.1110/ps.04682504.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hQ5w6LyxUupVG0TQd8oUcmjer7dy2FKzs11R2Q-n&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15340161">PubMed</a>]</span></li><li id="B28">Datasets used in SignalP and provided for download. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0NMKeNOjX8pEfKpawcGLVvDwlcKG0vsOaHCZmSACf&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.cbs.dtu.dk/services/SignalP/background/trainingset.php#trainingset" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.cbs.dtu.dk/services/SignalP/background/trainingset.php#trainingset</a></span></li><li id="B29">Pennisi, E. Keeping genome databases clean and up to date. <span><span class="ref-journal">Science. </span>1999;<span class="ref-vol">286</span>:447–450. doi: 10.1126/science.286.5439.447.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pKvNU096-Ixt_oxzzBdcEMKU8IKkaH_qhii7oAes&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10577208">PubMed</a>]</span></li><li id="B30">Wiley, HS; Michaels, GS. Should software hold data hostage? <span><span class="ref-journal">Nat Biotechnol. </span>2004;<span class="ref-vol">22</span>:1037–1038. doi: 10.1038/nbt0804-1037.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0LWsEjjHArwF0Bha5EhAccJdgOnXrKX82eZBB6kDV&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/15286656">PubMed</a>]</span></li><li id="B31">Signal Peptide Database (SPdb). <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0nMqFsG3wKQMTaOyir-olgEcetBLgtrJxdorLnQaM&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://proline.bic.nus.edu.sg/spdb" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://proline.bic.nus.edu.sg/spdb</a></span></li><li id="B32">MySQL database management system. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0jS0VT-7JWuPQ2PYFAmjENyWWSN9OPL5jz2Pm3kVY&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.mysql.com/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.mysql.com/</a></span></li><li id="B33">Perl Programming Language. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0bEDLXwsZTYS6tNtx5zkh43dZYEF2XHcGS41CEQHd&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.perl.org/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.perl.org/</a></span></li><li id="B34">Nielsen, H; Engelbrecht, J; von Heijne, G; Brunak, S. Defining a similarity threshold for a functional protein sequence pattern: the signal peptide cleavage site. <span><span class="ref-journal">Proteins. </span>1996;<span class="ref-vol">24</span>:165–177. doi: 10.1002/(SICI)1097-0134(199602)24:2&#x0003c;165::AID-PROT4&#x0003e;3.0.CO;2-I.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Rc1_gbLuHrynNxRHAYyk_ubmKSTUl0nSTN1NLQKK&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8820484">PubMed</a>]</span></li><li id="B35">SignalP - A Popular Signal Peptide Prediction Server. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0hWPEkbd1HVgPIBYz8L9uc-UmxMoLgn2MSrhHnkCI&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://cbs.dtu.dk/services/SignalP/" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://cbs.dtu.dk/services/SignalP/</a></span></li><li id="B36">Swiss-Prot Manual (FT Field). <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0XqKqVufzD55ybCuOJZjU9nQugwexnQ0Z47uh0kHT&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.expasy.org/sprot/userman.html#FT_line" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.expasy.org/sprot/userman.html#FT_line</a></span></li><li id="B37">Bork, P. Powers and pitfalls in sequence analysis: the 70% hurdle. <span><span class="ref-journal">Genome Res. </span>2000;<span class="ref-vol">10</span>:398–400. doi: 10.1101/gr.10.4.398.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0hCIYJ3COGVAgxGno0c16NXVNewkD9-eN677_PAFS&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/10779480">PubMed</a>]</span></li><li id="B38">EMBL Database Release Note. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0XYkdRoqLtwrhCLuJ0CSr0komiZwDa07L08weB5VV&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.ebi.ac.uk/embl/Documentation/Release_notes/current/relnotes.html" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.ebi.ac.uk/embl/Documentation/Release_notes/current/relnotes.html</a></span></li><li id="B39">Swiss-Prot Manual (DR Field). <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0xOqlx06-OTCX_av48yLvhrAbY-M49QAV2h6y1ku9&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.expasy.org/sprot/userman.html#DR_line" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.expasy.org/sprot/userman.html#DR_line</a></span></li><li id="B40">Andersson, H; von Heijne, G. A 30-residue-long "export initiation domain" adjacent to the signal sequence is critical for protein translocation across the inner membrane of Escherichia coli. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>1991;<span class="ref-vol">88</span>:9751–9754.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=02cQjJZTyC9hm6UW5-bgiOt_s7ww6WnlF-fygRQp-&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/1946398">PubMed</a>]</span></li><li id="B41">RasMol amino colour scheme. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0Ti1lq3d6UAH82_B18v3VOWDZerTsjhscXptvdQ27&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://www.openrasmol.org/doc/rasmol.html#aminocolours" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://www.openrasmol.org/doc/rasmol.html#aminocolours</a></span></li><li id="B42">Hinuma, S; Habata, Y; Fujii, R; Kawamata, Y; Hosoya, M; Fukusumi, S; Kitada, C; Masuo, Y; Asano, T; Matsumoto, H; Sekiguchi, M; Kurokawa, T; Nishimura, O; Onda, H; Fujino, M. A prolactin-releasing peptide in the brain. <span><span class="ref-journal">Nature. </span>1998;<span class="ref-vol">393</span>:272–276. doi: 10.1038/30515.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0pALUNSAguejT33a68l5EgYEOdwL2M8fOijOuBQrg&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9607765">PubMed</a>]</span></li><li id="B43">Nakashima, H; Nishikawa, K. Discrimination of intracellular and extracellular proteins using amino acid composition and residue-pair frequencies. <span><span class="ref-journal">J Mol Biol. </span>1994;<span class="ref-vol">238</span>:54–61. doi: 10.1006/jmbi.1994.1267.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0P_G_Stv38pCOYwi0quvg16weLBGwOmwHOiYno5gR&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/8145256">PubMed</a>]</span></li><li id="B44">Kyte, J; Doolittle, RF. A simple method for displaying the hydropathic character of a protein. <span><span class="ref-journal">J Mol Biol. </span>1982;<span class="ref-vol">157</span>:105–132. doi: 10.1016/0022-2836(82)90515-0.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0p5k4-q8IMnCl6xfAAaatArAufLowM_vrxQ7DK5XL&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7108955">PubMed</a>]</span></li><li id="B45">Sweet, RM; Eisenberg, D. Correlation of sequence hydrophobicities measures similarity in three-dimensional protein structure. <span><span class="ref-journal">J Mol Biol. </span>1983;<span class="ref-vol">171</span>:479–488. doi: 10.1016/0022-2836(83)90041-4.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0XszqX6OuA3ZPdi3g2vB8bDa20YKDoqCwf9ChT5rA&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/6663622">PubMed</a>]</span></li><li id="B46">Eisenberg, D; Weiss, RM; Terwilliger, TC. The helical hydrophobic moment: a measure of the amphiphilicity of a helix. <span><span class="ref-journal">Nature. </span>1982;<span class="ref-vol">299</span>:371–374. doi: 10.1038/299371a0.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0tIKPL8uwqlD4CHn2RumuET6h1h-fuxHlCxohNkBs&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7110359">PubMed</a>]</span></li><li id="B47">EMBOSS : Sequence analysis application suite. <span class="ext-reflink"><a href="redirect3.cgi?&amp;&amp;auth=0B3nuE2eZbBdU4PBwWIvM3jcPGbfNps4CKsOfTQVP&amp;reftype=extlink&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI&amp;rendering-type=normal&amp;&amp;http://emboss.sourceforge.net" class="ext-reflink" target="pmc_ext" onclick="focuswin('pmc_ext')">http://emboss.sourceforge.net</a></span></li><li id="B48">de Gier, JW; Scotti, PA; Saaf, A; Valent, QA; Kuhn, A; Luirink, J; von Heijne, G. Differential use of the signal recognition particle translocase targeting pathway for inner membrane protein assembly in Escherichia coli. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>1998;<span class="ref-vol">95</span>:14646–14651. doi: 10.1073/pnas.95.25.14646.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0RQaBDRQjr4_U0pRahW0f0OPRDXSVt-bWLHBDAAbW&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9843943">PubMed</a>]</span></li><li id="B49">Ye, RD; Wun, TC; Sadler, JE. Mammalian protein secretion without signal peptide removal. Biosynthesis of plasminogen activator inhibitor-2 in U-937 cells. <span><span class="ref-journal">J Biol Chem. </span>1988;<span class="ref-vol">263</span>:4869–4875.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Be0IoctLh9yi-pFk2nvwy5ZGT4ld-DDS-FCLf5KO&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/3127394">PubMed</a>]</span></li><li id="B50">Lingappa, VR; Lingappa, JR; Blobel, G. Chicken ovalbumin contains an internal signal sequence. <span><span class="ref-journal">Nature. </span>1979;<span class="ref-vol">281</span>:117–121. doi: 10.1038/281117a0.</span>  <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Rau1xYBKbr6OO4X8AZA65Nx9GeSq_I_1rUYxfA_g&amp;reftype=pubmed&amp;artid=1276010&amp;article-id=1276010&amp;iid=18092&amp;issue-id=18092&amp;jid=13&amp;journal-id=13&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/471058">PubMed</a>]</span></li></ul></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">BMC Bioinformatics</span> are provided here courtesy of <br><b>BioMed Central</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc3-->

