<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"><html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE"><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/"><meta name="citation_journal_title" content="Journal of the American Medical Informatics Association"><meta name="citation_title" content="An Enriched Unified Medical Language System Semantic Network with a Multiple Subsumption Hierarchy"><meta name="citation_authors" content="Li Zhang, Yehoshua Perl, Michael Halper, James Geller, and  James J. Cimino"><meta name="citation_date" content="May&#x02013;Jun 2004"><meta name="citation_issue" content="3"><meta name="citation_volume" content="11"><meta name="citation_firstpage" content="195"><meta name="citation_doi" content="10.1197/jamia.M1269"><meta name="citation_pdf_url" content="http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=400518&amp;blobtype=pdf"><meta name="citation_abstract_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=400518&amp;rendertype=abstract"><meta name="citation_fulltext_html_url" content="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=400518"><meta name="citation_pmid" content="14764611"><meta name="DC.Title" content="An Enriched Unified Medical Language System Semantic Network with a Multiple Subsumption Hierarchy"><meta name="DC.Type" content="Text"><meta name="DC.Publisher" content="American Medical Informatics Association"><meta name="DC.Contributor" content="Li Zhang"><meta name="DC.Contributor" content="Yehoshua Perl"><meta name="DC.Contributor" content="Michael Halper"><meta name="DC.Contributor" content="James Geller"><meta name="DC.Contributor" content="James J. Cimino"><meta name="DC.Date" content="May&#x02013;Jun 2004"><meta name="DC.Identifier" content="10.1197/jamia.M1269"><meta name="DC.Language" content="en"><link rel="stylesheet" href="corehtml/pmc/css/pmcstatic.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbase1.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbars-black.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcbody6.css" type="text/css"><link rel="stylesheet" href="corehtml/pmc/css/pmcrefs1.css" type="text/css"><script type="text/javascript" src="corehtml/pmc/js/common.js"></script><script type="text/javascript">window.name="mainwindow";</script><link href="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/css/tileshop_pmc.1/tileshop_pmc.1.css" type="text/css" rel="stylesheet"><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/utils.1.js"></script><script>utils.jsLoader.sBase = "http://www.ncbi.nlm.nih.gov/corehtml/jsutils/";</script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corehtml/jsutils/tileshop_pmc.1/tileshop_pmc.1.js"></script><script type="text/javascript" src="http://www.ncbi.nlm.nih.gov/corecgi/tileshop/tileshop_data_db.1.js"></script><title>An Enriched Unified Medical Language System Semantic Network with a Multiple Subsumption Hierarchy</title></head><body class="pmc-body"><a id="top" name="top"></a><table cellpadding="0" cellspacing="0" width="100%" border="0"><tr valign="top"><td colspan="3"><table border="0" cellspacing="0" cellpadding="0" style="border-collapse: collapse;"><tr><td width="145" valign="top"><img src="corehtml/pmc/pmcgifs/pmc3_logo_v5.gif" border="0" usemap="#pmclogo" alt="pmc logo image"><map name="pmclogo"><area shape="rect" alt="Journal List" coords="65,52,137,67" href="fprender.fcgi"><area shape="rect" alt="Search" coords="7,52,52,67" href="redirect3.cgi?&amp;&amp;auth=0Dyq4s47hM2tJhJ1rTwXUtRHCOxwYf62Oh8_muAlr&amp;reftype=other&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CLogo&amp;TO=Entrez%7CSearch%7CAll%20PMC&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc"><area shape="rect" alt="pmc logo image" coords="0,0,145,75" href="./"></map></td><td height="75" valign="top" style="padding-left: 4px;"><div><img align="top" border="0" src="corehtml/pmc/pmcgifs/jamialogo.gif" alt="Logo of jamia" usemap="#logo-imagemap"><map id="logo-imagemap" name="logo-imagemap"><area shape="RECT" coords="0,0,349,62" href="redirect3.cgi?&amp;&amp;auth=0TKB9n9ZWnaKQCQ-zTfYpN0PO3_yg4crjKQn86AXP&amp;reftype=publisher&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.jamia.org" alt="JAMIA - The Journal of the American Medical Informatics Association" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="RECT" coords="0,63,85,74" href="redirect3.cgi?&amp;&amp;auth=0bko6EvAUK4EyygtuKO4qv3MtgoH7PRse8o_cB5JM&amp;reftype=publisher&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.jamia.org/subscriptions/online.shtml" alt="Subscribe to JAMIA" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="RECT" coords="87,63,173,74" href="redirect3.cgi?&amp;&amp;auth=0Dg0ZjM1vUrYLXwGiX7FcHqWhGd1MwPjdBDRShQ9Y&amp;reftype=publisher&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.amia.org" alt="AMIA Home Page" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="RECT" coords="175,63,261,74" href="redirect3.cgi?&amp;&amp;auth=00_dotWUzoRoGmlRq2TX5XF39COjHdLN4xQ4KwkqL&amp;reftype=publisher&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.amia.org/membership/fmembr.html" alt="AMIA Membership" target="pmc_ext" onclick="focuswin('pmc_ext')"><area shape="RECT" coords="263,63,349,74" href="redirect3.cgi?&amp;&amp;auth=0HGvHWEVFDJb3yZlJcsBB4MTlA93Bvtri0A4eYQq3&amp;reftype=publisher&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN%2FA&amp;rendering-type=normal&amp;&amp;http://www.amia.org/search/fsearch.html" alt="Search the AMIA Web Site" target="pmc_ext" onclick="focuswin('pmc_ext')"></map></div></td></tr></table></td></tr></table><table cellpadding="0" cellspacing="3" width="100%" border="0"><tr><td></td><td colspan="2"><div class="navlink-box navlink-box-gray"><a href="fprender.fcgi" class="navlink">Journal List</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?journal=76&amp;action=archive">J Am Med Inform Assoc</a><span> &gt; </span><a class="navlink" href="tocrender.fcgi?iid=134762">v.11(3); May&#x02013;Jun 2004</a></div></td></tr><tr><td width="145" valign="top" class="sidebar-cell"><div class="sidebar-article-navigation-box"><div class="sidefm-pmclink-item"><a href="articlerender.fcgi?artid=400518&amp;rendertype=abstract" class="sidefm-pmclink">Abstract</a></div><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Full Text</a></div><div class="sidefm-pmclink-item"><a href="picrender.fcgi?artid=400518&amp;blobtype=pdf" class="sidefm-pmclink">PDF (223K)</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?iid=134762" class="sidefm-pmclink">Contents</a></div><div class="sidefm-pmclink-item"><a href="tocrender.fcgi?journal=76&amp;action=archive" class="sidefm-pmclink">Archive</a></div></div><div class="sidebar-pubmed-box"><div class="sidebar-pubmed-links-to-box"><form name="pubmedLinks" onsubmit="return pubMedDbLinkSubmit(this)"><div class="sidefm-pmsubhead"><label for="pubmedOption">Related material:</label></div><select name="pubmedOption" id="pubmedOption" class="sidefm-pmart" onchange="return pubMedDbLinkSubmit(this)"><option value="redirect3.cgi?&amp;&amp;auth=06eAlM3TqpXJKaIqHUigTfqlRQee4z6kiSW7X5Qhj&amp;reftype=abs&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Record&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14764611">PubMed record</option><option selected value="redirect3.cgi?&amp;&amp;auth=0virNI3ABbwuV36IciTKx5jNonzMvwuDoVkgyl5ty&amp;reftype=relart&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Related%20Records&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&amp;cmd=Link&amp;dbFrom=PubMed&amp;from_uid=14764611">PubMed related arts</option><option value="redirect3.cgi?&amp;&amp;auth=0hkqeUs5x8g_hQAV_JoKcFLBAZJK2jilUMQ3FYk_Y&amp;reftype=linkout&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article|Navigation&amp;TO=Entrez|PubMed|Linkout&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowLinkOut&amp;TermToSearch=14764611">PubMed LinkOut</option><option value="redirect3.cgi?&amp;&amp;auth=0hOH4Ul8yKM5yXc4MqY8mRGtQesNYj7fk9FkEB596&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pmc&amp;cmd=Display&amp;from_uid=400518&amp;dopt=pmc_taxonomy">Taxonomy</option><option value="redirect3.cgi?&amp;&amp;auth=0Hw_6B59GGJpmNZjDNAy_4T3RCobW5aOVgQ7OP55P&amp;reftype=Taxonomy&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article|Navigation&amp;TO=Entrez|Crosslink|Taxonomy&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?Db=taxonomy&amp;DbFrom=pmc&amp;cmd=Link&amp;LinkName=pmc_taxonomy&amp;doptcmdl=TxTree&amp;IdsFromResult=400518">Taxonomy tree</option></select><br><input type="image" src="corehtml/pmc/pmcgifs/go.gif" alt="Go" class="sidefm-pmart"></form></div><div class="sidebar-pubmed-author-box"><div class="sidefm-pmsubhead">PubMed articles by:</div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0r1UeBazzl6MdsQPHNNs-A9Mks6P2aqwCwDdlw59K&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Zhang+L%5Bauth%5D" class="sidefm-pmclink"> Zhang, L.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=009ck02t_3_WTQVnKrb94oJfx_dBKze56U3PL8kWn&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Perl+Y%5Bauth%5D" class="sidefm-pmclink"> Perl, Y.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0rUUUnHIijIouA3t7mDbF0bgZW9aQHsoI2HbNNkz7&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Halper+M%5Bauth%5D" class="sidefm-pmclink"> Halper, M.</a></div><div class="sidefm-pmclink-item"><a href="redirect3.cgi?&amp;&amp;auth=0NuiKPEre0a2nlor7GUcl_WLWYTif25nhMq5eCkC_&amp;reftype=authsrch&amp;refto=entrez&amp;reffrom=sidebar&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CNavigation&amp;TO=Entrez%7CPubMed%7CAuthor%20Search&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=PubMed&amp;term=%20Cimino+JJ%5Bauth%5D" class="sidefm-pmclink"> Cimino, J.</a></div></div></div></td><td rowspan="104" width="20px" height="300" background="corehtml/pmc/pmcgifs/wm-jamia.gif"><div style="width:20px"></div></td><td valign="top" class="content-cell"><div class="front-matter-section"><table cellspacing="0" cellpadding="0" width="100%"><tr style="vertical-align: top"><td><div class="fm-citation"><div><span class="citation-abbreviation">J Am Med Inform Assoc. </span><span class="citation-publication-date">2004 May&#x02013;Jun; </span><span class="citation-volume">11</span><span class="citation-issue">(3)</span><span class="citation-flpages">: 195–206. </span></div><div><span class="fm-vol-iss-date"> </span><span class="fm-vol-iss-date">doi: 10.1197/jamia.M1269.</span></div></div></td><td class="fm-citation-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span>PMC400518</div></td></tr></table><div class="fm-copyright"><a class="int-reflink" href="about/copyright.html">Copyright</a>  &#x000a9; 2004, American Medical Informatics Association</div><div class="fm-title">An Enriched Unified Medical Language System Semantic Network with a Multiple Subsumption Hierarchy</div><div class="fm-author contrib-group">Li Zhang, MS, Yehoshua Perl, PhD, Michael Halper, PhD, James Geller, PhD, and  James J. Cimino, MD<div class="fm-affl">Affiliations of the authors: CS Department, New Jersey Institute of Technology, Newark, NJ (LZ, YP, JG); Mathematics &#x00026; ComputerScience Department, Kean University, Union, NJ (MH); Department of Medical Informatics, Columbia University, New York, NY (JJC).</div></div><div class="fm-footnote"></div><div class="fm-footnote"><div class="p">Correspondence and reprints: Li Zhang, MS, Room 4214, GITC Building, CS Department, NJIT, University Heights, Newark, NJ 07102; e-mail: &#x0003c;<span class="e_id375260">lxz1853/at/njit.edu</span><script type="text/javascript" language="JavaScript"><!--
                                    try{initUnObscureEmail ("e_id375260", '<a class="ext-reflink" href="' + reverseAndReplaceString('ude.tijn/ta/3581zxl:otliam', '/at/', '@') + '">' + reverseAndReplaceString('ude.tijn/ta/3581zxl', '/at/','@') + '</a>')}catch(e){}
                                //--></script>&#x0003e;.</div></div><div class="fm-pubdate">Received October 14, 2002; Accepted January 13, 2004.</div><div class="links-box"><div class="fm-footnote"><img src="corehtml/pmc/pmcgifs/rt-arrow.gif" alt="Small right arrow pointing to:" style="vertical-align: middle;"> This article has been <a class="int-reflink" href="tocrender.fcgi?artid=400518&amp;action=cited">cited by</a> other articles in PMC.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Abstract</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id455913" class="sidefm-pmclink" style="text-transform: none;">Methods to Enhance the SN'S IS-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />A Hierarchy</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457388" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457737" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458169" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458193" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id441162">Abstract</div><div class="section-content"><!--article-meta--><strong>Objective:</strong> The Unified Medical Language System's (UMLS's) Semantic Network's (SN's) two-tree structure is restrictive because it does not allow a semantic type to be a specialization of several other semantic types. In this article, the SN is expanded into a multiple subsumption structure with a directed acyclic graph (DAG) IS-A hierarchy, allowing a semantic type to have multiple parents. New viable IS-A links are added as warranted.<p><strong>Design:</strong> Two methodologies are presented to identify and add new viable IS-A links. The first methodology is based on imposing the characteristic of connectivity on a previously presented partition of the SN. Four transformations are provided to find viable IS-A links in the process of converting the partition's disconnected groups into connected ones. The second methodology identifies new IS-A links through a string matching process involving names and definitions of various semantic types in the SN. A domain expert is needed to review all the results to determine the validity of the new IS-A links.</p><p><strong>Results:</strong> Nineteen new IS-A links are added to the SN, and four new semantic types are also created to support the multiple subsumption framework. The resulting network, called the Enriched Semantic Network (ESN), exhibits a DAG-structured hierarchy. A partition of the ESN containing 19 connected groups is also derived.</p><p><strong>Conclusion:</strong> The ESN is an expanded abstraction of the UMLS compared with the original SN. Its multiple subsumption hierarchy can accommodate semantic types with multiple parents. Its representation thus provides direct access to a broader range of subsumption knowledge.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id441162" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id455913" class="sidefm-pmclink" style="text-transform: none;">Methods to Enhance the SN'S IS-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />A Hierarchy</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457388" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457737" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458169" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458193" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id328830"> </div><div class="section-content">The Unified Medical Language System (UMLS) is a large terminology whose Metathesaurus (META) contains approximately 900,000 concepts.<sup><a href="#bib1" class="cite-reflink">1</a></sup><sup>,</sup><sup><a href="#bib2" class="cite-reflink">2</a></sup><sup>,</sup><sup><a href="#bib3" class="cite-reflink">3</a></sup> The Semantic Network (SN) has 135 (134 at the time of our study) semantic types and provides a high-level abstract view of the META.<sup><a href="#bib4" class="cite-reflink">4</a></sup><sup>,</sup><sup><a href="#bib5" class="cite-reflink">5</a></sup> As expressed in the study by McCray and Nelson,<sup><a href="#bib6" class="cite-reflink">6</a></sup> &#x0201c;The Semantic Network encompasses and provides a unifying structure for the Metathesaurus constituent vocabularies.&#x0201d;<p>The SN contains a hierarchy consisting of two trees rooted at the semantic types <strong>Event</strong> and <strong>Entity</strong>,<sup><a href="#fn1" class="cite-reflink">&#x0002a;</a></sup> respectively.<sup><a href="#bib7" class="cite-reflink">7</a></sup> This hierarchy is based on the IS-A (subsumption) relationship, which connects a more specialized semantic type (a child) to a more generalized semantic type (its parent). Each semantic type, except for <strong>Event</strong> and <strong>Entity</strong>, is a specialization of exactly one semantic type (its parent) and inherits semantic relationships only from this unique parent.</p><p>Many researchers have suggested that concept-oriented<sup><a href="#bib8" class="cite-reflink">8</a></sup><sup>,</sup><sup><a href="#bib9" class="cite-reflink">9</a></sup> and logic-based<sup><a href="#bib10" class="cite-reflink">10</a></sup><sup>,</sup><sup><a href="#bib11" class="cite-reflink">11</a></sup><sup>,</sup><sup><a href="#bib12" class="cite-reflink">12</a></sup><sup>,</sup><sup><a href="#bib13" class="cite-reflink">13</a></sup> approaches are beneficial for creating categorical terminological structures,<sup><a href="#bib14" class="cite-reflink">14</a></sup> especially when their purpose is to support, as the UMLS does, cross-thesaurus mappings.<sup><a href="#bib14" class="cite-reflink">14</a></sup><sup>,</sup><sup><a href="#bib15" class="cite-reflink">15</a></sup> However, the SN does not provide sufficient logic-based structures to apply such methods; we must, therefore, seek alternative methods to improve the consistency and utility of the SN.</p><p>Although the SN's tree structure is easy to implement and process, it is restrictive in that it does not allow multiple parent types when warranted. There are, in fact, some semantic types that could naturally be specializations of more than one semantic type. For example, <strong>Gene or Genome</strong> could conceptually be a child of two semantic types: one is its current parent <strong>Fully Formed Anatomical Structure</strong>; the other is <strong>Molecular Sequence</strong>. Hence, <strong>Gene or Genome</strong> should inherit from <strong>Molecular Sequence</strong> the semantic relationship <em>result_of</em> to <strong>Mental Process</strong>. In a case such as this, the modeling of the SN omits an aspect of current medical knowledge.</p><p>In this article, we present two methodologies to structurally enrich the SN by transforming its hierarchy into a directed acyclic graph (DAG) structure that allows multiple parents. The methodologies are based on the identification of viable new IS-A relationships currently not included in the SN. These omissions may have been due to the tree-structure restriction on the SN, noted previously in this article. We add new semantic types to the SN as necessary to accommodate the new multiple subsumption framework. In the first methodology, the identification of new IS-As is guided by imposing connectivity on an existing partition of the SN.<sup><a href="#bib16" class="cite-reflink">16</a></sup> In the second methodology, the identification is based on partial string matching between names of semantic types and the definitions of other semantic types. These identified potential IS-A relationships are then reviewed by a domain expert to decide whether they are semantically valid. With the addition of these new IS-A relationships, we get a new DAG version of the SN that we refer to as the <em>Enriched Semantic Network</em> (<em>ESN</em>). Furthermore, we obtain a partition of the ESN consisting of groups, each of which has a tree structure. The ESN serves as an enhanced abstraction of the UMLS. The accompanying partition enables the creation of a metaschema,<sup><a href="#bib17" class="cite-reflink">17</a></sup> an additional abstract layer of the ESN that can help users in their orientation to the UMLS.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id441162" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Methods to Enhance the SN'S IS-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />A Hierarchy</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457388" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457737" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458169" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458193" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id455913">Methods to Enhance the SN'S IS-A Hierarchy</div><div class="section-content"><p><div class="head2 head-separate">Imposing Connectivity on an SN Partition</div><p><span class="head3">Basis </span> McCray et al.<sup><a href="#bib16" class="cite-reflink">16</a></sup> presented a partition of the SN into 15 groups, with each group representing a subject area. Six principles that such a partition should satisfy were proposed. One of them, semantic validity, can be assessed by seeing if a group's semantic types together with their IS-A links form a connected subgraph of the SN. We refer to this as the &#x0201c;connectivity property.&#x0201d; Because the SN's IS-A hierarchy consists of two trees, such a connected subgraph in the current SN must form a tree with a unique root.<p>In the analysis of McCray et al.,<sup><a href="#bib16" class="cite-reflink">16</a></sup> it was noted that: &#x0201c;In some cases, it was not possible to resolve anomalies in our attempt to create a coherent and semantically valid set of groupings.&#x0201d; In fact, some of the partition's groups do not satisfy the connectivity property. Such groups contain a forest, comprising two or more trees, or perhaps isolated semantic types. (Some groups have both.) For example, the <em>Physiology</em><sup><a href="#fn2" class="cite-reflink">&#x02020;</a></sup> group contains a forest of two trees (<a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig1" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 1</a>). There are no hierarchical relationships between a semantic type of one tree and a semantic type of the other tree. Therefore, the <em>Physiology</em> group is not connected.  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig1" name="fig1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig1" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=400518&amp;blobname=195f01.gif" border="1" class="icon-reflink" alt="Figure 1." title="Figure 1."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 1.</strong></a><div class="figure-table-caption-in-article"><span><em>Physiology</em> group.</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>In our previous work,<sup><a href="#bib18" class="cite-reflink">18</a></sup> we presented an alternative partition of the SN based on the sets of relationships exhibited by the semantic types. In our technique, we required that the hierarchy of each group of the partition be a tree exhibiting the connectivity property. In this way, we obtained a partition that is strictly semantically uniform. A difference between the partition of McCray et al.<sup><a href="#bib16" class="cite-reflink">16</a></sup> and that of Chen et al.<sup><a href="#bib18" class="cite-reflink">18</a></sup> is that the connectivity is only a preferred, not required, property in McCray et al.,<sup><a href="#bib16" class="cite-reflink">16</a></sup> whereas it is required and enforced in Chen et al.<sup><a href="#bib18" class="cite-reflink">18</a></sup></p><p>In our semantic technique, we use the partition of McCray et al.<sup><a href="#bib16" class="cite-reflink">16</a></sup> as a basis for augmenting the SN's hierarchy, and, in particular, for identifying new viable IS-A relationships. The basic idea is to bridge the gap between the two partitioning techniques by imposing the connectivity property on the partition of McCray et al.<sup><a href="#bib16" class="cite-reflink">16</a></sup> To convert the disconnected groups of McCray et al.<sup><a href="#bib16" class="cite-reflink">16</a></sup> into connected groups, we need to identify and insert additional IS-As. This will yield a first version of our desired multiple subsumption hierarchy and an accompanying partition. Analysis of the definitions of semantic types within each disconnected group will guide the introduction of the new IS-A links. In this context, we will use four kinds of transformations with respect to the groups of McCray et al.<sup><a href="#bib16" class="cite-reflink">16</a></sup> Another methodology using exact string matching will then be used in a following subsection.</p></p><p><span class="head3">Four transformations to identify new IS-A links </span> The possible transformations that can be applied to disconnected groups to make them connected are listed in the following. The choice of which transformation to use is based on reviews of the definitions of all semantic types within a group.  <ul style="list-style-type: none;"><li><span><strong>IS-A Addition Transformation:</strong> Identify a viable IS-A and add it to transform the group into a connected subtree.</span></li><li><span><strong>Split Transformation:</strong> Split a group into several groups, each of which is either a rooted tree structure or can be transformed into a rooted tree structure by adding IS-A relationships.</span></li><li><span><strong>Root-addition Transformation:</strong> Create a new semantic type that will be an ancestor of all roots in the group. Make the new semantic type the group's root by adding additional IS-A relationships from all the roots of the group's connected components (either directly or through more new semantic types, if necessary).</span></li><li><span><strong>Root-moving Transformation:</strong> Locate a semantic type (from another group) that is a lowest common ancestor of the roots of all the disconnected group's subtrees and/or isolated semantic types. Move that lowest common ancestor into the disconnected group, making it the root of the group and thereby connecting the group. Also, move all the new root's existing descendants into the group.</span></li></ul> <p>The new network obtained by applying these transformations is called the <em>Enriched Semantic Network (ESN)</em>. It has a DAG structure rather than a two-tree structure. We now demonstrate the various transformations and show their impact on different disconnected groups.</p><p>As demonstrations of the IS-A addition transformation and split transformation, we consider the group <em>Disorders</em> (<a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig2" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 2</a>). This group contains 12 semantic types, 11 of which belong to three trees rooted at <strong>Pathologic Function, Anatomical Abnormality,</strong> and <strong>Finding</strong>, respectively. <strong>Injury or Poisoning</strong> is an isolated member of the group. Clearly, <em>Disorders</em> does not satisfy connectivity.  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig2" name="fig2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig2" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=400518&amp;blobname=195f02.gif" border="1" class="icon-reflink" alt="Figure 2." title="Figure 2."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 2.</strong></a><div class="figure-table-caption-in-article"><span><em>Disorders</em> group.</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>The IS-A addition transformation is first applied to this group to connect <strong>Injury or Poisoning</strong> to the tree rooted at <strong>Pathologic Function</strong>. Actually, <strong>Injury or Poisoning</strong> should have a subsumption relationship to <strong>Disease or Syndrome</strong> and inherit its semantic relationships. Thus, we add an IS-A link to capture this.</p><p>Because in the original SN, <strong>Disease or Syndrome</strong> is a descendant of <strong>Phenomenon or Process</strong>, the original IS-A from <strong>Injury or Poisoning</strong> to <strong>Phenomenon or Process</strong> can be removed because it can be inferred transitively through the new IS-A link from <strong>Injury or Poisoning</strong> to <strong>Disease or Syndrome</strong>.</p><p>At this point, the group is still a collection of disconnected trees. To rectify this, we apply the split transformation to form three new groups. According to the definitions of the 12 semantic types, we find that <strong>Pathologic Function</strong> and its six descendant semantic types, including the new descendant <strong>Injury or Poisoning</strong>, emphasize phenomenon or process and are in the <strong>Event</strong> tree, whereas the remaining semantic types emphasize an entity or object and are in the <strong>Entity</strong> tree. Furthermore, <strong>Anatomical Abnormality</strong> and its children are descendants of <strong>Physical Object</strong>, whereas <strong>Finding</strong> and its child are conceptual entities. So, it is natural to partition this group into three smaller connected groups, each comprising a tree. These groups, <em>Pathologic Function</em>, <em>Anatomical Abnormality</em>, and <em>Finding</em>, are shown in <a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig3" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 3</a>. Note that using a root-addition transformation for all or any two trees is not an option because this new root could not be placed anywhere in the SN due to the differences in the contents of the trees. The new groups are named after their roots.  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig3" name="fig3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig3" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=400518&amp;blobname=195f03.gif" border="1" class="icon-reflink" alt="Figure 3." title="Figure 3."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 3.</strong></a><div class="figure-table-caption-in-article"><span>Three new groups: (a) <em>Pathologic Function</em>, (b) <em>Anatomical Abnormality</em>, and (c) <em>Finding</em> (via IS-A Addition transformation and Split transformation).</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>In the next example, the <em>Anatomy</em> group undergoes a root-addition transformation; that is, we add new semantic types to make the group connected. The group contains a tree of seven semantic types rooted at <strong>Anatomical Structure</strong> and four isolated semantic types: <strong>Body Substance, Body System, Body Location or Region,</strong> and <strong>Body Space or Junction</strong> (<a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig4" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 4</a>). In carrying out this transformation, we follow the analysis of Michael et al.<sup><a href="#bib19" class="cite-reflink">19</a></sup> for definitions of anatomical concepts. For example, the new semantic type <strong>Material Physical Anatomical Entity</strong> is defined as &#x0201c;IS-A Physical Anatomical Entity which has a mass.&#x0201d;<sup><a href="#bib19" class="cite-reflink">19</a></sup> <strong>Body Substance</strong> is not an <strong>Anatomical Structure</strong> because it does not have a three-dimensional shape, but it is a <strong>Material Physical Anatomical Entity</strong> because it has mass. Thus, both <strong>Body Substance</strong> and <strong>Anatomical Structure</strong> are made children of the new semantic type <strong>Material Physical Anatomical Entity</strong>.  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig4" name="fig4"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig4" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=400518&amp;blobname=195f04.gif" border="1" class="icon-reflink" alt="Figure 4." title="Figure 4."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig4" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 4.</strong></a><div class="figure-table-caption-in-article"><span><em>Anatomy</em> group.</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>Furthermore, <strong>Body Space or Junction</strong> is not a <strong>Material Physical Anatomical Entity</strong>, but it is a <strong>Physical Anatomical Entity</strong> (defined in Michael et al.<sup><a href="#bib19" class="cite-reflink">19</a></sup> to have spatial dimensions). Hence, both <strong>Body Space or Junction</strong> and <strong>Material Physical Anatomical Entity</strong> are made children of the newly introduced <strong>Physical Anatomical Entity</strong>, which in turn IS-A <strong>Physical Object</strong>. The original IS-A from <strong>Anatomical Structure</strong> to <strong>Physical Object</strong> is cut because it can be inferred from the new IS-A from <strong>Anatomical Structure</strong> to <strong>Physical Anatomical Entity</strong>.</p><p>On the other hand, <strong>Body Location or Region</strong> and <strong>Body System</strong> do not have either mass or spatial dimension and thus cannot be descendants of <strong>Physical Anatomical Entity</strong>. Nevertheless, both obviously should belong to the <em>Anatomy</em> group. Following Michael et al.,<sup><a href="#bib19" class="cite-reflink">19</a></sup> we introduce the new semantic type <strong>Conceptual Anatomical Entity</strong>, which in turn IS-A <strong>Conceptual Entity</strong>, to complement <strong>Physical Anatomical Entity</strong> and serve as the parent of <strong>Body Location or Region</strong> and <strong>Body System</strong>.</p><p>Finally, the new semantic type <strong>Anatomical Entity</strong> is added as the parent of both <strong>Physical Anatomical Entity</strong> and <strong>Conceptual Anatomical Entity</strong>. In turn, <strong>Anatomical Entity</strong> IS-A <strong>Entity</strong>. In this way, the whole <em>Anatomy</em> group is transformed into the new group <em>Anatomical Entity</em> (<a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig5" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 5</a>). The dashed rectangles in the figure represent the newly added semantic types, and the dashed arrows represent the newly added IS-A links. We note that each of the four new semantic types should have at least the corresponding concepts suggested in Michael et al.<sup><a href="#bib19" class="cite-reflink">19</a></sup> assigned to it. (These concepts have been submitted to the NLM for inclusion in the next UMLS release [Rosse C, personal communication, 2002].)  </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig5" name="fig5"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig5" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=400518&amp;blobname=195f05.gif" border="1" class="icon-reflink" alt="Figure 5." title="Figure 5."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig5" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 5.</strong></a><div class="figure-table-caption-in-article"><span><em>Anatomical Entity</em> group.</span></div></td></tr></table></div></div><div style="clear:both;"></div><p>In the next example, the root-moving transformation is applied to the disconnected <em>Procedures</em> group to make it connected. The group contains seven semantic types, with two trees rooted at <strong>Health Care Activity</strong> and <strong>Research Activity</strong>, respectively, and the isolated <strong>Educational Activity</strong> (<a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig6" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 6</a>). These three are children of <strong>Occupational Activity</strong>, which has another child, <strong>Governmental or Regulatory Activity</strong>. Both of these semantic types, in turn, belong to the <em>Activities and Behaviors</em> group. In the context of the UMLS, these five semantic types refer to health care-related issues. They describe activities of health care professionals. Thus, <strong>Occupational Activity</strong>, the lowest ancestor of the seven semantic types in the group, and its child <strong>Governmental or Regulatory Activity</strong> are moved to this group. By doing this, the group is transformed into the new <em>Occupational Activity</em> connected group (<a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig7" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Fig. 7</a>).    </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig6" name="fig6"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig6" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=400518&amp;blobname=195f06.gif" border="1" class="icon-reflink" alt="Figure 6." title="Figure 6."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig6" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 6.</strong></a><div class="figure-table-caption-in-article"><span><em>Procedures</em> group.</span></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig7" name="fig7"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig7" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=400518&amp;blobname=195f07.gif" border="1" class="icon-reflink" alt="Figure 7." title="Figure 7."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig7" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 7.</strong></a><div class="figure-table-caption-in-article"><span><em>Occupational Activity</em> group.</span></div></td></tr></table></div></div><div style="clear:both;"></div></p></p><p><div class="head2 head-separate">String Matching</div>Additional IS-A links can be found by using string matching involving names and definitions of various semantic types in the SN. To be more formal, we define a string match as follows:<p><span class="head3">Definition (string match) </span> A string match from a semantic type <strong>T</strong><sub>1</sub> to another semantic type <strong>T</strong><sub>2</sub> is a triple (<strong>T</strong><sub>1</sub>; <strong>T</strong><sub>2</sub>; <em>S</em>) such that <em>S</em> is a string appearing both in the definition of <strong>T</strong><sub>1</sub> and in the name of <strong>T</strong><sub>2</sub>. <em>S</em> is called the common string and must contain one or more (not necessarily consecutive) complete words (ignoring case).<p>For example, the definition of <strong>Plant</strong> contains the word &#x0201c;organism,&#x0201d; which happens to be the name of a semantic type. Hence, a string match (<strong>Plant, Organism</strong>, &#x0201c;organism&#x0201d;) exists.</p><p>The motivation for using this kind of string matching to find viable new IS-A links is based on the evaluation of string matches among the 132 pairs of semantic types that currently have IS-A relationships between them in the SN. By analyzing the definitions of the children in the pairs, we found that there are string matches from 88 children to their respective parents. The string match (<strong>Plant, Organism</strong>, &#x0201c;organism&#x0201d;) is one of them. Thus, the sensitivity of this approach with <em>known</em> IS-A links is 67%. This finding leads us to the following observation.</p></p><p><span class="head3">Observation </span> If <strong>T</strong><sub>1</sub> IS-A <strong>T</strong><sub>2</sub>, then there is a high likelihood of a string match from <strong>T</strong><sub>1</sub> to <strong>T</strong><sub>2</sub>.<p>This leads us to formulate the inverse hypothesis.</p></p><p><span class="head3">Hypothesis </span> If there is a string match from one semantic type to another, then it is likely to imply a viable subsumption relationship between them.<p>Based on this hypothesis, we developed the string matching method to identify additional viable IS-A relationships not already appearing in the SN. Our methodology is a human&#x02013;computer interactive methodology and contains three steps:  <ul style="list-style-type: decimal;"><li><span>Preprocess names and definitions of semantic types to obtain the input file;</span></li><li><span>Apply the &#x0201c;AllMatches&#x0201d; algorithm to the input file to get all string matches; and</span></li><li><span>Manually review all resulting string matches and determine which constitute additional viable IS-A links between semantic types.</span></li></ul> </p><p>In step 1, we use some common techniques from the data mining and information retrieval fields.<sup><a href="#bib20" class="cite-reflink">20</a></sup>  <ul style="list-style-type: none;"><li><span><strong>Stop-words:</strong> All stop-words such as &#x0201c;a,&#x0201d; &#x0201c;the,&#x0201d; &#x0201c;of,&#x0201d; &#x0201c;for,&#x0201d; &#x0201c;with,&#x0201d; and so on are removed from names and definitions.</span></li><li><span><strong>Verb variant processing:</strong> All verbs and verb variants are removed from definitions of semantic types. In the string matching, we do not consider verbs and verb variants. The reason is that most semantic types' names consist only of nouns, adjectives, and adverbs.</span></li><li><span><strong>Lexical normalization:</strong> The Specialist Lexicon (coupled with highly efficient &#x0201c;lexical variant generator&#x0201d; code)<sup><a href="#bib21" class="cite-reflink">21</a></sup> is applied to stem-word variations. All adjectives and adverbs are converted to nouns, and all plurals are converted to singular forms. Also, uppercase letters are changed to corresponding lowercase.</span></li></ul> </p><p>In step 2, the following AllMatches algorithm is used to find string matches between any two semantic types not currently connected by a single IS-A link or a path of such links. The input file to the algorithm contains the names and definitions of semantic types after the preprocessing step.</p><p>In the description of the AllMatches algorithm, we assume that <strong>T</strong><sub>1</sub>, <strong>T</strong><sub>2</sub>, &#x02026;, <strong>T</strong><sub><em>m</em></sub> are all semantic types in the SN. (In the 2002 version, <em>m</em> &#x0003d; 134.) We use the notation <em>DEF</em>(<strong>T</strong><sub><em>i</em></sub>) to represent the definition of the semantic type <strong>T</strong><sub><em>i</em></sub>, 1&#x02264;i&#x02264;134, after preprocessing. <em>NAME</em>(<strong>T</strong><sub><em>i</em></sub>) is used to represent the name of <strong>T</strong><sub><em>i</em></sub>, in the form of a string, after preprocessing. For example, suppose <strong>T</strong><sub><em>i</em></sub> &#x0003d; <strong>Anatomical Structure</strong>, which is defined as: &#x0201c;a normal or pathological part of the anatomy or structural organization of an organism.&#x0201d; After preprocessing, <em>NAME</em>(<strong>T</strong><sub><em>i</em></sub>) &#x0003d; &#x0201c;anatomy structure&#x0201d; and <em>DEF</em>(<strong>T</strong><sub><em>i</em></sub>) &#x0003d; &#x0201c;normal pathology part anatomy structure organization organism.&#x0201d;</p><p>In the following AllMatches algorithm, we use a list <em>L</em> to hold all common strings. We also use the following functions defined for lists:  <ul style="list-style-type: none;"><li><span>Length(): Return the number of elements in the list</span></li><li><span>Retrieve(<em>k</em>): Retrieve the <em>k</em>th element of the list</span></li><li><span><strong>AllMatches algorithm: Find all string matches in the SN.</strong></span></li><li><span>For (<em>i</em> &#x0003d; 1 to <em>m</em>)</span></li><li><span><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />For (all <strong>T</strong><em><sub>j</sub></em>, 1&#x02264;<em>j</em>&#x02264;<em>m</em> &#x00026; <em>j &#x02260; i</em>)</span></li><li><span><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />If (<strong>T</strong><em><sub>j</sub></em> is not the parent or an ancestor of <strong>T</strong><em><sub>i</sub></em>)</span></li><li><span><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />&#x0007b; <em>L</em> &#x0003d; FindCommonStrings(<em>DEF</em>(<strong>T</strong><em><sub>i</sub></em>),<em>NAME</em>(<strong>T</strong><em><sub>j</sub></em>));</span></li><li><span><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />//write string matches to the output file</span></li><li><span><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />For (<em>k</em> &#x0003d; 1 to <em>L</em>.Length())</span></li><li><span><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />&#x0007b; <em>S &#x0003d; L</em>.Retrieve(<em>k</em>); // get <em>k</em>th element of the list</span></li><li><span><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />write (<strong>T</strong><em><sub>i</sub></em>; <strong>T</strong><em><sub>j</sub></em>; <em>S</em>) to output file;</span></li><li><span><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />&#x0007d;<img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " /><img src="corehtml/pmc/pmcents/x2003.gif" border="0" alt=" " />&#x0007d;</span></li></ul> </p><p>The function FindCommonStrings(<em>R</em><sub>1</sub>, <em>R</em><sub>2</sub>) is used to find all common strings involving a given pair of strings <em>R</em><sub>1</sub> and <em>R</em><sub>2</sub>. During a call, <em>R</em><sub>1</sub> is the definition of a semantic type <strong>T</strong><sub><em>i</em></sub> in a string format, and <em>R</em><sub>2</sub> is the name of a semantic type <strong>T</strong><sub><em>j</em></sub> as a string. For each pair (<strong>T</strong><sub><em>i</em></sub>, <strong>T</strong><sub><em>j</em></sub>) that has no direct IS-A relationship or directed path of IS-A relationships between its components, we call FindCommonStrings(<em>DEF</em>(<strong>T</strong><sub><em>i</em></sub>), <em>NAME</em>(<strong>T</strong><sub><em>j</em></sub>)) to get all possible common strings between <em>DEF</em>(<strong>T</strong><sub><em>i</em></sub>) and <em>NAME</em>(<strong>T</strong><sub><em>j</em></sub>). Each such common string is inserted into <em>L</em>. We say that a match M is redundant if its constituent common string <em>S</em> is a substring of another match's common string (again, ignoring case). FindCommonStrings(<em>DEF</em>(<strong>T</strong><sub><em>i</em></sub>), <em>NAME</em>(<strong>T</strong><sub><em>j</em></sub>)) identifies the redundant matches and does not return them. So, <em>L</em> contains no redundant common strings. Finally, all string matches (<strong>T</strong><sub><em>i</em></sub>; <strong>T</strong><sub><em>j</em></sub>; <em>S</em>) are written to the output file. After AllMatches has been executed, we have a file containing all string matches between pairs of semantic types not connected by IS-A relationships in the SN.</p><p>As an example, consider <strong>Enzyme</strong> whose definition is &#x0201c;a complex chemical, usually a protein, that is produced by living cells and which catalyzes specific biochemical reactions.&#x0201d; The AllMatches algorithm finds three string matches:  <ul style="list-style-type: none;"><li><span>(<strong>Enzyme, Cell,</strong> &#x0201c;cell&#x0201d;)</span></li><li><span>(<strong>Enzyme, Cell Component,</strong> &#x0201c;cell&#x0201d;)</span></li><li><span>(<strong>Enzyme; Amino Acid, Peptide, or Protein;</strong> &#x0201c;protein&#x0201d;)</span></li></ul> </p><p>In step 3, an expert is called on to review all resulting string matches to find new IS-A links not currently appearing in the SN. These newly discovered IS-A links can then be added to the ESN. As it happens, in the case of the three string matches involving <strong>Enzyme</strong>, the third match implies the existence of a new IS-A link, because any enzyme must be a kind of protein. Hence, <strong>Enzyme</strong> IS-A <strong>Amino Acid, Peptide,</strong> or <strong>Protein</strong>.</p><p>As noted previously in this article, the sensitivity of the string matching approach, when applied to <em>known</em> IS-A links, is 67%. To determine the sensitivity of our method for detecting <em>unknown</em> IS-A links, we established a gold standard by performing a manual review of randomly generated relationship pairs.</p></p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id441162" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id455913" class="sidefm-pmclink" style="text-transform: none;">Methods to Enhance the SN'S IS-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />A Hierarchy</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Results</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457737" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458169" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458193" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id457388">Results</div><div class="section-content"><p><div class="head2 head-separate">Results of Imposing Connectivity on the Partition</div>Besides the three disconnected groups described in Methods, the original partition of McCray et al.<sup><a href="#bib16" class="cite-reflink">16</a></sup> contains six other disconnected groups. <a href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a> presents the six groups and the six transformations applied to them.  <div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="tbl1" name="tbl1"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl1" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 1." title="Table 1."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl1" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 1.</strong></a><div class="figure-table-caption-in-article"><span>The Six Transformations Applied to Six Disconnected Groups of the Partition of McCray et al.<sup><a href="#bib16" class="cite-reflink">16</a></sup></span></div></td></tr></table></div></div><div style="clear:both;"></div><p>For each such group, <a href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a> shows the isolated semantic types or trees that existed in the group and the transformations used. In the second column of <a href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl1" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 1</a>, each tree in the group is denoted by placing its constituent semantic types in braces &#x0201c;&#x0007b;&#x0007d;.&#x0201d; In the fourth column, we use the notation (<strong>A</strong> IS-A <strong>B</strong>) to denote a single IS-A link that was added to the group, where <strong>A</strong> and <strong>B</strong> are semantic types. The new groups are named after their respective roots.</p><p>Overall, using the four kinds of transformations, we converted all disconnected groups into new connected groups, each with an internal tree structure. During this process, a total of ten transformations were applied: the IS-A addition transformation was used four times; the split transformation was used three times; the root-addition transformation was used once (on the <em>Anatomy</em> group); and the root-moving transformation was used twice. Note that multiple transformations might have been applied to a single group. For example, see the <em>Disorders</em> group. The application of the four transformations yielded the preliminary ESN with 15 new IS-A links. Its DAG structure allows semantic types to have multiple parents.</p><p>A total of 19 disjoint groups, which together constitute a partition of the ESN, was also obtained. See <a href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl2" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 2</a>, where we use an asterisk (<sup>&#x0002a;</sup>) to denote a group different from that originally appearing in McCray et al.<sup><a href="#bib16" class="cite-reflink">16</a></sup> Each group is a connected subgraph of the ESN. Hence, the partition satisfies the connectivity property preferred for semantic validity.  </p><div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="tbl2" name="tbl2"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl2" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 2." title="Table 2."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl2" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 2.</strong></a><div class="figure-table-caption-in-article"><span>Partition of the ESN into 19 Connected Groups</span></div></td></tr></table></div></div><div style="clear:both;"></div></p><p><div class="head2 head-separate">Results of String Matching</div>For our manual review, we randomly selected 550 (3%) of the 17,396 possible pairs of semantic types for which no ancestor/descendant relationship currently exists. Neither of the two reviewers judged any of the 550 pairs to represent a true parent&#x02013;child relationship. This corresponds to a prevalence of unknown pairs of 0%, with a 95% confidence interval of 0&#x02013;0.54%.<p>A total of 665 string matches were found by our algorithm. Only five of these were judged to represent true parent&#x02013;child relationships, for a precision of 0.75%. However, these five positive results suggest a prevalence of 0.029% (five of 17,396), which is within the 95% confidence interval of our gold standard analysis.</p><p>Our semantic method resulted in the addition of 15 new IS-A links. However, 11 of these links involved the addition of new semantic types, leaving four previously undiscovered IS-A links. One of these, <strong>Gene or Genome</strong> IS-A <strong>Molecular Sequence</strong>, was also detected by the string matching method. Thus, a total of eight new parent&#x02013;child relationships were discovered (prevalence eight of 17,396 &#x0003d; 0.046%, still within the range found by the gold standard). The string matching method detected five of the eight true parent&#x02013;child relationships discovered by both methods, yielding a sensitivity (or recall) of 62.5%. At the maximum prevalence suggested by the 95% confidence interval (0.54%), the sensitivity could be as low as 5.3%.</p><p>Let us review the four additional IS-A links. One is the new IS-A link from <strong>Enzyme</strong> to <strong>Amino Acid, Peptide, or Protein,</strong> which was demonstrated above.</p><p>Another example relates to <strong>Receptor</strong>, for which there were five string matches:  <ul style="list-style-type: none;"><li><span>(<strong>Receptor, Cell Component,</strong> &#x0201c;cell&#x0201d;)</span></li><li><span>(<strong>Receptor, Cell,</strong> &#x0201c;cell&#x0201d;)</span></li><li><span>(<strong>Receptor, Anatomical Structure,</strong> &#x0201c;structure&#x0201d;)</span></li><li><span>(<strong>Receptor; Amino Acid, Peptide, or Protein;</strong> &#x0201c;protein&#x0201d;)</span></li><li><span>(<strong>Receptor, Hormone,</strong> &#x0201c;hormone&#x0201d;)</span></li></ul> </p><p>In accordance with the review of the domain expert, an IS-A link from <strong>Receptor</strong> to <strong>Cell Component</strong> was added. The other string matches did not imply IS-A links.</p><p>The third valid IS-A link involves <strong>Vitamin</strong>, which had four matches:  <ul style="list-style-type: none;"><li><span>(<strong>Vitamin, Pharmacologic Substance,</strong> &#x0201c;substance&#x0201d;)</span></li><li><span>(<strong>Vitamin, Organic Chemical,</strong> &#x0201c;organic chemical&#x0201d;)</span></li><li><span>(<strong>Vitamin, Body Substance,</strong> &#x0201c;substance&#x0201d;)</span></li><li><span>(<strong>Vitamin, Animal,</strong> &#x0201c;animal&#x0201d;)</span></li></ul> </p><p>Based on the domain expert's review, two IS-A links were added: one IS-A from <strong>Vitamin</strong> to <strong>Pharmacologic Substance</strong>, and another IS-A from <strong>Vitamin</strong> to <strong>Organic Chemical</strong>.</p></p><p><div class="head2 head-separate">Summary of Results of Two Methodologies</div>After adding the new IS-A links derived by our two methodologies, we get the new ESN. Compared with the original SN, the ESN has four new semantic types and 19 new IS-A links. Two IS-A links appearing in the SN were not included in the ESN. Hence, the ESN has 149 IS-A links and 138 semantic types, among which 12 semantic types (approximately 8%) have multiple parents, giving the ESN a DAG-structured IS-A (subsumption) hierarchy. See <a href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl3" class="fig-table-link" onclick="startTarget(this, 'table', 1024, 800)">Table 3</a> for these 12 semantic types and their parents.  <div style="margin: 1em; margin-right: 2em;                                 border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="tbl3" name="tbl3"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl3" class="icon-reflink" onclick="startTarget(this, 'table', 1024, 800)"><img src="corehtml/pmc/pmcgifs/table-icon.gif" border="1" class="icon-reflink" alt="Table 3." title="Table 3."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=table&amp;id=tbl3" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Table 3.</strong></a><div class="figure-table-caption-in-article"><span>Semantic Types (STs) with Multiple Parents in the ESN</span></div></td></tr></table></div></div><div style="clear:both;"></div><p><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig8" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 8</a> shows the portion of the ESN's hierarchy rooted at <strong>Event</strong>, and <a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig9" class="fig-table-link" onclick="startTarget(this, 'figure', 1024, 800)">Figure 9</a> shows part of the portion rooted at <strong>Entity</strong>. To emphasize the changes from the original SN, we use dashed arrows to denote the new IS-A links and thick dashed rectangles to denote new semantic types. Thin dashed rectangles denote semantic types that originally resided in the other tree of the SN. Ellipses in a rectangle indicate that the names of one or several semantic types are not shown due to lack of space.    </p><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig8" name="fig8"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig8" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=400518&amp;blobname=195f08.gif" border="1" class="icon-reflink" alt="Figure 8." title="Figure 8."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig8" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 8.</strong></a><div class="figure-table-caption-in-article"><span><strong>Event</strong> Portion of the Enriched Semantic Network.</span></div></td></tr></table></div></div><div style="clear:both;"></div><div style="margin: 1em; margin-right: 2em;        border: 1px solid #999999;border-left: 1px solid #AAAAAA;border-bottom: 1px solid #AAAAAA;"><div style="border: 3px solid #F0F0F0; border-left: 1px solid #F8F8F8; border-bottom: 1px solid #F8F8F8;"><a id="fig9" name="fig9"></a><table border="0" cellpadding="5" cellspacing="5" style="clear:both; width: 100%;"><tr valign="top" align="left"><td width="100" align="center"><a href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig9" class="icon-reflink" onclick="startTarget(this, 'figure', 1024, 800)"><img src="picrender.fcgi?artid=400518&amp;blobname=195f09.gif" border="1" class="icon-reflink" alt="Figure 9." title="Figure 9."></a></td><td><a class="side-caption" href="articlerender.fcgi?artid=400518&amp;rendertype=figure&amp;id=fig9" onclick="startTarget(this, 'figure', 1024, 800)"><strong>Figure 9.</strong></a><div class="figure-table-caption-in-article"><span>Part of the <strong>Entity</strong> portion of the Enriched Semantic Network.</span></div></td></tr></table></div></div><div style="clear:both;"></div></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id441162" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id455913" class="sidefm-pmclink" style="text-transform: none;">Methods to Enhance the SN'S IS-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />A Hierarchy</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457388" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>Discussion</a></div><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458169" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458193" class="sidefm-pmclink" style="text-transform: none;">References</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id457737">Discussion</div><div class="section-content"><p><div class="head2 head-separate">Advantages of the Enriched Semantic Network</div>The ESN has 12 semantic types with multiple parents. As it happens, most such semantic types are leaves or parents of leaves. As such, the changes are local, not influencing other semantic types. An exception is the modeling of the four new semantic types, <strong>Anatomical Entity</strong>, its two children <strong>Conceptual Anatomical Entity</strong> and <strong>Physical Anatomical Entity</strong>, and the child of the latter, <strong>Material Physical Anatomical Entity</strong>. This is the most visible difference from the original SN's two-tree structure, because it happens close to the root <strong>Entity</strong> rather than at the bottom levels of the SN. As such, it is not a local change.<p>The ESN has a number of advantages over the original SN. The multiple subsumption hierarchy enables better modeling of IS-A relationships for those semantic types having multiple parents. In the ESN, some semantic types will have more semantic relationships than they had in the SN. Specifically, semantic types with multiple parents will inherit relationships independently from each of those parents. Thus, such a semantic type will have a larger relationship set than before. For example, <strong>Organism Attribute</strong> and its child <strong>Clinical Attribute</strong> will now have a relationship <em>result_of</em> to <strong>Anatomical Abnormality</strong>. This relationship is inherited by <strong>Organism Attribute</strong> from its new parent <strong>Physiologic Function</strong>, and further inherited by its child <strong>Clinical Attribute</strong>.</p><p>One might consider the introduction of multiple inheritance as a potential problem in that inconsistent information from different parents might be inherited. However, when the placement of a concept into two classes is semantically correct, then the inheritance of definitional attributes from multiple parents is, by definition, also correct. Multiple inheritance will allow the identification of inconsistencies that were already there implicitly; it will not introduce new ones.</p><p>The addition of the IS-A links helps to expose missing classifications of concepts of the META to semantic types. Let us demonstrate this with regard to the concepts assigned to the semantic type <strong>Vitamin</strong>.</p><p>We checked all 1,204 concepts from the META assigned to <strong>Vitamin</strong> and found that 957 are also assigned to <strong>Pharmacologic Substance</strong>. One of them is also assigned to <strong>Antibiotic</strong>, which is a child of <strong>Pharmacologic Substance</strong>. The other 246 concepts are not assigned to <strong>Pharmacologic Substance.</strong> For example, the concept F<span class="small-caps">olate</span><sup><a href="#fn3" class="cite-reflink">&#x02021;</a></sup> is not assigned to <strong>Pharmacologic Substance</strong>. However, we know that some drugs, for example, vitamins given to pregnant women, contain folate to prevent possible congenital deficiencies of the baby. Hence, F<span class="small-caps">olate</span> should indeed be assigned to <strong>Pharmacologic Substance</strong>. As a matter of fact, all the remaining 246 concepts should also have been assigned to <strong>Pharmacologic Substance</strong> because all vitamins can be ingredients of drugs.</p><p>Similarly, all concepts in <strong>Vitamin</strong> should also be assigned to <strong>Organic Chemical</strong> or one of its descendants. Among the 1,204 concepts assigned to <strong>Vitamin</strong>, 735 are also assigned to <strong>Organic Chemical</strong> or children of <strong>Organic Chemical</strong>. However, there are 469 concepts assigned to <strong>Vitamin</strong> that are not assigned to <strong>Organic Chemical</strong> or to any of its children. An example is 24,25-D<span class="small-caps">ihydroxyvitamin</span> D, which is a kind of vitamin D that is helpful for the absorption of calcium and is certainly an organic chemical. In fact, all these 469 concepts should have been assigned to <strong>Organic Chemical</strong>.</p><p>Another advantage of the multiple-parent hierarchy is that it can simplify the assignment of META's concepts to semantic types. An important rule promoted by the SN's designers states that a concept should be explicitly assigned to the most specialized possible semantic type in the SN's IS-A hierarchy.<sup><a href="#bib6" class="cite-reflink">6</a></sup> Suppose a concept was assigned to two semantic types, <strong>T</strong><sub>1</sub> and <strong>T</strong><sub>2</sub>, that originally had no IS-A path between them in the SN. If in the ESN there is a direct IS-A link or path from <strong>T</strong><sub>1</sub> to <strong>T</strong><sub>2</sub> (i.e., <strong>T</strong><sub>2</sub> is now a parent or ancestor of <strong>T</strong><sub>1</sub>), then the assignment of the concept to <strong>T</strong><sub>2</sub> is considered redundant<sup><a href="#bib22" class="cite-reflink">22</a></sup><sup>,</sup><sup><a href="#bib23" class="cite-reflink">23</a></sup> and should be removed because it can be inferred from the assignment to <strong>T</strong><sub>1</sub>.</p><p>As an example, consider the new IS-A link from <strong>Vitamin</strong> to <strong>Pharmacologic Substance</strong>. After adding this IS-A link, the 957 assignments of <strong>Vitamin</strong>'s concepts to <strong>Pharmacologic Substance</strong> should be removed because <strong>Vitamin</strong> is now more specialized than <strong>Pharmacologic Substance</strong> in the network. The 957 concepts should only be assigned to <strong>Vitamin</strong>, because implicit assignments to <strong>Pharmacologic Substance</strong> can be inferred via the new IS-A. After the addition of <strong>Vitamin</strong> IS-A <strong>Organic Chemical</strong>, the 735 assignments to <strong>Organic Chemical</strong> should also be removed for the same reason.</p><p>Let us note that in the preceding discussion, we proposed both the assignment of additional concepts to <strong>Organic Chemical</strong> and then their subsequent removal. However, the proposed additions were strictly in the context of the current SN hierarchy, in which simultaneous assignment to <strong>Vitamin</strong> and <strong>Organic Chemical</strong> is not redundant and is in fact warranted. In the ESN, with <strong>Vitamin</strong> now being a child of <strong>Organic Chemical</strong>, such assignments become redundant and therefore unnecessary. This further supports the validity of the new IS-A link and demonstrates that the ESN hierarchy requires fewer explicit assignments of META's concepts to the semantic types.</p><p>The partition of the ESN can enable the design of a metaschema,<sup><a href="#bib24" class="cite-reflink">24</a></sup> a higher-level abstraction network that can aid user orientation. Among other things, a metaschema allows a user to focus on a subject area of interest without losing sight of the overall ESN layout. Previously, we developed metaschemas for the current SN. The connected partition of the ESN allows us to do the same for the ESN.<sup><a href="#bib17" class="cite-reflink">17</a></sup></p><p>Regarding limitations, our first methodology was applied only to the partition presented in McCray et al.,<sup><a href="#bib16" class="cite-reflink">16</a></sup> and decisions were made with respect to the current definitions of semantic types. Of course, there are many possible partitions of the SN. If we consider other partitions, we might decide on different IS-A links.</p><p>The string matching methodology is dependent on the definitions of the semantic types. We realize that the current definitions are not necessarily the only ones possible for the given semantic types. Another team of designers might come up with slightly different definitions. Because we used the exact wording of the definitions, our results may very well be altered by alternative definitions. Furthermore, the average time complexity of the algorithm is approximately <em>O</em>(<em>n</em><sup>2</sup>), and this limits its scalability. It is thus applicable only to a compact upper-level abstraction ontology (like the SN), not a full-scale ontology. For example, the algorithm would be very time-consuming if it were applied to find string matches for the META's concepts. Word-level synonymy (or phrase synonymy) was not considered in our algorithm. If used, it could increase the string match cases and maybe the number of new viable IS-A links found. However, this would likely erode the algorithm's efficiency, which is already low, and might increase the number of false-positives, which is already high.</p></p><p><div class="head2 head-separate">Evaluation</div>Without an exhaustive examination of all 17,396 pairs of unrelated semantic types, it is impossible to know the exact prevalence of undiscovered parent&#x02013;child relationships. However, all of the methods we used (semantic modeling, manual review, and string matching) suggest that the number of such relationships is very low. In the absence of a precise figure for prevalence, estimating the sensitivity of our automated methods is impossible. However, the semantic modeling revealed 15 links and the string matching revealed five links; either of these counts represents a significant contribution to the number of links in the SN; taken together as 19 (because one was repeated), they increase the number of links by 14.3%.<p>Although at first glance, the precision of the string matching method (0.75%) appears poor, applying it to the SN reduces the number of semantic-type pairs that must be manually reviewed by 94%. We note that there is no inherent reason why a string match from the definition of a semantic type to the name of another semantic type would necessarily indicate an IS-A link. The match could indicate another kind of connection such as a semantic relationship. Our hypothesis was that if there is a shared string, then the likelihood of a parent&#x02013;child relationship is substantially higher. The results support our hypothesis.</p></p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id458169">Conclusion</div><div class="section-content"><p>We have enhanced the UMLS's SN hierarchy by adding new IS-A links and new semantic types to accommodate multiple parents. We obtained a new semantic network that has a DAG structure instead of a two-tree structure. This new semantic network, containing 138 semantic types and 149 IS-A relationships, is referred to as the ESN. The ESN can express cases of multiple subsumption for some semantic types. Furthermore, a partition of the ESN comprising 19 groups was derived; each group in the partition exhibits connectivity and semantic uniformity. This new partition can enable the design of a metaschema<sup><a href="#bib17" class="cite-reflink">17</a></sup> to help further improve user orientation to the ESN.</p></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"> </td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="notes">Notes</div><div class="section-content"><!--front--><div><div class="fm-footnote"><div class="p">This research was partially supported by contract &#x00023;N01-1-3543 from the National Library of Medicine (NLM), and by the New Jersey Commission for Science and Technology.</div></div><div class="fm-footnote" id="fn1"><sup>&#x0002a;</sup>Semantic types appear in boldface type.</div><div class="fm-footnote" id="fn2"><sup>&#x02020;</sup>Group names appear in italic type.</div><div class="fm-footnote" id="fn3"><sup>&#x02021;</sup>Concept names appear in small caps.</div></div></div></td></tr><tr valign="top"><td class="sidebar-cell" width="145"><div class="side-section-group"><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#top" class="sidefm-pmclink" style="text-transform: none;">Top</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id441162" class="sidefm-pmclink" style="text-transform: none;">Abstract</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id455913" class="sidefm-pmclink" style="text-transform: none;">Methods to Enhance the SN'S IS-<img src="corehtml/pmc/pmcents/dot.gif" border="0" />A Hierarchy</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457388" class="sidefm-pmclink" style="text-transform: none;">Results</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id457737" class="sidefm-pmclink" style="text-transform: none;">Discussion</a></div></span><span style="text-transform: none;"><div class="sidefm-pmclink-item"><a href="#id458169" class="sidefm-pmclink" style="text-transform: none;">Conclusion</a></div></span><div class="sidefm-pmccurrent-item"><a class="sidefm-pmclink" href="javascript:return(false);" style="text-transform: none;"><span class="sidebar-menu-square-image-holder"><img border="0" src="corehtml/pmc/pmcgifs/square.gif" alt="&gt;"></span>References</a></div></div></td><td class="content-cell"><div class="head1 section-title" style="text-transform: none;" id="id458193">References</div><div class="section-content"><div class="back-matter-section"><div class="ref-cit-blk" id="bib1"><div class="ref-label">1.</div><div class="ref-cit">Campbell KE, Oliver DE, Shortliffe EH. The Unified Medical Language System: toward a collaborative approach for solving terminologic problems.  <span class="ref-journal">J Am Med Inform Assoc.</span> 1998;<span class="ref-vol">5</span>:12&#x02013;6.   <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0ps5JBdLOOfuOPZFd_-8zirGrVfc7riP3HARKM5SW&amp;reftype=pubmed&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9452982">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="bib2"><div class="ref-label">2.</div><div class="ref-cit">Humphreys BL, Lindberg DAB, Schoolman HM, Barnett GO. The Unified Medical Language System: an informatics research collaboration.  <span class="ref-journal">J Am Med Inform Assoc.</span> 1998;<span class="ref-vol">5</span>:1&#x02013;11.   <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Ruef0Rv0du9nYY80_r8z2jPkSSBsHArw5CASRAwB&amp;reftype=pubmed&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9452981">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="bib3"><div class="ref-label">3.</div><div class="ref-cit">US Department of Health and Human Services, National Institutes of Health, National Library of Medicine. Washington, DC: Unified Medical Language System (UMLS), 2003. </div></div><div class="ref-cit-blk" id="bib4"><div class="ref-label">4.</div><div class="ref-cit">McCray AT, Hole WT. The scope and structure of the first version of the UMLS Semantic Network.  <span class="ref-journal">Proc Annu Symp Comput Appl Med Care.</span> 1990:126&#x02013;30. </div></div><div class="ref-cit-blk" id="bib5"><div class="ref-label">5.</div><div class="ref-cit">McCray AT. Representing biomedical knowledge in the UMLS Semantic Network. In: Broering NC, (ed). High-Performance Medical Libraries: Advances in Information Management for the Virtual Era. Westport, CT: Mekler, 1993, pp. 45&#x02013;55. </div></div><div class="ref-cit-blk" id="bib6"><div class="ref-label">6.</div><div class="ref-cit">McCray AT, Nelson SJ. The representation of meaning in the UMLS.  <span class="ref-journal">Methods Inf Med.</span> 1995;<span class="ref-vol">34</span>:193&#x02013;201.   <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0tKC5qmRF10DN_Qa7BLDWGsUhV04KDYovu6ggUkPi&amp;reftype=pubmed&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9082131">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="bib7"><div class="ref-label">7.</div><div class="ref-cit">McCray AT. UMLS Semantic Network.  <span class="ref-journal">Proc Annu Symp Comput Appl Med Care.</span> 1989:503&#x02013;7. </div></div><div class="ref-cit-blk" id="bib8"><div class="ref-label">8.</div><div class="ref-cit">Chute CG. The Copernican era of healthcare terminology: a re-centering of health information systems.  <span class="ref-journal">Proc AMIA Annu Symp.</span> 1998:68&#x02013;73. </div></div><div class="ref-cit-blk" id="bib9"><div class="ref-label">9.</div><div class="ref-cit">Cimino JJ. Desiderata for controlled medical vocabularies in the twenty-first century.  <span class="ref-journal">Methods Inf Med.</span> 1998;<span class="ref-vol">37</span>:394&#x02013;403.   <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0jE19DMdwjnAmSwaqcgAqUxGfz-1kQGs6t4s0R50l&amp;reftype=pubmed&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9865037">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="bib10"><div class="ref-label">10.</div><div class="ref-cit">Campbell KE, Das AK, Musen MA. A logical foundation for representation of clinical data.  <span class="ref-journal">J Am Med Inform Assoc.</span> 1994;<span class="ref-vol">1</span>:218&#x02013;32.   <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Xi1CW4eFQtfeNkcWtArVFefbbz0I6a-hubS0BkGy&amp;reftype=pubmed&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/7719805">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="bib11"><div class="ref-label">11.</div><div class="ref-cit">Campbell KE, Cohn SP, Chute CG, Shortliffe EH, Rennels G. Scalable methodologies for distributed development of logic-based convergent medical terminology.  <span class="ref-journal">J Am Med Inform Assoc.</span> 1998;<span class="ref-vol">37</span>:426&#x02013;39. </div></div><div class="ref-cit-blk" id="bib12"><div class="ref-label">12.</div><div class="ref-cit">Rector AL, Rogers J, Roberts A, Wroe C. Scale and context: issues in ontologies to link health and bio-informatics.  <span class="ref-journal">Proc AMIA Annu Symp.</span> 2002:642&#x02013;4. </div></div><div class="ref-cit-blk" id="bib13"><div class="ref-label">13.</div><div class="ref-cit">Rector AL. Thesauri and formal classifications: terminologies for people and machines.  <span class="ref-journal">Methods Inf Med.</span> 1998;<span class="ref-vol">37</span>:501&#x02013;9.   <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0Nub6qc5oE2_12LeaOXfBOOoZ_VhEyzAeBRbGKQue&amp;reftype=pubmed&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9865048">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="bib14"><div class="ref-label">14.</div><div class="ref-cit">Rossi Mori A, Consorti F, Galeazzi E. Standards to support development of terminological systems for healthcare telematics.  <span class="ref-journal">Methods Inf Med.</span> 1998;<span class="ref-vol">37</span>:551&#x02013;63.   <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0LY1UBm4Q3Dec2YKnOLTMxGtvXuXTSTcA4hRK0Qi-&amp;reftype=pubmed&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/9865053">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="bib15"><div class="ref-label">15.</div><div class="ref-cit">Dessena S, Rossi Mori A, Galeazzi E. Building cross-thesauri with the support of UMLS. In: Cesnik B, (ed). Medinfo 98. Amsterdam, The Netherlands: IOS Press, 1998, pp. 654&#x02013;9. </div></div><div class="ref-cit-blk" id="bib16"><div class="ref-label">16.</div><div class="ref-cit">McCray AT, Burgun A, Bodenreider O. Aggregating UMLS semantic types for reducing conceptual complexity. Medinfo 2001. Amsterdam: IOS Press, 2001, pp 171&#x02013;5. </div></div><div class="ref-cit-blk" id="bib17"><div class="ref-label">17.</div><div class="ref-cit">Zhang L, Perl Y, Halper M, Geller J. Designing metaschemas for the UMLS Enriched Semantic Network.  <span class="ref-journal">J Biomed Inform.</span> 2003;<span class="ref-vol">36</span>:433&#x02013;49.   <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=0TU8ywNJpVKGepJcj2dy1LslazrScUDhPr33B75QC&amp;reftype=pubmed&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/14759817">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="bib18"><div class="ref-label">18.</div><div class="ref-cit">Chen Z, Perl Y, Halper M, Geller J, Gu H. Partitioning the UMLS Semantic Network.  <span class="ref-journal">IEEE Trans Inf Technol Biomed.</span> 2002;<span class="ref-vol">6</span>:102&#x02013;8.   <span style="white-space: nowrap;">[<a class="ref-extlink" href="redirect3.cgi?&amp;&amp;auth=06ODdVXG8x64maBAx1IiVARTHUozlHqLstQyBjQb_&amp;reftype=pubmed&amp;artid=400518&amp;article-id=400518&amp;iid=134762&amp;issue-id=134762&amp;jid=76&amp;journal-id=76&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord&amp;rendering-type=normal&amp;&amp;http://www.ncbi.nlm.nih.gov/pubmed/12075663">PubMed</a>]</span>.</div></div><div class="ref-cit-blk" id="bib19"><div class="ref-label">19.</div><div class="ref-cit">Michael J, Mejino J, Rosse C. The role of definitions in biomedical concept representation. In: Bakken S (ed). Proc AMIA Annu Symp. 2001:463&#x02013;7. </div></div><div class="ref-cit-blk" id="bib20"><div class="ref-label">20.</div><div class="ref-cit">Han J, Kamber M. Data Mining: Concepts and Techniques. San Francisco, CA: Morgan Kaufmann Publishers, 2000. </div></div><div class="ref-cit-blk" id="bib21"><div class="ref-label">21.</div><div class="ref-cit">McCray AT, Srinivasan S, Browne AC. Lexical methods for managing variation in biomedical terminologies.  <span class="ref-journal">Proc Annu Symp Comput Applic Med Care.</span> 1994:235&#x02013;9. </div></div><div class="ref-cit-blk" id="bib22"><div class="ref-label">22.</div><div class="ref-cit">Gu H, Perl Y, Geller J, Halper M, Liu L, Cimino JJ. Representing the UMLS as an OODB: modeling issues and advantages. J Am Med Inform Assoc. 2000;7:66&#x02013;80. Selected for reprint in Haux R, Kulikowski C (eds). Yearbook of Medical Informatics. Rotterdam: International Medical Informatics Association, 2001, pp 271&#x02013;285. </div></div><div class="ref-cit-blk" id="bib23"><div class="ref-label">23.</div><div class="ref-cit">Peng Y, Halper M, Perl Y, Geller J. Auditing the UMLS for redundant classifications.  <span class="ref-journal">Proc AMIA Annu Symp.</span> 2002:612&#x02013;6. </div></div><div class="ref-cit-blk" id="bib24"><div class="ref-label">24.</div><div class="ref-cit">Perl Y, Chen Z, Halper M, Geller J, Zhang L, Peng Y. The cohesive metaschema: a higher-level abstraction of the UMLS Semantic Network.  <span class="ref-journal">J Biomed Inform.</span> 2003;<span class="ref-vol">35</span>:194&#x02013;212. </div></div></div></div></td></tr><tr><td></td><td class="content-cell"><div class="footer-section"><hr><div class="sans90" style="margin: 0em 1em 1em; text-align: center;">Articles from <span class="acknowledgment-journal-title">Journal of the American Medical Informatics Association : JAMIA</span> are provided here courtesy of <br><b>American Medical Informatics Association</b></div>
      <div class="footer-link" id="fp-footer">
        <div>
          <a class="link" href="mailto:pubmedcentral@nih.gov">Write to PMC</a>
           | 
          <a class="link" href="./">PMC Home</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/pubmed">PubMed</a>
        </div>
        <div>
          <a class="link" href="http://www.ncbi.nlm.nih.gov">NCBI</a>
           | 
          <a class="link" href="http://www.nlm.nih.gov">U.S. National Library of Medicine</a>
        </div>
        <div>
          <a class="link" href="http://www.nih.gov">NIH</a>
           | 
          <a class="link" href="http://www.hhs.gov/">Department of Health and Human Services</a>
        </div>
        <div>
          <a class="link" href="about/privacypolicy.html">Privacy Policy</a>
           | 
          <a class="link" href="http://www.ncbi.nlm.nih.gov/About/disclaimer.html">Disclaimer</a>
           | 
          <a class="link" href="http://www.nih.gov/icd/od/foia/index.htm">Freedom of Information Act</a>
        </div>
      </div>
    </div></td></tr></table></body></html><!--host:ipmc1-->

